// @composio/core/generated/active_campaign.ts
/**
 * Map of Composio's ACTIVE_CAMPAIGN toolkit.
 */
export const ACTIVE_CAMPAIGN = {
  slug: "active_campaign",
  tools: {
    ADD_CONTACT_NOTE: "ACTIVE_CAMPAIGN_ADD_CONTACT_NOTE",
    ADD_CONTACT_TO_AUTOMATION: "ACTIVE_CAMPAIGN_ADD_CONTACT_TO_AUTOMATION",
    ADD_SECONDARY_CONTACT_TO_DEAL: "ACTIVE_CAMPAIGN_ADD_SECONDARY_CONTACT_TO_DEAL",
    CREATE_ACCOUNT: "ACTIVE_CAMPAIGN_CREATE_ACCOUNT",
    CREATE_CONTACT: "ACTIVE_CAMPAIGN_CREATE_CONTACT",
    CREATE_CONTACT_TASK: "ACTIVE_CAMPAIGN_CREATE_CONTACT_TASK",
    CREATE_DEAL_PIPELINE: "ACTIVE_CAMPAIGN_CREATE_DEAL_PIPELINE",
    CREATE_DEAL_TASK_TYPE: "ACTIVE_CAMPAIGN_CREATE_DEAL_TASK_TYPE",
    DELETE_ACCOUNT: "ACTIVE_CAMPAIGN_DELETE_ACCOUNT",
    DELETE_DEAL_PIPELINE: "ACTIVE_CAMPAIGN_DELETE_DEAL_PIPELINE",
    DELETE_DEAL_STAGE: "ACTIVE_CAMPAIGN_DELETE_DEAL_STAGE",
    FIND_CONTACT: "ACTIVE_CAMPAIGN_FIND_CONTACT",
    FIND_CONTACT_TASK: "ACTIVE_CAMPAIGN_FIND_CONTACT_TASK",
    FIND_USER_ACTION: "ACTIVE_CAMPAIGN_FIND_USER_ACTION",
    LIST_ALL_CONTACTS: "ACTIVE_CAMPAIGN_LIST_ALL_CONTACTS",
    MANAGE_CONTACT_TAG: "ACTIVE_CAMPAIGN_MANAGE_CONTACT_TAG",
    MANAGE_LIST_SUBSCRIPTION: "ACTIVE_CAMPAIGN_MANAGE_LIST_SUBSCRIPTION",
    REMOVE_CONTACT_FROM_AUTOMATION: "ACTIVE_CAMPAIGN_REMOVE_CONTACT_FROM_AUTOMATION",
    TRACK_EVENT: "ACTIVE_CAMPAIGN_TRACK_EVENT",
    UPDATE_ACCOUNT: "ACTIVE_CAMPAIGN_UPDATE_ACCOUNT",
    UPSERT_ACCOUNT: "ACTIVE_CAMPAIGN_UPSERT_ACCOUNT",
  },
  triggerTypes: {},
}
