// @composio/core/generated/affinity.ts
/**
 * Map of Composio's AFFINITY toolkit.
 */
export const AFFINITY = {
  slug: "affinity",
  tools: {
    GET_ALL_COMPANIES: "AFFINITY_GET_ALL_COMPANIES",
    GET_ALL_LIST_ENTRIES_ON_A_LIST: "AFFINITY_GET_ALL_LIST_ENTRIES_ON_A_LIST",
    GET_ALL_LIST_ENTRIES_ON_A_SAVED_VIEW: "AFFINITY_GET_ALL_LIST_ENTRIES_ON_A_SAVED_VIEW",
    GET_ALL_OPPORTUNITIES: "AFFINITY_GET_ALL_OPPORTUNITIES",
    GET_ALL_PERSONS: "AFFINITY_GET_ALL_PERSONS",
    GET_A_COMPANY_S_LISTS: "AFFINITY_GET_A_COMPANY_S_LISTS",
    GET_A_COMPANY_S_LIST_ENTRIES: "AFFINITY_GET_A_COMPANY_S_LIST_ENTRIES",
    GET_A_PERSON_S_LISTS: "AFFINITY_GET_A_PERSON_S_LISTS",
    GET_A_PERSON_S_LIST_ENTRIES: "AFFINITY_GET_A_PERSON_S_LIST_ENTRIES",
    GET_A_SINGLE_COMPANY: "AFFINITY_GET_A_SINGLE_COMPANY",
    GET_A_SINGLE_OPPORTUNITY: "AFFINITY_GET_A_SINGLE_OPPORTUNITY",
    GET_A_SINGLE_PERSON: "AFFINITY_GET_A_SINGLE_PERSON",
    GET_CURRENT_USER: "AFFINITY_GET_CURRENT_USER",
    GET_METADATA_ON_ALL_LISTS: "AFFINITY_GET_METADATA_ON_ALL_LISTS",
    GET_METADATA_ON_A_SINGLE_LIST: "AFFINITY_GET_METADATA_ON_A_SINGLE_LIST",
    GET_METADATA_ON_A_SINGLE_LIST_S_FIELDS: "AFFINITY_GET_METADATA_ON_A_SINGLE_LIST_S_FIELDS",
    GET_METADATA_ON_A_SINGLE_SAVED_VIEW: "AFFINITY_GET_METADATA_ON_A_SINGLE_SAVED_VIEW",
    GET_METADATA_ON_COMPANY_FIELDS: "AFFINITY_GET_METADATA_ON_COMPANY_FIELDS",
    GET_METADATA_ON_PERSON_FIELDS: "AFFINITY_GET_METADATA_ON_PERSON_FIELDS",
    GET_METADATA_ON_SAVED_VIEWS: "AFFINITY_GET_METADATA_ON_SAVED_VIEWS",
  },
  triggerTypes: {},
}
