// @composio/core/generated/agencyzoom.ts
/**
 * Map of Composio's AGENCYZOOM toolkit.
 */
export const AGENCYZOOM = {
  slug: "agencyzoom",
  tools: {
    AUTHENTICATE_FOR_JWTVIA_V4_SSO: "AGENCYZOOM_AUTHENTICATE_FOR_JWTVIA_V4_SSO",
    BATCH_CREATE_CONTACT: "AGENCYZOOM_BATCH_CREATE_CONTACT",
    BATCH_CREATE_LEAD: "AGENCYZOOM_BATCH_CREATE_LEAD",
    BATCH_DELETE_TASK: "AGENCYZOOM_BATCH_DELETE_TASK",
    CHANGE_STATUS_FOR_LEAD: "AGENCYZOOM_CHANGE_STATUS_FOR_LEAD",
    COMPLETE_TASK: "AGENCYZOOM_COMPLETE_TASK",
    CREATE_AN_OPPORTUNITY: "AGENCYZOOM_CREATE_AN_OPPORTUNITY",
    CREATE_A_CUSTOMER_NOTE: "AGENCYZOOM_CREATE_A_CUSTOMER_NOTE",
    CREATE_A_DRIVER_FOR_AN_OPPORTUNITY: "AGENCYZOOM_CREATE_A_DRIVER_FOR_AN_OPPORTUNITY",
    CREATE_A_LEAD_NOTE: "AGENCYZOOM_CREATE_A_LEAD_NOTE",
    CREATE_A_LEAD_OPPORTUNITY: "AGENCYZOOM_CREATE_A_LEAD_OPPORTUNITY",
    CREATE_A_LEAD_QUOTE: "AGENCYZOOM_CREATE_A_LEAD_QUOTE",
    CREATE_A_VEHICLE_FOR_AN_OPPORTUNITY: "AGENCYZOOM_CREATE_A_VEHICLE_FOR_AN_OPPORTUNITY",
    CREATE_BIZ_LEAD: "AGENCYZOOM_CREATE_BIZ_LEAD",
    CREATE_LEAD: "AGENCYZOOM_CREATE_LEAD",
    CREATE_TASK: "AGENCYZOOM_CREATE_TASK",
    DELETE_AN_OPPORTUNITY: "AGENCYZOOM_DELETE_AN_OPPORTUNITY",
    DELETE_A_CUSTOMER: "AGENCYZOOM_DELETE_A_CUSTOMER",
    DELETE_A_CUSTOMER_FILE: "AGENCYZOOM_DELETE_A_CUSTOMER_FILE",
    DELETE_A_CUSTOMER_POLICY: "AGENCYZOOM_DELETE_A_CUSTOMER_POLICY",
    DELETE_A_DRIVER: "AGENCYZOOM_DELETE_A_DRIVER",
    DELETE_A_LEAD_FILE: "AGENCYZOOM_DELETE_A_LEAD_FILE",
    DELETE_A_LEAD_OPPORTUNITY: "AGENCYZOOM_DELETE_A_LEAD_OPPORTUNITY",
    DELETE_A_LEAD_QUOTE: "AGENCYZOOM_DELETE_A_LEAD_QUOTE",
    DELETE_A_TASK: "AGENCYZOOM_DELETE_A_TASK",
    DELETE_A_VEHICLE: "AGENCYZOOM_DELETE_A_VEHICLE",
    DELETE_MESSAGE: "AGENCYZOOM_DELETE_MESSAGE",
    DELETE_THREAD: "AGENCYZOOM_DELETE_THREAD",
    GET_AMS_POLICIES_FOR_A_CUSTOMER: "AGENCYZOOM_GET_AMS_POLICIES_FOR_A_CUSTOMER",
    GET_AUTH_URL_FOR_V4SSO: "AGENCYZOOM_GET_AUTH_URL_FOR_V4SSO",
    GET_A_LIST_OF_ASSIGN_GROUPS: "AGENCYZOOM_GET_A_LIST_OF_ASSIGN_GROUPS",
    GET_A_LIST_OF_CARRIERS: "AGENCYZOOM_GET_A_LIST_OF_CARRIERS",
    GET_A_LIST_OF_CSRS: "AGENCYZOOM_GET_A_LIST_OF_CSRS",
    GET_A_LIST_OF_CUSTOM_FIELDS: "AGENCYZOOM_GET_A_LIST_OF_CUSTOM_FIELDS",
    GET_A_LIST_OF_DRIVERS_FOR_AN_OPPORTUNITY: "AGENCYZOOM_GET_A_LIST_OF_DRIVERS_FOR_AN_OPPORTUNITY",
    GET_A_LIST_OF_EMPLOYEES: "AGENCYZOOM_GET_A_LIST_OF_EMPLOYEES",
    GET_A_LIST_OF_LEAD_SOURCES: "AGENCYZOOM_GET_A_LIST_OF_LEAD_SOURCES",
    GET_A_LIST_OF_LEAD_SOURCE_CATEGORIES: "AGENCYZOOM_GET_A_LIST_OF_LEAD_SOURCE_CATEGORIES",
    GET_A_LIST_OF_LIFE_PROFESSIONALS: "AGENCYZOOM_GET_A_LIST_OF_LIFE_PROFESSIONALS",
    GET_A_LIST_OF_LOCATIONS: "AGENCYZOOM_GET_A_LIST_OF_LOCATIONS",
    GET_A_LIST_OF_LOSS_REASONS: "AGENCYZOOM_GET_A_LIST_OF_LOSS_REASONS",
    GET_A_LIST_OF_PIPELINES: "AGENCYZOOM_GET_A_LIST_OF_PIPELINES",
    GET_A_LIST_OF_PRODUCER: "AGENCYZOOM_GET_A_LIST_OF_PRODUCER",
    GET_A_LIST_OF_PRODUCT_LINES_POLICY_TYPES: "AGENCYZOOM_GET_A_LIST_OF_PRODUCT_LINES_POLICY_TYPES",
    GET_A_LIST_OF_RECYCLE_EVENTS: "AGENCYZOOM_GET_A_LIST_OF_RECYCLE_EVENTS",
    GET_A_LIST_OF_VEHICLES_FOR_AN_OPPORTUNITY: "AGENCYZOOM_GET_A_LIST_OF_VEHICLES_FOR_AN_OPPORTUNITY",
    GET_DEPARTMENTS_GROUPS: "AGENCYZOOM_GET_DEPARTMENTS_GROUPS",
    GET_LEAD_FILES: "AGENCYZOOM_GET_LEAD_FILES",
    GET_LEAD_NOTES: "AGENCYZOOM_GET_LEAD_NOTES",
    GET_LEAD_QUOTES: "AGENCYZOOM_GET_LEAD_QUOTES",
    GET_LEAD_TASKS: "AGENCYZOOM_GET_LEAD_TASKS",
    GET_LIST_OF_END_STAGES: "AGENCYZOOM_GET_LIST_OF_END_STAGES",
    GET_POLICIES_FOR_A_CUSTOMER: "AGENCYZOOM_GET_POLICIES_FOR_A_CUSTOMER",
    GET_THE_CUSTOMER_DETAILS: "AGENCYZOOM_GET_THE_CUSTOMER_DETAILS",
    GET_THE_CUSTOMER_TASKS: "AGENCYZOOM_GET_THE_CUSTOMER_TASKS",
    GET_THE_DRIVER_DETAILS: "AGENCYZOOM_GET_THE_DRIVER_DETAILS",
    GET_THE_LEAD_DETAILS: "AGENCYZOOM_GET_THE_LEAD_DETAILS",
    GET_THE_OPPORTUNITIES_FOR_A_LEAD: "AGENCYZOOM_GET_THE_OPPORTUNITIES_FOR_A_LEAD",
    GET_THE_OPPORTUNITY_DETAILS: "AGENCYZOOM_GET_THE_OPPORTUNITY_DETAILS",
    GET_THE_TASK_DETAILS: "AGENCYZOOM_GET_THE_TASK_DETAILS",
    GET_THE_VEHICLE_DETAILS: "AGENCYZOOM_GET_THE_VEHICLE_DETAILS",
    GET_THREAD_DETAILS: "AGENCYZOOM_GET_THREAD_DETAILS",
    LINK_A_DRIVER_TO_OPPORTUNITY: "AGENCYZOOM_LINK_A_DRIVER_TO_OPPORTUNITY",
    LINK_A_VEHICLE_TO_OPPORTUNITY: "AGENCYZOOM_LINK_A_VEHICLE_TO_OPPORTUNITY",
    LIST_PRODUCT_CATEGORIES: "AGENCYZOOM_LIST_PRODUCT_CATEGORIES",
    LOG_THE_USER_IN: "AGENCYZOOM_LOG_THE_USER_IN",
    LOG_THE_USER_OUT: "AGENCYZOOM_LOG_THE_USER_OUT",
    MARK_THREAD_AS_UNREAD_API_ENDPOINT: "AGENCYZOOM_MARK_THREAD_AS_UNREAD_API_ENDPOINT",
    MOVE_LEAD_TO_SOLD: "AGENCYZOOM_MOVE_LEAD_TO_SOLD",
    REMOVE_TEXT_THREAD_ENDPOINT: "AGENCYZOOM_REMOVE_TEXT_THREAD_ENDPOINT",
    REOPEN_A_TASK: "AGENCYZOOM_REOPEN_A_TASK",
    SEARCH_BUSINESS_CLASSIFICATIONS: "AGENCYZOOM_SEARCH_BUSINESS_CLASSIFICATIONS",
    SEARCH_CUSTOMERS: "AGENCYZOOM_SEARCH_CUSTOMERS",
    SEARCH_EMAIL_THREADS: "AGENCYZOOM_SEARCH_EMAIL_THREADS",
    SEARCH_LEADS: "AGENCYZOOM_SEARCH_LEADS",
    SEARCH_LEADS_COUNT: "AGENCYZOOM_SEARCH_LEADS_COUNT",
    SEARCH_LIFE_AND_HEALTH_LEADS: "AGENCYZOOM_SEARCH_LIFE_AND_HEALTH_LEADS",
    SEARCH_SMS_THREADS: "AGENCYZOOM_SEARCH_SMS_THREADS",
    SEARCH_TASKS: "AGENCYZOOM_SEARCH_TASKS",
    SERVICE_TICKET_LIST: "AGENCYZOOM_SERVICE_TICKET_LIST",
    TEXT_DETAIL_THREAD: "AGENCYZOOM_TEXT_DETAIL_THREAD",
    UNLINK_A_DRIVER_FROM_OPPORTUNITY: "AGENCYZOOM_UNLINK_A_DRIVER_FROM_OPPORTUNITY",
    UNLINK_A_VEHICLE_FROM_OPPORTUNITY: "AGENCYZOOM_UNLINK_A_VEHICLE_FROM_OPPORTUNITY",
    UNREAD_THREAD: "AGENCYZOOM_UNREAD_THREAD",
    UPDATE_AN_OPPORTUNITY: "AGENCYZOOM_UPDATE_AN_OPPORTUNITY",
    UPDATE_A_DRIVER_S_DETAILS: "AGENCYZOOM_UPDATE_A_DRIVER_S_DETAILS",
    UPDATE_A_LEAD_FILE_NAME: "AGENCYZOOM_UPDATE_A_LEAD_FILE_NAME",
    UPDATE_A_LEAD_OPPORTUNITY: "AGENCYZOOM_UPDATE_A_LEAD_OPPORTUNITY",
    UPDATE_A_LEAD_QUOTE: "AGENCYZOOM_UPDATE_A_LEAD_QUOTE",
    UPDATE_A_POLICY: "AGENCYZOOM_UPDATE_A_POLICY",
    UPDATE_A_VEHICLE_S_DETAILS: "AGENCYZOOM_UPDATE_A_VEHICLE_S_DETAILS",
    UPDATE_BUSINESS_LEAD: "AGENCYZOOM_UPDATE_BUSINESS_LEAD",
    UPDATE_CUSTOMER: "AGENCYZOOM_UPDATE_CUSTOMER",
    UPDATE_LEAD: "AGENCYZOOM_UPDATE_LEAD",
    UPDATE_LEAD_STATUS_BY_ID: "AGENCYZOOM_UPDATE_LEAD_STATUS_BY_ID",
    UPDATE_MY_PROFILE: "AGENCYZOOM_UPDATE_MY_PROFILE",
    UPDATE_TAGS_FOR_A_POLICY: "AGENCYZOOM_UPDATE_TAGS_FOR_A_POLICY",
    UPDATE_TASK: "AGENCYZOOM_UPDATE_TASK",
    V4_SSO_LOG_THE_USER_IN: "AGENCYZOOM_V4_SSO_LOG_THE_USER_IN",
  },
  triggerTypes: {},
}
