// @composio/core/generated/ahrefs.ts
/**
 * Map of Composio's AHREFS toolkit.
 */
export const AHREFS = {
  slug: "ahrefs",
  tools: {
    BACKLINKS_STATS_RETRIEVAL: "AHREFS_BACKLINKS_STATS_RETRIEVAL",
    BATCH_URL_ANALYSIS: "AHREFS_BATCH_URL_ANALYSIS",
    DOMAIN_RATING_FOR_SITE_EXPLORER: "AHREFS_DOMAIN_RATING_FOR_SITE_EXPLORER",
    DOMAIN_RATING_HISTORY: "AHREFS_DOMAIN_RATING_HISTORY",
    EXPLORE_KEYWORDS_OVERVIEW: "AHREFS_EXPLORE_KEYWORDS_OVERVIEW",
    EXPLORE_KEYWORD_VOLUME_BY_COUNTRY: "AHREFS_EXPLORE_KEYWORD_VOLUME_BY_COUNTRY",
    EXPLORE_LINKED_DOMAINS_OF_A_SITE: "AHREFS_EXPLORE_LINKED_DOMAINS_OF_A_SITE",
    EXPLORE_MATCHING_TERMS_FOR_KEYWORDS: "AHREFS_EXPLORE_MATCHING_TERMS_FOR_KEYWORDS",
    FETCH_ALL_BACKLINKS: "AHREFS_FETCH_ALL_BACKLINKS",
    FETCH_BROKEN_BACKLINKS_DATA: "AHREFS_FETCH_BROKEN_BACKLINKS_DATA",
    FETCH_COMPETITORS_OVERVIEW: "AHREFS_FETCH_COMPETITORS_OVERVIEW",
    FETCH_RANK_TRACKER_OVERVIEW: "AHREFS_FETCH_RANK_TRACKER_OVERVIEW",
    FETCH_SITE_EXPLORER_REFERRING_DOMAINS: "AHREFS_FETCH_SITE_EXPLORER_REFERRING_DOMAINS",
    FETCH_TOTAL_SEARCH_VOLUME_HISTORY: "AHREFS_FETCH_TOTAL_SEARCH_VOLUME_HISTORY",
    GET_SERP_OVERVIEW: "AHREFS_GET_SERP_OVERVIEW",
    GET_SITE_AUDIT_PROJECTS: "AHREFS_GET_SITE_AUDIT_PROJECTS",
    GET_SITE_EXPLORER_COUNTRY_METRICS: "AHREFS_GET_SITE_EXPLORER_COUNTRY_METRICS",
    GET_SITE_EXPLORER_LINKED_ANCHORS_EXTERNAL: "AHREFS_GET_SITE_EXPLORER_LINKED_ANCHORS_EXTERNAL",
    GET_URL_RATING_HISTORY: "AHREFS_GET_URL_RATING_HISTORY",
    LINKED_ANCHORS_EXPLORER: "AHREFS_LINKED_ANCHORS_EXPLORER",
    LIST_BEST_BY_EXTERNAL_LINKS: "AHREFS_LIST_BEST_BY_EXTERNAL_LINKS",
    PAGES_BY_TRAFFIC_OVERVIEW: "AHREFS_PAGES_BY_TRAFFIC_OVERVIEW",
    RETRIEVE_ANCHOR_DATA: "AHREFS_RETRIEVE_ANCHOR_DATA",
    RETRIEVE_BEST_BY_INTERNAL_LINKS: "AHREFS_RETRIEVE_BEST_BY_INTERNAL_LINKS",
    RETRIEVE_CRAWLER_IP_RANGES: "AHREFS_RETRIEVE_CRAWLER_IP_RANGES",
    RETRIEVE_ORGANIC_COMPETITORS: "AHREFS_RETRIEVE_ORGANIC_COMPETITORS",
    RETRIEVE_ORGANIC_KEYWORDS: "AHREFS_RETRIEVE_ORGANIC_KEYWORDS",
    RETRIEVE_OUTLINKS_STATS: "AHREFS_RETRIEVE_OUTLINKS_STATS",
    RETRIEVE_PAID_PAGES_DATA: "AHREFS_RETRIEVE_PAID_PAGES_DATA",
    RETRIEVE_PUBLIC_CRAWLER_IPS: "AHREFS_RETRIEVE_PUBLIC_CRAWLER_IPS",
    RETRIEVE_RELATED_TERMS: "AHREFS_RETRIEVE_RELATED_TERMS",
    RETRIEVE_SITE_EXPLORER_KEYWORDS_HISTORY: "AHREFS_RETRIEVE_SITE_EXPLORER_KEYWORDS_HISTORY",
    RETRIEVE_SITE_EXPLORER_METRICS: "AHREFS_RETRIEVE_SITE_EXPLORER_METRICS",
    RETRIEVE_SITE_EXPLORER_METRICS_HISTORY: "AHREFS_RETRIEVE_SITE_EXPLORER_METRICS_HISTORY",
    RETRIEVE_SITE_EXPLORER_PAGES_HISTORY: "AHREFS_RETRIEVE_SITE_EXPLORER_PAGES_HISTORY",
    RETRIEVE_SITE_EXPLORER_REFERRING_DOMAINS_HISTORY: "AHREFS_RETRIEVE_SITE_EXPLORER_REFERRING_DOMAINS_HISTORY",
    RETRIEVE_SUBSCRIPTION_LIMITS_AND_USAGE: "AHREFS_RETRIEVE_SUBSCRIPTION_LIMITS_AND_USAGE",
    RETRIEVE_TOP_PAGES_FROM_SITE_EXPLORER: "AHREFS_RETRIEVE_TOP_PAGES_FROM_SITE_EXPLORER",
    RETRIEVE_VOLUME_HISTORY: "AHREFS_RETRIEVE_VOLUME_HISTORY",
    SEARCH_SUGGESTIONS_EXPLORER: "AHREFS_SEARCH_SUGGESTIONS_EXPLORER",
  },
  triggerTypes: {},
}
