// @composio/core/generated/algolia.ts
/**
 * Map of Composio's ALGOLIA toolkit.
 */
export const ALGOLIA = {
  slug: "algolia",
  tools: {
    ADD_AB_TEST: "ALGOLIA_ADD_AB_TEST",
    BROWSE_INDEX: "ALGOLIA_BROWSE_INDEX",
    CLEAR_OBJECTS: "ALGOLIA_CLEAR_OBJECTS",
    CLEAR_RULES: "ALGOLIA_CLEAR_RULES",
    CLICKED_OBJECT_IDS: "ALGOLIA_CLICKED_OBJECT_IDS",
    CLICKED_OBJECT_IDS_AFTER_SEARCH: "ALGOLIA_CLICKED_OBJECT_IDS_AFTER_SEARCH",
    CONVERTED_OBJECT_IDS: "ALGOLIA_CONVERTED_OBJECT_IDS",
    COPY_INDEX: "ALGOLIA_COPY_INDEX",
    COPY_RULES: "ALGOLIA_COPY_RULES",
    COPY_SETTINGS: "ALGOLIA_COPY_SETTINGS",
    DELETE_INDEX: "ALGOLIA_DELETE_INDEX",
    DELETE_OBJECTS: "ALGOLIA_DELETE_OBJECTS",
    DELETE_RULE: "ALGOLIA_DELETE_RULE",
    DELETE_SYNONYM: "ALGOLIA_DELETE_SYNONYM",
    EXPORT_RULES: "ALGOLIA_EXPORT_RULES",
    FIND_OBJECT: "ALGOLIA_FIND_OBJECT",
    GET_OBJECTS: "ALGOLIA_GET_OBJECTS",
    GET_OBJECT_POSITION: "ALGOLIA_GET_OBJECT_POSITION",
    GET_SETTINGS: "ALGOLIA_GET_SETTINGS",
    INDEX_EXISTS: "ALGOLIA_INDEX_EXISTS",
    INIT_INSIGHTS: "ALGOLIA_INIT_INSIGHTS",
    LIST_INDICES: "ALGOLIA_LIST_INDICES",
    PARTIAL_UPDATE_OBJECTS: "ALGOLIA_PARTIAL_UPDATE_OBJECTS",
    REPLACE_ALL_RULES: "ALGOLIA_REPLACE_ALL_RULES",
    SAVE_SYNONYM: "ALGOLIA_SAVE_SYNONYM",
    SEARCH_INDEX: "ALGOLIA_SEARCH_INDEX",
    SEARCH_MULTIPLE_INDICES: "ALGOLIA_SEARCH_MULTIPLE_INDICES",
    SEARCH_RULES: "ALGOLIA_SEARCH_RULES",
    SEARCH_SYNONYMS: "ALGOLIA_SEARCH_SYNONYMS",
    SET_SETTINGS: "ALGOLIA_SET_SETTINGS",
  },
  triggerTypes: {},
}
