// @composio/core/generated/apaleo.ts
/**
 * Map of Composio's APALEO toolkit.
 */
export const APALEO = {
  slug: "apaleo",
  tools: {
    ARCHIVE_A_PROPERTY: "APALEO_ARCHIVE_A_PROPERTY",
    CHECK_IF_A_PROPERTY_EXISTS: "APALEO_CHECK_IF_A_PROPERTY_EXISTS",
    CHECK_IF_A_UNIT_ATTRIBUTE_EXISTS: "APALEO_CHECK_IF_A_UNIT_ATTRIBUTE_EXISTS",
    CHECK_IF_A_UNIT_EXISTS: "APALEO_CHECK_IF_A_UNIT_EXISTS",
    CHECK_IF_A_UNIT_GROUP_EXISTS: "APALEO_CHECK_IF_A_UNIT_GROUP_EXISTS",
    CLONES_A_PROPERTY: "APALEO_CLONES_A_PROPERTY",
    CREATES_A_PROPERTY: "APALEO_CREATES_A_PROPERTY",
    CREATE_A_UNIT: "APALEO_CREATE_A_UNIT",
    CREATE_A_UNIT_ATTRIBUTE: "APALEO_CREATE_A_UNIT_ATTRIBUTE",
    CREATE_A_UNIT_GROUP: "APALEO_CREATE_A_UNIT_GROUP",
    CREATE_MULTIPLE_UNITS: "APALEO_CREATE_MULTIPLE_UNITS",
    DELETES_UNIT_ATTRIBUTE: "APALEO_DELETES_UNIT_ATTRIBUTE",
    DELETE_A_UNIT: "APALEO_DELETE_A_UNIT",
    DELETE_A_UNIT_GROUP: "APALEO_DELETE_A_UNIT_GROUP",
    GET_A_PROPERTIES_LIST: "APALEO_GET_A_PROPERTIES_LIST",
    GET_A_PROPERTY: "APALEO_GET_A_PROPERTY",
    GET_A_UNIT: "APALEO_GET_A_UNIT",
    GET_A_UNITS_LIST: "APALEO_GET_A_UNITS_LIST",
    GET_A_UNIT_GROUP: "APALEO_GET_A_UNIT_GROUP",
    GET_UNIT_ATTRIBUTE_BY_ID: "APALEO_GET_UNIT_ATTRIBUTE_BY_ID",
    GET_UNIT_ATTRIBUTE_LIST: "APALEO_GET_UNIT_ATTRIBUTE_LIST",
    LIST_UNIT_GROUPS: "APALEO_LIST_UNIT_GROUPS",
    MOVE_PROPERTY_TO_LIVE: "APALEO_MOVE_PROPERTY_TO_LIVE",
    REPLACE_A_UNIT_GROUP: "APALEO_REPLACE_A_UNIT_GROUP",
    RESET_PROPERTY_DATA: "APALEO_RESET_PROPERTY_DATA",
    RETURNS_A_LIST_OF_SUPPORTED_COUNTRIES: "APALEO_RETURNS_A_LIST_OF_SUPPORTED_COUNTRIES",
    RETURNS_NUMBER_OF_UNITS: "APALEO_RETURNS_NUMBER_OF_UNITS",
    RETURNS_NUMBER_OF_UNIT_GROUPS: "APALEO_RETURNS_NUMBER_OF_UNIT_GROUPS",
    RETURN_TOTAL_COUNT_OF_PROPERTIES: "APALEO_RETURN_TOTAL_COUNT_OF_PROPERTIES",
  },
  triggerTypes: {},
}
