// @composio/core/generated/apollo.ts
/**
 * Map of Composio's APOLLO toolkit.
 */
export const APOLLO = {
  slug: "apollo",
  tools: {
    ADD_CONTACTS_TO_SEQUENCE: "APOLLO_ADD_CONTACTS_TO_SEQUENCE",
    BULK_ORGANIZATION_ENRICHMENT: "APOLLO_BULK_ORGANIZATION_ENRICHMENT",
    BULK_PEOPLE_ENRICHMENT: "APOLLO_BULK_PEOPLE_ENRICHMENT",
    BULK_UPDATE_ACCOUNT_STAGE: "APOLLO_BULK_UPDATE_ACCOUNT_STAGE",
    CREATE_ACCOUNT: "APOLLO_CREATE_ACCOUNT",
    CREATE_CONTACT: "APOLLO_CREATE_CONTACT",
    CREATE_DEAL: "APOLLO_CREATE_DEAL",
    CREATE_TASK: "APOLLO_CREATE_TASK",
    GET_LABELS: "APOLLO_GET_LABELS",
    GET_OPPORTUNITY_STAGES: "APOLLO_GET_OPPORTUNITY_STAGES",
    GET_ORGANIZATION_JOB_POSTINGS: "APOLLO_GET_ORGANIZATION_JOB_POSTINGS",
    GET_TYPED_CUSTOM_FIELDS: "APOLLO_GET_TYPED_CUSTOM_FIELDS",
    LIST_ACCOUNT_STAGES: "APOLLO_LIST_ACCOUNT_STAGES",
    LIST_CONTACT_STAGES: "APOLLO_LIST_CONTACT_STAGES",
    LIST_DEALS: "APOLLO_LIST_DEALS",
    LIST_EMAIL_ACCOUNTS: "APOLLO_LIST_EMAIL_ACCOUNTS",
    LIST_USERS: "APOLLO_LIST_USERS",
    ORGANIZATION_ENRICHMENT: "APOLLO_ORGANIZATION_ENRICHMENT",
    ORGANIZATION_SEARCH: "APOLLO_ORGANIZATION_SEARCH",
    PEOPLE_ENRICHMENT: "APOLLO_PEOPLE_ENRICHMENT",
    PEOPLE_SEARCH: "APOLLO_PEOPLE_SEARCH",
    SEARCH_ACCOUNTS: "APOLLO_SEARCH_ACCOUNTS",
    SEARCH_CONTACTS: "APOLLO_SEARCH_CONTACTS",
    SEARCH_SEQUENCES: "APOLLO_SEARCH_SEQUENCES",
    SEARCH_TASKS: "APOLLO_SEARCH_TASKS",
    UPDATE_ACCOUNT: "APOLLO_UPDATE_ACCOUNT",
    UPDATE_CONTACT: "APOLLO_UPDATE_CONTACT",
    UPDATE_CONTACT_OWNERSHIP: "APOLLO_UPDATE_CONTACT_OWNERSHIP",
    UPDATE_CONTACT_STAGE: "APOLLO_UPDATE_CONTACT_STAGE",
    UPDATE_CONTACT_STATUS_IN_SEQUENCE: "APOLLO_UPDATE_CONTACT_STATUS_IN_SEQUENCE",
    UPDATE_DEALS: "APOLLO_UPDATE_DEALS",
  },
  triggerTypes: {},
}
