// @composio/core/generated/asana.ts
/**
 * Map of Composio's ASANA toolkit.
 */
export const ASANA = {
  slug: "asana",
  tools: {
    ADD_FOLLOWERS_TO_TASK: "ASANA_ADD_FOLLOWERS_TO_TASK",
    ADD_SUPPORTING_RELATIONSHIP: "ASANA_ADD_SUPPORTING_RELATIONSHIP",
    ADD_TASK_TO_SECTION: "ASANA_ADD_TASK_TO_SECTION",
    CREATE_ALLOCATION: "ASANA_CREATE_ALLOCATION",
    CREATE_ATTACHMENT_FOR_TASK: "ASANA_CREATE_ATTACHMENT_FOR_TASK",
    CREATE_A_PROJECT: "ASANA_CREATE_A_PROJECT",
    CREATE_A_TAG_IN_A_WORKSPACE: "ASANA_CREATE_A_TAG_IN_A_WORKSPACE",
    CREATE_A_TASK: "ASANA_CREATE_A_TASK",
    CREATE_CUSTOM_FIELD: "ASANA_CREATE_CUSTOM_FIELD",
    CREATE_ENUM_OPTION_FOR_CUSTOM_FIELD: "ASANA_CREATE_ENUM_OPTION_FOR_CUSTOM_FIELD",
    CREATE_PROJECT_STATUS_UPDATE: "ASANA_CREATE_PROJECT_STATUS_UPDATE",
    CREATE_SECTION_IN_PROJECT: "ASANA_CREATE_SECTION_IN_PROJECT",
    CREATE_SUBTASK: "ASANA_CREATE_SUBTASK",
    CREATE_TASK_COMMENT: "ASANA_CREATE_TASK_COMMENT",
    CREATE_TEAM: "ASANA_CREATE_TEAM",
    DELETE_ALLOCATION: "ASANA_DELETE_ALLOCATION",
    DELETE_ATTACHMENT: "ASANA_DELETE_ATTACHMENT",
    DELETE_CUSTOM_FIELD: "ASANA_DELETE_CUSTOM_FIELD",
    DELETE_PROJECT: "ASANA_DELETE_PROJECT",
    DELETE_TAG: "ASANA_DELETE_TAG",
    DELETE_TASK: "ASANA_DELETE_TASK",
    DUPLICATE_PROJECT: "ASANA_DUPLICATE_PROJECT",
    DUPLICATE_TASK: "ASANA_DUPLICATE_TASK",
    GET_ALLOCATION: "ASANA_GET_ALLOCATION",
    GET_ALLOCATIONS: "ASANA_GET_ALLOCATIONS",
    GET_ATTACHMENT: "ASANA_GET_ATTACHMENT",
    GET_AUDIT_LOG_EVENTS: "ASANA_GET_AUDIT_LOG_EVENTS",
    GET_A_PROJECT: "ASANA_GET_A_PROJECT",
    GET_A_TASK: "ASANA_GET_A_TASK",
    GET_A_USER_TASK_LIST: "ASANA_GET_A_USER_TASK_LIST",
    GET_CURRENT_USER: "ASANA_GET_CURRENT_USER",
    GET_CUSTOM_FIELD: "ASANA_GET_CUSTOM_FIELD",
    GET_CUSTOM_FIELDS_FOR_WORKSPACE: "ASANA_GET_CUSTOM_FIELDS_FOR_WORKSPACE",
    GET_EVENTS: "ASANA_GET_EVENTS",
    GET_GOAL: "ASANA_GET_GOAL",
    GET_GOALS: "ASANA_GET_GOALS",
    GET_GOAL_RELATIONSHIPS: "ASANA_GET_GOAL_RELATIONSHIPS",
    GET_MEMBERSHIPS: "ASANA_GET_MEMBERSHIPS",
    GET_MULTIPLE_PROJECTS: "ASANA_GET_MULTIPLE_PROJECTS",
    GET_MULTIPLE_TASKS: "ASANA_GET_MULTIPLE_TASKS",
    GET_MULTIPLE_USERS: "ASANA_GET_MULTIPLE_USERS",
    GET_MULTIPLE_WORKSPACES: "ASANA_GET_MULTIPLE_WORKSPACES",
    GET_PORTFOLIO: "ASANA_GET_PORTFOLIO",
    GET_PORTFOLIOS: "ASANA_GET_PORTFOLIOS",
    GET_PORTFOLIO_ITEMS: "ASANA_GET_PORTFOLIO_ITEMS",
    GET_PORTFOLIO_MEMBERSHIPS: "ASANA_GET_PORTFOLIO_MEMBERSHIPS",
    GET_PROJECTS_FOR_TEAM: "ASANA_GET_PROJECTS_FOR_TEAM",
    GET_PROJECT_BRIEF: "ASANA_GET_PROJECT_BRIEF",
    GET_PROJECT_MEMBERSHIPS: "ASANA_GET_PROJECT_MEMBERSHIPS",
    GET_PROJECT_STATUS: "ASANA_GET_PROJECT_STATUS",
    GET_PROJECT_STATUS_UPDATES: "ASANA_GET_PROJECT_STATUS_UPDATES",
    GET_PROJECT_TEMPLATES: "ASANA_GET_PROJECT_TEMPLATES",
    GET_SECTION: "ASANA_GET_SECTION",
    GET_SECTIONS_IN_PROJECT: "ASANA_GET_SECTIONS_IN_PROJECT",
    GET_STATUS_UPDATES: "ASANA_GET_STATUS_UPDATES",
    GET_STORIES_FOR_TASK: "ASANA_GET_STORIES_FOR_TASK",
    GET_STORY: "ASANA_GET_STORY",
    GET_TAG: "ASANA_GET_TAG",
    GET_TAGS: "ASANA_GET_TAGS",
    GET_TASKS_FROM_A_PROJECT: "ASANA_GET_TASKS_FROM_A_PROJECT",
    GET_TASK_ATTACHMENTS: "ASANA_GET_TASK_ATTACHMENTS",
    GET_TASK_SUBTASKS: "ASANA_GET_TASK_SUBTASKS",
    GET_TASK_TEMPLATES: "ASANA_GET_TASK_TEMPLATES",
    GET_TEAM: "ASANA_GET_TEAM",
    GET_TEAMS_IN_WORKSPACE: "ASANA_GET_TEAMS_IN_WORKSPACE",
    GET_TEAM_MEMBERSHIPS: "ASANA_GET_TEAM_MEMBERSHIPS",
    GET_TIME_PERIODS: "ASANA_GET_TIME_PERIODS",
    GET_TYPEAHEAD_OBJECTS: "ASANA_GET_TYPEAHEAD_OBJECTS",
    GET_USER: "ASANA_GET_USER",
    GET_USERS_FOR_TEAM: "ASANA_GET_USERS_FOR_TEAM",
    GET_USERS_FOR_WORKSPACE: "ASANA_GET_USERS_FOR_WORKSPACE",
    GET_WORKSPACE: "ASANA_GET_WORKSPACE",
    GET_WORKSPACE_MEMBERSHIPS: "ASANA_GET_WORKSPACE_MEMBERSHIPS",
    GET_WORKSPACE_PROJECTS: "ASANA_GET_WORKSPACE_PROJECTS",
    INSERT_ENUM_OPTION_FOR_CUSTOM_FIELD: "ASANA_INSERT_ENUM_OPTION_FOR_CUSTOM_FIELD",
    REMOVE_FOLLOWER_FROM_TASK: "ASANA_REMOVE_FOLLOWER_FROM_TASK",
    SUBMIT_PARALLEL_REQUESTS: "ASANA_SUBMIT_PARALLEL_REQUESTS",
    UPDATE_ALLOCATION: "ASANA_UPDATE_ALLOCATION",
    UPDATE_A_TASK: "ASANA_UPDATE_A_TASK",
    UPDATE_CUSTOM_FIELD: "ASANA_UPDATE_CUSTOM_FIELD",
    UPDATE_ENUM_OPTION: "ASANA_UPDATE_ENUM_OPTION",
    UPDATE_PROJECT: "ASANA_UPDATE_PROJECT",
    UPDATE_TAG: "ASANA_UPDATE_TAG",
    UPDATE_TEAM: "ASANA_UPDATE_TEAM",
  },
  triggerTypes: {
    TASK_TRIGGER: "ASANA_TASK_TRIGGER",
  },
}
