// @composio/core/generated/bamboohr.ts
/**
 * Map of Composio's BAMBOOHR toolkit.
 */
export const BAMBOOHR = {
  slug: "bamboohr",
  tools: {
    ADDS_A_TABLE_ROW: "BAMBOOHR_ADDS_A_TABLE_ROW",
    ADD_AN_EMPLOYEE_BENEFIT: "BAMBOOHR_ADD_AN_EMPLOYEE_BENEFIT",
    ADD_AN_EMPLOYEE_DEPENDENT: "BAMBOOHR_ADD_AN_EMPLOYEE_DEPENDENT",
    ADD_AN_EMPLOYEE_S_DEFAULT_WITHHOLDINGS: "BAMBOOHR_ADD_AN_EMPLOYEE_S_DEFAULT_WITHHOLDINGS",
    ADD_AN_EMPLOYEE_S_DIRECT_DEPOSIT_INFORMATION: "BAMBOOHR_ADD_AN_EMPLOYEE_S_DIRECT_DEPOSIT_INFORMATION",
    ADD_AN_EMPLOYEE_S_PAYSTUB: "BAMBOOHR_ADD_AN_EMPLOYEE_S_PAYSTUB",
    ADD_AN_EMPLOYEE_S_UNPAID_PAYSTUBS: "BAMBOOHR_ADD_AN_EMPLOYEE_S_UNPAID_PAYSTUBS",
    ADD_AN_HOUR_RECORD: "BAMBOOHR_ADD_AN_HOUR_RECORD",
    ADD_APPLICATION_COMMENT: "BAMBOOHR_ADD_APPLICATION_COMMENT",
    ADD_A_BENEFIT_GROUP_EMPLOYEE: "BAMBOOHR_ADD_A_BENEFIT_GROUP_EMPLOYEE",
    ADD_A_NEW_COMPANY_BENEFIT: "BAMBOOHR_ADD_A_NEW_COMPANY_BENEFIT",
    ADD_A_TIME_OFF_HISTORY_ITEM_FOR_TIME_OFF_REQUEST: "BAMBOOHR_ADD_A_TIME_OFF_HISTORY_ITEM_FOR_TIME_OFF_REQUEST",
    ADD_A_TIME_OFF_REQUEST: "BAMBOOHR_ADD_A_TIME_OFF_REQUEST",
    ADD_EDIT_TIMESHEET_CLOCK_ENTRIES: "BAMBOOHR_ADD_EDIT_TIMESHEET_CLOCK_ENTRIES",
    ADD_EDIT_TIMESHEET_HOUR_ENTRIES: "BAMBOOHR_ADD_EDIT_TIMESHEET_HOUR_ENTRIES",
    ADD_EMPLOYEE: "BAMBOOHR_ADD_EMPLOYEE",
    ADD_NEW_CANDIDATE: "BAMBOOHR_ADD_NEW_CANDIDATE",
    ADD_NEW_EMPLOYEE_TRAINING_RECORD: "BAMBOOHR_ADD_NEW_EMPLOYEE_TRAINING_RECORD",
    ADD_NEW_JOB_OPENING: "BAMBOOHR_ADD_NEW_JOB_OPENING",
    ADD_OR_UPDATE_VALUES_FOR_LIST_FIELDS: "BAMBOOHR_ADD_OR_UPDATE_VALUES_FOR_LIST_FIELDS",
    ADD_TIMESHEET_CLOCK_IN_ENTRY: "BAMBOOHR_ADD_TIMESHEET_CLOCK_IN_ENTRY",
    ADD_TIMESHEET_CLOCK_OUT_ENTRY: "BAMBOOHR_ADD_TIMESHEET_CLOCK_OUT_ENTRY",
    ADD_TRAINING_CATEGORY: "BAMBOOHR_ADD_TRAINING_CATEGORY",
    ADD_TRAINING_TYPE: "BAMBOOHR_ADD_TRAINING_TYPE",
    ADD_WEBHOOK: "BAMBOOHR_ADD_WEBHOOK",
    ADJUST_TIME_OFF_BALANCE: "BAMBOOHR_ADJUST_TIME_OFF_BALANCE",
    ALIGNABLE_GOAL_OPTIONS: "BAMBOOHR_ALIGNABLE_GOAL_OPTIONS",
    APPROVE_ACTIVE_TIMESHEETS: "BAMBOOHR_APPROVE_ACTIVE_TIMESHEETS",
    APPROVE_EMPLOYEE_TIMESHEETS: "BAMBOOHR_APPROVE_EMPLOYEE_TIMESHEETS",
    AVAILABLE_GOAL_SHARING_OPTIONS: "BAMBOOHR_AVAILABLE_GOAL_SHARING_OPTIONS",
    CAN_CREATE_A_GOAL: "BAMBOOHR_CAN_CREATE_A_GOAL",
    CHANGE_APPLICANT_S_STATUS: "BAMBOOHR_CHANGE_APPLICANT_S_STATUS",
    CHANGE_A_REQUEST_STATUS: "BAMBOOHR_CHANGE_A_REQUEST_STATUS",
    CLEAR_AN_EMPLOYEE_S_DEFAULT_WITHHOLDINGS: "BAMBOOHR_CLEAR_AN_EMPLOYEE_S_DEFAULT_WITHHOLDINGS",
    CLEAR_AN_EMPLOYEE_S_DIRECT_DEPOSIT_INFORMATION: "BAMBOOHR_CLEAR_AN_EMPLOYEE_S_DIRECT_DEPOSIT_INFORMATION",
    CLEAR_AN_EMPLOYEE_S_UNPAID_PAYSTUBS: "BAMBOOHR_CLEAR_AN_EMPLOYEE_S_UNPAID_PAYSTUBS",
    CLOCK_IN_EMPLOYEE_ID_OPTIONAL: "BAMBOOHR_CLOCK_IN_EMPLOYEE_ID_OPTIONAL",
    CLOCK_OUT_AN_EMPLOYEE_AT_A_SPECIFIC_TIME: "BAMBOOHR_CLOCK_OUT_AN_EMPLOYEE_AT_A_SPECIFIC_TIME",
    CLOCK_OUT_EMPLOYEE_ID_OPTIONAL: "BAMBOOHR_CLOCK_OUT_EMPLOYEE_ID_OPTIONAL",
    CLOSE_GOAL: "BAMBOOHR_CLOSE_GOAL",
    CREATE_A_TIME_TRACKING_PROJECT: "BAMBOOHR_CREATE_A_TIME_TRACKING_PROJECT",
    CREATE_GOAL: "BAMBOOHR_CREATE_GOAL",
    CREATE_GOAL_COMMENT: "BAMBOOHR_CREATE_GOAL_COMMENT",
    DELETES_A_TABLE_ROW: "BAMBOOHR_DELETES_A_TABLE_ROW",
    DELETE_AN_EMPLOYEE_S_PAYSTUB: "BAMBOOHR_DELETE_AN_EMPLOYEE_S_PAYSTUB",
    DELETE_AN_HOUR_RECORD: "BAMBOOHR_DELETE_AN_HOUR_RECORD",
    DELETE_A_COMPANY_BENEFIT: "BAMBOOHR_DELETE_A_COMPANY_BENEFIT",
    DELETE_CLOCK_ENTRIES: "BAMBOOHR_DELETE_CLOCK_ENTRIES",
    DELETE_COMPANY_FILE: "BAMBOOHR_DELETE_COMPANY_FILE",
    DELETE_EMPLOYEE_FILE: "BAMBOOHR_DELETE_EMPLOYEE_FILE",
    DELETE_EMPLOYEE_TRAINING_RECORD: "BAMBOOHR_DELETE_EMPLOYEE_TRAINING_RECORD",
    DELETE_GOAL: "BAMBOOHR_DELETE_GOAL",
    DELETE_GOAL_COMMENT: "BAMBOOHR_DELETE_GOAL_COMMENT",
    DELETE_TIMESHEET_CLOCK_ENTRIES: "BAMBOOHR_DELETE_TIMESHEET_CLOCK_ENTRIES",
    DELETE_TIMESHEET_HOUR_ENTRIES: "BAMBOOHR_DELETE_TIMESHEET_HOUR_ENTRIES",
    DELETE_TRAINING_CATEGORY: "BAMBOOHR_DELETE_TRAINING_CATEGORY",
    DELETE_TRAINING_TYPE: "BAMBOOHR_DELETE_TRAINING_TYPE",
    DELETE_WEBHOOK: "BAMBOOHR_DELETE_WEBHOOK",
    EDIT_AN_HOUR_RECORD: "BAMBOOHR_EDIT_AN_HOUR_RECORD",
    EDIT_CURRENT_CLOCK_IN_ENTRY: "BAMBOOHR_EDIT_CURRENT_CLOCK_IN_ENTRY",
    ENSURE_EMPLOYEE_FIELDS_FOR_TABLE_ROW: "BAMBOOHR_ENSURE_EMPLOYEE_FIELDS_FOR_TABLE_ROW",
    ESTIMATE_FUTURE_TIME_OFF_BALANCES: "BAMBOOHR_ESTIMATE_FUTURE_TIME_OFF_BALANCES",
    GETS_ALL_UPDATED_EMPLOYEE_IDS: "BAMBOOHR_GETS_ALL_UPDATED_EMPLOYEE_IDS",
    GETS_ALL_UPDATED_EMPLOYEE_TABLE_DATA: "BAMBOOHR_GETS_ALL_UPDATED_EMPLOYEE_TABLE_DATA",
    GETS_AS_LIST_OF_WEBHOOKS_FOR_THE_USER_API_KEY: "BAMBOOHR_GETS_AS_LIST_OF_WEBHOOKS_FOR_THE_USER_API_KEY",
    GET_AGGREGATE_GOAL_INFO: "BAMBOOHR_GET_AGGREGATE_GOAL_INFO",
    GET_ALL_AGGREGATE_GOAL_INFO: "BAMBOOHR_GET_ALL_AGGREGATE_GOAL_INFO",
    GET_ALL_AGGREGATE_GOAL_INFO_VERSION_1_1: "BAMBOOHR_GET_ALL_AGGREGATE_GOAL_INFO_VERSION_1_1",
    GET_ALL_AGGREGATE_GOAL_INFO_VERSION_1_2: "BAMBOOHR_GET_ALL_AGGREGATE_GOAL_INFO_VERSION_1_2",
    GET_ALL_EMPLOYEE_DEPENDENTS: "BAMBOOHR_GET_ALL_EMPLOYEE_DEPENDENTS",
    GET_AN_COMPANY_FILE: "BAMBOOHR_GET_AN_COMPANY_FILE",
    GET_AN_EMPLOYEE_FILE: "BAMBOOHR_GET_AN_EMPLOYEE_FILE",
    GET_AN_EMPLOYEE_PHOTO: "BAMBOOHR_GET_AN_EMPLOYEE_PHOTO",
    GET_AN_EMPLOYEE_S_DEFAULT_WITHHOLDINGS: "BAMBOOHR_GET_AN_EMPLOYEE_S_DEFAULT_WITHHOLDINGS",
    GET_AN_EMPLOYEE_S_DIRECT_DEPOSIT_INFORMATION: "BAMBOOHR_GET_AN_EMPLOYEE_S_DIRECT_DEPOSIT_INFORMATION",
    GET_AN_EMPLOYEE_S_PAYSTUB: "BAMBOOHR_GET_AN_EMPLOYEE_S_PAYSTUB",
    GET_AN_EMPLOYEE_S_UNPAID_PAYSTUBS: "BAMBOOHR_GET_AN_EMPLOYEE_S_UNPAID_PAYSTUBS",
    GET_AN_HOUR_RECORD: "BAMBOOHR_GET_AN_HOUR_RECORD",
    GET_APPLICATIONS: "BAMBOOHR_GET_APPLICATIONS",
    GET_APPLICATION_DETAILS: "BAMBOOHR_GET_APPLICATION_DETAILS",
    GET_A_CLOCK_ENTRY: "BAMBOOHR_GET_A_CLOCK_ENTRY",
    GET_A_COMPANY_BENEFIT: "BAMBOOHR_GET_A_COMPANY_BENEFIT",
    GET_A_DAILY_ENTRY: "BAMBOOHR_GET_A_DAILY_ENTRY",
    GET_A_LIST_OF_COMPANY_BENEFITS: "BAMBOOHR_GET_A_LIST_OF_COMPANY_BENEFITS",
    GET_A_LIST_OF_COMPANY_BENEFIT_TYPES: "BAMBOOHR_GET_A_LIST_OF_COMPANY_BENEFIT_TYPES",
    GET_A_LIST_OF_EMPLOYEE_BENEFITS: "BAMBOOHR_GET_A_LIST_OF_EMPLOYEE_BENEFITS",
    GET_A_LIST_OF_FIELDS: "BAMBOOHR_GET_A_LIST_OF_FIELDS",
    GET_A_LIST_OF_TABULAR_FIELDS: "BAMBOOHR_GET_A_LIST_OF_TABULAR_FIELDS",
    GET_A_LIST_OF_USERS: "BAMBOOHR_GET_A_LIST_OF_USERS",
    GET_A_LIST_OF_WHO_S_OUT: "BAMBOOHR_GET_A_LIST_OF_WHO_S_OUT",
    GET_A_TIMESHEET_FOR_AN_EMPLOYEE: "BAMBOOHR_GET_A_TIMESHEET_FOR_AN_EMPLOYEE",
    GET_BENEFIT_COVERAGES: "BAMBOOHR_GET_BENEFIT_COVERAGES",
    GET_BENEFIT_DEDUCTIONS_FOR_EMPLOYEE: "BAMBOOHR_GET_BENEFIT_DEDUCTIONS_FOR_EMPLOYEE",
    GET_BENEFIT_DEDUCTION_TYPES: "BAMBOOHR_GET_BENEFIT_DEDUCTION_TYPES",
    GET_BENEFIT_GROUPS: "BAMBOOHR_GET_BENEFIT_GROUPS",
    GET_BENEFIT_GROUP_EMPLOYEES: "BAMBOOHR_GET_BENEFIT_GROUP_EMPLOYEES",
    GET_BENEFIT_GROUP_PLANS: "BAMBOOHR_GET_BENEFIT_GROUP_PLANS",
    GET_BENEFIT_GROUP_PLAN_COSTS: "BAMBOOHR_GET_BENEFIT_GROUP_PLAN_COSTS",
    GET_BENEFIT_PLANS: "BAMBOOHR_GET_BENEFIT_PLANS",
    GET_BENEFIT_PLAN_COVERAGES: "BAMBOOHR_GET_BENEFIT_PLAN_COVERAGES",
    GET_BENEFIT_PLAN_DEDUCTIONS_BY_EMPLOYEE: "BAMBOOHR_GET_BENEFIT_PLAN_DEDUCTIONS_BY_EMPLOYEE",
    GET_COMPANY_LOCATIONS: "BAMBOOHR_GET_COMPANY_LOCATIONS",
    GET_COMPANY_REPORT: "BAMBOOHR_GET_COMPANY_REPORT",
    GET_DETAILS_FOR_LIST_FIELDS: "BAMBOOHR_GET_DETAILS_FOR_LIST_FIELDS",
    GET_EMPLOYEE: "BAMBOOHR_GET_EMPLOYEE",
    GET_EMPLOYEE_DEDUCTIONS_BY_BENEFIT_PLAN: "BAMBOOHR_GET_EMPLOYEE_DEDUCTIONS_BY_BENEFIT_PLAN",
    GET_EMPLOYEE_DEPENDENT: "BAMBOOHR_GET_EMPLOYEE_DEPENDENT",
    GET_EMPLOYEE_DIRECTORY: "BAMBOOHR_GET_EMPLOYEE_DIRECTORY",
    GET_EMPLOYEE_PROJECTS: "BAMBOOHR_GET_EMPLOYEE_PROJECTS",
    GET_EMPLOYEE_TABLE_ROWS: "BAMBOOHR_GET_EMPLOYEE_TABLE_ROWS",
    GET_GOALS: "BAMBOOHR_GET_GOALS",
    GET_GOAL_COMMENTS: "BAMBOOHR_GET_GOAL_COMMENTS",
    GET_GOAL_STATUS_COUNTS: "BAMBOOHR_GET_GOAL_STATUS_COUNTS",
    GET_GOAL_STATUS_COUNTS_VERSION_1_1: "BAMBOOHR_GET_GOAL_STATUS_COUNTS_VERSION_1_1",
    GET_HIRING_LEADS: "BAMBOOHR_GET_HIRING_LEADS",
    GET_JOB_SUMMARIES: "BAMBOOHR_GET_JOB_SUMMARIES",
    GET_MONITOR_FIELDS: "BAMBOOHR_GET_MONITOR_FIELDS",
    GET_PROJECTS_AN_EMPLOYEE_CAN_LOG_TIME_TO: "BAMBOOHR_GET_PROJECTS_AN_EMPLOYEE_CAN_LOG_TIME_TO",
    GET_STATUSES: "BAMBOOHR_GET_STATUSES",
    GET_TASKS_FOR_A_PROJECT: "BAMBOOHR_GET_TASKS_FOR_A_PROJECT",
    GET_TIMESHEETS_BY_IDS: "BAMBOOHR_GET_TIMESHEETS_BY_IDS",
    GET_TIMESHEET_ENTRIES: "BAMBOOHR_GET_TIMESHEET_ENTRIES",
    GET_TIME_OFF_POLICIES: "BAMBOOHR_GET_TIME_OFF_POLICIES",
    GET_TIME_OFF_REQUESTS: "BAMBOOHR_GET_TIME_OFF_REQUESTS",
    GET_TIME_OFF_TYPES: "BAMBOOHR_GET_TIME_OFF_TYPES",
    GET_WEBHOOK: "BAMBOOHR_GET_WEBHOOK",
    GET_WEBHOOK_LOGS: "BAMBOOHR_GET_WEBHOOK_LOGS",
    IS_EMPLOYEE_CLOCKED_IN: "BAMBOOHR_IS_EMPLOYEE_CLOCKED_IN",
    LIST_COMPANY_FILES_AND_CATEGORIES: "BAMBOOHR_LIST_COMPANY_FILES_AND_CATEGORIES",
    LIST_EMPLOYEE_FILES_AND_CATEGORIES: "BAMBOOHR_LIST_EMPLOYEE_FILES_AND_CATEGORIES",
    LIST_EMPLOYEE_TRAININGS: "BAMBOOHR_LIST_EMPLOYEE_TRAININGS",
    LIST_TIME_OFF_POLICIES_FOR_EMPLOYEE: "BAMBOOHR_LIST_TIME_OFF_POLICIES_FOR_EMPLOYEE",
    LIST_TIME_OFF_POLICIES_FOR_EMPLOYEE_VERSION_1_1: "BAMBOOHR_LIST_TIME_OFF_POLICIES_FOR_EMPLOYEE_VERSION_1_1",
    LIST_TRAINING_CATEGORIES: "BAMBOOHR_LIST_TRAINING_CATEGORIES",
    LIST_TRAINING_TYPES: "BAMBOOHR_LIST_TRAINING_TYPES",
    MODIFIES_ROW_CONTENTS: "BAMBOOHR_MODIFIES_ROW_CONTENTS",
    REOPEN_A_GOAL: "BAMBOOHR_REOPEN_A_GOAL",
    REQUEST_A_CUSTOM_REPORT: "BAMBOOHR_REQUEST_A_CUSTOM_REPORT",
    STORE_A_NEW_EMPLOYEE_PHOTO: "BAMBOOHR_STORE_A_NEW_EMPLOYEE_PHOTO",
    STORE_CLOCK_ENTRIES: "BAMBOOHR_STORE_CLOCK_ENTRIES",
    STORE_DAILY_ENTRIES: "BAMBOOHR_STORE_DAILY_ENTRIES",
    UPDATES_A_TABLE_ROW: "BAMBOOHR_UPDATES_A_TABLE_ROW",
    UPDATE_AN_EMPLOYEE_DEPENDENT: "BAMBOOHR_UPDATE_AN_EMPLOYEE_DEPENDENT",
    UPDATE_A_COMPANY_BENEFIT: "BAMBOOHR_UPDATE_A_COMPANY_BENEFIT",
    UPDATE_COMPANY_FILE: "BAMBOOHR_UPDATE_COMPANY_FILE",
    UPDATE_EMPLOYEE: "BAMBOOHR_UPDATE_EMPLOYEE",
    UPDATE_EMPLOYEE_FILE: "BAMBOOHR_UPDATE_EMPLOYEE_FILE",
    UPDATE_EMPLOYEE_TRAINING_RECORD: "BAMBOOHR_UPDATE_EMPLOYEE_TRAINING_RECORD",
    UPDATE_GOAL: "BAMBOOHR_UPDATE_GOAL",
    UPDATE_GOAL_COMMENT: "BAMBOOHR_UPDATE_GOAL_COMMENT",
    UPDATE_GOAL_PROGRESS: "BAMBOOHR_UPDATE_GOAL_PROGRESS",
    UPDATE_GOAL_SHARING: "BAMBOOHR_UPDATE_GOAL_SHARING",
    UPDATE_GOAL_V1_1: "BAMBOOHR_UPDATE_GOAL_V1_1",
    UPDATE_TRAINING_CATEGORY: "BAMBOOHR_UPDATE_TRAINING_CATEGORY",
    UPDATE_TRAINING_TYPE: "BAMBOOHR_UPDATE_TRAINING_TYPE",
    UPDATE_WEBHOOK: "BAMBOOHR_UPDATE_WEBHOOK",
    UPLOAD_COMPANY_FILE: "BAMBOOHR_UPLOAD_COMPANY_FILE",
    UPLOAD_EMPLOYEE_FILE: "BAMBOOHR_UPLOAD_EMPLOYEE_FILE",
    USER_LOGIN: "BAMBOOHR_USER_LOGIN",
  },
  triggerTypes: {},
}
