// @composio/core/generated/bill.ts
/**
 * Map of Composio's BILL toolkit.
 */
export const BILL = {
  slug: "bill",
  tools: {
    ACCEPT_NETWORK_INVITATION: "BILL_ACCEPT_NETWORK_INVITATION",
    ADD_PHONE_FOR_MFA_SETUP: "BILL_ADD_PHONE_FOR_MFA_SETUP",
    ADD_PHONE_FOR_RISK_VERIFICATION: "BILL_ADD_PHONE_FOR_RISK_VERIFICATION",
    API_LOGIN: "BILL_API_LOGIN",
    API_LOGIN_AS_USER: "BILL_API_LOGIN_AS_USER",
    API_LOGOUT: "BILL_API_LOGOUT",
    API_PARTNER_LOGIN: "BILL_API_PARTNER_LOGIN",
    APPROVE_OR_DENY_A_REIMBURSEMENT: "BILL_APPROVE_OR_DENY_A_REIMBURSEMENT",
    ARCHIVE_AN_ACCOUNTING_CLASS: "BILL_ARCHIVE_AN_ACCOUNTING_CLASS",
    ARCHIVE_AN_EMPLOYEE: "BILL_ARCHIVE_AN_EMPLOYEE",
    ARCHIVE_AN_INVOICE: "BILL_ARCHIVE_AN_INVOICE",
    ARCHIVE_AN_ITEM: "BILL_ARCHIVE_AN_ITEM",
    ARCHIVE_A_BANK_ACCOUNT: "BILL_ARCHIVE_A_BANK_ACCOUNT",
    ARCHIVE_A_BANK_ACCOUNT_USER: "BILL_ARCHIVE_A_BANK_ACCOUNT_USER",
    ARCHIVE_A_BILL: "BILL_ARCHIVE_A_BILL",
    ARCHIVE_A_CHART_OF_ACCOUNTS: "BILL_ARCHIVE_A_CHART_OF_ACCOUNTS",
    ARCHIVE_A_CUSTOMER: "BILL_ARCHIVE_A_CUSTOMER",
    ARCHIVE_A_DEPARTMENT: "BILL_ARCHIVE_A_DEPARTMENT",
    ARCHIVE_A_JOB: "BILL_ARCHIVE_A_JOB",
    ARCHIVE_A_LOCATION: "BILL_ARCHIVE_A_LOCATION",
    ARCHIVE_A_RECURRING_BILL: "BILL_ARCHIVE_A_RECURRING_BILL",
    ARCHIVE_A_USER: "BILL_ARCHIVE_A_USER",
    ARCHIVE_A_VENDOR: "BILL_ARCHIVE_A_VENDOR",
    ARCHIVE_MULTIPLE_ACCOUNTING_CLASSES: "BILL_ARCHIVE_MULTIPLE_ACCOUNTING_CLASSES",
    ARCHIVE_MULTIPLE_DEPARTMENTS: "BILL_ARCHIVE_MULTIPLE_DEPARTMENTS",
    ARCHIVE_MULTIPLE_EMPLOYEES: "BILL_ARCHIVE_MULTIPLE_EMPLOYEES",
    ARCHIVE_MULTIPLE_ITEMS: "BILL_ARCHIVE_MULTIPLE_ITEMS",
    ARCHIVE_MULTIPLE_JOBS: "BILL_ARCHIVE_MULTIPLE_JOBS",
    ARCHIVE_MULTIPLE_LOCATIONS: "BILL_ARCHIVE_MULTIPLE_LOCATIONS",
    ARCHIVE_MUTLIPLE_CHART_OF_ACCOUNTS: "BILL_ARCHIVE_MUTLIPLE_CHART_OF_ACCOUNTS",
    CANCEL_A_PAYMENT: "BILL_CANCEL_A_PAYMENT",
    CHECK_APP_HEALTH: "BILL_CHECK_APP_HEALTH",
    CREATE_AN_ACCOUNTING_CLASS: "BILL_CREATE_AN_ACCOUNTING_CLASS",
    CREATE_AN_EMPLOYEE: "BILL_CREATE_AN_EMPLOYEE",
    CREATE_AN_IMAGE_UPLOAD_URL_FOR_A_REIMBURSEMENT: "BILL_CREATE_AN_IMAGE_UPLOAD_URL_FOR_A_REIMBURSEMENT",
    CREATE_AN_INVOICE: "BILL_CREATE_AN_INVOICE",
    CREATE_AN_ITEM: "BILL_CREATE_AN_ITEM",
    CREATE_AN_ORGANIZATION: "BILL_CREATE_AN_ORGANIZATION",
    CREATE_A_BANK_ACCOUNT: "BILL_CREATE_A_BANK_ACCOUNT",
    CREATE_A_BILL: "BILL_CREATE_A_BILL",
    CREATE_A_BUDGET: "BILL_CREATE_A_BUDGET",
    CREATE_A_BULK_PAYMENT: "BILL_CREATE_A_BULK_PAYMENT",
    CREATE_A_CHART_OF_ACCOUNTS: "BILL_CREATE_A_CHART_OF_ACCOUNTS",
    CREATE_A_CUSTOMER: "BILL_CREATE_A_CUSTOMER",
    CREATE_A_DEPARTMENT: "BILL_CREATE_A_DEPARTMENT",
    CREATE_A_JOB: "BILL_CREATE_A_JOB",
    CREATE_A_LOCATION: "BILL_CREATE_A_LOCATION",
    CREATE_A_PAYMENT: "BILL_CREATE_A_PAYMENT",
    CREATE_A_RECURRING_BILL: "BILL_CREATE_A_RECURRING_BILL",
    CREATE_A_REIMBURSEMENT: "BILL_CREATE_A_REIMBURSEMENT",
    CREATE_A_USER: "BILL_CREATE_A_USER",
    CREATE_A_VENDOR: "BILL_CREATE_A_VENDOR",
    CREATE_A_VENDOR_BANK_ACCOUNT: "BILL_CREATE_A_VENDOR_BANK_ACCOUNT",
    CREATE_A_VENDOR_CARD: "BILL_CREATE_A_VENDOR_CARD",
    CREATE_CUSTOM_FIELD: "BILL_CREATE_CUSTOM_FIELD",
    CREATE_CUSTOM_FIELD_VALUES: "BILL_CREATE_CUSTOM_FIELD_VALUES",
    DELETE_A_BUDGET: "BILL_DELETE_A_BUDGET",
    DELETE_A_CARD: "BILL_DELETE_A_CARD",
    DELETE_A_CUSTOM_FIELD: "BILL_DELETE_A_CUSTOM_FIELD",
    DELETE_A_MEMBER_FROM_A_BUDGET: "BILL_DELETE_A_MEMBER_FROM_A_BUDGET",
    DELETE_A_REIMBURSEMENT: "BILL_DELETE_A_REIMBURSEMENT",
    DELETE_A_USER: "BILL_DELETE_A_USER",
    DELETE_A_VENDOR_BANK_ACCOUNT: "BILL_DELETE_A_VENDOR_BANK_ACCOUNT",
    DELETE_CUSTOMER_CONNECTION: "BILL_DELETE_CUSTOMER_CONNECTION",
    DELETE_CUSTOM_FIELD_VALUES: "BILL_DELETE_CUSTOM_FIELD_VALUES",
    DELETE_VENDOR_CONNECTION: "BILL_DELETE_VENDOR_CONNECTION",
    GENERATE_MFA_CHALLENGE: "BILL_GENERATE_MFA_CHALLENGE",
    GET_ACCOUNTING_CLASS_DETAILS: "BILL_GET_ACCOUNTING_CLASS_DETAILS",
    GET_API_SESSION_DETAILS: "BILL_GET_API_SESSION_DETAILS",
    GET_ATTACHMENT_DETAILS: "BILL_GET_ATTACHMENT_DETAILS",
    GET_AUDIT_TRAIL_DETAILS_FOR_A_VENDOR: "BILL_GET_AUDIT_TRAIL_DETAILS_FOR_A_VENDOR",
    GET_A_SINGLE_MEMBER_FOR_A_BUDGET: "BILL_GET_A_SINGLE_MEMBER_FOR_A_BUDGET",
    GET_BANK_ACCOUNT_DETAILS: "BILL_GET_BANK_ACCOUNT_DETAILS",
    GET_BILL_DETAILS: "BILL_GET_BILL_DETAILS",
    GET_BUDGET_DETAILS: "BILL_GET_BUDGET_DETAILS",
    GET_CARD_ACCOUNT_DETAILS: "BILL_GET_CARD_ACCOUNT_DETAILS",
    GET_CARD_DETAILS: "BILL_GET_CARD_DETAILS",
    GET_CARD_FUNDING_PURPOSE: "BILL_GET_CARD_FUNDING_PURPOSE",
    GET_CHART_OF_ACCOUNTS_DETAILS: "BILL_GET_CHART_OF_ACCOUNTS_DETAILS",
    GET_CHECK_IMAGE_DATA: "BILL_GET_CHECK_IMAGE_DATA",
    GET_CURRENT_USER_DETAILS: "BILL_GET_CURRENT_USER_DETAILS",
    GET_CUSTOMER_DETAILS: "BILL_GET_CUSTOMER_DETAILS",
    GET_CUSTOMER_INVITATION_STATUS: "BILL_GET_CUSTOMER_INVITATION_STATUS",
    GET_CUSTOM_FIELD_DETAILS: "BILL_GET_CUSTOM_FIELD_DETAILS",
    GET_CUSTOM_FIELD_VALUE: "BILL_GET_CUSTOM_FIELD_VALUE",
    GET_DEPARTMENT_DETAILS: "BILL_GET_DEPARTMENT_DETAILS",
    GET_DOCUMENT_DETAILS: "BILL_GET_DOCUMENT_DETAILS",
    GET_DOCUMENT_UPLOAD_STATUS: "BILL_GET_DOCUMENT_UPLOAD_STATUS",
    GET_EMPLOYEE_DETAILS: "BILL_GET_EMPLOYEE_DETAILS",
    GET_FUNDING_ACCOUNT_PERMISSIONS: "BILL_GET_FUNDING_ACCOUNT_PERMISSIONS",
    GET_INTERNATIONAL_PAYMENTS_CONFIGURATION: "BILL_GET_INTERNATIONAL_PAYMENTS_CONFIGURATION",
    GET_INVOICE_DETAILS: "BILL_GET_INVOICE_DETAILS",
    GET_ITEM_DETAILS: "BILL_GET_ITEM_DETAILS",
    GET_JOB_DETAILS: "BILL_GET_JOB_DETAILS",
    GET_LIST_OF_ACCOUNTING_CLASSES: "BILL_GET_LIST_OF_ACCOUNTING_CLASSES",
    GET_LIST_OF_AP_CARDS: "BILL_GET_LIST_OF_AP_CARDS",
    GET_LIST_OF_BANK_ACCOUNTS: "BILL_GET_LIST_OF_BANK_ACCOUNTS",
    GET_LIST_OF_BANK_ACCOUNT_USERS: "BILL_GET_LIST_OF_BANK_ACCOUNT_USERS",
    GET_LIST_OF_BILLS: "BILL_GET_LIST_OF_BILLS",
    GET_LIST_OF_BUDGETS: "BILL_GET_LIST_OF_BUDGETS",
    GET_LIST_OF_CARDS: "BILL_GET_LIST_OF_CARDS",
    GET_LIST_OF_CARD_ACCOUNTS: "BILL_GET_LIST_OF_CARD_ACCOUNTS",
    GET_LIST_OF_CARD_ACCOUNT_USERS: "BILL_GET_LIST_OF_CARD_ACCOUNT_USERS",
    GET_LIST_OF_CHART_OF_ACCOUNTS: "BILL_GET_LIST_OF_CHART_OF_ACCOUNTS",
    GET_LIST_OF_CUSTOMERS: "BILL_GET_LIST_OF_CUSTOMERS",
    GET_LIST_OF_CUSTOM_FIELDS: "BILL_GET_LIST_OF_CUSTOM_FIELDS",
    GET_LIST_OF_DEPARTMENTS: "BILL_GET_LIST_OF_DEPARTMENTS",
    GET_LIST_OF_DOCUMENTS: "BILL_GET_LIST_OF_DOCUMENTS",
    GET_LIST_OF_EMPLOYEES: "BILL_GET_LIST_OF_EMPLOYEES",
    GET_LIST_OF_INVOICES: "BILL_GET_LIST_OF_INVOICES",
    GET_LIST_OF_INVOICE_ATTACHMENTS: "BILL_GET_LIST_OF_INVOICE_ATTACHMENTS",
    GET_LIST_OF_ITEMS: "BILL_GET_LIST_OF_ITEMS",
    GET_LIST_OF_JOBS: "BILL_GET_LIST_OF_JOBS",
    GET_LIST_OF_LOCATIONS: "BILL_GET_LIST_OF_LOCATIONS",
    GET_LIST_OF_MEMBERS_FOR_A_BUDGET: "BILL_GET_LIST_OF_MEMBERS_FOR_A_BUDGET",
    GET_LIST_OF_MFA_PHONE_NUMBERS: "BILL_GET_LIST_OF_MFA_PHONE_NUMBERS",
    GET_LIST_OF_ORGANIZATIONS: "BILL_GET_LIST_OF_ORGANIZATIONS",
    GET_LIST_OF_ORGANIZATION_INDUSTRIES: "BILL_GET_LIST_OF_ORGANIZATION_INDUSTRIES",
    GET_LIST_OF_PAYMENTS: "BILL_GET_LIST_OF_PAYMENTS",
    GET_LIST_OF_RECURRING_BILLS: "BILL_GET_LIST_OF_RECURRING_BILLS",
    GET_LIST_OF_REIMBURSEMENTS: "BILL_GET_LIST_OF_REIMBURSEMENTS",
    GET_LIST_OF_TRANSACTIONS: "BILL_GET_LIST_OF_TRANSACTIONS",
    GET_LIST_OF_USERS: "BILL_GET_LIST_OF_USERS",
    GET_LIST_OF_USER_ROLES: "BILL_GET_LIST_OF_USER_ROLES",
    GET_LIST_OF_VALUES_FOR_CUSTOM_FIELD: "BILL_GET_LIST_OF_VALUES_FOR_CUSTOM_FIELD",
    GET_LIST_OF_VENDORS: "BILL_GET_LIST_OF_VENDORS",
    GET_LIST_OF_VENDOR_ATTACHMENTS: "BILL_GET_LIST_OF_VENDOR_ATTACHMENTS",
    GET_LIST_OF_VENDOR_PAYMENT_OPTIONS: "BILL_GET_LIST_OF_VENDOR_PAYMENT_OPTIONS",
    GET_LOCATION_DETAILS: "BILL_GET_LOCATION_DETAILS",
    GET_ORGANIZATION_DETAILS: "BILL_GET_ORGANIZATION_DETAILS",
    GET_ORGANIZATION_PRICE_PLAN_DETAILS: "BILL_GET_ORGANIZATION_PRICE_PLAN_DETAILS",
    GET_PAN_JWT: "BILL_GET_PAN_JWT",
    GET_PAYMENT_DETAILS: "BILL_GET_PAYMENT_DETAILS",
    GET_PHONE_STATUS_FOR_RISK_VERIFICATION: "BILL_GET_PHONE_STATUS_FOR_RISK_VERIFICATION",
    GET_RECURRING_BILL_DETAILS: "BILL_GET_RECURRING_BILL_DETAILS",
    GET_REIMBURSEMENT_DETAILS: "BILL_GET_REIMBURSEMENT_DETAILS",
    GET_RISK_VERIFICATION_DETAILS: "BILL_GET_RISK_VERIFICATION_DETAILS",
    GET_TRANSACTION_CUSTOM_FIELD_DETAILS: "BILL_GET_TRANSACTION_CUSTOM_FIELD_DETAILS",
    GET_TRANSACTION_CUSTOM_FIELD_VALUE_DETAILS: "BILL_GET_TRANSACTION_CUSTOM_FIELD_VALUE_DETAILS",
    GET_TRANSACTION_DETAILS: "BILL_GET_TRANSACTION_DETAILS",
    GET_USER_DETAILS: "BILL_GET_USER_DETAILS",
    GET_USER_ROLE_DETAILS: "BILL_GET_USER_ROLE_DETAILS",
    GET_VENDOR_BANK_ACCOUNT_DETAILS: "BILL_GET_VENDOR_BANK_ACCOUNT_DETAILS",
    GET_VENDOR_CONFIGURATION: "BILL_GET_VENDOR_CONFIGURATION",
    GET_VENDOR_DETAILS: "BILL_GET_VENDOR_DETAILS",
    GET_VENDOR_INVITATION_STATUS: "BILL_GET_VENDOR_INVITATION_STATUS",
    INITIATE_RISK_VERIFICATION_FOR_AN_ORGANIZATION: "BILL_INITIATE_RISK_VERIFICATION_FOR_AN_ORGANIZATION",
    INVITE_A_CUSTOMER_IN_THE_BILL_NETWORK: "BILL_INVITE_A_CUSTOMER_IN_THE_BILL_NETWORK",
    INVITE_A_VENDOR_IN_THE_BILL_NETWORK: "BILL_INVITE_A_VENDOR_IN_THE_BILL_NETWORK",
    MFA_STEP_UP_FOR_API_SESSION: "BILL_MFA_STEP_UP_FOR_API_SESSION",
    NOMINATE_A_BANK_ACCOUNT_USER: "BILL_NOMINATE_A_BANK_ACCOUNT_USER",
    RECORD_AR_PAYMENT: "BILL_RECORD_AR_PAYMENT",
    REPLACE_AN_INVOICE: "BILL_REPLACE_AN_INVOICE",
    REPLACE_A_BILL: "BILL_REPLACE_A_BILL",
    REPLACE_A_RECURRING_BILL: "BILL_REPLACE_A_RECURRING_BILL",
    RESTORE_AN_ARCHIVED_ACCOUNTING_CLASS: "BILL_RESTORE_AN_ARCHIVED_ACCOUNTING_CLASS",
    RESTORE_AN_ARCHIVED_BILL: "BILL_RESTORE_AN_ARCHIVED_BILL",
    RESTORE_AN_ARCHIVED_CHART_OF_ACCOUNTS: "BILL_RESTORE_AN_ARCHIVED_CHART_OF_ACCOUNTS",
    RESTORE_AN_ARCHIVED_CUSTOMER: "BILL_RESTORE_AN_ARCHIVED_CUSTOMER",
    RESTORE_AN_ARCHIVED_DEPARTMENT: "BILL_RESTORE_AN_ARCHIVED_DEPARTMENT",
    RESTORE_AN_ARCHIVED_EMPLOYEE: "BILL_RESTORE_AN_ARCHIVED_EMPLOYEE",
    RESTORE_AN_ARCHIVED_INVOICE: "BILL_RESTORE_AN_ARCHIVED_INVOICE",
    RESTORE_AN_ARCHIVED_ITEM: "BILL_RESTORE_AN_ARCHIVED_ITEM",
    RESTORE_AN_ARCHIVED_JOB: "BILL_RESTORE_AN_ARCHIVED_JOB",
    RESTORE_AN_ARCHIVED_LOCATION: "BILL_RESTORE_AN_ARCHIVED_LOCATION",
    RESTORE_AN_ARCHIVED_RECURRING_BILL: "BILL_RESTORE_AN_ARCHIVED_RECURRING_BILL",
    RESTORE_AN_ARCHIVED_USER: "BILL_RESTORE_AN_ARCHIVED_USER",
    RESTORE_AN_ARCHIVED_VENDOR: "BILL_RESTORE_AN_ARCHIVED_VENDOR",
    RESTORE_MULTIPLE_ACCOUNTING_CLASSES: "BILL_RESTORE_MULTIPLE_ACCOUNTING_CLASSES",
    RESTORE_MULTIPLE_CHART_OF_ACCOUNTS: "BILL_RESTORE_MULTIPLE_CHART_OF_ACCOUNTS",
    RESTORE_MULTIPLE_DEPARTMENTS: "BILL_RESTORE_MULTIPLE_DEPARTMENTS",
    RESTORE_MULTIPLE_EMPLOYEES: "BILL_RESTORE_MULTIPLE_EMPLOYEES",
    RESTORE_MULTIPLE_ITEMS: "BILL_RESTORE_MULTIPLE_ITEMS",
    RESTORE_MULTIPLE_JOBS: "BILL_RESTORE_MULTIPLE_JOBS",
    RESTORE_MULTIPLE_LOCATIONS: "BILL_RESTORE_MULTIPLE_LOCATIONS",
    RETRIEVE_CUSTOMER_ATTACHMENTS: "BILL_RETRIEVE_CUSTOMER_ATTACHMENTS",
    SEARCH_FOR_AN_ORGANIZATION_IN_THE_BILL_NETWORKS: "BILL_SEARCH_FOR_AN_ORGANIZATION_IN_THE_BILL_NETWORKS",
    SEND_AN_INVOICE: "BILL_SEND_AN_INVOICE",
    UPDATE_AN_ACCOUNTING_CLASS: "BILL_UPDATE_AN_ACCOUNTING_CLASS",
    UPDATE_AN_EMPLOYEE: "BILL_UPDATE_AN_EMPLOYEE",
    UPDATE_AN_INVOICE: "BILL_UPDATE_AN_INVOICE",
    UPDATE_AN_ITEM: "BILL_UPDATE_AN_ITEM",
    UPDATE_AN_ORGANIZATION: "BILL_UPDATE_AN_ORGANIZATION",
    UPDATE_A_BANK_ACCOUNT: "BILL_UPDATE_A_BANK_ACCOUNT",
    UPDATE_A_BILL: "BILL_UPDATE_A_BILL",
    UPDATE_A_BUDGET: "BILL_UPDATE_A_BUDGET",
    UPDATE_A_CHART_OF_ACCOUNTS: "BILL_UPDATE_A_CHART_OF_ACCOUNTS",
    UPDATE_A_CUSTOMER: "BILL_UPDATE_A_CUSTOMER",
    UPDATE_A_CUSTOM_FIELD_AND_VALUES_ON_A_TRANSACTION: "BILL_UPDATE_A_CUSTOM_FIELD_AND_VALUES_ON_A_TRANSACTION",
    UPDATE_A_DEPARTMENT: "BILL_UPDATE_A_DEPARTMENT",
    UPDATE_A_JOB: "BILL_UPDATE_A_JOB",
    UPDATE_A_LIST_OF_BUDGET_MEMBERS_IN_A_BUDGET: "BILL_UPDATE_A_LIST_OF_BUDGET_MEMBERS_IN_A_BUDGET",
    UPDATE_A_LOCATION: "BILL_UPDATE_A_LOCATION",
    UPDATE_A_RECURRING_BILL: "BILL_UPDATE_A_RECURRING_BILL",
    UPDATE_A_REIMBURSEMENT: "BILL_UPDATE_A_REIMBURSEMENT",
    UPDATE_A_USER: "BILL_UPDATE_A_USER",
    UPDATE_A_VENDOR: "BILL_UPDATE_A_VENDOR",
    UPDATE_A_VENDOR_CARD: "BILL_UPDATE_A_VENDOR_CARD",
    UPDATE_BUDGET_MEMBER_FUNDS: "BILL_UPDATE_BUDGET_MEMBER_FUNDS",
    UPDATE_CUSTOM_FIELD_DETAILS: "BILL_UPDATE_CUSTOM_FIELD_DETAILS",
    UPDATE_TRANSACTION: "BILL_UPDATE_TRANSACTION",
    UPLOAD_BILL_DOCUMENT: "BILL_UPLOAD_BILL_DOCUMENT",
    UPLOAD_CUSTOMER_ATTACHMENT: "BILL_UPLOAD_CUSTOMER_ATTACHMENT",
    UPLOAD_INVOICE_ATTACHMENT: "BILL_UPLOAD_INVOICE_ATTACHMENT",
    UPLOAD_VENDOR_ATTACHMENT: "BILL_UPLOAD_VENDOR_ATTACHMENT",
    VALIDATE_MFA_CHALLENGE: "BILL_VALIDATE_MFA_CHALLENGE",
    VALIDATE_PHONE_FOR_MFA_SETUP: "BILL_VALIDATE_PHONE_FOR_MFA_SETUP",
    VERIFY_A_BANK_ACCOUNT: "BILL_VERIFY_A_BANK_ACCOUNT",
    VOID_A_PAYMENT: "BILL_VOID_A_PAYMENT",
  },
  triggerTypes: {},
}
