// @composio/core/generated/bitbucket.ts
/**
 * Map of Composio's BITBUCKET toolkit.
 */
export const BITBUCKET = {
  slug: "bitbucket",
  tools: {
    CREATE_BRANCH: "BITBUCKET_CREATE_BRANCH",
    CREATE_ISSUE: "BITBUCKET_CREATE_ISSUE",
    CREATE_ISSUE_COMMENT: "BITBUCKET_CREATE_ISSUE_COMMENT",
    CREATE_PULL_REQUEST: "BITBUCKET_CREATE_PULL_REQUEST",
    CREATE_REPOSITORY: "BITBUCKET_CREATE_REPOSITORY",
    CREATE_SNIPPET_COMMENT: "BITBUCKET_CREATE_SNIPPET_COMMENT",
    DELETE_ISSUE: "BITBUCKET_DELETE_ISSUE",
    DELETE_REPOSITORY: "BITBUCKET_DELETE_REPOSITORY",
    GET_CURRENT_USER: "BITBUCKET_GET_CURRENT_USER",
    GET_FILE_FROM_REPOSITORY: "BITBUCKET_GET_FILE_FROM_REPOSITORY",
    GET_SNIPPET: "BITBUCKET_GET_SNIPPET",
    LIST_PULL_REQUESTS: "BITBUCKET_LIST_PULL_REQUESTS",
    LIST_REPOSITORIES_IN_WORKSPACE: "BITBUCKET_LIST_REPOSITORIES_IN_WORKSPACE",
    LIST_WORKSPACES: "BITBUCKET_LIST_WORKSPACES",
    LIST_WORKSPACE_MEMBERS: "BITBUCKET_LIST_WORKSPACE_MEMBERS",
    UPDATE_ISSUE: "BITBUCKET_UPDATE_ISSUE",
  },
  triggerTypes: {},
}
