// @composio/core/generated/blackboard.ts
/**
 * Map of Composio's BLACKBOARD toolkit.
 */
export const BLACKBOARD = {
  slug: "blackboard",
  tools: {
    ACCESS_COURSE_ANNOUNCEMENTS_BASED_ON_PERMISSIONS: "BLACKBOARD_ACCESS_COURSE_ANNOUNCEMENTS_BASED_ON_PERMISSIONS",
    ADD_CHILD_COURSE: "BLACKBOARD_ADD_CHILD_COURSE",
    ATTACH_FILE: "BLACKBOARD_ATTACH_FILE",
    AUTHORIZATION_CODE: "BLACKBOARD_AUTHORIZATION_CODE",
    CONDITIONAL_GRADE_ACCESS: "BLACKBOARD_CONDITIONAL_GRADE_ACCESS",
    COPY_COURSE: "BLACKBOARD_COPY_COURSE",
    COURSE_AND_ORG_DUPLICATION_PERMISSIONS_UPDATE: "BLACKBOARD_COURSE_AND_ORG_DUPLICATION_PERMISSIONS_UPDATE",
    COURSE_ANNOUNCEMENTS_ACCESS: "BLACKBOARD_COURSE_ANNOUNCEMENTS_ACCESS",
    COURSE_ANNOUNCEMENT_CREATION_PERMISSION: "BLACKBOARD_COURSE_ANNOUNCEMENT_CREATION_PERMISSION",
    COURSE_DATA_ACCESS_CONTROL_BY_USER_ENTITLEMENT: "BLACKBOARD_COURSE_DATA_ACCESS_CONTROL_BY_USER_ENTITLEMENT",
    COURSE_ENDPOINT_ENTITLEMENT_ACCESS: "BLACKBOARD_COURSE_ENDPOINT_ENTITLEMENT_ACCESS",
    COURSE_ENTITLEMENT_AND_DATA_ACCESS_CONTROL: "BLACKBOARD_COURSE_ENTITLEMENT_AND_DATA_ACCESS_CONTROL",
    COURSE_GROUP_INITIATION: "BLACKBOARD_COURSE_GROUP_INITIATION",
    COURSE_GROUP_MODIFICATION: "BLACKBOARD_COURSE_GROUP_MODIFICATION",
    COURSE_MODIFICATION_PERMISSIONS: "BLACKBOARD_COURSE_MODIFICATION_PERMISSIONS",
    COURSE_MODIFICATION_PERMISSION_CHECKS: "BLACKBOARD_COURSE_MODIFICATION_PERMISSION_CHECKS",
    COURSE_ORG_SYNC_REMOVAL: "BLACKBOARD_COURSE_ORG_SYNC_REMOVAL",
    COURSE_OR_ORGANIZATION_CREATION: "BLACKBOARD_COURSE_OR_ORGANIZATION_CREATION",
    COURSE_USER_REMOVAL: "BLACKBOARD_COURSE_USER_REMOVAL",
    CREATE_ADAPTIVE_RELEASE_ACL_USER_PREDICATE: "BLACKBOARD_CREATE_ADAPTIVE_RELEASE_ACL_USER_PREDICATE",
    CREATE_ADAPTIVE_RELEASE_RULE_CRITERIA: "BLACKBOARD_CREATE_ADAPTIVE_RELEASE_RULE_CRITERIA",
    CREATE_ANNOUNCEMENT: "BLACKBOARD_CREATE_ANNOUNCEMENT",
    CREATE_ASSIGNMENT: "BLACKBOARD_CREATE_ASSIGNMENT",
    CREATE_ATTENDANCE_RECORD: "BLACKBOARD_CREATE_ATTENDANCE_RECORD",
    CREATE_CALENDAR_ITEM: "BLACKBOARD_CREATE_CALENDAR_ITEM",
    CREATE_CATEGORY: "BLACKBOARD_CREATE_CATEGORY",
    CREATE_CHILD: "BLACKBOARD_CREATE_CHILD",
    CREATE_CHILD_NODE: "BLACKBOARD_CREATE_CHILD_NODE",
    CREATE_COLUMN_ATTEMPT: "BLACKBOARD_CREATE_COLUMN_ATTEMPT",
    CREATE_COLUMN_GROUP_ATTEMPT: "BLACKBOARD_CREATE_COLUMN_GROUP_ATTEMPT",
    CREATE_CONTENT: "BLACKBOARD_CREATE_CONTENT",
    CREATE_CONTENT_GROUP: "BLACKBOARD_CREATE_CONTENT_GROUP",
    CREATE_COURSE: "BLACKBOARD_CREATE_COURSE",
    CREATE_COURSE_MEETING: "BLACKBOARD_CREATE_COURSE_MEETING",
    CREATE_DATA_SOURCE: "BLACKBOARD_CREATE_DATA_SOURCE",
    CREATE_DISCUSSION: "BLACKBOARD_CREATE_DISCUSSION",
    CREATE_DOMAIN_CONFIG: "BLACKBOARD_CREATE_DOMAIN_CONFIG",
    CREATE_FILE_ATTACHMENT: "BLACKBOARD_CREATE_FILE_ATTACHMENT",
    CREATE_GRADE_COLUMN: "BLACKBOARD_CREATE_GRADE_COLUMN",
    CREATE_GRADE_NOTATION: "BLACKBOARD_CREATE_GRADE_NOTATION",
    CREATE_GRADE_SCHEMA: "BLACKBOARD_CREATE_GRADE_SCHEMA",
    CREATE_GRADING_PERIOD: "BLACKBOARD_CREATE_GRADING_PERIOD",
    CREATE_GROUP: "BLACKBOARD_CREATE_GROUP",
    CREATE_GROUP_MEMBERSHIP: "BLACKBOARD_CREATE_GROUP_MEMBERSHIP",
    CREATE_GROUP_PREDICATE: "BLACKBOARD_CREATE_GROUP_PREDICATE",
    CREATE_GROUP_SET: "BLACKBOARD_CREATE_GROUP_SET",
    CREATE_GROUP_SET_CHILD: "BLACKBOARD_CREATE_GROUP_SET_CHILD",
    CREATE_MEMBERSHIP: "BLACKBOARD_CREATE_MEMBERSHIP",
    CREATE_MESSAGE: "BLACKBOARD_CREATE_MESSAGE",
    CREATE_MESSAGE_REPLY: "BLACKBOARD_CREATE_MESSAGE_REPLY",
    CREATE_NODE: "BLACKBOARD_CREATE_NODE",
    CREATE_NODE_COURSE_ASSOCIATION: "BLACKBOARD_CREATE_NODE_COURSE_ASSOCIATION",
    CREATE_NODE_USER_ASSOCIATION: "BLACKBOARD_CREATE_NODE_USER_ASSOCIATION",
    CREATE_OBSERVER: "BLACKBOARD_CREATE_OBSERVER",
    CREATE_PLACEMENT: "BLACKBOARD_CREATE_PLACEMENT",
    CREATE_PRONOUN: "BLACKBOARD_CREATE_PRONOUN",
    CREATE_QUESTION: "BLACKBOARD_CREATE_QUESTION",
    CREATE_RUBRIC: "BLACKBOARD_CREATE_RUBRIC",
    CREATE_RUBRIC_ASSOCIATION: "BLACKBOARD_CREATE_RUBRIC_ASSOCIATION",
    CREATE_RUBRIC_EVALUATIONS: "BLACKBOARD_CREATE_RUBRIC_EVALUATIONS",
    CREATE_RULE: "BLACKBOARD_CREATE_RULE",
    CREATE_TERM: "BLACKBOARD_CREATE_TERM",
    CREATE_USER: "BLACKBOARD_CREATE_USER",
    DELETE_ACL_USER_PREDICATE: "BLACKBOARD_DELETE_ACL_USER_PREDICATE",
    DELETE_ADAPTIVE_RELEASE_RULE_CRITERION: "BLACKBOARD_DELETE_ADAPTIVE_RELEASE_RULE_CRITERION",
    DELETE_ALL_ATTENDANCE_BY_USER_ID: "BLACKBOARD_DELETE_ALL_ATTENDANCE_BY_USER_ID",
    DELETE_ALL_MEETINGS_IN_COURSE: "BLACKBOARD_DELETE_ALL_MEETINGS_IN_COURSE",
    DELETE_ALL_RECORDS_IN_MEETING: "BLACKBOARD_DELETE_ALL_RECORDS_IN_MEETING",
    DELETE_ANNOUNCEMENT: "BLACKBOARD_DELETE_ANNOUNCEMENT",
    DELETE_ATTACHMENT: "BLACKBOARD_DELETE_ATTACHMENT",
    DELETE_ATTENDANCE_RECORD: "BLACKBOARD_DELETE_ATTENDANCE_RECORD",
    DELETE_CALENDAR_ITEM: "BLACKBOARD_DELETE_CALENDAR_ITEM",
    DELETE_CATEGORY: "BLACKBOARD_DELETE_CATEGORY",
    DELETE_CONTENT: "BLACKBOARD_DELETE_CONTENT",
    DELETE_CONTENT_GROUP: "BLACKBOARD_DELETE_CONTENT_GROUP",
    DELETE_COURSE: "BLACKBOARD_DELETE_COURSE",
    DELETE_COURSE_MEETING: "BLACKBOARD_DELETE_COURSE_MEETING",
    DELETE_DATA_SOURCE: "BLACKBOARD_DELETE_DATA_SOURCE",
    DELETE_DOMAIN_CONFIG: "BLACKBOARD_DELETE_DOMAIN_CONFIG",
    DELETE_FILE_ATTACHMENT: "BLACKBOARD_DELETE_FILE_ATTACHMENT",
    DELETE_GRADE_COLUMN: "BLACKBOARD_DELETE_GRADE_COLUMN",
    DELETE_GRADE_NOTATION: "BLACKBOARD_DELETE_GRADE_NOTATION",
    DELETE_GRADING_PERIOD: "BLACKBOARD_DELETE_GRADING_PERIOD",
    DELETE_GROUP: "BLACKBOARD_DELETE_GROUP",
    DELETE_GROUP_MEMBERSHIP: "BLACKBOARD_DELETE_GROUP_MEMBERSHIP",
    DELETE_GROUP_PREDICATE: "BLACKBOARD_DELETE_GROUP_PREDICATE",
    DELETE_GROUP_SET: "BLACKBOARD_DELETE_GROUP_SET",
    DELETE_MEMBERSHIP: "BLACKBOARD_DELETE_MEMBERSHIP",
    DELETE_MESSAGE: "BLACKBOARD_DELETE_MESSAGE",
    DELETE_NODE: "BLACKBOARD_DELETE_NODE",
    DELETE_NODE_ADMIN: "BLACKBOARD_DELETE_NODE_ADMIN",
    DELETE_NODE_COURSE_ASSOCIATION: "BLACKBOARD_DELETE_NODE_COURSE_ASSOCIATION",
    DELETE_NODE_USER_ASSOCIATION: "BLACKBOARD_DELETE_NODE_USER_ASSOCIATION",
    DELETE_OBSERVER: "BLACKBOARD_DELETE_OBSERVER",
    DELETE_PLACEMENT: "BLACKBOARD_DELETE_PLACEMENT",
    DELETE_PRONOUN: "BLACKBOARD_DELETE_PRONOUN",
    DELETE_QUESTION: "BLACKBOARD_DELETE_QUESTION",
    DELETE_RUBRIC: "BLACKBOARD_DELETE_RUBRIC",
    DELETE_RUBRIC_ASSOCIATION: "BLACKBOARD_DELETE_RUBRIC_ASSOCIATION",
    DELETE_TERM: "BLACKBOARD_DELETE_TERM",
    DELETE_USER: "BLACKBOARD_DELETE_USER",
    DELETE_USER_ATTENDANCE: "BLACKBOARD_DELETE_USER_ATTENDANCE",
    DEPRECATED_GRADE_COLUMN_ACCESS: "BLACKBOARD_DEPRECATED_GRADE_COLUMN_ACCESS",
    DEPRECATE_COURSE_GROUP_DELETE: "BLACKBOARD_DEPRECATE_COURSE_GROUP_DELETE",
    DOWNLOAD: "BLACKBOARD_DOWNLOAD",
    ENHANCE_GRADEBOOK_ENTRIES: "BLACKBOARD_ENHANCE_GRADEBOOK_ENTRIES",
    ENROLL_COURSE_WITH_PERMISSIONS: "BLACKBOARD_ENROLL_COURSE_WITH_PERMISSIONS",
    FETCH_COURSE_CATEGORY_ASSOCIATIONS: "BLACKBOARD_FETCH_COURSE_CATEGORY_ASSOCIATIONS",
    FETCH_COURSE_RUBRIC_ASSESSMENTS: "BLACKBOARD_FETCH_COURSE_RUBRIC_ASSESSMENTS",
    FETCH_GROUP_ATTEMPT_RUBRIC_SCORES: "BLACKBOARD_FETCH_GROUP_ATTEMPT_RUBRIC_SCORES",
    FILTER_GRADES_WITH_EXCLUSIONS_AND_SECURITY: "BLACKBOARD_FILTER_GRADES_WITH_EXCLUSIONS_AND_SECURITY",
    GENERATE_ATTENDANCE_DATA_DOWNLOAD_URL: "BLACKBOARD_GENERATE_ATTENDANCE_DATA_DOWNLOAD_URL",
    GET_ACTIVE_SESSIONS: "BLACKBOARD_GET_ACTIVE_SESSIONS",
    GET_ADAPTIVE_RELEASE_ACL_GROUP_PREDICATES: "BLACKBOARD_GET_ADAPTIVE_RELEASE_ACL_GROUP_PREDICATES",
    GET_ADAPTIVE_RELEASE_ACL_USER_PREDICATES: "BLACKBOARD_GET_ADAPTIVE_RELEASE_ACL_USER_PREDICATES",
    GET_ADAPTIVE_RELEASE_RULE_CRITERION: "BLACKBOARD_GET_ADAPTIVE_RELEASE_RULE_CRITERION",
    GET_ANNOUNCEMENT: "BLACKBOARD_GET_ANNOUNCEMENT",
    GET_ANNOUNCEMENTS: "BLACKBOARD_GET_ANNOUNCEMENTS",
    GET_ATTEMPT_FILE_META_DATA: "BLACKBOARD_GET_ATTEMPT_FILE_META_DATA",
    GET_ATTEMPT_FILE_META_DATA_LIST: "BLACKBOARD_GET_ATTEMPT_FILE_META_DATA_LIST",
    GET_ATTEMPT_RECEIPT: "BLACKBOARD_GET_ATTEMPT_RECEIPT",
    GET_ATTENDANCE_RECORD: "BLACKBOARD_GET_ATTENDANCE_RECORD",
    GET_ATTENDANCE_RECORDS_BY_MEETING_ID: "BLACKBOARD_GET_ATTENDANCE_RECORDS_BY_MEETING_ID",
    GET_ATTENDANCE_RECORDS_BY_USER_ID: "BLACKBOARD_GET_ATTENDANCE_RECORDS_BY_USER_ID",
    GET_CALENDARS: "BLACKBOARD_GET_CALENDARS",
    GET_CALENDAR_ITEM: "BLACKBOARD_GET_CALENDAR_ITEM",
    GET_CALENDAR_ITEMS: "BLACKBOARD_GET_CALENDAR_ITEMS",
    GET_CATEGORIES: "BLACKBOARD_GET_CATEGORIES",
    GET_CATEGORY: "BLACKBOARD_GET_CATEGORY",
    GET_CHILD: "BLACKBOARD_GET_CHILD",
    GET_CHILDREN_GOALS: "BLACKBOARD_GET_CHILDREN_GOALS",
    GET_CHILD_CATEGORIES: "BLACKBOARD_GET_CHILD_CATEGORIES",
    GET_COLUMN_ATTEMPT: "BLACKBOARD_GET_COLUMN_ATTEMPT",
    GET_COLUMN_ATTEMPTS: "BLACKBOARD_GET_COLUMN_ATTEMPTS",
    GET_COLUMN_GRADE: "BLACKBOARD_GET_COLUMN_GRADE",
    GET_COLUMN_GRADES: "BLACKBOARD_GET_COLUMN_GRADES",
    GET_COLUMN_GRADE_LAST_CHANGED: "BLACKBOARD_GET_COLUMN_GRADE_LAST_CHANGED",
    GET_COLUMN_GROUP_ATTEMPTS: "BLACKBOARD_GET_COLUMN_GROUP_ATTEMPTS",
    GET_CONTENT: "BLACKBOARD_GET_CONTENT",
    GET_CONTENTS: "BLACKBOARD_GET_CONTENTS",
    GET_CONTENT_CHILDREN: "BLACKBOARD_GET_CONTENT_CHILDREN",
    GET_CONTENT_COLLECTION_RESOURCE: "BLACKBOARD_GET_CONTENT_COLLECTION_RESOURCE",
    GET_CONTENT_GROUP: "BLACKBOARD_GET_CONTENT_GROUP",
    GET_CONTENT_GROUPS: "BLACKBOARD_GET_CONTENT_GROUPS",
    GET_COURSE: "BLACKBOARD_GET_COURSE",
    GET_COURSES: "BLACKBOARD_GET_COURSES",
    GET_COURSE_CHILDREN: "BLACKBOARD_GET_COURSE_CHILDREN",
    GET_COURSE_COLUMN_LOGS: "BLACKBOARD_GET_COURSE_COLUMN_LOGS",
    GET_COURSE_GOAL_ALIGNMENTS: "BLACKBOARD_GET_COURSE_GOAL_ALIGNMENTS",
    GET_COURSE_GRADEBOOK_LOGS: "BLACKBOARD_GET_COURSE_GRADEBOOK_LOGS",
    GET_COURSE_MEETING: "BLACKBOARD_GET_COURSE_MEETING",
    GET_COURSE_MEETINGS: "BLACKBOARD_GET_COURSE_MEETINGS",
    GET_COURSE_MEMBERSHIPS: "BLACKBOARD_GET_COURSE_MEMBERSHIPS",
    GET_COURSE_RESOURCE: "BLACKBOARD_GET_COURSE_RESOURCE",
    GET_COURSE_RESOURCE_CHILDREN: "BLACKBOARD_GET_COURSE_RESOURCE_CHILDREN",
    GET_COURSE_ROLE: "BLACKBOARD_GET_COURSE_ROLE",
    GET_COURSE_ROLES: "BLACKBOARD_GET_COURSE_ROLES",
    GET_CRITERIA: "BLACKBOARD_GET_CRITERIA",
    GET_CROSS_LIST_SET: "BLACKBOARD_GET_CROSS_LIST_SET",
    GET_CURRENT_ACTIVE_USER_BY_ID: "BLACKBOARD_GET_CURRENT_ACTIVE_USER_BY_ID",
    GET_DATA_SOURCE: "BLACKBOARD_GET_DATA_SOURCE",
    GET_DATA_SOURCES: "BLACKBOARD_GET_DATA_SOURCES",
    GET_DISCUSSION: "BLACKBOARD_GET_DISCUSSION",
    GET_DISCUSSIONS: "BLACKBOARD_GET_DISCUSSIONS",
    GET_DISCUSSION_MESSAGES: "BLACKBOARD_GET_DISCUSSION_MESSAGES",
    GET_DOMAIN_CONFIG: "BLACKBOARD_GET_DOMAIN_CONFIG",
    GET_DOMAIN_CONFIGS: "BLACKBOARD_GET_DOMAIN_CONFIGS",
    GET_FILE_ATTACHMENT: "BLACKBOARD_GET_FILE_ATTACHMENT",
    GET_FILE_ATTACHMENTS: "BLACKBOARD_GET_FILE_ATTACHMENTS",
    GET_FOLDERS: "BLACKBOARD_GET_FOLDERS",
    GET_GOALS: "BLACKBOARD_GET_GOALS",
    GET_GOALS_FROM_SET_AND_CATEGORY: "BLACKBOARD_GET_GOALS_FROM_SET_AND_CATEGORY",
    GET_GOAL_ALIGNMENTS: "BLACKBOARD_GET_GOAL_ALIGNMENTS",
    GET_GOAL_BY_ID: "BLACKBOARD_GET_GOAL_BY_ID",
    GET_GOAL_SETS: "BLACKBOARD_GET_GOAL_SETS",
    GET_GOAL_SET_BY_ID: "BLACKBOARD_GET_GOAL_SET_BY_ID",
    GET_GOAL_SET_CATEGORY_BY_ID: "BLACKBOARD_GET_GOAL_SET_CATEGORY_BY_ID",
    GET_GRADEBOOK_CATEGORIES: "BLACKBOARD_GET_GRADEBOOK_CATEGORIES",
    GET_GRADEBOOK_CATEGORY: "BLACKBOARD_GET_GRADEBOOK_CATEGORY",
    GET_GRADE_COLUMN: "BLACKBOARD_GET_GRADE_COLUMN",
    GET_GRADE_COLUMNS: "BLACKBOARD_GET_GRADE_COLUMNS",
    GET_GRADE_NOTATION: "BLACKBOARD_GET_GRADE_NOTATION",
    GET_GRADE_NOTATIONS: "BLACKBOARD_GET_GRADE_NOTATIONS",
    GET_GRADE_SCHEMAS: "BLACKBOARD_GET_GRADE_SCHEMAS",
    GET_GRADING_PERIOD: "BLACKBOARD_GET_GRADING_PERIOD",
    GET_GRADING_PERIODS: "BLACKBOARD_GET_GRADING_PERIODS",
    GET_GROUP: "BLACKBOARD_GET_GROUP",
    GET_GROUPS: "BLACKBOARD_GET_GROUPS",
    GET_GROUP_MEMBERSHIP: "BLACKBOARD_GET_GROUP_MEMBERSHIP",
    GET_GROUP_MEMBERSHIPS: "BLACKBOARD_GET_GROUP_MEMBERSHIPS",
    GET_GROUP_SET: "BLACKBOARD_GET_GROUP_SET",
    GET_GROUP_SETS: "BLACKBOARD_GET_GROUP_SETS",
    GET_GROUP_SET_CHILDREN: "BLACKBOARD_GET_GROUP_SET_CHILDREN",
    GET_INFO: "BLACKBOARD_GET_INFO",
    GET_INSTITUTION_ROLE: "BLACKBOARD_GET_INSTITUTION_ROLE",
    GET_INSTITUTION_ROLES: "BLACKBOARD_GET_INSTITUTION_ROLES",
    GET_LOGIN_AS_SESSIONS: "BLACKBOARD_GET_LOGIN_AS_SESSIONS",
    GET_MEMBERSHIP: "BLACKBOARD_GET_MEMBERSHIP",
    GET_MEMBERSHIPS: "BLACKBOARD_GET_MEMBERSHIPS",
    GET_MESSAGES: "BLACKBOARD_GET_MESSAGES",
    GET_MESSAGE_PARTICIPANTS: "BLACKBOARD_GET_MESSAGE_PARTICIPANTS",
    GET_MESSAGE_REPLIES: "BLACKBOARD_GET_MESSAGE_REPLIES",
    GET_NODE: "BLACKBOARD_GET_NODE",
    GET_NODES: "BLACKBOARD_GET_NODES",
    GET_NODES_FOR_COURSE: "BLACKBOARD_GET_NODES_FOR_COURSE",
    GET_NODES_FOR_USER: "BLACKBOARD_GET_NODES_FOR_USER",
    GET_NODE_ADMIN: "BLACKBOARD_GET_NODE_ADMIN",
    GET_NODE_ADMINS: "BLACKBOARD_GET_NODE_ADMINS",
    GET_NODE_CHILDREN: "BLACKBOARD_GET_NODE_CHILDREN",
    GET_NODE_COURSE_ASSOCIATIONS: "BLACKBOARD_GET_NODE_COURSE_ASSOCIATIONS",
    GET_NODE_USER_ASSOCIATIONS: "BLACKBOARD_GET_NODE_USER_ASSOCIATIONS",
    GET_OBSERVEES: "BLACKBOARD_GET_OBSERVEES",
    GET_OBSERVERS: "BLACKBOARD_GET_OBSERVERS",
    GET_PLACEMENT: "BLACKBOARD_GET_PLACEMENT",
    GET_PLACEMENTS: "BLACKBOARD_GET_PLACEMENTS",
    GET_POLICIES: "BLACKBOARD_GET_POLICIES",
    GET_PROCTORING_SERVICE: "BLACKBOARD_GET_PROCTORING_SERVICE",
    GET_PROCTORING_SERVICES: "BLACKBOARD_GET_PROCTORING_SERVICES",
    GET_PRONOUNS: "BLACKBOARD_GET_PRONOUNS",
    GET_QUESTIONS: "BLACKBOARD_GET_QUESTIONS",
    GET_QUESTION_BY_ID: "BLACKBOARD_GET_QUESTION_BY_ID",
    GET_RESOURCES: "BLACKBOARD_GET_RESOURCES",
    GET_RESOURCE_CHILDREN: "BLACKBOARD_GET_RESOURCE_CHILDREN",
    GET_REVIEW_STATUS: "BLACKBOARD_GET_REVIEW_STATUS",
    GET_REVIEW_STATUS_BY_COURSE_ID: "BLACKBOARD_GET_REVIEW_STATUS_BY_COURSE_ID",
    GET_RUBRICS: "BLACKBOARD_GET_RUBRICS",
    GET_RUBRIC_ASSOCIATIONS: "BLACKBOARD_GET_RUBRIC_ASSOCIATIONS",
    GET_RUBRIC_ASSOCIATIONS_BY_COLUMN_ID: "BLACKBOARD_GET_RUBRIC_ASSOCIATIONS_BY_COLUMN_ID",
    GET_RUBRIC_ASSOCIATION_BY_ID: "BLACKBOARD_GET_RUBRIC_ASSOCIATION_BY_ID",
    GET_RUBRIC_BY_ID: "BLACKBOARD_GET_RUBRIC_BY_ID",
    GET_RUBRIC_EVALUATIONS: "BLACKBOARD_GET_RUBRIC_EVALUATIONS",
    GET_RUBRIC_EVALUATION_BY_ID: "BLACKBOARD_GET_RUBRIC_EVALUATION_BY_ID",
    GET_RULES: "BLACKBOARD_GET_RULES",
    GET_SETTINGS: "BLACKBOARD_GET_SETTINGS",
    GET_SIS_LOGS_BY_DATA_SET_UID: "BLACKBOARD_GET_SIS_LOGS_BY_DATA_SET_UID",
    GET_SYSTEM_ROLE: "BLACKBOARD_GET_SYSTEM_ROLE",
    GET_SYSTEM_ROLES: "BLACKBOARD_GET_SYSTEM_ROLES",
    GET_SYSTEM_TASK: "BLACKBOARD_GET_SYSTEM_TASK",
    GET_TASK: "BLACKBOARD_GET_TASK",
    GET_TERM: "BLACKBOARD_GET_TERM",
    GET_TERMS: "BLACKBOARD_GET_TERMS",
    GET_TOC_ITEMS: "BLACKBOARD_GET_TOC_ITEMS",
    GET_TOKEN_INFO: "BLACKBOARD_GET_TOKEN_INFO",
    GET_TOOLS: "BLACKBOARD_GET_TOOLS",
    GET_TOP_LEVEL_COURSE_RESOURCES: "BLACKBOARD_GET_TOP_LEVEL_COURSE_RESOURCES",
    GET_TYPES: "BLACKBOARD_GET_TYPES",
    GET_USER: "BLACKBOARD_GET_USER",
    GET_USERS: "BLACKBOARD_GET_USERS",
    GET_USER_AVATAR: "BLACKBOARD_GET_USER_AVATAR",
    GET_USER_GRADES: "BLACKBOARD_GET_USER_GRADES",
    GET_USER_MEMBERSHIPS: "BLACKBOARD_GET_USER_MEMBERSHIPS",
    GET_USER_PRONUNCIATION_AUDIO: "BLACKBOARD_GET_USER_PRONUNCIATION_AUDIO",
    GET_VERSION: "BLACKBOARD_GET_VERSION",
    GRADE_ACCESS_WITH_RIGHTS_VALIDATION: "BLACKBOARD_GRADE_ACCESS_WITH_RIGHTS_VALIDATION",
    GROUP_MEMBERSHIP_API_DETAILS: "BLACKBOARD_GROUP_MEMBERSHIP_API_DETAILS",
    INITIALIZE_EDUCATIONAL_ENTITY: "BLACKBOARD_INITIALIZE_EDUCATIONAL_ENTITY",
    INITIATE_GROUP_MEMBER_ADDITION: "BLACKBOARD_INITIATE_GROUP_MEMBER_ADDITION",
    LIST_GOAL_SET_CATEGORIES: "BLACKBOARD_LIST_GOAL_SET_CATEGORIES",
    LIST_GRADEBOOK_COLUMNS: "BLACKBOARD_LIST_GRADEBOOK_COLUMNS",
    LOAD_COURSE_GROUP_DETAILS: "BLACKBOARD_LOAD_COURSE_GROUP_DETAILS",
    LOAD_COURSE_GROUP_USER_INFO: "BLACKBOARD_LOAD_COURSE_GROUP_USER_INFO",
    MANAGE_COURSE_DATA_ACCESS: "BLACKBOARD_MANAGE_COURSE_DATA_ACCESS",
    MANUAL_GRADE_COLUMN_CREATION: "BLACKBOARD_MANUAL_GRADE_COLUMN_CREATION",
    MANUAL_GRADE_COLUMN_MODIFICATION: "BLACKBOARD_MANUAL_GRADE_COLUMN_MODIFICATION",
    MODIFY_COURSE_ANNOUNCEMENT_ACCESS: "BLACKBOARD_MODIFY_COURSE_ANNOUNCEMENT_ACCESS",
    PATCH_TOOL: "BLACKBOARD_PATCH_TOOL",
    POST_ULTRA_COURSE_DISCUSSION_REPLY: "BLACKBOARD_POST_ULTRA_COURSE_DISCUSSION_REPLY",
    REMOVE_CHILD_COURSE: "BLACKBOARD_REMOVE_CHILD_COURSE",
    REMOVE_COURSE_GROUP: "BLACKBOARD_REMOVE_COURSE_GROUP",
    REMOVE_COURSE_NOTICE: "BLACKBOARD_REMOVE_COURSE_NOTICE",
    REMOVE_SPECIFIC_GRADE_FIELD: "BLACKBOARD_REMOVE_SPECIFIC_GRADE_FIELD",
    REMOVE_ULTRA_COURSE_DISCUSSION_POST: "BLACKBOARD_REMOVE_ULTRA_COURSE_DISCUSSION_POST",
    REQUEST_TOKEN: "BLACKBOARD_REQUEST_TOKEN",
    RETRIEVE_STUDENT_SUBMISSION_FILE: "BLACKBOARD_RETRIEVE_STUDENT_SUBMISSION_FILE",
    REVISE_ULTRA_COURSE_DISCUSSION_POST: "BLACKBOARD_REVISE_ULTRA_COURSE_DISCUSSION_POST",
    ROLE_BASED_GRADE_ATTEMPT_VISIBILITY: "BLACKBOARD_ROLE_BASED_GRADE_ATTEMPT_VISIBILITY",
    SECURE_GRADE_ATTEMPT_ACCESS_WITH_VERSION_CONTROL: "BLACKBOARD_SECURE_GRADE_ATTEMPT_ACCESS_WITH_VERSION_CONTROL",
    SET_NODE_ADMIN_ASSOCIATION: "BLACKBOARD_SET_NODE_ADMIN_ASSOCIATION",
    SYNCHRONOUS_COURSE_OR_ORG_REMOVAL: "BLACKBOARD_SYNCHRONOUS_COURSE_OR_ORG_REMOVAL",
    UPDATE_ADAPTIVE_RELEASE_RULE_CRITERION: "BLACKBOARD_UPDATE_ADAPTIVE_RELEASE_RULE_CRITERION",
    UPDATE_ANNOUNCEMENT: "BLACKBOARD_UPDATE_ANNOUNCEMENT",
    UPDATE_ATTENDANCE_RECORD: "BLACKBOARD_UPDATE_ATTENDANCE_RECORD",
    UPDATE_ATTENDANCE_RECORDS: "BLACKBOARD_UPDATE_ATTENDANCE_RECORDS",
    UPDATE_CALENDAR_ITEM: "BLACKBOARD_UPDATE_CALENDAR_ITEM",
    UPDATE_CATEGORY: "BLACKBOARD_UPDATE_CATEGORY",
    UPDATE_COLUMN_ATTEMPT: "BLACKBOARD_UPDATE_COLUMN_ATTEMPT",
    UPDATE_COLUMN_GRADE: "BLACKBOARD_UPDATE_COLUMN_GRADE",
    UPDATE_CONTENT: "BLACKBOARD_UPDATE_CONTENT",
    UPDATE_COURSE: "BLACKBOARD_UPDATE_COURSE",
    UPDATE_COURSE_MEETING: "BLACKBOARD_UPDATE_COURSE_MEETING",
    UPDATE_DATA_SOURCE: "BLACKBOARD_UPDATE_DATA_SOURCE",
    UPDATE_DISCUSSION: "BLACKBOARD_UPDATE_DISCUSSION",
    UPDATE_DOMAIN_CONFIG: "BLACKBOARD_UPDATE_DOMAIN_CONFIG",
    UPDATE_GRADE_COLUMN: "BLACKBOARD_UPDATE_GRADE_COLUMN",
    UPDATE_GRADE_NOTATION: "BLACKBOARD_UPDATE_GRADE_NOTATION",
    UPDATE_GRADING_PERIOD: "BLACKBOARD_UPDATE_GRADING_PERIOD",
    UPDATE_GROUP: "BLACKBOARD_UPDATE_GROUP",
    UPDATE_GROUP_SET: "BLACKBOARD_UPDATE_GROUP_SET",
    UPDATE_MEMBERSHIP: "BLACKBOARD_UPDATE_MEMBERSHIP",
    UPDATE_MESSAGE: "BLACKBOARD_UPDATE_MESSAGE",
    UPDATE_NODE: "BLACKBOARD_UPDATE_NODE",
    UPDATE_NODE_COURSE_ASSOCIATION: "BLACKBOARD_UPDATE_NODE_COURSE_ASSOCIATION",
    UPDATE_PLACEMENT: "BLACKBOARD_UPDATE_PLACEMENT",
    UPDATE_PRONOUN: "BLACKBOARD_UPDATE_PRONOUN",
    UPDATE_QUESTION: "BLACKBOARD_UPDATE_QUESTION",
    UPDATE_REVIEW_STATUS: "BLACKBOARD_UPDATE_REVIEW_STATUS",
    UPDATE_RUBRIC: "BLACKBOARD_UPDATE_RUBRIC",
    UPDATE_RUBRIC_ASSOCIATION: "BLACKBOARD_UPDATE_RUBRIC_ASSOCIATION",
    UPDATE_RUBRIC_EVALUATION: "BLACKBOARD_UPDATE_RUBRIC_EVALUATION",
    UPDATE_TERM: "BLACKBOARD_UPDATE_TERM",
    UPDATE_TOC_ITEM: "BLACKBOARD_UPDATE_TOC_ITEM",
    UPDATE_USER: "BLACKBOARD_UPDATE_USER",
    UPLOAD: "BLACKBOARD_UPLOAD",
    VIEW_COURSE_GROUP_ACCESSIBILITIES: "BLACKBOARD_VIEW_COURSE_GROUP_ACCESSIBILITIES",
  },
  triggerTypes: {},
}
