// @composio/core/generated/bolna.ts
/**
 * Map of Composio's BOLNA toolkit.
 */
export const BOLNA = {
  slug: "bolna",
  tools: {
    DELETE_AGENT_BY_ID: "BOLNA_DELETE_AGENT_BY_ID",
    DELETE_BATCH_BY_ID: "BOLNA_DELETE_BATCH_BY_ID",
    FETCH_ALL_BATCHES_BY_AGENT_ID: "BOLNA_FETCH_ALL_BATCHES_BY_AGENT_ID",
    GET_ALL_AGENTS: "BOLNA_GET_ALL_AGENTS",
    GET_ALL_PHONE_NUMBERS: "BOLNA_GET_ALL_PHONE_NUMBERS",
    GET_EXECUTION_BY_ID: "BOLNA_GET_EXECUTION_BY_ID",
    MAKE_A_PHONE_CALL_FROM_AGENT: "BOLNA_MAKE_A_PHONE_CALL_FROM_AGENT",
    RETRIEVE_AGENT_BY_ID: "BOLNA_RETRIEVE_AGENT_BY_ID",
    RETRIEVE_AGENT_EXECUTION_DETAILS: "BOLNA_RETRIEVE_AGENT_EXECUTION_DETAILS",
    RETRIEVE_AGENT_EXECUTION_STATUS: "BOLNA_RETRIEVE_AGENT_EXECUTION_STATUS",
    RETRIEVE_BATCH_DETAILS_BY_ID: "BOLNA_RETRIEVE_BATCH_DETAILS_BY_ID",
    RETRIEVE_BATCH_EXECUTION_LIST: "BOLNA_RETRIEVE_BATCH_EXECUTION_LIST",
    SCHEDULE_BATCH_BY_ID: "BOLNA_SCHEDULE_BATCH_BY_ID",
    SETUP_INBOUND_CALL_FOR_AGENT: "BOLNA_SETUP_INBOUND_CALL_FOR_AGENT",
    STOP_BATCH_BY_ID: "BOLNA_STOP_BATCH_BY_ID",
  },
  triggerTypes: {},
}
