// @composio/core/generated/borneo.ts
/**
 * Map of Composio's BORNEO toolkit.
 */
export const BORNEO = {
  slug: "borneo",
  tools: {
    ACCESS_SCAN_ITERATION_BY_ID: "BORNEO_ACCESS_SCAN_ITERATION_BY_ID",
    ADD_DISCOVERED_RECIPIENTS: "BORNEO_ADD_DISCOVERED_RECIPIENTS",
    ARCHIVE_DISCOVERED_RECIPIENT: "BORNEO_ARCHIVE_DISCOVERED_RECIPIENT",
    CREATE_AND_SCHEDULE_CLOUD_RESOURCE_SCAN: "BORNEO_CREATE_AND_SCHEDULE_CLOUD_RESOURCE_SCAN",
    CREATE_DASHBOARD_USER: "BORNEO_CREATE_DASHBOARD_USER",
    CREATE_DEPARTMENT_WITH_TRANSLATIONS: "BORNEO_CREATE_DEPARTMENT_WITH_TRANSLATIONS",
    CREATE_DOMAIN_WITH_POLLING_FREQUENCY: "BORNEO_CREATE_DOMAIN_WITH_POLLING_FREQUENCY",
    CREATE_DPIA_FOR_PROCESSING_ACTIVITY: "BORNEO_CREATE_DPIA_FOR_PROCESSING_ACTIVITY",
    CREATE_EMPLOYEE_WITH_JSON_PAYLOAD: "BORNEO_CREATE_EMPLOYEE_WITH_JSON_PAYLOAD",
    CREATE_HEADQUARTER_ENTRY: "BORNEO_CREATE_HEADQUARTER_ENTRY",
    CREATE_LEGAL_DOCUMENT_ENTRY: "BORNEO_CREATE_LEGAL_DOCUMENT_ENTRY",
    CREATE_NEW_ASSET: "BORNEO_CREATE_NEW_ASSET",
    CREATE_NEW_INFOTYPE_CATEGORY: "BORNEO_CREATE_NEW_INFOTYPE_CATEGORY",
    CREATE_PROCESSING_ACTIVITY: "BORNEO_CREATE_PROCESSING_ACTIVITY",
    CREATE_PROCESSING_ACTIVITY_THRESHOLD: "BORNEO_CREATE_PROCESSING_ACTIVITY_THRESHOLD",
    CREATE_RECIPIENT_WITH_DETAILS: "BORNEO_CREATE_RECIPIENT_WITH_DETAILS",
    CREATE_THRESHOLD_FOR_PROCESSING_ACTIVITY: "BORNEO_CREATE_THRESHOLD_FOR_PROCESSING_ACTIVITY",
    DELETE_ASSET_BY_ID: "BORNEO_DELETE_ASSET_BY_ID",
    DELETE_CATEGORY_BY_LABEL: "BORNEO_DELETE_CATEGORY_BY_LABEL",
    DELETE_DASHBOARD_REPORT_BY_ID: "BORNEO_DELETE_DASHBOARD_REPORT_BY_ID",
    DELETE_DATA_BREACH_BY_ID: "BORNEO_DELETE_DATA_BREACH_BY_ID",
    DELETE_DEPARTMENT_BY_ID: "BORNEO_DELETE_DEPARTMENT_BY_ID",
    DELETE_DOMAIN_BY_ID: "BORNEO_DELETE_DOMAIN_BY_ID",
    DELETE_DPIA_BY_ID: "BORNEO_DELETE_DPIA_BY_ID",
    DELETE_EMPLOYEE_BY_ID: "BORNEO_DELETE_EMPLOYEE_BY_ID",
    DELETE_HEADQUARTERS_BY_ID: "BORNEO_DELETE_HEADQUARTERS_BY_ID",
    DELETE_LEGAL_DOCUMENT_BY_ID: "BORNEO_DELETE_LEGAL_DOCUMENT_BY_ID",
    DELETE_LOPDP_THRESHOLD_BY_ID: "BORNEO_DELETE_LOPDP_THRESHOLD_BY_ID",
    DELETE_PROCESSING_ACTIVITY_BY_ID: "BORNEO_DELETE_PROCESSING_ACTIVITY_BY_ID",
    DELETE_RECIPIENT_BY_ID: "BORNEO_DELETE_RECIPIENT_BY_ID",
    DELETE_TAG_FROM_RESOURCE: "BORNEO_DELETE_TAG_FROM_RESOURCE",
    DELETE_THRESHOLD_BY_ID: "BORNEO_DELETE_THRESHOLD_BY_ID",
    DISABLE_DASHBOARD_USER_BY_USERNAME: "BORNEO_DISABLE_DASHBOARD_USER_BY_USERNAME",
    DOWNLOAD_DASHBOARD_REPORT: "BORNEO_DOWNLOAD_DASHBOARD_REPORT",
    DOWNLOAD_DASHBOARD_REPORT_EDITION: "BORNEO_DOWNLOAD_DASHBOARD_REPORT_EDITION",
    ENABLE_DASHBOARD_USER: "BORNEO_ENABLE_DASHBOARD_USER",
    EVALUATE_DATA_BREACH_IMPACT: "BORNEO_EVALUATE_DATA_BREACH_IMPACT",
    EXPORT_FILTERED_LEAF_RESOURCES: "BORNEO_EXPORT_FILTERED_LEAF_RESOURCES",
    EXPORT_INSIGHT_PAGE_USING_SCAN_ID: "BORNEO_EXPORT_INSIGHT_PAGE_USING_SCAN_ID",
    EXPORT_INVENTORY_RESOURCE_LIST: "BORNEO_EXPORT_INVENTORY_RESOURCE_LIST",
    EXPORT_PROCESSING_ACTIVITIES_LIST: "BORNEO_EXPORT_PROCESSING_ACTIVITIES_LIST",
    EXPORT_RECIPIENTS_LIST_WITH_FILTER: "BORNEO_EXPORT_RECIPIENTS_LIST_WITH_FILTER",
    FETCH_DASHBOARD_REPORT_BY_ID: "BORNEO_FETCH_DASHBOARD_REPORT_BY_ID",
    FETCH_DATA_BREACH_EVALUATION: "BORNEO_FETCH_DATA_BREACH_EVALUATION",
    FILTER_AND_LIST_INSPECTION_RESULTS: "BORNEO_FILTER_AND_LIST_INSPECTION_RESULTS",
    FILTER_AND_SORT_ASSETS_LIST: "BORNEO_FILTER_AND_SORT_ASSETS_LIST",
    FILTER_EMPLOYEE_LIST: "BORNEO_FILTER_EMPLOYEE_LIST",
    FILTER_RECIPIENTS_LIST: "BORNEO_FILTER_RECIPIENTS_LIST",
    GET_CATEGORY_BY_LABEL: "BORNEO_GET_CATEGORY_BY_LABEL",
    GET_CLOUD_ACCOUNT_BY_ID: "BORNEO_GET_CLOUD_ACCOUNT_BY_ID",
    GET_DASHBOARD_REPORT_EDITION_BY_ID: "BORNEO_GET_DASHBOARD_REPORT_EDITION_BY_ID",
    GET_DEPARTMENT_FILTER_LIST: "BORNEO_GET_DEPARTMENT_FILTER_LIST",
    GET_DOMAIN_BY_ID: "BORNEO_GET_DOMAIN_BY_ID",
    GET_HEADQUARTERS_BY_ID: "BORNEO_GET_HEADQUARTERS_BY_ID",
    GET_INSIGHT_BY_TYPE_AND_ID: "BORNEO_GET_INSIGHT_BY_TYPE_AND_ID",
    GET_RESOURCE_INVENTORY_BY_ID: "BORNEO_GET_RESOURCE_INVENTORY_BY_ID",
    GET_SCAN_BY_SCAN_ID: "BORNEO_GET_SCAN_BY_SCAN_ID",
    GET_THRESHOLD_BY_ID: "BORNEO_GET_THRESHOLD_BY_ID",
    GET_USER_PROFILE_BY_ID: "BORNEO_GET_USER_PROFILE_BY_ID",
    LIST_DASHBOARD_REPORTS_WITH_FILTERS: "BORNEO_LIST_DASHBOARD_REPORTS_WITH_FILTERS",
    LIST_DASHBOARD_REPORT_EDITIONS: "BORNEO_LIST_DASHBOARD_REPORT_EDITIONS",
    LIST_DASHBOARD_USERS_WITH_FILTERS: "BORNEO_LIST_DASHBOARD_USERS_WITH_FILTERS",
    LIST_DATA_BREACHES_WITH_FILTERS: "BORNEO_LIST_DATA_BREACHES_WITH_FILTERS",
    LIST_DATA_BREACH_FILTERS: "BORNEO_LIST_DATA_BREACH_FILTERS",
    LIST_DEPARTMENTS_WITH_SORT_AND_PAGINATION: "BORNEO_LIST_DEPARTMENTS_WITH_SORT_AND_PAGINATION",
    LIST_DISCOVERED_DOCUMENT: "BORNEO_LIST_DISCOVERED_DOCUMENT",
    LIST_DISCOVERED_INFOTYPES: "BORNEO_LIST_DISCOVERED_INFOTYPES",
    LIST_DISCOVERED_RECIPIENTS: "BORNEO_LIST_DISCOVERED_RECIPIENTS",
    LIST_DOMAINS_WITH_PAGINATION_AND_SORTING: "BORNEO_LIST_DOMAINS_WITH_PAGINATION_AND_SORTING",
    LIST_EMPLOYEES_WITH_FILTERS: "BORNEO_LIST_EMPLOYEES_WITH_FILTERS",
    LIST_ERROR_DETAILS_FROM_FILTERED_SCAN_ITERATIONS: "BORNEO_LIST_ERROR_DETAILS_FROM_FILTERED_SCAN_ITERATIONS",
    LIST_EVENTS_WITH_FILTERS: "BORNEO_LIST_EVENTS_WITH_FILTERS",
    LIST_FILTERED_SORTED_CATEGORIES: "BORNEO_LIST_FILTERED_SORTED_CATEGORIES",
    LIST_FILTER_OPTIONS_FOR_RECIPIENTS: "BORNEO_LIST_FILTER_OPTIONS_FOR_RECIPIENTS",
    LIST_HEADQUARTERS_WITH_SORTING: "BORNEO_LIST_HEADQUARTERS_WITH_SORTING",
    LIST_INSIGHT_FILTERS: "BORNEO_LIST_INSIGHT_FILTERS",
    LIST_INVENTORY_RESOURCES_WITH_FILTERS: "BORNEO_LIST_INVENTORY_RESOURCES_WITH_FILTERS",
    LIST_ISSUES_WITH_FILTERS: "BORNEO_LIST_ISSUES_WITH_FILTERS",
    LIST_LEAF_RESOURCES_WITH_FILTERS: "BORNEO_LIST_LEAF_RESOURCES_WITH_FILTERS",
    LIST_LEGAL_DOCUMENTS_WITH_PAGINATION: "BORNEO_LIST_LEGAL_DOCUMENTS_WITH_PAGINATION",
    LIST_OR_FILTER_RECIPIENTS: "BORNEO_LIST_OR_FILTER_RECIPIENTS",
    LIST_PROCESSING_ACTIVITIES: "BORNEO_LIST_PROCESSING_ACTIVITIES",
    LIST_PROCESSING_ACTIVITIES_FILTERS: "BORNEO_LIST_PROCESSING_ACTIVITIES_FILTERS",
    LIST_SCANS_WITH_FILTERS: "BORNEO_LIST_SCANS_WITH_FILTERS",
    LIST_SCAN_EXECUTION_RESULTS: "BORNEO_LIST_SCAN_EXECUTION_RESULTS",
    LIST_SCAN_ITERATIONS_WITH_FILTER: "BORNEO_LIST_SCAN_ITERATIONS_WITH_FILTER",
    LIST_TOMS_WITH_FILTER_AND_PAGINATION_OPTIONS: "BORNEO_LIST_TOMS_WITH_FILTER_AND_PAGINATION_OPTIONS",
    LIST_USER_PROFILE_WITH_FILTERS_AND_SORTING: "BORNEO_LIST_USER_PROFILE_WITH_FILTERS_AND_SORTING",
    MARK_SCAN_FALSE_POSITIVES_BY_ID: "BORNEO_MARK_SCAN_FALSE_POSITIVES_BY_ID",
    PAUSE_SCAN_BY_ID: "BORNEO_PAUSE_SCAN_BY_ID",
    POLL_DOMAIN_BY_ID: "BORNEO_POLL_DOMAIN_BY_ID",
    POST_ACCOUNTS_WITH_FILTER_AND_SORT_OPTIONS: "BORNEO_POST_ACCOUNTS_WITH_FILTER_AND_SORT_OPTIONS",
    POST_CLASSIFICATION_STATS: "BORNEO_POST_CLASSIFICATION_STATS",
    POST_CONNECTOR_WITH_FILTERING_OPTIONS: "BORNEO_POST_CONNECTOR_WITH_FILTERING_OPTIONS",
    POST_CURRENT_DASHBOARD_USER: "BORNEO_POST_CURRENT_DASHBOARD_USER",
    POST_DASHBOARD_REPORT: "BORNEO_POST_DASHBOARD_REPORT",
    POST_DATA_BREACH_INFORMATION: "BORNEO_POST_DATA_BREACH_INFORMATION",
    POST_DISCOVERED_RECIPIENT_BY_ID: "BORNEO_POST_DISCOVERED_RECIPIENT_BY_ID",
    POST_FILTERED_ACCESS_LOGS: "BORNEO_POST_FILTERED_ACCESS_LOGS",
    POST_LOG_AUDIT_RECORDS_WITH_FILTER_CRITERIA: "BORNEO_POST_LOG_AUDIT_RECORDS_WITH_FILTER_CRITERIA",
    POST_RESOURCE_LINEAGE_FILTER: "BORNEO_POST_RESOURCE_LINEAGE_FILTER",
    POST_RESOURCE_STATS_WITH_DELETED_RESOURCES: "BORNEO_POST_RESOURCE_STATS_WITH_DELETED_RESOURCES",
    POST_SCAN_RESOURCE_STATUS: "BORNEO_POST_SCAN_RESOURCE_STATUS",
    POST_SUPPORT_CHAT_QUERY: "BORNEO_POST_SUPPORT_CHAT_QUERY",
    PUT_TOM_STATUS_AND_NOTE: "BORNEO_PUT_TOM_STATUS_AND_NOTE",
    REMOVE_DASHBOARD_USER_BY_USERNAME: "BORNEO_REMOVE_DASHBOARD_USER_BY_USERNAME",
    RESET_DASHBOARD_USER_PASSWORD: "BORNEO_RESET_DASHBOARD_USER_PASSWORD",
    RESUME_SCAN_BY_ID: "BORNEO_RESUME_SCAN_BY_ID",
    RETRIEVE_ACCOUNT_DETAILS_BY_ID: "BORNEO_RETRIEVE_ACCOUNT_DETAILS_BY_ID",
    RETRIEVE_ASSET_BY_ID: "BORNEO_RETRIEVE_ASSET_BY_ID",
    RETRIEVE_CONNECTOR_BY_ID: "BORNEO_RETRIEVE_CONNECTOR_BY_ID",
    RETRIEVE_DATA_BREACH_BY_ID: "BORNEO_RETRIEVE_DATA_BREACH_BY_ID",
    RETRIEVE_DATA_RESOURCE_STATISTICS: "BORNEO_RETRIEVE_DATA_RESOURCE_STATISTICS",
    RETRIEVE_DEPARTMENT_INFORMATION: "BORNEO_RETRIEVE_DEPARTMENT_INFORMATION",
    RETRIEVE_DISCOVERED_DOCUMENT_BY_ID: "BORNEO_RETRIEVE_DISCOVERED_DOCUMENT_BY_ID",
    RETRIEVE_DISCOVERED_INFOTYPE_BY_ID: "BORNEO_RETRIEVE_DISCOVERED_INFOTYPE_BY_ID",
    RETRIEVE_DISCOVERED_RECIPIENT_BY_ID: "BORNEO_RETRIEVE_DISCOVERED_RECIPIENT_BY_ID",
    RETRIEVE_DPIA_BY_ID: "BORNEO_RETRIEVE_DPIA_BY_ID",
    RETRIEVE_EMPLOYEE_DETAILS_BY_ID: "BORNEO_RETRIEVE_EMPLOYEE_DETAILS_BY_ID",
    RETRIEVE_ERROR_DETAILS_BY_ID: "BORNEO_RETRIEVE_ERROR_DETAILS_BY_ID",
    RETRIEVE_ISSUE_BY_ID: "BORNEO_RETRIEVE_ISSUE_BY_ID",
    RETRIEVE_LEGAL_DOCUMENT_BY_ID: "BORNEO_RETRIEVE_LEGAL_DOCUMENT_BY_ID",
    RETRIEVE_LOPDP_THRESHOLD_BY_ID: "BORNEO_RETRIEVE_LOPDP_THRESHOLD_BY_ID",
    RETRIEVE_PROCESSING_ACTIVITY_BY_ID: "BORNEO_RETRIEVE_PROCESSING_ACTIVITY_BY_ID",
    RETRIEVE_RECIPIENT_DETAILS: "BORNEO_RETRIEVE_RECIPIENT_DETAILS",
    RETRIEVE_RECIPIENT_PROCESSING_ACTIVITIES: "BORNEO_RETRIEVE_RECIPIENT_PROCESSING_ACTIVITIES",
    RETRIEVE_RESOURCE_CATALOG_BY_ID: "BORNEO_RETRIEVE_RESOURCE_CATALOG_BY_ID",
    RETRIEVE_RESOURCE_COLUMNS: "BORNEO_RETRIEVE_RESOURCE_COLUMNS",
    RETRIEVE_TOM_BY_ID: "BORNEO_RETRIEVE_TOM_BY_ID",
    SCAN_LEGAL_DOCUMENT_BY_ID: "BORNEO_SCAN_LEGAL_DOCUMENT_BY_ID",
    STOP_SCAN_VIA_SCAN_ID: "BORNEO_STOP_SCAN_VIA_SCAN_ID",
    SUBMIT_CHAT_FEEDBACK: "BORNEO_SUBMIT_CHAT_FEEDBACK",
    SUBMIT_DETAILED_SCAN_RESULTS: "BORNEO_SUBMIT_DETAILED_SCAN_RESULTS",
    TRIGGER_DASHBOARD_REPORT_BY_REPORT_ID: "BORNEO_TRIGGER_DASHBOARD_REPORT_BY_REPORT_ID",
    UPDATE_ASSET_INFORMATION_BY_ID: "BORNEO_UPDATE_ASSET_INFORMATION_BY_ID",
    UPDATE_CATEGORY_INFOTYPES: "BORNEO_UPDATE_CATEGORY_INFOTYPES",
    UPDATE_DASHBOARD_REPORT_FREQUENCY_AND_RECIPIENTS: "BORNEO_UPDATE_DASHBOARD_REPORT_FREQUENCY_AND_RECIPIENTS",
    UPDATE_DASHBOARD_USER_DETAILS: "BORNEO_UPDATE_DASHBOARD_USER_DETAILS",
    UPDATE_DASHBOARD_USER_ROLES: "BORNEO_UPDATE_DASHBOARD_USER_ROLES",
    UPDATE_DATA_BREACH_ENTRY: "BORNEO_UPDATE_DATA_BREACH_ENTRY",
    UPDATE_DEPARTMENT_NAME: "BORNEO_UPDATE_DEPARTMENT_NAME",
    UPDATE_DISCOVERED_DOCUMENT_STATUS: "BORNEO_UPDATE_DISCOVERED_DOCUMENT_STATUS",
    UPDATE_DISCOVERED_INFOTYPE_STATUS: "BORNEO_UPDATE_DISCOVERED_INFOTYPE_STATUS",
    UPDATE_DOMAIN_DETAILS: "BORNEO_UPDATE_DOMAIN_DETAILS",
    UPDATE_DPIA_BY_ID: "BORNEO_UPDATE_DPIA_BY_ID",
    UPDATE_EMPLOYEE_BY_ID: "BORNEO_UPDATE_EMPLOYEE_BY_ID",
    UPDATE_HEADQUARTER_DETAILS_BY_ID: "BORNEO_UPDATE_HEADQUARTER_DETAILS_BY_ID",
    UPDATE_LOPDP_THRESHOLD_BY_ID: "BORNEO_UPDATE_LOPDP_THRESHOLD_BY_ID",
    UPDATE_PROCESSING_ACTIVITY_DETAILS: "BORNEO_UPDATE_PROCESSING_ACTIVITY_DETAILS",
    UPDATE_RECIPIENT_DETAILS_BY_ID: "BORNEO_UPDATE_RECIPIENT_DETAILS_BY_ID",
    UPDATE_RECIPIENT_STATUS_VIA_ID: "BORNEO_UPDATE_RECIPIENT_STATUS_VIA_ID",
    UPDATE_THRESHOLD_BY_ID: "BORNEO_UPDATE_THRESHOLD_BY_ID",
    VERIFY_EMAIL_WITH_ID_AND_TOKEN: "BORNEO_VERIFY_EMAIL_WITH_ID_AND_TOKEN",
  },
  triggerTypes: {},
}
