// @composio/core/generated/box.ts
/**
 * Map of Composio's BOX toolkit.
 */
export const BOX = {
  slug: "box",
  tools: {
    ADD_CLASSIFICATION_TO_FILE: "BOX_ADD_CLASSIFICATION_TO_FILE",
    ADD_CLASSIFICATION_TO_FOLDER: "BOX_ADD_CLASSIFICATION_TO_FOLDER",
    ADD_DOMAIN_TO_LIST_OF_ALLOWED_COLLABORATION_DOMAINS: "BOX_ADD_DOMAIN_TO_LIST_OF_ALLOWED_COLLABORATION_DOMAINS",
    ADD_INITIAL_CLASSIFICATIONS: "BOX_ADD_INITIAL_CLASSIFICATIONS",
    ADD_OR_UPDATE_USER_AVATAR: "BOX_ADD_OR_UPDATE_USER_AVATAR",
    ADD_SHARED_LINK_TO_FILE: "BOX_ADD_SHARED_LINK_TO_FILE",
    ADD_SHARED_LINK_TO_FOLDER: "BOX_ADD_SHARED_LINK_TO_FOLDER",
    ADD_SHARED_LINK_TO_WEB_LINK: "BOX_ADD_SHARED_LINK_TO_WEB_LINK",
    ADD_USER_TO_GROUP: "BOX_ADD_USER_TO_GROUP",
    APPLY_WATERMARK_TO_FILE: "BOX_APPLY_WATERMARK_TO_FILE",
    APPLY_WATERMARK_TO_FOLDER: "BOX_APPLY_WATERMARK_TO_FOLDER",
    ASK_QUESTION: "BOX_ASK_QUESTION",
    ASSIGN_LEGAL_HOLD_POLICY: "BOX_ASSIGN_LEGAL_HOLD_POLICY",
    ASSIGN_RETENTION_POLICY: "BOX_ASSIGN_RETENTION_POLICY",
    ASSIGN_STORAGE_POLICY: "BOX_ASSIGN_STORAGE_POLICY",
    ASSIGN_TASK: "BOX_ASSIGN_TASK",
    AUTHORIZE_USER: "BOX_AUTHORIZE_USER",
    CANCEL_BOX_SIGN_REQUEST: "BOX_CANCEL_BOX_SIGN_REQUEST",
    CHANGE_SHIELD_INFORMATION_BARRIER_STATUS: "BOX_CHANGE_SHIELD_INFORMATION_BARRIER_STATUS",
    COMMIT_UPLOAD_SESSION: "BOX_COMMIT_UPLOAD_SESSION",
    COPY_FILE: "BOX_COPY_FILE",
    COPY_FILE_REQUEST: "BOX_COPY_FILE_REQUEST",
    COPY_FOLDER: "BOX_COPY_FOLDER",
    CREATE_AI_AGENT: "BOX_CREATE_AI_AGENT",
    CREATE_BOX_SIGN_REQUEST: "BOX_CREATE_BOX_SIGN_REQUEST",
    CREATE_BOX_SKILL_CARDS_ON_FILE: "BOX_CREATE_BOX_SKILL_CARDS_ON_FILE",
    CREATE_COLLABORATION: "BOX_CREATE_COLLABORATION",
    CREATE_COLLAB_EXEMPTION: "BOX_CREATE_COLLAB_EXEMPTION",
    CREATE_COMMENT: "BOX_CREATE_COMMENT",
    CREATE_EMAIL_ALIAS: "BOX_CREATE_EMAIL_ALIAS",
    CREATE_FOLDER: "BOX_CREATE_FOLDER",
    CREATE_FOLDER_LOCK: "BOX_CREATE_FOLDER_LOCK",
    CREATE_GROUP: "BOX_CREATE_GROUP",
    CREATE_JOBS_TO_TERMINATE_USERS_SESSION: "BOX_CREATE_JOBS_TO_TERMINATE_USERS_SESSION",
    CREATE_JOBS_TO_TERMINATE_USER_GROUP_SESSION: "BOX_CREATE_JOBS_TO_TERMINATE_USER_GROUP_SESSION",
    CREATE_LEGAL_HOLD_POLICY: "BOX_CREATE_LEGAL_HOLD_POLICY",
    CREATE_METADATA_CASCADE_POLICY: "BOX_CREATE_METADATA_CASCADE_POLICY",
    CREATE_METADATA_INSTANCE_ON_FILE: "BOX_CREATE_METADATA_INSTANCE_ON_FILE",
    CREATE_METADATA_INSTANCE_ON_FOLDER: "BOX_CREATE_METADATA_INSTANCE_ON_FOLDER",
    CREATE_METADATA_TEMPLATE: "BOX_CREATE_METADATA_TEMPLATE",
    CREATE_RETENTION_POLICY: "BOX_CREATE_RETENTION_POLICY",
    CREATE_SHIELD_INFORMATION_BARRIER: "BOX_CREATE_SHIELD_INFORMATION_BARRIER",
    CREATE_SHIELD_INFORMATION_BARRIER_REPORT: "BOX_CREATE_SHIELD_INFORMATION_BARRIER_REPORT",
    CREATE_SHIELD_INFORMATION_BARRIER_SEGMENT: "BOX_CREATE_SHIELD_INFORMATION_BARRIER_SEGMENT",
    CREATE_SHIELD_INFORMATION_BARRIER_SEGMENT_MEMBER: "BOX_CREATE_SHIELD_INFORMATION_BARRIER_SEGMENT_MEMBER",
    CREATE_SHIELD_INFORMATION_BARRIER_SEGMENT_RESTRICTION: "BOX_CREATE_SHIELD_INFORMATION_BARRIER_SEGMENT_RESTRICTION",
    CREATE_SLACK_INTEGRATION_MAPPING: "BOX_CREATE_SLACK_INTEGRATION_MAPPING",
    CREATE_TASK: "BOX_CREATE_TASK",
    CREATE_TEAMS_INTEGRATION_MAPPING: "BOX_CREATE_TEAMS_INTEGRATION_MAPPING",
    CREATE_TERMS_OF_SERVICE: "BOX_CREATE_TERMS_OF_SERVICE",
    CREATE_TERMS_OF_SERVICE_STATUS_FOR_NEW_USER: "BOX_CREATE_TERMS_OF_SERVICE_STATUS_FOR_NEW_USER",
    CREATE_UPLOAD_SESSION: "BOX_CREATE_UPLOAD_SESSION",
    CREATE_UPLOAD_SESSION_FOR_EXISTING_FILE: "BOX_CREATE_UPLOAD_SESSION_FOR_EXISTING_FILE",
    CREATE_USER: "BOX_CREATE_USER",
    CREATE_USER_INVITE: "BOX_CREATE_USER_INVITE",
    CREATE_WEBHOOK: "BOX_CREATE_WEBHOOK",
    CREATE_WEB_LINK: "BOX_CREATE_WEB_LINK",
    CREATE_ZIP_DOWNLOAD: "BOX_CREATE_ZIP_DOWNLOAD",
    DELETE_AI_AGENT: "BOX_DELETE_AI_AGENT",
    DELETE_BARRIER_RESTRICTION: "BOX_DELETE_BARRIER_RESTRICTION",
    DELETE_FILE: "BOX_DELETE_FILE",
    DELETE_FILE_REQUEST: "BOX_DELETE_FILE_REQUEST",
    DELETE_FOLDER: "BOX_DELETE_FOLDER",
    DELETE_FOLDER_LOCK: "BOX_DELETE_FOLDER_LOCK",
    DELETE_RETENTION_POLICY: "BOX_DELETE_RETENTION_POLICY",
    DELETE_SHIELD_INFORMATION_BARRIER_SEGMENT: "BOX_DELETE_SHIELD_INFORMATION_BARRIER_SEGMENT",
    DELETE_SHIELD_INFORMATION_BARRIER_SEGMENT_MEMBER_BY_ID: "BOX_DELETE_SHIELD_INFORMATION_BARRIER_SEGMENT_MEMBER_BY_ID",
    DELETE_SLACK_INTEGRATION_MAPPING: "BOX_DELETE_SLACK_INTEGRATION_MAPPING",
    DELETE_TEAMS_INTEGRATION_MAPPING: "BOX_DELETE_TEAMS_INTEGRATION_MAPPING",
    DELETE_USER: "BOX_DELETE_USER",
    DELETE_USER_AVATAR: "BOX_DELETE_USER_AVATAR",
    DOWNLOAD_FILE: "BOX_DOWNLOAD_FILE",
    DOWNLOAD_ZIP_ARCHIVE: "BOX_DOWNLOAD_ZIP_ARCHIVE",
    EXTRACT_METADATA_FREEFORM: "BOX_EXTRACT_METADATA_FREEFORM",
    EXTRACT_METADATA_STRUCTURED: "BOX_EXTRACT_METADATA_STRUCTURED",
    FIND_APP_ITEM_FOR_SHARED_LINK: "BOX_FIND_APP_ITEM_FOR_SHARED_LINK",
    FIND_FILE_FOR_SHARED_LINK: "BOX_FIND_FILE_FOR_SHARED_LINK",
    FIND_FOLDER_FOR_SHARED_LINK: "BOX_FIND_FOLDER_FOR_SHARED_LINK",
    FIND_METADATA_TEMPLATE_BY_INSTANCE_ID: "BOX_FIND_METADATA_TEMPLATE_BY_INSTANCE_ID",
    FIND_WEB_LINK_FOR_SHARED_LINK: "BOX_FIND_WEB_LINK_FOR_SHARED_LINK",
    FORCE_APPLY_METADATA_CASCADE_POLICY_TO_FOLDER: "BOX_FORCE_APPLY_METADATA_CASCADE_POLICY_TO_FOLDER",
    GENERATE_TEXT: "BOX_GENERATE_TEXT",
    GET_AI_AGENT_BY_AGENT_ID: "BOX_GET_AI_AGENT_BY_AGENT_ID",
    GET_AI_AGENT_DEFAULT_CONFIGURATION: "BOX_GET_AI_AGENT_DEFAULT_CONFIGURATION",
    GET_ALLOWED_COLLABORATION_DOMAIN: "BOX_GET_ALLOWED_COLLABORATION_DOMAIN",
    GET_BOX_SIGN_REQUEST_BY_ID: "BOX_GET_BOX_SIGN_REQUEST_BY_ID",
    GET_BOX_SIGN_TEMPLATE_BY_ID: "BOX_GET_BOX_SIGN_TEMPLATE_BY_ID",
    GET_CLASSIFICATION_ON_FILE: "BOX_GET_CLASSIFICATION_ON_FILE",
    GET_CLASSIFICATION_ON_FOLDER: "BOX_GET_CLASSIFICATION_ON_FOLDER",
    GET_COLLABORATION: "BOX_GET_COLLABORATION",
    GET_COLLECTION_BY_ID: "BOX_GET_COLLECTION_BY_ID",
    GET_COMMENT: "BOX_GET_COMMENT",
    GET_CURRENT_USER: "BOX_GET_CURRENT_USER",
    GET_DEVICE_PIN: "BOX_GET_DEVICE_PIN",
    GET_EVENTS_LONG_POLL_ENDPOINT: "BOX_GET_EVENTS_LONG_POLL_ENDPOINT",
    GET_FILES_UNDER_RETENTION: "BOX_GET_FILES_UNDER_RETENTION",
    GET_FILE_INFORMATION: "BOX_GET_FILE_INFORMATION",
    GET_FILE_REQUEST: "BOX_GET_FILE_REQUEST",
    GET_FILE_THUMBNAIL: "BOX_GET_FILE_THUMBNAIL",
    GET_FILE_VERSION: "BOX_GET_FILE_VERSION",
    GET_FILE_VERSIONS_UNDER_RETENTION: "BOX_GET_FILE_VERSIONS_UNDER_RETENTION",
    GET_FILE_VERSION_LEGAL_HOLD: "BOX_GET_FILE_VERSION_LEGAL_HOLD",
    GET_FOLDER_INFORMATION: "BOX_GET_FOLDER_INFORMATION",
    GET_GROUP: "BOX_GET_GROUP",
    GET_GROUP_MEMBERSHIP: "BOX_GET_GROUP_MEMBERSHIP",
    GET_LEGAL_HOLD_POLICY: "BOX_GET_LEGAL_HOLD_POLICY",
    GET_LEGAL_HOLD_POLICY_ASSIGNMENT: "BOX_GET_LEGAL_HOLD_POLICY_ASSIGNMENT",
    GET_METADATA_CASCADE_POLICY: "BOX_GET_METADATA_CASCADE_POLICY",
    GET_METADATA_INSTANCE_ON_FILE: "BOX_GET_METADATA_INSTANCE_ON_FILE",
    GET_METADATA_INSTANCE_ON_FOLDER: "BOX_GET_METADATA_INSTANCE_ON_FOLDER",
    GET_METADATA_TEMPLATE_BY_ID: "BOX_GET_METADATA_TEMPLATE_BY_ID",
    GET_METADATA_TEMPLATE_BY_NAME: "BOX_GET_METADATA_TEMPLATE_BY_NAME",
    GET_RETENTION_ON_FILE: "BOX_GET_RETENTION_ON_FILE",
    GET_RETENTION_POLICY: "BOX_GET_RETENTION_POLICY",
    GET_RETENTION_POLICY_ASSIGNMENT: "BOX_GET_RETENTION_POLICY_ASSIGNMENT",
    GET_SHARED_LINK_FOR_FILE: "BOX_GET_SHARED_LINK_FOR_FILE",
    GET_SHARED_LINK_FOR_FOLDER: "BOX_GET_SHARED_LINK_FOR_FOLDER",
    GET_SHARED_LINK_FOR_WEB_LINK: "BOX_GET_SHARED_LINK_FOR_WEB_LINK",
    GET_SHIELD_INFORMATION_BARRIER_REPORT_BY_ID: "BOX_GET_SHIELD_INFORMATION_BARRIER_REPORT_BY_ID",
    GET_SHIELD_INFORMATION_BARRIER_SEGMENT_MEMBER_BY_ID: "BOX_GET_SHIELD_INFORMATION_BARRIER_SEGMENT_MEMBER_BY_ID",
    GET_SHIELD_INFORMATION_BARRIER_SEGMENT_RESTRICTION_BY_ID: "BOX_GET_SHIELD_INFORMATION_BARRIER_SEGMENT_RESTRICTION_BY_ID",
    GET_SHIELD_INFORMATION_BARRIER_SEGMENT_WITH_SPECIFIED_ID: "BOX_GET_SHIELD_INFORMATION_BARRIER_SEGMENT_WITH_SPECIFIED_ID",
    GET_SHIELD_INFORMATION_BARRIER_WITH_SPECIFIED_ID: "BOX_GET_SHIELD_INFORMATION_BARRIER_WITH_SPECIFIED_ID",
    GET_STORAGE_POLICY: "BOX_GET_STORAGE_POLICY",
    GET_STORAGE_POLICY_ASSIGNMENT: "BOX_GET_STORAGE_POLICY_ASSIGNMENT",
    GET_TASK: "BOX_GET_TASK",
    GET_TASK_ASSIGNMENT: "BOX_GET_TASK_ASSIGNMENT",
    GET_TERMS_OF_SERVICE: "BOX_GET_TERMS_OF_SERVICE",
    GET_TRASHED_FILE: "BOX_GET_TRASHED_FILE",
    GET_TRASHED_FOLDER: "BOX_GET_TRASHED_FOLDER",
    GET_TRASHED_WEB_LINK: "BOX_GET_TRASHED_WEB_LINK",
    GET_UPLOAD_SESSION: "BOX_GET_UPLOAD_SESSION",
    GET_USER: "BOX_GET_USER",
    GET_USER_AVATAR: "BOX_GET_USER_AVATAR",
    GET_USER_EXEMPT_FROM_COLLABORATION_DOMAIN_RESTRICTIONS: "BOX_GET_USER_EXEMPT_FROM_COLLABORATION_DOMAIN_RESTRICTIONS",
    GET_USER_INVITE_STATUS: "BOX_GET_USER_INVITE_STATUS",
    GET_WATERMARK_FOR_FOLDER: "BOX_GET_WATERMARK_FOR_FOLDER",
    GET_WATERMARK_ON_FILE: "BOX_GET_WATERMARK_ON_FILE",
    GET_WEBHOOK: "BOX_GET_WEBHOOK",
    GET_WEB_LINK: "BOX_GET_WEB_LINK",
    GET_ZIP_DOWNLOAD_STATUS: "BOX_GET_ZIP_DOWNLOAD_STATUS",
    LIST_AI_AGENTS: "BOX_LIST_AI_AGENTS",
    LIST_ALLOWED_COLLABORATION_DOMAINS: "BOX_LIST_ALLOWED_COLLABORATION_DOMAINS",
    LIST_ALL_CLASSIFICATIONS: "BOX_LIST_ALL_CLASSIFICATIONS",
    LIST_ALL_COLLECTIONS: "BOX_LIST_ALL_COLLECTIONS",
    LIST_ALL_FILE_VERSIONS: "BOX_LIST_ALL_FILE_VERSIONS",
    LIST_ALL_GLOBAL_METADATA_TEMPLATES: "BOX_LIST_ALL_GLOBAL_METADATA_TEMPLATES",
    LIST_ALL_LEGAL_HOLD_POLICIES: "BOX_LIST_ALL_LEGAL_HOLD_POLICIES",
    LIST_ALL_METADATA_TEMPLATES_FOR_ENTERPRISE: "BOX_LIST_ALL_METADATA_TEMPLATES_FOR_ENTERPRISE",
    LIST_ALL_WEBHOOKS: "BOX_LIST_ALL_WEBHOOKS",
    LIST_BOX_SIGN_REQUESTS: "BOX_LIST_BOX_SIGN_REQUESTS",
    LIST_BOX_SIGN_TEMPLATES: "BOX_LIST_BOX_SIGN_TEMPLATES",
    LIST_BOX_SKILL_CARDS_ON_FILE: "BOX_LIST_BOX_SKILL_CARDS_ON_FILE",
    LIST_COLLECTION_ITEMS: "BOX_LIST_COLLECTION_ITEMS",
    LIST_ENTERPRISE_DEVICE_PINS: "BOX_LIST_ENTERPRISE_DEVICE_PINS",
    LIST_ENTERPRISE_USERS: "BOX_LIST_ENTERPRISE_USERS",
    LIST_FILE_APP_ITEM_ASSOCIATIONS: "BOX_LIST_FILE_APP_ITEM_ASSOCIATIONS",
    LIST_FILE_COLLABORATIONS: "BOX_LIST_FILE_COLLABORATIONS",
    LIST_FILE_COMMENTS: "BOX_LIST_FILE_COMMENTS",
    LIST_FILE_VERSION_LEGAL_HOLDS: "BOX_LIST_FILE_VERSION_LEGAL_HOLDS",
    LIST_FILE_VERSION_RETENTIONS: "BOX_LIST_FILE_VERSION_RETENTIONS",
    LIST_FOLDER_APP_ITEM_ASSOCIATIONS: "BOX_LIST_FOLDER_APP_ITEM_ASSOCIATIONS",
    LIST_FOLDER_COLLABORATIONS: "BOX_LIST_FOLDER_COLLABORATIONS",
    LIST_FOLDER_LOCKS: "BOX_LIST_FOLDER_LOCKS",
    LIST_GROUPS_FOR_ENTERPRISE: "BOX_LIST_GROUPS_FOR_ENTERPRISE",
    LIST_GROUP_COLLABORATIONS: "BOX_LIST_GROUP_COLLABORATIONS",
    LIST_HOLD_FILE_VERSIONS: "BOX_LIST_HOLD_FILE_VERSIONS",
    LIST_ITEMS_IN_FOLDER: "BOX_LIST_ITEMS_IN_FOLDER",
    LIST_LEGAL_HOLD_POLICY_ASSIGNMENTS: "BOX_LIST_LEGAL_HOLD_POLICY_ASSIGNMENTS",
    LIST_MEMBERS_OF_GROUP: "BOX_LIST_MEMBERS_OF_GROUP",
    LIST_METADATA_CASCADE_POLICIES: "BOX_LIST_METADATA_CASCADE_POLICIES",
    LIST_METADATA_INSTANCES_ON_FILE: "BOX_LIST_METADATA_INSTANCES_ON_FILE",
    LIST_METADATA_INSTANCES_ON_FOLDER: "BOX_LIST_METADATA_INSTANCES_ON_FOLDER",
    LIST_PARTS: "BOX_LIST_PARTS",
    LIST_PENDING_COLLABORATIONS: "BOX_LIST_PENDING_COLLABORATIONS",
    LIST_RECENTLY_ACCESSED_ITEMS: "BOX_LIST_RECENTLY_ACCESSED_ITEMS",
    LIST_RETENTION_POLICIES: "BOX_LIST_RETENTION_POLICIES",
    LIST_RETENTION_POLICY_ASSIGNMENTS: "BOX_LIST_RETENTION_POLICY_ASSIGNMENTS",
    LIST_SHIELD_INFORMATION_BARRIERS: "BOX_LIST_SHIELD_INFORMATION_BARRIERS",
    LIST_SHIELD_INFORMATION_BARRIER_REPORTS: "BOX_LIST_SHIELD_INFORMATION_BARRIER_REPORTS",
    LIST_SHIELD_INFORMATION_BARRIER_SEGMENTS: "BOX_LIST_SHIELD_INFORMATION_BARRIER_SEGMENTS",
    LIST_SHIELD_INFORMATION_BARRIER_SEGMENT_MEMBERS: "BOX_LIST_SHIELD_INFORMATION_BARRIER_SEGMENT_MEMBERS",
    LIST_SHIELD_INFORMATION_BARRIER_SEGMENT_RESTRICTIONS: "BOX_LIST_SHIELD_INFORMATION_BARRIER_SEGMENT_RESTRICTIONS",
    LIST_SLACK_INTEGRATION_MAPPINGS: "BOX_LIST_SLACK_INTEGRATION_MAPPINGS",
    LIST_STORAGE_POLICIES: "BOX_LIST_STORAGE_POLICIES",
    LIST_STORAGE_POLICY_ASSIGNMENTS: "BOX_LIST_STORAGE_POLICY_ASSIGNMENTS",
    LIST_TASKS_ON_FILE: "BOX_LIST_TASKS_ON_FILE",
    LIST_TASK_ASSIGNMENTS: "BOX_LIST_TASK_ASSIGNMENTS",
    LIST_TEAMS_INTEGRATION_MAPPINGS: "BOX_LIST_TEAMS_INTEGRATION_MAPPINGS",
    LIST_TERMS_OF_SERVICES: "BOX_LIST_TERMS_OF_SERVICES",
    LIST_TERMS_OF_SERVICE_USER_STATUSES: "BOX_LIST_TERMS_OF_SERVICE_USER_STATUSES",
    LIST_TRASHED_ITEMS: "BOX_LIST_TRASHED_ITEMS",
    LIST_USERS_EXEMPT_FROM_COLLABORATION_DOMAIN_RESTRICTIONS: "BOX_LIST_USERS_EXEMPT_FROM_COLLABORATION_DOMAIN_RESTRICTIONS",
    LIST_USER_AND_ENTERPRISE_EVENTS: "BOX_LIST_USER_AND_ENTERPRISE_EVENTS",
    LIST_USER_S_EMAIL_ALIASES: "BOX_LIST_USER_S_EMAIL_ALIASES",
    LIST_USER_S_GROUPS: "BOX_LIST_USER_S_GROUPS",
    LIST_WORKFLOWS: "BOX_LIST_WORKFLOWS",
    PERMANENTLY_REMOVE_FILE: "BOX_PERMANENTLY_REMOVE_FILE",
    PERMANENTLY_REMOVE_FOLDER: "BOX_PERMANENTLY_REMOVE_FOLDER",
    PERMANENTLY_REMOVE_WEB_LINK: "BOX_PERMANENTLY_REMOVE_WEB_LINK",
    PREFLIGHT_CHECK_BEFORE_UPLOAD: "BOX_PREFLIGHT_CHECK_BEFORE_UPLOAD",
    PROMOTE_FILE_VERSION: "BOX_PROMOTE_FILE_VERSION",
    QUERY_FILES_FOLDERS_BY_METADATA: "BOX_QUERY_FILES_FOLDERS_BY_METADATA",
    REFRESH_ACCESS_TOKEN: "BOX_REFRESH_ACCESS_TOKEN",
    REMOVE_BOX_SKILL_CARDS_FROM_FILE: "BOX_REMOVE_BOX_SKILL_CARDS_FROM_FILE",
    REMOVE_CLASSIFICATION_FROM_FILE: "BOX_REMOVE_CLASSIFICATION_FROM_FILE",
    REMOVE_CLASSIFICATION_FROM_FOLDER: "BOX_REMOVE_CLASSIFICATION_FROM_FOLDER",
    REMOVE_COLLABORATION: "BOX_REMOVE_COLLABORATION",
    REMOVE_COMMENT: "BOX_REMOVE_COMMENT",
    REMOVE_DEVICE_PIN: "BOX_REMOVE_DEVICE_PIN",
    REMOVE_DOMAIN_FROM_LIST_OF_ALLOWED_COLLABORATION_DOMAINS: "BOX_REMOVE_DOMAIN_FROM_LIST_OF_ALLOWED_COLLABORATION_DOMAINS",
    REMOVE_EMAIL_ALIAS: "BOX_REMOVE_EMAIL_ALIAS",
    REMOVE_FILE_VERSION: "BOX_REMOVE_FILE_VERSION",
    REMOVE_GROUP: "BOX_REMOVE_GROUP",
    REMOVE_LEGAL_HOLD_POLICY: "BOX_REMOVE_LEGAL_HOLD_POLICY",
    REMOVE_METADATA_CASCADE_POLICY: "BOX_REMOVE_METADATA_CASCADE_POLICY",
    REMOVE_METADATA_INSTANCE_FROM_FILE: "BOX_REMOVE_METADATA_INSTANCE_FROM_FILE",
    REMOVE_METADATA_INSTANCE_FROM_FOLDER: "BOX_REMOVE_METADATA_INSTANCE_FROM_FOLDER",
    REMOVE_METADATA_TEMPLATE: "BOX_REMOVE_METADATA_TEMPLATE",
    REMOVE_RETENTION_POLICY_ASSIGNMENT: "BOX_REMOVE_RETENTION_POLICY_ASSIGNMENT",
    REMOVE_SHARED_LINK_FROM_FILE: "BOX_REMOVE_SHARED_LINK_FROM_FILE",
    REMOVE_SHARED_LINK_FROM_FOLDER: "BOX_REMOVE_SHARED_LINK_FROM_FOLDER",
    REMOVE_SHARED_LINK_FROM_WEB_LINK: "BOX_REMOVE_SHARED_LINK_FROM_WEB_LINK",
    REMOVE_TASK: "BOX_REMOVE_TASK",
    REMOVE_UPLOAD_SESSION: "BOX_REMOVE_UPLOAD_SESSION",
    REMOVE_USER_DOMAIN_EXEMPTION: "BOX_REMOVE_USER_DOMAIN_EXEMPTION",
    REMOVE_USER_FROM_GROUP: "BOX_REMOVE_USER_FROM_GROUP",
    REMOVE_WATERMARK_FROM_FILE: "BOX_REMOVE_WATERMARK_FROM_FILE",
    REMOVE_WATERMARK_FROM_FOLDER: "BOX_REMOVE_WATERMARK_FROM_FOLDER",
    REMOVE_WEBHOOK: "BOX_REMOVE_WEBHOOK",
    REMOVE_WEB_LINK: "BOX_REMOVE_WEB_LINK",
    REQUEST_ACCESS_TOKEN: "BOX_REQUEST_ACCESS_TOKEN",
    RESEND_BOX_SIGN_REQUEST: "BOX_RESEND_BOX_SIGN_REQUEST",
    RESTORE_FILE: "BOX_RESTORE_FILE",
    RESTORE_FILE_VERSION: "BOX_RESTORE_FILE_VERSION",
    RESTORE_FOLDER: "BOX_RESTORE_FOLDER",
    RESTORE_WEB_LINK: "BOX_RESTORE_WEB_LINK",
    REVIEW_FILES_ON_LEGAL_HOLD_POLICY_ASSIGNMENT: "BOX_REVIEW_FILES_ON_LEGAL_HOLD_POLICY_ASSIGNMENT",
    REVOKE_ACCESS_TOKEN: "BOX_REVOKE_ACCESS_TOKEN",
    SEARCH_FOR_CONTENT: "BOX_SEARCH_FOR_CONTENT",
    STARTS_WORKFLOW_BASED_ON_REQUEST_BODY: "BOX_STARTS_WORKFLOW_BASED_ON_REQUEST_BODY",
    TRANSFER_OWNED_FOLDERS: "BOX_TRANSFER_OWNED_FOLDERS",
    UNASSIGN_LEGAL_HOLD_POLICY: "BOX_UNASSIGN_LEGAL_HOLD_POLICY",
    UNASSIGN_STORAGE_POLICY: "BOX_UNASSIGN_STORAGE_POLICY",
    UNASSIGN_TASK: "BOX_UNASSIGN_TASK",
    UPDATE_AI_AGENT: "BOX_UPDATE_AI_AGENT",
    UPDATE_ALL_BOX_SKILL_CARDS_ON_FILE: "BOX_UPDATE_ALL_BOX_SKILL_CARDS_ON_FILE",
    UPDATE_BARRIER_SEGMENT: "BOX_UPDATE_BARRIER_SEGMENT",
    UPDATE_COLLABORATION: "BOX_UPDATE_COLLABORATION",
    UPDATE_COMMENT: "BOX_UPDATE_COMMENT",
    UPDATE_FILE: "BOX_UPDATE_FILE",
    UPDATE_FILE_REQUEST: "BOX_UPDATE_FILE_REQUEST",
    UPDATE_FOLDER: "BOX_UPDATE_FOLDER",
    UPDATE_GROUP: "BOX_UPDATE_GROUP",
    UPDATE_GROUP_MEMBERSHIP: "BOX_UPDATE_GROUP_MEMBERSHIP",
    UPDATE_LEGAL_HOLD_POLICY: "BOX_UPDATE_LEGAL_HOLD_POLICY",
    UPDATE_RETENTION_POLICY: "BOX_UPDATE_RETENTION_POLICY",
    UPDATE_SHARED_LINK_ON_FILE: "BOX_UPDATE_SHARED_LINK_ON_FILE",
    UPDATE_SHARED_LINK_ON_FOLDER: "BOX_UPDATE_SHARED_LINK_ON_FOLDER",
    UPDATE_SHARED_LINK_ON_WEB_LINK: "BOX_UPDATE_SHARED_LINK_ON_WEB_LINK",
    UPDATE_SLACK_INTEGRATION_MAPPING: "BOX_UPDATE_SLACK_INTEGRATION_MAPPING",
    UPDATE_STORAGE_POLICY_ASSIGNMENT: "BOX_UPDATE_STORAGE_POLICY_ASSIGNMENT",
    UPDATE_TASK: "BOX_UPDATE_TASK",
    UPDATE_TASK_ASSIGNMENT: "BOX_UPDATE_TASK_ASSIGNMENT",
    UPDATE_TEAMS_INTEGRATION_MAPPING: "BOX_UPDATE_TEAMS_INTEGRATION_MAPPING",
    UPDATE_TERMS_OF_SERVICE: "BOX_UPDATE_TERMS_OF_SERVICE",
    UPDATE_TERMS_OF_SERVICE_STATUS_FOR_EXISTING_USER: "BOX_UPDATE_TERMS_OF_SERVICE_STATUS_FOR_EXISTING_USER",
    UPDATE_USER: "BOX_UPDATE_USER",
    UPDATE_WEBHOOK: "BOX_UPDATE_WEBHOOK",
    UPDATE_WEB_LINK: "BOX_UPDATE_WEB_LINK",
    UPLOAD_FILE: "BOX_UPLOAD_FILE",
    UPLOAD_FILE_VERSION: "BOX_UPLOAD_FILE_VERSION",
    UPLOAD_PART_OF_FILE: "BOX_UPLOAD_PART_OF_FILE",
  },
  triggerTypes: {},
}
