// @composio/core/generated/brevo.ts
/**
 * Map of Composio's BREVO toolkit.
 */
export const BREVO = {
  slug: "brevo",
  tools: {
    CREATE_A_COMPANY: "BREVO_CREATE_A_COMPANY",
    CREATE_OR_UPDATE_EMAIL_TEMPLATE: "BREVO_CREATE_OR_UPDATE_EMAIL_TEMPLATE",
    CREATE_SMS_CAMPAIGN: "BREVO_CREATE_SMS_CAMPAIGN",
    DELETE_COMPANY: "BREVO_DELETE_COMPANY",
    DELETE_CONTACT: "BREVO_DELETE_CONTACT",
    DELETE_EMAIL_TEMPLATE: "BREVO_DELETE_EMAIL_TEMPLATE",
    DELETE_SMS_CAMPAIGN: "BREVO_DELETE_SMS_CAMPAIGN",
    FIND_CONTACT: "BREVO_FIND_CONTACT",
    GET_ACCOUNT_INFO: "BREVO_GET_ACCOUNT_INFO",
    GET_ALL_CONTACTS: "BREVO_GET_ALL_CONTACTS",
    GET_ALL_EMAIL_TEMPLATES: "BREVO_GET_ALL_EMAIL_TEMPLATES",
    GET_ALL_SENDERS: "BREVO_GET_ALL_SENDERS",
    GET_COMPANY_DETAILS: "BREVO_GET_COMPANY_DETAILS",
    GET_CONTACT_DETAILS: "BREVO_GET_CONTACT_DETAILS",
    GET_SMS_CAMPAIGNS: "BREVO_GET_SMS_CAMPAIGNS",
    GET_SMS_CAMPAIGN_DETAILS: "BREVO_GET_SMS_CAMPAIGN_DETAILS",
    LIST_ALL_COMPANIES: "BREVO_LIST_ALL_COMPANIES",
    LIST_EMAIL_CAMPAIGNS: "BREVO_LIST_EMAIL_CAMPAIGNS",
    UPDATE_EMAIL_CAMPAIGN: "BREVO_UPDATE_EMAIL_CAMPAIGN",
  },
  triggerTypes: {},
}
