// @composio/core/generated/cal.ts
/**
 * Map of Composio's CAL toolkit.
 */
export const CAL = {
  slug: "cal",
  tools: {
    ADD_MEMBER_TO_TEAM_USING_ORG_AND_TEAM_ID: "CAL_ADD_MEMBER_TO_TEAM_USING_ORG_AND_TEAM_ID",
    ADD_ORGANIZATION_ATTRIBUTE_OPTION: "CAL_ADD_ORGANIZATION_ATTRIBUTE_OPTION",
    CANCEL_BOOKING_VIA_UID: "CAL_CANCEL_BOOKING_VIA_UID",
    CHECK_CALENDAR_VERSION_2: "CAL_CHECK_CALENDAR_VERSION_2",
    CHECK_GCAL_SYNCHRONIZATION_STATUS: "CAL_CHECK_GCAL_SYNCHRONIZATION_STATUS",
    CHECK_ICS_FEED_CALENDAR_ENDPOINT: "CAL_CHECK_ICS_FEED_CALENDAR_ENDPOINT",
    CHECK_STRIPE_STATUS: "CAL_CHECK_STRIPE_STATUS",
    CHECK_TEAM_STRIPE_INTEGRATION_STATUS: "CAL_CHECK_TEAM_STRIPE_INTEGRATION_STATUS",
    CONFIRM_BOOKING_BY_UID: "CAL_CONFIRM_BOOKING_BY_UID",
    CONNECT_TO_CALENDAR: "CAL_CONNECT_TO_CALENDAR",
    CREATE_MEMBERSHIP_FOR_ORGANIZATION: "CAL_CREATE_MEMBERSHIP_FOR_ORGANIZATION",
    CREATE_OAUTH_CLIENT_USER: "CAL_CREATE_OAUTH_CLIENT_USER",
    CREATE_OAUTH_CLIENT_WEBHOOK_CONFIGURATION: "CAL_CREATE_OAUTH_CLIENT_WEBHOOK_CONFIGURATION",
    CREATE_ORGANIZATION_ATTRIBUTES: "CAL_CREATE_ORGANIZATION_ATTRIBUTES",
    CREATE_ORGANIZATION_WEBHOOK_BY_ORG_ID: "CAL_CREATE_ORGANIZATION_WEBHOOK_BY_ORG_ID",
    CREATE_OR_UPDATE_TEAM_PROFILE: "CAL_CREATE_OR_UPDATE_TEAM_PROFILE",
    CREATE_PHONE_CALL_EVENT: "CAL_CREATE_PHONE_CALL_EVENT",
    CREATE_PHONE_CALL_FOR_EVENT_TYPE: "CAL_CREATE_PHONE_CALL_FOR_EVENT_TYPE",
    CREATE_TEAM_EVENT_TYPE: "CAL_CREATE_TEAM_EVENT_TYPE",
    CREATE_TEAM_EVENT_TYPES_WITH_CUSTOM_OPTIONS: "CAL_CREATE_TEAM_EVENT_TYPES_WITH_CUSTOM_OPTIONS",
    CREATE_TEAM_IN_ORGANIZATION: "CAL_CREATE_TEAM_IN_ORGANIZATION",
    CREATE_TEAM_MEMBERSHIP_WITH_ROLE: "CAL_CREATE_TEAM_MEMBERSHIP_WITH_ROLE",
    CREATE_USER_AVAILABILITY_SCHEDULE: "CAL_CREATE_USER_AVAILABILITY_SCHEDULE",
    CREATE_USER_SCHEDULE_IN_ORGANIZATION: "CAL_CREATE_USER_SCHEDULE_IN_ORGANIZATION",
    CREATE_WEBHOOK_FOR_EVENT_TYPE: "CAL_CREATE_WEBHOOK_FOR_EVENT_TYPE",
    DECLINE_BOOKING_WITH_REASON: "CAL_DECLINE_BOOKING_WITH_REASON",
    DELETE_CONFERENCE_APP_CONNECTION: "CAL_DELETE_CONFERENCE_APP_CONNECTION",
    DELETE_EVENT_TYPE_BY_ID: "CAL_DELETE_EVENT_TYPE_BY_ID",
    DELETE_EVENT_TYPE_IN_TEAM: "CAL_DELETE_EVENT_TYPE_IN_TEAM",
    DELETE_MEMBERSHIP_IN_TEAM: "CAL_DELETE_MEMBERSHIP_IN_TEAM",
    DELETE_OAUTH_CLIENT_USER: "CAL_DELETE_OAUTH_CLIENT_USER",
    DELETE_OAUTH_CLIENT_WEBHOOK: "CAL_DELETE_OAUTH_CLIENT_WEBHOOK",
    DELETE_ORGANIZATION_ATTRIBUTE: "CAL_DELETE_ORGANIZATION_ATTRIBUTE",
    DELETE_ORGANIZATION_ATTRIBUTE_OPTION: "CAL_DELETE_ORGANIZATION_ATTRIBUTE_OPTION",
    DELETE_ORGANIZATION_MEMBERSHIP: "CAL_DELETE_ORGANIZATION_MEMBERSHIP",
    DELETE_ORG_WEBHOOK: "CAL_DELETE_ORG_WEBHOOK",
    DELETE_SCHEDULE_BY_ID: "CAL_DELETE_SCHEDULE_BY_ID",
    DELETE_SELECTED_CALENDARS: "CAL_DELETE_SELECTED_CALENDARS",
    DELETE_SELECTED_SLOT: "CAL_DELETE_SELECTED_SLOT",
    DELETE_TEAM_BY_ID: "CAL_DELETE_TEAM_BY_ID",
    DELETE_TEAM_FROM_ORGANIZATION: "CAL_DELETE_TEAM_FROM_ORGANIZATION",
    DELETE_TEAM_MEMBERSHIPS_BY_ID: "CAL_DELETE_TEAM_MEMBERSHIPS_BY_ID",
    DELETE_USER_ATTRIBUTE_OPTION: "CAL_DELETE_USER_ATTRIBUTE_OPTION",
    DELETE_USER_FROM_ORGANIZATION: "CAL_DELETE_USER_FROM_ORGANIZATION",
    DELETE_USER_SCHEDULE: "CAL_DELETE_USER_SCHEDULE",
    DELETE_WEBHOOKS_FOR_EVENT_TYPE: "CAL_DELETE_WEBHOOKS_FOR_EVENT_TYPE",
    DELETE_WEBHOOK_BY_ID: "CAL_DELETE_WEBHOOK_BY_ID",
    DELETE_WEBHOOK_FOR_EVENT_TYPE: "CAL_DELETE_WEBHOOK_FOR_EVENT_TYPE",
    DISCONNECT_CALENDAR_USING_CREDENTIAL_ID: "CAL_DISCONNECT_CALENDAR_USING_CREDENTIAL_ID",
    FETCH_ALL_BOOKINGS: "CAL_FETCH_ALL_BOOKINGS",
    FETCH_EVENT_TYPE_DETAILS: "CAL_FETCH_EVENT_TYPE_DETAILS",
    FETCH_ORGANIZATION_ATTRIBUTE_BY_ID: "CAL_FETCH_ORGANIZATION_ATTRIBUTE_BY_ID",
    FETCH_PROVIDER_ACCESS_TOKEN: "CAL_FETCH_PROVIDER_ACCESS_TOKEN",
    FETCH_SCHEDULE_BY_ID: "CAL_FETCH_SCHEDULE_BY_ID",
    FETCH_USER_SCHEDULE_BY_ORG_ID: "CAL_FETCH_USER_SCHEDULE_BY_ORG_ID",
    FETCH_WEBHOOK_BY_EVENT_TYPE_ID: "CAL_FETCH_WEBHOOK_BY_EVENT_TYPE_ID",
    FORCE_REFRESH_USER_OAUTH_CLIENT: "CAL_FORCE_REFRESH_USER_OAUTH_CLIENT",
    GET_ALL_TIMEZONES: "CAL_GET_ALL_TIMEZONES",
    GET_AVAILABLE_SLOTS_INFO: "CAL_GET_AVAILABLE_SLOTS_INFO",
    GET_CONFERENCE_OAUTH_AUTHORIZATION_URL: "CAL_GET_CONFERENCE_OAUTH_AUTHORIZATION_URL",
    GET_DEFAULT_SCHEDULE_DETAILS: "CAL_GET_DEFAULT_SCHEDULE_DETAILS",
    GET_EVENT_TYPE_BY_TEAM_ID: "CAL_GET_EVENT_TYPE_BY_TEAM_ID",
    GET_GOOGLE_CALENDAR_OAUTH_AUTHENTICATION_URL: "CAL_GET_GOOGLE_CALENDAR_OAUTH_AUTHENTICATION_URL",
    GET_OAUTH_CLIENTS_USER: "CAL_GET_OAUTH_CLIENTS_USER",
    GET_ORGANIZATION_ID: "CAL_GET_ORGANIZATION_ID",
    GET_ORGANIZATION_SCHEDULES: "CAL_GET_ORGANIZATION_SCHEDULES",
    GET_ORGANIZATION_TEAMS_EVENT_TYPES: "CAL_GET_ORGANIZATION_TEAMS_EVENT_TYPES",
    GET_ORGANIZATION_USER_SCHEDULES: "CAL_GET_ORGANIZATION_USER_SCHEDULES",
    GET_SCHEDULE_FOR_USER_IN_TEAM: "CAL_GET_SCHEDULE_FOR_USER_IN_TEAM",
    GET_STRIPE_CONNECT_INFO: "CAL_GET_STRIPE_CONNECT_INFO",
    GET_TEAMS_LIST: "CAL_GET_TEAMS_LIST",
    GET_TEAM_DETAILS_BY_ORG_ID_AND_TEAM_ID: "CAL_GET_TEAM_DETAILS_BY_ORG_ID_AND_TEAM_ID",
    GET_TEAM_INFORMATION_BY_TEAM_ID: "CAL_GET_TEAM_INFORMATION_BY_TEAM_ID",
    GET_WEBHOOK_BY_ID: "CAL_GET_WEBHOOK_BY_ID",
    HANDLE_CONFERENCING_OAUTH_CALLBACK_FOR_APP: "CAL_HANDLE_CONFERENCING_OAUTH_CALLBACK_FOR_APP",
    LIST_EVENT_TYPES: "CAL_LIST_EVENT_TYPES",
    LIST_ORGANIZATION_MEMBERSHIPS: "CAL_LIST_ORGANIZATION_MEMBERSHIPS",
    LIST_TEAM_EVENT_TYPES_BY_ORG_AND_TEAM_ID: "CAL_LIST_TEAM_EVENT_TYPES_BY_ORG_AND_TEAM_ID",
    MARK_BOOKING_ABSENT_FOR_UID: "CAL_MARK_BOOKING_ABSENT_FOR_UID",
    MODIFY_ORGANIZATION_MEMBERSHIP_BY_ID: "CAL_MODIFY_ORGANIZATION_MEMBERSHIP_BY_ID",
    MODIFY_ORG_ATTRIBUTE_BY_ID: "CAL_MODIFY_ORG_ATTRIBUTE_BY_ID",
    PATCH_ORGANIZATION_ATTRIBUTE_OPTION: "CAL_PATCH_ORGANIZATION_ATTRIBUTE_OPTION",
    PATCH_ORGANIZATION_USER_DETAILS: "CAL_PATCH_ORGANIZATION_USER_DETAILS",
    PATCH_TEAM_DETAILS_BY_ID: "CAL_PATCH_TEAM_DETAILS_BY_ID",
    PATCH_WEBHOOK_EVENT_TYPE: "CAL_PATCH_WEBHOOK_EVENT_TYPE",
    POST_CALENDAR_CREDENTIALS: "CAL_POST_CALENDAR_CREDENTIALS",
    POST_CONFERENCING_APP_CONNECT: "CAL_POST_CONFERENCING_APP_CONNECT",
    POST_NEW_BOOKING_REQUEST: "CAL_POST_NEW_BOOKING_REQUEST",
    POST_ORG_ATTRIBUTE_OPTION_BY_USER: "CAL_POST_ORG_ATTRIBUTE_OPTION_BY_USER",
    POST_SELECTED_CALENDARS: "CAL_POST_SELECTED_CALENDARS",
    POST_USER_TO_ORGANIZATION: "CAL_POST_USER_TO_ORGANIZATION",
    POST_WEBHOOK_EVENT: "CAL_POST_WEBHOOK_EVENT",
    REASSIGN_BOOKING_TO_ANOTHER_USER: "CAL_REASSIGN_BOOKING_TO_ANOTHER_USER",
    REASSIGN_BOOKING_WITH_UID: "CAL_REASSIGN_BOOKING_WITH_UID",
    REFRESH_OAUTH_TOKEN_FOR_CLIENT_ID: "CAL_REFRESH_OAUTH_TOKEN_FOR_CLIENT_ID",
    RESCHEDULE_BOOKING_BY_UID: "CAL_RESCHEDULE_BOOKING_BY_UID",
    RESERVE_SLOT_FOR_EVENT: "CAL_RESERVE_SLOT_FOR_EVENT",
    RETRIEVE_ATTRIBUTE_OPTIONS_FOR_ORG: "CAL_RETRIEVE_ATTRIBUTE_OPTIONS_FOR_ORG",
    RETRIEVE_BOOKING_DETAILS_BY_UID: "CAL_RETRIEVE_BOOKING_DETAILS_BY_UID",
    RETRIEVE_CALENDAR_BUSY_TIMES: "CAL_RETRIEVE_CALENDAR_BUSY_TIMES",
    RETRIEVE_CALENDAR_LIST: "CAL_RETRIEVE_CALENDAR_LIST",
    RETRIEVE_CURRENT_TEAM_FOR_ORGANIZATION: "CAL_RETRIEVE_CURRENT_TEAM_FOR_ORGANIZATION",
    RETRIEVE_DEFAULT_CONFERENCING_SETTINGS: "CAL_RETRIEVE_DEFAULT_CONFERENCING_SETTINGS",
    RETRIEVE_EVENT_TYPE_BY_ID: "CAL_RETRIEVE_EVENT_TYPE_BY_ID",
    RETRIEVE_MEMBERSHIP_FROM_ORGANIZATION: "CAL_RETRIEVE_MEMBERSHIP_FROM_ORGANIZATION",
    RETRIEVE_MY_INFORMATION: "CAL_RETRIEVE_MY_INFORMATION",
    RETRIEVE_OAUTH_CLIENT_USER_BY_ID: "CAL_RETRIEVE_OAUTH_CLIENT_USER_BY_ID",
    RETRIEVE_OAUTH_CLIENT_WEBHOOK_BY_ID: "CAL_RETRIEVE_OAUTH_CLIENT_WEBHOOK_BY_ID",
    RETRIEVE_ORGANIZATION_ATTRIBUTES: "CAL_RETRIEVE_ORGANIZATION_ATTRIBUTES",
    RETRIEVE_ORGANIZATION_ATTRIBUTES_OPTIONS: "CAL_RETRIEVE_ORGANIZATION_ATTRIBUTES_OPTIONS",
    RETRIEVE_ORGANIZATION_WEBHOOKS_BY_ORG_ID: "CAL_RETRIEVE_ORGANIZATION_WEBHOOKS_BY_ORG_ID",
    RETRIEVE_ORGANIZATION_WEBHOOK_BY_ID: "CAL_RETRIEVE_ORGANIZATION_WEBHOOK_BY_ID",
    RETRIEVE_PROVIDER_DETAILS: "CAL_RETRIEVE_PROVIDER_DETAILS",
    RETRIEVE_SCHEDULES_LIST: "CAL_RETRIEVE_SCHEDULES_LIST",
    RETRIEVE_TEAM_DETAILS_IN_ORGANIZATION: "CAL_RETRIEVE_TEAM_DETAILS_IN_ORGANIZATION",
    RETRIEVE_TEAM_EVENT_TYPES: "CAL_RETRIEVE_TEAM_EVENT_TYPES",
    RETRIEVE_TEAM_MEMBERSHIPS: "CAL_RETRIEVE_TEAM_MEMBERSHIPS",
    RETRIEVE_TEAM_MEMBERSHIPS_FOR_ORGANIZATION: "CAL_RETRIEVE_TEAM_MEMBERSHIPS_FOR_ORGANIZATION",
    RETRIEVE_TEAM_MEMBERSHIP_BY_ID: "CAL_RETRIEVE_TEAM_MEMBERSHIP_BY_ID",
    RETRIEVE_TEAM_MEMBERSHIP_DETAILS: "CAL_RETRIEVE_TEAM_MEMBERSHIP_DETAILS",
    RETRIEVE_USERS_IN_ORGANIZATION: "CAL_RETRIEVE_USERS_IN_ORGANIZATION",
    RETRIEVE_V2_CONFERENCING_INFO: "CAL_RETRIEVE_V2_CONFERENCING_INFO",
    RETRIEVE_WEBHOOKS_FOR_EVENT_TYPE: "CAL_RETRIEVE_WEBHOOKS_FOR_EVENT_TYPE",
    RETRIEVE_WEBHOOKS_LIST: "CAL_RETRIEVE_WEBHOOKS_LIST",
    RETRIEVE_WEBHOOK_DETAILS_FOR_OAUTH_CLIENT: "CAL_RETRIEVE_WEBHOOK_DETAILS_FOR_OAUTH_CLIENT",
    SAVE_CALENDAR_ENTRY: "CAL_SAVE_CALENDAR_ENTRY",
    SAVE_CALENDAR_ICS_FEEDS: "CAL_SAVE_CALENDAR_ICS_FEEDS",
    SAVE_OAUTH_CREDENTIALS_VIA_GCAL_API: "CAL_SAVE_OAUTH_CREDENTIALS_VIA_GCAL_API",
    SAVE_STRIPE_DETAILS: "CAL_SAVE_STRIPE_DETAILS",
    SET_DEFAULT_CONFERENCING_APP: "CAL_SET_DEFAULT_CONFERENCING_APP",
    UPDATE_DESTINATION_CALENDAR_INTEGRATION: "CAL_UPDATE_DESTINATION_CALENDAR_INTEGRATION",
    UPDATE_OAUTH_CLIENT_USER_SETTINGS: "CAL_UPDATE_OAUTH_CLIENT_USER_SETTINGS",
    UPDATE_OAUTH_CLIENT_WEBHOOK: "CAL_UPDATE_OAUTH_CLIENT_WEBHOOK",
    UPDATE_SCHEDULE_BY_ID: "CAL_UPDATE_SCHEDULE_BY_ID",
    UPDATE_TEAM_INFORMATION_BY_ID: "CAL_UPDATE_TEAM_INFORMATION_BY_ID",
    UPDATE_TEAM_MEMBERSHIP_BY_ID: "CAL_UPDATE_TEAM_MEMBERSHIP_BY_ID",
    UPDATE_TEAM_MEMBERSHIP_PROPERTIES: "CAL_UPDATE_TEAM_MEMBERSHIP_PROPERTIES",
    UPDATE_USER_PROFILE_DETAILS: "CAL_UPDATE_USER_PROFILE_DETAILS",
    UPDATE_USER_SCHEDULE_IN_ORGANIZATION: "CAL_UPDATE_USER_SCHEDULE_IN_ORGANIZATION",
    UPDATE_WEBHOOK_BY_ID: "CAL_UPDATE_WEBHOOK_BY_ID",
    UPDATE_WEBHOOK_FOR_ORGANIZATION: "CAL_UPDATE_WEBHOOK_FOR_ORGANIZATION",
  },
  triggerTypes: {},
}
