// @composio/core/generated/calendly.ts
/**
 * Map of Composio's CALENDLY toolkit.
 */
export const CALENDLY = {
  slug: "calendly",
  tools: {
    CANCEL_EVENT: "CALENDLY_CANCEL_EVENT",
    CREATE_INVITEE_NO_SHOW: "CALENDLY_CREATE_INVITEE_NO_SHOW",
    CREATE_ONE_OFF_EVENT_TYPE: "CALENDLY_CREATE_ONE_OFF_EVENT_TYPE",
    CREATE_SCHEDULING_LINK: "CALENDLY_CREATE_SCHEDULING_LINK",
    CREATE_SHARE: "CALENDLY_CREATE_SHARE",
    CREATE_SINGLE_USE_SCHEDULING_LINK: "CALENDLY_CREATE_SINGLE_USE_SCHEDULING_LINK",
    CREATE_WEBHOOK_SUBSCRIPTION: "CALENDLY_CREATE_WEBHOOK_SUBSCRIPTION",
    DELETE_INVITEE_DATA: "CALENDLY_DELETE_INVITEE_DATA",
    DELETE_INVITEE_NO_SHOW: "CALENDLY_DELETE_INVITEE_NO_SHOW",
    DELETE_SCHEDULED_EVENT_DATA: "CALENDLY_DELETE_SCHEDULED_EVENT_DATA",
    DELETE_WEBHOOK_SUBSCRIPTION: "CALENDLY_DELETE_WEBHOOK_SUBSCRIPTION",
    GET_CURRENT_USER: "CALENDLY_GET_CURRENT_USER",
    GET_EVENT: "CALENDLY_GET_EVENT",
    GET_EVENT_INVITEE: "CALENDLY_GET_EVENT_INVITEE",
    GET_EVENT_TYPE: "CALENDLY_GET_EVENT_TYPE",
    GET_GROUP: "CALENDLY_GET_GROUP",
    GET_GROUP_RELATIONSHIP: "CALENDLY_GET_GROUP_RELATIONSHIP",
    GET_INVITEE_NO_SHOW: "CALENDLY_GET_INVITEE_NO_SHOW",
    GET_ORGANIZATION_INVITATION: "CALENDLY_GET_ORGANIZATION_INVITATION",
    GET_ORGANIZATION_MEMBERSHIP: "CALENDLY_GET_ORGANIZATION_MEMBERSHIP",
    GET_ROUTING_FORM: "CALENDLY_GET_ROUTING_FORM",
    GET_USER: "CALENDLY_GET_USER",
    GET_USER_AVAILABILITY_SCHEDULE: "CALENDLY_GET_USER_AVAILABILITY_SCHEDULE",
    GET_WEBHOOK_SUBSCRIPTION: "CALENDLY_GET_WEBHOOK_SUBSCRIPTION",
    INVITE_USER_TO_ORGANIZATION: "CALENDLY_INVITE_USER_TO_ORGANIZATION",
    LIST_ACTIVITY_LOG_ENTRIES: "CALENDLY_LIST_ACTIVITY_LOG_ENTRIES",
    LIST_EVENTS: "CALENDLY_LIST_EVENTS",
    LIST_EVENT_INVITEES: "CALENDLY_LIST_EVENT_INVITEES",
    LIST_EVENT_TYPE_AVAILABLE_TIMES: "CALENDLY_LIST_EVENT_TYPE_AVAILABLE_TIMES",
    LIST_EVENT_TYPE_HOSTS: "CALENDLY_LIST_EVENT_TYPE_HOSTS",
    LIST_GROUPS: "CALENDLY_LIST_GROUPS",
    LIST_GROUP_RELATIONSHIPS: "CALENDLY_LIST_GROUP_RELATIONSHIPS",
    LIST_ORGANIZATION_INVITATIONS: "CALENDLY_LIST_ORGANIZATION_INVITATIONS",
    LIST_ORGANIZATION_MEMBERSHIPS: "CALENDLY_LIST_ORGANIZATION_MEMBERSHIPS",
    LIST_OUTGOING_COMMUNICATIONS: "CALENDLY_LIST_OUTGOING_COMMUNICATIONS",
    LIST_ROUTING_FORMS: "CALENDLY_LIST_ROUTING_FORMS",
    LIST_USER_AVAILABILITY_SCHEDULES: "CALENDLY_LIST_USER_AVAILABILITY_SCHEDULES",
    LIST_USER_BUSY_TIMES: "CALENDLY_LIST_USER_BUSY_TIMES",
    LIST_USER_S_EVENT_TYPES: "CALENDLY_LIST_USER_S_EVENT_TYPES",
    LIST_WEBHOOK_SUBSCRIPTIONS: "CALENDLY_LIST_WEBHOOK_SUBSCRIPTIONS",
    REMOVE_USER_FROM_ORGANIZATION: "CALENDLY_REMOVE_USER_FROM_ORGANIZATION",
    REVOKE_USER_S_ORGANIZATION_INVITATION: "CALENDLY_REVOKE_USER_S_ORGANIZATION_INVITATION",
  },
  triggerTypes: {},
}
