// @composio/core/generated/canvas.ts
/**
 * Map of Composio's CANVAS toolkit.
 */
export const CANVAS = {
  slug: "canvas",
  tools: {
    CREATE_ACCOUNT_NOTIFICATION: "CANVAS_CREATE_ACCOUNT_NOTIFICATION",
    CREATE_APPOINTMENT_GROUP: "CANVAS_CREATE_APPOINTMENT_GROUP",
    CREATE_ASSIGNMENT: "CANVAS_CREATE_ASSIGNMENT",
    CREATE_ASSIGNMENT_OVERRIDE: "CANVAS_CREATE_ASSIGNMENT_OVERRIDE",
    CREATE_CALENDAR_EVENT: "CANVAS_CREATE_CALENDAR_EVENT",
    CREATE_COMMUNICATION_CHANNEL: "CANVAS_CREATE_COMMUNICATION_CHANNEL",
    CREATE_CONTENT_SHARE: "CANVAS_CREATE_CONTENT_SHARE",
    CREATE_CONVERSATION: "CANVAS_CREATE_CONVERSATION",
    CREATE_COURSE: "CANVAS_CREATE_COURSE",
    CREATE_DISCUSSION_TOPIC: "CANVAS_CREATE_DISCUSSION_TOPIC",
    CREATE_ENROLLMENT: "CANVAS_CREATE_ENROLLMENT",
    CREATE_FOLDER: "CANVAS_CREATE_FOLDER",
    CREATE_MODULE: "CANVAS_CREATE_MODULE",
    CREATE_PAGE_FOR_COURSE: "CANVAS_CREATE_PAGE_FOR_COURSE",
    CREATE_QUIZ: "CANVAS_CREATE_QUIZ",
    CREATE_QUIZ_QUESTION: "CANVAS_CREATE_QUIZ_QUESTION",
    DELETE_APPOINTMENT_GROUP: "CANVAS_DELETE_APPOINTMENT_GROUP",
    DELETE_ASSIGNMENT: "CANVAS_DELETE_ASSIGNMENT",
    DELETE_FOLDER: "CANVAS_DELETE_FOLDER",
    DELETE_QUIZ: "CANVAS_DELETE_QUIZ",
    DEPARTMENT_LEVEL_GRADE_DATA_FOR_CURRENT_GRADES: "CANVAS_DEPARTMENT_LEVEL_GRADE_DATA_FOR_CURRENT_GRADES",
    DEPARTMENT_LEVEL_GRADE_DATA_FOR_TERM_GRADES: "CANVAS_DEPARTMENT_LEVEL_GRADE_DATA_FOR_TERM_GRADES",
    EDIT_ASSIGNMENT: "CANVAS_EDIT_ASSIGNMENT",
    EDIT_QUIZ: "CANVAS_EDIT_QUIZ",
    EXPORT_CONTENT: "CANVAS_EXPORT_CONTENT",
    FETCH_DATA: "CANVAS_FETCH_DATA",
    GET_ACCOUNTS: "CANVAS_GET_ACCOUNTS",
    GET_ALL_ASSIGNMENTS: "CANVAS_GET_ALL_ASSIGNMENTS",
    GET_ALL_USERS: "CANVAS_GET_ALL_USERS",
    GET_ASSIGNMENT: "CANVAS_GET_ASSIGNMENT",
    GET_ASSIGNMENT_RUBRIC: "CANVAS_GET_ASSIGNMENT_RUBRIC",
    GET_COURSE_LEVEL_PARTICIPATION_DATA: "CANVAS_GET_COURSE_LEVEL_PARTICIPATION_DATA",
    GET_CURRENT_USER: "CANVAS_GET_CURRENT_USER",
    GET_DEPARTMENT_LEVEL_COMPLETED_STATISTICS: "CANVAS_GET_DEPARTMENT_LEVEL_COMPLETED_STATISTICS",
    GET_DEPARTMENT_LEVEL_CURRENT_STATISTICS: "CANVAS_GET_DEPARTMENT_LEVEL_CURRENT_STATISTICS",
    GET_DEPARTMENT_LEVEL_GRADE_DATA_COMPLETED_GRADES: "CANVAS_GET_DEPARTMENT_LEVEL_GRADE_DATA_COMPLETED_GRADES",
    GET_DEPARTMENT_LEVEL_TERM_STATISTICS: "CANVAS_GET_DEPARTMENT_LEVEL_TERM_STATISTICS",
    GET_ENROLLMENT_BY_ID: "CANVAS_GET_ENROLLMENT_BY_ID",
    GET_GRADEBOOK_HISTORY_DAYS: "CANVAS_GET_GRADEBOOK_HISTORY_DAYS",
    GET_PAGE_FOR_COURSE: "CANVAS_GET_PAGE_FOR_COURSE",
    GET_QUIZ_SUBMISSIONS: "CANVAS_GET_QUIZ_SUBMISSIONS",
    GET_REPORT_STATUS: "CANVAS_GET_REPORT_STATUS",
    GET_SINGLE_COURSE: "CANVAS_GET_SINGLE_COURSE",
    GET_SINGLE_SUBMISSION: "CANVAS_GET_SINGLE_SUBMISSION",
    GET_SUBMISSION_FILES: "CANVAS_GET_SUBMISSION_FILES",
    GET_USER_ASSIGNMENT_ANALYTICS: "CANVAS_GET_USER_ASSIGNMENT_ANALYTICS",
    GET_USER_COURSE_PROGRESS: "CANVAS_GET_USER_COURSE_PROGRESS",
    GET_USER_PARTICIPATION_ANALYTICS: "CANVAS_GET_USER_PARTICIPATION_ANALYTICS",
    GET_USER_PROFILE: "CANVAS_GET_USER_PROFILE",
    GRADE_COMMENT_SUBMISSION: "CANVAS_GRADE_COMMENT_SUBMISSION",
    LIST_ACCOUNTS_FOR_COURSE_ADMINS: "CANVAS_LIST_ACCOUNTS_FOR_COURSE_ADMINS",
    LIST_ALL_FOLDERS: "CANVAS_LIST_ALL_FOLDERS",
    LIST_APPOINTMENT_GROUPS: "CANVAS_LIST_APPOINTMENT_GROUPS",
    LIST_ASSIGNMENT_SUBMISSIONS: "CANVAS_LIST_ASSIGNMENT_SUBMISSIONS",
    LIST_AVAILABLE_REPORTS: "CANVAS_LIST_AVAILABLE_REPORTS",
    LIST_CALENDAR_EVENTS_FOR_USER: "CANVAS_LIST_CALENDAR_EVENTS_FOR_USER",
    LIST_COMMUNICATION_CHANNELS: "CANVAS_LIST_COMMUNICATION_CHANNELS",
    LIST_COMM_MESSAGES: "CANVAS_LIST_COMM_MESSAGES",
    LIST_CONTENT_EXPORTS: "CANVAS_LIST_CONTENT_EXPORTS",
    LIST_COURSES: "CANVAS_LIST_COURSES",
    LIST_COURSES_FOR_USER: "CANVAS_LIST_COURSES_FOR_USER",
    LIST_COURSE_USERS: "CANVAS_LIST_COURSE_USERS",
    LIST_DISCUSSION_TOPICS: "CANVAS_LIST_DISCUSSION_TOPICS",
    LIST_FILES: "CANVAS_LIST_FILES",
    LIST_FOLDERS: "CANVAS_LIST_FOLDERS",
    LIST_PAGES_FOR_COURSE: "CANVAS_LIST_PAGES_FOR_COURSE",
    LIST_QUIZZES_IN_COURSE: "CANVAS_LIST_QUIZZES_IN_COURSE",
    LIST_SENT_CONTENT_SHARES: "CANVAS_LIST_SENT_CONTENT_SHARES",
    LIST_SUBMISSIONS: "CANVAS_LIST_SUBMISSIONS",
    LIST_SUBMISSIONSFOR_MULTIPLE_ASSIGNMENTS: "CANVAS_LIST_SUBMISSIONSFOR_MULTIPLE_ASSIGNMENTS",
    LIST_UNCOLLATED_SUBMISSION_VERSIONS: "CANVAS_LIST_UNCOLLATED_SUBMISSION_VERSIONS",
    LIST_USERS_IN_ACCOUNT: "CANVAS_LIST_USERS_IN_ACCOUNT",
    RETRIEVE_ENROLLMENT_TERM: "CANVAS_RETRIEVE_ENROLLMENT_TERM",
    START_REPORT: "CANVAS_START_REPORT",
    TRANSLATE_FILE_REFERENCE: "CANVAS_TRANSLATE_FILE_REFERENCE",
    UPDATE_APPOINTMENT_GROUP: "CANVAS_UPDATE_APPOINTMENT_GROUP",
    UPDATE_COURSE: "CANVAS_UPDATE_COURSE",
    UPDATE_COURSE_SETTINGS: "CANVAS_UPDATE_COURSE_SETTINGS",
    UPDATE_FILE: "CANVAS_UPDATE_FILE",
    UPDATE_PAGE_FOR_COURSE: "CANVAS_UPDATE_PAGE_FOR_COURSE",
    UPDATE_QUIZ_SUBMISSION_SCORES: "CANVAS_UPDATE_QUIZ_SUBMISSION_SCORES",
    UPDATE_USER_SETTINGS: "CANVAS_UPDATE_USER_SETTINGS",
    UPLOAD_COURSE_FILE: "CANVAS_UPLOAD_COURSE_FILE",
    UPLOAD_SUBMISSION_FILE: "CANVAS_UPLOAD_SUBMISSION_FILE",
  },
  triggerTypes: {
    ASSIGNMENT_GRADED_TRIGGER: "CANVAS_ASSIGNMENT_GRADED_TRIGGER",
    NEW_ASSIGNMENT_SUBMISSION_TRIGGER: "CANVAS_NEW_ASSIGNMENT_SUBMISSION_TRIGGER",
    NEW_CANVAS_DISCUSSION_MESSAGE_TRIGGER: "CANVAS_NEW_CANVAS_DISCUSSION_MESSAGE_TRIGGER",
    NEW_CANVAS_DISCUSSION_TRIGGER: "CANVAS_NEW_CANVAS_DISCUSSION_TRIGGER",
    NEW_COURSE_CREATED_TRIGGER: "CANVAS_NEW_COURSE_CREATED_TRIGGER",
    NEW_FILE_UPLOADED_TRIGGER: "CANVAS_NEW_FILE_UPLOADED_TRIGGER",
  },
}
