// @composio/core/generated/canva.ts
/**
 * Map of Composio's CANVA toolkit.
 */
export const CANVA = {
  slug: "canva",
  tools: {
    ACCESS_USER_SPECIFIC_BRAND_TEMPLATES_LIST: "CANVA_ACCESS_USER_SPECIFIC_BRAND_TEMPLATES_LIST",
    CREATE_CANVA_DESIGN_WITH_OPTIONAL_ASSET: "CANVA_CREATE_CANVA_DESIGN_WITH_OPTIONAL_ASSET",
    CREATE_COMMENT_REPLY_IN_DESIGN: "CANVA_CREATE_COMMENT_REPLY_IN_DESIGN",
    CREATE_DESIGN_COMMENT_IN_PREVIEW_API: "CANVA_CREATE_DESIGN_COMMENT_IN_PREVIEW_API",
    CREATE_USER_OR_SUB_FOLDER: "CANVA_CREATE_USER_OR_SUB_FOLDER",
    DELETE_ASSET_BY_ID: "CANVA_DELETE_ASSET_BY_ID",
    EXCHANGE_OAUTH_2_0_ACCESS_OR_REFRESH_TOKEN: "CANVA_EXCHANGE_OAUTH_2_0_ACCESS_OR_REFRESH_TOKEN",
    FETCH_ASSET_UPLOAD_JOB_STATUS: "CANVA_FETCH_ASSET_UPLOAD_JOB_STATUS",
    FETCH_CANVA_CONNECT_SIGNING_PUBLIC_KEYS: "CANVA_FETCH_CANVA_CONNECT_SIGNING_PUBLIC_KEYS",
    FETCH_CURRENT_USER_DETAILS: "CANVA_FETCH_CURRENT_USER_DETAILS",
    FETCH_DESIGN_METADATA_AND_ACCESS_INFORMATION: "CANVA_FETCH_DESIGN_METADATA_AND_ACCESS_INFORMATION",
    GET_DESIGN_EXPORT_JOB_RESULT: "CANVA_GET_DESIGN_EXPORT_JOB_RESULT",
    INITIATES_CANVA_DESIGN_EXPORT_JOB: "CANVA_INITIATES_CANVA_DESIGN_EXPORT_JOB",
    INITIATE_CANVA_DESIGN_AUTOFILL_JOB: "CANVA_INITIATE_CANVA_DESIGN_AUTOFILL_JOB",
    LIST_DESIGN_PAGES_WITH_PAGINATION: "CANVA_LIST_DESIGN_PAGES_WITH_PAGINATION",
    LIST_FOLDER_ITEMS_BY_TYPE_WITH_SORTING: "CANVA_LIST_FOLDER_ITEMS_BY_TYPE_WITH_SORTING",
    LIST_USER_DESIGNS: "CANVA_LIST_USER_DESIGNS",
    MOVE_ITEM_TO_SPECIFIED_FOLDER: "CANVA_MOVE_ITEM_TO_SPECIFIED_FOLDER",
    REMOVE_FOLDER_AND_MOVE_CONTENTS_TO_TRASH: "CANVA_REMOVE_FOLDER_AND_MOVE_CONTENTS_TO_TRASH",
    RETRIEVE_APP_PUBLIC_KEY_SET: "CANVA_RETRIEVE_APP_PUBLIC_KEY_SET",
    RETRIEVE_ASSET_METADATA_BY_ID: "CANVA_RETRIEVE_ASSET_METADATA_BY_ID",
    RETRIEVE_A_SPECIFIC_DESIGN_COMMENT: "CANVA_RETRIEVE_A_SPECIFIC_DESIGN_COMMENT",
    RETRIEVE_BRAND_TEMPLATE_DATASET_DEFINITION: "CANVA_RETRIEVE_BRAND_TEMPLATE_DATASET_DEFINITION",
    RETRIEVE_CANVA_ENTERPRISE_BRAND_TEMPLATE_METADATA: "CANVA_RETRIEVE_CANVA_ENTERPRISE_BRAND_TEMPLATE_METADATA",
    RETRIEVE_DESIGN_AUTOFILL_JOB_STATUS: "CANVA_RETRIEVE_DESIGN_AUTOFILL_JOB_STATUS",
    RETRIEVE_DESIGN_IMPORT_JOB_STATUS: "CANVA_RETRIEVE_DESIGN_IMPORT_JOB_STATUS",
    RETRIEVE_FOLDER_DETAILS_BY_ID: "CANVA_RETRIEVE_FOLDER_DETAILS_BY_ID",
    RETRIEVE_USER_PROFILE_DATA: "CANVA_RETRIEVE_USER_PROFILE_DATA",
    REVOKE_OAUTH_TOKENS: "CANVA_REVOKE_OAUTH_TOKENS",
    UPDATE_ASSET_S_NAME_AND_TAGS_BY_ID: "CANVA_UPDATE_ASSET_S_NAME_AND_TAGS_BY_ID",
    UPDATE_FOLDER_DETAILS_BY_ID: "CANVA_UPDATE_FOLDER_DETAILS_BY_ID",
    VALIDATE_OAUTH_TOKEN_PROPERTIES: "CANVA_VALIDATE_OAUTH_TOKEN_PROPERTIES",
  },
  triggerTypes: {},
}
