// @composio/core/generated/clickup.ts
/**
 * Map of Composio's CLICKUP toolkit.
 */
export const CLICKUP = {
  slug: "clickup",
  tools: {
    ADD_DEPENDENCY: "CLICKUP_ADD_DEPENDENCY",
    ADD_GUEST_TO_FOLDER: "CLICKUP_ADD_GUEST_TO_FOLDER",
    ADD_GUEST_TO_LIST: "CLICKUP_ADD_GUEST_TO_LIST",
    ADD_GUEST_TO_TASK: "CLICKUP_ADD_GUEST_TO_TASK",
    ADD_TAGS_FROM_TIME_ENTRIES: "CLICKUP_ADD_TAGS_FROM_TIME_ENTRIES",
    ADD_TAG_TO_TASK: "CLICKUP_ADD_TAG_TO_TASK",
    ADD_TASK_LINK: "CLICKUP_ADD_TASK_LINK",
    ADD_TASK_TO_LIST: "CLICKUP_ADD_TASK_TO_LIST",
    ATTACHMENTS_UPLOAD_FILE_TO_TASK_AS_ATTACHMENT: "CLICKUP_ATTACHMENTS_UPLOAD_FILE_TO_TASK_AS_ATTACHMENT",
    AUTHORIZATION_GET_ACCESS_TOKEN: "CLICKUP_AUTHORIZATION_GET_ACCESS_TOKEN",
    AUTHORIZATION_GET_WORK_SPACE_LIST: "CLICKUP_AUTHORIZATION_GET_WORK_SPACE_LIST",
    AUTHORIZATION_VIEW_ACCOUNT_DETAILS: "CLICKUP_AUTHORIZATION_VIEW_ACCOUNT_DETAILS",
    CHANGE_TAG_NAMES_FROM_TIME_ENTRIES: "CLICKUP_CHANGE_TAG_NAMES_FROM_TIME_ENTRIES",
    CREATE_A_TIME_ENTRY: "CLICKUP_CREATE_A_TIME_ENTRY",
    CREATE_CHAT_VIEW_COMMENT: "CLICKUP_CREATE_CHAT_VIEW_COMMENT",
    CREATE_CHECKLIST: "CLICKUP_CREATE_CHECKLIST",
    CREATE_CHECKLIST_ITEM: "CLICKUP_CREATE_CHECKLIST_ITEM",
    CREATE_FOLDER: "CLICKUP_CREATE_FOLDER",
    CREATE_FOLDERLESS_LIST: "CLICKUP_CREATE_FOLDERLESS_LIST",
    CREATE_FOLDER_VIEW: "CLICKUP_CREATE_FOLDER_VIEW",
    CREATE_GOAL: "CLICKUP_CREATE_GOAL",
    CREATE_KEY_RESULT: "CLICKUP_CREATE_KEY_RESULT",
    CREATE_LIST: "CLICKUP_CREATE_LIST",
    CREATE_LIST_COMMENT: "CLICKUP_CREATE_LIST_COMMENT",
    CREATE_LIST_VIEW: "CLICKUP_CREATE_LIST_VIEW",
    CREATE_SPACE: "CLICKUP_CREATE_SPACE",
    CREATE_SPACE_TAG: "CLICKUP_CREATE_SPACE_TAG",
    CREATE_SPACE_VIEW: "CLICKUP_CREATE_SPACE_VIEW",
    CREATE_TASK: "CLICKUP_CREATE_TASK",
    CREATE_TASK_ATTACHMENT: "CLICKUP_CREATE_TASK_ATTACHMENT",
    CREATE_TASK_COMMENT: "CLICKUP_CREATE_TASK_COMMENT",
    CREATE_TASK_FROM_TEMPLATE: "CLICKUP_CREATE_TASK_FROM_TEMPLATE",
    CREATE_TEAM: "CLICKUP_CREATE_TEAM",
    CREATE_WEBHOOK: "CLICKUP_CREATE_WEBHOOK",
    CREATE_WORKSPACE_EVERYTHING_LEVEL_VIEW: "CLICKUP_CREATE_WORKSPACE_EVERYTHING_LEVEL_VIEW",
    DELETE_A_TIME_ENTRY: "CLICKUP_DELETE_A_TIME_ENTRY",
    DELETE_CHECKLIST: "CLICKUP_DELETE_CHECKLIST",
    DELETE_CHECKLIST_ITEM: "CLICKUP_DELETE_CHECKLIST_ITEM",
    DELETE_COMMENT: "CLICKUP_DELETE_COMMENT",
    DELETE_DEPENDENCY: "CLICKUP_DELETE_DEPENDENCY",
    DELETE_FOLDER: "CLICKUP_DELETE_FOLDER",
    DELETE_GOAL: "CLICKUP_DELETE_GOAL",
    DELETE_KEY_RESULT: "CLICKUP_DELETE_KEY_RESULT",
    DELETE_LIST: "CLICKUP_DELETE_LIST",
    DELETE_SPACE: "CLICKUP_DELETE_SPACE",
    DELETE_SPACE_TAG: "CLICKUP_DELETE_SPACE_TAG",
    DELETE_TASK: "CLICKUP_DELETE_TASK",
    DELETE_TASK_LINK: "CLICKUP_DELETE_TASK_LINK",
    DELETE_TEAM: "CLICKUP_DELETE_TEAM",
    DELETE_TIME_TRACKED: "CLICKUP_DELETE_TIME_TRACKED",
    DELETE_VIEW: "CLICKUP_DELETE_VIEW",
    DELETE_WEBHOOK: "CLICKUP_DELETE_WEBHOOK",
    EDIT_CHECKLIST: "CLICKUP_EDIT_CHECKLIST",
    EDIT_CHECKLIST_ITEM: "CLICKUP_EDIT_CHECKLIST_ITEM",
    EDIT_GUEST_ON_WORKSPACE: "CLICKUP_EDIT_GUEST_ON_WORKSPACE",
    EDIT_KEY_RESULT: "CLICKUP_EDIT_KEY_RESULT",
    EDIT_SPACE_TAG: "CLICKUP_EDIT_SPACE_TAG",
    EDIT_TIME_TRACKED: "CLICKUP_EDIT_TIME_TRACKED",
    EDIT_USER_ON_WORKSPACE: "CLICKUP_EDIT_USER_ON_WORKSPACE",
    FOLDERS_CREATE_NEW_FOLDER: "CLICKUP_FOLDERS_CREATE_NEW_FOLDER",
    FOLDERS_GET_CONTENTS_OF: "CLICKUP_FOLDERS_GET_CONTENTS_OF",
    FOLDERS_GET_FOLDER_CONTENT: "CLICKUP_FOLDERS_GET_FOLDER_CONTENT",
    GET_ACCESSIBLE_CUSTOM_FIELDS: "CLICKUP_GET_ACCESSIBLE_CUSTOM_FIELDS",
    GET_ACCESS_TOKEN: "CLICKUP_GET_ACCESS_TOKEN",
    GET_ALL_TAGS_FROM_TIME_ENTRIES: "CLICKUP_GET_ALL_TAGS_FROM_TIME_ENTRIES",
    GET_AUTHORIZED_TEAMS_WORKSPACES: "CLICKUP_GET_AUTHORIZED_TEAMS_WORKSPACES",
    GET_AUTHORIZED_USER: "CLICKUP_GET_AUTHORIZED_USER",
    GET_BULK_TASKS_TIME_IN_STATUS: "CLICKUP_GET_BULK_TASKS_TIME_IN_STATUS",
    GET_CHAT_VIEW_COMMENTS: "CLICKUP_GET_CHAT_VIEW_COMMENTS",
    GET_CUSTOM_ROLES: "CLICKUP_GET_CUSTOM_ROLES",
    GET_CUSTOM_TASK_TYPES: "CLICKUP_GET_CUSTOM_TASK_TYPES",
    GET_FILTERED_TEAM_TASKS: "CLICKUP_GET_FILTERED_TEAM_TASKS",
    GET_FOLDER: "CLICKUP_GET_FOLDER",
    GET_FOLDERLESS_LISTS: "CLICKUP_GET_FOLDERLESS_LISTS",
    GET_FOLDERS: "CLICKUP_GET_FOLDERS",
    GET_FOLDER_VIEWS: "CLICKUP_GET_FOLDER_VIEWS",
    GET_GOAL: "CLICKUP_GET_GOAL",
    GET_GOALS: "CLICKUP_GET_GOALS",
    GET_GUEST: "CLICKUP_GET_GUEST",
    GET_LIST: "CLICKUP_GET_LIST",
    GET_LISTS: "CLICKUP_GET_LISTS",
    GET_LIST_COMMENTS: "CLICKUP_GET_LIST_COMMENTS",
    GET_LIST_MEMBERS: "CLICKUP_GET_LIST_MEMBERS",
    GET_LIST_VIEWS: "CLICKUP_GET_LIST_VIEWS",
    GET_RUNNING_TIME_ENTRY: "CLICKUP_GET_RUNNING_TIME_ENTRY",
    GET_SINGULAR_TIME_ENTRY: "CLICKUP_GET_SINGULAR_TIME_ENTRY",
    GET_SPACE: "CLICKUP_GET_SPACE",
    GET_SPACES: "CLICKUP_GET_SPACES",
    GET_SPACE_TAGS: "CLICKUP_GET_SPACE_TAGS",
    GET_SPACE_VIEWS: "CLICKUP_GET_SPACE_VIEWS",
    GET_TASK: "CLICKUP_GET_TASK",
    GET_TASKS: "CLICKUP_GET_TASKS",
    GET_TASK_COMMENTS: "CLICKUP_GET_TASK_COMMENTS",
    GET_TASK_MEMBERS: "CLICKUP_GET_TASK_MEMBERS",
    GET_TASK_S_TIME_IN_STATUS: "CLICKUP_GET_TASK_S_TIME_IN_STATUS",
    GET_TASK_TEMPLATES: "CLICKUP_GET_TASK_TEMPLATES",
    GET_TEAMS: "CLICKUP_GET_TEAMS",
    GET_TIME_ENTRIES_WITHIN_A_DATE_RANGE: "CLICKUP_GET_TIME_ENTRIES_WITHIN_A_DATE_RANGE",
    GET_TIME_ENTRY_HISTORY: "CLICKUP_GET_TIME_ENTRY_HISTORY",
    GET_TRACKED_TIME: "CLICKUP_GET_TRACKED_TIME",
    GET_USER: "CLICKUP_GET_USER",
    GET_VIEW: "CLICKUP_GET_VIEW",
    GET_VIEW_TASKS: "CLICKUP_GET_VIEW_TASKS",
    GET_WEBHOOKS: "CLICKUP_GET_WEBHOOKS",
    GET_WORKSPACE_EVERYTHING_LEVEL_VIEWS: "CLICKUP_GET_WORKSPACE_EVERYTHING_LEVEL_VIEWS",
    GET_WORKSPACE_PLAN: "CLICKUP_GET_WORKSPACE_PLAN",
    GET_WORKSPACE_SEATS: "CLICKUP_GET_WORKSPACE_SEATS",
    INVITE_GUEST_TO_WORKSPACE: "CLICKUP_INVITE_GUEST_TO_WORKSPACE",
    INVITE_USER_TO_WORKSPACE: "CLICKUP_INVITE_USER_TO_WORKSPACE",
    LISTS_GET_FOLDER_LISTS: "CLICKUP_LISTS_GET_FOLDER_LISTS",
    MEMBERS_GET_LIST_USERS: "CLICKUP_MEMBERS_GET_LIST_USERS",
    REMOVE_CUSTOM_FIELD_VALUE: "CLICKUP_REMOVE_CUSTOM_FIELD_VALUE",
    REMOVE_GUEST_FROM_FOLDER: "CLICKUP_REMOVE_GUEST_FROM_FOLDER",
    REMOVE_GUEST_FROM_LIST: "CLICKUP_REMOVE_GUEST_FROM_LIST",
    REMOVE_GUEST_FROM_TASK: "CLICKUP_REMOVE_GUEST_FROM_TASK",
    REMOVE_GUEST_FROM_WORKSPACE: "CLICKUP_REMOVE_GUEST_FROM_WORKSPACE",
    REMOVE_TAGS_FROM_TIME_ENTRIES: "CLICKUP_REMOVE_TAGS_FROM_TIME_ENTRIES",
    REMOVE_TAG_FROM_TASK: "CLICKUP_REMOVE_TAG_FROM_TASK",
    REMOVE_TASK_FROM_LIST: "CLICKUP_REMOVE_TASK_FROM_LIST",
    REMOVE_USER_FROM_WORKSPACE: "CLICKUP_REMOVE_USER_FROM_WORKSPACE",
    SET_CUSTOM_FIELD_VALUE: "CLICKUP_SET_CUSTOM_FIELD_VALUE",
    SHARED_HIERARCHY: "CLICKUP_SHARED_HIERARCHY",
    SPACES_GET_DETAILS: "CLICKUP_SPACES_GET_DETAILS",
    SPACES_GET_SPACE_DETAILS: "CLICKUP_SPACES_GET_SPACE_DETAILS",
    START_A_TIME_ENTRY: "CLICKUP_START_A_TIME_ENTRY",
    STOP_A_TIME_ENTRY: "CLICKUP_STOP_A_TIME_ENTRY",
    TASKS_GET_TASK_DETAILS: "CLICKUP_TASKS_GET_TASK_DETAILS",
    TASK_CHECKLISTS_CREATE_NEW_CHECKLIST: "CLICKUP_TASK_CHECKLISTS_CREATE_NEW_CHECKLIST",
    TEAMS_USER_GROUPS_CREATE_TEAM: "CLICKUP_TEAMS_USER_GROUPS_CREATE_TEAM",
    TEAMS_WORK_SPACES_GET_WORK_SPACE_PLAN: "CLICKUP_TEAMS_WORK_SPACES_GET_WORK_SPACE_PLAN",
    TEAMS_WORK_SPACES_GET_WORK_SPACE_SEATS: "CLICKUP_TEAMS_WORK_SPACES_GET_WORK_SPACE_SEATS",
    TRACK_TIME: "CLICKUP_TRACK_TIME",
    UPDATE_A_TIME_ENTRY: "CLICKUP_UPDATE_A_TIME_ENTRY",
    UPDATE_COMMENT: "CLICKUP_UPDATE_COMMENT",
    UPDATE_FOLDER: "CLICKUP_UPDATE_FOLDER",
    UPDATE_GOAL: "CLICKUP_UPDATE_GOAL",
    UPDATE_LIST: "CLICKUP_UPDATE_LIST",
    UPDATE_SPACE: "CLICKUP_UPDATE_SPACE",
    UPDATE_TASK: "CLICKUP_UPDATE_TASK",
    UPDATE_TEAM: "CLICKUP_UPDATE_TEAM",
    UPDATE_VIEW: "CLICKUP_UPDATE_VIEW",
    UPDATE_WEBHOOK: "CLICKUP_UPDATE_WEBHOOK",
    VIEWS_GET_EVERYTHING_LEVEL: "CLICKUP_VIEWS_GET_EVERYTHING_LEVEL",
    VIEWS_SPACE_VIEWS_GET: "CLICKUP_VIEWS_SPACE_VIEWS_GET",
  },
  triggerTypes: {},
}
