// @composio/core/generated/coda.ts
/**
 * Map of Composio's CODA toolkit.
 */
export const CODA = {
  slug: "coda",
  tools: {
    ADD_A_CATEGORY_FOR_PACK: "CODA_ADD_A_CATEGORY_FOR_PACK",
    ADD_A_MAKER_FOR_PACK: "CODA_ADD_A_MAKER_FOR_PACK",
    ADD_A_PERMISSION_FOR_PACK: "CODA_ADD_A_PERMISSION_FOR_PACK",
    ADD_CUSTOM_DOMAIN: "CODA_ADD_CUSTOM_DOMAIN",
    ADD_PERMISSION: "CODA_ADD_PERMISSION",
    BEGIN_CONTENT_EXPORT: "CODA_BEGIN_CONTENT_EXPORT",
    CONTENT_EXPORT_STATUS: "CODA_CONTENT_EXPORT_STATUS",
    COPY_DOC: "CODA_COPY_DOC",
    CREATE_A_NEW_PACK_RELEASE: "CODA_CREATE_A_NEW_PACK_RELEASE",
    CREATE_A_PAGE: "CODA_CREATE_A_PAGE",
    CREATE_DOC: "CODA_CREATE_DOC",
    CREATE_PACK: "CODA_CREATE_PACK",
    DELETES_A_CUSTOM_DOMAIN: "CODA_DELETES_A_CUSTOM_DOMAIN",
    DELETE_A_CATEGORY_FOR_PACK: "CODA_DELETE_A_CATEGORY_FOR_PACK",
    DELETE_A_MAKER_FOR_PACK: "CODA_DELETE_A_MAKER_FOR_PACK",
    DELETE_A_PAGE: "CODA_DELETE_A_PAGE",
    DELETE_A_PERMISSION_FOR_PACK: "CODA_DELETE_A_PERMISSION_FOR_PACK",
    DELETE_DOC: "CODA_DELETE_DOC",
    DELETE_MULTIPLE_ROWS: "CODA_DELETE_MULTIPLE_ROWS",
    DELETE_PACK: "CODA_DELETE_PACK",
    DELETE_PERMISSION: "CODA_DELETE_PERMISSION",
    DELETE_ROW: "CODA_DELETE_ROW",
    FETCH_GROUPED_LOGS_BY_PACK_ORG_ROOT_INGESTION_ID: "CODA_FETCH_GROUPED_LOGS_BY_PACK_ORG_ROOT_INGESTION_ID",
    FETCH_INGESTION_EXECUTIONS_FOR_PACK: "CODA_FETCH_INGESTION_EXECUTIONS_FOR_PACK",
    GETS_CUSTOM_DOC_DOMAINS_PROVIDERS: "CODA_GETS_CUSTOM_DOC_DOMAINS_PROVIDERS",
    GETS_THE_JSON_SCHEMA_FOR_PACK_CONFIGURATION: "CODA_GETS_THE_JSON_SCHEMA_FOR_PACK_CONFIGURATION",
    GET_ACL_SETTINGS: "CODA_GET_ACL_SETTINGS",
    GET_ANALYTICS_LAST_UPDATED_DAY: "CODA_GET_ANALYTICS_LAST_UPDATED_DAY",
    GET_A_COLUMN: "CODA_GET_A_COLUMN",
    GET_A_CONTROL: "CODA_GET_A_CONTROL",
    GET_A_FORMULA: "CODA_GET_A_FORMULA",
    GET_A_PAGE: "CODA_GET_A_PAGE",
    GET_A_ROW: "CODA_GET_A_ROW",
    GET_A_SINGLE_PACK: "CODA_GET_A_SINGLE_PACK",
    GET_A_TABLE: "CODA_GET_A_TABLE",
    GET_DETAILED_LISTING_INFORMATION_FOR_A_PACK: "CODA_GET_DETAILED_LISTING_INFORMATION_FOR_A_PACK",
    GET_DOC_ANALYTICS_SUMMARY: "CODA_GET_DOC_ANALYTICS_SUMMARY",
    GET_DOC_CATEGORIES: "CODA_GET_DOC_CATEGORIES",
    GET_INFO_ABOUT_A_DOC: "CODA_GET_INFO_ABOUT_A_DOC",
    GET_MUTATION_STATUS: "CODA_GET_MUTATION_STATUS",
    GET_PACK_ANALYTICS_SUMMARY: "CODA_GET_PACK_ANALYTICS_SUMMARY",
    GET_SHARING_METADATA: "CODA_GET_SHARING_METADATA",
    GET_THE_DIFFERENCE_BETWEEN_TWO_PACK_VERSIONS: "CODA_GET_THE_DIFFERENCE_BETWEEN_TWO_PACK_VERSIONS",
    GET_THE_NEXT_VALID_VERSION_FOR_A_PACK: "CODA_GET_THE_NEXT_VALID_VERSION_FOR_A_PACK",
    GET_THE_SOURCE_CODE_FOR_A_PACK_VERSION: "CODA_GET_THE_SOURCE_CODE_FOR_A_PACK_VERSION",
    GET_USER_INFO: "CODA_GET_USER_INFO",
    LIST_AVAILABLE_DOCS: "CODA_LIST_AVAILABLE_DOCS",
    LIST_CATEGORIES_FOR_PACK: "CODA_LIST_CATEGORIES_FOR_PACK",
    LIST_COLUMNS: "CODA_LIST_COLUMNS",
    LIST_CONTROLS: "CODA_LIST_CONTROLS",
    LIST_CUSTOM_DOC_DOMAINS: "CODA_LIST_CUSTOM_DOC_DOMAINS",
    LIST_DOC_ANALYTICS: "CODA_LIST_DOC_ANALYTICS",
    LIST_FEATURED_DOCS_FOR_A_PACK: "CODA_LIST_FEATURED_DOCS_FOR_A_PACK",
    LIST_FORMULAS: "CODA_LIST_FORMULAS",
    LIST_MAKERS_FOR_PACK: "CODA_LIST_MAKERS_FOR_PACK",
    LIST_PACKS: "CODA_LIST_PACKS",
    LIST_PACK_ANALYTICS: "CODA_LIST_PACK_ANALYTICS",
    LIST_PACK_FORMULA_ANALYTICS: "CODA_LIST_PACK_FORMULA_ANALYTICS",
    LIST_PAGES: "CODA_LIST_PAGES",
    LIST_PAGE_ANALYTICS: "CODA_LIST_PAGE_ANALYTICS",
    LIST_PERMISSIONS: "CODA_LIST_PERMISSIONS",
    LIST_PERMISSIONS_FOR_A_PACK: "CODA_LIST_PERMISSIONS_FOR_A_PACK",
    LIST_TABLES: "CODA_LIST_TABLES",
    LIST_TABLE_ROWS: "CODA_LIST_TABLE_ROWS",
    LIST_THE_PACK_LISTINGS_ACCESSIBLE_TO_A_USER: "CODA_LIST_THE_PACK_LISTINGS_ACCESSIBLE_TO_A_USER",
    LIST_THE_RELEASES_FOR_A_PACK: "CODA_LIST_THE_RELEASES_FOR_A_PACK",
    LIST_THE_VERSIONS_FOR_A_PACK: "CODA_LIST_THE_VERSIONS_FOR_A_PACK",
    LIST_WORKSPACE_ROLES: "CODA_LIST_WORKSPACE_ROLES",
    LIST_WORKSPACE_USERS: "CODA_LIST_WORKSPACE_USERS",
    PACK_ASSET_UPLOAD_COMPLETE: "CODA_PACK_ASSET_UPLOAD_COMPLETE",
    PACK_SOURCE_CODE_UPLOAD_COMPLETE: "CODA_PACK_SOURCE_CODE_UPLOAD_COMPLETE",
    PACK_VERSION_UPLOAD_COMPLETE: "CODA_PACK_VERSION_UPLOAD_COMPLETE",
    PATCH_THE_SYSTEM_CONNECTION_CREDENTIALS_OF_THE_PACK: "CODA_PATCH_THE_SYSTEM_CONNECTION_CREDENTIALS_OF_THE_PACK",
    PUBLISH_DOC: "CODA_PUBLISH_DOC",
    PUSH_A_BUTTON: "CODA_PUSH_A_BUTTON",
    REGISTER_PACK_VERSION: "CODA_REGISTER_PACK_VERSION",
    RESOLVE_BROWSER_LINK: "CODA_RESOLVE_BROWSER_LINK",
    RETRIEVE_THE_GROUPED_LOGS_OF_A_PACK: "CODA_RETRIEVE_THE_GROUPED_LOGS_OF_A_PACK",
    RETRIEVE_THE_INFORMATION_FOR_A_SPECIFIC_LOG: "CODA_RETRIEVE_THE_INFORMATION_FOR_A_SPECIFIC_LOG",
    RETRIEVE_THE_LOGS_OF_A_INGESTION: "CODA_RETRIEVE_THE_LOGS_OF_A_INGESTION",
    RETRIEVE_THE_LOGS_OF_A_PACK: "CODA_RETRIEVE_THE_LOGS_OF_A_PACK",
    RETRIEVE_THE_OAUTH_CONFIGURATION_OF_THE_PACK: "CODA_RETRIEVE_THE_OAUTH_CONFIGURATION_OF_THE_PACK",
    RETRIEVE_THE_SYSTEM_CONNECTION_METADATA_OF_THE_PACK: "CODA_RETRIEVE_THE_SYSTEM_CONNECTION_METADATA_OF_THE_PACK",
    SEARCH_DOCS: "CODA_SEARCH_DOCS",
    SEARCH_PRINCIPALS: "CODA_SEARCH_PRINCIPALS",
    SEARCH_ROW: "CODA_SEARCH_ROW",
    SET_THE_OAUTH_CONFIGURATIONS_OF_THE_PACK: "CODA_SET_THE_OAUTH_CONFIGURATIONS_OF_THE_PACK",
    SET_THE_SYSTEM_CONNECTION_CREDENTIALS_OF_THE_PACK: "CODA_SET_THE_SYSTEM_CONNECTION_CREDENTIALS_OF_THE_PACK",
    TRIGGER_AUTOMATION: "CODA_TRIGGER_AUTOMATION",
    UNPUBLISH_DOC: "CODA_UNPUBLISH_DOC",
    UPDATES_A_CUSTOM_DOMAIN: "CODA_UPDATES_A_CUSTOM_DOMAIN",
    UPDATES_USER_ROLE: "CODA_UPDATES_USER_ROLE",
    UPDATE_ACL_SETTINGS: "CODA_UPDATE_ACL_SETTINGS",
    UPDATE_AN_EXISTING_PACK_RELEASE: "CODA_UPDATE_AN_EXISTING_PACK_RELEASE",
    UPDATE_A_PAGE: "CODA_UPDATE_A_PAGE",
    UPDATE_DOC: "CODA_UPDATE_DOC",
    UPDATE_FEATURED_DOCS_FOR_A_PACK: "CODA_UPDATE_FEATURED_DOCS_FOR_A_PACK",
    UPDATE_PACK: "CODA_UPDATE_PACK",
    UPLOAD_A_PACK_ASSET: "CODA_UPLOAD_A_PACK_ASSET",
    UPLOAD_PACK_SOURCE_CODE: "CODA_UPLOAD_PACK_SOURCE_CODE",
    UPSERT_ROWS: "CODA_UPSERT_ROWS",
  },
  triggerTypes: {
    CODA_DOCUMENT_DELETED: "CODA_CODA_DOCUMENT_DELETED",
    NEW_CODA_DOCUMENT: "CODA_NEW_CODA_DOCUMENT",
    NEW_CODA_PAGE: "CODA_NEW_CODA_PAGE",
    SPECIFIC_CODA_PAGE_UPDATED: "CODA_SPECIFIC_CODA_PAGE_UPDATED",
  },
}
