// @composio/core/generated/composio.ts
/**
 * Map of Composio's COMPOSIO toolkit.
 */
export const COMPOSIO = {
  slug: "composio",
  tools: {
    CHECK_ACTIVE_CONNECTION: "COMPOSIO_CHECK_ACTIVE_CONNECTION",
    DOWNLOAD_S3_FILE: "COMPOSIO_DOWNLOAD_S3_FILE",
    ENABLE_TRIGGER: "COMPOSIO_ENABLE_TRIGGER",
    EXECUTE_TOOL: "COMPOSIO_EXECUTE_TOOL",
    GET_REQUIRED_PARAMETERS: "COMPOSIO_GET_REQUIRED_PARAMETERS",
    GET_RESPONSE_SCHEMA: "COMPOSIO_GET_RESPONSE_SCHEMA",
    INITIATE_CONNECTION: "COMPOSIO_INITIATE_CONNECTION",
    LIST_TOOLKITS: "COMPOSIO_LIST_TOOLKITS",
    LIST_TRIGGERS: "COMPOSIO_LIST_TRIGGERS",
    RETRIEVE_TOOLKITS: "COMPOSIO_RETRIEVE_TOOLKITS",
    RETRIEVE_TOOLS: "COMPOSIO_RETRIEVE_TOOLS",
    SEARCH_ACTIONS: "COMPOSIO_SEARCH_ACTIONS",
    SEARCH_DUCK_DUCK_GO_SEARCH: "COMPOSIO_SEARCH_DUCK_DUCK_GO_SEARCH",
    SEARCH_EVENT_SEARCH: "COMPOSIO_SEARCH_EVENT_SEARCH",
    SEARCH_EXA_SIMILARLINK: "COMPOSIO_SEARCH_EXA_SIMILARLINK",
    SEARCH_FINANCE_SEARCH: "COMPOSIO_SEARCH_FINANCE_SEARCH",
    SEARCH_GOOGLE_MAPS_SEARCH: "COMPOSIO_SEARCH_GOOGLE_MAPS_SEARCH",
    SEARCH_IMAGE_SEARCH: "COMPOSIO_SEARCH_IMAGE_SEARCH",
    SEARCH_NEWS_SEARCH: "COMPOSIO_SEARCH_NEWS_SEARCH",
    SEARCH_SCHOLAR_SEARCH: "COMPOSIO_SEARCH_SCHOLAR_SEARCH",
    SEARCH_SEARCH: "COMPOSIO_SEARCH_SEARCH",
    SEARCH_SHOPPING_SEARCH: "COMPOSIO_SEARCH_SHOPPING_SEARCH",
    SEARCH_TAVILY_SEARCH: "COMPOSIO_SEARCH_TAVILY_SEARCH",
    SEARCH_TRENDS_SEARCH: "COMPOSIO_SEARCH_TRENDS_SEARCH",
    WAIT_FOR_CONNECTION: "COMPOSIO_WAIT_FOR_CONNECTION",
  },
  triggerTypes: {},
}
