// @composio/core/generated/confluence.ts
/**
 * Map of Composio's CONFLUENCE toolkit.
 */
export const CONFLUENCE = {
  slug: "confluence",
  tools: {
    CHECK_SITE_ACCESS_FOR_A_LIST_OF_EMAILS: "CONFLUENCE_CHECK_SITE_ACCESS_FOR_A_LIST_OF_EMAILS",
    CONVERT_CONTENT_IDS_TO_CONTENT_TYPES: "CONFLUENCE_CONVERT_CONTENT_IDS_TO_CONTENT_TYPES",
    CREATE_BLOG_POST: "CONFLUENCE_CREATE_BLOG_POST",
    CREATE_BULK_USER_LOOKUP_USING_IDS: "CONFLUENCE_CREATE_BULK_USER_LOOKUP_USING_IDS",
    CREATE_CONTENT_PROPERTY_FOR_ATTACHMENT: "CONFLUENCE_CREATE_CONTENT_PROPERTY_FOR_ATTACHMENT",
    CREATE_CONTENT_PROPERTY_FOR_BLOG_POST: "CONFLUENCE_CREATE_CONTENT_PROPERTY_FOR_BLOG_POST",
    CREATE_CONTENT_PROPERTY_FOR_COMMENT: "CONFLUENCE_CREATE_CONTENT_PROPERTY_FOR_COMMENT",
    CREATE_CONTENT_PROPERTY_FOR_CUSTOM_CONTENT: "CONFLUENCE_CREATE_CONTENT_PROPERTY_FOR_CUSTOM_CONTENT",
    CREATE_CONTENT_PROPERTY_FOR_DATABASE: "CONFLUENCE_CREATE_CONTENT_PROPERTY_FOR_DATABASE",
    CREATE_CONTENT_PROPERTY_FOR_FOLDER: "CONFLUENCE_CREATE_CONTENT_PROPERTY_FOR_FOLDER",
    CREATE_CONTENT_PROPERTY_FOR_PAGE: "CONFLUENCE_CREATE_CONTENT_PROPERTY_FOR_PAGE",
    CREATE_CONTENT_PROPERTY_FOR_WHITEBOARD: "CONFLUENCE_CREATE_CONTENT_PROPERTY_FOR_WHITEBOARD",
    CREATE_CUSTOM_CONTENT: "CONFLUENCE_CREATE_CUSTOM_CONTENT",
    CREATE_DATABASE: "CONFLUENCE_CREATE_DATABASE",
    CREATE_FOLDER: "CONFLUENCE_CREATE_FOLDER",
    CREATE_FOOTER_COMMENT: "CONFLUENCE_CREATE_FOOTER_COMMENT",
    CREATE_INLINE_COMMENT: "CONFLUENCE_CREATE_INLINE_COMMENT",
    CREATE_PAGE: "CONFLUENCE_CREATE_PAGE",
    CREATE_SMART_LINK_CONTENT_PROPERTY: "CONFLUENCE_CREATE_SMART_LINK_CONTENT_PROPERTY",
    CREATE_SMART_LINK_IN_THE_CONTENT_TREE: "CONFLUENCE_CREATE_SMART_LINK_IN_THE_CONTENT_TREE",
    CREATE_SPACE: "CONFLUENCE_CREATE_SPACE",
    CREATE_SPACE_PROPERTY_IN_SPACE: "CONFLUENCE_CREATE_SPACE_PROPERTY_IN_SPACE",
    CREATE_WHITEBOARD: "CONFLUENCE_CREATE_WHITEBOARD",
    DELETE_ATTACHMENT: "CONFLUENCE_DELETE_ATTACHMENT",
    DELETE_BLOG_POST: "CONFLUENCE_DELETE_BLOG_POST",
    DELETE_CONTENT_PROPERTY_FOR_ATTACHMENT_BY_ID: "CONFLUENCE_DELETE_CONTENT_PROPERTY_FOR_ATTACHMENT_BY_ID",
    DELETE_CONTENT_PROPERTY_FOR_BLOGPOST_BY_ID: "CONFLUENCE_DELETE_CONTENT_PROPERTY_FOR_BLOGPOST_BY_ID",
    DELETE_CONTENT_PROPERTY_FOR_COMMENT_BY_ID: "CONFLUENCE_DELETE_CONTENT_PROPERTY_FOR_COMMENT_BY_ID",
    DELETE_CONTENT_PROPERTY_FOR_DATABASE_BY_ID: "CONFLUENCE_DELETE_CONTENT_PROPERTY_FOR_DATABASE_BY_ID",
    DELETE_CONTENT_PROPERTY_FOR_FOLDER_BY_ID: "CONFLUENCE_DELETE_CONTENT_PROPERTY_FOR_FOLDER_BY_ID",
    DELETE_CONTENT_PROPERTY_FOR_PAGE_BY_ID: "CONFLUENCE_DELETE_CONTENT_PROPERTY_FOR_PAGE_BY_ID",
    DELETE_CONTENT_PROPERTY_FOR_WHITEBOARD_BY_ID: "CONFLUENCE_DELETE_CONTENT_PROPERTY_FOR_WHITEBOARD_BY_ID",
    DELETE_CUSTOM_CONTENT: "CONFLUENCE_DELETE_CUSTOM_CONTENT",
    DELETE_CUSTOM_CONTENT_PROPERTY_BY_ID: "CONFLUENCE_DELETE_CUSTOM_CONTENT_PROPERTY_BY_ID",
    DELETE_DATABASE: "CONFLUENCE_DELETE_DATABASE",
    DELETE_FOLDER: "CONFLUENCE_DELETE_FOLDER",
    DELETE_FOOTER_COMMENT: "CONFLUENCE_DELETE_FOOTER_COMMENT",
    DELETE_INLINE_COMMENT: "CONFLUENCE_DELETE_INLINE_COMMENT",
    DELETE_PAGE: "CONFLUENCE_DELETE_PAGE",
    DELETE_SMART_LINK_IN_THE_CONTENT_TREE: "CONFLUENCE_DELETE_SMART_LINK_IN_THE_CONTENT_TREE",
    DELETE_SMART_LINK_PROPERTY_BY_ID: "CONFLUENCE_DELETE_SMART_LINK_PROPERTY_BY_ID",
    DELETE_SPACE_DEFAULT_CLASSIFICATION_LEVEL: "CONFLUENCE_DELETE_SPACE_DEFAULT_CLASSIFICATION_LEVEL",
    DELETE_SPACE_PROPERTY_BY_ID: "CONFLUENCE_DELETE_SPACE_PROPERTY_BY_ID",
    DELETE_WHITEBOARD: "CONFLUENCE_DELETE_WHITEBOARD",
    GET_ACCOUNT_IDS_OF_LIKES_FOR_BLOG_POST: "CONFLUENCE_GET_ACCOUNT_IDS_OF_LIKES_FOR_BLOG_POST",
    GET_ACCOUNT_IDS_OF_LIKES_FOR_FOOTER_COMMENT: "CONFLUENCE_GET_ACCOUNT_IDS_OF_LIKES_FOR_FOOTER_COMMENT",
    GET_ACCOUNT_IDS_OF_LIKES_FOR_INLINE_COMMENT: "CONFLUENCE_GET_ACCOUNT_IDS_OF_LIKES_FOR_INLINE_COMMENT",
    GET_ACCOUNT_IDS_OF_LIKES_FOR_PAGE: "CONFLUENCE_GET_ACCOUNT_IDS_OF_LIKES_FOR_PAGE",
    GET_ALL_ANCESTORS_OF_DATABASE: "CONFLUENCE_GET_ALL_ANCESTORS_OF_DATABASE",
    GET_ALL_ANCESTORS_OF_FOLDER: "CONFLUENCE_GET_ALL_ANCESTORS_OF_FOLDER",
    GET_ALL_ANCESTORS_OF_PAGE: "CONFLUENCE_GET_ALL_ANCESTORS_OF_PAGE",
    GET_ALL_ANCESTORS_OF_SMART_LINK_IN_CONTENT_TREE: "CONFLUENCE_GET_ALL_ANCESTORS_OF_SMART_LINK_IN_CONTENT_TREE",
    GET_ALL_ANCESTORS_OF_WHITEBOARD: "CONFLUENCE_GET_ALL_ANCESTORS_OF_WHITEBOARD",
    GET_ATTACHMENTS: "CONFLUENCE_GET_ATTACHMENTS",
    GET_ATTACHMENTS_FOR_BLOG_POST: "CONFLUENCE_GET_ATTACHMENTS_FOR_BLOG_POST",
    GET_ATTACHMENTS_FOR_CUSTOM_CONTENT: "CONFLUENCE_GET_ATTACHMENTS_FOR_CUSTOM_CONTENT",
    GET_ATTACHMENTS_FOR_LABEL: "CONFLUENCE_GET_ATTACHMENTS_FOR_LABEL",
    GET_ATTACHMENTS_FOR_PAGE: "CONFLUENCE_GET_ATTACHMENTS_FOR_PAGE",
    GET_ATTACHMENT_BY_ID: "CONFLUENCE_GET_ATTACHMENT_BY_ID",
    GET_ATTACHMENT_COMMENTS: "CONFLUENCE_GET_ATTACHMENT_COMMENTS",
    GET_ATTACHMENT_VERSIONS: "CONFLUENCE_GET_ATTACHMENT_VERSIONS",
    GET_AVAILABLE_SPACE_PERMISSIONS: "CONFLUENCE_GET_AVAILABLE_SPACE_PERMISSIONS",
    GET_AVAILABLE_SPACE_ROLES: "CONFLUENCE_GET_AVAILABLE_SPACE_ROLES",
    GET_BLOG_POSTS: "CONFLUENCE_GET_BLOG_POSTS",
    GET_BLOG_POSTS_FOR_LABEL: "CONFLUENCE_GET_BLOG_POSTS_FOR_LABEL",
    GET_BLOG_POSTS_IN_SPACE: "CONFLUENCE_GET_BLOG_POSTS_IN_SPACE",
    GET_BLOG_POST_BY_ID: "CONFLUENCE_GET_BLOG_POST_BY_ID",
    GET_BLOG_POST_CLASSIFICATION_LEVEL: "CONFLUENCE_GET_BLOG_POST_CLASSIFICATION_LEVEL",
    GET_BLOG_POST_VERSIONS: "CONFLUENCE_GET_BLOG_POST_VERSIONS",
    GET_CHILDREN_FOOTER_COMMENTS: "CONFLUENCE_GET_CHILDREN_FOOTER_COMMENTS",
    GET_CHILDREN_INLINE_COMMENTS: "CONFLUENCE_GET_CHILDREN_INLINE_COMMENTS",
    GET_CHILD_CUSTOM_CONTENT: "CONFLUENCE_GET_CHILD_CUSTOM_CONTENT",
    GET_CHILD_PAGES: "CONFLUENCE_GET_CHILD_PAGES",
    GET_CONTENT_PROPERTIES_FOR_ATTACHMENT: "CONFLUENCE_GET_CONTENT_PROPERTIES_FOR_ATTACHMENT",
    GET_CONTENT_PROPERTIES_FOR_BLOG_POST: "CONFLUENCE_GET_CONTENT_PROPERTIES_FOR_BLOG_POST",
    GET_CONTENT_PROPERTIES_FOR_COMMENT: "CONFLUENCE_GET_CONTENT_PROPERTIES_FOR_COMMENT",
    GET_CONTENT_PROPERTIES_FOR_CUSTOM_CONTENT: "CONFLUENCE_GET_CONTENT_PROPERTIES_FOR_CUSTOM_CONTENT",
    GET_CONTENT_PROPERTIES_FOR_DATABASE: "CONFLUENCE_GET_CONTENT_PROPERTIES_FOR_DATABASE",
    GET_CONTENT_PROPERTIES_FOR_FOLDER: "CONFLUENCE_GET_CONTENT_PROPERTIES_FOR_FOLDER",
    GET_CONTENT_PROPERTIES_FOR_PAGE: "CONFLUENCE_GET_CONTENT_PROPERTIES_FOR_PAGE",
    GET_CONTENT_PROPERTIES_FOR_WHITEBOARD: "CONFLUENCE_GET_CONTENT_PROPERTIES_FOR_WHITEBOARD",
    GET_CONTENT_PROPERTY_FOR_ATTACHMENT_BY_ID: "CONFLUENCE_GET_CONTENT_PROPERTY_FOR_ATTACHMENT_BY_ID",
    GET_CONTENT_PROPERTY_FOR_BLOG_POST_BY_ID: "CONFLUENCE_GET_CONTENT_PROPERTY_FOR_BLOG_POST_BY_ID",
    GET_CONTENT_PROPERTY_FOR_COMMENT_BY_ID: "CONFLUENCE_GET_CONTENT_PROPERTY_FOR_COMMENT_BY_ID",
    GET_CONTENT_PROPERTY_FOR_CUSTOM_CONTENT_BY_ID: "CONFLUENCE_GET_CONTENT_PROPERTY_FOR_CUSTOM_CONTENT_BY_ID",
    GET_CONTENT_PROPERTY_FOR_DATABASE_BY_ID: "CONFLUENCE_GET_CONTENT_PROPERTY_FOR_DATABASE_BY_ID",
    GET_CONTENT_PROPERTY_FOR_FOLDER_BY_ID: "CONFLUENCE_GET_CONTENT_PROPERTY_FOR_FOLDER_BY_ID",
    GET_CONTENT_PROPERTY_FOR_PAGE_BY_ID: "CONFLUENCE_GET_CONTENT_PROPERTY_FOR_PAGE_BY_ID",
    GET_CONTENT_PROPERTY_FOR_WHITEBOARD_BY_ID: "CONFLUENCE_GET_CONTENT_PROPERTY_FOR_WHITEBOARD_BY_ID",
    GET_CUSTOM_CONTENT_BY_ID: "CONFLUENCE_GET_CUSTOM_CONTENT_BY_ID",
    GET_CUSTOM_CONTENT_BY_TYPE: "CONFLUENCE_GET_CUSTOM_CONTENT_BY_TYPE",
    GET_CUSTOM_CONTENT_BY_TYPE_IN_BLOG_POST: "CONFLUENCE_GET_CUSTOM_CONTENT_BY_TYPE_IN_BLOG_POST",
    GET_CUSTOM_CONTENT_BY_TYPE_IN_PAGE: "CONFLUENCE_GET_CUSTOM_CONTENT_BY_TYPE_IN_PAGE",
    GET_CUSTOM_CONTENT_BY_TYPE_IN_SPACE: "CONFLUENCE_GET_CUSTOM_CONTENT_BY_TYPE_IN_SPACE",
    GET_CUSTOM_CONTENT_COMMENTS: "CONFLUENCE_GET_CUSTOM_CONTENT_COMMENTS",
    GET_CUSTOM_CONTENT_VERSIONS: "CONFLUENCE_GET_CUSTOM_CONTENT_VERSIONS",
    GET_DATABASE_BY_ID: "CONFLUENCE_GET_DATABASE_BY_ID",
    GET_DATABASE_CLASSIFICATION_LEVEL: "CONFLUENCE_GET_DATABASE_CLASSIFICATION_LEVEL",
    GET_DATA_POLICY_METADATA_FOR_THE_WORKSPACE: "CONFLUENCE_GET_DATA_POLICY_METADATA_FOR_THE_WORKSPACE",
    GET_FOLDER_BY_ID: "CONFLUENCE_GET_FOLDER_BY_ID",
    GET_FOOTER_COMMENTS: "CONFLUENCE_GET_FOOTER_COMMENTS",
    GET_FOOTER_COMMENTS_FOR_BLOG_POST: "CONFLUENCE_GET_FOOTER_COMMENTS_FOR_BLOG_POST",
    GET_FOOTER_COMMENTS_FOR_PAGE: "CONFLUENCE_GET_FOOTER_COMMENTS_FOR_PAGE",
    GET_FOOTER_COMMENT_BY_ID: "CONFLUENCE_GET_FOOTER_COMMENT_BY_ID",
    GET_FOOTER_COMMENT_VERSIONS: "CONFLUENCE_GET_FOOTER_COMMENT_VERSIONS",
    GET_INLINE_COMMENTS: "CONFLUENCE_GET_INLINE_COMMENTS",
    GET_INLINE_COMMENTS_FOR_BLOG_POST: "CONFLUENCE_GET_INLINE_COMMENTS_FOR_BLOG_POST",
    GET_INLINE_COMMENTS_FOR_PAGE: "CONFLUENCE_GET_INLINE_COMMENTS_FOR_PAGE",
    GET_INLINE_COMMENT_BY_ID: "CONFLUENCE_GET_INLINE_COMMENT_BY_ID",
    GET_INLINE_COMMENT_VERSIONS: "CONFLUENCE_GET_INLINE_COMMENT_VERSIONS",
    GET_LABELS: "CONFLUENCE_GET_LABELS",
    GET_LABELS_FOR_ATTACHMENT: "CONFLUENCE_GET_LABELS_FOR_ATTACHMENT",
    GET_LABELS_FOR_BLOG_POST: "CONFLUENCE_GET_LABELS_FOR_BLOG_POST",
    GET_LABELS_FOR_CUSTOM_CONTENT: "CONFLUENCE_GET_LABELS_FOR_CUSTOM_CONTENT",
    GET_LABELS_FOR_PAGE: "CONFLUENCE_GET_LABELS_FOR_PAGE",
    GET_LABELS_FOR_SPACE: "CONFLUENCE_GET_LABELS_FOR_SPACE",
    GET_LABELS_FOR_SPACE_CONTENT: "CONFLUENCE_GET_LABELS_FOR_SPACE_CONTENT",
    GET_LIKE_COUNT_FOR_BLOG_POST: "CONFLUENCE_GET_LIKE_COUNT_FOR_BLOG_POST",
    GET_LIKE_COUNT_FOR_FOOTER_COMMENT: "CONFLUENCE_GET_LIKE_COUNT_FOR_FOOTER_COMMENT",
    GET_LIKE_COUNT_FOR_INLINE_COMMENT: "CONFLUENCE_GET_LIKE_COUNT_FOR_INLINE_COMMENT",
    GET_LIKE_COUNT_FOR_PAGE: "CONFLUENCE_GET_LIKE_COUNT_FOR_PAGE",
    GET_LIST_OF_CLASSIFICATION_LEVELS: "CONFLUENCE_GET_LIST_OF_CLASSIFICATION_LEVELS",
    GET_PAGES: "CONFLUENCE_GET_PAGES",
    GET_PAGES_FOR_LABEL: "CONFLUENCE_GET_PAGES_FOR_LABEL",
    GET_PAGES_IN_SPACE: "CONFLUENCE_GET_PAGES_IN_SPACE",
    GET_PAGE_BY_ID: "CONFLUENCE_GET_PAGE_BY_ID",
    GET_PAGE_CLASSIFICATION_LEVEL: "CONFLUENCE_GET_PAGE_CLASSIFICATION_LEVEL",
    GET_PAGE_VERSIONS: "CONFLUENCE_GET_PAGE_VERSIONS",
    GET_PERMITTED_OPERATIONS_FOR_ATTACHMENT: "CONFLUENCE_GET_PERMITTED_OPERATIONS_FOR_ATTACHMENT",
    GET_PERMITTED_OPERATIONS_FOR_A_DATABASE: "CONFLUENCE_GET_PERMITTED_OPERATIONS_FOR_A_DATABASE",
    GET_PERMITTED_OPERATIONS_FOR_A_FOLDER: "CONFLUENCE_GET_PERMITTED_OPERATIONS_FOR_A_FOLDER",
    GET_PERMITTED_OPERATIONS_FOR_A_WHITEBOARD: "CONFLUENCE_GET_PERMITTED_OPERATIONS_FOR_A_WHITEBOARD",
    GET_PERMITTED_OPERATIONS_FOR_BLOG_POST: "CONFLUENCE_GET_PERMITTED_OPERATIONS_FOR_BLOG_POST",
    GET_PERMITTED_OPERATIONS_FOR_CUSTOM_CONTENT: "CONFLUENCE_GET_PERMITTED_OPERATIONS_FOR_CUSTOM_CONTENT",
    GET_PERMITTED_OPERATIONS_FOR_FOOTER_COMMENT: "CONFLUENCE_GET_PERMITTED_OPERATIONS_FOR_FOOTER_COMMENT",
    GET_PERMITTED_OPERATIONS_FOR_INLINE_COMMENT: "CONFLUENCE_GET_PERMITTED_OPERATIONS_FOR_INLINE_COMMENT",
    GET_PERMITTED_OPERATIONS_FOR_PAGE: "CONFLUENCE_GET_PERMITTED_OPERATIONS_FOR_PAGE",
    GET_PERMITTED_OPERATIONS_FOR_SPACE: "CONFLUENCE_GET_PERMITTED_OPERATIONS_FOR_SPACE",
    GET_SMART_LINK_CONTENT_PROPS: "CONFLUENCE_GET_SMART_LINK_CONTENT_PROPS",
    GET_SMART_LINK_IN_THE_CONTENT_TREE_BY_ID: "CONFLUENCE_GET_SMART_LINK_IN_THE_CONTENT_TREE_BY_ID",
    GET_SMART_LINK_OPERATIONS: "CONFLUENCE_GET_SMART_LINK_OPERATIONS",
    GET_SMART_LINK_PROPERTY: "CONFLUENCE_GET_SMART_LINK_PROPERTY",
    GET_SPACES: "CONFLUENCE_GET_SPACES",
    GET_SPACES_WITH_DATA_POLICIES: "CONFLUENCE_GET_SPACES_WITH_DATA_POLICIES",
    GET_SPACE_BY_ID: "CONFLUENCE_GET_SPACE_BY_ID",
    GET_SPACE_DEFAULT_CLASSIFICATION_LEVEL: "CONFLUENCE_GET_SPACE_DEFAULT_CLASSIFICATION_LEVEL",
    GET_SPACE_PERMISSIONS_ASSIGNMENTS: "CONFLUENCE_GET_SPACE_PERMISSIONS_ASSIGNMENTS",
    GET_SPACE_PROPERTIES_IN_SPACE: "CONFLUENCE_GET_SPACE_PROPERTIES_IN_SPACE",
    GET_SPACE_PROPERTY_BY_ID: "CONFLUENCE_GET_SPACE_PROPERTY_BY_ID",
    GET_SPACE_ROLE_ASSIGNMENTS: "CONFLUENCE_GET_SPACE_ROLE_ASSIGNMENTS",
    GET_SPACE_ROLE_BY_ID: "CONFLUENCE_GET_SPACE_ROLE_BY_ID",
    GET_TASKS: "CONFLUENCE_GET_TASKS",
    GET_TASK_BY_ID: "CONFLUENCE_GET_TASK_BY_ID",
    GET_VERSION_DETAILS_FOR_ATTACHMENT_VERSION: "CONFLUENCE_GET_VERSION_DETAILS_FOR_ATTACHMENT_VERSION",
    GET_VERSION_DETAILS_FOR_BLOG_POST_VERSION: "CONFLUENCE_GET_VERSION_DETAILS_FOR_BLOG_POST_VERSION",
    GET_VERSION_DETAILS_FOR_CUSTOM_CONTENT_VERSION: "CONFLUENCE_GET_VERSION_DETAILS_FOR_CUSTOM_CONTENT_VERSION",
    GET_VERSION_DETAILS_FOR_FOOTER_COMMENT_VERSION: "CONFLUENCE_GET_VERSION_DETAILS_FOR_FOOTER_COMMENT_VERSION",
    GET_VERSION_DETAILS_FOR_INLINE_COMMENT_VERSION: "CONFLUENCE_GET_VERSION_DETAILS_FOR_INLINE_COMMENT_VERSION",
    GET_VERSION_DETAILS_FOR_PAGE_VERSION: "CONFLUENCE_GET_VERSION_DETAILS_FOR_PAGE_VERSION",
    GET_WHITEBOARD_BY_ID: "CONFLUENCE_GET_WHITEBOARD_BY_ID",
    GET_WHITEBOARD_CLASSIFICATION_LEVEL: "CONFLUENCE_GET_WHITEBOARD_CLASSIFICATION_LEVEL",
    INVITE_A_LIST_OF_EMAILS_TO_THE_SITE: "CONFLUENCE_INVITE_A_LIST_OF_EMAILS_TO_THE_SITE",
    RESET_BLOG_POST_CLASSIFICATION_LEVEL: "CONFLUENCE_RESET_BLOG_POST_CLASSIFICATION_LEVEL",
    RESET_DATABASE_CLASSIFICATION_LEVEL: "CONFLUENCE_RESET_DATABASE_CLASSIFICATION_LEVEL",
    RESET_PAGE_CLASSIFICATION_LEVEL: "CONFLUENCE_RESET_PAGE_CLASSIFICATION_LEVEL",
    RESET_WHITEBOARD_CLASSIFICATION_LEVEL: "CONFLUENCE_RESET_WHITEBOARD_CLASSIFICATION_LEVEL",
    SEARCH_CONTENT: "CONFLUENCE_SEARCH_CONTENT",
    SEARCH_USERS: "CONFLUENCE_SEARCH_USERS",
    SET_SPACE_ROLE_ASSIGNMENTS: "CONFLUENCE_SET_SPACE_ROLE_ASSIGNMENTS",
    UPDATE_BLOG_POST: "CONFLUENCE_UPDATE_BLOG_POST",
    UPDATE_BLOG_POST_CLASSIFICATION_LEVEL: "CONFLUENCE_UPDATE_BLOG_POST_CLASSIFICATION_LEVEL",
    UPDATE_CONTENT_PROPERTY_FOR_ATTACHMENT_BY_ID: "CONFLUENCE_UPDATE_CONTENT_PROPERTY_FOR_ATTACHMENT_BY_ID",
    UPDATE_CONTENT_PROPERTY_FOR_BLOG_POST_BY_ID: "CONFLUENCE_UPDATE_CONTENT_PROPERTY_FOR_BLOG_POST_BY_ID",
    UPDATE_CONTENT_PROPERTY_FOR_COMMENT_BY_ID: "CONFLUENCE_UPDATE_CONTENT_PROPERTY_FOR_COMMENT_BY_ID",
    UPDATE_CONTENT_PROPERTY_FOR_DATABASE_BY_ID: "CONFLUENCE_UPDATE_CONTENT_PROPERTY_FOR_DATABASE_BY_ID",
    UPDATE_CONTENT_PROPERTY_FOR_FOLDER_BY_ID: "CONFLUENCE_UPDATE_CONTENT_PROPERTY_FOR_FOLDER_BY_ID",
    UPDATE_CONTENT_PROPERTY_FOR_PAGE_BY_ID: "CONFLUENCE_UPDATE_CONTENT_PROPERTY_FOR_PAGE_BY_ID",
    UPDATE_CONTENT_PROPERTY_FOR_WHITEBOARD_BY_ID: "CONFLUENCE_UPDATE_CONTENT_PROPERTY_FOR_WHITEBOARD_BY_ID",
    UPDATE_CUSTOM_CONTENT: "CONFLUENCE_UPDATE_CUSTOM_CONTENT",
    UPDATE_CUSTOM_CONTENT_PROP_BY_ID: "CONFLUENCE_UPDATE_CUSTOM_CONTENT_PROP_BY_ID",
    UPDATE_DATABASE_CLASSIFICATION_LEVEL: "CONFLUENCE_UPDATE_DATABASE_CLASSIFICATION_LEVEL",
    UPDATE_FOOTER_COMMENT: "CONFLUENCE_UPDATE_FOOTER_COMMENT",
    UPDATE_INLINE_COMMENT: "CONFLUENCE_UPDATE_INLINE_COMMENT",
    UPDATE_PAGE: "CONFLUENCE_UPDATE_PAGE",
    UPDATE_PAGE_CLASSIFICATION_LEVEL: "CONFLUENCE_UPDATE_PAGE_CLASSIFICATION_LEVEL",
    UPDATE_SMART_LINK_PROPERTY: "CONFLUENCE_UPDATE_SMART_LINK_PROPERTY",
    UPDATE_SPACE_DEFAULT_CLASSIFICATION_LEVEL: "CONFLUENCE_UPDATE_SPACE_DEFAULT_CLASSIFICATION_LEVEL",
    UPDATE_SPACE_PROPERTY_BY_ID: "CONFLUENCE_UPDATE_SPACE_PROPERTY_BY_ID",
    UPDATE_TASK: "CONFLUENCE_UPDATE_TASK",
    UPDATE_WHITEBOARD_CLASSIFICATION_LEVEL: "CONFLUENCE_UPDATE_WHITEBOARD_CLASSIFICATION_LEVEL",
  },
  triggerTypes: {},
}
