// @composio/core/generated/crustdata.ts
/**
 * Map of Composio's CRUSTDATA toolkit.
 */
export const CRUSTDATA = {
  slug: "crustdata",
  tools: {
    ENRICH_PERSON_SCREENER: "CRUSTDATA_ENRICH_PERSON_SCREENER",
    FETCH_HEADCOUNT_BY_FACET_TIMESERIES: "CRUSTDATA_FETCH_HEADCOUNT_BY_FACET_TIMESERIES",
    FETCH_INVESTOR_PORTFOLIO_DATA: "CRUSTDATA_FETCH_INVESTOR_PORTFOLIO_DATA",
    FILTER_DECISION_MAKERS_DATA: "CRUSTDATA_FILTER_DECISION_MAKERS_DATA",
    POST_FUNDING_MILESTONE_TIME_SERIES_DATA: "CRUSTDATA_POST_FUNDING_MILESTONE_TIME_SERIES_DATA",
    POST_HEADCOUNT_TIMESERIES_DATA: "CRUSTDATA_POST_HEADCOUNT_TIMESERIES_DATA",
    POST_JOB_LISTINGS_TABLE_DATA: "CRUSTDATA_POST_JOB_LISTINGS_TABLE_DATA",
    POST_WEB_TRAFFIC_DATA: "CRUSTDATA_POST_WEB_TRAFFIC_DATA",
    RETRIEVE_LINKED_IN_POSTS: "CRUSTDATA_RETRIEVE_LINKED_IN_POSTS",
    SCREENER_COMPANY_INFORMATION: "CRUSTDATA_SCREENER_COMPANY_INFORMATION",
    SCREEN_METRICS_AND_FILTER_CONDITIONS: "CRUSTDATA_SCREEN_METRICS_AND_FILTER_CONDITIONS",
    SEARCH_COMPANIES_WITH_FILTERS: "CRUSTDATA_SEARCH_COMPANIES_WITH_FILTERS",
    SEARCH_FOR_JOB_ID_IN_SCREENER: "CRUSTDATA_SEARCH_FOR_JOB_ID_IN_SCREENER",
    SEARCH_LINKED_IN_POSTS_BY_KEYWORD: "CRUSTDATA_SEARCH_LINKED_IN_POSTS_BY_KEYWORD",
  },
  triggerTypes: {},
}
