// @composio/core/generated/d2lbrightspace.ts
/**
 * Map of Composio's D2LBRIGHTSPACE toolkit.
 */
export const D2LBRIGHTSPACE = {
  slug: "d2lbrightspace",
  tools: {
    COPY_ROLE: "D2LBRIGHTSPACE_COPY_ROLE",
    CREATE_A_COURSE_OFFERING: "D2LBRIGHTSPACE_CREATE_A_COURSE_OFFERING",
    CREATE_A_COURSE_TEMPLATE: "D2LBRIGHTSPACE_CREATE_A_COURSE_TEMPLATE",
    CREATE_GRADE_OBJECT: "D2LBRIGHTSPACE_CREATE_GRADE_OBJECT",
    CREATE_QUIZ: "D2LBRIGHTSPACE_CREATE_QUIZ",
    CREATE_QUIZ_CATEGORY: "D2LBRIGHTSPACE_CREATE_QUIZ_CATEGORY",
    CREATE_USER: "D2LBRIGHTSPACE_CREATE_USER",
    DELETE_A_COURSE_TEMPLATE: "D2LBRIGHTSPACE_DELETE_A_COURSE_TEMPLATE",
    DELETE_COURSE: "D2LBRIGHTSPACE_DELETE_COURSE",
    DELETE_GRADE_OBJECT: "D2LBRIGHTSPACE_DELETE_GRADE_OBJECT",
    DELETE_QUIZ: "D2LBRIGHTSPACE_DELETE_QUIZ",
    DELETE_QUIZ_CATEGORY: "D2LBRIGHTSPACE_DELETE_QUIZ_CATEGORY",
    DELETE_USER: "D2LBRIGHTSPACE_DELETE_USER",
    DELETE_USER_DEMOGRAPHICS: "D2LBRIGHTSPACE_DELETE_USER_DEMOGRAPHICS",
    GET_A_COURSE_OFFERING: "D2LBRIGHTSPACE_GET_A_COURSE_OFFERING",
    GET_A_COURSE_TEMPLATE: "D2LBRIGHTSPACE_GET_A_COURSE_TEMPLATE",
    GET_COURSE_SCHEMA: "D2LBRIGHTSPACE_GET_COURSE_SCHEMA",
    GET_COURSE_TEMPLATE_SCHEMA: "D2LBRIGHTSPACE_GET_COURSE_TEMPLATE_SCHEMA",
    GET_CURRENT_USER: "D2LBRIGHTSPACE_GET_CURRENT_USER",
    GET_ENROLLED_ROLES: "D2LBRIGHTSPACE_GET_ENROLLED_ROLES",
    GET_GRADE_ACCESS: "D2LBRIGHTSPACE_GET_GRADE_ACCESS",
    GET_GRADE_OBJECT: "D2LBRIGHTSPACE_GET_GRADE_OBJECT",
    GET_GRADE_OBJECTS: "D2LBRIGHTSPACE_GET_GRADE_OBJECTS",
    GET_GRADE_SETUP: "D2LBRIGHTSPACE_GET_GRADE_SETUP",
    GET_GRADE_STATISTICS: "D2LBRIGHTSPACE_GET_GRADE_STATISTICS",
    GET_ORG_UNIT_DEMOGRAPHICS: "D2LBRIGHTSPACE_GET_ORG_UNIT_DEMOGRAPHICS",
    GET_QUIZ: "D2LBRIGHTSPACE_GET_QUIZ",
    GET_QUIZZES: "D2LBRIGHTSPACE_GET_QUIZZES",
    GET_QUIZ_ACCESS: "D2LBRIGHTSPACE_GET_QUIZ_ACCESS",
    GET_QUIZ_ATTEMPT: "D2LBRIGHTSPACE_GET_QUIZ_ATTEMPT",
    GET_QUIZ_ATTEMPTS: "D2LBRIGHTSPACE_GET_QUIZ_ATTEMPTS",
    GET_QUIZ_CATEGORIES: "D2LBRIGHTSPACE_GET_QUIZ_CATEGORIES",
    GET_QUIZ_CATEGORY: "D2LBRIGHTSPACE_GET_QUIZ_CATEGORY",
    GET_QUIZ_QUESTIONS: "D2LBRIGHTSPACE_GET_QUIZ_QUESTIONS",
    GET_ROLES: "D2LBRIGHTSPACE_GET_ROLES",
    GET_ROLE_BY_ID: "D2LBRIGHTSPACE_GET_ROLE_BY_ID",
    GET_USERS: "D2LBRIGHTSPACE_GET_USERS",
    GET_USER_BY_ID: "D2LBRIGHTSPACE_GET_USER_BY_ID",
    UPDATE_A_COURSE_OFFERING: "D2LBRIGHTSPACE_UPDATE_A_COURSE_OFFERING",
    UPDATE_A_COURSE_TEMPLATE: "D2LBRIGHTSPACE_UPDATE_A_COURSE_TEMPLATE",
    UPDATE_GRADE_OBJECT: "D2LBRIGHTSPACE_UPDATE_GRADE_OBJECT",
    UPDATE_GRADE_SETUP: "D2LBRIGHTSPACE_UPDATE_GRADE_SETUP",
    UPDATE_QUIZ: "D2LBRIGHTSPACE_UPDATE_QUIZ",
    UPDATE_QUIZ_CATEGORY: "D2LBRIGHTSPACE_UPDATE_QUIZ_CATEGORY",
    UPDATE_USER: "D2LBRIGHTSPACE_UPDATE_USER",
  },
  triggerTypes: {},
}
