// @composio/core/generated/datadog.ts
/**
 * Map of Composio's DATADOG toolkit.
 */
export const DATADOG = {
  slug: "datadog",
  tools: {
    CREATE_DASHBOARD: "DATADOG_CREATE_DASHBOARD",
    CREATE_DOWNTIME: "DATADOG_CREATE_DOWNTIME",
    CREATE_EVENT: "DATADOG_CREATE_EVENT",
    CREATE_MONITOR: "DATADOG_CREATE_MONITOR",
    CREATE_SLO: "DATADOG_CREATE_SLO",
    CREATE_SYNTHETIC_API_TEST: "DATADOG_CREATE_SYNTHETIC_API_TEST",
    CREATE_WEBHOOK: "DATADOG_CREATE_WEBHOOK",
    DELETE_DASHBOARD: "DATADOG_DELETE_DASHBOARD",
    DELETE_MONITOR: "DATADOG_DELETE_MONITOR",
    GET_DASHBOARD: "DATADOG_GET_DASHBOARD",
    GET_MONITOR: "DATADOG_GET_MONITOR",
    GET_SYNTHETICS_LOCATIONS: "DATADOG_GET_SYNTHETICS_LOCATIONS",
    GET_TAGS: "DATADOG_GET_TAGS",
    GET_USAGE_SUMMARY: "DATADOG_GET_USAGE_SUMMARY",
    LIST_ALL_TAGS: "DATADOG_LIST_ALL_TAGS",
    LIST_API_KEYS: "DATADOG_LIST_API_KEYS",
    LIST_APM_SERVICES: "DATADOG_LIST_APM_SERVICES",
    LIST_AWS_INTEGRATION: "DATADOG_LIST_AWS_INTEGRATION",
    LIST_DASHBOARDS: "DATADOG_LIST_DASHBOARDS",
    LIST_EVENTS: "DATADOG_LIST_EVENTS",
    LIST_HOSTS: "DATADOG_LIST_HOSTS",
    LIST_INCIDENTS: "DATADOG_LIST_INCIDENTS",
    LIST_LOG_INDEXES: "DATADOG_LIST_LOG_INDEXES",
    LIST_MONITORS: "DATADOG_LIST_MONITORS",
    LIST_ROLES: "DATADOG_LIST_ROLES",
    LIST_SERVICE_CHECKS: "DATADOG_LIST_SERVICE_CHECKS",
    LIST_SL_OS: "DATADOG_LIST_SL_OS",
    LIST_SYNTHETICS: "DATADOG_LIST_SYNTHETICS",
    LIST_USERS: "DATADOG_LIST_USERS",
    LIST_WEBHOOKS: "DATADOG_LIST_WEBHOOKS",
    MUTE_MONITOR: "DATADOG_MUTE_MONITOR",
    QUERY_METRICS: "DATADOG_QUERY_METRICS",
    SEARCH_LOGS: "DATADOG_SEARCH_LOGS",
    SUBMIT_METRICS: "DATADOG_SUBMIT_METRICS",
    UNMUTE_MONITOR: "DATADOG_UNMUTE_MONITOR",
    UPDATE_DASHBOARD: "DATADOG_UPDATE_DASHBOARD",
    UPDATE_HOST_TAGS: "DATADOG_UPDATE_HOST_TAGS",
    UPDATE_MONITOR: "DATADOG_UPDATE_MONITOR",
  },
  triggerTypes: {},
}
