// @composio/core/generated/dialpad.ts
/**
 * Map of Composio's DIALPAD toolkit.
 */
export const DIALPAD = {
  slug: "dialpad",
  tools: {
    ACCESS_CONTROL_POLICIES_LISTING: "DIALPAD_ACCESS_CONTROL_POLICIES_LISTING",
    ADD_BLOCKED_PHONE_NUMBERS: "DIALPAD_ADD_BLOCKED_PHONE_NUMBERS",
    ADD_DEPARTMENT_OPERATOR_BY_ID: "DIALPAD_ADD_DEPARTMENT_OPERATOR_BY_ID",
    ADD_MEMBER_TO_CHANNEL: "DIALPAD_ADD_MEMBER_TO_CHANNEL",
    ADD_MEMBER_TO_COACHING_TEAM: "DIALPAD_ADD_MEMBER_TO_COACHING_TEAM",
    ADD_OPERATOR_TO_CALL_CENTER: "DIALPAD_ADD_OPERATOR_TO_CALL_CENTER",
    ADD_OPERATOR_TO_OFFICE: "DIALPAD_ADD_OPERATOR_TO_OFFICE",
    ADD_PARTICIPANT_TO_CALL: "DIALPAD_ADD_PARTICIPANT_TO_CALL",
    ASSIGN_FAX_LINE_TO_TARGET: "DIALPAD_ASSIGN_FAX_LINE_TO_TARGET",
    ASSIGN_NUMBER_TO_CALL_ROUTER: "DIALPAD_ASSIGN_NUMBER_TO_CALL_ROUTER",
    ASSIGN_PHONE_NUMBER_TO_OFFICE: "DIALPAD_ASSIGN_PHONE_NUMBER_TO_OFFICE",
    ASSIGN_PHONE_NUMBER_TO_ROOM: "DIALPAD_ASSIGN_PHONE_NUMBER_TO_ROOM",
    ASSIGN_PHONE_NUMBER_TO_TARGET: "DIALPAD_ASSIGN_PHONE_NUMBER_TO_TARGET",
    ASSIGN_PHONE_NUMBER_TO_USER: "DIALPAD_ASSIGN_PHONE_NUMBER_TO_USER",
    ASSIGN_POLICY_TO_USER_BY_ID: "DIALPAD_ASSIGN_POLICY_TO_USER_BY_ID",
    ATTACH_LABELS_TO_CALL: "DIALPAD_ATTACH_LABELS_TO_CALL",
    CONFIGURE_CALL_CENTER_SETTINGS: "DIALPAD_CONFIGURE_CALL_CENTER_SETTINGS",
    CREATE_ACCESS_CONTROL_POLICY: "DIALPAD_CREATE_ACCESS_CONTROL_POLICY",
    CREATE_AGENT_STATUS_SUBSCRIPTION: "DIALPAD_CREATE_AGENT_STATUS_SUBSCRIPTION",
    CREATE_CALLBACK: "DIALPAD_CREATE_CALLBACK",
    CREATE_CALL_REVIEW_SHARE_LINK: "DIALPAD_CREATE_CALL_REVIEW_SHARE_LINK",
    CREATE_CALL_ROUTER_CONFIGURATION: "DIALPAD_CREATE_CALL_ROUTER_CONFIGURATION",
    CREATE_CHANNEL_ENDPOINT: "DIALPAD_CREATE_CHANNEL_ENDPOINT",
    CREATE_CUSTOM_IVR_WITH_AUDIO_FILE: "DIALPAD_CREATE_CUSTOM_IVR_WITH_AUDIO_FILE",
    CREATE_NEW_CONTACT_ENTRY: "DIALPAD_CREATE_NEW_CONTACT_ENTRY",
    CREATE_NEW_DEPARTMENT_RECORD: "DIALPAD_CREATE_NEW_DEPARTMENT_RECORD",
    CREATE_PIN_FOR_INTERNATIONAL_ROOM_CALLS: "DIALPAD_CREATE_PIN_FOR_INTERNATIONAL_ROOM_CALLS",
    CREATE_ROOM_IN_OFFICE_VIA_POST: "DIALPAD_CREATE_ROOM_IN_OFFICE_VIA_POST",
    CREATE_SCHEDULE_REPORTS_ENDPOINT: "DIALPAD_CREATE_SCHEDULE_REPORTS_ENDPOINT",
    CREATE_SECONDARY_OFFICE: "DIALPAD_CREATE_SECONDARY_OFFICE",
    CREATE_SMS_EVENT_SUBSCRIPTION: "DIALPAD_CREATE_SMS_EVENT_SUBSCRIPTION",
    CREATE_USER_WITH_AUTO_ASSIGN: "DIALPAD_CREATE_USER_WITH_AUTO_ASSIGN",
    CREATE_WEBHOOK_POST_ENDPOINT: "DIALPAD_CREATE_WEBHOOK_POST_ENDPOINT",
    DEAUTHORIZE_OAUTH2_SESSION: "DIALPAD_DEAUTHORIZE_OAUTH2_SESSION",
    DELETE_ACCESS_CONTROL_POLICY_BY_ID: "DIALPAD_DELETE_ACCESS_CONTROL_POLICY_BY_ID",
    DELETE_AGENT_STATUS_BY_ID: "DIALPAD_DELETE_AGENT_STATUS_BY_ID",
    DELETE_CALL_CENTER_BY_ID: "DIALPAD_DELETE_CALL_CENTER_BY_ID",
    DELETE_CALL_REVIEW_SHARE_LINK_BY_ID: "DIALPAD_DELETE_CALL_REVIEW_SHARE_LINK_BY_ID",
    DELETE_CALL_ROUTER_BY_ID: "DIALPAD_DELETE_CALL_ROUTER_BY_ID",
    DELETE_CALL_SUBSCRIPTION_BY_ID: "DIALPAD_DELETE_CALL_SUBSCRIPTION_BY_ID",
    DELETE_CHANNEL_BY_ID: "DIALPAD_DELETE_CHANNEL_BY_ID",
    DELETE_CHANNEL_MEMBER_BY_ID: "DIALPAD_DELETE_CHANNEL_MEMBER_BY_ID",
    DELETE_CONTACT_BY_ID: "DIALPAD_DELETE_CONTACT_BY_ID",
    DELETE_CUSTOM_IVR_BY_TARGET_TYPE_AND_IVRTYPE: "DIALPAD_DELETE_CUSTOM_IVR_BY_TARGET_TYPE_AND_IVRTYPE",
    DELETE_DEPARTMENT_RESOURCE: "DIALPAD_DELETE_DEPARTMENT_RESOURCE",
    DELETE_DESKPHONE_BY_USER_ID: "DIALPAD_DELETE_DESKPHONE_BY_USER_ID",
    DELETE_DESKPHONE_IN_ROOM: "DIALPAD_DELETE_DESKPHONE_IN_ROOM",
    DELETE_NUMBER_VIA_API: "DIALPAD_DELETE_NUMBER_VIA_API",
    DELETE_OFFICE_OPERATOR_BY_ID: "DIALPAD_DELETE_OFFICE_OPERATOR_BY_ID",
    DELETE_OPERATOR_FROM_CALL_CENTER: "DIALPAD_DELETE_OPERATOR_FROM_CALL_CENTER",
    DELETE_OPERATOR_FROM_DEPARTMENT: "DIALPAD_DELETE_OPERATOR_FROM_DEPARTMENT",
    DELETE_RECORDING_SHARE_LINK_BY_ID: "DIALPAD_DELETE_RECORDING_SHARE_LINK_BY_ID",
    DELETE_ROOM_BY_ID: "DIALPAD_DELETE_ROOM_BY_ID",
    DELETE_SCHEDULE_REPORT_BY_ID: "DIALPAD_DELETE_SCHEDULE_REPORT_BY_ID",
    DELETE_SMS_SUBSCRIPTION_BY_ID: "DIALPAD_DELETE_SMS_SUBSCRIPTION_BY_ID",
    DELETE_SUBSCRIPTION_CHANGELOG_BY_ID: "DIALPAD_DELETE_SUBSCRIPTION_CHANGELOG_BY_ID",
    DELETE_SUBSCRIPTION_CONTACT_BY_ID: "DIALPAD_DELETE_SUBSCRIPTION_CONTACT_BY_ID",
    DELETE_USER_BY_ID: "DIALPAD_DELETE_USER_BY_ID",
    DELETE_WEBHOOK_BY_ID: "DIALPAD_DELETE_WEBHOOK_BY_ID",
    DELETE_WEBSOCKET_CONNECTION_BY_ID: "DIALPAD_DELETE_WEBSOCKET_CONNECTION_BY_ID",
    FETCH_CALL_BY_ID: "DIALPAD_FETCH_CALL_BY_ID",
    FETCH_CALL_TRANSCRIPT_BY_ID: "DIALPAD_FETCH_CALL_TRANSCRIPT_BY_ID",
    FETCH_CUSTOM_IVRS: "DIALPAD_FETCH_CUSTOM_IVRS",
    FETCH_OPERATORS_FOR_OFFICE_ID: "DIALPAD_FETCH_OPERATORS_FOR_OFFICE_ID",
    FETCH_SCHEDULED_REPORTS: "DIALPAD_FETCH_SCHEDULED_REPORTS",
    FETCH_TRANSCRIPT_URL_BY_CALL_ID: "DIALPAD_FETCH_TRANSCRIPT_URL_BY_CALL_ID",
    FETCH_USER_DETAILS_BY_ID: "DIALPAD_FETCH_USER_DETAILS_BY_ID",
    FORMAT_PHONE_NUMBERS: "DIALPAD_FORMAT_PHONE_NUMBERS",
    GET_AVAILABLE_LICENSES_FOR_AN_OFFICE: "DIALPAD_GET_AVAILABLE_LICENSES_FOR_AN_OFFICE",
    GET_CALL_CENTER_STATUS_BY_ID: "DIALPAD_GET_CALL_CENTER_STATUS_BY_ID",
    GET_CALL_LABELS: "DIALPAD_GET_CALL_LABELS",
    GET_CALL_REVIEW_SHARE_LINK_BY_ID: "DIALPAD_GET_CALL_REVIEW_SHARE_LINK_BY_ID",
    GET_CALL_SUBSCRIPTION_DETAILS: "DIALPAD_GET_CALL_SUBSCRIPTION_DETAILS",
    GET_CHANNEL_MEMBERS_BY_ID: "DIALPAD_GET_CHANNEL_MEMBERS_BY_ID",
    GET_DEPARTMENTS_BY_OFFICE_ID: "DIALPAD_GET_DEPARTMENTS_BY_OFFICE_ID",
    GET_DESKPHONES_FOR_USER_PARENT_ID: "DIALPAD_GET_DESKPHONES_FOR_USER_PARENT_ID",
    GET_DESKPHONES_IN_ROOMS: "DIALPAD_GET_DESKPHONES_IN_ROOMS",
    GET_DESKPHONE_BY_PARENT_ID: "DIALPAD_GET_DESKPHONE_BY_PARENT_ID",
    GET_OFFICE_CALL_CENTERS: "DIALPAD_GET_OFFICE_CALL_CENTERS",
    GET_OPERATOR_SKILLS_BY_CALL_CENTER_AND_USER: "DIALPAD_GET_OPERATOR_SKILLS_BY_CALL_CENTER_AND_USER",
    GET_PHONE_NUMBER_DETAILS: "DIALPAD_GET_PHONE_NUMBER_DETAILS",
    GET_RECORDING_SHARE_LINK_BY_ID: "DIALPAD_GET_RECORDING_SHARE_LINK_BY_ID",
    GET_ROOMS_LIST: "DIALPAD_GET_ROOMS_LIST",
    GET_SUBSCRIPTION_CONTACT_BY_ID: "DIALPAD_GET_SUBSCRIPTION_CONTACT_BY_ID",
    GET_USER_CALLER_ID: "DIALPAD_GET_USER_CALLER_ID",
    GET_USER_DESKPHONES: "DIALPAD_GET_USER_DESKPHONES",
    GET_USER_E911_DETAILS: "DIALPAD_GET_USER_E911_DETAILS",
    GET_WEBHOOKS_INFORMATION: "DIALPAD_GET_WEBHOOKS_INFORMATION",
    HANGUP_CALL_VIA_ID: "DIALPAD_HANGUP_CALL_VIA_ID",
    INITIATE_IVR_CALL_API_ENDPOINT: "DIALPAD_INITIATE_IVR_CALL_API_ENDPOINT",
    INITIATE_OUTBOUND_CALL_VIA_API: "DIALPAD_INITIATE_OUTBOUND_CALL_VIA_API",
    INITIATE_USER_CALL_WITH_GROUP_OPTIONS: "DIALPAD_INITIATE_USER_CALL_WITH_GROUP_OPTIONS",
    LIST_CHANNELS_API: "DIALPAD_LIST_CHANNELS_API",
    LIST_CONFERENCE_ROOMS: "DIALPAD_LIST_CONFERENCE_ROOMS",
    LIST_OFFICES_ENDPOINT: "DIALPAD_LIST_OFFICES_ENDPOINT",
    LIST_USERS: "DIALPAD_LIST_USERS",
    MODIFY_CONTACT_DETAILS_USING_ID: "DIALPAD_MODIFY_CONTACT_DETAILS_USING_ID",
    MODIFY_CUSTOM_IVR_SETTINGS: "DIALPAD_MODIFY_CUSTOM_IVR_SETTINGS",
    MODIFY_OPERATOR_SKILL_LEVEL: "DIALPAD_MODIFY_OPERATOR_SKILL_LEVEL",
    MODIFY_WEBHOOK_CONFIGURATION: "DIALPAD_MODIFY_WEBHOOK_CONFIGURATION",
    MOVE_USER_TO_SPECIFIED_OFFICE: "DIALPAD_MOVE_USER_TO_SPECIFIED_OFFICE",
    OAUTH2_AUTHORIZE_ENDPOINT: "DIALPAD_OAUTH2_AUTHORIZE_ENDPOINT",
    PATCH_ACCESS_CONTROL_POLICY_BY_ID: "DIALPAD_PATCH_ACCESS_CONTROL_POLICY_BY_ID",
    PATCH_AGENT_STATUS_EVENT_SUBSCRIPTION: "DIALPAD_PATCH_AGENT_STATUS_EVENT_SUBSCRIPTION",
    PATCH_CALL_ROUTER_BY_ID: "DIALPAD_PATCH_CALL_ROUTER_BY_ID",
    PATCH_DEPARTMENT_DETAILS_BY_ID: "DIALPAD_PATCH_DEPARTMENT_DETAILS_BY_ID",
    PATCH_SUBSCRIPTION_CHANGELOG_ENDPOINT: "DIALPAD_PATCH_SUBSCRIPTION_CHANGELOG_ENDPOINT",
    PATCH_USER_ACTIVE_CALL_RECORDING: "DIALPAD_PATCH_USER_ACTIVE_CALL_RECORDING",
    PATCH_USER_STATUS: "DIALPAD_PATCH_USER_STATUS",
    PATCH_WEBSOCKET_SIGNATURE_SECRET: "DIALPAD_PATCH_WEBSOCKET_SIGNATURE_SECRET",
    POST_CALL_EVENT_SUBSCRIPTION: "DIALPAD_POST_CALL_EVENT_SUBSCRIPTION",
    POST_RECORDING_SHARE_LINK: "DIALPAD_POST_RECORDING_SHARE_LINK",
    POST_SUBSCRIPTION_CHANGE_LOG_EVENT: "DIALPAD_POST_SUBSCRIPTION_CHANGE_LOG_EVENT",
    POST_USERS_SCREENPOP_URI: "DIALPAD_POST_USERS_SCREENPOP_URI",
    POST_WEBSOCKET_CONNECTION_SECRET: "DIALPAD_POST_WEBSOCKET_CONNECTION_SECRET",
    REASSIGN_PHONE_NUMBER_TO_TARGET: "DIALPAD_REASSIGN_PHONE_NUMBER_TO_TARGET",
    REDEEM_ACCESS_OR_REFRESH_TOKEN: "DIALPAD_REDEEM_ACCESS_OR_REFRESH_TOKEN",
    REMOVE_BLOCKED_NUMBERS: "DIALPAD_REMOVE_BLOCKED_NUMBERS",
    RETRIEVE_ACCESS_CONTROL_POLICY_ASSIGNMENTS: "DIALPAD_RETRIEVE_ACCESS_CONTROL_POLICY_ASSIGNMENTS",
    RETRIEVE_ACCESS_CONTROL_POLICY_BY_ID: "DIALPAD_RETRIEVE_ACCESS_CONTROL_POLICY_BY_ID",
    RETRIEVE_AGENT_STATUS_BY_ID: "DIALPAD_RETRIEVE_AGENT_STATUS_BY_ID",
    RETRIEVE_AGENT_STATUS_SUBSCRIPTION: "DIALPAD_RETRIEVE_AGENT_STATUS_SUBSCRIPTION",
    RETRIEVE_APP_SETTINGS: "DIALPAD_RETRIEVE_APP_SETTINGS",
    RETRIEVE_BLOCKED_NUMBER_DETAILS: "DIALPAD_RETRIEVE_BLOCKED_NUMBER_DETAILS",
    RETRIEVE_BLOCKED_PHONE_NUMBERS: "DIALPAD_RETRIEVE_BLOCKED_PHONE_NUMBERS",
    RETRIEVE_CALLCENTER_OPERATORS_BY_ID: "DIALPAD_RETRIEVE_CALLCENTER_OPERATORS_BY_ID",
    RETRIEVE_CALL_CENTERS_INFORMATION: "DIALPAD_RETRIEVE_CALL_CENTERS_INFORMATION",
    RETRIEVE_CALL_CENTER_BY_ID: "DIALPAD_RETRIEVE_CALL_CENTER_BY_ID",
    RETRIEVE_CALL_INFORMATION: "DIALPAD_RETRIEVE_CALL_INFORMATION",
    RETRIEVE_CALL_ROUTERS_V2_API: "DIALPAD_RETRIEVE_CALL_ROUTERS_V2_API",
    RETRIEVE_CALL_ROUTER_BY_ID: "DIALPAD_RETRIEVE_CALL_ROUTER_BY_ID",
    RETRIEVE_CALL_SUBSCRIPTION_BY_ID: "DIALPAD_RETRIEVE_CALL_SUBSCRIPTION_BY_ID",
    RETRIEVE_CHANNEL_BY_ID: "DIALPAD_RETRIEVE_CHANNEL_BY_ID",
    RETRIEVE_COACHING_STATS_V2: "DIALPAD_RETRIEVE_COACHING_STATS_V2",
    RETRIEVE_COACHING_TEAMS_INFORMATION: "DIALPAD_RETRIEVE_COACHING_TEAMS_INFORMATION",
    RETRIEVE_COACHING_TEAM_BY_ID: "DIALPAD_RETRIEVE_COACHING_TEAM_BY_ID",
    RETRIEVE_COMPANY_DATA: "DIALPAD_RETRIEVE_COMPANY_DATA",
    RETRIEVE_CONFERENCE_MEETINGS: "DIALPAD_RETRIEVE_CONFERENCE_MEETINGS",
    RETRIEVE_CONTACT_BY_ID: "DIALPAD_RETRIEVE_CONTACT_BY_ID",
    RETRIEVE_CONTACT_LIST: "DIALPAD_RETRIEVE_CONTACT_LIST",
    RETRIEVE_DEPARTMENT_BY_ID: "DIALPAD_RETRIEVE_DEPARTMENT_BY_ID",
    RETRIEVE_DEPARTMENT_OPERATORS_BY_ID: "DIALPAD_RETRIEVE_DEPARTMENT_OPERATORS_BY_ID",
    RETRIEVE_E911_OFFICE_DETAILS: "DIALPAD_RETRIEVE_E911_OFFICE_DETAILS",
    RETRIEVE_LIST_OF_DEPARTMENTS: "DIALPAD_RETRIEVE_LIST_OF_DEPARTMENTS",
    RETRIEVE_MEMBERS_OF_COACHING_TEAM_BY_ID: "DIALPAD_RETRIEVE_MEMBERS_OF_COACHING_TEAM_BY_ID",
    RETRIEVE_NUMBERS_VIA_API_V2: "DIALPAD_RETRIEVE_NUMBERS_VIA_API_V2",
    RETRIEVE_OFFICE_BY_ID: "DIALPAD_RETRIEVE_OFFICE_BY_ID",
    RETRIEVE_OFFICE_PLAN_BY_ID: "DIALPAD_RETRIEVE_OFFICE_PLAN_BY_ID",
    RETRIEVE_OFF_DUTY_STATUS_BY_OFFICE_ID: "DIALPAD_RETRIEVE_OFF_DUTY_STATUS_BY_OFFICE_ID",
    RETRIEVE_OPERATOR_DUTY_STATUS: "DIALPAD_RETRIEVE_OPERATOR_DUTY_STATUS",
    RETRIEVE_ROOM_INFORMATION_BY_ID: "DIALPAD_RETRIEVE_ROOM_INFORMATION_BY_ID",
    RETRIEVE_SCHEDULE_REPORT_BY_ID: "DIALPAD_RETRIEVE_SCHEDULE_REPORT_BY_ID",
    RETRIEVE_SMS_OPT_OUT_BY_COMPANY_ID: "DIALPAD_RETRIEVE_SMS_OPT_OUT_BY_COMPANY_ID",
    RETRIEVE_SMS_SUBSCRIPTIONS: "DIALPAD_RETRIEVE_SMS_SUBSCRIPTIONS",
    RETRIEVE_SMS_SUBSCRIPTION_BY_ID: "DIALPAD_RETRIEVE_SMS_SUBSCRIPTION_BY_ID",
    RETRIEVE_SPECIFIC_WEBHOOK_BY_ID: "DIALPAD_RETRIEVE_SPECIFIC_WEBHOOK_BY_ID",
    RETRIEVE_STAT_BY_ID: "DIALPAD_RETRIEVE_STAT_BY_ID",
    RETRIEVE_SUBSCRIPTION_CHANGE_LOG: "DIALPAD_RETRIEVE_SUBSCRIPTION_CHANGE_LOG",
    RETRIEVE_SUBSCRIPTION_CHANGE_LOG_BY_ID: "DIALPAD_RETRIEVE_SUBSCRIPTION_CHANGE_LOG_BY_ID",
    RETRIEVE_SUBSCRIPTION_CONTACTS: "DIALPAD_RETRIEVE_SUBSCRIPTION_CONTACTS",
    RETRIEVE_TEAMS_FOR_OFFICE_ID: "DIALPAD_RETRIEVE_TEAMS_FOR_OFFICE_ID",
    RETRIEVE_USER_DEVICES_LIST: "DIALPAD_RETRIEVE_USER_DEVICES_LIST",
    RETRIEVE_USER_DEVICE_BY_ID: "DIALPAD_RETRIEVE_USER_DEVICE_BY_ID",
    RETRIEVE_USER_PERSONAS_BY_ID: "DIALPAD_RETRIEVE_USER_PERSONAS_BY_ID",
    RETRIEVE_WEBSOCKET_BY_ID: "DIALPAD_RETRIEVE_WEBSOCKET_BY_ID",
    RETRIEVE_WEBSOCKET_CONNECTION: "DIALPAD_RETRIEVE_WEBSOCKET_CONNECTION",
    SEND_SMS_WITH_OPTIONAL_MEDIA_AND_GROUP_SENDER: "DIALPAD_SEND_SMS_WITH_OPTIONAL_MEDIA_AND_GROUP_SENDER",
    SET_USER_CALLER_ID_BY_ID: "DIALPAD_SET_USER_CALLER_ID_BY_ID",
    SUBSCRIBE_CONTACT_EVENT: "DIALPAD_SUBSCRIBE_CONTACT_EVENT",
    TOGGLE_USER_DO_NOT_DISTURB_STATUS: "DIALPAD_TOGGLE_USER_DO_NOT_DISTURB_STATUS",
    TOGGLE_VI_CALL_FOR_USER_BY_ID: "DIALPAD_TOGGLE_VI_CALL_FOR_USER_BY_ID",
    TRANSFER_CALL_TO_DESTINATION: "DIALPAD_TRANSFER_CALL_TO_DESTINATION",
    UNASSIGN_ACCESS_CONTROL_POLICY_FROM_USER: "DIALPAD_UNASSIGN_ACCESS_CONTROL_POLICY_FROM_USER",
    UNASSIGN_OFFICE_NUMBER: "DIALPAD_UNASSIGN_OFFICE_NUMBER",
    UNASSIGN_ROOM_PHONE_NUMBER: "DIALPAD_UNASSIGN_ROOM_PHONE_NUMBER",
    UNASSIGN_USER_PHONE_NUMBER: "DIALPAD_UNASSIGN_USER_PHONE_NUMBER",
    UNPARK_CALL_BY_USER_ID: "DIALPAD_UNPARK_CALL_BY_USER_ID",
    UPDATE_CALL_CENTER_SETTINGS_BY_ID: "DIALPAD_UPDATE_CALL_CENTER_SETTINGS_BY_ID",
    UPDATE_CALL_REVIEW_SHARE_LINK_PRIVACY: "DIALPAD_UPDATE_CALL_REVIEW_SHARE_LINK_PRIVACY",
    UPDATE_CALL_SUBSCRIPTION_BY_ID: "DIALPAD_UPDATE_CALL_SUBSCRIPTION_BY_ID",
    UPDATE_CONTACT_INFORMATION: "DIALPAD_UPDATE_CONTACT_INFORMATION",
    UPDATE_CONTACT_SUBSCRIPTION_BY_ID: "DIALPAD_UPDATE_CONTACT_SUBSCRIPTION_BY_ID",
    UPDATE_E911_ADDRESS_FOR_OFFICE: "DIALPAD_UPDATE_E911_ADDRESS_FOR_OFFICE",
    UPDATE_E911_ADDRESS_FOR_USER: "DIALPAD_UPDATE_E911_ADDRESS_FOR_USER",
    UPDATE_IVR_DETAILS_BY_ID: "DIALPAD_UPDATE_IVR_DETAILS_BY_ID",
    UPDATE_OPERATOR_DUTY_STATUS: "DIALPAD_UPDATE_OPERATOR_DUTY_STATUS",
    UPDATE_RECORDING_SHARE_LINK_PRIVACY: "DIALPAD_UPDATE_RECORDING_SHARE_LINK_PRIVACY",
    UPDATE_ROOM_DETAILS: "DIALPAD_UPDATE_ROOM_DETAILS",
    UPDATE_SCHEDULE_REPORT_SETTINGS: "DIALPAD_UPDATE_SCHEDULE_REPORT_SETTINGS",
    UPDATE_SMS_SUBSCRIPTION_SETTINGS: "DIALPAD_UPDATE_SMS_SUBSCRIPTION_SETTINGS",
    UPDATE_USER_INFORMATION_BY_ID: "DIALPAD_UPDATE_USER_INFORMATION_BY_ID",
    VALIDATE_CALLBACK_REQUEST: "DIALPAD_VALIDATE_CALLBACK_REQUEST",
  },
  triggerTypes: {},
}
