// @composio/core/generated/discordbot.ts
/**
 * Map of Composio's DISCORDBOT toolkit.
 */
export const DISCORDBOT = {
  slug: "discordbot",
  tools: {
    ADD_GROUP_DM_USER: "DISCORDBOT_ADD_GROUP_DM_USER",
    ADD_GUILD_MEMBER: "DISCORDBOT_ADD_GUILD_MEMBER",
    ADD_GUILD_MEMBER_ROLE: "DISCORDBOT_ADD_GUILD_MEMBER_ROLE",
    ADD_MY_MESSAGE_REACTION: "DISCORDBOT_ADD_MY_MESSAGE_REACTION",
    ADD_THREAD_MEMBER: "DISCORDBOT_ADD_THREAD_MEMBER",
    BAN_USER_FROM_GUILD: "DISCORDBOT_BAN_USER_FROM_GUILD",
    BULK_BAN_USERS_FROM_GUILD: "DISCORDBOT_BULK_BAN_USERS_FROM_GUILD",
    BULK_DELETE_MESSAGES: "DISCORDBOT_BULK_DELETE_MESSAGES",
    CREATE_APPLICATION_COMMAND: "DISCORDBOT_CREATE_APPLICATION_COMMAND",
    CREATE_AUTO_MODERATION_RULE: "DISCORDBOT_CREATE_AUTO_MODERATION_RULE",
    CREATE_CHANNEL_INVITE: "DISCORDBOT_CREATE_CHANNEL_INVITE",
    CREATE_DM: "DISCORDBOT_CREATE_DM",
    CREATE_GUILD: "DISCORDBOT_CREATE_GUILD",
    CREATE_GUILD_APPLICATION_COMMAND: "DISCORDBOT_CREATE_GUILD_APPLICATION_COMMAND",
    CREATE_GUILD_CHANNEL: "DISCORDBOT_CREATE_GUILD_CHANNEL",
    CREATE_GUILD_EMOJI: "DISCORDBOT_CREATE_GUILD_EMOJI",
    CREATE_GUILD_FROM_TEMPLATE: "DISCORDBOT_CREATE_GUILD_FROM_TEMPLATE",
    CREATE_GUILD_ROLE: "DISCORDBOT_CREATE_GUILD_ROLE",
    CREATE_GUILD_SCHEDULED_EVENT: "DISCORDBOT_CREATE_GUILD_SCHEDULED_EVENT",
    CREATE_GUILD_STICKER: "DISCORDBOT_CREATE_GUILD_STICKER",
    CREATE_GUILD_TEMPLATE: "DISCORDBOT_CREATE_GUILD_TEMPLATE",
    CREATE_INTERACTION_RESPONSE: "DISCORDBOT_CREATE_INTERACTION_RESPONSE",
    CREATE_MESSAGE: "DISCORDBOT_CREATE_MESSAGE",
    CREATE_STAGE_INSTANCE: "DISCORDBOT_CREATE_STAGE_INSTANCE",
    CREATE_THREAD: "DISCORDBOT_CREATE_THREAD",
    CREATE_THREAD_FROM_MESSAGE: "DISCORDBOT_CREATE_THREAD_FROM_MESSAGE",
    CREATE_WEBHOOK: "DISCORDBOT_CREATE_WEBHOOK",
    CROSSPOST_MESSAGE: "DISCORDBOT_CROSSPOST_MESSAGE",
    DELETE_ALL_MESSAGE_REACTIONS: "DISCORDBOT_DELETE_ALL_MESSAGE_REACTIONS",
    DELETE_ALL_MESSAGE_REACTIONS_BY_EMOJI: "DISCORDBOT_DELETE_ALL_MESSAGE_REACTIONS_BY_EMOJI",
    DELETE_APPLICATION_COMMAND: "DISCORDBOT_DELETE_APPLICATION_COMMAND",
    DELETE_AUTO_MODERATION_RULE: "DISCORDBOT_DELETE_AUTO_MODERATION_RULE",
    DELETE_CHANNEL: "DISCORDBOT_DELETE_CHANNEL",
    DELETE_CHANNEL_PERMISSION_OVERWRITE: "DISCORDBOT_DELETE_CHANNEL_PERMISSION_OVERWRITE",
    DELETE_GROUP_DM_USER: "DISCORDBOT_DELETE_GROUP_DM_USER",
    DELETE_GUILD: "DISCORDBOT_DELETE_GUILD",
    DELETE_GUILD_APPLICATION_COMMAND: "DISCORDBOT_DELETE_GUILD_APPLICATION_COMMAND",
    DELETE_GUILD_EMOJI: "DISCORDBOT_DELETE_GUILD_EMOJI",
    DELETE_GUILD_INTEGRATION: "DISCORDBOT_DELETE_GUILD_INTEGRATION",
    DELETE_GUILD_MEMBER: "DISCORDBOT_DELETE_GUILD_MEMBER",
    DELETE_GUILD_MEMBER_ROLE: "DISCORDBOT_DELETE_GUILD_MEMBER_ROLE",
    DELETE_GUILD_ROLE: "DISCORDBOT_DELETE_GUILD_ROLE",
    DELETE_GUILD_SCHEDULED_EVENT: "DISCORDBOT_DELETE_GUILD_SCHEDULED_EVENT",
    DELETE_GUILD_STICKER: "DISCORDBOT_DELETE_GUILD_STICKER",
    DELETE_GUILD_TEMPLATE: "DISCORDBOT_DELETE_GUILD_TEMPLATE",
    DELETE_MESSAGE: "DISCORDBOT_DELETE_MESSAGE",
    DELETE_MY_MESSAGE_REACTION: "DISCORDBOT_DELETE_MY_MESSAGE_REACTION",
    DELETE_ORIGINAL_WEBHOOK_MESSAGE: "DISCORDBOT_DELETE_ORIGINAL_WEBHOOK_MESSAGE",
    DELETE_STAGE_INSTANCE: "DISCORDBOT_DELETE_STAGE_INSTANCE",
    DELETE_THREAD_MEMBER: "DISCORDBOT_DELETE_THREAD_MEMBER",
    DELETE_USER_MESSAGE_REACTION: "DISCORDBOT_DELETE_USER_MESSAGE_REACTION",
    DELETE_WEBHOOK: "DISCORDBOT_DELETE_WEBHOOK",
    DELETE_WEBHOOK_BY_TOKEN: "DISCORDBOT_DELETE_WEBHOOK_BY_TOKEN",
    DELETE_WEBHOOK_MESSAGE: "DISCORDBOT_DELETE_WEBHOOK_MESSAGE",
    EXECUTE_GITHUB_COMPATIBLE_WEBHOOK: "DISCORDBOT_EXECUTE_GITHUB_COMPATIBLE_WEBHOOK",
    EXECUTE_SLACK_COMPATIBLE_WEBHOOK: "DISCORDBOT_EXECUTE_SLACK_COMPATIBLE_WEBHOOK",
    EXECUTE_WEBHOOK: "DISCORDBOT_EXECUTE_WEBHOOK",
    FOLLOW_CHANNEL: "DISCORDBOT_FOLLOW_CHANNEL",
    GET_ACTIVE_GUILD_THREADS: "DISCORDBOT_GET_ACTIVE_GUILD_THREADS",
    GET_APPLICATION: "DISCORDBOT_GET_APPLICATION",
    GET_APPLICATION_COMMAND: "DISCORDBOT_GET_APPLICATION_COMMAND",
    GET_APPLICATION_ROLE_CONNECTIONS_METADATA: "DISCORDBOT_GET_APPLICATION_ROLE_CONNECTIONS_METADATA",
    GET_APPLICATION_USER_ROLE_CONNECTION: "DISCORDBOT_GET_APPLICATION_USER_ROLE_CONNECTION",
    GET_AUTO_MODERATION_RULE: "DISCORDBOT_GET_AUTO_MODERATION_RULE",
    GET_BOT_GATEWAY: "DISCORDBOT_GET_BOT_GATEWAY",
    GET_CHANNEL: "DISCORDBOT_GET_CHANNEL",
    GET_GATEWAY: "DISCORDBOT_GET_GATEWAY",
    GET_GUILD: "DISCORDBOT_GET_GUILD",
    GET_GUILDS_ONBOARDING: "DISCORDBOT_GET_GUILDS_ONBOARDING",
    GET_GUILD_APPLICATION_COMMAND: "DISCORDBOT_GET_GUILD_APPLICATION_COMMAND",
    GET_GUILD_APPLICATION_COMMAND_PERMISSIONS: "DISCORDBOT_GET_GUILD_APPLICATION_COMMAND_PERMISSIONS",
    GET_GUILD_BAN: "DISCORDBOT_GET_GUILD_BAN",
    GET_GUILD_EMOJI: "DISCORDBOT_GET_GUILD_EMOJI",
    GET_GUILD_MEMBER: "DISCORDBOT_GET_GUILD_MEMBER",
    GET_GUILD_NEW_MEMBER_WELCOME: "DISCORDBOT_GET_GUILD_NEW_MEMBER_WELCOME",
    GET_GUILD_PREVIEW: "DISCORDBOT_GET_GUILD_PREVIEW",
    GET_GUILD_SCHEDULED_EVENT: "DISCORDBOT_GET_GUILD_SCHEDULED_EVENT",
    GET_GUILD_STICKER: "DISCORDBOT_GET_GUILD_STICKER",
    GET_GUILD_TEMPLATE: "DISCORDBOT_GET_GUILD_TEMPLATE",
    GET_GUILD_VANITY_URL: "DISCORDBOT_GET_GUILD_VANITY_URL",
    GET_GUILD_WEBHOOKS: "DISCORDBOT_GET_GUILD_WEBHOOKS",
    GET_GUILD_WELCOME_SCREEN: "DISCORDBOT_GET_GUILD_WELCOME_SCREEN",
    GET_GUILD_WIDGET: "DISCORDBOT_GET_GUILD_WIDGET",
    GET_GUILD_WIDGET_SETTINGS: "DISCORDBOT_GET_GUILD_WIDGET_SETTINGS",
    GET_MESSAGE: "DISCORDBOT_GET_MESSAGE",
    GET_MY_APPLICATION: "DISCORDBOT_GET_MY_APPLICATION",
    GET_MY_OAUTH2_APPLICATION: "DISCORDBOT_GET_MY_OAUTH2_APPLICATION",
    GET_ORIGINAL_WEBHOOK_MESSAGE: "DISCORDBOT_GET_ORIGINAL_WEBHOOK_MESSAGE",
    GET_PUBLIC_KEYS: "DISCORDBOT_GET_PUBLIC_KEYS",
    GET_STAGE_INSTANCE: "DISCORDBOT_GET_STAGE_INSTANCE",
    GET_STICKER: "DISCORDBOT_GET_STICKER",
    GET_THREAD_MEMBER: "DISCORDBOT_GET_THREAD_MEMBER",
    GET_USER: "DISCORDBOT_GET_USER",
    GET_WEBHOOK: "DISCORDBOT_GET_WEBHOOK",
    GET_WEBHOOK_BY_TOKEN: "DISCORDBOT_GET_WEBHOOK_BY_TOKEN",
    GET_WEBHOOK_MESSAGE: "DISCORDBOT_GET_WEBHOOK_MESSAGE",
    INVITE_RESOLVE: "DISCORDBOT_INVITE_RESOLVE",
    INVITE_REVOKE: "DISCORDBOT_INVITE_REVOKE",
    JOIN_THREAD: "DISCORDBOT_JOIN_THREAD",
    LEAVE_GUILD: "DISCORDBOT_LEAVE_GUILD",
    LEAVE_THREAD: "DISCORDBOT_LEAVE_THREAD",
    LIST_APPLICATION_COMMANDS: "DISCORDBOT_LIST_APPLICATION_COMMANDS",
    LIST_AUTO_MODERATION_RULES: "DISCORDBOT_LIST_AUTO_MODERATION_RULES",
    LIST_CHANNEL_INVITES: "DISCORDBOT_LIST_CHANNEL_INVITES",
    LIST_CHANNEL_WEBHOOKS: "DISCORDBOT_LIST_CHANNEL_WEBHOOKS",
    LIST_GUILD_APPLICATION_COMMANDS: "DISCORDBOT_LIST_GUILD_APPLICATION_COMMANDS",
    LIST_GUILD_APPLICATION_COMMAND_PERMISSIONS: "DISCORDBOT_LIST_GUILD_APPLICATION_COMMAND_PERMISSIONS",
    LIST_GUILD_AUDIT_LOG_ENTRIES: "DISCORDBOT_LIST_GUILD_AUDIT_LOG_ENTRIES",
    LIST_GUILD_BANS: "DISCORDBOT_LIST_GUILD_BANS",
    LIST_GUILD_CHANNELS: "DISCORDBOT_LIST_GUILD_CHANNELS",
    LIST_GUILD_EMOJIS: "DISCORDBOT_LIST_GUILD_EMOJIS",
    LIST_GUILD_INTEGRATIONS: "DISCORDBOT_LIST_GUILD_INTEGRATIONS",
    LIST_GUILD_INVITES: "DISCORDBOT_LIST_GUILD_INVITES",
    LIST_GUILD_MEMBERS: "DISCORDBOT_LIST_GUILD_MEMBERS",
    LIST_GUILD_ROLES: "DISCORDBOT_LIST_GUILD_ROLES",
    LIST_GUILD_SCHEDULED_EVENTS: "DISCORDBOT_LIST_GUILD_SCHEDULED_EVENTS",
    LIST_GUILD_SCHEDULED_EVENT_USERS: "DISCORDBOT_LIST_GUILD_SCHEDULED_EVENT_USERS",
    LIST_GUILD_STICKERS: "DISCORDBOT_LIST_GUILD_STICKERS",
    LIST_GUILD_TEMPLATES: "DISCORDBOT_LIST_GUILD_TEMPLATES",
    LIST_GUILD_VOICE_REGIONS: "DISCORDBOT_LIST_GUILD_VOICE_REGIONS",
    LIST_MESSAGES: "DISCORDBOT_LIST_MESSAGES",
    LIST_MESSAGE_REACTIONS_BY_EMOJI: "DISCORDBOT_LIST_MESSAGE_REACTIONS_BY_EMOJI",
    LIST_MY_PRIVATE_ARCHIVED_THREADS: "DISCORDBOT_LIST_MY_PRIVATE_ARCHIVED_THREADS",
    LIST_PINNED_MESSAGES: "DISCORDBOT_LIST_PINNED_MESSAGES",
    LIST_PRIVATE_ARCHIVED_THREADS: "DISCORDBOT_LIST_PRIVATE_ARCHIVED_THREADS",
    LIST_PUBLIC_ARCHIVED_THREADS: "DISCORDBOT_LIST_PUBLIC_ARCHIVED_THREADS",
    LIST_STICKER_PACKS: "DISCORDBOT_LIST_STICKER_PACKS",
    LIST_THREAD_MEMBERS: "DISCORDBOT_LIST_THREAD_MEMBERS",
    LIST_VOICE_REGIONS: "DISCORDBOT_LIST_VOICE_REGIONS",
    PIN_MESSAGE: "DISCORDBOT_PIN_MESSAGE",
    PREVIEW_PRUNE_GUILD: "DISCORDBOT_PREVIEW_PRUNE_GUILD",
    PRUNE_GUILD: "DISCORDBOT_PRUNE_GUILD",
    PUT_GUILDS_ONBOARDING: "DISCORDBOT_PUT_GUILDS_ONBOARDING",
    SEARCH_GUILD_MEMBERS: "DISCORDBOT_SEARCH_GUILD_MEMBERS",
    SET_CHANNEL_PERMISSION_OVERWRITE: "DISCORDBOT_SET_CHANNEL_PERMISSION_OVERWRITE",
    SYNC_GUILD_TEMPLATE: "DISCORDBOT_SYNC_GUILD_TEMPLATE",
    TRIGGER_TYPING_INDICATOR: "DISCORDBOT_TRIGGER_TYPING_INDICATOR",
    UNBAN_USER_FROM_GUILD: "DISCORDBOT_UNBAN_USER_FROM_GUILD",
    UNPIN_MESSAGE: "DISCORDBOT_UNPIN_MESSAGE",
    UPDATE_APPLICATION: "DISCORDBOT_UPDATE_APPLICATION",
    UPDATE_APPLICATION_COMMAND: "DISCORDBOT_UPDATE_APPLICATION_COMMAND",
    UPDATE_APPLICATION_USER_ROLE_CONNECTION: "DISCORDBOT_UPDATE_APPLICATION_USER_ROLE_CONNECTION",
    UPDATE_AUTO_MODERATION_RULE: "DISCORDBOT_UPDATE_AUTO_MODERATION_RULE",
    UPDATE_CHANNEL: "DISCORDBOT_UPDATE_CHANNEL",
    UPDATE_GUILD: "DISCORDBOT_UPDATE_GUILD",
    UPDATE_GUILD_APPLICATION_COMMAND: "DISCORDBOT_UPDATE_GUILD_APPLICATION_COMMAND",
    UPDATE_GUILD_EMOJI: "DISCORDBOT_UPDATE_GUILD_EMOJI",
    UPDATE_GUILD_MEMBER: "DISCORDBOT_UPDATE_GUILD_MEMBER",
    UPDATE_GUILD_ROLE: "DISCORDBOT_UPDATE_GUILD_ROLE",
    UPDATE_GUILD_SCHEDULED_EVENT: "DISCORDBOT_UPDATE_GUILD_SCHEDULED_EVENT",
    UPDATE_GUILD_STICKER: "DISCORDBOT_UPDATE_GUILD_STICKER",
    UPDATE_GUILD_TEMPLATE: "DISCORDBOT_UPDATE_GUILD_TEMPLATE",
    UPDATE_GUILD_WELCOME_SCREEN: "DISCORDBOT_UPDATE_GUILD_WELCOME_SCREEN",
    UPDATE_GUILD_WIDGET_SETTINGS: "DISCORDBOT_UPDATE_GUILD_WIDGET_SETTINGS",
    UPDATE_MESSAGE: "DISCORDBOT_UPDATE_MESSAGE",
    UPDATE_MY_APPLICATION: "DISCORDBOT_UPDATE_MY_APPLICATION",
    UPDATE_MY_GUILD_MEMBER: "DISCORDBOT_UPDATE_MY_GUILD_MEMBER",
    UPDATE_MY_USER: "DISCORDBOT_UPDATE_MY_USER",
    UPDATE_ORIGINAL_WEBHOOK_MESSAGE: "DISCORDBOT_UPDATE_ORIGINAL_WEBHOOK_MESSAGE",
    UPDATE_SELF_VOICE_STATE: "DISCORDBOT_UPDATE_SELF_VOICE_STATE",
    UPDATE_VOICE_STATE: "DISCORDBOT_UPDATE_VOICE_STATE",
    UPDATE_WEBHOOK: "DISCORDBOT_UPDATE_WEBHOOK",
    UPDATE_WEBHOOK_BY_TOKEN: "DISCORDBOT_UPDATE_WEBHOOK_BY_TOKEN",
    UPDATE_WEBHOOK_MESSAGE: "DISCORDBOT_UPDATE_WEBHOOK_MESSAGE",
  },
  triggerTypes: {},
}
