// @composio/core/generated/docusign.ts
/**
 * Map of Composio's DOCUSIGN toolkit.
 */
export const DOCUSIGN = {
  slug: "docusign",
  tools: {
    ADD_CONTACTS_TO_CONTACTS_LIST: "DOCUSIGN_ADD_CONTACTS_TO_CONTACTS_LIST",
    ADD_EMAIL_OVERRIDES_TO_ENVELOPE: "DOCUSIGN_ADD_EMAIL_OVERRIDES_TO_ENVELOPE",
    ADD_ENVELOPE_ATTACHMENTS: "DOCUSIGN_ADD_ENVELOPE_ATTACHMENTS",
    ADD_EXISTING_BRAND_TO_GROUP: "DOCUSIGN_ADD_EXISTING_BRAND_TO_GROUP",
    ADD_FILE_TO_WORKSPACE: "DOCUSIGN_ADD_FILE_TO_WORKSPACE",
    ADD_MEMBERS_TO_SIGNING_GROUP: "DOCUSIGN_ADD_MEMBERS_TO_SIGNING_GROUP",
    ADD_NEW_USERS_TO_A_SPECIFIED_ACCOUNT: "DOCUSIGN_ADD_NEW_USERS_TO_A_SPECIFIED_ACCOUNT",
    ADD_OR_UPDATE_ACCOUNT_STAMPS: "DOCUSIGN_ADD_OR_UPDATE_ACCOUNT_STAMPS",
    ADD_OR_UPDATE_USER_CUSTOM_SETTINGS: "DOCUSIGN_ADD_OR_UPDATE_USER_CUSTOM_SETTINGS",
    ADD_OR_UPDATE_USER_SIGNATURE: "DOCUSIGN_ADD_OR_UPDATE_USER_SIGNATURE",
    ADD_PART_TO_CHUNKED_UPLOAD: "DOCUSIGN_ADD_PART_TO_CHUNKED_UPLOAD",
    ADD_STEP_TO_ENVELOPE_WORKFLOW: "DOCUSIGN_ADD_STEP_TO_ENVELOPE_WORKFLOW",
    ADD_TEMPLATES_TO_DOCUMENT_IN_ENVELOPE: "DOCUSIGN_ADD_TEMPLATES_TO_DOCUMENT_IN_ENVELOPE",
    ADD_TEMPLATES_TO_ENVELOPE: "DOCUSIGN_ADD_TEMPLATES_TO_ENVELOPE",
    ADD_USERS_TO_EXISTING_GROUP: "DOCUSIGN_ADD_USERS_TO_EXISTING_GROUP",
    ADD_USER_SIGNATURE_AND_INITIALS_IMAGES: "DOCUSIGN_ADD_USER_SIGNATURE_AND_INITIALS_IMAGES",
    APPLY_ACTION_TO_BULK_SEND_ENVELOPES: "DOCUSIGN_APPLY_ACTION_TO_BULK_SEND_ENVELOPES",
    CHANGE_USERS_IN_ACCOUNT: "DOCUSIGN_CHANGE_USERS_IN_ACCOUNT",
    CLOSE_USERS_IN_ACCOUNT: "DOCUSIGN_CLOSE_USERS_IN_ACCOUNT",
    COMMIT_CHUNKED_UPLOAD_FOR_ENVELOPES: "DOCUSIGN_COMMIT_CHUNKED_UPLOAD_FOR_ENVELOPES",
    CONFIGURE_CLOUD_STORAGE_REDIRECT_URL: "DOCUSIGN_CONFIGURE_CLOUD_STORAGE_REDIRECT_URL",
    CREATE_ACCOUNT_CUSTOM_FIELD: "DOCUSIGN_CREATE_ACCOUNT_CUSTOM_FIELD",
    CREATE_A_COLLABORATIVE_WORKSPACE: "DOCUSIGN_CREATE_A_COLLABORATIVE_WORKSPACE",
    CREATE_BCC_EMAIL_ARCHIVE_CONFIGURATION: "DOCUSIGN_CREATE_BCC_EMAIL_ARCHIVE_CONFIGURATION",
    CREATE_BRAND_PROFILES_FOR_ACCOUNT: "DOCUSIGN_CREATE_BRAND_PROFILES_FOR_ACCOUNT",
    CREATE_BULK_SEND_REQUEST: "DOCUSIGN_CREATE_BULK_SEND_REQUEST",
    CREATE_BULK_SEND_TEST_REQUEST: "DOCUSIGN_CREATE_BULK_SEND_TEST_REQUEST",
    CREATE_CONNECT_CONFIGURATION_FOR_ACCOUNT: "DOCUSIGN_CREATE_CONNECT_CONFIGURATION_FOR_ACCOUNT",
    CREATE_CUSTOM_DOCUMENT_FIELDS_IN_ENVELOPE: "DOCUSIGN_CREATE_CUSTOM_DOCUMENT_FIELDS_IN_ENVELOPE",
    CREATE_CUSTOM_FIELDS_FOR_ENVELOPE: "DOCUSIGN_CREATE_CUSTOM_FIELDS_FOR_ENVELOPE",
    CREATE_CUSTOM_FIELDS_IN_TEMPLATE_DOCUMENT: "DOCUSIGN_CREATE_CUSTOM_FIELDS_IN_TEMPLATE_DOCUMENT",
    CREATE_CUSTOM_TAB_WITH_PROPERTIES: "DOCUSIGN_CREATE_CUSTOM_TAB_WITH_PROPERTIES",
    CREATE_DOCUMENT_RESPONSIVE_HTML_PREVIEW: "DOCUSIGN_CREATE_DOCUMENT_RESPONSIVE_HTML_PREVIEW",
    CREATE_ENVELOPE_FROM_TEMPLATE: "DOCUSIGN_CREATE_ENVELOPE_FROM_TEMPLATE",
    CREATE_GROUPS_FOR_ACCOUNT: "DOCUSIGN_CREATE_GROUPS_FOR_ACCOUNT",
    CREATE_IDPROOF_RESOURCE_TOKEN_FOR_RECIPIENT: "DOCUSIGN_CREATE_IDPROOF_RESOURCE_TOKEN_FOR_RECIPIENT",
    CREATE_NEW_ACCOUNT_PERMISSION_PROFILE: "DOCUSIGN_CREATE_NEW_ACCOUNT_PERMISSION_PROFILE",
    CREATE_NOTARY_JURISDICTION_OBJECT: "DOCUSIGN_CREATE_NOTARY_JURISDICTION_OBJECT",
    CREATE_OR_UPDATE_USER_AUTHORIZATIONS: "DOCUSIGN_CREATE_OR_UPDATE_USER_AUTHORIZATIONS",
    CREATE_PREVIEW_OF_RESPONSIVE_HTML: "DOCUSIGN_CREATE_PREVIEW_OF_RESPONSIVE_HTML",
    CREATE_PREVIEW_OF_RESPONSIVE_HTML_IN_ENVELOPE: "DOCUSIGN_CREATE_PREVIEW_OF_RESPONSIVE_HTML_IN_ENVELOPE",
    CREATE_RECIPIENT_MANUAL_REVIEW_LINK: "DOCUSIGN_CREATE_RECIPIENT_MANUAL_REVIEW_LINK",
    CREATE_RECIPIENT_PREVIEW_FOR_ENVELOPE: "DOCUSIGN_CREATE_RECIPIENT_PREVIEW_FOR_ENVELOPE",
    CREATE_RECIPIENT_VIEW_URL: "DOCUSIGN_CREATE_RECIPIENT_VIEW_URL",
    CREATE_SENDER_VIEW_URL_FOR_ENVELOPE: "DOCUSIGN_CREATE_SENDER_VIEW_URL_FOR_ENVELOPE",
    CREATE_SIGNING_GROUP: "DOCUSIGN_CREATE_SIGNING_GROUP",
    CREATE_TEMPLATE_DOCUMENT_CUSTOM_FIELDS: "DOCUSIGN_CREATE_TEMPLATE_DOCUMENT_CUSTOM_FIELDS",
    CREATE_TEMPLATE_RECIPIENT_PREVIEW_URL: "DOCUSIGN_CREATE_TEMPLATE_RECIPIENT_PREVIEW_URL",
    CREATE_TEMPLATE_RESPONSIVE_HTML_PREVIEW: "DOCUSIGN_CREATE_TEMPLATE_RESPONSIVE_HTML_PREVIEW",
    CREATE_TEMPLATE_WORKFLOW_STEP: "DOCUSIGN_CREATE_TEMPLATE_WORKFLOW_STEP",
    CREATE_URLFOR_TEMPLATE_EDIT_VIEW: "DOCUSIGN_CREATE_URLFOR_TEMPLATE_EDIT_VIEW",
    CREATE_USER_AUTHORIZATION_FOR_AGENT_USER: "DOCUSIGN_CREATE_USER_AUTHORIZATION_FOR_AGENT_USER",
    DELETE_ACCOUNT_BRAND: "DOCUSIGN_DELETE_ACCOUNT_BRAND",
    DELETE_ACCOUNT_CUSTOM_FIELD: "DOCUSIGN_DELETE_ACCOUNT_CUSTOM_FIELD",
    DELETE_ACCOUNT_PERMISSION_PROFILE: "DOCUSIGN_DELETE_ACCOUNT_PERMISSION_PROFILE",
    DELETE_ACCOUNT_STAMP: "DOCUSIGN_DELETE_ACCOUNT_STAMP",
    DELETE_BCC_EMAIL_ARCHIVE_CONFIGURATION: "DOCUSIGN_DELETE_BCC_EMAIL_ARCHIVE_CONFIGURATION",
    DELETE_BRAND_FROM_GROUP: "DOCUSIGN_DELETE_BRAND_FROM_GROUP",
    DELETE_BRAND_LOGO_BY_TYPE: "DOCUSIGN_DELETE_BRAND_LOGO_BY_TYPE",
    DELETE_BRAND_PROFILES: "DOCUSIGN_DELETE_BRAND_PROFILES",
    DELETE_CHUNKED_UPLOAD: "DOCUSIGN_DELETE_CHUNKED_UPLOAD",
    DELETE_CONNECT_CONFIGURATION: "DOCUSIGN_DELETE_CONNECT_CONFIGURATION",
    DELETE_CONNECT_FAILURE_LOG_ENTRY: "DOCUSIGN_DELETE_CONNECT_FAILURE_LOG_ENTRY",
    DELETE_CONNECT_LOG_ENTRIES: "DOCUSIGN_DELETE_CONNECT_LOG_ENTRIES",
    DELETE_CONNECT_OAUTH_CONFIGURATION: "DOCUSIGN_DELETE_CONNECT_OAUTH_CONFIGURATION",
    DELETE_CONTACT_FROM_ACCOUNT: "DOCUSIGN_DELETE_CONTACT_FROM_ACCOUNT",
    DELETE_CUSTOM_DOCUMENT_FIELDS: "DOCUSIGN_DELETE_CUSTOM_DOCUMENT_FIELDS",
    DELETE_CUSTOM_DOCUMENT_FIELDS_FROM_TEMPLATE: "DOCUSIGN_DELETE_CUSTOM_DOCUMENT_FIELDS_FROM_TEMPLATE",
    DELETE_CUSTOM_FIELDS_IN_TEMPLATE: "DOCUSIGN_DELETE_CUSTOM_FIELDS_IN_TEMPLATE",
    DELETE_CUSTOM_TAB_INFORMATION: "DOCUSIGN_DELETE_CUSTOM_TAB_INFORMATION",
    DELETE_CUSTOM_USER_SETTINGS: "DOCUSIGN_DELETE_CUSTOM_USER_SETTINGS",
    DELETE_DELAYED_ROUTING_RULES_FOR_TEMPLATE: "DOCUSIGN_DELETE_DELAYED_ROUTING_RULES_FOR_TEMPLATE",
    DELETE_DELAYED_ROUTING_RULE_FOR_ENVELOPE_STEP: "DOCUSIGN_DELETE_DELAYED_ROUTING_RULE_FOR_ENVELOPE_STEP",
    DELETE_DRAFT_ENVELOPE_ATTACHMENTS: "DOCUSIGN_DELETE_DRAFT_ENVELOPE_ATTACHMENTS",
    DELETE_ENOTE_CONFIGURATION: "DOCUSIGN_DELETE_ENOTE_CONFIGURATION",
    DELETE_ENVELOPE_CUSTOM_FIELDS: "DOCUSIGN_DELETE_ENVELOPE_CUSTOM_FIELDS",
    DELETE_ENVELOPE_EMAIL_SETTINGS: "DOCUSIGN_DELETE_ENVELOPE_EMAIL_SETTINGS",
    DELETE_ENVELOPE_LOCK: "DOCUSIGN_DELETE_ENVELOPE_LOCK",
    DELETE_ENVELOPE_SCHEDULED_SENDING_RULES: "DOCUSIGN_DELETE_ENVELOPE_SCHEDULED_SENDING_RULES",
    DELETE_ENVELOPE_TRANSFER_RULE: "DOCUSIGN_DELETE_ENVELOPE_TRANSFER_RULE",
    DELETE_ENVELOPE_WORKFLOW_DEFINITION: "DOCUSIGN_DELETE_ENVELOPE_WORKFLOW_DEFINITION",
    DELETE_ENVELOPE_WORKFLOW_STEP: "DOCUSIGN_DELETE_ENVELOPE_WORKFLOW_STEP",
    DELETE_EXISTING_WORKSPACE_LOGICALLY: "DOCUSIGN_DELETE_EXISTING_WORKSPACE_LOGICALLY",
    DELETE_MEMBERS_FROM_SIGNING_GROUP: "DOCUSIGN_DELETE_MEMBERS_FROM_SIGNING_GROUP",
    DELETE_MULTIPLE_CONTACTS_FROM_ACCOUNT: "DOCUSIGN_DELETE_MULTIPLE_CONTACTS_FROM_ACCOUNT",
    DELETE_NOTARY_JURISDICTION: "DOCUSIGN_DELETE_NOTARY_JURISDICTION",
    DELETE_ONE_OR_MORE_SIGNING_GROUPS: "DOCUSIGN_DELETE_ONE_OR_MORE_SIGNING_GROUPS",
    DELETE_PAGE_FROM_DOCUMENT_IN_ENVELOPE: "DOCUSIGN_DELETE_PAGE_FROM_DOCUMENT_IN_ENVELOPE",
    DELETE_PAGE_FROM_TEMPLATE_DOCUMENT: "DOCUSIGN_DELETE_PAGE_FROM_TEMPLATE_DOCUMENT",
    DELETE_POWER_FORM_BY_ID: "DOCUSIGN_DELETE_POWER_FORM_BY_ID",
    DELETE_REQUEST_LOG_FILES: "DOCUSIGN_DELETE_REQUEST_LOG_FILES",
    DELETE_SCHEDULED_SENDING_RULES: "DOCUSIGN_DELETE_SCHEDULED_SENDING_RULES",
    DELETE_SIGNATURE_FOR_CAPTIVE_RECIPIENTS: "DOCUSIGN_DELETE_SIGNATURE_FOR_CAPTIVE_RECIPIENTS",
    DELETE_SPECIFIC_CONNECT_LOG_ENTRY: "DOCUSIGN_DELETE_SPECIFIC_CONNECT_LOG_ENTRY",
    DELETE_SPECIFIED_ACCOUNT: "DOCUSIGN_DELETE_SPECIFIED_ACCOUNT",
    DELETE_STAMP_IMAGE_FOR_ACCOUNT: "DOCUSIGN_DELETE_STAMP_IMAGE_FOR_ACCOUNT",
    DELETE_TEMPLATE_FROM_ENVELOPE_DOCUMENT: "DOCUSIGN_DELETE_TEMPLATE_FROM_ENVELOPE_DOCUMENT",
    DELETE_TEMPLATE_LOCK: "DOCUSIGN_DELETE_TEMPLATE_LOCK",
    DELETE_TEMPLATE_WORKFLOW_STEP: "DOCUSIGN_DELETE_TEMPLATE_WORKFLOW_STEP",
    DELETE_USERS_FROM_GROUP: "DOCUSIGN_DELETE_USERS_FROM_GROUP",
    DELETE_USER_AUTHORIZATION: "DOCUSIGN_DELETE_USER_AUTHORIZATION",
    DELETE_USER_AUTHORIZATIONS: "DOCUSIGN_DELETE_USER_AUTHORIZATIONS",
    DELETE_USER_AUTH_FOR_CLOUD_STORAGE_PROVIDERS: "DOCUSIGN_DELETE_USER_AUTH_FOR_CLOUD_STORAGE_PROVIDERS",
    DELETE_USER_CLOUD_STORAGE_AUTHENTICATION: "DOCUSIGN_DELETE_USER_CLOUD_STORAGE_AUTHENTICATION",
    DELETE_USER_GROUP: "DOCUSIGN_DELETE_USER_GROUP",
    DELETE_USER_INITIALS_OR_SIGNATURE_IMAGE: "DOCUSIGN_DELETE_USER_INITIALS_OR_SIGNATURE_IMAGE",
    DELETE_USER_PROFILE_IMAGE: "DOCUSIGN_DELETE_USER_PROFILE_IMAGE",
    DELETE_WORKFLOW_DEFINITION_FOR_TEMPLATE: "DOCUSIGN_DELETE_WORKFLOW_DEFINITION_FOR_TEMPLATE",
    DELETE_WORKSPACE_FILES_OR_FOLDERS: "DOCUSIGN_DELETE_WORKSPACE_FILES_OR_FOLDERS",
    DEPRECATED_ENDPOINT_FOR_TAB_BLOB: "DOCUSIGN_DEPRECATED_ENDPOINT_FOR_TAB_BLOB",
    EXPORT_BRAND_TO_XMLFILE: "DOCUSIGN_EXPORT_BRAND_TO_XMLFILE",
    FETCH_RECIPIENT_NAMES_FOR_EMAIL: "DOCUSIGN_FETCH_RECIPIENT_NAMES_FOR_EMAIL",
    GENERATE_EDIT_VIEW_URLFOR_ENVELOPE: "DOCUSIGN_GENERATE_EDIT_VIEW_URLFOR_ENVELOPE",
    GENERATE_ENVELOPE_CORRECTION_URL: "DOCUSIGN_GENERATE_ENVELOPE_CORRECTION_URL",
    GENERATE_RECIPIENT_SHARED_VIEW_URL: "DOCUSIGN_GENERATE_RECIPIENT_SHARED_VIEW_URL",
    GET_ACCOUNT_BILLING_CHARGES_LIST: "DOCUSIGN_GET_ACCOUNT_BILLING_CHARGES_LIST",
    GET_ACCOUNT_PASSWORD_RULES: "DOCUSIGN_GET_ACCOUNT_PASSWORD_RULES",
    GET_ACCOUNT_STAMP_IMAGE: "DOCUSIGN_GET_ACCOUNT_STAMP_IMAGE",
    GET_AGENT_USER_AUTHORIZATIONS: "DOCUSIGN_GET_AGENT_USER_AUTHORIZATIONS",
    GET_API_REQUEST_LOG_SETTINGS: "DOCUSIGN_GET_API_REQUEST_LOG_SETTINGS",
    GET_BCCEMAIL_ARCHIVE_HISTORY: "DOCUSIGN_GET_BCCEMAIL_ARCHIVE_HISTORY",
    GET_BCC_EMAIL_ARCHIVE_CONFIGURATIONS: "DOCUSIGN_GET_BCC_EMAIL_ARCHIVE_CONFIGURATIONS",
    GET_BRANDS_INFORMATION_FOR_GROUP: "DOCUSIGN_GET_BRANDS_INFORMATION_FOR_GROUP",
    GET_BRAND_LOGO_BY_TYPE: "DOCUSIGN_GET_BRAND_LOGO_BY_TYPE",
    GET_BULK_SEND_BATCH_STATUS: "DOCUSIGN_GET_BULK_SEND_BATCH_STATUS",
    GET_BULK_SEND_LISTS_INFORMATION: "DOCUSIGN_GET_BULK_SEND_LISTS_INFORMATION",
    GET_DEFAULT_CONSUMER_DISCLOSURE: "DOCUSIGN_GET_DEFAULT_CONSUMER_DISCLOSURE",
    GET_DOCUSIGN_ACCOUNT_CONTACTS: "DOCUSIGN_GET_DOCUSIGN_ACCOUNT_CONTACTS",
    GET_DOWNGRADE_PLAN_INFO_FOR_ACCOUNT: "DOCUSIGN_GET_DOWNGRADE_PLAN_INFO_FOR_ACCOUNT",
    GET_ELECTRONIC_DISCLOSURE_FOR_RECIPIENT: "DOCUSIGN_GET_ELECTRONIC_DISCLOSURE_FOR_RECIPIENT",
    GET_ELECTRONIC_RECORD_AND_SIGNATURE_DISCLOSURE: "DOCUSIGN_GET_ELECTRONIC_RECORD_AND_SIGNATURE_DISCLOSURE",
    GET_ENOTE_INTEGRATION_SETTINGS: "DOCUSIGN_GET_ENOTE_INTEGRATION_SETTINGS",
    GET_ENVELOPE: "DOCUSIGN_GET_ENVELOPE",
    GET_ENVELOPE_DOCUMENT_FIELDS: "DOCUSIGN_GET_ENVELOPE_DOCUMENT_FIELDS",
    GET_ENVELOPE_DOC_GEN_FORM_FIELDS: "DOCUSIGN_GET_ENVELOPE_DOC_GEN_FORM_FIELDS",
    GET_ENVELOPE_LOCK_INFORMATION: "DOCUSIGN_GET_ENVELOPE_LOCK_INFORMATION",
    GET_ENVELOPE_NOTIFICATION_DEFAULTS: "DOCUSIGN_GET_ENVELOPE_NOTIFICATION_DEFAULTS",
    GET_ENVELOPE_WORKFLOW_DEFINITION: "DOCUSIGN_GET_ENVELOPE_WORKFLOW_DEFINITION",
    GET_ENVELOPE_WORKFLOW_DELAYED_ROUTING_DEFINITION: "DOCUSIGN_GET_ENVELOPE_WORKFLOW_DELAYED_ROUTING_DEFINITION",
    GET_GROUPS_INFORMATION_FOR_ACCOUNT: "DOCUSIGN_GET_GROUPS_INFORMATION_FOR_ACCOUNT",
    GET_LIST_OF_ACCOUNT_PERMISSION_PROFILES: "DOCUSIGN_GET_LIST_OF_ACCOUNT_PERMISSION_PROFILES",
    GET_MEMBERS_OF_A_SIGNING_GROUP: "DOCUSIGN_GET_MEMBERS_OF_A_SIGNING_GROUP",
    GET_NOTARY_JURISDICTIONS_FOR_USER: "DOCUSIGN_GET_NOTARY_JURISDICTIONS_FOR_USER",
    GET_NOTARY_JURISDICTION_OBJECT: "DOCUSIGN_GET_NOTARY_JURISDICTION_OBJECT",
    GET_NOTARY_USER_SETTINGS: "DOCUSIGN_GET_NOTARY_USER_SETTINGS",
    GET_ORIGINAL_HTML_DEFINITION_FOR_TEMPLATE: "DOCUSIGN_GET_ORIGINAL_HTML_DEFINITION_FOR_TEMPLATE",
    GET_PAGE_IMAGE_FROM_ENVELOPE: "DOCUSIGN_GET_PAGE_IMAGE_FROM_ENVELOPE",
    GET_PDFTRANSCRIPT_OF_ENVELOPE_COMMENTS: "DOCUSIGN_GET_PDFTRANSCRIPT_OF_ENVELOPE_COMMENTS",
    GET_PERMISSION_PROFILE_FOR_ACCOUNT: "DOCUSIGN_GET_PERMISSION_PROFILE_FOR_ACCOUNT",
    GET_POWER_FORM_SENDERS: "DOCUSIGN_GET_POWER_FORM_SENDERS",
    GET_RECIPIENT_DOCUMENT_VISIBILITY: "DOCUSIGN_GET_RECIPIENT_DOCUMENT_VISIBILITY",
    GET_REQUEST_LOGGING_LOG_FILE: "DOCUSIGN_GET_REQUEST_LOGGING_LOG_FILE",
    GET_SIGNATURE_INFORMATION_FOR_RECIPIENT: "DOCUSIGN_GET_SIGNATURE_INFORMATION_FOR_RECIPIENT",
    GET_SIGNING_GROUP_INFORMATION: "DOCUSIGN_GET_SIGNING_GROUP_INFORMATION",
    GET_SPECIFIC_BRAND_RESOURCE_FILE: "DOCUSIGN_GET_SPECIFIC_BRAND_RESOURCE_FILE",
    GET_SPECIFIED_ACCOUNT_STAMP_INFO: "DOCUSIGN_GET_SPECIFIED_ACCOUNT_STAMP_INFO",
    GET_SUPPORTED_LANGUAGES_FOR_RECIPIENTS: "DOCUSIGN_GET_SUPPORTED_LANGUAGES_FOR_RECIPIENTS",
    GET_TABS_BLOB_FOR_ENVELOPE: "DOCUSIGN_GET_TABS_BLOB_FOR_ENVELOPE",
    GET_TAB_SETTINGS_FOR_ACCOUNT: "DOCUSIGN_GET_TAB_SETTINGS_FOR_ACCOUNT",
    GET_TEMPLATE: "DOCUSIGN_GET_TEMPLATE",
    GET_TEMPLATES_FOR_ENVELOPE_DOCUMENT: "DOCUSIGN_GET_TEMPLATES_FOR_ENVELOPE_DOCUMENT",
    GET_TEMPLATES_USED_IN_ENVELOPE: "DOCUSIGN_GET_TEMPLATES_USED_IN_ENVELOPE",
    GET_TEMPLATE_DELAYED_ROUTING_RULES: "DOCUSIGN_GET_TEMPLATE_DELAYED_ROUTING_RULES",
    GET_TEMPLATE_DOCUMENT_HTML_DEFINITION: "DOCUSIGN_GET_TEMPLATE_DOCUMENT_HTML_DEFINITION",
    GET_TEMPLATE_LOCK_INFORMATION: "DOCUSIGN_GET_TEMPLATE_LOCK_INFORMATION",
    GET_TEMPLATE_NOTIFICATION_INFORMATION: "DOCUSIGN_GET_TEMPLATE_NOTIFICATION_INFORMATION",
    GET_TEMPLATE_RECIPIENT_DOCUMENT_VISIBILITY: "DOCUSIGN_GET_TEMPLATE_RECIPIENT_DOCUMENT_VISIBILITY",
    GET_TEMPLATE_SCHEDULED_SENDING_SETTINGS: "DOCUSIGN_GET_TEMPLATE_SCHEDULED_SENDING_SETTINGS",
    GET_UNSUPPORTED_FILE_TYPES_LIST: "DOCUSIGN_GET_UNSUPPORTED_FILE_TYPES_LIST",
    GET_URLFOR_EMBEDDING_DOCUSIGN_UI: "DOCUSIGN_GET_URLFOR_EMBEDDING_DOCUSIGN_UI",
    GET_USERS_IN_GROUP: "DOCUSIGN_GET_USERS_IN_GROUP",
    GET_USER_AUTHORIZATIONS_FOR_PRINCIPAL: "DOCUSIGN_GET_USER_AUTHORIZATIONS_FOR_PRINCIPAL",
    GET_USER_AUTHORIZATION_DETAILS: "DOCUSIGN_GET_USER_AUTHORIZATION_DETAILS",
    GET_USER_CLOUD_STORAGE_PROVIDER_CONFIGURATION: "DOCUSIGN_GET_USER_CLOUD_STORAGE_PROVIDER_CONFIGURATION",
    GET_USER_INFORMATION_BY_ID: "DOCUSIGN_GET_USER_INFORMATION_BY_ID",
    GET_WATERMARK_DETAILS_FOR_ACCOUNT: "DOCUSIGN_GET_WATERMARK_DETAILS_FOR_ACCOUNT",
    GET_WORKFLOW_STEP_FOR_ENVELOPE: "DOCUSIGN_GET_WORKFLOW_STEP_FOR_ENVELOPE",
    GET_WORKSPACE_FILE: "DOCUSIGN_GET_WORKSPACE_FILE",
    GET_WORKSPACE_FOLDER_CONTENTS: "DOCUSIGN_GET_WORKSPACE_FOLDER_CONTENTS",
    INITIATE_NEW_CHUNKED_UPLOAD: "DOCUSIGN_INITIATE_NEW_CHUNKED_UPLOAD",
    LIST_ALL_TEMPLATES: "DOCUSIGN_LIST_ALL_TEMPLATES",
    LIST_BILLING_INVOICES: "DOCUSIGN_LIST_BILLING_INVOICES",
    LIST_BRANDS_FOR_ACCOUNT: "DOCUSIGN_LIST_BRANDS_FOR_ACCOUNT",
    LIST_BULK_SEND_BATCH_SUMMARIES: "DOCUSIGN_LIST_BULK_SEND_BATCH_SUMMARIES",
    LIST_CLOUD_STORAGE_ITEMS: "DOCUSIGN_LIST_CLOUD_STORAGE_ITEMS",
    LIST_ENVELOPE_AND_DOCUMENT_CUSTOM_FIELDS: "DOCUSIGN_LIST_ENVELOPE_AND_DOCUMENT_CUSTOM_FIELDS",
    LIST_ENVELOPE_ATTACHMENTS_BY_ENVELOPE_ID: "DOCUSIGN_LIST_ENVELOPE_ATTACHMENTS_BY_ENVELOPE_ID",
    LIST_PAST_DUE_INVOICES: "DOCUSIGN_LIST_PAST_DUE_INVOICES",
    LIST_PAYMENT_GATEWAY_ACCOUNTS_INFO: "DOCUSIGN_LIST_PAYMENT_GATEWAY_ACCOUNTS_INFO",
    LIST_REGISTERED_NOTARY_JURISDICTIONS: "DOCUSIGN_LIST_REGISTERED_NOTARY_JURISDICTIONS",
    LIST_RESOURCES_FOR_RESTVERSION_SPECIFIED: "DOCUSIGN_LIST_RESOURCES_FOR_RESTVERSION_SPECIFIED",
    LIST_SIGNATURE_PROVIDERS_FOR_ACCOUNT: "DOCUSIGN_LIST_SIGNATURE_PROVIDERS_FOR_ACCOUNT",
    LIST_USERS_FOR_ACCOUNT: "DOCUSIGN_LIST_USERS_FOR_ACCOUNT",
    LIST_WORKSPACES: "DOCUSIGN_LIST_WORKSPACES",
    LIST_WORKSPACE_FILE_PAGES: "DOCUSIGN_LIST_WORKSPACE_FILE_PAGES",
    LOCK_AN_ENVELOPE_FOR_EDITING: "DOCUSIGN_LOCK_AN_ENVELOPE_FOR_EDITING",
    LOCK_TEMPLATE_FOR_EDITING: "DOCUSIGN_LOCK_TEMPLATE_FOR_EDITING",
    MODIFY_ACCOUNT_NOTIFICATION_SETTINGS: "DOCUSIGN_MODIFY_ACCOUNT_NOTIFICATION_SETTINGS",
    MODIFY_TAB_SETTINGS_FOR_ACCOUNT: "DOCUSIGN_MODIFY_TAB_SETTINGS_FOR_ACCOUNT",
    POST_PAYMENT_TO_INVOICE: "DOCUSIGN_POST_PAYMENT_TO_INVOICE",
    PURCHASE_ADDITIONAL_ENVELOPES: "DOCUSIGN_PURCHASE_ADDITIONAL_ENVELOPES",
    QUEUE_DOWNGRADE_BILLING_PLAN_REQUEST: "DOCUSIGN_QUEUE_DOWNGRADE_BILLING_PLAN_REQUEST",
    REGISTER_CURRENT_USER_AS_NOTARY: "DOCUSIGN_REGISTER_CURRENT_USER_AS_NOTARY",
    REMOVE_BULK_SEND_LIST: "DOCUSIGN_REMOVE_BULK_SEND_LIST",
    REMOVE_GROUP_SHARING_PERMISSIONS_FOR_TEMPLATE: "DOCUSIGN_REMOVE_GROUP_SHARING_PERMISSIONS_FOR_TEMPLATE",
    REMOVE_TEMPLATES_FROM_FAVORITES: "DOCUSIGN_REMOVE_TEMPLATES_FROM_FAVORITES",
    REMOVE_USER_SIGNATURE_INFORMATION: "DOCUSIGN_REMOVE_USER_SIGNATURE_INFORMATION",
    REPUBLISH_CONNECT_DATA_FOR_ENVELOPE: "DOCUSIGN_REPUBLISH_CONNECT_DATA_FOR_ENVELOPE",
    REPUBLISH_CONNECT_INFO_FOR_ENVELOPES: "DOCUSIGN_REPUBLISH_CONNECT_INFO_FOR_ENVELOPES",
    RETRIEVE_ACCOUNT_BILLING_PLAN: "DOCUSIGN_RETRIEVE_ACCOUNT_BILLING_PLAN",
    RETRIEVE_ACCOUNT_BRAND_INFORMATION: "DOCUSIGN_RETRIEVE_ACCOUNT_BRAND_INFORMATION",
    RETRIEVE_ACCOUNT_ENVELOPE_PURGE_CONFIG: "DOCUSIGN_RETRIEVE_ACCOUNT_ENVELOPE_PURGE_CONFIG",
    RETRIEVE_ACCOUNT_FAVORITE_TEMPLATES: "DOCUSIGN_RETRIEVE_ACCOUNT_FAVORITE_TEMPLATES",
    RETRIEVE_ACCOUNT_INFORMATION_FOR_SPECIFIED_ACCOUNT: "DOCUSIGN_RETRIEVE_ACCOUNT_INFORMATION_FOR_SPECIFIED_ACCOUNT",
    RETRIEVE_ACCOUNT_PASSWORD_RULES: "DOCUSIGN_RETRIEVE_ACCOUNT_PASSWORD_RULES",
    RETRIEVE_ACCOUNT_PROVISIONING_INFO: "DOCUSIGN_RETRIEVE_ACCOUNT_PROVISIONING_INFO",
    RETRIEVE_ACCOUNT_SEAL_PROVIDERS: "DOCUSIGN_RETRIEVE_ACCOUNT_SEAL_PROVIDERS",
    RETRIEVE_ACCOUNT_SETTINGS_INFORMATION: "DOCUSIGN_RETRIEVE_ACCOUNT_SETTINGS_INFORMATION",
    RETRIEVE_ACCOUNT_SIGNING_GROUPS: "DOCUSIGN_RETRIEVE_ACCOUNT_SIGNING_GROUPS",
    RETRIEVE_ACCOUNT_VERIFICATION_WORKFLOWS: "DOCUSIGN_RETRIEVE_ACCOUNT_VERIFICATION_WORKFLOWS",
    RETRIEVE_ALL_ACCOUNT_TABS: "DOCUSIGN_RETRIEVE_ALL_ACCOUNT_TABS",
    RETRIEVE_APIREQUEST_LOG_FILES: "DOCUSIGN_RETRIEVE_APIREQUEST_LOG_FILES",
    RETRIEVE_AVAILABLE_ACCOUNT_STAMPS: "DOCUSIGN_RETRIEVE_AVAILABLE_ACCOUNT_STAMPS",
    RETRIEVE_BILLING_INVOICE: "DOCUSIGN_RETRIEVE_BILLING_INVOICE",
    RETRIEVE_BILLING_PAYMENT_INFO: "DOCUSIGN_RETRIEVE_BILLING_PAYMENT_INFO",
    RETRIEVE_BILLING_PLAN_DETAILS: "DOCUSIGN_RETRIEVE_BILLING_PLAN_DETAILS",
    RETRIEVE_CHUNKED_UPLOAD_METADATA: "DOCUSIGN_RETRIEVE_CHUNKED_UPLOAD_METADATA",
    RETRIEVE_CLOUD_STORAGE_PROVIDER_CONFIGURATION: "DOCUSIGN_RETRIEVE_CLOUD_STORAGE_PROVIDER_CONFIGURATION",
    RETRIEVE_CONNECT_CONFIGURATIONS: "DOCUSIGN_RETRIEVE_CONNECT_CONFIGURATIONS",
    RETRIEVE_CONNECT_CONFIGURATION_DETAILS: "DOCUSIGN_RETRIEVE_CONNECT_CONFIGURATION_DETAILS",
    RETRIEVE_CONNECT_FAILURE_LOGS: "DOCUSIGN_RETRIEVE_CONNECT_FAILURE_LOGS",
    RETRIEVE_CONNECT_LOGS: "DOCUSIGN_RETRIEVE_CONNECT_LOGS",
    RETRIEVE_CONNECT_LOG_ENTRY: "DOCUSIGN_RETRIEVE_CONNECT_LOG_ENTRY",
    RETRIEVE_CONNECT_OAUTH_CONFIGURATION: "DOCUSIGN_RETRIEVE_CONNECT_OAUTH_CONFIGURATION",
    RETRIEVE_CREDIT_CARD_INFO: "DOCUSIGN_RETRIEVE_CREDIT_CARD_INFO",
    RETRIEVE_CUSTOM_FIELDS_FOR_ENVELOPE: "DOCUSIGN_RETRIEVE_CUSTOM_FIELDS_FOR_ENVELOPE",
    RETRIEVE_CUSTOM_FIELDS_FOR_TEMPLATE: "DOCUSIGN_RETRIEVE_CUSTOM_FIELDS_FOR_TEMPLATE",
    RETRIEVE_CUSTOM_TAB_INFORMATION: "DOCUSIGN_RETRIEVE_CUSTOM_TAB_INFORMATION",
    RETRIEVE_CUSTOM_USER_SETTINGS: "DOCUSIGN_RETRIEVE_CUSTOM_USER_SETTINGS",
    RETRIEVE_DEFAULT_DISCLOSURE_FOR_ENVELOPE: "DOCUSIGN_RETRIEVE_DEFAULT_DISCLOSURE_FOR_ENVELOPE",
    RETRIEVE_ENVELOPE_ATTACHMENT: "DOCUSIGN_RETRIEVE_ENVELOPE_ATTACHMENT",
    RETRIEVE_ENVELOPE_AUDIT_EVENTS: "DOCUSIGN_RETRIEVE_ENVELOPE_AUDIT_EVENTS",
    RETRIEVE_ENVELOPE_DOCUMENTS: "DOCUSIGN_RETRIEVE_ENVELOPE_DOCUMENTS",
    RETRIEVE_ENVELOPE_DOCUMENT_FIELDS: "DOCUSIGN_RETRIEVE_ENVELOPE_DOCUMENT_FIELDS",
    RETRIEVE_ENVELOPE_EMAIL_OVERRIDES: "DOCUSIGN_RETRIEVE_ENVELOPE_EMAIL_OVERRIDES",
    RETRIEVE_ENVELOPE_HTML_DEFINITION: "DOCUSIGN_RETRIEVE_ENVELOPE_HTML_DEFINITION",
    RETRIEVE_ENVELOPE_NOTIFICATION_DETAILS: "DOCUSIGN_RETRIEVE_ENVELOPE_NOTIFICATION_DETAILS",
    RETRIEVE_HTMLDEFINITION_FOR_RESPONSIVE_DOCUMENT: "DOCUSIGN_RETRIEVE_HTMLDEFINITION_FOR_RESPONSIVE_DOCUMENT",
    RETRIEVE_ITEMS_IN_CLOUD_STORAGE_FOLDER: "DOCUSIGN_RETRIEVE_ITEMS_IN_CLOUD_STORAGE_FOLDER",
    RETRIEVE_LIST_OF_BILLING_PLANS: "DOCUSIGN_RETRIEVE_LIST_OF_BILLING_PLANS",
    RETRIEVE_PAYMENT_INFORMATION_LIST: "DOCUSIGN_RETRIEVE_PAYMENT_INFORMATION_LIST",
    RETRIEVE_PDF_FROM_SPECIFIED_TEMPLATE: "DOCUSIGN_RETRIEVE_PDF_FROM_SPECIFIED_TEMPLATE",
    RETRIEVE_POWER_FORM_DATA_ENTRIES: "DOCUSIGN_RETRIEVE_POWER_FORM_DATA_ENTRIES",
    RETRIEVE_PROPERTIES_ABOUT_WORKSPACE: "DOCUSIGN_RETRIEVE_PROPERTIES_ABOUT_WORKSPACE",
    RETRIEVE_REST_API_VERSIONS: "DOCUSIGN_RETRIEVE_REST_API_VERSIONS",
    RETRIEVE_SHARED_ITEM_STATUS_FOR_USERS: "DOCUSIGN_RETRIEVE_SHARED_ITEM_STATUS_FOR_USERS",
    RETRIEVE_SIGNER_SIGNATURE_IMAGE_INFORMATION: "DOCUSIGN_RETRIEVE_SIGNER_SIGNATURE_IMAGE_INFORMATION",
    RETRIEVE_TEMPLATE_CUSTOM_FIELDS: "DOCUSIGN_RETRIEVE_TEMPLATE_CUSTOM_FIELDS",
    RETRIEVE_TEMPLATE_DOCUMENT_PAGE_IMAGES: "DOCUSIGN_RETRIEVE_TEMPLATE_DOCUMENT_PAGE_IMAGES",
    RETRIEVE_TEMPLATE_PAGE_IMAGE: "DOCUSIGN_RETRIEVE_TEMPLATE_PAGE_IMAGE",
    RETRIEVE_USER_ACCOUNT_SETTINGS: "DOCUSIGN_RETRIEVE_USER_ACCOUNT_SETTINGS",
    RETRIEVE_USER_INITIALS_IMAGE_FOR_ENVELOPES: "DOCUSIGN_RETRIEVE_USER_INITIALS_IMAGE_FOR_ENVELOPES",
    RETRIEVE_USER_PROFILE_IMAGE: "DOCUSIGN_RETRIEVE_USER_PROFILE_IMAGE",
    RETRIEVE_USER_PROFILE_INFORMATION: "DOCUSIGN_RETRIEVE_USER_PROFILE_INFORMATION",
    RETRIEVE_USER_SIGNATURE_DEFINITIONS: "DOCUSIGN_RETRIEVE_USER_SIGNATURE_DEFINITIONS",
    RETRIEVE_USER_SIGNATURE_IMAGE: "DOCUSIGN_RETRIEVE_USER_SIGNATURE_IMAGE",
    RETRIEVE_USER_SIGNATURE_INFORMATION: "DOCUSIGN_RETRIEVE_USER_SIGNATURE_INFORMATION",
    RETRIEVE_WORKFLOW_STEP_FOR_TEMPLATE: "DOCUSIGN_RETRIEVE_WORKFLOW_STEP_FOR_TEMPLATE",
    RETURNS_DOCUMENT_PAGE_IMAGES_BASED_ON_INPUT: "DOCUSIGN_RETURNS_DOCUMENT_PAGE_IMAGES_BASED_ON_INPUT",
    RETURN_ALL_CONNECT_SERVICE_USERS: "DOCUSIGN_RETURN_ALL_CONNECT_SERVICE_USERS",
    RETURN_BRAND_RESOURCE_METADATA_FOR_ACCOUNT: "DOCUSIGN_RETURN_BRAND_RESOURCE_METADATA_FOR_ACCOUNT",
    RETURN_ENVELOPE_TAB_DATA_FOR_EXISTING_ENVELOPE: "DOCUSIGN_RETURN_ENVELOPE_TAB_DATA_FOR_EXISTING_ENVELOPE",
    RETURN_SCHEDULED_SENDING_RULES_FOR_ENVELOPE: "DOCUSIGN_RETURN_SCHEDULED_SENDING_RULES_FOR_ENVELOPE",
    RETURN_TEMPLATE_WORKFLOW_DEFINITION: "DOCUSIGN_RETURN_TEMPLATE_WORKFLOW_DEFINITION",
    RETURN_USERS_FROM_CONNECT_SERVICE: "DOCUSIGN_RETURN_USERS_FROM_CONNECT_SERVICE",
    REVOKE_ENVELOPE_CORRECTION_URL: "DOCUSIGN_REVOKE_ENVELOPE_CORRECTION_URL",
    ROTATE_PAGE_IMAGE_FOR_ENVELOPE: "DOCUSIGN_ROTATE_PAGE_IMAGE_FOR_ENVELOPE",
    ROTATE_TEMPLATE_PAGE_IMAGE: "DOCUSIGN_ROTATE_TEMPLATE_PAGE_IMAGE",
    SEND_ENVELOPE: "DOCUSIGN_SEND_ENVELOPE",
    SET_ENVELOPE_PURGE_CONFIG_FOR_ACCOUNT: "DOCUSIGN_SET_ENVELOPE_PURGE_CONFIG_FOR_ACCOUNT",
    SET_INITIALS_IMAGE_FOR_ACCOUNTLESS_SIGNER: "DOCUSIGN_SET_INITIALS_IMAGE_FOR_ACCOUNTLESS_SIGNER",
    SET_SHARED_ACCESS_FOR_USERS_AND_TEMPLATES: "DOCUSIGN_SET_SHARED_ACCESS_FOR_USERS_AND_TEMPLATES",
    SET_SIGNATURE_IMAGE_FOR_NO_ACCOUNT_SIGNER: "DOCUSIGN_SET_SIGNATURE_IMAGE_FOR_NO_ACCOUNT_SIGNER",
    SET_TEMPLATES_AS_ACCOUNT_FAVORITES: "DOCUSIGN_SET_TEMPLATES_AS_ACCOUNT_FAVORITES",
    SET_UP_CONNECT_OAUTH_CONFIGURATION: "DOCUSIGN_SET_UP_CONNECT_OAUTH_CONFIGURATION",
    SET_USER_SIGNATURE_IMAGE: "DOCUSIGN_SET_USER_SIGNATURE_IMAGE",
    SHARE_TEMPLATE_WITH_GROUP: "DOCUSIGN_SHARE_TEMPLATE_WITH_GROUP",
    SUBMIT_BATCH_HISTORICAL_ENVELOPES_TO_WEBHOOK: "DOCUSIGN_SUBMIT_BATCH_HISTORICAL_ENVELOPES_TO_WEBHOOK",
    TOGGLE_API_REQUEST_LOGGING: "DOCUSIGN_TOGGLE_API_REQUEST_LOGGING",
    UPDATE_ACCOUNT_BRAND_SETTINGS: "DOCUSIGN_UPDATE_ACCOUNT_BRAND_SETTINGS",
    UPDATE_ACCOUNT_CUSTOM_FIELD: "DOCUSIGN_UPDATE_ACCOUNT_CUSTOM_FIELD",
    UPDATE_ACCOUNT_PASSWORD_RULES: "DOCUSIGN_UPDATE_ACCOUNT_PASSWORD_RULES",
    UPDATE_ACCOUNT_SETTINGS: "DOCUSIGN_UPDATE_ACCOUNT_SETTINGS",
    UPDATE_ACCOUNT_STAMPS: "DOCUSIGN_UPDATE_ACCOUNT_STAMPS",
    UPDATE_ACCOUNT_STAMP_BY_ID: "DOCUSIGN_UPDATE_ACCOUNT_STAMP_BY_ID",
    UPDATE_ACCOUNT_WATERMARK_INFORMATION: "DOCUSIGN_UPDATE_ACCOUNT_WATERMARK_INFORMATION",
    UPDATE_ACCOUNT_WATERMARK_PREVIEW: "DOCUSIGN_UPDATE_ACCOUNT_WATERMARK_PREVIEW",
    UPDATE_BILLING_PLAN_FOR_ACCOUNT: "DOCUSIGN_UPDATE_BILLING_PLAN_FOR_ACCOUNT",
    UPDATE_BULK_SEND_BATCH_NAME: "DOCUSIGN_UPDATE_BULK_SEND_BATCH_NAME",
    UPDATE_CONNECT_OAUTH_CONFIGURATION: "DOCUSIGN_UPDATE_CONNECT_OAUTH_CONFIGURATION",
    UPDATE_CONTACTS_INFORMATION: "DOCUSIGN_UPDATE_CONTACTS_INFORMATION",
    UPDATE_CUSTOM_FIELDS_IN_ENVELOPE_DOCUMENT: "DOCUSIGN_UPDATE_CUSTOM_FIELDS_IN_ENVELOPE_DOCUMENT",
    UPDATE_CUSTOM_TAB_INFORMATION_FOR_ACCOUNT: "DOCUSIGN_UPDATE_CUSTOM_TAB_INFORMATION_FOR_ACCOUNT",
    UPDATE_DOCUMENT_VISIBILITY_RECIPIENTS: "DOCUSIGN_UPDATE_DOCUMENT_VISIBILITY_RECIPIENTS",
    UPDATE_DOCUSIGN_CONNECT_CONFIGURATION: "DOCUSIGN_UPDATE_DOCUSIGN_CONNECT_CONFIGURATION",
    UPDATE_ELECTRONIC_RECORD_DISCLOSURE: "DOCUSIGN_UPDATE_ELECTRONIC_RECORD_DISCLOSURE",
    UPDATE_ENOTE_INTEGRATION_CONFIG: "DOCUSIGN_UPDATE_ENOTE_INTEGRATION_CONFIG",
    UPDATE_ENVELOPE_ATTACHMENT: "DOCUSIGN_UPDATE_ENVELOPE_ATTACHMENT",
    UPDATE_ENVELOPE_CUSTOM_FIELDS: "DOCUSIGN_UPDATE_ENVELOPE_CUSTOM_FIELDS",
    UPDATE_ENVELOPE_DELAYED_ROUTING_RULES: "DOCUSIGN_UPDATE_ENVELOPE_DELAYED_ROUTING_RULES",
    UPDATE_ENVELOPE_DOC_GEN_FORM_FIELDS: "DOCUSIGN_UPDATE_ENVELOPE_DOC_GEN_FORM_FIELDS",
    UPDATE_ENVELOPE_EMAIL_SETTINGS: "DOCUSIGN_UPDATE_ENVELOPE_EMAIL_SETTINGS",
    UPDATE_ENVELOPE_NOTIFICATION_SETTINGS: "DOCUSIGN_UPDATE_ENVELOPE_NOTIFICATION_SETTINGS",
    UPDATE_ENVELOPE_SCHEDULED_SENDING_RULES: "DOCUSIGN_UPDATE_ENVELOPE_SCHEDULED_SENDING_RULES",
    UPDATE_ENVELOPE_WORKFLOW_DEFINITION: "DOCUSIGN_UPDATE_ENVELOPE_WORKFLOW_DEFINITION",
    UPDATE_ENVELOPE_WORKFLOW_STEP: "DOCUSIGN_UPDATE_ENVELOPE_WORKFLOW_STEP",
    UPDATE_GROUP_INFORMATION: "DOCUSIGN_UPDATE_GROUP_INFORMATION",
    UPDATE_LOCK_FOR_ENVELOPE: "DOCUSIGN_UPDATE_LOCK_FOR_ENVELOPE",
    UPDATE_NOTARY_JURISDICTION_INFO: "DOCUSIGN_UPDATE_NOTARY_JURISDICTION_INFO",
    UPDATE_PERMISSION_PROFILE_SETTINGS: "DOCUSIGN_UPDATE_PERMISSION_PROFILE_SETTINGS",
    UPDATE_RECIPIENT_DOCUMENT_VISIBILITY: "DOCUSIGN_UPDATE_RECIPIENT_DOCUMENT_VISIBILITY",
    UPDATE_SIGNING_GROUP_DETAILS: "DOCUSIGN_UPDATE_SIGNING_GROUP_DETAILS",
    UPDATE_SIGNING_GROUP_NAMES: "DOCUSIGN_UPDATE_SIGNING_GROUP_NAMES",
    UPDATE_TEMPLATE_CUSTOM_FIELDS: "DOCUSIGN_UPDATE_TEMPLATE_CUSTOM_FIELDS",
    UPDATE_TEMPLATE_DELAYED_ROUTING_RULES: "DOCUSIGN_UPDATE_TEMPLATE_DELAYED_ROUTING_RULES",
    UPDATE_TEMPLATE_DOCUMENT_CUSTOM_FIELDS: "DOCUSIGN_UPDATE_TEMPLATE_DOCUMENT_CUSTOM_FIELDS",
    UPDATE_TEMPLATE_DOC_VISIBILITY: "DOCUSIGN_UPDATE_TEMPLATE_DOC_VISIBILITY",
    UPDATE_TEMPLATE_LOCK_INFORMATION: "DOCUSIGN_UPDATE_TEMPLATE_LOCK_INFORMATION",
    UPDATE_TEMPLATE_NOTIFICATION_SETTINGS: "DOCUSIGN_UPDATE_TEMPLATE_NOTIFICATION_SETTINGS",
    UPDATE_TEMPLATE_RECIPIENT_DOCUMENT_VISIBILITY: "DOCUSIGN_UPDATE_TEMPLATE_RECIPIENT_DOCUMENT_VISIBILITY",
    UPDATE_TEMPLATE_SCHEDULED_SENDING_RULES: "DOCUSIGN_UPDATE_TEMPLATE_SCHEDULED_SENDING_RULES",
    UPDATE_TEMPLATE_WORKFLOW_DEFINITION: "DOCUSIGN_UPDATE_TEMPLATE_WORKFLOW_DEFINITION",
    UPDATE_USER_ACCOUNT_SETTINGS: "DOCUSIGN_UPDATE_USER_ACCOUNT_SETTINGS",
    UPDATE_USER_AUTHORIZATION_DATES: "DOCUSIGN_UPDATE_USER_AUTHORIZATION_DATES",
    UPDATE_USER_INFORMATION_FOR_SPECIFIED_USER: "DOCUSIGN_UPDATE_USER_INFORMATION_FOR_SPECIFIED_USER",
    UPDATE_USER_NOTARY_INFORMATION: "DOCUSIGN_UPDATE_USER_NOTARY_INFORMATION",
    UPDATE_USER_PROFILE_IMAGE: "DOCUSIGN_UPDATE_USER_PROFILE_IMAGE",
    UPDATE_USER_PROFILE_INFORMATION: "DOCUSIGN_UPDATE_USER_PROFILE_INFORMATION",
    UPDATE_USER_SIGNATURE_BY_ID: "DOCUSIGN_UPDATE_USER_SIGNATURE_BY_ID",
    UPDATE_WORKFLOW_STEP_FOR_TEMPLATE: "DOCUSIGN_UPDATE_WORKFLOW_STEP_FOR_TEMPLATE",
    UPDATE_WORKSPACE_INFORMATION: "DOCUSIGN_UPDATE_WORKSPACE_INFORMATION",
    UPDATE_WORKSPACE_ITEM_METADATA: "DOCUSIGN_UPDATE_WORKSPACE_ITEM_METADATA",
  },
  triggerTypes: {},
}
