// @composio/core/generated/elevenlabs.ts
/**
 * Map of Composio's ELEVENLABS toolkit.
 */
export const ELEVENLABS = {
  slug: "elevenlabs",
  tools: {
    ADD_A_PRONUNCIATION_DICTIONARY: "ELEVENLABS_ADD_A_PRONUNCIATION_DICTIONARY",
    ADD_PROJECT: "ELEVENLABS_ADD_PROJECT",
    ADD_RULES_TO_THE_PRONUNCIATION_DICTIONARY: "ELEVENLABS_ADD_RULES_TO_THE_PRONUNCIATION_DICTIONARY",
    ADD_SHARING_VOICE: "ELEVENLABS_ADD_SHARING_VOICE",
    ADD_VOICE: "ELEVENLABS_ADD_VOICE",
    CONVERT_CHAPTER: "ELEVENLABS_CONVERT_CHAPTER",
    CONVERT_PROJECT: "ELEVENLABS_CONVERT_PROJECT",
    CREATES_AUDIONATIVE_ENABLED_PROJECT: "ELEVENLABS_CREATES_AUDIONATIVE_ENABLED_PROJECT",
    CREATE_A_PREVIOUSLY_GENERATED_VOICE: "ELEVENLABS_CREATE_A_PREVIOUSLY_GENERATED_VOICE",
    DELETE_CHAPTER: "ELEVENLABS_DELETE_CHAPTER",
    DELETE_DUBBING_PROJECT: "ELEVENLABS_DELETE_DUBBING_PROJECT",
    DELETE_HISTORY_ITEM: "ELEVENLABS_DELETE_HISTORY_ITEM",
    DELETE_PROJECT: "ELEVENLABS_DELETE_PROJECT",
    DELETE_SAMPLE: "ELEVENLABS_DELETE_SAMPLE",
    DELETE_VOICE: "ELEVENLABS_DELETE_VOICE",
    DOWNLOAD_HISTORY_ITEMS: "ELEVENLABS_DOWNLOAD_HISTORY_ITEMS",
    DUB_A_VIDEO_OR_AN_AUDIOFILE_V_1_DUBBING_POST: "ELEVENLABS_DUB_A_VIDEO_OR_AN_AUDIOFILE_V_1_DUBBING_POST",
    DUB_A_VIDEO_OR_AN_AUDIO_FILE: "ELEVENLABS_DUB_A_VIDEO_OR_AN_AUDIO_FILE",
    EDIT_VOICE: "ELEVENLABS_EDIT_VOICE",
    EDIT_VOICE_SETTINGS: "ELEVENLABS_EDIT_VOICE_SETTINGS",
    GENERATE_A_RANDOM_VOICE: "ELEVENLABS_GENERATE_A_RANDOM_VOICE",
    GET_AUDIO_FROM_HISTORY_ITEM: "ELEVENLABS_GET_AUDIO_FROM_HISTORY_ITEM",
    GET_AUDIO_FROM_SAMPLE: "ELEVENLABS_GET_AUDIO_FROM_SAMPLE",
    GET_A_PROFILE_PAGE: "ELEVENLABS_GET_A_PROFILE_PAGE",
    GET_CHAPTERS: "ELEVENLABS_GET_CHAPTERS",
    GET_CHAPTER_BY_ID: "ELEVENLABS_GET_CHAPTER_BY_ID",
    GET_CHAPTER_SNAPSHOTS: "ELEVENLABS_GET_CHAPTER_SNAPSHOTS",
    GET_DEFAULT_VOICE_SETTINGS: "ELEVENLABS_GET_DEFAULT_VOICE_SETTINGS",
    GET_DUBBED_FILE: "ELEVENLABS_GET_DUBBED_FILE",
    GET_DUBBING_PROJECT_METADATA: "ELEVENLABS_GET_DUBBING_PROJECT_METADATA",
    GET_GENERATED_ITEMS: "ELEVENLABS_GET_GENERATED_ITEMS",
    GET_HISTORY_ITEM_BY_ID: "ELEVENLABS_GET_HISTORY_ITEM_BY_ID",
    GET_METADATA_FOR_A_PRONUNCIATION_DICTIONARY: "ELEVENLABS_GET_METADATA_FOR_A_PRONUNCIATION_DICTIONARY",
    GET_MODELS: "ELEVENLABS_GET_MODELS",
    GET_MODELS_V_1_MODELS_GET: "ELEVENLABS_GET_MODELS_V_1_MODELS_GET",
    GET_PROJECTS: "ELEVENLABS_GET_PROJECTS",
    GET_PROJECT_BY_ID: "ELEVENLABS_GET_PROJECT_BY_ID",
    GET_PROJECT_SNAPSHOTS: "ELEVENLABS_GET_PROJECT_SNAPSHOTS",
    GET_PRONUNCIATION_DICTIONARIES: "ELEVENLABS_GET_PRONUNCIATION_DICTIONARIES",
    GET_PRON_DICT_VERSION: "ELEVENLABS_GET_PRON_DICT_VERSION",
    GET_SHARED_VOICES: "ELEVENLABS_GET_SHARED_VOICES",
    GET_SSO_PROVIDER_ADMIN: "ELEVENLABS_GET_SSO_PROVIDER_ADMIN",
    GET_TRANSCRIPT_FOR_DUB: "ELEVENLABS_GET_TRANSCRIPT_FOR_DUB",
    GET_USER_INFO: "ELEVENLABS_GET_USER_INFO",
    GET_USER_INFO_V_1_USER_GET: "ELEVENLABS_GET_USER_INFO_V_1_USER_GET",
    GET_USER_SUBSCRIPTION_INFO: "ELEVENLABS_GET_USER_SUBSCRIPTION_INFO",
    GET_VOICE: "ELEVENLABS_GET_VOICE",
    GET_VOICES: "ELEVENLABS_GET_VOICES",
    GET_VOICES_V_1_VOICES_GET: "ELEVENLABS_GET_VOICES_V_1_VOICES_GET",
    GET_VOICE_SETTINGS: "ELEVENLABS_GET_VOICE_SETTINGS",
    REDIRECT_TO_MINTLIFY: "ELEVENLABS_REDIRECT_TO_MINTLIFY",
    REMOVE_RULES_FROM_THE_PRONUNCIATION_DICTIONARY: "ELEVENLABS_REMOVE_RULES_FROM_THE_PRONUNCIATION_DICTIONARY",
    SPEECH_TO_SPEECH: "ELEVENLABS_SPEECH_TO_SPEECH",
    SPEECH_TO_SPEECH_STREAMING: "ELEVENLABS_SPEECH_TO_SPEECH_STREAMING",
    STREAMS_ARCHIVE_WITH_PROJECT_AUDIO: "ELEVENLABS_STREAMS_ARCHIVE_WITH_PROJECT_AUDIO",
    STREAM_CHAPTER_AUDIO: "ELEVENLABS_STREAM_CHAPTER_AUDIO",
    STREAM_PROJECT_AUDIO: "ELEVENLABS_STREAM_PROJECT_AUDIO",
    TEXT_TO_SPEECH: "ELEVENLABS_TEXT_TO_SPEECH",
    TEXT_TO_SPEECH_STREAM: "ELEVENLABS_TEXT_TO_SPEECH_STREAM",
    UPDATE_PRONUNCIATION_DICTIONARIES: "ELEVENLABS_UPDATE_PRONUNCIATION_DICTIONARIES",
    VOICE_GENERATION_PARAMETERS: "ELEVENLABS_VOICE_GENERATION_PARAMETERS",
  },
  triggerTypes: {},
}
