// @composio/core/generated/excel.ts
/**
 * Map of Composio's EXCEL toolkit.
 */
export const EXCEL = {
  slug: "excel",
  tools: {
    ADD_CHART: "EXCEL_ADD_CHART",
    ADD_TABLE: "EXCEL_ADD_TABLE",
    ADD_TABLE_COLUMN: "EXCEL_ADD_TABLE_COLUMN",
    ADD_TABLE_ROW: "EXCEL_ADD_TABLE_ROW",
    ADD_WORKSHEET: "EXCEL_ADD_WORKSHEET",
    APPLY_TABLE_FILTER: "EXCEL_APPLY_TABLE_FILTER",
    APPLY_TABLE_SORT: "EXCEL_APPLY_TABLE_SORT",
    CLEAR_TABLE_FILTER: "EXCEL_CLEAR_TABLE_FILTER",
    CONVERT_TABLE_TO_RANGE: "EXCEL_CONVERT_TABLE_TO_RANGE",
    DELETE_TABLE_COLUMN: "EXCEL_DELETE_TABLE_COLUMN",
    DELETE_TABLE_ROW: "EXCEL_DELETE_TABLE_ROW",
    GET_RANGE: "EXCEL_GET_RANGE",
    GET_SESSION: "EXCEL_GET_SESSION",
    GET_WORKSHEET: "EXCEL_GET_WORKSHEET",
    LIST_CHARTS: "EXCEL_LIST_CHARTS",
    LIST_NAMED_ITEMS: "EXCEL_LIST_NAMED_ITEMS",
    LIST_TABLES: "EXCEL_LIST_TABLES",
    LIST_TABLE_COLUMNS: "EXCEL_LIST_TABLE_COLUMNS",
    LIST_TABLE_ROWS: "EXCEL_LIST_TABLE_ROWS",
    LIST_WORKSHEETS: "EXCEL_LIST_WORKSHEETS",
    SORT_RANGE: "EXCEL_SORT_RANGE",
    UPDATE_CHART: "EXCEL_UPDATE_CHART",
    UPDATE_RANGE: "EXCEL_UPDATE_RANGE",
    UPDATE_TABLE: "EXCEL_UPDATE_TABLE",
    UPDATE_WORKSHEET: "EXCEL_UPDATE_WORKSHEET",
  },
  triggerTypes: {},
}
