// @composio/core/generated/figma.ts
/**
 * Map of Composio's FIGMA toolkit.
 */
export const FIGMA = {
  slug: "figma",
  tools: {
    ADD_A_COMMENT_TO_A_FILE: "FIGMA_ADD_A_COMMENT_TO_A_FILE",
    ADD_A_REACTION_TO_A_COMMENT: "FIGMA_ADD_A_REACTION_TO_A_COMMENT",
    CREATE_A_WEBHOOK: "FIGMA_CREATE_A_WEBHOOK",
    CREATE_DEV_RESOURCES: "FIGMA_CREATE_DEV_RESOURCES",
    CREATE_MODIFY_DELETE_VARIABLES: "FIGMA_CREATE_MODIFY_DELETE_VARIABLES",
    DELETE_A_COMMENT: "FIGMA_DELETE_A_COMMENT",
    DELETE_A_REACTION: "FIGMA_DELETE_A_REACTION",
    DELETE_A_WEBHOOK: "FIGMA_DELETE_A_WEBHOOK",
    DELETE_DEV_RESOURCE: "FIGMA_DELETE_DEV_RESOURCE",
    DESIGN_TOKENS_TO_TAILWIND: "FIGMA_DESIGN_TOKENS_TO_TAILWIND",
    DETECT_BACKGROUND: "FIGMA_DETECT_BACKGROUND",
    DISCOVER_FIGMA_RESOURCES: "FIGMA_DISCOVER_FIGMA_RESOURCES",
    DOWNLOAD_FIGMA_IMAGES: "FIGMA_DOWNLOAD_FIGMA_IMAGES",
    EXTRACT_DESIGN_TOKENS: "FIGMA_EXTRACT_DESIGN_TOKENS",
    EXTRACT_PROTOTYPE_INTERACTIONS: "FIGMA_EXTRACT_PROTOTYPE_INTERACTIONS",
    GET_ACTIVITY_LOGS: "FIGMA_GET_ACTIVITY_LOGS",
    GET_A_WEBHOOK: "FIGMA_GET_A_WEBHOOK",
    GET_COMMENTS_IN_A_FILE: "FIGMA_GET_COMMENTS_IN_A_FILE",
    GET_COMPONENT: "FIGMA_GET_COMPONENT",
    GET_COMPONENT_SET: "FIGMA_GET_COMPONENT_SET",
    GET_CURRENT_USER: "FIGMA_GET_CURRENT_USER",
    GET_DEV_RESOURCES: "FIGMA_GET_DEV_RESOURCES",
    GET_FILES_IN_A_PROJECT: "FIGMA_GET_FILES_IN_A_PROJECT",
    GET_FILE_COMPONENTS: "FIGMA_GET_FILE_COMPONENTS",
    GET_FILE_COMPONENT_SETS: "FIGMA_GET_FILE_COMPONENT_SETS",
    GET_FILE_JSON: "FIGMA_GET_FILE_JSON",
    GET_FILE_STYLES: "FIGMA_GET_FILE_STYLES",
    GET_IMAGE_FILLS: "FIGMA_GET_IMAGE_FILLS",
    GET_LIBRARY_ANALYTICS_COMPONENT_ACTION_DATA: "FIGMA_GET_LIBRARY_ANALYTICS_COMPONENT_ACTION_DATA",
    GET_LIBRARY_ANALYTICS_COMPONENT_USAGE_DATA: "FIGMA_GET_LIBRARY_ANALYTICS_COMPONENT_USAGE_DATA",
    GET_LIBRARY_ANALYTICS_STYLE_ACTION_DATA: "FIGMA_GET_LIBRARY_ANALYTICS_STYLE_ACTION_DATA",
    GET_LIBRARY_ANALYTICS_STYLE_USAGE_DATA: "FIGMA_GET_LIBRARY_ANALYTICS_STYLE_USAGE_DATA",
    GET_LIBRARY_ANALYTICS_VARIABLE_ACTION_DATA: "FIGMA_GET_LIBRARY_ANALYTICS_VARIABLE_ACTION_DATA",
    GET_LIBRARY_ANALYTICS_VARIABLE_USAGE_DATA: "FIGMA_GET_LIBRARY_ANALYTICS_VARIABLE_USAGE_DATA",
    GET_LOCAL_VARIABLES: "FIGMA_GET_LOCAL_VARIABLES",
    GET_ME: "FIGMA_GET_ME",
    GET_PAYMENTS: "FIGMA_GET_PAYMENTS",
    GET_PROJECTS_IN_A_TEAM: "FIGMA_GET_PROJECTS_IN_A_TEAM",
    GET_PUBLISHED_VARIABLES: "FIGMA_GET_PUBLISHED_VARIABLES",
    GET_REACTIONS_FOR_A_COMMENT: "FIGMA_GET_REACTIONS_FOR_A_COMMENT",
    GET_STYLE: "FIGMA_GET_STYLE",
    GET_TEAM_COMPONENTS: "FIGMA_GET_TEAM_COMPONENTS",
    GET_TEAM_COMPONENT_SETS: "FIGMA_GET_TEAM_COMPONENT_SETS",
    GET_TEAM_STYLES: "FIGMA_GET_TEAM_STYLES",
    GET_TEAM_WEBHOOKS: "FIGMA_GET_TEAM_WEBHOOKS",
    GET_VERSIONS_OF_A_FILE: "FIGMA_GET_VERSIONS_OF_A_FILE",
    GET_WEBHOOK_REQUESTS: "FIGMA_GET_WEBHOOK_REQUESTS",
    RENDER_IMAGES_OF_FILE_NODES: "FIGMA_RENDER_IMAGES_OF_FILE_NODES",
    UPDATE_A_WEBHOOK: "FIGMA_UPDATE_A_WEBHOOK",
    UPDATE_DEV_RESOURCES: "FIGMA_UPDATE_DEV_RESOURCES",
  },
  triggerTypes: {},
}
