// @composio/core/generated/flutterwave.ts
/**
 * Map of Composio's FLUTTERWAVE toolkit.
 */
export const FLUTTERWAVE = {
  slug: "flutterwave",
  tools: {
    CREATE_BENEFICIARY: "FLUTTERWAVE_CREATE_BENEFICIARY",
    CREATE_BULK_VIRTUAL_ACCOUNT_NUMBERS: "FLUTTERWAVE_CREATE_BULK_VIRTUAL_ACCOUNT_NUMBERS",
    CREATE_PAYMENT_LINK: "FLUTTERWAVE_CREATE_PAYMENT_LINK",
    CREATE_PAYMENT_PLAN: "FLUTTERWAVE_CREATE_PAYMENT_PLAN",
    CREATE_SUBACCOUNT: "FLUTTERWAVE_CREATE_SUBACCOUNT",
    CREATE_VIRTUAL_ACCOUNT: "FLUTTERWAVE_CREATE_VIRTUAL_ACCOUNT",
    DELETE_BENEFICIARY: "FLUTTERWAVE_DELETE_BENEFICIARY",
    DISABLE_PAYMENT_LINK: "FLUTTERWAVE_DISABLE_PAYMENT_LINK",
    FETCH_BENEFICIARY: "FLUTTERWAVE_FETCH_BENEFICIARY",
    FETCH_SUBACCOUNT: "FLUTTERWAVE_FETCH_SUBACCOUNT",
    GENERATE_TRANSACTION_REFERENCE: "FLUTTERWAVE_GENERATE_TRANSACTION_REFERENCE",
    GET_ALL_SUBSCRIPTIONS: "FLUTTERWAVE_GET_ALL_SUBSCRIPTIONS",
    GET_ALL_TRANSACTIONS: "FLUTTERWAVE_GET_ALL_TRANSACTIONS",
    GET_ALL_WALLET_BALANCES: "FLUTTERWAVE_GET_ALL_WALLET_BALANCES",
    GET_BALANCES_PER_CURRENCY: "FLUTTERWAVE_GET_BALANCES_PER_CURRENCY",
    GET_BILL_CATEGORIES: "FLUTTERWAVE_GET_BILL_CATEGORIES",
    GET_MULTIPLE_REFUND_TRANSACTIONS: "FLUTTERWAVE_GET_MULTIPLE_REFUND_TRANSACTIONS",
    GET_PAYMENT_PLANS: "FLUTTERWAVE_GET_PAYMENT_PLANS",
    GET_TRANSACTION: "FLUTTERWAVE_GET_TRANSACTION",
    GET_TRANSACTION_FEE: "FLUTTERWAVE_GET_TRANSACTION_FEE",
    GET_TRANSFER_FEE: "FLUTTERWAVE_GET_TRANSFER_FEE",
    INITIATE_MOBILE_MONEY_TANZANIA: "FLUTTERWAVE_INITIATE_MOBILE_MONEY_TANZANIA",
    LIST_ALL_BENEFICIARIES: "FLUTTERWAVE_LIST_ALL_BENEFICIARIES",
    VIEW_TRANSACTION_TIMELINE: "FLUTTERWAVE_VIEW_TRANSACTION_TIMELINE",
  },
  triggerTypes: {},
}
