// @composio/core/generated/github.ts
/**
 * Map of Composio's GITHUB toolkit.
 */
export const GITHUB = {
  slug: "github",
  tools: {
    ACCEPT_A_REPOSITORY_INVITATION: "GITHUB_ACCEPT_A_REPOSITORY_INVITATION",
    ACTIVITY_LIST_REPO_S_STARRED_BY_AUTHENTICATED_USER: "GITHUB_ACTIVITY_LIST_REPO_S_STARRED_BY_AUTHENTICATED_USER",
    ACTIVITY_LIST_STARGAZERS_FOR_REPO: "GITHUB_ACTIVITY_LIST_STARGAZERS_FOR_REPO",
    ACTIVITY_STAR_REPO_FOR_AUTHENTICATED_USER: "GITHUB_ACTIVITY_STAR_REPO_FOR_AUTHENTICATED_USER",
    ADD_APP_ACCESS_RESTRICTIONS: "GITHUB_ADD_APP_ACCESS_RESTRICTIONS",
    ADD_ASSIGNEES_TO_AN_ISSUE: "GITHUB_ADD_ASSIGNEES_TO_AN_ISSUE",
    ADD_A_REPOSITORY_COLLABORATOR: "GITHUB_ADD_A_REPOSITORY_COLLABORATOR",
    ADD_A_REPOSITORY_TO_AN_APP_INSTALLATION: "GITHUB_ADD_A_REPOSITORY_TO_AN_APP_INSTALLATION",
    ADD_A_SECURITY_MANAGER_TEAM: "GITHUB_ADD_A_SECURITY_MANAGER_TEAM",
    ADD_A_SELECTED_REPOSITORY_TO_A_USER_SECRET: "GITHUB_ADD_A_SELECTED_REPOSITORY_TO_A_USER_SECRET",
    ADD_EMAIL_FOR_AUTH_USER: "GITHUB_ADD_EMAIL_FOR_AUTH_USER",
    ADD_LABELS_TO_AN_ISSUE: "GITHUB_ADD_LABELS_TO_AN_ISSUE",
    ADD_ORG_RUNNER_LABELS: "GITHUB_ADD_ORG_RUNNER_LABELS",
    ADD_OR_UPDATE_TEAM_MEMBERSHIP_FOR_A_USER: "GITHUB_ADD_OR_UPDATE_TEAM_MEMBERSHIP_FOR_A_USER",
    ADD_OR_UPDATE_TEAM_MEMBERSHIP_FOR_A_USER_LEGACY: "GITHUB_ADD_OR_UPDATE_TEAM_MEMBERSHIP_FOR_A_USER_LEGACY",
    ADD_OR_UPDATE_TEAM_PROJECT_PERMISSIONS: "GITHUB_ADD_OR_UPDATE_TEAM_PROJECT_PERMISSIONS",
    ADD_OR_UPDATE_TEAM_PROJECT_PERMISSIONS_LEGACY: "GITHUB_ADD_OR_UPDATE_TEAM_PROJECT_PERMISSIONS_LEGACY",
    ADD_OR_UPDATE_TEAM_REPOSITORY_PERMISSIONS: "GITHUB_ADD_OR_UPDATE_TEAM_REPOSITORY_PERMISSIONS",
    ADD_OR_UPDATE_TEAM_REPOSITORY_PERMISSIONS_LEGACY: "GITHUB_ADD_OR_UPDATE_TEAM_REPOSITORY_PERMISSIONS_LEGACY",
    ADD_PROJECT_COLLABORATOR: "GITHUB_ADD_PROJECT_COLLABORATOR",
    ADD_REPO_TO_ORG_SECRET_WITH_SELECTED_ACCESS: "GITHUB_ADD_REPO_TO_ORG_SECRET_WITH_SELECTED_ACCESS",
    ADD_REPO_TO_ORG_SECRET_WITH_SELECTED_VISIBILITY: "GITHUB_ADD_REPO_TO_ORG_SECRET_WITH_SELECTED_VISIBILITY",
    ADD_RUNNER_LABELS: "GITHUB_ADD_RUNNER_LABELS",
    ADD_SELECTED_REPOSITORY_TO_AN_ORGANIZATION_SECRET: "GITHUB_ADD_SELECTED_REPOSITORY_TO_AN_ORGANIZATION_SECRET",
    ADD_SELECTED_REPOSITORY_TO_AN_ORGANIZATION_VARIABLE: "GITHUB_ADD_SELECTED_REPOSITORY_TO_AN_ORGANIZATION_VARIABLE",
    ADD_SOCIAL_ACCOUNTS_FOR_THE_AUTHENTICATED_USER: "GITHUB_ADD_SOCIAL_ACCOUNTS_FOR_THE_AUTHENTICATED_USER",
    ADD_STATUS_CHECK_CONTEXTS: "GITHUB_ADD_STATUS_CHECK_CONTEXTS",
    ADD_TEAMS_TO_COPILOT_SUBSCRIPTION_FOR_ORG: "GITHUB_ADD_TEAMS_TO_COPILOT_SUBSCRIPTION_FOR_ORG",
    ADD_TEAM_ACCESS_RESTRICTIONS: "GITHUB_ADD_TEAM_ACCESS_RESTRICTIONS",
    ADD_TEAM_MEMBER_LEGACY: "GITHUB_ADD_TEAM_MEMBER_LEGACY",
    ADD_USERS_TO_CODESPACES_ACCESS_FOR_AN_ORGANIZATION: "GITHUB_ADD_USERS_TO_CODESPACES_ACCESS_FOR_AN_ORGANIZATION",
    ADD_USERS_TO_COPILOT_SUBSCRIPTION_FOR_ORG: "GITHUB_ADD_USERS_TO_COPILOT_SUBSCRIPTION_FOR_ORG",
    ADD_USER_ACCESS_RESTRICTIONS: "GITHUB_ADD_USER_ACCESS_RESTRICTIONS",
    APPROVE_A_WORKFLOW_RUN_FOR_A_FORK_PULL_REQUEST: "GITHUB_APPROVE_A_WORKFLOW_RUN_FOR_A_FORK_PULL_REQUEST",
    ASSIGN_AN_ORGANIZATION_ROLE_TO_A_TEAM: "GITHUB_ASSIGN_AN_ORGANIZATION_ROLE_TO_A_TEAM",
    ASSIGN_AN_ORGANIZATION_ROLE_TO_A_USER: "GITHUB_ASSIGN_AN_ORGANIZATION_ROLE_TO_A_USER",
    AUTH_USER_DOCKER_CONFLICT_PACKAGES_LIST: "GITHUB_AUTH_USER_DOCKER_CONFLICT_PACKAGES_LIST",
    BLOCK_A_USER: "GITHUB_BLOCK_A_USER",
    BLOCK_A_USER_FROM_AN_ORGANIZATION: "GITHUB_BLOCK_A_USER_FROM_AN_ORGANIZATION",
    CANCEL_AN_IMPORT: "GITHUB_CANCEL_AN_IMPORT",
    CANCEL_AN_ORGANIZATION_INVITATION: "GITHUB_CANCEL_AN_ORGANIZATION_INVITATION",
    CANCEL_A_GITHUB_PAGES_DEPLOYMENT: "GITHUB_CANCEL_A_GITHUB_PAGES_DEPLOYMENT",
    CANCEL_A_WORKFLOW_RUN: "GITHUB_CANCEL_A_WORKFLOW_RUN",
    CHECK_AUTO_SECURITY_FIXES_FOR_REPO: "GITHUB_CHECK_AUTO_SECURITY_FIXES_FOR_REPO",
    CHECK_A_TOKEN: "GITHUB_CHECK_A_TOKEN",
    CHECK_IF_A_GIST_IS_STARRED: "GITHUB_CHECK_IF_A_GIST_IS_STARRED",
    CHECK_IF_A_USER_CAN_BE_ASSIGNED: "GITHUB_CHECK_IF_A_USER_CAN_BE_ASSIGNED",
    CHECK_IF_A_USER_CAN_BE_ASSIGNED_TO_A_ISSUE: "GITHUB_CHECK_IF_A_USER_CAN_BE_ASSIGNED_TO_A_ISSUE",
    CHECK_IF_A_USER_FOLLOWS_ANOTHER_USER: "GITHUB_CHECK_IF_A_USER_FOLLOWS_ANOTHER_USER",
    CHECK_IF_A_USER_IS_A_REPOSITORY_COLLABORATOR: "GITHUB_CHECK_IF_A_USER_IS_A_REPOSITORY_COLLABORATOR",
    CHECK_IF_A_USER_IS_BLOCKED_BY_AN_ORGANIZATION: "GITHUB_CHECK_IF_A_USER_IS_BLOCKED_BY_AN_ORGANIZATION",
    CHECK_IF_PULL_REQUEST_MERGED: "GITHUB_CHECK_IF_PULL_REQUEST_MERGED",
    CHECK_IF_REPO_STARRED_BY_AUTH_USER: "GITHUB_CHECK_IF_REPO_STARRED_BY_AUTH_USER",
    CHECK_ORGANIZATION_MEMBERSHIP_FOR_A_USER: "GITHUB_CHECK_ORGANIZATION_MEMBERSHIP_FOR_A_USER",
    CHECK_PERSON_FOLLOWED_BY_AUTH_USER: "GITHUB_CHECK_PERSON_FOLLOWED_BY_AUTH_USER",
    CHECK_PRIVATE_VULNERABILITY_REPORTING_STATUS: "GITHUB_CHECK_PRIVATE_VULNERABILITY_REPORTING_STATUS",
    CHECK_PUBLIC_ORGANIZATION_MEMBERSHIP_FOR_A_USER: "GITHUB_CHECK_PUBLIC_ORGANIZATION_MEMBERSHIP_FOR_A_USER",
    CHECK_TEAM_PERMISSIONS_FOR_A_PROJECT: "GITHUB_CHECK_TEAM_PERMISSIONS_FOR_A_PROJECT",
    CHECK_TEAM_PERMISSIONS_FOR_A_PROJECT_LEGACY: "GITHUB_CHECK_TEAM_PERMISSIONS_FOR_A_PROJECT_LEGACY",
    CHECK_TEAM_PERMISSIONS_FOR_A_REPOSITORY: "GITHUB_CHECK_TEAM_PERMISSIONS_FOR_A_REPOSITORY",
    CHECK_TEAM_PERMISSIONS_FOR_A_REPOSITORY_LEGACY: "GITHUB_CHECK_TEAM_PERMISSIONS_FOR_A_REPOSITORY_LEGACY",
    CHECK_USER_BLOCKED_BY_AUTH_USER: "GITHUB_CHECK_USER_BLOCKED_BY_AUTH_USER",
    CHECK_VULNERABILITY_ALERTS_ENABLED_FOR_REPO: "GITHUB_CHECK_VULNERABILITY_ALERTS_ENABLED_FOR_REPO",
    CLEAR_REPOSITORY_CACHE_BY_KEY: "GITHUB_CLEAR_REPOSITORY_CACHE_BY_KEY",
    CLEAR_SELF_HOSTED_RUNNER_ORG_LABELS: "GITHUB_CLEAR_SELF_HOSTED_RUNNER_ORG_LABELS",
    COMPARE_TWO_COMMITS: "GITHUB_COMPARE_TWO_COMMITS",
    CONFIGURE_JITRUNNER_FOR_ORG: "GITHUB_CONFIGURE_JITRUNNER_FOR_ORG",
    CONFIGURE_OIDCSUBJECT_CLAIM_TEMPLATE: "GITHUB_CONFIGURE_OIDCSUBJECT_CLAIM_TEMPLATE",
    CONVERT_ORG_MEMBER_TO_OUTSIDE_COLLABORATOR: "GITHUB_CONVERT_ORG_MEMBER_TO_OUTSIDE_COLLABORATOR",
    CREATE_AN_AUTOLINK_REFERENCE_FOR_A_REPOSITORY: "GITHUB_CREATE_AN_AUTOLINK_REFERENCE_FOR_A_REPOSITORY",
    CREATE_AN_ENVIRONMENT_VARIABLE: "GITHUB_CREATE_AN_ENVIRONMENT_VARIABLE",
    CREATE_AN_ISSUE: "GITHUB_CREATE_AN_ISSUE",
    CREATE_AN_ISSUE_COMMENT: "GITHUB_CREATE_AN_ISSUE_COMMENT",
    CREATE_AN_ORGANIZATION_INVITATION: "GITHUB_CREATE_AN_ORGANIZATION_INVITATION",
    CREATE_AN_ORGANIZATION_PROJECT: "GITHUB_CREATE_AN_ORGANIZATION_PROJECT",
    CREATE_AN_ORGANIZATION_REPOSITORY: "GITHUB_CREATE_AN_ORGANIZATION_REPOSITORY",
    CREATE_AN_ORGANIZATION_REPOSITORY_RULESET: "GITHUB_CREATE_AN_ORGANIZATION_REPOSITORY_RULESET",
    CREATE_AN_ORGANIZATION_VARIABLE: "GITHUB_CREATE_AN_ORGANIZATION_VARIABLE",
    CREATE_AN_ORGANIZATION_WEBHOOK: "GITHUB_CREATE_AN_ORGANIZATION_WEBHOOK",
    CREATE_A_BLOB: "GITHUB_CREATE_A_BLOB",
    CREATE_A_CHECK_RUN: "GITHUB_CREATE_A_CHECK_RUN",
    CREATE_A_CHECK_SUITE: "GITHUB_CREATE_A_CHECK_SUITE",
    CREATE_A_CODESPACE_FOR_THE_AUTHENTICATED_USER: "GITHUB_CREATE_A_CODESPACE_FOR_THE_AUTHENTICATED_USER",
    CREATE_A_CODESPACE_FROM_A_PULL_REQUEST: "GITHUB_CREATE_A_CODESPACE_FROM_A_PULL_REQUEST",
    CREATE_A_CODESPACE_IN_A_REPOSITORY: "GITHUB_CREATE_A_CODESPACE_IN_A_REPOSITORY",
    CREATE_A_COMMIT: "GITHUB_CREATE_A_COMMIT",
    CREATE_A_COMMIT_COMMENT: "GITHUB_CREATE_A_COMMIT_COMMENT",
    CREATE_A_COMMIT_STATUS: "GITHUB_CREATE_A_COMMIT_STATUS",
    CREATE_A_CUSTOM_ORGANIZATION_ROLE: "GITHUB_CREATE_A_CUSTOM_ORGANIZATION_ROLE",
    CREATE_A_DEPLOYMENT: "GITHUB_CREATE_A_DEPLOYMENT",
    CREATE_A_DEPLOYMENT_BRANCH_POLICY: "GITHUB_CREATE_A_DEPLOYMENT_BRANCH_POLICY",
    CREATE_A_DEPLOYMENT_STATUS: "GITHUB_CREATE_A_DEPLOYMENT_STATUS",
    CREATE_A_DEPLOY_KEY: "GITHUB_CREATE_A_DEPLOY_KEY",
    CREATE_A_DISCUSSION: "GITHUB_CREATE_A_DISCUSSION",
    CREATE_A_DISCUSSION_COMMENT: "GITHUB_CREATE_A_DISCUSSION_COMMENT",
    CREATE_A_DISCUSSION_COMMENT_LEGACY: "GITHUB_CREATE_A_DISCUSSION_COMMENT_LEGACY",
    CREATE_A_DISCUSSION_LEGACY: "GITHUB_CREATE_A_DISCUSSION_LEGACY",
    CREATE_A_FORK: "GITHUB_CREATE_A_FORK",
    CREATE_A_GIST: "GITHUB_CREATE_A_GIST",
    CREATE_A_GIST_COMMENT: "GITHUB_CREATE_A_GIST_COMMENT",
    CREATE_A_GITHUB_APP_FROM_A_MANIFEST: "GITHUB_CREATE_A_GITHUB_APP_FROM_A_MANIFEST",
    CREATE_A_GITHUB_PAGES_DEPLOYMENT: "GITHUB_CREATE_A_GITHUB_PAGES_DEPLOYMENT",
    CREATE_A_GITHUB_PAGES_SITE: "GITHUB_CREATE_A_GITHUB_PAGES_SITE",
    CREATE_A_GPG_KEY_FOR_THE_AUTHENTICATED_USER: "GITHUB_CREATE_A_GPG_KEY_FOR_THE_AUTHENTICATED_USER",
    CREATE_A_LABEL: "GITHUB_CREATE_A_LABEL",
    CREATE_A_MILESTONE: "GITHUB_CREATE_A_MILESTONE",
    CREATE_A_PROJECT_CARD: "GITHUB_CREATE_A_PROJECT_CARD",
    CREATE_A_PROJECT_COLUMN: "GITHUB_CREATE_A_PROJECT_COLUMN",
    CREATE_A_PUBLIC_SSH_KEY_FOR_THE_AUTHENTICATED_USER: "GITHUB_CREATE_A_PUBLIC_SSH_KEY_FOR_THE_AUTHENTICATED_USER",
    CREATE_A_PULL_REQUEST: "GITHUB_CREATE_A_PULL_REQUEST",
    CREATE_A_REFERENCE: "GITHUB_CREATE_A_REFERENCE",
    CREATE_A_REGISTRATION_TOKEN_FOR_AN_ORGANIZATION: "GITHUB_CREATE_A_REGISTRATION_TOKEN_FOR_AN_ORGANIZATION",
    CREATE_A_REGISTRATION_TOKEN_FOR_A_REPOSITORY: "GITHUB_CREATE_A_REGISTRATION_TOKEN_FOR_A_REPOSITORY",
    CREATE_A_RELEASE: "GITHUB_CREATE_A_RELEASE",
    CREATE_A_REMOVE_TOKEN_FOR_AN_ORGANIZATION: "GITHUB_CREATE_A_REMOVE_TOKEN_FOR_AN_ORGANIZATION",
    CREATE_A_REMOVE_TOKEN_FOR_A_REPOSITORY: "GITHUB_CREATE_A_REMOVE_TOKEN_FOR_A_REPOSITORY",
    CREATE_A_REPLY_FOR_A_REVIEW_COMMENT: "GITHUB_CREATE_A_REPLY_FOR_A_REVIEW_COMMENT",
    CREATE_A_REPOSITORY_DISPATCH_EVENT: "GITHUB_CREATE_A_REPOSITORY_DISPATCH_EVENT",
    CREATE_A_REPOSITORY_FOR_THE_AUTHENTICATED_USER: "GITHUB_CREATE_A_REPOSITORY_FOR_THE_AUTHENTICATED_USER",
    CREATE_A_REPOSITORY_FROM_AN_UNPUBLISHED_CODESPACE: "GITHUB_CREATE_A_REPOSITORY_FROM_AN_UNPUBLISHED_CODESPACE",
    CREATE_A_REPOSITORY_PROJECT: "GITHUB_CREATE_A_REPOSITORY_PROJECT",
    CREATE_A_REPOSITORY_RULESET: "GITHUB_CREATE_A_REPOSITORY_RULESET",
    CREATE_A_REPOSITORY_SECURITY_ADVISORY: "GITHUB_CREATE_A_REPOSITORY_SECURITY_ADVISORY",
    CREATE_A_REPOSITORY_USING_A_TEMPLATE: "GITHUB_CREATE_A_REPOSITORY_USING_A_TEMPLATE",
    CREATE_A_REPOSITORY_VARIABLE: "GITHUB_CREATE_A_REPOSITORY_VARIABLE",
    CREATE_A_REPOSITORY_WEBHOOK: "GITHUB_CREATE_A_REPOSITORY_WEBHOOK",
    CREATE_A_REVIEW_COMMENT_FOR_A_PULL_REQUEST: "GITHUB_CREATE_A_REVIEW_COMMENT_FOR_A_PULL_REQUEST",
    CREATE_A_REVIEW_FOR_A_PULL_REQUEST: "GITHUB_CREATE_A_REVIEW_FOR_A_PULL_REQUEST",
    CREATE_A_SCOPED_ACCESS_TOKEN: "GITHUB_CREATE_A_SCOPED_ACCESS_TOKEN",
    CREATE_A_SNAPSHOT_OF_DEPENDENCIES_FOR_A_REPOSITORY: "GITHUB_CREATE_A_SNAPSHOT_OF_DEPENDENCIES_FOR_A_REPOSITORY",
    CREATE_A_SSH_SIGNING_KEY_FOR_THE_AUTHENTICATED_USER: "GITHUB_CREATE_A_SSH_SIGNING_KEY_FOR_THE_AUTHENTICATED_USER",
    CREATE_A_TAG_OBJECT: "GITHUB_CREATE_A_TAG_OBJECT",
    CREATE_A_TAG_PROTECTION_STATE_FOR_A_REPOSITORY: "GITHUB_CREATE_A_TAG_PROTECTION_STATE_FOR_A_REPOSITORY",
    CREATE_A_TEAM: "GITHUB_CREATE_A_TEAM",
    CREATE_A_TEMPORARY_PRIVATE_FORK: "GITHUB_CREATE_A_TEMPORARY_PRIVATE_FORK",
    CREATE_A_TREE: "GITHUB_CREATE_A_TREE",
    CREATE_A_USER_PROJECT: "GITHUB_CREATE_A_USER_PROJECT",
    CREATE_A_WORKFLOW_DISPATCH_EVENT: "GITHUB_CREATE_A_WORKFLOW_DISPATCH_EVENT",
    CREATE_COMMIT_SIGNATURE_PROTECTION: "GITHUB_CREATE_COMMIT_SIGNATURE_PROTECTION",
    CREATE_CUSTOM_DEPLOYMENT_PROTECTION_RULE_ON_ENV: "GITHUB_CREATE_CUSTOM_DEPLOYMENT_PROTECTION_RULE_ON_ENV",
    CREATE_JIT_RUNNER_CONFIG_FOR_REPO: "GITHUB_CREATE_JIT_RUNNER_CONFIG_FOR_REPO",
    CREATE_OR_UPDATE_AN_ENVIRONMENT: "GITHUB_CREATE_OR_UPDATE_AN_ENVIRONMENT",
    CREATE_OR_UPDATE_AN_ENVIRONMENT_SECRET: "GITHUB_CREATE_OR_UPDATE_AN_ENVIRONMENT_SECRET",
    CREATE_OR_UPDATE_AN_ORGANIZATION_SECRET: "GITHUB_CREATE_OR_UPDATE_AN_ORGANIZATION_SECRET",
    CREATE_OR_UPDATE_A_REPOSITORY_SECRET: "GITHUB_CREATE_OR_UPDATE_A_REPOSITORY_SECRET",
    CREATE_OR_UPDATE_CUSTOM_PROPERTIES_FOR_ORG: "GITHUB_CREATE_OR_UPDATE_CUSTOM_PROPERTIES_FOR_ORG",
    CREATE_OR_UPDATE_CUSTOM_PROPERTY_ORG: "GITHUB_CREATE_OR_UPDATE_CUSTOM_PROPERTY_ORG",
    CREATE_OR_UPDATE_FILE_CONTENTS: "GITHUB_CREATE_OR_UPDATE_FILE_CONTENTS",
    CREATE_OR_UPDATE_REPO_SECRET_WITH_ENCRYPTED_VALUE: "GITHUB_CREATE_OR_UPDATE_REPO_SECRET_WITH_ENCRYPTED_VALUE",
    CREATE_OR_UPDATE_SECRET_FOR_AUTH_USER: "GITHUB_CREATE_OR_UPDATE_SECRET_FOR_AUTH_USER",
    CREATE_REACTION_FOR_AN_ISSUE: "GITHUB_CREATE_REACTION_FOR_AN_ISSUE",
    CREATE_REACTION_FOR_AN_ISSUE_COMMENT: "GITHUB_CREATE_REACTION_FOR_AN_ISSUE_COMMENT",
    CREATE_REACTION_FOR_A_COMMIT_COMMENT: "GITHUB_CREATE_REACTION_FOR_A_COMMIT_COMMENT",
    CREATE_REACTION_FOR_A_PULL_REQUEST_REVIEW_COMMENT: "GITHUB_CREATE_REACTION_FOR_A_PULL_REQUEST_REVIEW_COMMENT",
    CREATE_REACTION_FOR_A_RELEASE: "GITHUB_CREATE_REACTION_FOR_A_RELEASE",
    CREATE_REACTION_FOR_A_TEAM_DISCUSSION: "GITHUB_CREATE_REACTION_FOR_A_TEAM_DISCUSSION",
    CREATE_REACTION_FOR_A_TEAM_DISCUSSION_COMMENT: "GITHUB_CREATE_REACTION_FOR_A_TEAM_DISCUSSION_COMMENT",
    CREATE_REACTION_FOR_A_TEAM_DISCUSSION_LEGACY: "GITHUB_CREATE_REACTION_FOR_A_TEAM_DISCUSSION_LEGACY",
    CREATE_REACTION_FOR_TEAM_DISCUSSION_COMMENT_LEGACY: "GITHUB_CREATE_REACTION_FOR_TEAM_DISCUSSION_COMMENT_LEGACY",
    CREATE_UPDATE_ORG_SECRET_WITH_LIB_SODIUM: "GITHUB_CREATE_UPDATE_ORG_SECRET_WITH_LIB_SODIUM",
    CUSTOMIZE_OIDC_SUBJECT_CLAIM_TEMPLATE: "GITHUB_CUSTOMIZE_OIDC_SUBJECT_CLAIM_TEMPLATE",
    CUSTOM_OIDCSUBJECT_CLAIM_TEMPLATE: "GITHUB_CUSTOM_OIDCSUBJECT_CLAIM_TEMPLATE",
    CUSTOM_OIDCSUBJECT_CLAIM_TEMPLATE_SETTER: "GITHUB_CUSTOM_OIDCSUBJECT_CLAIM_TEMPLATE_SETTER",
    DECLINE_A_REPOSITORY_INVITATION: "GITHUB_DECLINE_A_REPOSITORY_INVITATION",
    DELETE_ACCESS_RESTRICTIONS: "GITHUB_DELETE_ACCESS_RESTRICTIONS",
    DELETE_ADMIN_BRANCH_PROTECTION: "GITHUB_DELETE_ADMIN_BRANCH_PROTECTION",
    DELETE_AN_APP_AUTHORIZATION: "GITHUB_DELETE_AN_APP_AUTHORIZATION",
    DELETE_AN_APP_TOKEN: "GITHUB_DELETE_AN_APP_TOKEN",
    DELETE_AN_ARTIFACT: "GITHUB_DELETE_AN_ARTIFACT",
    DELETE_AN_AUTOLINK_REFERENCE_FROM_A_REPOSITORY: "GITHUB_DELETE_AN_AUTOLINK_REFERENCE_FROM_A_REPOSITORY",
    DELETE_AN_EMAIL_ADDRESS_FOR_THE_AUTHENTICATED_USER: "GITHUB_DELETE_AN_EMAIL_ADDRESS_FOR_THE_AUTHENTICATED_USER",
    DELETE_AN_ENVIRONMENT: "GITHUB_DELETE_AN_ENVIRONMENT",
    DELETE_AN_ENVIRONMENT_SECRET: "GITHUB_DELETE_AN_ENVIRONMENT_SECRET",
    DELETE_AN_ENVIRONMENT_VARIABLE: "GITHUB_DELETE_AN_ENVIRONMENT_VARIABLE",
    DELETE_AN_ISSUE_COMMENT: "GITHUB_DELETE_AN_ISSUE_COMMENT",
    DELETE_AN_ISSUE_COMMENT_REACTION: "GITHUB_DELETE_AN_ISSUE_COMMENT_REACTION",
    DELETE_AN_ISSUE_REACTION: "GITHUB_DELETE_AN_ISSUE_REACTION",
    DELETE_AN_ORGANIZATION: "GITHUB_DELETE_AN_ORGANIZATION",
    DELETE_AN_ORGANIZATION_MIGRATION_ARCHIVE: "GITHUB_DELETE_AN_ORGANIZATION_MIGRATION_ARCHIVE",
    DELETE_AN_ORGANIZATION_REPOSITORY_RULESET: "GITHUB_DELETE_AN_ORGANIZATION_REPOSITORY_RULESET",
    DELETE_AN_ORGANIZATION_SECRET: "GITHUB_DELETE_AN_ORGANIZATION_SECRET",
    DELETE_AN_ORGANIZATION_VARIABLE: "GITHUB_DELETE_AN_ORGANIZATION_VARIABLE",
    DELETE_AN_ORGANIZATION_WEBHOOK: "GITHUB_DELETE_AN_ORGANIZATION_WEBHOOK",
    DELETE_A_CODESPACE_FOR_THE_AUTHENTICATED_USER: "GITHUB_DELETE_A_CODESPACE_FOR_THE_AUTHENTICATED_USER",
    DELETE_A_CODESPACE_FROM_THE_ORGANIZATION: "GITHUB_DELETE_A_CODESPACE_FROM_THE_ORGANIZATION",
    DELETE_A_CODE_SCANNING_ANALYSIS_FROM_A_REPOSITORY: "GITHUB_DELETE_A_CODE_SCANNING_ANALYSIS_FROM_A_REPOSITORY",
    DELETE_A_COMMIT_COMMENT: "GITHUB_DELETE_A_COMMIT_COMMENT",
    DELETE_A_COMMIT_COMMENT_REACTION: "GITHUB_DELETE_A_COMMIT_COMMENT_REACTION",
    DELETE_A_CUSTOM_ORGANIZATION_ROLE: "GITHUB_DELETE_A_CUSTOM_ORGANIZATION_ROLE",
    DELETE_A_DEPLOYMENT: "GITHUB_DELETE_A_DEPLOYMENT",
    DELETE_A_DEPLOYMENT_BRANCH_POLICY: "GITHUB_DELETE_A_DEPLOYMENT_BRANCH_POLICY",
    DELETE_A_DEPLOY_KEY: "GITHUB_DELETE_A_DEPLOY_KEY",
    DELETE_A_DISCUSSION: "GITHUB_DELETE_A_DISCUSSION",
    DELETE_A_DISCUSSION_COMMENT: "GITHUB_DELETE_A_DISCUSSION_COMMENT",
    DELETE_A_DISCUSSION_COMMENT_LEGACY: "GITHUB_DELETE_A_DISCUSSION_COMMENT_LEGACY",
    DELETE_A_DISCUSSION_LEGACY: "GITHUB_DELETE_A_DISCUSSION_LEGACY",
    DELETE_A_FILE: "GITHUB_DELETE_A_FILE",
    DELETE_A_GIST: "GITHUB_DELETE_A_GIST",
    DELETE_A_GIST_COMMENT: "GITHUB_DELETE_A_GIST_COMMENT",
    DELETE_A_GITHUB_PAGES_SITE: "GITHUB_DELETE_A_GITHUB_PAGES_SITE",
    DELETE_A_GPG_KEY_FOR_THE_AUTHENTICATED_USER: "GITHUB_DELETE_A_GPG_KEY_FOR_THE_AUTHENTICATED_USER",
    DELETE_A_LABEL: "GITHUB_DELETE_A_LABEL",
    DELETE_A_MILESTONE: "GITHUB_DELETE_A_MILESTONE",
    DELETE_A_PACKAGE_FOR_AN_ORGANIZATION: "GITHUB_DELETE_A_PACKAGE_FOR_AN_ORGANIZATION",
    DELETE_A_PACKAGE_FOR_A_USER: "GITHUB_DELETE_A_PACKAGE_FOR_A_USER",
    DELETE_A_PACKAGE_FOR_THE_AUTHENTICATED_USER: "GITHUB_DELETE_A_PACKAGE_FOR_THE_AUTHENTICATED_USER",
    DELETE_A_PACKAGE_VERSION_FOR_THE_AUTHENTICATED_USER: "GITHUB_DELETE_A_PACKAGE_VERSION_FOR_THE_AUTHENTICATED_USER",
    DELETE_A_PENDING_REVIEW_FOR_A_PULL_REQUEST: "GITHUB_DELETE_A_PENDING_REVIEW_FOR_A_PULL_REQUEST",
    DELETE_A_PROJECT: "GITHUB_DELETE_A_PROJECT",
    DELETE_A_PROJECT_CARD: "GITHUB_DELETE_A_PROJECT_CARD",
    DELETE_A_PROJECT_COLUMN: "GITHUB_DELETE_A_PROJECT_COLUMN",
    DELETE_A_PUBLIC_SSH_KEY_FOR_THE_AUTHENTICATED_USER: "GITHUB_DELETE_A_PUBLIC_SSH_KEY_FOR_THE_AUTHENTICATED_USER",
    DELETE_A_PULL_REQUEST_COMMENT_REACTION: "GITHUB_DELETE_A_PULL_REQUEST_COMMENT_REACTION",
    DELETE_A_REFERENCE: "GITHUB_DELETE_A_REFERENCE",
    DELETE_A_RELEASE: "GITHUB_DELETE_A_RELEASE",
    DELETE_A_RELEASE_ASSET: "GITHUB_DELETE_A_RELEASE_ASSET",
    DELETE_A_RELEASE_REACTION: "GITHUB_DELETE_A_RELEASE_REACTION",
    DELETE_A_REPOSITORY: "GITHUB_DELETE_A_REPOSITORY",
    DELETE_A_REPOSITORY_INVITATION: "GITHUB_DELETE_A_REPOSITORY_INVITATION",
    DELETE_A_REPOSITORY_RULESET: "GITHUB_DELETE_A_REPOSITORY_RULESET",
    DELETE_A_REPOSITORY_SECRET: "GITHUB_DELETE_A_REPOSITORY_SECRET",
    DELETE_A_REPOSITORY_SUBSCRIPTION: "GITHUB_DELETE_A_REPOSITORY_SUBSCRIPTION",
    DELETE_A_REPOSITORY_VARIABLE: "GITHUB_DELETE_A_REPOSITORY_VARIABLE",
    DELETE_A_REPOSITORY_WEBHOOK: "GITHUB_DELETE_A_REPOSITORY_WEBHOOK",
    DELETE_A_REVIEW_COMMENT_FOR_A_PULL_REQUEST: "GITHUB_DELETE_A_REVIEW_COMMENT_FOR_A_PULL_REQUEST",
    DELETE_A_SECRET_FOR_THE_AUTHENTICATED_USER: "GITHUB_DELETE_A_SECRET_FOR_THE_AUTHENTICATED_USER",
    DELETE_A_SELF_HOSTED_RUNNER_FROM_AN_ORGANIZATION: "GITHUB_DELETE_A_SELF_HOSTED_RUNNER_FROM_AN_ORGANIZATION",
    DELETE_A_SELF_HOSTED_RUNNER_FROM_A_REPOSITORY: "GITHUB_DELETE_A_SELF_HOSTED_RUNNER_FROM_A_REPOSITORY",
    DELETE_A_TAG_PROTECTION_STATE_FOR_A_REPOSITORY: "GITHUB_DELETE_A_TAG_PROTECTION_STATE_FOR_A_REPOSITORY",
    DELETE_A_TEAM: "GITHUB_DELETE_A_TEAM",
    DELETE_A_TEAM_LEGACY: "GITHUB_DELETE_A_TEAM_LEGACY",
    DELETE_A_THREAD_SUBSCRIPTION: "GITHUB_DELETE_A_THREAD_SUBSCRIPTION",
    DELETE_A_USER_MIGRATION_ARCHIVE: "GITHUB_DELETE_A_USER_MIGRATION_ARCHIVE",
    DELETE_A_WORKFLOW_RUN: "GITHUB_DELETE_A_WORKFLOW_RUN",
    DELETE_BRANCH_PROTECTION: "GITHUB_DELETE_BRANCH_PROTECTION",
    DELETE_COMMIT_SIGNATURE_PROTECTION: "GITHUB_DELETE_COMMIT_SIGNATURE_PROTECTION",
    DELETE_DEPENDEBOT_SECRET_BY_NAME: "GITHUB_DELETE_DEPENDEBOT_SECRET_BY_NAME",
    DELETE_GITHUB_ACTIONS_CACHE_BY_ID: "GITHUB_DELETE_GITHUB_ACTIONS_CACHE_BY_ID",
    DELETE_PACKAGE_VERSION_FOR_AN_ORGANIZATION: "GITHUB_DELETE_PACKAGE_VERSION_FOR_AN_ORGANIZATION",
    DELETE_PACKAGE_VERSION_FOR_A_USER: "GITHUB_DELETE_PACKAGE_VERSION_FOR_A_USER",
    DELETE_PULL_REQUEST_REVIEW_PROTECTION: "GITHUB_DELETE_PULL_REQUEST_REVIEW_PROTECTION",
    DELETE_REPO_CODESPACE_SECRET_BY_NAME: "GITHUB_DELETE_REPO_CODESPACE_SECRET_BY_NAME",
    DELETE_SOCIAL_ACCOUNTS_FOR_THE_AUTHENTICATED_USER: "GITHUB_DELETE_SOCIAL_ACCOUNTS_FOR_THE_AUTHENTICATED_USER",
    DELETE_SSH_SIGNING_KEY_AUTH_USER: "GITHUB_DELETE_SSH_SIGNING_KEY_AUTH_USER",
    DELETE_TEAM_DISCUSSION_COMMENT_REACTION: "GITHUB_DELETE_TEAM_DISCUSSION_COMMENT_REACTION",
    DELETE_TEAM_DISCUSSION_REACTION: "GITHUB_DELETE_TEAM_DISCUSSION_REACTION",
    DELETE_WORKFLOW_RUN_LOGS: "GITHUB_DELETE_WORKFLOW_RUN_LOGS",
    DISABLE_AUTOMATED_SECURITY_FIXES: "GITHUB_DISABLE_AUTOMATED_SECURITY_FIXES",
    DISABLE_A_CUSTOM_PROTECTION_RULE_FOR_AN_ENVIRONMENT: "GITHUB_DISABLE_A_CUSTOM_PROTECTION_RULE_FOR_AN_ENVIRONMENT",
    DISABLE_A_WORKFLOW: "GITHUB_DISABLE_A_WORKFLOW",
    DISABLE_REPOSITORY_ACTIONS_IN_ORG: "GITHUB_DISABLE_REPOSITORY_ACTIONS_IN_ORG",
    DISABLE_VULNERABILITY_ALERTS: "GITHUB_DISABLE_VULNERABILITY_ALERTS",
    DISMISS_A_REVIEW_FOR_A_PULL_REQUEST: "GITHUB_DISMISS_A_REVIEW_FOR_A_PULL_REQUEST",
    DOWNLOAD_AN_ARTIFACT: "GITHUB_DOWNLOAD_AN_ARTIFACT",
    DOWNLOAD_AN_ORGANIZATION_MIGRATION_ARCHIVE: "GITHUB_DOWNLOAD_AN_ORGANIZATION_MIGRATION_ARCHIVE",
    DOWNLOAD_A_REPOSITORY_ARCHIVE_TAR: "GITHUB_DOWNLOAD_A_REPOSITORY_ARCHIVE_TAR",
    DOWNLOAD_A_REPOSITORY_ARCHIVE_ZIP: "GITHUB_DOWNLOAD_A_REPOSITORY_ARCHIVE_ZIP",
    DOWNLOAD_A_USER_MIGRATION_ARCHIVE: "GITHUB_DOWNLOAD_A_USER_MIGRATION_ARCHIVE",
    DOWNLOAD_JOB_LOGS_FOR_A_WORKFLOW_RUN: "GITHUB_DOWNLOAD_JOB_LOGS_FOR_A_WORKFLOW_RUN",
    DOWNLOAD_WORKFLOW_RUN_ATTEMPT_LOGS: "GITHUB_DOWNLOAD_WORKFLOW_RUN_ATTEMPT_LOGS",
    DOWNLOAD_WORKFLOW_RUN_LOGS: "GITHUB_DOWNLOAD_WORKFLOW_RUN_LOGS",
    EMO_J_IS_GET: "GITHUB_EMO_J_IS_GET",
    ENABLE_AUTOMATED_SECURITY_FIXES: "GITHUB_ENABLE_AUTOMATED_SECURITY_FIXES",
    ENABLE_A_WORKFLOW: "GITHUB_ENABLE_A_WORKFLOW",
    ENABLE_GITHUB_ACTIONS_IN_SELECTED_REPOSITORIES: "GITHUB_ENABLE_GITHUB_ACTIONS_IN_SELECTED_REPOSITORIES",
    ENABLE_OR_DISABLE_SECURITY_FEATURE_FOR_ORG: "GITHUB_ENABLE_OR_DISABLE_SECURITY_FEATURE_FOR_ORG",
    ENABLE_REPO_FORGITHUB_ACTIONS: "GITHUB_ENABLE_REPO_FORGITHUB_ACTIONS",
    ENABLE_VULNERABILITY_ALERTS: "GITHUB_ENABLE_VULNERABILITY_ALERTS",
    ENCRYPT_AND_UPDATE_DEV_SECRET: "GITHUB_ENCRYPT_AND_UPDATE_DEV_SECRET",
    ENCRYPT_ORG_DEV_ENV_SECRET: "GITHUB_ENCRYPT_ORG_DEV_ENV_SECRET",
    ENCRYPT_ORG_SECRETS_USING_PUBLIC_KEY: "GITHUB_ENCRYPT_ORG_SECRETS_USING_PUBLIC_KEY",
    EXPORT_A_CODESPACE_FOR_THE_AUTHENTICATED_USER: "GITHUB_EXPORT_A_CODESPACE_FOR_THE_AUTHENTICATED_USER",
    EXPORT_SBOM_FOR_REPO: "GITHUB_EXPORT_SBOM_FOR_REPO",
    FETCH_PUBLIC_KEY_FOR_SECRET_ENCRYPTION: "GITHUB_FETCH_PUBLIC_KEY_FOR_SECRET_ENCRYPTION",
    FIND_CONFLICTING_PACKAGES_FOR_DOCKER_MIGRATION: "GITHUB_FIND_CONFLICTING_PACKAGES_FOR_DOCKER_MIGRATION",
    FOLLOW_A_USER: "GITHUB_FOLLOW_A_USER",
    FORCE_CANCEL_A_WORKFLOW_RUN: "GITHUB_FORCE_CANCEL_A_WORKFLOW_RUN",
    FORK_A_GIST: "GITHUB_FORK_A_GIST",
    GENERATE_RELEASE_NOTES_CONTENT_FOR_A_RELEASE: "GITHUB_GENERATE_RELEASE_NOTES_CONTENT_FOR_A_RELEASE",
    GET_ACCESS_RESTRICTIONS: "GITHUB_GET_ACCESS_RESTRICTIONS",
    GET_ADMIN_BRANCH_PROTECTION: "GITHUB_GET_ADMIN_BRANCH_PROTECTION",
    GET_ALLOWED_ACTIONS_WORKFLOWS_FOR_ORG: "GITHUB_GET_ALLOWED_ACTIONS_WORKFLOWS_FOR_ORG",
    GET_ALL_API_VERSIONS: "GITHUB_GET_ALL_API_VERSIONS",
    GET_ALL_AUTOLINKS_OF_A_REPOSITORY: "GITHUB_GET_ALL_AUTOLINKS_OF_A_REPOSITORY",
    GET_ALL_CODES_OF_CONDUCT: "GITHUB_GET_ALL_CODES_OF_CONDUCT",
    GET_ALL_COMMONLY_USED_LICENSES: "GITHUB_GET_ALL_COMMONLY_USED_LICENSES",
    GET_ALL_CONTRIBUTOR_COMMIT_ACTIVITY: "GITHUB_GET_ALL_CONTRIBUTOR_COMMIT_ACTIVITY",
    GET_ALL_CUSTOM_PROPERTIES_FOR_AN_ORGANIZATION: "GITHUB_GET_ALL_CUSTOM_PROPERTIES_FOR_AN_ORGANIZATION",
    GET_ALL_CUSTOM_PROPERTY_VALUES_FOR_A_REPOSITORY: "GITHUB_GET_ALL_CUSTOM_PROPERTY_VALUES_FOR_A_REPOSITORY",
    GET_ALL_DEPLOYMENT_PROTECTION_RULES_FOR_ENV: "GITHUB_GET_ALL_DEPLOYMENT_PROTECTION_RULES_FOR_ENV",
    GET_ALL_GITIGNORE_TEMPLATES: "GITHUB_GET_ALL_GITIGNORE_TEMPLATES",
    GET_ALL_ORGANIZATION_REPOSITORY_RULESETS: "GITHUB_GET_ALL_ORGANIZATION_REPOSITORY_RULESETS",
    GET_ALL_ORGANIZATION_ROLES_FOR_AN_ORGANIZATION: "GITHUB_GET_ALL_ORGANIZATION_ROLES_FOR_AN_ORGANIZATION",
    GET_ALL_REPOSITORY_RULESETS: "GITHUB_GET_ALL_REPOSITORY_RULESETS",
    GET_ALL_REPOSITORY_TOPICS: "GITHUB_GET_ALL_REPOSITORY_TOPICS",
    GET_ALL_REQUESTED_REVIEWERS_FOR_A_PULL_REQUEST: "GITHUB_GET_ALL_REQUESTED_REVIEWERS_FOR_A_PULL_REQUEST",
    GET_ALL_STATUS_CHECK_CONTEXTS: "GITHUB_GET_ALL_STATUS_CHECK_CONTEXTS",
    GET_AN_APP: "GITHUB_GET_AN_APP",
    GET_AN_ARTIFACT: "GITHUB_GET_AN_ARTIFACT",
    GET_AN_ASSIGNMENT: "GITHUB_GET_AN_ASSIGNMENT",
    GET_AN_AUTOLINK_REFERENCE_OF_A_REPOSITORY: "GITHUB_GET_AN_AUTOLINK_REFERENCE_OF_A_REPOSITORY",
    GET_AN_ENVIRONMENT: "GITHUB_GET_AN_ENVIRONMENT",
    GET_AN_ENVIRONMENT_PUBLIC_KEY: "GITHUB_GET_AN_ENVIRONMENT_PUBLIC_KEY",
    GET_AN_ENVIRONMENT_SECRET: "GITHUB_GET_AN_ENVIRONMENT_SECRET",
    GET_AN_ENVIRONMENT_VARIABLE: "GITHUB_GET_AN_ENVIRONMENT_VARIABLE",
    GET_AN_IMPORT_STATUS: "GITHUB_GET_AN_IMPORT_STATUS",
    GET_AN_ISSUE: "GITHUB_GET_AN_ISSUE",
    GET_AN_ISSUE_COMMENT: "GITHUB_GET_AN_ISSUE_COMMENT",
    GET_AN_ISSUE_EVENT: "GITHUB_GET_AN_ISSUE_EVENT",
    GET_AN_ORGANIZATION: "GITHUB_GET_AN_ORGANIZATION",
    GET_AN_ORGANIZATION_MIGRATION_STATUS: "GITHUB_GET_AN_ORGANIZATION_MIGRATION_STATUS",
    GET_AN_ORGANIZATION_PUBLIC_KEY: "GITHUB_GET_AN_ORGANIZATION_PUBLIC_KEY",
    GET_AN_ORGANIZATION_REPOSITORY_RULESET: "GITHUB_GET_AN_ORGANIZATION_REPOSITORY_RULESET",
    GET_AN_ORGANIZATION_ROLE: "GITHUB_GET_AN_ORGANIZATION_ROLE",
    GET_AN_ORGANIZATION_RULE_SUITE: "GITHUB_GET_AN_ORGANIZATION_RULE_SUITE",
    GET_AN_ORGANIZATION_SECRET: "GITHUB_GET_AN_ORGANIZATION_SECRET",
    GET_AN_ORGANIZATION_VARIABLE: "GITHUB_GET_AN_ORGANIZATION_VARIABLE",
    GET_AN_ORGANIZATION_WEBHOOK: "GITHUB_GET_AN_ORGANIZATION_WEBHOOK",
    GET_AN_SSH_SIGNING_KEY_FOR_THE_AUTHENTICATED_USER: "GITHUB_GET_AN_SSH_SIGNING_KEY_FOR_THE_AUTHENTICATED_USER",
    GET_APPS_WITH_ACCESS_TO_THE_PROTECTED_BRANCH: "GITHUB_GET_APPS_WITH_ACCESS_TO_THE_PROTECTED_BRANCH",
    GET_ASSIGNMENT_GRADES: "GITHUB_GET_ASSIGNMENT_GRADES",
    GET_A_BLOB: "GITHUB_GET_A_BLOB",
    GET_A_BRANCH: "GITHUB_GET_A_BRANCH",
    GET_A_CHECK_RUN: "GITHUB_GET_A_CHECK_RUN",
    GET_A_CHECK_SUITE: "GITHUB_GET_A_CHECK_SUITE",
    GET_A_CLASSROOM: "GITHUB_GET_A_CLASSROOM",
    GET_A_CODESPACE_FOR_THE_AUTHENTICATED_USER: "GITHUB_GET_A_CODESPACE_FOR_THE_AUTHENTICATED_USER",
    GET_A_CODE_OF_CONDUCT: "GITHUB_GET_A_CODE_OF_CONDUCT",
    GET_A_CODE_QL_DATABASE_FOR_A_REPOSITORY: "GITHUB_GET_A_CODE_QL_DATABASE_FOR_A_REPOSITORY",
    GET_A_CODE_SCANNING_ALERT: "GITHUB_GET_A_CODE_SCANNING_ALERT",
    GET_A_CODE_SCANNING_ANALYSIS_FOR_A_REPOSITORY: "GITHUB_GET_A_CODE_SCANNING_ANALYSIS_FOR_A_REPOSITORY",
    GET_A_CODE_SCANNING_DEFAULT_SETUP_CONFIGURATION: "GITHUB_GET_A_CODE_SCANNING_DEFAULT_SETUP_CONFIGURATION",
    GET_A_COMMIT: "GITHUB_GET_A_COMMIT",
    GET_A_COMMIT_COMMENT: "GITHUB_GET_A_COMMIT_COMMENT",
    GET_A_COMMIT_OBJECT: "GITHUB_GET_A_COMMIT_OBJECT",
    GET_A_CUSTOM_DEPLOYMENT_PROTECTION_RULE: "GITHUB_GET_A_CUSTOM_DEPLOYMENT_PROTECTION_RULE",
    GET_A_CUSTOM_PROPERTY_FOR_AN_ORGANIZATION: "GITHUB_GET_A_CUSTOM_PROPERTY_FOR_AN_ORGANIZATION",
    GET_A_DELIVERY_FOR_A_REPOSITORY_WEBHOOK: "GITHUB_GET_A_DELIVERY_FOR_A_REPOSITORY_WEBHOOK",
    GET_A_DEPENDABOT_ALERT: "GITHUB_GET_A_DEPENDABOT_ALERT",
    GET_A_DEPLOYMENT: "GITHUB_GET_A_DEPLOYMENT",
    GET_A_DEPLOYMENT_BRANCH_POLICY: "GITHUB_GET_A_DEPLOYMENT_BRANCH_POLICY",
    GET_A_DEPLOYMENT_STATUS: "GITHUB_GET_A_DEPLOYMENT_STATUS",
    GET_A_DEPLOY_KEY: "GITHUB_GET_A_DEPLOY_KEY",
    GET_A_DIFF_OF_THE_DEPENDENCIES_BETWEEN_COMMITS: "GITHUB_GET_A_DIFF_OF_THE_DEPENDENCIES_BETWEEN_COMMITS",
    GET_A_DISCUSSION: "GITHUB_GET_A_DISCUSSION",
    GET_A_DISCUSSION_COMMENT: "GITHUB_GET_A_DISCUSSION_COMMENT",
    GET_A_DISCUSSION_COMMENT_LEGACY: "GITHUB_GET_A_DISCUSSION_COMMENT_LEGACY",
    GET_A_DISCUSSION_LEGACY: "GITHUB_GET_A_DISCUSSION_LEGACY",
    GET_A_DNS_HEALTH_CHECK_FOR_GITHUB_PAGES: "GITHUB_GET_A_DNS_HEALTH_CHECK_FOR_GITHUB_PAGES",
    GET_A_GIST: "GITHUB_GET_A_GIST",
    GET_A_GIST_COMMENT: "GITHUB_GET_A_GIST_COMMENT",
    GET_A_GIST_REVISION: "GITHUB_GET_A_GIST_REVISION",
    GET_A_GITHUB_PAGES_SITE: "GITHUB_GET_A_GITHUB_PAGES_SITE",
    GET_A_GITIGNORE_TEMPLATE: "GITHUB_GET_A_GITIGNORE_TEMPLATE",
    GET_A_GLOBAL_SECURITY_ADVISORY: "GITHUB_GET_A_GLOBAL_SECURITY_ADVISORY",
    GET_A_GPG_KEY_FOR_THE_AUTHENTICATED_USER: "GITHUB_GET_A_GPG_KEY_FOR_THE_AUTHENTICATED_USER",
    GET_A_JOB_FOR_A_WORKFLOW_RUN: "GITHUB_GET_A_JOB_FOR_A_WORKFLOW_RUN",
    GET_A_LABEL: "GITHUB_GET_A_LABEL",
    GET_A_LICENSE: "GITHUB_GET_A_LICENSE",
    GET_A_MILESTONE: "GITHUB_GET_A_MILESTONE",
    GET_A_PACKAGE_FOR_AN_ORGANIZATION: "GITHUB_GET_A_PACKAGE_FOR_AN_ORGANIZATION",
    GET_A_PACKAGE_FOR_A_USER: "GITHUB_GET_A_PACKAGE_FOR_A_USER",
    GET_A_PACKAGE_FOR_THE_AUTHENTICATED_USER: "GITHUB_GET_A_PACKAGE_FOR_THE_AUTHENTICATED_USER",
    GET_A_PACKAGE_VERSION_FOR_AN_ORGANIZATION: "GITHUB_GET_A_PACKAGE_VERSION_FOR_AN_ORGANIZATION",
    GET_A_PACKAGE_VERSION_FOR_A_USER: "GITHUB_GET_A_PACKAGE_VERSION_FOR_A_USER",
    GET_A_PACKAGE_VERSION_FOR_THE_AUTHENTICATED_USER: "GITHUB_GET_A_PACKAGE_VERSION_FOR_THE_AUTHENTICATED_USER",
    GET_A_PROJECT: "GITHUB_GET_A_PROJECT",
    GET_A_PROJECT_CARD: "GITHUB_GET_A_PROJECT_CARD",
    GET_A_PROJECT_COLUMN: "GITHUB_GET_A_PROJECT_COLUMN",
    GET_A_PUBLIC_SSH_KEY_FOR_THE_AUTHENTICATED_USER: "GITHUB_GET_A_PUBLIC_SSH_KEY_FOR_THE_AUTHENTICATED_USER",
    GET_A_PULL_REQUEST: "GITHUB_GET_A_PULL_REQUEST",
    GET_A_REFERENCE: "GITHUB_GET_A_REFERENCE",
    GET_A_RELEASE: "GITHUB_GET_A_RELEASE",
    GET_A_RELEASE_ASSET: "GITHUB_GET_A_RELEASE_ASSET",
    GET_A_RELEASE_BY_TAG_NAME: "GITHUB_GET_A_RELEASE_BY_TAG_NAME",
    GET_A_REPOSITORY: "GITHUB_GET_A_REPOSITORY",
    GET_A_REPOSITORY_PUBLIC_KEY: "GITHUB_GET_A_REPOSITORY_PUBLIC_KEY",
    GET_A_REPOSITORY_README: "GITHUB_GET_A_REPOSITORY_README",
    GET_A_REPOSITORY_README_FOR_A_DIRECTORY: "GITHUB_GET_A_REPOSITORY_README_FOR_A_DIRECTORY",
    GET_A_REPOSITORY_RULESET: "GITHUB_GET_A_REPOSITORY_RULESET",
    GET_A_REPOSITORY_RULE_SUITE: "GITHUB_GET_A_REPOSITORY_RULE_SUITE",
    GET_A_REPOSITORY_SECRET: "GITHUB_GET_A_REPOSITORY_SECRET",
    GET_A_REPOSITORY_SECURITY_ADVISORY: "GITHUB_GET_A_REPOSITORY_SECURITY_ADVISORY",
    GET_A_REPOSITORY_SUBSCRIPTION: "GITHUB_GET_A_REPOSITORY_SUBSCRIPTION",
    GET_A_REPOSITORY_VARIABLE: "GITHUB_GET_A_REPOSITORY_VARIABLE",
    GET_A_REPOSITORY_WEBHOOK: "GITHUB_GET_A_REPOSITORY_WEBHOOK",
    GET_A_REVIEW_COMMENT_FOR_A_PULL_REQUEST: "GITHUB_GET_A_REVIEW_COMMENT_FOR_A_PULL_REQUEST",
    GET_A_REVIEW_FOR_A_PULL_REQUEST: "GITHUB_GET_A_REVIEW_FOR_A_PULL_REQUEST",
    GET_A_SECRET_FOR_THE_AUTHENTICATED_USER: "GITHUB_GET_A_SECRET_FOR_THE_AUTHENTICATED_USER",
    GET_A_SECRET_SCANNING_ALERT: "GITHUB_GET_A_SECRET_SCANNING_ALERT",
    GET_A_SELF_HOSTED_RUNNER_FOR_AN_ORGANIZATION: "GITHUB_GET_A_SELF_HOSTED_RUNNER_FOR_AN_ORGANIZATION",
    GET_A_SELF_HOSTED_RUNNER_FOR_A_REPOSITORY: "GITHUB_GET_A_SELF_HOSTED_RUNNER_FOR_A_REPOSITORY",
    GET_A_TAG: "GITHUB_GET_A_TAG",
    GET_A_TEAM_BY_NAME: "GITHUB_GET_A_TEAM_BY_NAME",
    GET_A_TEAM_LEGACY: "GITHUB_GET_A_TEAM_LEGACY",
    GET_A_THREAD: "GITHUB_GET_A_THREAD",
    GET_A_TREE: "GITHUB_GET_A_TREE",
    GET_A_USER: "GITHUB_GET_A_USER",
    GET_A_USER_MIGRATION_STATUS: "GITHUB_GET_A_USER_MIGRATION_STATUS",
    GET_A_WEBHOOK_CONFIGURATION_FOR_AN_ORGANIZATION: "GITHUB_GET_A_WEBHOOK_CONFIGURATION_FOR_AN_ORGANIZATION",
    GET_A_WEBHOOK_CONFIGURATION_FOR_A_REPOSITORY: "GITHUB_GET_A_WEBHOOK_CONFIGURATION_FOR_A_REPOSITORY",
    GET_A_WEBHOOK_DELIVERY_FOR_AN_ORGANIZATION_WEBHOOK: "GITHUB_GET_A_WEBHOOK_DELIVERY_FOR_AN_ORGANIZATION_WEBHOOK",
    GET_A_WORKFLOW: "GITHUB_GET_A_WORKFLOW",
    GET_A_WORKFLOW_RUN: "GITHUB_GET_A_WORKFLOW_RUN",
    GET_A_WORKFLOW_RUN_ATTEMPT: "GITHUB_GET_A_WORKFLOW_RUN_ATTEMPT",
    GET_BRANCH_PROTECTION: "GITHUB_GET_BRANCH_PROTECTION",
    GET_COMMIT_AUTHORS: "GITHUB_GET_COMMIT_AUTHORS",
    GET_COMMIT_SIGNATURE_PROTECTION: "GITHUB_GET_COMMIT_SIGNATURE_PROTECTION",
    GET_COMMUNITY_PROFILE_METRICS: "GITHUB_GET_COMMUNITY_PROFILE_METRICS",
    GET_CONTEXTUAL_INFORMATION_FOR_A_USER: "GITHUB_GET_CONTEXTUAL_INFORMATION_FOR_A_USER",
    GET_COPILOT_SEAT_ASSIGNMENT_DETAILS_FOR_A_USER: "GITHUB_GET_COPILOT_SEAT_ASSIGNMENT_DETAILS_FOR_A_USER",
    GET_COPILOT_SEAT_INFO_AND_SETTINGS_FOR_ORG: "GITHUB_GET_COPILOT_SEAT_INFO_AND_SETTINGS_FOR_ORG",
    GET_DEFAULT_ATTRIBUTES_FOR_A_CODESPACE: "GITHUB_GET_DEFAULT_ATTRIBUTES_FOR_A_CODESPACE",
    GET_DEFAULT_WORKFLOW_PERMISSIONS_FOR_A_REPOSITORY: "GITHUB_GET_DEFAULT_WORKFLOW_PERMISSIONS_FOR_A_REPOSITORY",
    GET_DEFAULT_WORKFLOW_PERMS_FOR_ORG: "GITHUB_GET_DEFAULT_WORKFLOW_PERMS_FOR_ORG",
    GET_DETAILS_ABOUT_A_CODESPACE_EXPORT: "GITHUB_GET_DETAILS_ABOUT_A_CODESPACE_EXPORT",
    GET_EMOJIS: "GITHUB_GET_EMOJIS",
    GET_FEEDS: "GITHUB_GET_FEEDS",
    GET_GITHUB_ACTIONS_BILLING_FOR_AN_ORGANIZATION: "GITHUB_GET_GITHUB_ACTIONS_BILLING_FOR_AN_ORGANIZATION",
    GET_GITHUB_ACTIONS_BILLING_FOR_A_USER: "GITHUB_GET_GITHUB_ACTIONS_BILLING_FOR_A_USER",
    GET_GITHUB_ACTIONS_CACHE_USAGE_FOR_AN_ORGANIZATION: "GITHUB_GET_GITHUB_ACTIONS_CACHE_USAGE_FOR_AN_ORGANIZATION",
    GET_GITHUB_ACTIONS_CACHE_USAGE_FOR_A_REPOSITORY: "GITHUB_GET_GITHUB_ACTIONS_CACHE_USAGE_FOR_A_REPOSITORY",
    GET_GITHUB_ACTIONS_PERMISSIONS_FOR_AN_ORGANIZATION: "GITHUB_GET_GITHUB_ACTIONS_PERMISSIONS_FOR_AN_ORGANIZATION",
    GET_GITHUB_ACTIONS_PERMISSIONS_FOR_A_REPOSITORY: "GITHUB_GET_GITHUB_ACTIONS_PERMISSIONS_FOR_A_REPOSITORY",
    GET_GITHUB_META_INFORMATION: "GITHUB_GET_GITHUB_META_INFORMATION",
    GET_GITHUB_PACKAGES_BILLING_FOR_AN_ORGANIZATION: "GITHUB_GET_GITHUB_PACKAGES_BILLING_FOR_AN_ORGANIZATION",
    GET_GITHUB_PACKAGES_BILLING_FOR_A_USER: "GITHUB_GET_GITHUB_PACKAGES_BILLING_FOR_A_USER",
    GET_GITHUB_PAGES_BUILD: "GITHUB_GET_GITHUB_PAGES_BUILD",
    GET_INFORMATION_ABOUT_A_SARIF_UPLOAD: "GITHUB_GET_INFORMATION_ABOUT_A_SARIF_UPLOAD",
    GET_INTERACTION_RESTRICTIONS_FOR_AN_ORGANIZATION: "GITHUB_GET_INTERACTION_RESTRICTIONS_FOR_AN_ORGANIZATION",
    GET_INTERACTION_RESTRICTIONS_FOR_A_REPOSITORY: "GITHUB_GET_INTERACTION_RESTRICTIONS_FOR_A_REPOSITORY",
    GET_INTERACTION_RESTRICTIONS_FOR_PUBLIC_REPOS: "GITHUB_GET_INTERACTION_RESTRICTIONS_FOR_PUBLIC_REPOS",
    GET_LARGE_FILES: "GITHUB_GET_LARGE_FILES",
    GET_LATEST_PAGES_BUILD: "GITHUB_GET_LATEST_PAGES_BUILD",
    GET_OCTOCAT: "GITHUB_GET_OCTOCAT",
    GET_ORGANIZATION_MEMBERSHIP_FOR_A_USER: "GITHUB_GET_ORGANIZATION_MEMBERSHIP_FOR_A_USER",
    GET_ORG_DEV_ENVIRONMENT_SECRET_SAFELY: "GITHUB_GET_ORG_DEV_ENVIRONMENT_SECRET_SAFELY",
    GET_ORG_MEMBERSHIP_FOR_AUTH_USER: "GITHUB_GET_ORG_MEMBERSHIP_FOR_AUTH_USER",
    GET_PAGE_VIEWS: "GITHUB_GET_PAGE_VIEWS",
    GET_PENDING_DEPLOYMENTS_FOR_A_WORKFLOW_RUN: "GITHUB_GET_PENDING_DEPLOYMENTS_FOR_A_WORKFLOW_RUN",
    GET_PROJECT_PERMISSION_FOR_A_USER: "GITHUB_GET_PROJECT_PERMISSION_FOR_A_USER",
    GET_PUBLIC_KEY_FOR_SECRET_ENCRYPTION: "GITHUB_GET_PUBLIC_KEY_FOR_SECRET_ENCRYPTION",
    GET_PUBLIC_KEY_FOR_THE_AUTHENTICATED_USER: "GITHUB_GET_PUBLIC_KEY_FOR_THE_AUTHENTICATED_USER",
    GET_PULL_REQUEST_REVIEW_PROTECTION: "GITHUB_GET_PULL_REQUEST_REVIEW_PROTECTION",
    GET_RATE_LIMIT_STATUS_FOR_THE_AUTHENTICATED_USER: "GITHUB_GET_RATE_LIMIT_STATUS_FOR_THE_AUTHENTICATED_USER",
    GET_REPOSITORY_CLONES: "GITHUB_GET_REPOSITORY_CLONES",
    GET_REPOSITORY_CONTENT: "GITHUB_GET_REPOSITORY_CONTENT",
    GET_REPOSITORY_PERMISSIONS_FOR_A_USER: "GITHUB_GET_REPOSITORY_PERMISSIONS_FOR_A_USER",
    GET_REPOSITORY_SECRET_SECURELY: "GITHUB_GET_REPOSITORY_SECRET_SECURELY",
    GET_REPO_ALLOWED_ACTIONS: "GITHUB_GET_REPO_ALLOWED_ACTIONS",
    GET_REPO_DEV_ENV_SECRET: "GITHUB_GET_REPO_DEV_ENV_SECRET",
    GET_RULES_FOR_A_BRANCH: "GITHUB_GET_RULES_FOR_A_BRANCH",
    GET_SHARED_STORAGE_BILLING_FOR_AN_ORGANIZATION: "GITHUB_GET_SHARED_STORAGE_BILLING_FOR_AN_ORGANIZATION",
    GET_SHARED_STORAGE_BILLING_FOR_A_USER: "GITHUB_GET_SHARED_STORAGE_BILLING_FOR_A_USER",
    GET_SINGLE_ORG_SECRET_WITHOUT_DECRYPTION: "GITHUB_GET_SINGLE_ORG_SECRET_WITHOUT_DECRYPTION",
    GET_STATUS_CHECKS_PROTECTION: "GITHUB_GET_STATUS_CHECKS_PROTECTION",
    GET_TEAMS_WITH_ACCESS_TO_THE_PROTECTED_BRANCH: "GITHUB_GET_TEAMS_WITH_ACCESS_TO_THE_PROTECTED_BRANCH",
    GET_TEAM_MEMBERSHIP_FOR_A_USER: "GITHUB_GET_TEAM_MEMBERSHIP_FOR_A_USER",
    GET_TEAM_MEMBERSHIP_FOR_A_USER_LEGACY: "GITHUB_GET_TEAM_MEMBERSHIP_FOR_A_USER_LEGACY",
    GET_TEAM_MEMBER_LEGACY: "GITHUB_GET_TEAM_MEMBER_LEGACY",
    GET_THE_AUTHENTICATED_USER: "GITHUB_GET_THE_AUTHENTICATED_USER",
    GET_THE_COMBINED_STATUS_FOR_A_SPECIFIC_REFERENCE: "GITHUB_GET_THE_COMBINED_STATUS_FOR_A_SPECIFIC_REFERENCE",
    GET_THE_HOURLY_COMMIT_COUNT_FOR_EACH_DAY: "GITHUB_GET_THE_HOURLY_COMMIT_COUNT_FOR_EACH_DAY",
    GET_THE_LAST_YEAR_OF_COMMIT_ACTIVITY: "GITHUB_GET_THE_LAST_YEAR_OF_COMMIT_ACTIVITY",
    GET_THE_LATEST_RELEASE: "GITHUB_GET_THE_LATEST_RELEASE",
    GET_THE_LICENSE_FOR_A_REPOSITORY: "GITHUB_GET_THE_LICENSE_FOR_A_REPOSITORY",
    GET_THE_REVIEW_HISTORY_FOR_A_WORKFLOW_RUN: "GITHUB_GET_THE_REVIEW_HISTORY_FOR_A_WORKFLOW_RUN",
    GET_THE_STATUS_OF_A_GITHUB_PAGES_DEPLOYMENT: "GITHUB_GET_THE_STATUS_OF_A_GITHUB_PAGES_DEPLOYMENT",
    GET_THE_WEEKLY_COMMIT_ACTIVITY: "GITHUB_GET_THE_WEEKLY_COMMIT_ACTIVITY",
    GET_THE_WEEKLY_COMMIT_COUNT: "GITHUB_GET_THE_WEEKLY_COMMIT_COUNT",
    GET_THE_ZEN_OF_GITHUB: "GITHUB_GET_THE_ZEN_OF_GITHUB",
    GET_THREAD_SUBSCRIPTION_FOR_AUTH_USER: "GITHUB_GET_THREAD_SUBSCRIPTION_FOR_AUTH_USER",
    GET_TOP_REFERRAL_PATHS: "GITHUB_GET_TOP_REFERRAL_PATHS",
    GET_TOP_REFERRAL_SOURCES: "GITHUB_GET_TOP_REFERRAL_SOURCES",
    GET_USERS_WITH_ACCESS_TO_THE_PROTECTED_BRANCH: "GITHUB_GET_USERS_WITH_ACCESS_TO_THE_PROTECTED_BRANCH",
    GET_WORKFLOW_ACCESS_LEVEL_OUTSIDE_REPO: "GITHUB_GET_WORKFLOW_ACCESS_LEVEL_OUTSIDE_REPO",
    GET_WORKFLOW_RUN_USAGE: "GITHUB_GET_WORKFLOW_RUN_USAGE",
    GET_WORKFLOW_USAGE: "GITHUB_GET_WORKFLOW_USAGE",
    GIST_S_CREATE: "GITHUB_GIST_S_CREATE",
    GIST_S_LIST_PUBLIC: "GITHUB_GIST_S_LIST_PUBLIC",
    GITHUB_API_ROOT: "GITHUB_GITHUB_API_ROOT",
    GITHUB_DISABLE_PRIVATE_VULN_REPORTING_FOR_REPO: "GITHUB_GITHUB_DISABLE_PRIVATE_VULN_REPORTING_FOR_REPO",
    GITHUB_ENABLE_PRIVATE_VULN_REPORTING_FOR_REPO: "GITHUB_GITHUB_ENABLE_PRIVATE_VULN_REPORTING_FOR_REPO",
    GITHUB_PULLS_CHECK_IF_MERGED: "GITHUB_GITHUB_PULLS_CHECK_IF_MERGED",
    ISSUES_CREATE: "GITHUB_ISSUES_CREATE",
    ISSUES_CREATE_COMMENT: "GITHUB_ISSUES_CREATE_COMMENT",
    ISSUES_GET: "GITHUB_ISSUES_GET",
    ISSUES_LIST_ASSIGN_EES: "GITHUB_ISSUES_LIST_ASSIGN_EES",
    ISSUES_LIST_EVENTS_FOR_REPO: "GITHUB_ISSUES_LIST_EVENTS_FOR_REPO",
    ISSUES_LIST_FOR_REPO: "GITHUB_ISSUES_LIST_FOR_REPO",
    LIST_ACCEPTED_ASSIGNMENTS_FOR_AN_ASSIGNMENT: "GITHUB_LIST_ACCEPTED_ASSIGNMENTS_FOR_AN_ASSIGNMENT",
    LIST_APP_INSTALLATIONS_FOR_AN_ORGANIZATION: "GITHUB_LIST_APP_INSTALLATIONS_FOR_AN_ORGANIZATION",
    LIST_APP_INSTALLATIONS_FOR_USER_TOKEN: "GITHUB_LIST_APP_INSTALLATIONS_FOR_USER_TOKEN",
    LIST_ARTIFACTS_FOR_A_REPOSITORY: "GITHUB_LIST_ARTIFACTS_FOR_A_REPOSITORY",
    LIST_ASSIGNEES: "GITHUB_LIST_ASSIGNEES",
    LIST_ASSIGNMENTS_FOR_A_CLASSROOM: "GITHUB_LIST_ASSIGNMENTS_FOR_A_CLASSROOM",
    LIST_AVAILABLE_MACHINE_TYPES_FOR_A_REPOSITORY: "GITHUB_LIST_AVAILABLE_MACHINE_TYPES_FOR_A_REPOSITORY",
    LIST_BRANCHES: "GITHUB_LIST_BRANCHES",
    LIST_BRANCHES_FOR_HEAD_COMMIT: "GITHUB_LIST_BRANCHES_FOR_HEAD_COMMIT",
    LIST_CHECK_RUNS_FOR_A_GIT_REFERENCE: "GITHUB_LIST_CHECK_RUNS_FOR_A_GIT_REFERENCE",
    LIST_CHECK_RUNS_IN_A_CHECK_SUITE: "GITHUB_LIST_CHECK_RUNS_IN_A_CHECK_SUITE",
    LIST_CHECK_RUN_ANNOTATIONS: "GITHUB_LIST_CHECK_RUN_ANNOTATIONS",
    LIST_CHECK_SUITES_FOR_A_GIT_REFERENCE: "GITHUB_LIST_CHECK_SUITES_FOR_A_GIT_REFERENCE",
    LIST_CHILD_TEAMS: "GITHUB_LIST_CHILD_TEAMS",
    LIST_CHILD_TEAMS_LEGACY: "GITHUB_LIST_CHILD_TEAMS_LEGACY",
    LIST_CLASSROOMS: "GITHUB_LIST_CLASSROOMS",
    LIST_CODEOWNERS_ERRORS: "GITHUB_LIST_CODEOWNERS_ERRORS",
    LIST_CODESPACES_FOR_A_USER_IN_ORGANIZATION: "GITHUB_LIST_CODESPACES_FOR_A_USER_IN_ORGANIZATION",
    LIST_CODESPACES_FOR_THE_AUTHENTICATED_USER: "GITHUB_LIST_CODESPACES_FOR_THE_AUTHENTICATED_USER",
    LIST_CODESPACES_FOR_THE_ORGANIZATION: "GITHUB_LIST_CODESPACES_FOR_THE_ORGANIZATION",
    LIST_CODE_QL_DATABASES_FOR_A_REPOSITORY: "GITHUB_LIST_CODE_QL_DATABASES_FOR_A_REPOSITORY",
    LIST_CODE_SCANNING_ALERTS_FOR_AN_ORGANIZATION: "GITHUB_LIST_CODE_SCANNING_ALERTS_FOR_AN_ORGANIZATION",
    LIST_CODE_SCANNING_ALERTS_FOR_A_REPOSITORY: "GITHUB_LIST_CODE_SCANNING_ALERTS_FOR_A_REPOSITORY",
    LIST_CODE_SCANNING_ANALYSES_FOR_A_REPOSITORY: "GITHUB_LIST_CODE_SCANNING_ANALYSES_FOR_A_REPOSITORY",
    LIST_COMMENTS_FOR_A_PULL_REQUEST_REVIEW: "GITHUB_LIST_COMMENTS_FOR_A_PULL_REQUEST_REVIEW",
    LIST_COMMITS: "GITHUB_LIST_COMMITS",
    LIST_COMMITS_ON_A_PULL_REQUEST: "GITHUB_LIST_COMMITS_ON_A_PULL_REQUEST",
    LIST_COMMIT_COMMENTS: "GITHUB_LIST_COMMIT_COMMENTS",
    LIST_COMMIT_COMMENTS_FOR_A_REPOSITORY: "GITHUB_LIST_COMMIT_COMMENTS_FOR_A_REPOSITORY",
    LIST_COMMIT_STATUSES_FOR_A_REFERENCE: "GITHUB_LIST_COMMIT_STATUSES_FOR_A_REFERENCE",
    LIST_COPILOT_SEAT_ASSIGNMENTS_FOR_ORG: "GITHUB_LIST_COPILOT_SEAT_ASSIGNMENTS_FOR_ORG",
    LIST_CUSTOM_PROPERTY_VALUES_FOR_ORG_REPOS: "GITHUB_LIST_CUSTOM_PROPERTY_VALUES_FOR_ORG_REPOS",
    LIST_DELIVERIES_FOR_AN_ORGANIZATION_WEBHOOK: "GITHUB_LIST_DELIVERIES_FOR_AN_ORGANIZATION_WEBHOOK",
    LIST_DELIVERIES_FOR_A_REPOSITORY_WEBHOOK: "GITHUB_LIST_DELIVERIES_FOR_A_REPOSITORY_WEBHOOK",
    LIST_DEPENDABOT_ALERTS_FOR_AN_ENTERPRISE: "GITHUB_LIST_DEPENDABOT_ALERTS_FOR_AN_ENTERPRISE",
    LIST_DEPENDABOT_ALERTS_FOR_AN_ORGANIZATION: "GITHUB_LIST_DEPENDABOT_ALERTS_FOR_AN_ORGANIZATION",
    LIST_DEPENDABOT_ALERTS_FOR_A_REPOSITORY: "GITHUB_LIST_DEPENDABOT_ALERTS_FOR_A_REPOSITORY",
    LIST_DEPLOYMENTS: "GITHUB_LIST_DEPLOYMENTS",
    LIST_DEPLOYMENT_BRANCH_POLICIES: "GITHUB_LIST_DEPLOYMENT_BRANCH_POLICIES",
    LIST_DEPLOYMENT_STATUSES: "GITHUB_LIST_DEPLOYMENT_STATUSES",
    LIST_DEPLOY_KEYS: "GITHUB_LIST_DEPLOY_KEYS",
    LIST_DISCUSSIONS: "GITHUB_LIST_DISCUSSIONS",
    LIST_DISCUSSIONS_LEGACY: "GITHUB_LIST_DISCUSSIONS_LEGACY",
    LIST_DISCUSSION_COMMENTS: "GITHUB_LIST_DISCUSSION_COMMENTS",
    LIST_DISCUSSION_COMMENTS_LEGACY: "GITHUB_LIST_DISCUSSION_COMMENTS_LEGACY",
    LIST_DOCKER_MIGRATION_CONFLICTS: "GITHUB_LIST_DOCKER_MIGRATION_CONFLICTS",
    LIST_EMAIL_ADDRESSES_FOR_THE_AUTHENTICATED_USER: "GITHUB_LIST_EMAIL_ADDRESSES_FOR_THE_AUTHENTICATED_USER",
    LIST_ENVIRONMENTS: "GITHUB_LIST_ENVIRONMENTS",
    LIST_ENVIRONMENT_CUSTOM_DEPLOYMENT_RULES: "GITHUB_LIST_ENVIRONMENT_CUSTOM_DEPLOYMENT_RULES",
    LIST_ENVIRONMENT_SECRETS: "GITHUB_LIST_ENVIRONMENT_SECRETS",
    LIST_ENVIRONMENT_VARIABLES: "GITHUB_LIST_ENVIRONMENT_VARIABLES",
    LIST_EVENTS_FOR_THE_AUTHENTICATED_USER: "GITHUB_LIST_EVENTS_FOR_THE_AUTHENTICATED_USER",
    LIST_EVENTS_RECEIVED_BY_THE_AUTHENTICATED_USER: "GITHUB_LIST_EVENTS_RECEIVED_BY_THE_AUTHENTICATED_USER",
    LIST_FAILED_ORGANIZATION_INVITATIONS: "GITHUB_LIST_FAILED_ORGANIZATION_INVITATIONS",
    LIST_FOLLOWERS_OF_A_USER: "GITHUB_LIST_FOLLOWERS_OF_A_USER",
    LIST_FOLLOWERS_OF_THE_AUTHENTICATED_USER: "GITHUB_LIST_FOLLOWERS_OF_THE_AUTHENTICATED_USER",
    LIST_FORKS: "GITHUB_LIST_FORKS",
    LIST_GISTS_FOR_A_USER: "GITHUB_LIST_GISTS_FOR_A_USER",
    LIST_GISTS_FOR_THE_AUTHENTICATED_USER: "GITHUB_LIST_GISTS_FOR_THE_AUTHENTICATED_USER",
    LIST_GIST_COMMENTS: "GITHUB_LIST_GIST_COMMENTS",
    LIST_GIST_COMMITS: "GITHUB_LIST_GIST_COMMITS",
    LIST_GIST_FORKS: "GITHUB_LIST_GIST_FORKS",
    LIST_GITHUB_ACTIONS_CACHES_FOR_A_REPOSITORY: "GITHUB_LIST_GITHUB_ACTIONS_CACHES_FOR_A_REPOSITORY",
    LIST_GITHUB_PAGES_BUILDS: "GITHUB_LIST_GITHUB_PAGES_BUILDS",
    LIST_GLOBAL_SECURITY_ADVISORIES: "GITHUB_LIST_GLOBAL_SECURITY_ADVISORIES",
    LIST_GPG_KEYS_FOR_A_USER: "GITHUB_LIST_GPG_KEYS_FOR_A_USER",
    LIST_GPG_KEYS_FOR_THE_AUTHENTICATED_USER: "GITHUB_LIST_GPG_KEYS_FOR_THE_AUTHENTICATED_USER",
    LIST_INSTALLATION_REQUESTS_FOR_AUTH_APP: "GITHUB_LIST_INSTALLATION_REQUESTS_FOR_AUTH_APP",
    LIST_INSTANCES_OF_A_CODE_SCANNING_ALERT: "GITHUB_LIST_INSTANCES_OF_A_CODE_SCANNING_ALERT",
    LIST_ISSUES_ASSIGNED_TO_THE_AUTHENTICATED_USER: "GITHUB_LIST_ISSUES_ASSIGNED_TO_THE_AUTHENTICATED_USER",
    LIST_ISSUE_COMMENTS: "GITHUB_LIST_ISSUE_COMMENTS",
    LIST_ISSUE_COMMENTS_FOR_A_REPOSITORY: "GITHUB_LIST_ISSUE_COMMENTS_FOR_A_REPOSITORY",
    LIST_ISSUE_EVENTS: "GITHUB_LIST_ISSUE_EVENTS",
    LIST_ISSUE_EVENTS_FOR_A_REPOSITORY: "GITHUB_LIST_ISSUE_EVENTS_FOR_A_REPOSITORY",
    LIST_JOBS_FOR_A_WORKFLOW_RUN: "GITHUB_LIST_JOBS_FOR_A_WORKFLOW_RUN",
    LIST_JOBS_FOR_A_WORKFLOW_RUN_ATTEMPT: "GITHUB_LIST_JOBS_FOR_A_WORKFLOW_RUN_ATTEMPT",
    LIST_LABELS_FOR_AN_ISSUE: "GITHUB_LIST_LABELS_FOR_AN_ISSUE",
    LIST_LABELS_FOR_A_REPOSITORY: "GITHUB_LIST_LABELS_FOR_A_REPOSITORY",
    LIST_LABELS_FOR_ISSUES_IN_A_MILESTONE: "GITHUB_LIST_LABELS_FOR_ISSUES_IN_A_MILESTONE",
    LIST_LABELS_FOR_SELF_HOSTED_RUNNER_FOR_REPO: "GITHUB_LIST_LABELS_FOR_SELF_HOSTED_RUNNER_FOR_REPO",
    LIST_LABELS_FOR_SELF_HOSTED_RUNNER_ORG: "GITHUB_LIST_LABELS_FOR_SELF_HOSTED_RUNNER_ORG",
    LIST_LOCATIONS_FOR_A_SECRET_SCANNING_ALERT: "GITHUB_LIST_LOCATIONS_FOR_A_SECRET_SCANNING_ALERT",
    LIST_MACHINE_TYPES_FOR_A_CODESPACE: "GITHUB_LIST_MACHINE_TYPES_FOR_A_CODESPACE",
    LIST_MATCHING_REFERENCES: "GITHUB_LIST_MATCHING_REFERENCES",
    LIST_MILESTONES: "GITHUB_LIST_MILESTONES",
    LIST_NOTIFICATIONS_FOR_THE_AUTHENTICATED_USER: "GITHUB_LIST_NOTIFICATIONS_FOR_THE_AUTHENTICATED_USER",
    LIST_ORGANIZATIONS: "GITHUB_LIST_ORGANIZATIONS",
    LIST_ORGANIZATIONS_FOR_A_USER: "GITHUB_LIST_ORGANIZATIONS_FOR_A_USER",
    LIST_ORGANIZATIONS_FOR_THE_AUTHENTICATED_USER: "GITHUB_LIST_ORGANIZATIONS_FOR_THE_AUTHENTICATED_USER",
    LIST_ORGANIZATION_EVENTS_FOR_THE_AUTHENTICATED_USER: "GITHUB_LIST_ORGANIZATION_EVENTS_FOR_THE_AUTHENTICATED_USER",
    LIST_ORGANIZATION_INVITATION_TEAMS: "GITHUB_LIST_ORGANIZATION_INVITATION_TEAMS",
    LIST_ORGANIZATION_MEMBERS: "GITHUB_LIST_ORGANIZATION_MEMBERS",
    LIST_ORGANIZATION_MIGRATIONS: "GITHUB_LIST_ORGANIZATION_MIGRATIONS",
    LIST_ORGANIZATION_PROJECTS: "GITHUB_LIST_ORGANIZATION_PROJECTS",
    LIST_ORGANIZATION_REPOSITORIES: "GITHUB_LIST_ORGANIZATION_REPOSITORIES",
    LIST_ORGANIZATION_RULE_SUITES: "GITHUB_LIST_ORGANIZATION_RULE_SUITES",
    LIST_ORGANIZATION_SECRETS: "GITHUB_LIST_ORGANIZATION_SECRETS",
    LIST_ORGANIZATION_VARIABLES: "GITHUB_LIST_ORGANIZATION_VARIABLES",
    LIST_ORGANIZATION_WEBHOOKS: "GITHUB_LIST_ORGANIZATION_WEBHOOKS",
    LIST_ORG_FINE_GRAINED_PERMISSIONS: "GITHUB_LIST_ORG_FINE_GRAINED_PERMISSIONS",
    LIST_ORG_ISSUES_ASSIGNED_AUTH_USER: "GITHUB_LIST_ORG_ISSUES_ASSIGNED_AUTH_USER",
    LIST_ORG_LEVEL_CODESPACES_SECRETS: "GITHUB_LIST_ORG_LEVEL_CODESPACES_SECRETS",
    LIST_ORG_MEMBERSHIPS_FOR_AUTH_USER: "GITHUB_LIST_ORG_MEMBERSHIPS_FOR_AUTH_USER",
    LIST_ORG_REPOS_WITHGITHUB_ACTIONS_ENABLED: "GITHUB_LIST_ORG_REPOS_WITHGITHUB_ACTIONS_ENABLED",
    LIST_ORG_RESOURCES_WITH_PERSONAL_TOKENS: "GITHUB_LIST_ORG_RESOURCES_WITH_PERSONAL_TOKENS",
    LIST_ORG_RESOURCE_ACCESS_TOKENS: "GITHUB_LIST_ORG_RESOURCE_ACCESS_TOKENS",
    LIST_ORG_SECRETS_WITHOUT_VALUES: "GITHUB_LIST_ORG_SECRETS_WITHOUT_VALUES",
    LIST_OUTSIDE_COLLABORATORS_FOR_AN_ORGANIZATION: "GITHUB_LIST_OUTSIDE_COLLABORATORS_FOR_AN_ORGANIZATION",
    LIST_OWNED_PACKAGE_VERSIONS: "GITHUB_LIST_OWNED_PACKAGE_VERSIONS",
    LIST_PACKAGES_FOR_AN_ORGANIZATION: "GITHUB_LIST_PACKAGES_FOR_AN_ORGANIZATION",
    LIST_PACKAGES_FOR_A_USER: "GITHUB_LIST_PACKAGES_FOR_A_USER",
    LIST_PACKAGES_FOR_THE_AUTHENTICATED_USER_S_NAMESPACE: "GITHUB_LIST_PACKAGES_FOR_THE_AUTHENTICATED_USER_S_NAMESPACE",
    LIST_PACKAGE_VERSIONS_FOR_A_PACKAGE_OWNED_BY_A_USER: "GITHUB_LIST_PACKAGE_VERSIONS_FOR_A_PACKAGE_OWNED_BY_A_USER",
    LIST_PACKAGE_VERSIONS_FOR_ORG_PACKAGE: "GITHUB_LIST_PACKAGE_VERSIONS_FOR_ORG_PACKAGE",
    LIST_PENDING_ORGANIZATION_INVITATIONS: "GITHUB_LIST_PENDING_ORGANIZATION_INVITATIONS",
    LIST_PENDING_TEAM_INVITATIONS: "GITHUB_LIST_PENDING_TEAM_INVITATIONS",
    LIST_PENDING_TEAM_INVITATIONS_LEGACY: "GITHUB_LIST_PENDING_TEAM_INVITATIONS_LEGACY",
    LIST_PROJECT_CARDS: "GITHUB_LIST_PROJECT_CARDS",
    LIST_PROJECT_COLLABORATORS: "GITHUB_LIST_PROJECT_COLLABORATORS",
    LIST_PROJECT_COLUMNS: "GITHUB_LIST_PROJECT_COLUMNS",
    LIST_PUBLIC_EMAILS_FOR_AUTH_USER: "GITHUB_LIST_PUBLIC_EMAILS_FOR_AUTH_USER",
    LIST_PUBLIC_EVENTS: "GITHUB_LIST_PUBLIC_EVENTS",
    LIST_PUBLIC_EVENTS_FOR_A_NETWORK_OF_REPOSITORIES: "GITHUB_LIST_PUBLIC_EVENTS_FOR_A_NETWORK_OF_REPOSITORIES",
    LIST_PUBLIC_EVENTS_FOR_A_USER: "GITHUB_LIST_PUBLIC_EVENTS_FOR_A_USER",
    LIST_PUBLIC_EVENTS_RECEIVED_BY_A_USER: "GITHUB_LIST_PUBLIC_EVENTS_RECEIVED_BY_A_USER",
    LIST_PUBLIC_GISTS: "GITHUB_LIST_PUBLIC_GISTS",
    LIST_PUBLIC_KEYS_FOR_A_USER: "GITHUB_LIST_PUBLIC_KEYS_FOR_A_USER",
    LIST_PUBLIC_ORGANIZATION_EVENTS: "GITHUB_LIST_PUBLIC_ORGANIZATION_EVENTS",
    LIST_PUBLIC_ORGANIZATION_MEMBERS: "GITHUB_LIST_PUBLIC_ORGANIZATION_MEMBERS",
    LIST_PUBLIC_REPOSITORIES: "GITHUB_LIST_PUBLIC_REPOSITORIES",
    LIST_PUBLIC_SSH_KEYS_FOR_THE_AUTHENTICATED_USER: "GITHUB_LIST_PUBLIC_SSH_KEYS_FOR_THE_AUTHENTICATED_USER",
    LIST_PULL_REQUESTS: "GITHUB_LIST_PULL_REQUESTS",
    LIST_PULL_REQUESTS_ASSOCIATED_WITH_A_COMMIT: "GITHUB_LIST_PULL_REQUESTS_ASSOCIATED_WITH_A_COMMIT",
    LIST_PULL_REQUESTS_FILES: "GITHUB_LIST_PULL_REQUESTS_FILES",
    LIST_REACTIONS_FOR_AN_ISSUE: "GITHUB_LIST_REACTIONS_FOR_AN_ISSUE",
    LIST_REACTIONS_FOR_AN_ISSUE_COMMENT: "GITHUB_LIST_REACTIONS_FOR_AN_ISSUE_COMMENT",
    LIST_REACTIONS_FOR_A_COMMIT_COMMENT: "GITHUB_LIST_REACTIONS_FOR_A_COMMIT_COMMENT",
    LIST_REACTIONS_FOR_A_PULL_REQUEST_REVIEW_COMMENT: "GITHUB_LIST_REACTIONS_FOR_A_PULL_REQUEST_REVIEW_COMMENT",
    LIST_REACTIONS_FOR_A_RELEASE: "GITHUB_LIST_REACTIONS_FOR_A_RELEASE",
    LIST_REACTIONS_FOR_A_TEAM_DISCUSSION: "GITHUB_LIST_REACTIONS_FOR_A_TEAM_DISCUSSION",
    LIST_REACTIONS_FOR_A_TEAM_DISCUSSION_COMMENT: "GITHUB_LIST_REACTIONS_FOR_A_TEAM_DISCUSSION_COMMENT",
    LIST_REACTIONS_FOR_A_TEAM_DISCUSSION_COMMENT_LEGACY: "GITHUB_LIST_REACTIONS_FOR_A_TEAM_DISCUSSION_COMMENT_LEGACY",
    LIST_REACTIONS_FOR_A_TEAM_DISCUSSION_LEGACY: "GITHUB_LIST_REACTIONS_FOR_A_TEAM_DISCUSSION_LEGACY",
    LIST_RELEASES: "GITHUB_LIST_RELEASES",
    LIST_RELEASE_ASSETS: "GITHUB_LIST_RELEASE_ASSETS",
    LIST_REPOSITORIES_FOR_A_USER: "GITHUB_LIST_REPOSITORIES_FOR_A_USER",
    LIST_REPOSITORIES_FOR_A_USER_MIGRATION: "GITHUB_LIST_REPOSITORIES_FOR_A_USER_MIGRATION",
    LIST_REPOSITORIES_FOR_THE_AUTHENTICATED_USER: "GITHUB_LIST_REPOSITORIES_FOR_THE_AUTHENTICATED_USER",
    LIST_REPOSITORIES_IN_AN_ORGANIZATION_MIGRATION: "GITHUB_LIST_REPOSITORIES_IN_AN_ORGANIZATION_MIGRATION",
    LIST_REPOSITORIES_STARRED_BY_A_USER: "GITHUB_LIST_REPOSITORIES_STARRED_BY_A_USER",
    LIST_REPOSITORIES_STARRED_BY_THE_AUTHENTICATED_USER: "GITHUB_LIST_REPOSITORIES_STARRED_BY_THE_AUTHENTICATED_USER",
    LIST_REPOSITORIES_WATCHED_BY_A_USER: "GITHUB_LIST_REPOSITORIES_WATCHED_BY_A_USER",
    LIST_REPOSITORIES_WATCHED_BY_THE_AUTHENTICATED_USER: "GITHUB_LIST_REPOSITORIES_WATCHED_BY_THE_AUTHENTICATED_USER",
    LIST_REPOSITORY_ACTIVITIES: "GITHUB_LIST_REPOSITORY_ACTIVITIES",
    LIST_REPOSITORY_COLLABORATORS: "GITHUB_LIST_REPOSITORY_COLLABORATORS",
    LIST_REPOSITORY_CONTRIBUTORS: "GITHUB_LIST_REPOSITORY_CONTRIBUTORS",
    LIST_REPOSITORY_EVENTS: "GITHUB_LIST_REPOSITORY_EVENTS",
    LIST_REPOSITORY_INVITATIONS: "GITHUB_LIST_REPOSITORY_INVITATIONS",
    LIST_REPOSITORY_ISSUES: "GITHUB_LIST_REPOSITORY_ISSUES",
    LIST_REPOSITORY_LANGUAGES: "GITHUB_LIST_REPOSITORY_LANGUAGES",
    LIST_REPOSITORY_ORGANIZATION_SECRETS: "GITHUB_LIST_REPOSITORY_ORGANIZATION_SECRETS",
    LIST_REPOSITORY_ORGANIZATION_VARIABLES: "GITHUB_LIST_REPOSITORY_ORGANIZATION_VARIABLES",
    LIST_REPOSITORY_PROJECTS: "GITHUB_LIST_REPOSITORY_PROJECTS",
    LIST_REPOSITORY_RULE_SUITES: "GITHUB_LIST_REPOSITORY_RULE_SUITES",
    LIST_REPOSITORY_SECRETS: "GITHUB_LIST_REPOSITORY_SECRETS",
    LIST_REPOSITORY_SECRETS_WITHOUT_DECRYPTING: "GITHUB_LIST_REPOSITORY_SECRETS_WITHOUT_DECRYPTING",
    LIST_REPOSITORY_SECURITY_ADVISORIES: "GITHUB_LIST_REPOSITORY_SECURITY_ADVISORIES",
    LIST_REPOSITORY_TAGS: "GITHUB_LIST_REPOSITORY_TAGS",
    LIST_REPOSITORY_TEAMS: "GITHUB_LIST_REPOSITORY_TEAMS",
    LIST_REPOSITORY_VARIABLES: "GITHUB_LIST_REPOSITORY_VARIABLES",
    LIST_REPOSITORY_WEBHOOKS: "GITHUB_LIST_REPOSITORY_WEBHOOKS",
    LIST_REPOSITORY_WORKFLOWS: "GITHUB_LIST_REPOSITORY_WORKFLOWS",
    LIST_REPOS_ACCESSIBLE_TO_APP_INSTALLATION: "GITHUB_LIST_REPOS_ACCESSIBLE_TO_APP_INSTALLATION",
    LIST_REPOS_ACCESSIBLE_TO_USER_TOKEN: "GITHUB_LIST_REPOS_ACCESSIBLE_TO_USER_TOKEN",
    LIST_REPOS_WITH_GHACTIONS_CACHE_USAGE: "GITHUB_LIST_REPOS_WITH_GHACTIONS_CACHE_USAGE",
    LIST_REPO_ACCESS_BY_TOKEN: "GITHUB_LIST_REPO_ACCESS_BY_TOKEN",
    LIST_REPO_CODESPACES_FOR_AUTH_USER: "GITHUB_LIST_REPO_CODESPACES_FOR_AUTH_USER",
    LIST_REPO_DEV_CONTAINER_CONFIGS_FOR_USER: "GITHUB_LIST_REPO_DEV_CONTAINER_CONFIGS_FOR_USER",
    LIST_REPO_INVITATIONS_FOR_AUTH_USER: "GITHUB_LIST_REPO_INVITATIONS_FOR_AUTH_USER",
    LIST_REPO_NOTIFICATIONS_FOR_AUTH_USER: "GITHUB_LIST_REPO_NOTIFICATIONS_FOR_AUTH_USER",
    LIST_REPO_SECRETS_WITHOUT_VALUES: "GITHUB_LIST_REPO_SECRETS_WITHOUT_VALUES",
    LIST_REPO_SECURITY_ADVISORIES_FOR_ORG: "GITHUB_LIST_REPO_SECURITY_ADVISORIES_FOR_ORG",
    LIST_REVIEWS_FOR_A_PULL_REQUEST: "GITHUB_LIST_REVIEWS_FOR_A_PULL_REQUEST",
    LIST_REVIEW_COMMENTS_IN_A_REPOSITORY: "GITHUB_LIST_REVIEW_COMMENTS_IN_A_REPOSITORY",
    LIST_REVIEW_COMMENTS_ON_A_PULL_REQUEST: "GITHUB_LIST_REVIEW_COMMENTS_ON_A_PULL_REQUEST",
    LIST_RUNNER_APPLICATIONS_FOR_AN_ORGANIZATION: "GITHUB_LIST_RUNNER_APPLICATIONS_FOR_AN_ORGANIZATION",
    LIST_RUNNER_APPLICATIONS_FOR_A_REPOSITORY: "GITHUB_LIST_RUNNER_APPLICATIONS_FOR_A_REPOSITORY",
    LIST_SECRETS_FOR_THE_AUTHENTICATED_USER: "GITHUB_LIST_SECRETS_FOR_THE_AUTHENTICATED_USER",
    LIST_SECRET_SCANNING_ALERTS_FOR_AN_ENTERPRISE: "GITHUB_LIST_SECRET_SCANNING_ALERTS_FOR_AN_ENTERPRISE",
    LIST_SECRET_SCANNING_ALERTS_FOR_AN_ORGANIZATION: "GITHUB_LIST_SECRET_SCANNING_ALERTS_FOR_AN_ORGANIZATION",
    LIST_SECRET_SCANNING_ALERTS_FOR_A_REPOSITORY: "GITHUB_LIST_SECRET_SCANNING_ALERTS_FOR_A_REPOSITORY",
    LIST_SECURITY_MANAGER_TEAMS: "GITHUB_LIST_SECURITY_MANAGER_TEAMS",
    LIST_SELECTED_REPOSITORIES_FOR_A_USER_SECRET: "GITHUB_LIST_SELECTED_REPOSITORIES_FOR_A_USER_SECRET",
    LIST_SELECTED_REPOS_FOR_ORG_SECRET: "GITHUB_LIST_SELECTED_REPOS_FOR_ORG_SECRET",
    LIST_SELECTED_REPOS_FOR_SECRET_ACCESS: "GITHUB_LIST_SELECTED_REPOS_FOR_SECRET_ACCESS",
    LIST_SELECTED_REPOS_ORG_VARIABLE: "GITHUB_LIST_SELECTED_REPOS_ORG_VARIABLE",
    LIST_SELF_HOSTED_RUNNERS_FOR_AN_ORGANIZATION: "GITHUB_LIST_SELF_HOSTED_RUNNERS_FOR_AN_ORGANIZATION",
    LIST_SELF_HOSTED_RUNNERS_FOR_A_REPOSITORY: "GITHUB_LIST_SELF_HOSTED_RUNNERS_FOR_A_REPOSITORY",
    LIST_SOCIAL_ACCOUNTS_FOR_A_USER: "GITHUB_LIST_SOCIAL_ACCOUNTS_FOR_A_USER",
    LIST_SOCIAL_ACCOUNTS_FOR_THE_AUTHENTICATED_USER: "GITHUB_LIST_SOCIAL_ACCOUNTS_FOR_THE_AUTHENTICATED_USER",
    LIST_SSH_SIGNING_KEYS_FOR_A_USER: "GITHUB_LIST_SSH_SIGNING_KEYS_FOR_A_USER",
    LIST_SSH_SIGNING_KEYS_FOR_THE_AUTHENTICATED_USER: "GITHUB_LIST_SSH_SIGNING_KEYS_FOR_THE_AUTHENTICATED_USER",
    LIST_STARGAZERS: "GITHUB_LIST_STARGAZERS",
    LIST_STARRED_GISTS: "GITHUB_LIST_STARRED_GISTS",
    LIST_SUBSCRIPTIONS_FOR_THE_AUTHENTICATED_USER: "GITHUB_LIST_SUBSCRIPTIONS_FOR_THE_AUTHENTICATED_USER",
    LIST_SUBS_FOR_AUTH_USER_STUBBED: "GITHUB_LIST_SUBS_FOR_AUTH_USER_STUBBED",
    LIST_TAG_PROTECTION_STATES_FOR_A_REPOSITORY: "GITHUB_LIST_TAG_PROTECTION_STATES_FOR_A_REPOSITORY",
    LIST_TEAMS: "GITHUB_LIST_TEAMS",
    LIST_TEAMS_ASSIGNED_TO_ORG_ROLE: "GITHUB_LIST_TEAMS_ASSIGNED_TO_ORG_ROLE",
    LIST_TEAMS_FOR_THE_AUTHENTICATED_USER: "GITHUB_LIST_TEAMS_FOR_THE_AUTHENTICATED_USER",
    LIST_TEAM_MEMBERS: "GITHUB_LIST_TEAM_MEMBERS",
    LIST_TEAM_MEMBERS_LEGACY: "GITHUB_LIST_TEAM_MEMBERS_LEGACY",
    LIST_TEAM_PROJECTS: "GITHUB_LIST_TEAM_PROJECTS",
    LIST_TEAM_PROJECTS_LEGACY: "GITHUB_LIST_TEAM_PROJECTS_LEGACY",
    LIST_TEAM_REPOSITORIES: "GITHUB_LIST_TEAM_REPOSITORIES",
    LIST_TEAM_REPOSITORIES_LEGACY: "GITHUB_LIST_TEAM_REPOSITORIES_LEGACY",
    LIST_THE_PEOPLE_A_USER_FOLLOWS: "GITHUB_LIST_THE_PEOPLE_A_USER_FOLLOWS",
    LIST_THE_PEOPLE_THE_AUTHENTICATED_USER_FOLLOWS: "GITHUB_LIST_THE_PEOPLE_THE_AUTHENTICATED_USER_FOLLOWS",
    LIST_TIMELINE_EVENTS_FOR_AN_ISSUE: "GITHUB_LIST_TIMELINE_EVENTS_FOR_AN_ISSUE",
    LIST_TOKEN_ACCESS_REPOSITORIES: "GITHUB_LIST_TOKEN_ACCESS_REPOSITORIES",
    LIST_USERS: "GITHUB_LIST_USERS",
    LIST_USERS_ASSIGNED_ORG_ROLE: "GITHUB_LIST_USERS_ASSIGNED_ORG_ROLE",
    LIST_USERS_BLOCKED_BY_AN_ORGANIZATION: "GITHUB_LIST_USERS_BLOCKED_BY_AN_ORGANIZATION",
    LIST_USERS_BLOCKED_BY_THE_AUTHENTICATED_USER: "GITHUB_LIST_USERS_BLOCKED_BY_THE_AUTHENTICATED_USER",
    LIST_USER_ISSUES_ASSIGNED_TO_AUTH_USER: "GITHUB_LIST_USER_ISSUES_ASSIGNED_TO_AUTH_USER",
    LIST_USER_MIGRATIONS: "GITHUB_LIST_USER_MIGRATIONS",
    LIST_USER_PROJECTS: "GITHUB_LIST_USER_PROJECTS",
    LIST_WATCHERS: "GITHUB_LIST_WATCHERS",
    LIST_WORKFLOW_RUNS_FOR_A_REPOSITORY: "GITHUB_LIST_WORKFLOW_RUNS_FOR_A_REPOSITORY",
    LIST_WORKFLOW_RUNS_FOR_A_WORKFLOW: "GITHUB_LIST_WORKFLOW_RUNS_FOR_A_WORKFLOW",
    LIST_WORKFLOW_RUN_ARTIFACTS: "GITHUB_LIST_WORKFLOW_RUN_ARTIFACTS",
    LOCK_AN_ISSUE: "GITHUB_LOCK_AN_ISSUE",
    MANAGE_ACCESS_CONTROL_FOR_ORGANIZATION_CODESPACES: "GITHUB_MANAGE_ACCESS_CONTROL_FOR_ORGANIZATION_CODESPACES",
    MANAGE_CUSTOM_PROPERTIES_FOR_ORG_REPOS: "GITHUB_MANAGE_CUSTOM_PROPERTIES_FOR_ORG_REPOS",
    MANAGE_SECRETS_SELECTED_REPOS_PROPER_ACCESS: "GITHUB_MANAGE_SECRETS_SELECTED_REPOS_PROPER_ACCESS",
    MAP_A_COMMIT_AUTHOR: "GITHUB_MAP_A_COMMIT_AUTHOR",
    MARK_A_THREAD_AS_DONE: "GITHUB_MARK_A_THREAD_AS_DONE",
    MARK_A_THREAD_AS_READ: "GITHUB_MARK_A_THREAD_AS_READ",
    MARK_NOTIFICATIONS_AS_READ: "GITHUB_MARK_NOTIFICATIONS_AS_READ",
    MARK_REPOSITORY_NOTIFICATIONS_AS_READ: "GITHUB_MARK_REPOSITORY_NOTIFICATIONS_AS_READ",
    MERGE_A_BRANCH: "GITHUB_MERGE_A_BRANCH",
    MERGE_A_PULL_REQUEST: "GITHUB_MERGE_A_PULL_REQUEST",
    META_ROOT: "GITHUB_META_ROOT",
    MOVE_A_PROJECT_CARD: "GITHUB_MOVE_A_PROJECT_CARD",
    MOVE_A_PROJECT_COLUMN: "GITHUB_MOVE_A_PROJECT_COLUMN",
    PING_AN_ORGANIZATION_WEBHOOK: "GITHUB_PING_AN_ORGANIZATION_WEBHOOK",
    PING_A_REPOSITORY_WEBHOOK: "GITHUB_PING_A_REPOSITORY_WEBHOOK",
    PRIVATELY_REPORT_A_SECURITY_VULNERABILITY: "GITHUB_PRIVATELY_REPORT_A_SECURITY_VULNERABILITY",
    PULLS_CREATE: "GITHUB_PULLS_CREATE",
    PULLS_CREATE_REVIEW: "GITHUB_PULLS_CREATE_REVIEW",
    PULLS_CREATE_REVIEW_COMMENT: "GITHUB_PULLS_CREATE_REVIEW_COMMENT",
    PULLS_GET: "GITHUB_PULLS_GET",
    PULLS_LIST: "GITHUB_PULLS_LIST",
    REDELIVER_A_DELIVERY_FOR_AN_ORGANIZATION_WEBHOOK: "GITHUB_REDELIVER_A_DELIVERY_FOR_AN_ORGANIZATION_WEBHOOK",
    REDELIVER_A_DELIVERY_FOR_A_REPOSITORY_WEBHOOK: "GITHUB_REDELIVER_A_DELIVERY_FOR_A_REPOSITORY_WEBHOOK",
    REMOVE_ALL_LABELS_FROM_AN_ISSUE: "GITHUB_REMOVE_ALL_LABELS_FROM_AN_ISSUE",
    REMOVE_ALL_ORGANIZATION_ROLES_FOR_A_TEAM: "GITHUB_REMOVE_ALL_ORGANIZATION_ROLES_FOR_A_TEAM",
    REMOVE_ALL_ORGANIZATION_ROLES_FOR_A_USER: "GITHUB_REMOVE_ALL_ORGANIZATION_ROLES_FOR_A_USER",
    REMOVE_AN_ORGANIZATION_MEMBER: "GITHUB_REMOVE_AN_ORGANIZATION_MEMBER",
    REMOVE_AN_ORGANIZATION_ROLE_FROM_A_TEAM: "GITHUB_REMOVE_AN_ORGANIZATION_ROLE_FROM_A_TEAM",
    REMOVE_AN_ORGANIZATION_ROLE_FROM_A_USER: "GITHUB_REMOVE_AN_ORGANIZATION_ROLE_FROM_A_USER",
    REMOVE_APP_ACCESS_RESTRICTIONS: "GITHUB_REMOVE_APP_ACCESS_RESTRICTIONS",
    REMOVE_ASSIGNEES_FROM_AN_ISSUE: "GITHUB_REMOVE_ASSIGNEES_FROM_AN_ISSUE",
    REMOVE_A_CUSTOM_PROPERTY_FOR_AN_ORGANIZATION: "GITHUB_REMOVE_A_CUSTOM_PROPERTY_FOR_AN_ORGANIZATION",
    REMOVE_A_LABEL_FROM_AN_ISSUE: "GITHUB_REMOVE_A_LABEL_FROM_AN_ISSUE",
    REMOVE_A_PROJECT_FROM_A_TEAM: "GITHUB_REMOVE_A_PROJECT_FROM_A_TEAM",
    REMOVE_A_PROJECT_FROM_A_TEAM_LEGACY: "GITHUB_REMOVE_A_PROJECT_FROM_A_TEAM_LEGACY",
    REMOVE_A_REPOSITORY_COLLABORATOR: "GITHUB_REMOVE_A_REPOSITORY_COLLABORATOR",
    REMOVE_A_REPOSITORY_FROM_AN_APP_INSTALLATION: "GITHUB_REMOVE_A_REPOSITORY_FROM_AN_APP_INSTALLATION",
    REMOVE_A_REPOSITORY_FROM_A_TEAM: "GITHUB_REMOVE_A_REPOSITORY_FROM_A_TEAM",
    REMOVE_A_REPOSITORY_FROM_A_TEAM_LEGACY: "GITHUB_REMOVE_A_REPOSITORY_FROM_A_TEAM_LEGACY",
    REMOVE_A_SECURITY_MANAGER_TEAM: "GITHUB_REMOVE_A_SECURITY_MANAGER_TEAM",
    REMOVE_A_SELECTED_REPOSITORY_FROM_A_USER_SECRET: "GITHUB_REMOVE_A_SELECTED_REPOSITORY_FROM_A_USER_SECRET",
    REMOVE_COPILOT_TEAMS: "GITHUB_REMOVE_COPILOT_TEAMS",
    REMOVE_CUSTOM_LABELS_FROM_SELF_HOSTED_REPO_RUNNER: "GITHUB_REMOVE_CUSTOM_LABELS_FROM_SELF_HOSTED_REPO_RUNNER",
    REMOVE_CUSTOM_LABEL_FROM_REPO_RUNNER: "GITHUB_REMOVE_CUSTOM_LABEL_FROM_REPO_RUNNER",
    REMOVE_CUSTOM_LABEL_FROM_SELF_HOSTED_RUNNER: "GITHUB_REMOVE_CUSTOM_LABEL_FROM_SELF_HOSTED_RUNNER",
    REMOVE_INTERACTION_RESTRICTIONS_FOR_AN_ORGANIZATION: "GITHUB_REMOVE_INTERACTION_RESTRICTIONS_FOR_AN_ORGANIZATION",
    REMOVE_INTERACTION_RESTRICTIONS_FOR_A_REPOSITORY: "GITHUB_REMOVE_INTERACTION_RESTRICTIONS_FOR_A_REPOSITORY",
    REMOVE_ORGANIZATION_MEMBERSHIP_FOR_A_USER: "GITHUB_REMOVE_ORGANIZATION_MEMBERSHIP_FOR_A_USER",
    REMOVE_ORG_DEV_ENV_SECRET_BY_NAME: "GITHUB_REMOVE_ORG_DEV_ENV_SECRET_BY_NAME",
    REMOVE_ORG_SECRET_BY_NAME: "GITHUB_REMOVE_ORG_SECRET_BY_NAME",
    REMOVE_OUTSIDE_COLLABORATOR_FROM_AN_ORGANIZATION: "GITHUB_REMOVE_OUTSIDE_COLLABORATOR_FROM_AN_ORGANIZATION",
    REMOVE_PUBLIC_ORG_MEMBERSHIP: "GITHUB_REMOVE_PUBLIC_ORG_MEMBERSHIP",
    REMOVE_REPO_FROM_ORG_DEV_ENV_SECRET: "GITHUB_REMOVE_REPO_FROM_ORG_DEV_ENV_SECRET",
    REMOVE_REPO_ORG_SECRET_WITH_SELECTED_VISIBILITY: "GITHUB_REMOVE_REPO_ORG_SECRET_WITH_SELECTED_VISIBILITY",
    REMOVE_REQUESTED_REVIEWERS_FROM_A_PULL_REQUEST: "GITHUB_REMOVE_REQUESTED_REVIEWERS_FROM_A_PULL_REQUEST",
    REMOVE_SELECTED_REPO_FROM_ORG_SECRET: "GITHUB_REMOVE_SELECTED_REPO_FROM_ORG_SECRET",
    REMOVE_SELECTED_REPO_FROM_ORG_VARIABLE: "GITHUB_REMOVE_SELECTED_REPO_FROM_ORG_VARIABLE",
    REMOVE_STATUS_CHECK_CONTEXTS: "GITHUB_REMOVE_STATUS_CHECK_CONTEXTS",
    REMOVE_STATUS_CHECK_PROTECTION: "GITHUB_REMOVE_STATUS_CHECK_PROTECTION",
    REMOVE_TEAM_ACCESS_RESTRICTIONS: "GITHUB_REMOVE_TEAM_ACCESS_RESTRICTIONS",
    REMOVE_TEAM_MEMBERSHIP_FOR_A_USER: "GITHUB_REMOVE_TEAM_MEMBERSHIP_FOR_A_USER",
    REMOVE_TEAM_MEMBERSHIP_FOR_A_USER_LEGACY: "GITHUB_REMOVE_TEAM_MEMBERSHIP_FOR_A_USER_LEGACY",
    REMOVE_TEAM_MEMBER_LEGACY: "GITHUB_REMOVE_TEAM_MEMBER_LEGACY",
    REMOVE_USERS_CODESPACES_ACCESS_ORG: "GITHUB_REMOVE_USERS_CODESPACES_ACCESS_ORG",
    REMOVE_USERS_FROM_COPILOT_SUBSCRIPTION_FOR_ORG: "GITHUB_REMOVE_USERS_FROM_COPILOT_SUBSCRIPTION_FOR_ORG",
    REMOVE_USER_ACCESS_RESTRICTIONS: "GITHUB_REMOVE_USER_ACCESS_RESTRICTIONS",
    REMOVE_USER_AS_A_COLLABORATOR: "GITHUB_REMOVE_USER_AS_A_COLLABORATOR",
    REMOVE_USER_PUBLIC_REPO_INTERACTION_RESTRICTIONS: "GITHUB_REMOVE_USER_PUBLIC_REPO_INTERACTION_RESTRICTIONS",
    RENAME_A_BRANCH: "GITHUB_RENAME_A_BRANCH",
    RENDER_A_MARKDOWN_DOCUMENT: "GITHUB_RENDER_A_MARKDOWN_DOCUMENT",
    REPLACE_ALL_REPOSITORY_TOPICS: "GITHUB_REPLACE_ALL_REPOSITORY_TOPICS",
    REPLACE_ORG_SECRET_VISIBILITY_TO_SELECTED: "GITHUB_REPLACE_ORG_SECRET_VISIBILITY_TO_SELECTED",
    REPLACE_REPO_ACCESS_ON_ORG_DEV_ENV_SECRET_SET: "GITHUB_REPLACE_REPO_ACCESS_ON_ORG_DEV_ENV_SECRET_SET",
    REPO_S_CREATE_FORK: "GITHUB_REPO_S_CREATE_FORK",
    REPO_S_CREATE_FOR_AUTHENTICATED_USER: "GITHUB_REPO_S_CREATE_FOR_AUTHENTICATED_USER",
    REPO_S_CREATE_IN_ORG: "GITHUB_REPO_S_CREATE_IN_ORG",
    REPO_S_CREATE_OR_UPDATE_FILE_CONTENTS: "GITHUB_REPO_S_CREATE_OR_UPDATE_FILE_CONTENTS",
    REPO_S_GET_CODE_FREQUENCY_STATS: "GITHUB_REPO_S_GET_CODE_FREQUENCY_STATS",
    REPO_S_GET_COMMIT: "GITHUB_REPO_S_GET_COMMIT",
    REPO_S_GET_CONTENT: "GITHUB_REPO_S_GET_CONTENT",
    REPO_S_GET_CONTRIBUTORS_STATS: "GITHUB_REPO_S_GET_CONTRIBUTORS_STATS",
    REPO_S_LIST_BRANCHES: "GITHUB_REPO_S_LIST_BRANCHES",
    REPO_S_LIST_COLLABORATORS: "GITHUB_REPO_S_LIST_COLLABORATORS",
    REPO_S_LIST_COMMITS: "GITHUB_REPO_S_LIST_COMMITS",
    REPO_S_LIST_CONTRIBUTORS: "GITHUB_REPO_S_LIST_CONTRIBUTORS",
    REPO_S_LIST_FOR_AUTHENTICATED_USER: "GITHUB_REPO_S_LIST_FOR_AUTHENTICATED_USER",
    REPO_S_LIST_FOR_ORG: "GITHUB_REPO_S_LIST_FOR_ORG",
    REPO_S_LIST_FOR_USER: "GITHUB_REPO_S_LIST_FOR_USER",
    REQUEST_A_CVE_FOR_A_REPOSITORY_SECURITY_ADVISORY: "GITHUB_REQUEST_A_CVE_FOR_A_REPOSITORY_SECURITY_ADVISORY",
    REQUEST_A_GITHUB_PAGES_BUILD: "GITHUB_REQUEST_A_GITHUB_PAGES_BUILD",
    REQUEST_REVIEWERS_FOR_A_PULL_REQUEST: "GITHUB_REQUEST_REVIEWERS_FOR_A_PULL_REQUEST",
    REREQUEST_A_CHECK_RUN: "GITHUB_REREQUEST_A_CHECK_RUN",
    REREQUEST_A_CHECK_SUITE: "GITHUB_REREQUEST_A_CHECK_SUITE",
    RESET_A_TOKEN: "GITHUB_RESET_A_TOKEN",
    RESTORE_A_PACKAGE_FOR_AN_ORGANIZATION: "GITHUB_RESTORE_A_PACKAGE_FOR_AN_ORGANIZATION",
    RESTORE_A_PACKAGE_FOR_A_USER: "GITHUB_RESTORE_A_PACKAGE_FOR_A_USER",
    RESTORE_A_PACKAGE_FOR_THE_AUTHENTICATED_USER: "GITHUB_RESTORE_A_PACKAGE_FOR_THE_AUTHENTICATED_USER",
    RESTORE_PACKAGE_VERSION_FOR_AN_ORGANIZATION: "GITHUB_RESTORE_PACKAGE_VERSION_FOR_AN_ORGANIZATION",
    RESTORE_PACKAGE_VERSION_FOR_AUTH_USER: "GITHUB_RESTORE_PACKAGE_VERSION_FOR_AUTH_USER",
    RESTORE_PACKAGE_VERSION_FOR_A_USER: "GITHUB_RESTORE_PACKAGE_VERSION_FOR_A_USER",
    RETRIEVE_REPO_PUBLIC_KEY_FOR_ENCRYPTION: "GITHUB_RETRIEVE_REPO_PUBLIC_KEY_FOR_ENCRYPTION",
    REVIEW_ACCESS_WITH_PERSONAL_TOKEN: "GITHUB_REVIEW_ACCESS_WITH_PERSONAL_TOKEN",
    REVIEW_CUSTOM_DEPLOYMENT_RULES_FOR_WORKFLOW_RUN: "GITHUB_REVIEW_CUSTOM_DEPLOYMENT_RULES_FOR_WORKFLOW_RUN",
    REVIEW_PENDING_DEPLOYMENTS_FOR_A_WORKFLOW_RUN: "GITHUB_REVIEW_PENDING_DEPLOYMENTS_FOR_A_WORKFLOW_RUN",
    REVIEW_RESOURCE_REQUESTS_WITH_FINE_GRAINED_TOKENS: "GITHUB_REVIEW_RESOURCE_REQUESTS_WITH_FINE_GRAINED_TOKENS",
    REVOKE_AN_INSTALLATION_ACCESS_TOKEN: "GITHUB_REVOKE_AN_INSTALLATION_ACCESS_TOKEN",
    RE_RUN_A_JOB_FROM_A_WORKFLOW_RUN: "GITHUB_RE_RUN_A_JOB_FROM_A_WORKFLOW_RUN",
    RE_RUN_A_WORKFLOW: "GITHUB_RE_RUN_A_WORKFLOW",
    RE_RUN_FAILED_JOBS_FROM_A_WORKFLOW_RUN: "GITHUB_RE_RUN_FAILED_JOBS_FROM_A_WORKFLOW_RUN",
    SEARCH_CODE: "GITHUB_SEARCH_CODE",
    SEARCH_COMMITS: "GITHUB_SEARCH_COMMITS",
    SEARCH_ISSUES_AND_PULL_REQUESTS: "GITHUB_SEARCH_ISSUES_AND_PULL_REQUESTS",
    SEARCH_LABELS: "GITHUB_SEARCH_LABELS",
    SEARCH_REPOSITORIES: "GITHUB_SEARCH_REPOSITORIES",
    SEARCH_REPO_S: "GITHUB_SEARCH_REPO_S",
    SEARCH_TOPICS: "GITHUB_SEARCH_TOPICS",
    SEARCH_USERS: "GITHUB_SEARCH_USERS",
    SECURITY_ADVISORIES_LIST_GLOBAL_ADVISORIES: "GITHUB_SECURITY_ADVISORIES_LIST_GLOBAL_ADVISORIES",
    SET_ADMIN_BRANCH_PROTECTION: "GITHUB_SET_ADMIN_BRANCH_PROTECTION",
    SET_ALLOWED_ACTIONS_AND_WORKFLOWS_FOR_ORG: "GITHUB_SET_ALLOWED_ACTIONS_AND_WORKFLOWS_FOR_ORG",
    SET_APP_ACCESS_RESTRICTIONS: "GITHUB_SET_APP_ACCESS_RESTRICTIONS",
    SET_A_REPOSITORY_SUBSCRIPTION: "GITHUB_SET_A_REPOSITORY_SUBSCRIPTION",
    SET_A_THREAD_SUBSCRIPTION: "GITHUB_SET_A_THREAD_SUBSCRIPTION",
    SET_DEFAULT_WORKFLOW_PERMISSIONS_FOR_A_REPOSITORY: "GITHUB_SET_DEFAULT_WORKFLOW_PERMISSIONS_FOR_A_REPOSITORY",
    SET_DEFAULT_WORKFLOW_PERMS_FOR_ORG: "GITHUB_SET_DEFAULT_WORKFLOW_PERMS_FOR_ORG",
    SET_EMAIL_VISIBILITY: "GITHUB_SET_EMAIL_VISIBILITY",
    SET_GITHUB_ACTIONS_PERMISSIONS_FOR_AN_ORGANIZATION: "GITHUB_SET_GITHUB_ACTIONS_PERMISSIONS_FOR_AN_ORGANIZATION",
    SET_GITHUB_ACTIONS_PERMISSIONS_FOR_A_REPOSITORY: "GITHUB_SET_GITHUB_ACTIONS_PERMISSIONS_FOR_A_REPOSITORY",
    SET_INTERACTION_RESTRICTIONS_FOR_AN_ORGANIZATION: "GITHUB_SET_INTERACTION_RESTRICTIONS_FOR_AN_ORGANIZATION",
    SET_INTERACTION_RESTRICTIONS_FOR_A_REPOSITORY: "GITHUB_SET_INTERACTION_RESTRICTIONS_FOR_A_REPOSITORY",
    SET_INTERACTION_RESTRICTIONS_FOR_YOUR_PUBLIC_REPOS: "GITHUB_SET_INTERACTION_RESTRICTIONS_FOR_YOUR_PUBLIC_REPOS",
    SET_LABELS_FOR_AN_ISSUE: "GITHUB_SET_LABELS_FOR_AN_ISSUE",
    SET_ORGANIZATION_MEMBERSHIP_FOR_A_USER: "GITHUB_SET_ORGANIZATION_MEMBERSHIP_FOR_A_USER",
    SET_ORG_RUNNER_LABELS: "GITHUB_SET_ORG_RUNNER_LABELS",
    SET_PUBLIC_ORG_MEMBERSHIP_FOR_AUTH_USER: "GITHUB_SET_PUBLIC_ORG_MEMBERSHIP_FOR_AUTH_USER",
    SET_REPO_ALLOWED_ACTIONS: "GITHUB_SET_REPO_ALLOWED_ACTIONS",
    SET_REPO_RUNNER_LABELS: "GITHUB_SET_REPO_RUNNER_LABELS",
    SET_SELECTED_REPOSITORIES_FOR_A_USER_SECRET: "GITHUB_SET_SELECTED_REPOSITORIES_FOR_A_USER_SECRET",
    SET_SELECTED_REPOS_FOR_ORG_SECRET: "GITHUB_SET_SELECTED_REPOS_FOR_ORG_SECRET",
    SET_SELECTED_REPOS_FOR_ORG_VARIABLE: "GITHUB_SET_SELECTED_REPOS_FOR_ORG_VARIABLE",
    SET_STATUS_CHECK_CONTEXTS: "GITHUB_SET_STATUS_CHECK_CONTEXTS",
    SET_TEAM_ACCESS_RESTRICTIONS: "GITHUB_SET_TEAM_ACCESS_RESTRICTIONS",
    SET_USER_ACCESS_RESTRICTIONS: "GITHUB_SET_USER_ACCESS_RESTRICTIONS",
    SET_WORKFLOW_ACCESS_LEVEL_OUTSIDE_REPO: "GITHUB_SET_WORKFLOW_ACCESS_LEVEL_OUTSIDE_REPO",
    START_AN_IMPORT: "GITHUB_START_AN_IMPORT",
    START_AN_ORGANIZATION_MIGRATION: "GITHUB_START_AN_ORGANIZATION_MIGRATION",
    START_A_CODESPACE_FOR_THE_AUTHENTICATED_USER: "GITHUB_START_A_CODESPACE_FOR_THE_AUTHENTICATED_USER",
    START_A_USER_MIGRATION: "GITHUB_START_A_USER_MIGRATION",
    STAR_A_GIST: "GITHUB_STAR_A_GIST",
    STAR_A_REPOSITORY_FOR_THE_AUTHENTICATED_USER: "GITHUB_STAR_A_REPOSITORY_FOR_THE_AUTHENTICATED_USER",
    STOP_A_CODESPACE_FOR_AN_ORGANIZATION_USER: "GITHUB_STOP_A_CODESPACE_FOR_AN_ORGANIZATION_USER",
    STOP_A_CODESPACE_FOR_THE_AUTHENTICATED_USER: "GITHUB_STOP_A_CODESPACE_FOR_THE_AUTHENTICATED_USER",
    SUBMIT_A_REVIEW_FOR_A_PULL_REQUEST: "GITHUB_SUBMIT_A_REVIEW_FOR_A_PULL_REQUEST",
    SYNC_A_FORK_BRANCH_WITH_THE_UPSTREAM_REPOSITORY: "GITHUB_SYNC_A_FORK_BRANCH_WITH_THE_UPSTREAM_REPOSITORY",
    TEST_THE_PUSH_REPOSITORY_WEBHOOK: "GITHUB_TEST_THE_PUSH_REPOSITORY_WEBHOOK",
    TRANSFER_A_REPOSITORY: "GITHUB_TRANSFER_A_REPOSITORY",
    UNBLOCK_A_USER: "GITHUB_UNBLOCK_A_USER",
    UNBLOCK_A_USER_FROM_AN_ORGANIZATION: "GITHUB_UNBLOCK_A_USER_FROM_AN_ORGANIZATION",
    UNFOLLOW_A_USER: "GITHUB_UNFOLLOW_A_USER",
    UNLOCK_AN_ISSUE: "GITHUB_UNLOCK_AN_ISSUE",
    UNLOCK_AN_ORGANIZATION_REPOSITORY: "GITHUB_UNLOCK_AN_ORGANIZATION_REPOSITORY",
    UNLOCK_A_USER_REPOSITORY: "GITHUB_UNLOCK_A_USER_REPOSITORY",
    UNSTAR_A_GIST: "GITHUB_UNSTAR_A_GIST",
    UNSTAR_A_REPOSITORY_FOR_THE_AUTHENTICATED_USER: "GITHUB_UNSTAR_A_REPOSITORY_FOR_THE_AUTHENTICATED_USER",
    UPDATE_AN_ENVIRONMENT_VARIABLE: "GITHUB_UPDATE_AN_ENVIRONMENT_VARIABLE",
    UPDATE_AN_EXISTING_PROJECT_CARD: "GITHUB_UPDATE_AN_EXISTING_PROJECT_CARD",
    UPDATE_AN_EXISTING_PROJECT_COLUMN: "GITHUB_UPDATE_AN_EXISTING_PROJECT_COLUMN",
    UPDATE_AN_IMPORT: "GITHUB_UPDATE_AN_IMPORT",
    UPDATE_AN_ISSUE: "GITHUB_UPDATE_AN_ISSUE",
    UPDATE_AN_ISSUE_COMMENT: "GITHUB_UPDATE_AN_ISSUE_COMMENT",
    UPDATE_AN_ORGANIZATION: "GITHUB_UPDATE_AN_ORGANIZATION",
    UPDATE_AN_ORGANIZATION_REPOSITORY_RULESET: "GITHUB_UPDATE_AN_ORGANIZATION_REPOSITORY_RULESET",
    UPDATE_AN_ORGANIZATION_VARIABLE: "GITHUB_UPDATE_AN_ORGANIZATION_VARIABLE",
    UPDATE_AN_ORGANIZATION_WEBHOOK: "GITHUB_UPDATE_AN_ORGANIZATION_WEBHOOK",
    UPDATE_A_CHECK_RUN: "GITHUB_UPDATE_A_CHECK_RUN",
    UPDATE_A_CODESPACE_FOR_THE_AUTHENTICATED_USER: "GITHUB_UPDATE_A_CODESPACE_FOR_THE_AUTHENTICATED_USER",
    UPDATE_A_CODE_SCANNING_ALERT: "GITHUB_UPDATE_A_CODE_SCANNING_ALERT",
    UPDATE_A_CODE_SCANNING_DEFAULT_SETUP_CONFIGURATION: "GITHUB_UPDATE_A_CODE_SCANNING_DEFAULT_SETUP_CONFIGURATION",
    UPDATE_A_COMMIT_COMMENT: "GITHUB_UPDATE_A_COMMIT_COMMENT",
    UPDATE_A_CUSTOM_ORGANIZATION_ROLE: "GITHUB_UPDATE_A_CUSTOM_ORGANIZATION_ROLE",
    UPDATE_A_DEPENDABOT_ALERT: "GITHUB_UPDATE_A_DEPENDABOT_ALERT",
    UPDATE_A_DEPLOYMENT_BRANCH_POLICY: "GITHUB_UPDATE_A_DEPLOYMENT_BRANCH_POLICY",
    UPDATE_A_DISCUSSION: "GITHUB_UPDATE_A_DISCUSSION",
    UPDATE_A_DISCUSSION_COMMENT: "GITHUB_UPDATE_A_DISCUSSION_COMMENT",
    UPDATE_A_DISCUSSION_COMMENT_LEGACY: "GITHUB_UPDATE_A_DISCUSSION_COMMENT_LEGACY",
    UPDATE_A_DISCUSSION_LEGACY: "GITHUB_UPDATE_A_DISCUSSION_LEGACY",
    UPDATE_A_GIST: "GITHUB_UPDATE_A_GIST",
    UPDATE_A_GIST_COMMENT: "GITHUB_UPDATE_A_GIST_COMMENT",
    UPDATE_A_LABEL: "GITHUB_UPDATE_A_LABEL",
    UPDATE_A_MILESTONE: "GITHUB_UPDATE_A_MILESTONE",
    UPDATE_A_PROJECT: "GITHUB_UPDATE_A_PROJECT",
    UPDATE_A_PULL_REQUEST: "GITHUB_UPDATE_A_PULL_REQUEST",
    UPDATE_A_PULL_REQUEST_BRANCH: "GITHUB_UPDATE_A_PULL_REQUEST_BRANCH",
    UPDATE_A_REFERENCE: "GITHUB_UPDATE_A_REFERENCE",
    UPDATE_A_RELEASE: "GITHUB_UPDATE_A_RELEASE",
    UPDATE_A_RELEASE_ASSET: "GITHUB_UPDATE_A_RELEASE_ASSET",
    UPDATE_A_REPOSITORY: "GITHUB_UPDATE_A_REPOSITORY",
    UPDATE_A_REPOSITORY_INVITATION: "GITHUB_UPDATE_A_REPOSITORY_INVITATION",
    UPDATE_A_REPOSITORY_RULESET: "GITHUB_UPDATE_A_REPOSITORY_RULESET",
    UPDATE_A_REPOSITORY_SECURITY_ADVISORY: "GITHUB_UPDATE_A_REPOSITORY_SECURITY_ADVISORY",
    UPDATE_A_REPOSITORY_VARIABLE: "GITHUB_UPDATE_A_REPOSITORY_VARIABLE",
    UPDATE_A_REPOSITORY_WEBHOOK: "GITHUB_UPDATE_A_REPOSITORY_WEBHOOK",
    UPDATE_A_REVIEW_COMMENT_FOR_A_PULL_REQUEST: "GITHUB_UPDATE_A_REVIEW_COMMENT_FOR_A_PULL_REQUEST",
    UPDATE_A_REVIEW_FOR_A_PULL_REQUEST: "GITHUB_UPDATE_A_REVIEW_FOR_A_PULL_REQUEST",
    UPDATE_A_SECRET_SCANNING_ALERT: "GITHUB_UPDATE_A_SECRET_SCANNING_ALERT",
    UPDATE_A_TEAM: "GITHUB_UPDATE_A_TEAM",
    UPDATE_A_TEAM_LEGACY: "GITHUB_UPDATE_A_TEAM_LEGACY",
    UPDATE_A_WEBHOOK_CONFIGURATION_FOR_AN_ORGANIZATION: "GITHUB_UPDATE_A_WEBHOOK_CONFIGURATION_FOR_AN_ORGANIZATION",
    UPDATE_A_WEBHOOK_CONFIGURATION_FOR_A_REPOSITORY: "GITHUB_UPDATE_A_WEBHOOK_CONFIGURATION_FOR_A_REPOSITORY",
    UPDATE_BRANCH_PROTECTION: "GITHUB_UPDATE_BRANCH_PROTECTION",
    UPDATE_GIT_LFS_PREFERENCE: "GITHUB_UPDATE_GIT_LFS_PREFERENCE",
    UPDATE_INFORMATION_ABOUT_A_GITHUB_PAGES_SITE: "GITHUB_UPDATE_INFORMATION_ABOUT_A_GITHUB_PAGES_SITE",
    UPDATE_ORG_MEMBERSHIP_FOR_AUTH_USER: "GITHUB_UPDATE_ORG_MEMBERSHIP_FOR_AUTH_USER",
    UPDATE_PULL_REQUEST_REVIEW_PROTECTION: "GITHUB_UPDATE_PULL_REQUEST_REVIEW_PROTECTION",
    UPDATE_REPOSITORY_PREFERENCES_FOR_CHECK_SUITES: "GITHUB_UPDATE_REPOSITORY_PREFERENCES_FOR_CHECK_SUITES",
    UPDATE_RESOURCE_ACCESS_WITH_TOKENS: "GITHUB_UPDATE_RESOURCE_ACCESS_WITH_TOKENS",
    UPDATE_STATUS_CHECK_PROTECTION: "GITHUB_UPDATE_STATUS_CHECK_PROTECTION",
    UPDATE_THE_AUTHENTICATED_USER: "GITHUB_UPDATE_THE_AUTHENTICATED_USER",
    UPDATE_TOKEN_ORG_ACCESS: "GITHUB_UPDATE_TOKEN_ORG_ACCESS",
    UPLOAD_AN_ANALYSIS_AS_SARIF_DATA: "GITHUB_UPLOAD_AN_ANALYSIS_AS_SARIF_DATA",
    UPSERT_REPO_CUSTOM_PROPERTY_VALUES: "GITHUB_UPSERT_REPO_CUSTOM_PROPERTY_VALUES",
    USERS_GET_AUTHENTICATED: "GITHUB_USERS_GET_AUTHENTICATED",
    USERS_GET_BY_USERNAME: "GITHUB_USERS_GET_BY_USERNAME",
    USERS_GET_CONTEXT_FOR_USER: "GITHUB_USERS_GET_CONTEXT_FOR_USER",
    USERS_LIST_EMAILS_FOR_AUTHENTICATED_USER: "GITHUB_USERS_LIST_EMAILS_FOR_AUTHENTICATED_USER",
    USERS_LIST_FOLLOWERS_FOR_AUTHENTICATED_USER: "GITHUB_USERS_LIST_FOLLOWERS_FOR_AUTHENTICATED_USER",
    USERS_LIST_PUBLIC_EMAILS_FOR_AUTHENTICATED_USER: "GITHUB_USERS_LIST_PUBLIC_EMAILS_FOR_AUTHENTICATED_USER",
    USERS_LIST_SOCIAL_ACCOUNTS_FOR_USER: "GITHUB_USERS_LIST_SOCIAL_ACCOUNTS_FOR_USER",
    VERIFY_DEV_CONTAINER_PERMISSIONS_ACCEPTED: "GITHUB_VERIFY_DEV_CONTAINER_PERMISSIONS_ACCEPTED",
  },
  triggerTypes: {
    COMMIT_EVENT: "GITHUB_COMMIT_EVENT",
    FOLLOWER_EVENT: "GITHUB_FOLLOWER_EVENT",
    ISSUE_ADDED_EVENT: "GITHUB_ISSUE_ADDED_EVENT",
    LABEL_ADDED_EVENT: "GITHUB_LABEL_ADDED_EVENT",
    PULL_REQUEST_EVENT: "GITHUB_PULL_REQUEST_EVENT",
    STAR_ADDED_EVENT: "GITHUB_STAR_ADDED_EVENT",
  },
}
