// @composio/core/generated/gitlab.ts
/**
 * Map of Composio's GITLAB toolkit.
 */
export const GITLAB = {
  slug: "gitlab",
  tools: {
    ADDS_A_BADGE_TO_A_GROUP: "GITLAB_ADDS_A_BADGE_TO_A_GROUP",
    ADDS_A_BADGE_TO_A_PROJECT: "GITLAB_ADDS_A_BADGE_TO_A_PROJECT",
    ADD_EXISTING_INSTANCE_CLUSTER: "GITLAB_ADD_EXISTING_INSTANCE_CLUSTER",
    APPROVES_AN_ACCESS_REQUEST_FOR_THE_GIVEN_USER: "GITLAB_APPROVES_AN_ACCESS_REQUEST_FOR_THE_GIVEN_USER",
    AUTHORIZE_METRIC_IMAGE_UPLOAD_FOR_ALERT: "GITLAB_AUTHORIZE_METRIC_IMAGE_UPLOAD_FOR_ALERT",
    CHANGE_PLAN_LIMITS: "GITLAB_CHANGE_PLAN_LIMITS",
    CHECK_BRANCH_EXISTENCE: "GITLAB_CHECK_BRANCH_EXISTENCE",
    CREATE_A_BROADCAST_MESSAGE: "GITLAB_CREATE_A_BROADCAST_MESSAGE",
    CREATE_A_NEW_APPLICATION: "GITLAB_CREATE_A_NEW_APPLICATION",
    CREATE_INSTANCE_LEVEL_CI_VARIABLE: "GITLAB_CREATE_INSTANCE_LEVEL_CI_VARIABLE",
    CREATE_PROJECT_REPOSITORY_BRANCH: "GITLAB_CREATE_PROJECT_REPOSITORY_BRANCH",
    DELETE_AN_APPLICATION: "GITLAB_DELETE_AN_APPLICATION",
    DELETE_A_BROADCAST_MESSAGE: "GITLAB_DELETE_A_BROADCAST_MESSAGE",
    DELETE_INSTANCE_CLUSTER: "GITLAB_DELETE_INSTANCE_CLUSTER",
    DELETE_INSTANCE_VARIABLE_BY_KEY: "GITLAB_DELETE_INSTANCE_VARIABLE_BY_KEY",
    DELETE_PROJECT_REPOSITORY_BRANCH: "GITLAB_DELETE_PROJECT_REPOSITORY_BRANCH",
    DENIES_AN_ACCESS_REQUEST_FOR_THE_GIVEN_USER: "GITLAB_DENIES_AN_ACCESS_REQUEST_FOR_THE_GIVEN_USER",
    EDIT_INSTANCE_CLUSTER: "GITLAB_EDIT_INSTANCE_CLUSTER",
    FETCH_INSTANCE_VARIABLE_DETAILS: "GITLAB_FETCH_INSTANCE_VARIABLE_DETAILS",
    FETCH_METRIC_IMAGES_FOR_AN_ALERT: "GITLAB_FETCH_METRIC_IMAGES_FOR_AN_ALERT",
    FETCH_PROJECT_REPOSITORY_BRANCHES: "GITLAB_FETCH_PROJECT_REPOSITORY_BRANCHES",
    FETCH_USER_AVATAR_URL: "GITLAB_FETCH_USER_AVATAR_URL",
    GETS_A_BADGE_OF_A_GROUP: "GITLAB_GETS_A_BADGE_OF_A_GROUP",
    GETS_A_BADGE_OF_A_PROJECT: "GITLAB_GETS_A_BADGE_OF_A_PROJECT",
    GETS_A_LIST_OF_ACCESS_REQUESTS_FOR_A_GROUP: "GITLAB_GETS_A_LIST_OF_ACCESS_REQUESTS_FOR_A_GROUP",
    GETS_A_LIST_OF_ACCESS_REQUESTS_FOR_A_PROJECT: "GITLAB_GETS_A_LIST_OF_ACCESS_REQUESTS_FOR_A_PROJECT",
    GET_ALL_BROADCAST_MESSAGES: "GITLAB_GET_ALL_BROADCAST_MESSAGES",
    GET_APPLICATIONS: "GITLAB_GET_APPLICATIONS",
    GET_A_SINGLE_INSTANCE_CLUSTER: "GITLAB_GET_A_SINGLE_INSTANCE_CLUSTER",
    GET_A_SINGLE_JOB_BY_ID: "GITLAB_GET_A_SINGLE_JOB_BY_ID",
    GET_A_SPECIFIC_BROADCAST_MESSAGE: "GITLAB_GET_A_SPECIFIC_BROADCAST_MESSAGE",
    GET_CURRENT_PLAN_LIMITS: "GITLAB_GET_CURRENT_PLAN_LIMITS",
    GET_GIT_LAB_MIGRATION_DETAILS: "GITLAB_GET_GIT_LAB_MIGRATION_DETAILS",
    GET_GIT_LAB_MIGRATION_ENTITY_DETAILS: "GITLAB_GET_GIT_LAB_MIGRATION_ENTITY_DETAILS",
    GET_INSTANCE_METADATA: "GITLAB_GET_INSTANCE_METADATA",
    GET_INSTANCE_VERSION: "GITLAB_GET_INSTANCE_VERSION",
    GET_USER_VIEWABLE_BADGES: "GITLAB_GET_USER_VIEWABLE_BADGES",
    GIT_LAB_USER_ACCESS_APPROVAL_FEATURE: "GITLAB_GIT_LAB_USER_ACCESS_APPROVAL_FEATURE",
    LIST_ALL_GIT_LAB_MIGRATIONS: "GITLAB_LIST_ALL_GIT_LAB_MIGRATIONS",
    LIST_ALL_GIT_LAB_MIGRATIONS_ENTITIES: "GITLAB_LIST_ALL_GIT_LAB_MIGRATIONS_ENTITIES",
    LIST_AUTHENTICATED_USER_S_PROJECT_BADGES: "GITLAB_LIST_AUTHENTICATED_USER_S_PROJECT_BADGES",
    LIST_BATCHED_BACKGROUND_MIGRATIONS: "GITLAB_LIST_BATCHED_BACKGROUND_MIGRATIONS",
    LIST_CI_INSTANCE_VARIABLES: "GITLAB_LIST_CI_INSTANCE_VARIABLES",
    LIST_GIT_LAB_MIGRATION_ENTITIES: "GITLAB_LIST_GIT_LAB_MIGRATION_ENTITIES",
    LIST_INSTANCE_CLUSTERS: "GITLAB_LIST_INSTANCE_CLUSTERS",
    LIST_JOBS_FOR_A_PROJECT: "GITLAB_LIST_JOBS_FOR_A_PROJECT",
    MARK_MIGRATION_AS_EXECUTED: "GITLAB_MARK_MIGRATION_AS_EXECUTED",
    MODIFY_APPLICATION_APPEARANCE: "GITLAB_MODIFY_APPLICATION_APPEARANCE",
    PAUSE_BATCHED_BACKGROUND_MIGRATION_BY_ID: "GITLAB_PAUSE_BATCHED_BACKGROUND_MIGRATION_BY_ID",
    PREVIEW_A_BADGE_FROM_A_GROUP: "GITLAB_PREVIEW_A_BADGE_FROM_A_GROUP",
    PREVIEW_A_BADGE_FROM_A_PROJECT: "GITLAB_PREVIEW_A_BADGE_FROM_A_PROJECT",
    PROTECT_A_PROJECT_BRANCH: "GITLAB_PROTECT_A_PROJECT_BRANCH",
    REJECT_USER_ACCESS_REQUEST: "GITLAB_REJECT_USER_ACCESS_REQUEST",
    REMOVES_A_BADGE_FROM_THE_GROUP: "GITLAB_REMOVES_A_BADGE_FROM_THE_GROUP",
    REMOVES_A_BADGE_FROM_THE_PROJECT: "GITLAB_REMOVES_A_BADGE_FROM_THE_PROJECT",
    REMOVE_MERGED_BRANCHES_BY_PROJECT_ID: "GITLAB_REMOVE_MERGED_BRANCHES_BY_PROJECT_ID",
    REMOVE_METRIC_IMAGE_FROM_ALERT: "GITLAB_REMOVE_METRIC_IMAGE_FROM_ALERT",
    REQUESTS_ACCESS_FOR_THE_AUTHENTICATED_USER_TO_A_GROUP: "GITLAB_REQUESTS_ACCESS_FOR_THE_AUTHENTICATED_USER_TO_A_GROUP",
    REQUEST_PROJECT_ACCESS: "GITLAB_REQUEST_PROJECT_ACCESS",
    RESUME_BATCH_BACKGROUND_MIGRATION_BY_ID: "GITLAB_RESUME_BATCH_BACKGROUND_MIGRATION_BY_ID",
    RETRIEVE_BATCHED_BACKGROUND_MIGRATION: "GITLAB_RETRIEVE_BATCHED_BACKGROUND_MIGRATION",
    RETRIEVE_CURRENT_APPEARANCE_SETTINGS: "GITLAB_RETRIEVE_CURRENT_APPEARANCE_SETTINGS",
    RETRIEVE_DICTIONARY_TABLE_DETAILS: "GITLAB_RETRIEVE_DICTIONARY_TABLE_DETAILS",
    RETRIEVE_SPECIFIC_PROJECT_REPOSITORY_BRANCH: "GITLAB_RETRIEVE_SPECIFIC_PROJECT_REPOSITORY_BRANCH",
    RUN_A_MANUAL_JOB: "GITLAB_RUN_A_MANUAL_JOB",
    START_A_NEW_GIT_LAB_MIGRATION: "GITLAB_START_A_NEW_GIT_LAB_MIGRATION",
    UNPROTECT_A_SINGLE_REPOSITORY_BRANCH: "GITLAB_UNPROTECT_A_SINGLE_REPOSITORY_BRANCH",
    UPDATES_A_BADGE_OF_A_GROUP: "GITLAB_UPDATES_A_BADGE_OF_A_GROUP",
    UPDATES_A_BADGE_OF_A_PROJECT: "GITLAB_UPDATES_A_BADGE_OF_A_PROJECT",
    UPDATE_A_BROADCAST_MESSAGE: "GITLAB_UPDATE_A_BROADCAST_MESSAGE",
    UPDATE_INSTANCE_LEVEL_CI_VARIABLE: "GITLAB_UPDATE_INSTANCE_LEVEL_CI_VARIABLE",
    UPDATE_METRIC_IMAGE_FOR_AN_ALERT: "GITLAB_UPDATE_METRIC_IMAGE_FOR_AN_ALERT",
    UPLOAD_METRIC_IMAGE_FOR_SPECIFIC_ALERT: "GITLAB_UPLOAD_METRIC_IMAGE_FOR_SPECIFIC_ALERT",
  },
  triggerTypes: {},
}
