// @composio/core/generated/gong.ts
/**
 * Map of Composio's GONG toolkit.
 */
export const GONG = {
  slug: "gong",
  tools: {
    ADD_CALL_MEDIA_V2_CALLS_ID_MEDIA: "GONG_ADD_CALL_MEDIA_V2_CALLS_ID_MEDIA",
    ADD_NEW_CALL_V2_CALLS: "GONG_ADD_NEW_CALL_V2_CALLS",
    AGGREGATE_ACTIVITY_BY_PERIOD_VIA_API: "GONG_AGGREGATE_ACTIVITY_BY_PERIOD_VIA_API",
    AGGREGATE_USER_ACTIVITY_STATISTICS: "GONG_AGGREGATE_USER_ACTIVITY_STATISTICS",
    ASSIGN_PROSPECTS_TO_FLOW: "GONG_ASSIGN_PROSPECTS_TO_FLOW",
    CREATE_ACTIVITY_SCORECARDS_REPORT: "GONG_CREATE_ACTIVITY_SCORECARDS_REPORT",
    CREATE_A_NEW_GONG_MEETING_V2_MEETINGS: "GONG_CREATE_A_NEW_GONG_MEETING_V2_MEETINGS",
    CREATE_PERMISSION_PROFILE_V2_PERMISSION_PROFILE: "GONG_CREATE_PERMISSION_PROFILE_V2_PERMISSION_PROFILE",
    DATA_PRIVACY_FOR_PHONE_NUMBER: "GONG_DATA_PRIVACY_FOR_PHONE_NUMBER",
    DELETE_A_GENERIC_CRM_INTEGRATION_V2_CRM_INTEGRATIONS: "GONG_DELETE_A_GENERIC_CRM_INTEGRATION_V2_CRM_INTEGRATIONS",
    DELETE_A_GONG_MEETING_V2_MEETINGS: "GONG_DELETE_A_GONG_MEETING_V2_MEETINGS",
    DELETE_USERS_FROM_CALL_ACCESS_LIST: "GONG_DELETE_USERS_FROM_CALL_ACCESS_LIST",
    ERASE_DATA_FOR_EMAIL_ADDRESS: "GONG_ERASE_DATA_FOR_EMAIL_ADDRESS",
    ERASE_DATA_FOR_PHONE_NUMBER: "GONG_ERASE_DATA_FOR_PHONE_NUMBER",
    FETCH_ALL_PERMISSION_PROFILES: "GONG_FETCH_ALL_PERMISSION_PROFILES",
    GET_CRM_INTEGRATION_DETAILS: "GONG_GET_CRM_INTEGRATION_DETAILS",
    GET_CRM_OBJECTS_V2_CRM_ENTITIES: "GONG_GET_CRM_OBJECTS_V2_CRM_ENTITIES",
    GET_PERMISSION_PROFILE: "GONG_GET_PERMISSION_PROFILE",
    GET_REQUEST_STATUS_V2_CRM_REQUEST_STATUS: "GONG_GET_REQUEST_STATUS_V2_CRM_REQUEST_STATUS",
    INTEGRATION_SETTINGS_V2_INTEGRATION_SETTINGS: "GONG_INTEGRATION_SETTINGS_V2_INTEGRATION_SETTINGS",
    LIST_ALL_COACHING_METRICS_V2_COACHING: "GONG_LIST_ALL_COACHING_METRICS_V2_COACHING",
    LIST_ALL_COMPANY_WORKSPACES_V2_WORKSPACES: "GONG_LIST_ALL_COMPANY_WORKSPACES_V2_WORKSPACES",
    LIST_ALL_USERS_V2_USERS: "GONG_LIST_ALL_USERS_V2_USERS",
    LIST_FLOWS_FOR_CRM_PROSPECTS: "GONG_LIST_FLOWS_FOR_CRM_PROSPECTS",
    LIST_FOLDER_CALLS: "GONG_LIST_FOLDER_CALLS",
    LIST_GONG_ENGAGE_FLOWS_V2_FLOWS: "GONG_LIST_GONG_ENGAGE_FLOWS_V2_FLOWS",
    LIST_SCHEMA_FIELDS_V2_CRM_ENTITY_SCHEMA: "GONG_LIST_SCHEMA_FIELDS_V2_CRM_ENTITY_SCHEMA",
    LIST_USERS_BY_FILTER_V2_USERS_EXTENSIVE: "GONG_LIST_USERS_BY_FILTER_V2_USERS_EXTENSIVE",
    MANAGE_USER_CALL_ACCESS: "GONG_MANAGE_USER_CALL_ACCESS",
    POST_A_DIGITAL_INTERACTION_V2_DIGITAL_INTERACTION: "GONG_POST_A_DIGITAL_INTERACTION_V2_DIGITAL_INTERACTION",
    POST_DAY_BY_DAY_ACTIVITY_STATS: "GONG_POST_DAY_BY_DAY_ACTIVITY_STATS",
    POST_INTERACTION_FILTER_STATS: "GONG_POST_INTERACTION_FILTER_STATS",
    POST_MEETING_INTEGRATION_STATUS: "GONG_POST_MEETING_INTEGRATION_STATUS",
    REGISTER_CRM_INTEGRATION: "GONG_REGISTER_CRM_INTEGRATION",
    REPORT_CONTENT_VIEWED_EVENT: "GONG_REPORT_CONTENT_VIEWED_EVENT",
    REPORT_CUSTOM_ENGAGEMENT_EVENT: "GONG_REPORT_CUSTOM_ENGAGEMENT_EVENT",
    RETRIEVE_CALL_DATA_BY_DATE_RANGE_V2_CALLS: "GONG_RETRIEVE_CALL_DATA_BY_DATE_RANGE_V2_CALLS",
    RETRIEVE_DATA_FOR_A_SPECIFIC_CALL_V2_CALLS_ID: "GONG_RETRIEVE_DATA_FOR_A_SPECIFIC_CALL_V2_CALLS_ID",
    RETRIEVE_DATA_PRIVACY_INFO_FOR_EMAIL_ADDRESS: "GONG_RETRIEVE_DATA_PRIVACY_INFO_FOR_EMAIL_ADDRESS",
    RETRIEVE_FILTERED_CALL_DETAILS: "GONG_RETRIEVE_FILTERED_CALL_DETAILS",
    RETRIEVE_LIBRARY_FOLDERS_V2_LIBRARY_FOLDERS: "GONG_RETRIEVE_LIBRARY_FOLDERS_V2_LIBRARY_FOLDERS",
    RETRIEVE_LOGS_DATA_BY_TYPE_AND_TIME_RANGE_V2_LOGS: "GONG_RETRIEVE_LOGS_DATA_BY_TYPE_AND_TIME_RANGE_V2_LOGS",
    RETRIEVE_MANUAL_CRM_CALL_ASSOCIATIONS: "GONG_RETRIEVE_MANUAL_CRM_CALL_ASSOCIATIONS",
    RETRIEVE_SCORECARDS_DETAILS_V2_SETTINGS_SCORECARDS: "GONG_RETRIEVE_SCORECARDS_DETAILS_V2_SETTINGS_SCORECARDS",
    RETRIEVE_TRACKER_DETAILS_V2_SETTINGS_TRACKERS: "GONG_RETRIEVE_TRACKER_DETAILS_V2_SETTINGS_TRACKERS",
    RETRIEVE_TRANSCRIPTS_OF_CALLS_V2_CALLS_TRANSCRIPT: "GONG_RETRIEVE_TRANSCRIPTS_OF_CALLS_V2_CALLS_TRANSCRIPT",
    RETRIEVE_USERS_FROM_PERMISSION_PROFILE: "GONG_RETRIEVE_USERS_FROM_PERMISSION_PROFILE",
    RETRIEVE_USER_SETTINGS_HISTORY: "GONG_RETRIEVE_USER_SETTINGS_HISTORY",
    RETRIEVE_USER_V2_USERS_ID: "GONG_RETRIEVE_USER_V2_USERS_ID",
    SET_USER_CALL_ACCESS: "GONG_SET_USER_CALL_ACCESS",
    UPDATE_A_GONG_MEETING_V2_MEETINGS_MEETING_ID: "GONG_UPDATE_A_GONG_MEETING_V2_MEETINGS_MEETING_ID",
    UPDATE_PERMISSION_PROFILE_V2_PERMISSION_PROFILE: "GONG_UPDATE_PERMISSION_PROFILE_V2_PERMISSION_PROFILE",
    UPDATE_SHARED_CONTENT_EVENT: "GONG_UPDATE_SHARED_CONTENT_EVENT",
    UPLOAD_CRM_OBJECTS_V2_CRM_ENTITIES: "GONG_UPLOAD_CRM_OBJECTS_V2_CRM_ENTITIES",
  },
  triggerTypes: {},
}
