// @composio/core/generated/googledocs.ts
/**
 * Map of Composio's GOOGLEDOCS toolkit.
 */
export const GOOGLEDOCS = {
  slug: "googledocs",
  tools: {
    COPY_DOCUMENT: "GOOGLEDOCS_COPY_DOCUMENT",
    CREATE_DOCUMENT: "GOOGLEDOCS_CREATE_DOCUMENT",
    CREATE_DOCUMENT_MARKDOWN: "GOOGLEDOCS_CREATE_DOCUMENT_MARKDOWN",
    CREATE_FOOTER: "GOOGLEDOCS_CREATE_FOOTER",
    CREATE_FOOTNOTE: "GOOGLEDOCS_CREATE_FOOTNOTE",
    CREATE_HEADER: "GOOGLEDOCS_CREATE_HEADER",
    CREATE_NAMED_RANGE: "GOOGLEDOCS_CREATE_NAMED_RANGE",
    CREATE_PARAGRAPH_BULLETS: "GOOGLEDOCS_CREATE_PARAGRAPH_BULLETS",
    DELETE_CONTENT_RANGE: "GOOGLEDOCS_DELETE_CONTENT_RANGE",
    DELETE_FOOTER: "GOOGLEDOCS_DELETE_FOOTER",
    DELETE_HEADER: "GOOGLEDOCS_DELETE_HEADER",
    DELETE_NAMED_RANGE: "GOOGLEDOCS_DELETE_NAMED_RANGE",
    DELETE_PARAGRAPH_BULLETS: "GOOGLEDOCS_DELETE_PARAGRAPH_BULLETS",
    DELETE_TABLE: "GOOGLEDOCS_DELETE_TABLE",
    DELETE_TABLE_COLUMN: "GOOGLEDOCS_DELETE_TABLE_COLUMN",
    DELETE_TABLE_ROW: "GOOGLEDOCS_DELETE_TABLE_ROW",
    GET_CHARTS_FROM_SPREADSHEET: "GOOGLEDOCS_GET_CHARTS_FROM_SPREADSHEET",
    GET_DOCUMENT_BY_ID: "GOOGLEDOCS_GET_DOCUMENT_BY_ID",
    INSERT_INLINE_IMAGE: "GOOGLEDOCS_INSERT_INLINE_IMAGE",
    INSERT_PAGE_BREAK: "GOOGLEDOCS_INSERT_PAGE_BREAK",
    INSERT_TABLE_ACTION: "GOOGLEDOCS_INSERT_TABLE_ACTION",
    INSERT_TABLE_COLUMN: "GOOGLEDOCS_INSERT_TABLE_COLUMN",
    INSERT_TEXT_ACTION: "GOOGLEDOCS_INSERT_TEXT_ACTION",
    LIST_SPREADSHEET_CHARTS_ACTION: "GOOGLEDOCS_LIST_SPREADSHEET_CHARTS_ACTION",
    REPLACE_ALL_TEXT: "GOOGLEDOCS_REPLACE_ALL_TEXT",
    REPLACE_IMAGE: "GOOGLEDOCS_REPLACE_IMAGE",
    SEARCH_DOCUMENTS: "GOOGLEDOCS_SEARCH_DOCUMENTS",
    UNMERGE_TABLE_CELLS: "GOOGLEDOCS_UNMERGE_TABLE_CELLS",
    UPDATE_DOCUMENT_MARKDOWN: "GOOGLEDOCS_UPDATE_DOCUMENT_MARKDOWN",
    UPDATE_DOCUMENT_STYLE: "GOOGLEDOCS_UPDATE_DOCUMENT_STYLE",
    UPDATE_EXISTING_DOCUMENT: "GOOGLEDOCS_UPDATE_EXISTING_DOCUMENT",
    UPDATE_TABLE_ROW_STYLE: "GOOGLEDOCS_UPDATE_TABLE_ROW_STYLE",
  },
  triggerTypes: {},
}
