// @composio/core/generated/googlesheets.ts
/**
 * Map of Composio's GOOGLESHEETS toolkit.
 */
export const GOOGLESHEETS = {
  slug: "googlesheets",
  tools: {
    ADD_SHEET: "GOOGLESHEETS_ADD_SHEET",
    AGGREGATE_COLUMN_DATA: "GOOGLESHEETS_AGGREGATE_COLUMN_DATA",
    APPEND_DIMENSION: "GOOGLESHEETS_APPEND_DIMENSION",
    BATCH_GET: "GOOGLESHEETS_BATCH_GET",
    BATCH_UPDATE: "GOOGLESHEETS_BATCH_UPDATE",
    BATCH_UPDATE_VALUES_BY_DATA_FILTER: "GOOGLESHEETS_BATCH_UPDATE_VALUES_BY_DATA_FILTER",
    CLEAR_BASIC_FILTER: "GOOGLESHEETS_CLEAR_BASIC_FILTER",
    CLEAR_VALUES: "GOOGLESHEETS_CLEAR_VALUES",
    CREATE_GOOGLE_SHEET1: "GOOGLESHEETS_CREATE_GOOGLE_SHEET1",
    CREATE_SPREADSHEET_COLUMN: "GOOGLESHEETS_CREATE_SPREADSHEET_COLUMN",
    CREATE_SPREADSHEET_ROW: "GOOGLESHEETS_CREATE_SPREADSHEET_ROW",
    DELETE_DIMENSION: "GOOGLESHEETS_DELETE_DIMENSION",
    DELETE_SHEET: "GOOGLESHEETS_DELETE_SHEET",
    FIND_WORKSHEET_BY_TITLE: "GOOGLESHEETS_FIND_WORKSHEET_BY_TITLE",
    FORMAT_CELL: "GOOGLESHEETS_FORMAT_CELL",
    GET_SHEET_NAMES: "GOOGLESHEETS_GET_SHEET_NAMES",
    GET_SPREADSHEET_BY_DATA_FILTER: "GOOGLESHEETS_GET_SPREADSHEET_BY_DATA_FILTER",
    GET_SPREADSHEET_INFO: "GOOGLESHEETS_GET_SPREADSHEET_INFO",
    INSERT_DIMENSION: "GOOGLESHEETS_INSERT_DIMENSION",
    LOOKUP_SPREADSHEET_ROW: "GOOGLESHEETS_LOOKUP_SPREADSHEET_ROW",
    SEARCH_DEVELOPER_METADATA: "GOOGLESHEETS_SEARCH_DEVELOPER_METADATA",
    SEARCH_SPREADSHEETS: "GOOGLESHEETS_SEARCH_SPREADSHEETS",
    SET_BASIC_FILTER: "GOOGLESHEETS_SET_BASIC_FILTER",
    SHEET_FROM_JSON: "GOOGLESHEETS_SHEET_FROM_JSON",
    SPREADSHEETS_SHEETS_COPY_TO: "GOOGLESHEETS_SPREADSHEETS_SHEETS_COPY_TO",
    SPREADSHEETS_VALUES_APPEND: "GOOGLESHEETS_SPREADSHEETS_VALUES_APPEND",
    SPREADSHEETS_VALUES_BATCH_CLEAR: "GOOGLESHEETS_SPREADSHEETS_VALUES_BATCH_CLEAR",
    SPREADSHEETS_VALUES_BATCH_CLEAR_BY_DATA_FILTER: "GOOGLESHEETS_SPREADSHEETS_VALUES_BATCH_CLEAR_BY_DATA_FILTER",
    SPREADSHEETS_VALUES_BATCH_GET_BY_DATA_FILTER: "GOOGLESHEETS_SPREADSHEETS_VALUES_BATCH_GET_BY_DATA_FILTER",
    UPDATE_SHEET_PROPERTIES: "GOOGLESHEETS_UPDATE_SHEET_PROPERTIES",
    UPDATE_SPREADSHEET_PROPERTIES: "GOOGLESHEETS_UPDATE_SPREADSHEET_PROPERTIES",
  },
  triggerTypes: {},
}
