// @composio/core/generated/googlesuper.ts
/**
 * Map of Composio's GOOGLESUPER toolkit.
 */
export const GOOGLESUPER = {
  slug: "googlesuper",
  tools: {
    ADD_ENRICHMENT: "GOOGLESUPER_ADD_ENRICHMENT",
    ADD_FILE_SHARING_PREFERENCE: "GOOGLESUPER_ADD_FILE_SHARING_PREFERENCE",
    ADD_LABEL_TO_EMAIL: "GOOGLESUPER_ADD_LABEL_TO_EMAIL",
    ADD_OR_REMOVE_TO_CUSTOMER_LIST: "GOOGLESUPER_ADD_OR_REMOVE_TO_CUSTOMER_LIST",
    ADD_SHEET: "GOOGLESUPER_ADD_SHEET",
    AGGREGATE_COLUMN_DATA: "GOOGLESUPER_AGGREGATE_COLUMN_DATA",
    APPEND_DIMENSION: "GOOGLESUPER_APPEND_DIMENSION",
    BATCH_ADD_MEDIA_ITEMS: "GOOGLESUPER_BATCH_ADD_MEDIA_ITEMS",
    BATCH_CREATE_MEDIA_ITEMS: "GOOGLESUPER_BATCH_CREATE_MEDIA_ITEMS",
    BATCH_GET: "GOOGLESUPER_BATCH_GET",
    BATCH_GET_MEDIA_ITEMS: "GOOGLESUPER_BATCH_GET_MEDIA_ITEMS",
    BATCH_UPDATE: "GOOGLESUPER_BATCH_UPDATE",
    BATCH_UPDATE_VALUES_BY_DATA_FILTER: "GOOGLESUPER_BATCH_UPDATE_VALUES_BY_DATA_FILTER",
    CALENDARS_DELETE: "GOOGLESUPER_CALENDARS_DELETE",
    CALENDARS_UPDATE: "GOOGLESUPER_CALENDARS_UPDATE",
    CALENDAR_LIST_INSERT: "GOOGLESUPER_CALENDAR_LIST_INSERT",
    CALENDAR_LIST_UPDATE: "GOOGLESUPER_CALENDAR_LIST_UPDATE",
    CLEAR_BASIC_FILTER: "GOOGLESUPER_CLEAR_BASIC_FILTER",
    CLEAR_CALENDAR: "GOOGLESUPER_CLEAR_CALENDAR",
    CLEAR_TASKS: "GOOGLESUPER_CLEAR_TASKS",
    CLEAR_VALUES: "GOOGLESUPER_CLEAR_VALUES",
    COPY_DOCUMENT: "GOOGLESUPER_COPY_DOCUMENT",
    COPY_FILE: "GOOGLESUPER_COPY_FILE",
    CREATE_ALBUM: "GOOGLESUPER_CREATE_ALBUM",
    CREATE_COMMENT: "GOOGLESUPER_CREATE_COMMENT",
    CREATE_CUSTOMER_LIST: "GOOGLESUPER_CREATE_CUSTOMER_LIST",
    CREATE_DOCUMENT: "GOOGLESUPER_CREATE_DOCUMENT",
    CREATE_DOCUMENT_MARKDOWN: "GOOGLESUPER_CREATE_DOCUMENT_MARKDOWN",
    CREATE_DRIVE: "GOOGLESUPER_CREATE_DRIVE",
    CREATE_EMAIL_DRAFT: "GOOGLESUPER_CREATE_EMAIL_DRAFT",
    CREATE_EVENT: "GOOGLESUPER_CREATE_EVENT",
    CREATE_EXPANDED_DATA_SET: "GOOGLESUPER_CREATE_EXPANDED_DATA_SET",
    CREATE_FILE: "GOOGLESUPER_CREATE_FILE",
    CREATE_FILE_FROM_TEXT: "GOOGLESUPER_CREATE_FILE_FROM_TEXT",
    CREATE_FOLDER: "GOOGLESUPER_CREATE_FOLDER",
    CREATE_FOOTER: "GOOGLESUPER_CREATE_FOOTER",
    CREATE_FOOTNOTE: "GOOGLESUPER_CREATE_FOOTNOTE",
    CREATE_GOOGLE_SHEET1: "GOOGLESUPER_CREATE_GOOGLE_SHEET1",
    CREATE_HEADER: "GOOGLESUPER_CREATE_HEADER",
    CREATE_LABEL: "GOOGLESUPER_CREATE_LABEL",
    CREATE_MEET: "GOOGLESUPER_CREATE_MEET",
    CREATE_NAMED_RANGE: "GOOGLESUPER_CREATE_NAMED_RANGE",
    CREATE_PARAGRAPH_BULLETS: "GOOGLESUPER_CREATE_PARAGRAPH_BULLETS",
    CREATE_REPLY: "GOOGLESUPER_CREATE_REPLY",
    CREATE_SHORTCUT_TO_FILE: "GOOGLESUPER_CREATE_SHORTCUT_TO_FILE",
    CREATE_SPREADSHEET_COLUMN: "GOOGLESUPER_CREATE_SPREADSHEET_COLUMN",
    CREATE_SPREADSHEET_ROW: "GOOGLESUPER_CREATE_SPREADSHEET_ROW",
    CREATE_TASK_LIST: "GOOGLESUPER_CREATE_TASK_LIST",
    DELETE_COMMENT: "GOOGLESUPER_DELETE_COMMENT",
    DELETE_CONTENT_RANGE: "GOOGLESUPER_DELETE_CONTENT_RANGE",
    DELETE_DIMENSION: "GOOGLESUPER_DELETE_DIMENSION",
    DELETE_DRAFT: "GOOGLESUPER_DELETE_DRAFT",
    DELETE_DRIVE: "GOOGLESUPER_DELETE_DRIVE",
    DELETE_EVENT: "GOOGLESUPER_DELETE_EVENT",
    DELETE_FOOTER: "GOOGLESUPER_DELETE_FOOTER",
    DELETE_HEADER: "GOOGLESUPER_DELETE_HEADER",
    DELETE_MESSAGE: "GOOGLESUPER_DELETE_MESSAGE",
    DELETE_NAMED_RANGE: "GOOGLESUPER_DELETE_NAMED_RANGE",
    DELETE_PARAGRAPH_BULLETS: "GOOGLESUPER_DELETE_PARAGRAPH_BULLETS",
    DELETE_PERMISSION: "GOOGLESUPER_DELETE_PERMISSION",
    DELETE_REPLY: "GOOGLESUPER_DELETE_REPLY",
    DELETE_SHEET: "GOOGLESUPER_DELETE_SHEET",
    DELETE_TABLE: "GOOGLESUPER_DELETE_TABLE",
    DELETE_TABLE_COLUMN: "GOOGLESUPER_DELETE_TABLE_COLUMN",
    DELETE_TABLE_ROW: "GOOGLESUPER_DELETE_TABLE_ROW",
    DELETE_TASK: "GOOGLESUPER_DELETE_TASK",
    DELETE_TASK_LIST: "GOOGLESUPER_DELETE_TASK_LIST",
    DISTANCE_MATRIX_API: "GOOGLESUPER_DISTANCE_MATRIX_API",
    DOWNLOAD_FILE: "GOOGLESUPER_DOWNLOAD_FILE",
    DUPLICATE_CALENDAR: "GOOGLESUPER_DUPLICATE_CALENDAR",
    EDIT_FILE: "GOOGLESUPER_EDIT_FILE",
    EMPTY_TRASH: "GOOGLESUPER_EMPTY_TRASH",
    EVENTS_INSTANCES: "GOOGLESUPER_EVENTS_INSTANCES",
    EVENTS_LIST: "GOOGLESUPER_EVENTS_LIST",
    EVENTS_MOVE: "GOOGLESUPER_EVENTS_MOVE",
    EVENTS_WATCH: "GOOGLESUPER_EVENTS_WATCH",
    FETCH_EMAILS: "GOOGLESUPER_FETCH_EMAILS",
    FETCH_MESSAGE_BY_MESSAGE_ID: "GOOGLESUPER_FETCH_MESSAGE_BY_MESSAGE_ID",
    FETCH_MESSAGE_BY_THREAD_ID: "GOOGLESUPER_FETCH_MESSAGE_BY_THREAD_ID",
    FILES_MODIFY_LABELS: "GOOGLESUPER_FILES_MODIFY_LABELS",
    FIND_EVENT: "GOOGLESUPER_FIND_EVENT",
    FIND_FILE: "GOOGLESUPER_FIND_FILE",
    FIND_FOLDER: "GOOGLESUPER_FIND_FOLDER",
    FIND_FREE_SLOTS: "GOOGLESUPER_FIND_FREE_SLOTS",
    FIND_WORKSHEET_BY_TITLE: "GOOGLESUPER_FIND_WORKSHEET_BY_TITLE",
    FORMAT_CELL: "GOOGLESUPER_FORMAT_CELL",
    FREE_BUSY_QUERY: "GOOGLESUPER_FREE_BUSY_QUERY",
    GENERATE_IDS: "GOOGLESUPER_GENERATE_IDS",
    GEOCODING_API: "GOOGLESUPER_GEOCODING_API",
    GET_ABOUT: "GOOGLESUPER_GET_ABOUT",
    GET_ACCOUNT: "GOOGLESUPER_GET_ACCOUNT",
    GET_ALBUM: "GOOGLESUPER_GET_ALBUM",
    GET_ATTACHMENT: "GOOGLESUPER_GET_ATTACHMENT",
    GET_CALENDAR: "GOOGLESUPER_GET_CALENDAR",
    GET_CAMPAIGN_BY_ID: "GOOGLESUPER_GET_CAMPAIGN_BY_ID",
    GET_CAMPAIGN_BY_NAME: "GOOGLESUPER_GET_CAMPAIGN_BY_NAME",
    GET_CHANGES_START_PAGE_TOKEN: "GOOGLESUPER_GET_CHANGES_START_PAGE_TOKEN",
    GET_CHARTS_FROM_SPREADSHEET: "GOOGLESUPER_GET_CHARTS_FROM_SPREADSHEET",
    GET_COMMENT: "GOOGLESUPER_GET_COMMENT",
    GET_CONFERENCE_RECORD_FOR_MEET: "GOOGLESUPER_GET_CONFERENCE_RECORD_FOR_MEET",
    GET_CONTACTS: "GOOGLESUPER_GET_CONTACTS",
    GET_CURRENT_DATE_TIME: "GOOGLESUPER_GET_CURRENT_DATE_TIME",
    GET_CUSTOMER_LISTS: "GOOGLESUPER_GET_CUSTOMER_LISTS",
    GET_DIRECTION: "GOOGLESUPER_GET_DIRECTION",
    GET_DOCUMENT_BY_ID: "GOOGLESUPER_GET_DOCUMENT_BY_ID",
    GET_DRIVE: "GOOGLESUPER_GET_DRIVE",
    GET_FILE_METADATA: "GOOGLESUPER_GET_FILE_METADATA",
    GET_MEDIA_ITEM_DOWNLOAD: "GOOGLESUPER_GET_MEDIA_ITEM_DOWNLOAD",
    GET_MEET: "GOOGLESUPER_GET_MEET",
    GET_PARTICIPANT_SESSION: "GOOGLESUPER_GET_PARTICIPANT_SESSION",
    GET_PEOPLE: "GOOGLESUPER_GET_PEOPLE",
    GET_PERMISSION: "GOOGLESUPER_GET_PERMISSION",
    GET_PROFILE: "GOOGLESUPER_GET_PROFILE",
    GET_RECORDINGS_BY_CONFERENCE_RECORD_ID: "GOOGLESUPER_GET_RECORDINGS_BY_CONFERENCE_RECORD_ID",
    GET_REPLY: "GOOGLESUPER_GET_REPLY",
    GET_REVISION: "GOOGLESUPER_GET_REVISION",
    GET_ROUTE: "GOOGLESUPER_GET_ROUTE",
    GET_SHEET_NAMES: "GOOGLESUPER_GET_SHEET_NAMES",
    GET_SPREADSHEET_BY_DATA_FILTER: "GOOGLESUPER_GET_SPREADSHEET_BY_DATA_FILTER",
    GET_SPREADSHEET_INFO: "GOOGLESUPER_GET_SPREADSHEET_INFO",
    GET_TASK: "GOOGLESUPER_GET_TASK",
    GET_TASK_LIST: "GOOGLESUPER_GET_TASK_LIST",
    GET_TRANSCRIPTS_BY_CONFERENCE_RECORD_ID: "GOOGLESUPER_GET_TRANSCRIPTS_BY_CONFERENCE_RECORD_ID",
    GOOGLE_DRIVE_DELETE_FOLDER_OR_FILE_ACTION: "GOOGLESUPER_GOOGLE_DRIVE_DELETE_FOLDER_OR_FILE_ACTION",
    HIDE_DRIVE: "GOOGLESUPER_HIDE_DRIVE",
    INSERT_DIMENSION: "GOOGLESUPER_INSERT_DIMENSION",
    INSERT_INLINE_IMAGE: "GOOGLESUPER_INSERT_INLINE_IMAGE",
    INSERT_PAGE_BREAK: "GOOGLESUPER_INSERT_PAGE_BREAK",
    INSERT_TABLE_ACTION: "GOOGLESUPER_INSERT_TABLE_ACTION",
    INSERT_TABLE_COLUMN: "GOOGLESUPER_INSERT_TABLE_COLUMN",
    INSERT_TASK: "GOOGLESUPER_INSERT_TASK",
    INSERT_TEXT_ACTION: "GOOGLESUPER_INSERT_TEXT_ACTION",
    LIST_ACCOUNTS: "GOOGLESUPER_LIST_ACCOUNTS",
    LIST_ACL_RULES: "GOOGLESUPER_LIST_ACL_RULES",
    LIST_ALBUMS: "GOOGLESUPER_LIST_ALBUMS",
    LIST_AUDIENCES: "GOOGLESUPER_LIST_AUDIENCES",
    LIST_CALENDARS: "GOOGLESUPER_LIST_CALENDARS",
    LIST_CHANGES: "GOOGLESUPER_LIST_CHANGES",
    LIST_COMMENTS: "GOOGLESUPER_LIST_COMMENTS",
    LIST_CONFERENCE_RECORDS: "GOOGLESUPER_LIST_CONFERENCE_RECORDS",
    LIST_DRAFTS: "GOOGLESUPER_LIST_DRAFTS",
    LIST_FILES: "GOOGLESUPER_LIST_FILES",
    LIST_FILE_LABELS: "GOOGLESUPER_LIST_FILE_LABELS",
    LIST_LABELS: "GOOGLESUPER_LIST_LABELS",
    LIST_MEDIA_ITEMS: "GOOGLESUPER_LIST_MEDIA_ITEMS",
    LIST_PARTICIPANT_SESSIONS: "GOOGLESUPER_LIST_PARTICIPANT_SESSIONS",
    LIST_PERMISSIONS: "GOOGLESUPER_LIST_PERMISSIONS",
    LIST_REPLIES_TO_COMMENT: "GOOGLESUPER_LIST_REPLIES_TO_COMMENT",
    LIST_REVISIONS: "GOOGLESUPER_LIST_REVISIONS",
    LIST_SHARED_DRIVES: "GOOGLESUPER_LIST_SHARED_DRIVES",
    LIST_SPREADSHEET_CHARTS_ACTION: "GOOGLESUPER_LIST_SPREADSHEET_CHARTS_ACTION",
    LIST_TASKS: "GOOGLESUPER_LIST_TASKS",
    LIST_TASK_LISTS: "GOOGLESUPER_LIST_TASK_LISTS",
    LIST_THREADS: "GOOGLESUPER_LIST_THREADS",
    LOOKUP_SPREADSHEET_ROW: "GOOGLESUPER_LOOKUP_SPREADSHEET_ROW",
    MAPS_EMBED_API: "GOOGLESUPER_MAPS_EMBED_API",
    MODIFY_THREAD_LABELS: "GOOGLESUPER_MODIFY_THREAD_LABELS",
    MOVE_FILE: "GOOGLESUPER_MOVE_FILE",
    MOVE_TASK: "GOOGLESUPER_MOVE_TASK",
    MOVE_TO_TRASH: "GOOGLESUPER_MOVE_TO_TRASH",
    NEARBY_SEARCH: "GOOGLESUPER_NEARBY_SEARCH",
    PARSE_FILE: "GOOGLESUPER_PARSE_FILE",
    PATCH_CALENDAR: "GOOGLESUPER_PATCH_CALENDAR",
    PATCH_EVENT: "GOOGLESUPER_PATCH_EVENT",
    PATCH_LABEL: "GOOGLESUPER_PATCH_LABEL",
    PATCH_TASK: "GOOGLESUPER_PATCH_TASK",
    PATCH_TASK_LIST: "GOOGLESUPER_PATCH_TASK_LIST",
    QUICK_ADD: "GOOGLESUPER_QUICK_ADD",
    REMOVE_ATTENDEE: "GOOGLESUPER_REMOVE_ATTENDEE",
    REMOVE_LABEL: "GOOGLESUPER_REMOVE_LABEL",
    REPLACE_ALL_TEXT: "GOOGLESUPER_REPLACE_ALL_TEXT",
    REPLACE_IMAGE: "GOOGLESUPER_REPLACE_IMAGE",
    REPLY_TO_THREAD: "GOOGLESUPER_REPLY_TO_THREAD",
    SEARCH_DEVELOPER_METADATA: "GOOGLESUPER_SEARCH_DEVELOPER_METADATA",
    SEARCH_DOCUMENTS: "GOOGLESUPER_SEARCH_DOCUMENTS",
    SEARCH_MEDIA_ITEMS: "GOOGLESUPER_SEARCH_MEDIA_ITEMS",
    SEARCH_PEOPLE: "GOOGLESUPER_SEARCH_PEOPLE",
    SEARCH_SPREADSHEETS: "GOOGLESUPER_SEARCH_SPREADSHEETS",
    SEND_DRAFT: "GOOGLESUPER_SEND_DRAFT",
    SEND_EMAIL: "GOOGLESUPER_SEND_EMAIL",
    SETTINGS_LIST: "GOOGLESUPER_SETTINGS_LIST",
    SETTINGS_WATCH: "GOOGLESUPER_SETTINGS_WATCH",
    SET_BASIC_FILTER: "GOOGLESUPER_SET_BASIC_FILTER",
    SHEET_FROM_JSON: "GOOGLESUPER_SHEET_FROM_JSON",
    SPREADSHEETS_SHEETS_COPY_TO: "GOOGLESUPER_SPREADSHEETS_SHEETS_COPY_TO",
    SPREADSHEETS_VALUES_APPEND: "GOOGLESUPER_SPREADSHEETS_VALUES_APPEND",
    SPREADSHEETS_VALUES_BATCH_CLEAR: "GOOGLESUPER_SPREADSHEETS_VALUES_BATCH_CLEAR",
    SPREADSHEETS_VALUES_BATCH_CLEAR_BY_DATA_FILTER: "GOOGLESUPER_SPREADSHEETS_VALUES_BATCH_CLEAR_BY_DATA_FILTER",
    SPREADSHEETS_VALUES_BATCH_GET_BY_DATA_FILTER: "GOOGLESUPER_SPREADSHEETS_VALUES_BATCH_GET_BY_DATA_FILTER",
    STOP_WATCH_CHANNEL: "GOOGLESUPER_STOP_WATCH_CHANNEL",
    SYNC_EVENTS: "GOOGLESUPER_SYNC_EVENTS",
    TEXT_SEARCH: "GOOGLESUPER_TEXT_SEARCH",
    UNHIDE_DRIVE: "GOOGLESUPER_UNHIDE_DRIVE",
    UNMERGE_TABLE_CELLS: "GOOGLESUPER_UNMERGE_TABLE_CELLS",
    UNTRASH_FILE: "GOOGLESUPER_UNTRASH_FILE",
    UPDATE_ACL_RULE: "GOOGLESUPER_UPDATE_ACL_RULE",
    UPDATE_ALBUM: "GOOGLESUPER_UPDATE_ALBUM",
    UPDATE_COMMENT: "GOOGLESUPER_UPDATE_COMMENT",
    UPDATE_DOCUMENT_MARKDOWN: "GOOGLESUPER_UPDATE_DOCUMENT_MARKDOWN",
    UPDATE_DOCUMENT_STYLE: "GOOGLESUPER_UPDATE_DOCUMENT_STYLE",
    UPDATE_DRIVE: "GOOGLESUPER_UPDATE_DRIVE",
    UPDATE_EVENT: "GOOGLESUPER_UPDATE_EVENT",
    UPDATE_EXISTING_DOCUMENT: "GOOGLESUPER_UPDATE_EXISTING_DOCUMENT",
    UPDATE_FILE_PUT: "GOOGLESUPER_UPDATE_FILE_PUT",
    UPDATE_FILE_REVISION_METADATA: "GOOGLESUPER_UPDATE_FILE_REVISION_METADATA",
    UPDATE_MEDIA_ITEM: "GOOGLESUPER_UPDATE_MEDIA_ITEM",
    UPDATE_PERMISSION: "GOOGLESUPER_UPDATE_PERMISSION",
    UPDATE_REPLY: "GOOGLESUPER_UPDATE_REPLY",
    UPDATE_SHEET_PROPERTIES: "GOOGLESUPER_UPDATE_SHEET_PROPERTIES",
    UPDATE_SPACE: "GOOGLESUPER_UPDATE_SPACE",
    UPDATE_SPREADSHEET_PROPERTIES: "GOOGLESUPER_UPDATE_SPREADSHEET_PROPERTIES",
    UPDATE_TABLE_ROW_STYLE: "GOOGLESUPER_UPDATE_TABLE_ROW_STYLE",
    UPDATE_TASK: "GOOGLESUPER_UPDATE_TASK",
    UPDATE_TASK_LIST: "GOOGLESUPER_UPDATE_TASK_LIST",
    UPLOAD_FILE: "GOOGLESUPER_UPLOAD_FILE",
    UPLOAD_MEDIA: "GOOGLESUPER_UPLOAD_MEDIA",
    WATCH_CHANGES: "GOOGLESUPER_WATCH_CHANGES",
  },
  triggerTypes: {
    GOOGLE_CALENDAR_EVENT_CHANGE_TRIGGER: "GOOGLESUPER_GOOGLE_CALENDAR_EVENT_CHANGE_TRIGGER",
    GOOGLE_DRIVE_CHANGES: "GOOGLESUPER_GOOGLE_DRIVE_CHANGES",
    NEW_GMAIL_MESSAGE: "GOOGLESUPER_NEW_GMAIL_MESSAGE",
  },
}
