// @composio/core/generated/googledrive.ts
/**
 * Map of Composio's GOOGLEDRIVE toolkit.
 */
export const GOOGLEDRIVE = {
  slug: "googledrive",
  tools: {
    ADD_FILE_SHARING_PREFERENCE: "GOOGLEDRIVE_ADD_FILE_SHARING_PREFERENCE",
    COPY_FILE: "GOOGLEDRIVE_COPY_FILE",
    CREATE_COMMENT: "GOOGLEDRIVE_CREATE_COMMENT",
    CREATE_DRIVE: "GOOGLEDRIVE_CREATE_DRIVE",
    CREATE_FILE: "GOOGLEDRIVE_CREATE_FILE",
    CREATE_FILE_FROM_TEXT: "GOOGLEDRIVE_CREATE_FILE_FROM_TEXT",
    CREATE_FOLDER: "GOOGLEDRIVE_CREATE_FOLDER",
    CREATE_REPLY: "GOOGLEDRIVE_CREATE_REPLY",
    CREATE_SHORTCUT_TO_FILE: "GOOGLEDRIVE_CREATE_SHORTCUT_TO_FILE",
    DELETE_COMMENT: "GOOGLEDRIVE_DELETE_COMMENT",
    DELETE_DRIVE: "GOOGLEDRIVE_DELETE_DRIVE",
    DELETE_PERMISSION: "GOOGLEDRIVE_DELETE_PERMISSION",
    DELETE_REPLY: "GOOGLEDRIVE_DELETE_REPLY",
    DOWNLOAD_FILE: "GOOGLEDRIVE_DOWNLOAD_FILE",
    EDIT_FILE: "GOOGLEDRIVE_EDIT_FILE",
    EMPTY_TRASH: "GOOGLEDRIVE_EMPTY_TRASH",
    FILES_MODIFY_LABELS: "GOOGLEDRIVE_FILES_MODIFY_LABELS",
    FIND_FILE: "GOOGLEDRIVE_FIND_FILE",
    FIND_FOLDER: "GOOGLEDRIVE_FIND_FOLDER",
    GENERATE_IDS: "GOOGLEDRIVE_GENERATE_IDS",
    GET_ABOUT: "GOOGLEDRIVE_GET_ABOUT",
    GET_CHANGES_START_PAGE_TOKEN: "GOOGLEDRIVE_GET_CHANGES_START_PAGE_TOKEN",
    GET_COMMENT: "GOOGLEDRIVE_GET_COMMENT",
    GET_DRIVE: "GOOGLEDRIVE_GET_DRIVE",
    GET_FILE_METADATA: "GOOGLEDRIVE_GET_FILE_METADATA",
    GET_PERMISSION: "GOOGLEDRIVE_GET_PERMISSION",
    GET_REPLY: "GOOGLEDRIVE_GET_REPLY",
    GET_REVISION: "GOOGLEDRIVE_GET_REVISION",
    GOOGLE_DRIVE_DELETE_FOLDER_OR_FILE_ACTION: "GOOGLEDRIVE_GOOGLE_DRIVE_DELETE_FOLDER_OR_FILE_ACTION",
    HIDE_DRIVE: "GOOGLEDRIVE_HIDE_DRIVE",
    LIST_CHANGES: "GOOGLEDRIVE_LIST_CHANGES",
    LIST_COMMENTS: "GOOGLEDRIVE_LIST_COMMENTS",
    LIST_FILES: "GOOGLEDRIVE_LIST_FILES",
    LIST_FILE_LABELS: "GOOGLEDRIVE_LIST_FILE_LABELS",
    LIST_PERMISSIONS: "GOOGLEDRIVE_LIST_PERMISSIONS",
    LIST_REPLIES_TO_COMMENT: "GOOGLEDRIVE_LIST_REPLIES_TO_COMMENT",
    LIST_REVISIONS: "GOOGLEDRIVE_LIST_REVISIONS",
    LIST_SHARED_DRIVES: "GOOGLEDRIVE_LIST_SHARED_DRIVES",
    MOVE_FILE: "GOOGLEDRIVE_MOVE_FILE",
    PARSE_FILE: "GOOGLEDRIVE_PARSE_FILE",
    STOP_WATCH_CHANNEL: "GOOGLEDRIVE_STOP_WATCH_CHANNEL",
    UNHIDE_DRIVE: "GOOGLEDRIVE_UNHIDE_DRIVE",
    UNTRASH_FILE: "GOOGLEDRIVE_UNTRASH_FILE",
    UPDATE_COMMENT: "GOOGLEDRIVE_UPDATE_COMMENT",
    UPDATE_DRIVE: "GOOGLEDRIVE_UPDATE_DRIVE",
    UPDATE_FILE_PUT: "GOOGLEDRIVE_UPDATE_FILE_PUT",
    UPDATE_FILE_REVISION_METADATA: "GOOGLEDRIVE_UPDATE_FILE_REVISION_METADATA",
    UPDATE_PERMISSION: "GOOGLEDRIVE_UPDATE_PERMISSION",
    UPDATE_REPLY: "GOOGLEDRIVE_UPDATE_REPLY",
    UPLOAD_FILE: "GOOGLEDRIVE_UPLOAD_FILE",
    WATCH_CHANGES: "GOOGLEDRIVE_WATCH_CHANGES",
  },
  triggerTypes: {
    GOOGLE_DRIVE_CHANGES: "GOOGLEDRIVE_GOOGLE_DRIVE_CHANGES",
  },
}
