// @composio/core/generated/googlecalendar.ts
/**
 * Map of Composio's GOOGLECALENDAR toolkit.
 */
export const GOOGLECALENDAR = {
  slug: "googlecalendar",
  tools: {
    CALENDARS_DELETE: "GOOGLECALENDAR_CALENDARS_DELETE",
    CALENDARS_UPDATE: "GOOGLECALENDAR_CALENDARS_UPDATE",
    CALENDAR_LIST_INSERT: "GOOGLECALENDAR_CALENDAR_LIST_INSERT",
    CALENDAR_LIST_UPDATE: "GOOGLECALENDAR_CALENDAR_LIST_UPDATE",
    CLEAR_CALENDAR: "GOOGLECALENDAR_CLEAR_CALENDAR",
    CREATE_EVENT: "GOOGLECALENDAR_CREATE_EVENT",
    DELETE_EVENT: "GOOGLECALENDAR_DELETE_EVENT",
    DUPLICATE_CALENDAR: "GOOGLECALENDAR_DUPLICATE_CALENDAR",
    EVENTS_INSTANCES: "GOOGLECALENDAR_EVENTS_INSTANCES",
    EVENTS_LIST: "GOOGLECALENDAR_EVENTS_LIST",
    EVENTS_MOVE: "GOOGLECALENDAR_EVENTS_MOVE",
    EVENTS_WATCH: "GOOGLECALENDAR_EVENTS_WATCH",
    FIND_EVENT: "GOOGLECALENDAR_FIND_EVENT",
    FIND_FREE_SLOTS: "GOOGLECALENDAR_FIND_FREE_SLOTS",
    FREE_BUSY_QUERY: "GOOGLECALENDAR_FREE_BUSY_QUERY",
    GET_CALENDAR: "GOOGLECALENDAR_GET_CALENDAR",
    GET_CURRENT_DATE_TIME: "GOOGLECALENDAR_GET_CURRENT_DATE_TIME",
    LIST_ACL_RULES: "GOOGLECALENDAR_LIST_ACL_RULES",
    LIST_CALENDARS: "GOOGLECALENDAR_LIST_CALENDARS",
    PATCH_CALENDAR: "GOOGLECALENDAR_PATCH_CALENDAR",
    PATCH_EVENT: "GOOGLECALENDAR_PATCH_EVENT",
    QUICK_ADD: "GOOGLECALENDAR_QUICK_ADD",
    REMOVE_ATTENDEE: "GOOGLECALENDAR_REMOVE_ATTENDEE",
    SETTINGS_LIST: "GOOGLECALENDAR_SETTINGS_LIST",
    SETTINGS_WATCH: "GOOGLECALENDAR_SETTINGS_WATCH",
    SYNC_EVENTS: "GOOGLECALENDAR_SYNC_EVENTS",
    UPDATE_ACL_RULE: "GOOGLECALENDAR_UPDATE_ACL_RULE",
    UPDATE_EVENT: "GOOGLECALENDAR_UPDATE_EVENT",
  },
  triggerTypes: {
    GOOGLE_CALENDAR_EVENT_CHANGE_TRIGGER: "GOOGLECALENDAR_GOOGLE_CALENDAR_EVENT_CHANGE_TRIGGER",
  },
}
