// @composio/core/generated/gorgias.ts
/**
 * Map of Composio's GORGIAS toolkit.
 */
export const GORGIAS = {
  slug: "gorgias",
  tools: {
    ADD_TICKET_TAGS: "GORGIAS_ADD_TICKET_TAGS",
    CREATE_ACCOUNT_SETTING: "GORGIAS_CREATE_ACCOUNT_SETTING",
    CREATE_CUSTOMER: "GORGIAS_CREATE_CUSTOMER",
    CREATE_TEAM: "GORGIAS_CREATE_TEAM",
    CREATE_TICKET: "GORGIAS_CREATE_TICKET",
    DELETE_CUSTOMER: "GORGIAS_DELETE_CUSTOMER",
    DELETE_CUSTOMERS: "GORGIAS_DELETE_CUSTOMERS",
    DELETE_CUSTOMER_FIELD_VALUE: "GORGIAS_DELETE_CUSTOMER_FIELD_VALUE",
    DELETE_TEAM: "GORGIAS_DELETE_TEAM",
    DELETE_TICKET: "GORGIAS_DELETE_TICKET",
    DELETE_TICKET_FIELD_VALUE: "GORGIAS_DELETE_TICKET_FIELD_VALUE",
    GET_ACCOUNT: "GORGIAS_GET_ACCOUNT",
    GET_CUSTOMER: "GORGIAS_GET_CUSTOMER",
    GET_EVENT: "GORGIAS_GET_EVENT",
    GET_TEAM: "GORGIAS_GET_TEAM",
    GET_TICKET: "GORGIAS_GET_TICKET",
    LIST_ACCOUNT_SETTINGS: "GORGIAS_LIST_ACCOUNT_SETTINGS",
    LIST_CUSTOMERS: "GORGIAS_LIST_CUSTOMERS",
    LIST_CUSTOMER_FIELD_VALUES: "GORGIAS_LIST_CUSTOMER_FIELD_VALUES",
    LIST_EVENTS: "GORGIAS_LIST_EVENTS",
    LIST_TEAMS: "GORGIAS_LIST_TEAMS",
    LIST_TICKETS: "GORGIAS_LIST_TICKETS",
    LIST_TICKET_FIELD_VALUES: "GORGIAS_LIST_TICKET_FIELD_VALUES",
    LIST_TICKET_TAGS: "GORGIAS_LIST_TICKET_TAGS",
    MERGE_CUSTOMERS: "GORGIAS_MERGE_CUSTOMERS",
    REMOVE_TICKET_TAGS: "GORGIAS_REMOVE_TICKET_TAGS",
    SET_CUSTOMER_DATA: "GORGIAS_SET_CUSTOMER_DATA",
    SET_TICKET_TAGS: "GORGIAS_SET_TICKET_TAGS",
    UPDATE_ACCOUNT_SETTING: "GORGIAS_UPDATE_ACCOUNT_SETTING",
    UPDATE_CUSTOMER: "GORGIAS_UPDATE_CUSTOMER",
    UPDATE_TEAM: "GORGIAS_UPDATE_TEAM",
    UPDATE_TICKET: "GORGIAS_UPDATE_TICKET",
  },
  triggerTypes: {},
}
