// @composio/core/generated/harvest.ts
/**
 * Map of Composio's HARVEST toolkit.
 */
export const HARVEST = {
  slug: "harvest",
  tools: {
    CREATE_CLIENT: "HARVEST_CREATE_CLIENT",
    CREATE_CLIENT_CONTACT: "HARVEST_CREATE_CLIENT_CONTACT",
    CREATE_ESTIMATE: "HARVEST_CREATE_ESTIMATE",
    CREATE_ESTIMATE_ITEM_CATEGORY: "HARVEST_CREATE_ESTIMATE_ITEM_CATEGORY",
    CREATE_ESTIMATE_MESSAGE: "HARVEST_CREATE_ESTIMATE_MESSAGE",
    CREATE_INVOICE: "HARVEST_CREATE_INVOICE",
    CREATE_INVOICE_ITEM_CATEGORY: "HARVEST_CREATE_INVOICE_ITEM_CATEGORY",
    CREATE_INVOICE_MESSAGE: "HARVEST_CREATE_INVOICE_MESSAGE",
    CREATE_PROJECT: "HARVEST_CREATE_PROJECT",
    CREATE_TASK: "HARVEST_CREATE_TASK",
    CREATE_TIME_ENTRY: "HARVEST_CREATE_TIME_ENTRY",
    CREATE_USER: "HARVEST_CREATE_USER",
    DELETE_CLIENT: "HARVEST_DELETE_CLIENT",
    DELETE_CLIENT_CONTACT: "HARVEST_DELETE_CLIENT_CONTACT",
    DELETE_ESTIMATE: "HARVEST_DELETE_ESTIMATE",
    DELETE_ESTIMATE_ITEM_CATEGORY: "HARVEST_DELETE_ESTIMATE_ITEM_CATEGORY",
    DELETE_ESTIMATE_MESSAGE: "HARVEST_DELETE_ESTIMATE_MESSAGE",
    DELETE_INVOICE: "HARVEST_DELETE_INVOICE",
    DELETE_INVOICE_ITEM_CATEGORY: "HARVEST_DELETE_INVOICE_ITEM_CATEGORY",
    DELETE_INVOICE_MESSAGE: "HARVEST_DELETE_INVOICE_MESSAGE",
    DELETE_PROJECT: "HARVEST_DELETE_PROJECT",
    DELETE_TASK: "HARVEST_DELETE_TASK",
    DELETE_TIME_ENTRY: "HARVEST_DELETE_TIME_ENTRY",
    DELETE_USER: "HARVEST_DELETE_USER",
    GET_COMPANY_INFO: "HARVEST_GET_COMPANY_INFO",
    LIST_CLIENTS: "HARVEST_LIST_CLIENTS",
    LIST_CLIENT_CONTACTS: "HARVEST_LIST_CLIENT_CONTACTS",
    LIST_ESTIMATE_MESSAGES: "HARVEST_LIST_ESTIMATE_MESSAGES",
    LIST_INVOICES: "HARVEST_LIST_INVOICES",
    LIST_INVOICE_ITEM_CATEGORIES: "HARVEST_LIST_INVOICE_ITEM_CATEGORIES",
    LIST_INVOICE_MESSAGES: "HARVEST_LIST_INVOICE_MESSAGES",
    LIST_INVOICE_PAYMENTS: "HARVEST_LIST_INVOICE_PAYMENTS",
    LIST_PROJECTS: "HARVEST_LIST_PROJECTS",
    LIST_TASKS: "HARVEST_LIST_TASKS",
    LIST_TIME_ENTRIES: "HARVEST_LIST_TIME_ENTRIES",
    LIST_USERS: "HARVEST_LIST_USERS",
    UPDATE_CLIENT: "HARVEST_UPDATE_CLIENT",
    UPDATE_CLIENT_CONTACT: "HARVEST_UPDATE_CLIENT_CONTACT",
    UPDATE_COMPANY_INFO: "HARVEST_UPDATE_COMPANY_INFO",
    UPDATE_ESTIMATE: "HARVEST_UPDATE_ESTIMATE",
    UPDATE_ESTIMATE_ITEM_CATEGORY: "HARVEST_UPDATE_ESTIMATE_ITEM_CATEGORY",
    UPDATE_INVOICE: "HARVEST_UPDATE_INVOICE",
    UPDATE_INVOICE_ITEM_CATEGORY: "HARVEST_UPDATE_INVOICE_ITEM_CATEGORY",
    UPDATE_PROJECT: "HARVEST_UPDATE_PROJECT",
    UPDATE_TASK: "HARVEST_UPDATE_TASK",
    UPDATE_TIME_ENTRY: "HARVEST_UPDATE_TIME_ENTRY",
    UPDATE_USER: "HARVEST_UPDATE_USER",
  },
  triggerTypes: {},
}
