// @composio/core/generated/heygen.ts
/**
 * Map of Composio's HEYGEN toolkit.
 */
export const HEYGEN = {
  slug: "heygen",
  tools: {
    ADD_NEW_ASSET: "HEYGEN_ADD_NEW_ASSET",
    PERSONALIZED_VIDEO_ADD_CONTACT: "HEYGEN_PERSONALIZED_VIDEO_ADD_CONTACT",
    PERSONALIZED_VIDEO_AUDIENCE_DETAIL: "HEYGEN_PERSONALIZED_VIDEO_AUDIENCE_DETAIL",
    PERSONALIZED_VIDEO_PROJECT_DETAIL: "HEYGEN_PERSONALIZED_VIDEO_PROJECT_DETAIL",
    POST_TALKING_PHOTO_BINARY_IMAGE: "HEYGEN_POST_TALKING_PHOTO_BINARY_IMAGE",
    STREAMING_AVATAR_LIST: "HEYGEN_STREAMING_AVATAR_LIST",
    STREAMING_CREATE_TOKEN: "HEYGEN_STREAMING_CREATE_TOKEN",
    STREAMING_ICE: "HEYGEN_STREAMING_ICE",
    STREAMING_INTERRUPT: "HEYGEN_STREAMING_INTERRUPT",
    STREAMING_LIST: "HEYGEN_STREAMING_LIST",
    STREAMING_NEW: "HEYGEN_STREAMING_NEW",
    STREAMING_START: "HEYGEN_STREAMING_START",
    STREAMING_STOP: "HEYGEN_STREAMING_STOP",
    STREAMING_TASK: "HEYGEN_STREAMING_TASK",
    V1_AVATAR_LIST: "HEYGEN_V1_AVATAR_LIST",
    V1_TALKING_PHOTO_LIST: "HEYGEN_V1_TALKING_PHOTO_LIST",
    V1_VIDEO_DELETE: "HEYGEN_V1_VIDEO_DELETE",
    V1_VIDEO_LIST: "HEYGEN_V1_VIDEO_LIST",
    V1_VIDEO_STATUS_GET_LATEST_ID: "HEYGEN_V1_VIDEO_STATUS_GET_LATEST_ID",
    V1_VOICE_LIST: "HEYGEN_V1_VOICE_LIST",
    V1_WEBHOOK_ENDPOINT_ADD: "HEYGEN_V1_WEBHOOK_ENDPOINT_ADD",
    V1_WEBHOOK_ENDPOINT_DELETE: "HEYGEN_V1_WEBHOOK_ENDPOINT_DELETE",
    V1_WEBHOOK_ENDPOINT_LIST: "HEYGEN_V1_WEBHOOK_ENDPOINT_LIST",
    V1_WEBHOOK_WEBHOOK_LIST: "HEYGEN_V1_WEBHOOK_WEBHOOK_LIST",
    V2_AVATARS: "HEYGEN_V2_AVATARS",
    V2_TALKING_PHOTO_ID_DELETE: "HEYGEN_V2_TALKING_PHOTO_ID_DELETE",
    V2_TEMPLATES: "HEYGEN_V2_TEMPLATES",
    V2_TEMPLATE_GENERATE: "HEYGEN_V2_TEMPLATE_GENERATE",
    V2_TEMPLATE_ID: "HEYGEN_V2_TEMPLATE_ID",
    V2_USER_REMAINING_QUOTA: "HEYGEN_V2_USER_REMAINING_QUOTA",
    V2_VIDEO_GENERATE: "HEYGEN_V2_VIDEO_GENERATE",
    V2_VIDEO_TRANSLATE: "HEYGEN_V2_VIDEO_TRANSLATE",
    V2_VIDEO_TRANSLATE_ID_STATUS: "HEYGEN_V2_VIDEO_TRANSLATE_ID_STATUS",
    V2_VIDEO_TRANSLATE_TARGET_LANGUAGES: "HEYGEN_V2_VIDEO_TRANSLATE_TARGET_LANGUAGES",
    V2_VOICES: "HEYGEN_V2_VOICES",
  },
  triggerTypes: {},
}
