// @composio/core/generated/hubspot.ts
/**
 * Map of Composio's HUBSPOT toolkit.
 */
export const HUBSPOT = {
  slug: "hubspot",
  tools: {
    ADD_ASSET_ASSOCIATION: "HUBSPOT_ADD_ASSET_ASSOCIATION",
    ADD_TOKEN_TO_EVENT_TEMPLATE: "HUBSPOT_ADD_TOKEN_TO_EVENT_TEMPLATE",
    ARCHIVE: "HUBSPOT_ARCHIVE",
    ARCHIVE_A_BATCH_OF_EMAILS_BY_ID: "HUBSPOT_ARCHIVE_A_BATCH_OF_EMAILS_BY_ID",
    ARCHIVE_A_BATCH_OF_QUOTES_BY_ID: "HUBSPOT_ARCHIVE_A_BATCH_OF_QUOTES_BY_ID",
    ARCHIVE_BATCH_OF_COMPANIES_BY_ID: "HUBSPOT_ARCHIVE_BATCH_OF_COMPANIES_BY_ID",
    ARCHIVE_BATCH_OF_CONTACTS_BY_ID: "HUBSPOT_ARCHIVE_BATCH_OF_CONTACTS_BY_ID",
    ARCHIVE_BATCH_OF_DEALS_BY_ID: "HUBSPOT_ARCHIVE_BATCH_OF_DEALS_BY_ID",
    ARCHIVE_BATCH_OF_FEEDBACK_SUBMISSIONS_BY_ID: "HUBSPOT_ARCHIVE_BATCH_OF_FEEDBACK_SUBMISSIONS_BY_ID",
    ARCHIVE_BATCH_OF_LINE_ITEMS_BY_ID: "HUBSPOT_ARCHIVE_BATCH_OF_LINE_ITEMS_BY_ID",
    ARCHIVE_BATCH_OF_OBJECTS_BY_ID: "HUBSPOT_ARCHIVE_BATCH_OF_OBJECTS_BY_ID",
    ARCHIVE_BATCH_OF_PROPERTIES: "HUBSPOT_ARCHIVE_BATCH_OF_PROPERTIES",
    ARCHIVE_BATCH_OF_TICKETS_BY_ID: "HUBSPOT_ARCHIVE_BATCH_OF_TICKETS_BY_ID",
    ARCHIVE_BATCH_PRODUCTS_BY_ID: "HUBSPOT_ARCHIVE_BATCH_PRODUCTS_BY_ID",
    ARCHIVE_COMPANY: "HUBSPOT_ARCHIVE_COMPANY",
    ARCHIVE_CONTACT_BY_ID: "HUBSPOT_ARCHIVE_CONTACT_BY_ID",
    ARCHIVE_CRM_OBJECT_BY_ID: "HUBSPOT_ARCHIVE_CRM_OBJECT_BY_ID",
    ARCHIVE_FEEDBACK_SUBMISSION: "HUBSPOT_ARCHIVE_FEEDBACK_SUBMISSION",
    ARCHIVE_LINE_ITEM_BY_ID: "HUBSPOT_ARCHIVE_LINE_ITEM_BY_ID",
    ARCHIVE_PRODUCT_BY_PRODUCT_ID: "HUBSPOT_ARCHIVE_PRODUCT_BY_PRODUCT_ID",
    ARCHIVE_PROPERTY_BY_OBJECT_TYPE_AND_NAME: "HUBSPOT_ARCHIVE_PROPERTY_BY_OBJECT_TYPE_AND_NAME",
    ARCHIVE_PROPERTY_GROUP: "HUBSPOT_ARCHIVE_PROPERTY_GROUP",
    ARCHIVE_QUOTE_OBJECT_BY_ID: "HUBSPOT_ARCHIVE_QUOTE_OBJECT_BY_ID",
    ARCHIVE_TICKET_BY_ID: "HUBSPOT_ARCHIVE_TICKET_BY_ID",
    AUDIT_PIPELINE_CHANGES_BY_ID: "HUBSPOT_AUDIT_PIPELINE_CHANGES_BY_ID",
    BATCH_READ_COMPANIES_BY_PROPERTIES: "HUBSPOT_BATCH_READ_COMPANIES_BY_PROPERTIES",
    BATCH_UPDATE_QUOTES: "HUBSPOT_BATCH_UPDATE_QUOTES",
    CAMPAIGN_SEARCH: "HUBSPOT_CAMPAIGN_SEARCH",
    CANCEL_ACTIVE_IMPORT: "HUBSPOT_CANCEL_ACTIVE_IMPORT",
    CLONE_MARKETING_EMAIL: "HUBSPOT_CLONE_MARKETING_EMAIL",
    CONFIGURE_CALLING_EXTENSION_SETTINGS: "HUBSPOT_CONFIGURE_CALLING_EXTENSION_SETTINGS",
    CREATE: "HUBSPOT_CREATE",
    CREATE_AB_TEST_VARIATION: "HUBSPOT_CREATE_AB_TEST_VARIATION",
    CREATE_AND_RETURN_A_NEW_PROPERTY_GROUP: "HUBSPOT_CREATE_AND_RETURN_A_NEW_PROPERTY_GROUP",
    CREATE_ASSOCIATION_FOR_OBJECT_TYPE: "HUBSPOT_CREATE_ASSOCIATION_FOR_OBJECT_TYPE",
    CREATE_A_BATCH_OF_CAMPAIGNS: "HUBSPOT_CREATE_A_BATCH_OF_CAMPAIGNS",
    CREATE_A_BATCH_OF_COMPANIES: "HUBSPOT_CREATE_A_BATCH_OF_COMPANIES",
    CREATE_A_BATCH_OF_CONTACTS: "HUBSPOT_CREATE_A_BATCH_OF_CONTACTS",
    CREATE_A_BATCH_OF_EMAILS: "HUBSPOT_CREATE_A_BATCH_OF_EMAILS",
    CREATE_A_CAMPAIGN: "HUBSPOT_CREATE_A_CAMPAIGN",
    CREATE_A_NEW_MARKETING_EMAIL: "HUBSPOT_CREATE_A_NEW_MARKETING_EMAIL",
    CREATE_BATCH_OF_CONTACTS: "HUBSPOT_CREATE_BATCH_OF_CONTACTS",
    CREATE_BATCH_OF_DEALS: "HUBSPOT_CREATE_BATCH_OF_DEALS",
    CREATE_BATCH_OF_FEEDBACK_SUBMISSIONS: "HUBSPOT_CREATE_BATCH_OF_FEEDBACK_SUBMISSIONS",
    CREATE_BATCH_OF_LINE_ITEMS: "HUBSPOT_CREATE_BATCH_OF_LINE_ITEMS",
    CREATE_BATCH_OF_OBJECTS: "HUBSPOT_CREATE_BATCH_OF_OBJECTS",
    CREATE_BATCH_OF_PROPERTIES: "HUBSPOT_CREATE_BATCH_OF_PROPERTIES",
    CREATE_BATCH_OF_QUOTES: "HUBSPOT_CREATE_BATCH_OF_QUOTES",
    CREATE_BATCH_OF_TICKET: "HUBSPOT_CREATE_BATCH_OF_TICKET",
    CREATE_COMPANY_OBJECT: "HUBSPOT_CREATE_COMPANY_OBJECT",
    CREATE_CONTACT_OBJECT_WITH_PROPERTIES: "HUBSPOT_CREATE_CONTACT_OBJECT_WITH_PROPERTIES",
    CREATE_CRM_OBJECT_WITH_PROPERTIES: "HUBSPOT_CREATE_CRM_OBJECT_WITH_PROPERTIES",
    CREATE_EVENT_TEMPLATE_FOR_APP: "HUBSPOT_CREATE_EVENT_TEMPLATE_FOR_APP",
    CREATE_FEEDBACK_SUBMISSION: "HUBSPOT_CREATE_FEEDBACK_SUBMISSION",
    CREATE_LINE_ITEM_OBJECT: "HUBSPOT_CREATE_LINE_ITEM_OBJECT",
    CREATE_MULTIPLE_TIMELINE_EVENTS_BATCH: "HUBSPOT_CREATE_MULTIPLE_TIMELINE_EVENTS_BATCH",
    CREATE_NEW_DEAL_OBJECT: "HUBSPOT_CREATE_NEW_DEAL_OBJECT",
    CREATE_NEW_OBJECT_SCHEMA_WITH_CUSTOM_PROPERTIES: "HUBSPOT_CREATE_NEW_OBJECT_SCHEMA_WITH_CUSTOM_PROPERTIES",
    CREATE_OR_UPDATE_DRAFT_VERSION: "HUBSPOT_CREATE_OR_UPDATE_DRAFT_VERSION",
    CREATE_PIPELINE_FOR_OBJECT_TYPE: "HUBSPOT_CREATE_PIPELINE_FOR_OBJECT_TYPE",
    CREATE_PIPELINE_STAGE: "HUBSPOT_CREATE_PIPELINE_STAGE",
    CREATE_PRODUCT_BATCH: "HUBSPOT_CREATE_PRODUCT_BATCH",
    CREATE_PRODUCT_OBJECT: "HUBSPOT_CREATE_PRODUCT_OBJECT",
    CREATE_PROPERTY_FOR_SPECIFIED_OBJECT_TYPE: "HUBSPOT_CREATE_PROPERTY_FOR_SPECIFIED_OBJECT_TYPE",
    CREATE_QUOTE_OBJECT: "HUBSPOT_CREATE_QUOTE_OBJECT",
    CREATE_TICKET_OBJECT: "HUBSPOT_CREATE_TICKET_OBJECT",
    CREATE_TIMELINE_EVENT_BASED_ON_TEMPLATE: "HUBSPOT_CREATE_TIMELINE_EVENT_BASED_ON_TEMPLATE",
    CREATE_WORKFLOW: "HUBSPOT_CREATE_WORKFLOW",
    CUSTOMIZABLE_CONTACTS_PAGE_RETRIEVAL: "HUBSPOT_CUSTOMIZABLE_CONTACTS_PAGE_RETRIEVAL",
    DELETE_A_BATCH_OF_CAMPAIGNS: "HUBSPOT_DELETE_A_BATCH_OF_CAMPAIGNS",
    DELETE_A_MARKETING_EMAIL: "HUBSPOT_DELETE_A_MARKETING_EMAIL",
    DELETE_CALLING_EXTENSION_SETTINGS: "HUBSPOT_DELETE_CALLING_EXTENSION_SETTINGS",
    DELETE_CAMPAIGN: "HUBSPOT_DELETE_CAMPAIGN",
    DELETE_CONTACT_FOR_GDPR_COMPLIANCE: "HUBSPOT_DELETE_CONTACT_FOR_GDPR_COMPLIANCE",
    DELETE_PIPELINE_BY_ID: "HUBSPOT_DELETE_PIPELINE_BY_ID",
    DELETE_PIPELINE_STAGE_BY_ID: "HUBSPOT_DELETE_PIPELINE_STAGE_BY_ID",
    DELETE_SCHEMA_BY_OBJECT_TYPE: "HUBSPOT_DELETE_SCHEMA_BY_OBJECT_TYPE",
    DELETE_TIMELINE_EVENT_TEMPLATE: "HUBSPOT_DELETE_TIMELINE_EVENT_TEMPLATE",
    DELETE_VIDEO_CONFERENCING_APP_SETTINGS: "HUBSPOT_DELETE_VIDEO_CONFERENCING_APP_SETTINGS",
    DELETE_WORKFLOW: "HUBSPOT_DELETE_WORKFLOW",
    FETCH_CONTACT_DETAILS_BY_ID: "HUBSPOT_FETCH_CONTACT_DETAILS_BY_ID",
    FETCH_CONTACT_IDS: "HUBSPOT_FETCH_CONTACT_IDS",
    FETCH_IMPORT_ERROR_DETAILS: "HUBSPOT_FETCH_IMPORT_ERROR_DETAILS",
    FETCH_OBJECT_DETAILS_BY_ID_OR_PROPERTY: "HUBSPOT_FETCH_OBJECT_DETAILS_BY_ID_OR_PROPERTY",
    FETCH_RECORDING_SETTINGS_BY_APP_ID: "HUBSPOT_FETCH_RECORDING_SETTINGS_BY_APP_ID",
    FETCH_REVENUE: "HUBSPOT_FETCH_REVENUE",
    GDPR_DELETE: "HUBSPOT_GDPR_DELETE",
    GET_ACTIVE_IMPORTS_LIST: "HUBSPOT_GET_ACTIVE_IMPORTS_LIST",
    GET_AGGREGATED_STATISTICS: "HUBSPOT_GET_AGGREGATED_STATISTICS",
    GET_AGGREGATED_STATISTIC_INTERVALS: "HUBSPOT_GET_AGGREGATED_STATISTIC_INTERVALS",
    GET_ALL_MARKETING_EMAILS_FOR_A_HUB_SPOT_ACCOUNT: "HUBSPOT_GET_ALL_MARKETING_EMAILS_FOR_A_HUB_SPOT_ACCOUNT",
    GET_ALL_WORKFLOWS: "HUBSPOT_GET_ALL_WORKFLOWS",
    GET_A_PAGE_OF_OWNERS: "HUBSPOT_GET_A_PAGE_OF_OWNERS",
    GET_A_REVISION_OF_A_MARKETING_EMAIL: "HUBSPOT_GET_A_REVISION_OF_A_MARKETING_EMAIL",
    GET_CAMPAIGN_METRICS: "HUBSPOT_GET_CAMPAIGN_METRICS",
    GET_DRAFT_VERSION_OF_A_MARKETING_EMAIL: "HUBSPOT_GET_DRAFT_VERSION_OF_A_MARKETING_EMAIL",
    GET_IMPORT_RECORD_INFORMATION: "HUBSPOT_GET_IMPORT_RECORD_INFORMATION",
    GET_MARKETING_EMAILS: "HUBSPOT_GET_MARKETING_EMAILS",
    GET_PIPELINE_STAGE_AUDIT: "HUBSPOT_GET_PIPELINE_STAGE_AUDIT",
    GET_QUOTE_BY_ID: "HUBSPOT_GET_QUOTE_BY_ID",
    GET_REVISIONS_OF_A_MARKETING_EMAIL: "HUBSPOT_GET_REVISIONS_OF_A_MARKETING_EMAIL",
    GET_SPECIFIC_EVENT_TEMPLATE_FOR_APP: "HUBSPOT_GET_SPECIFIC_EVENT_TEMPLATE_FOR_APP",
    GET_THE_DETAILS_OF_A_SPECIFIED_MARKETING_EMAIL: "HUBSPOT_GET_THE_DETAILS_OF_A_SPECIFIED_MARKETING_EMAIL",
    GET_THE_VARIATION_OF_A_AN_A_B_MARKETING_EMAIL: "HUBSPOT_GET_THE_VARIATION_OF_A_AN_A_B_MARKETING_EMAIL",
    GET_WORKFLOW_BY_ID: "HUBSPOT_GET_WORKFLOW_BY_ID",
    GET_WORKFLOW_EMAIL_CAMPAIGNS: "HUBSPOT_GET_WORKFLOW_EMAIL_CAMPAIGNS",
    INITIATE_DATA_IMPORT_PROCESS: "HUBSPOT_INITIATE_DATA_IMPORT_PROCESS",
    INSTANTIATE_AND_RETURN_NEW_COMPANY_OBJECT: "HUBSPOT_INSTANTIATE_AND_RETURN_NEW_COMPANY_OBJECT",
    LIST: "HUBSPOT_LIST",
    LIST_ALL_EVENT_TEMPLATES_FOR_APP: "HUBSPOT_LIST_ALL_EVENT_TEMPLATES_FOR_APP",
    LIST_ASSETS: "HUBSPOT_LIST_ASSETS",
    LIST_ASSOCIATION_TYPES: "HUBSPOT_LIST_ASSOCIATION_TYPES",
    LIST_CONTACTS_PAGE: "HUBSPOT_LIST_CONTACTS_PAGE",
    LIST_FEEDBACK_SUBMISSIONS_PAGE: "HUBSPOT_LIST_FEEDBACK_SUBMISSIONS_PAGE",
    LIST_PAGE_OF_TICKETS: "HUBSPOT_LIST_PAGE_OF_TICKETS",
    LIST_PRODUCTS_WITH_PAGING: "HUBSPOT_LIST_PRODUCTS_WITH_PAGING",
    LIST_QUOTES_PAGE: "HUBSPOT_LIST_QUOTES_PAGE",
    MERGE_TWO_COMPANIES_OF_SAME_TYPE: "HUBSPOT_MERGE_TWO_COMPANIES_OF_SAME_TYPE",
    MERGE_TWO_CONTACTS_OF_SAME_TYPE: "HUBSPOT_MERGE_TWO_CONTACTS_OF_SAME_TYPE",
    MERGE_TWO_DEALS_OF_SAME_TYPE: "HUBSPOT_MERGE_TWO_DEALS_OF_SAME_TYPE",
    MERGE_TWO_EMAILS_WITH_SAME_TYPE: "HUBSPOT_MERGE_TWO_EMAILS_WITH_SAME_TYPE",
    MERGE_TWO_FEEDBACK_SUBMISSIONS: "HUBSPOT_MERGE_TWO_FEEDBACK_SUBMISSIONS",
    MERGE_TWO_LINE_ITEMS_OF_SAME_TYPE: "HUBSPOT_MERGE_TWO_LINE_ITEMS_OF_SAME_TYPE",
    MERGE_TWO_OBJECTS_OF_SAME_TYPE: "HUBSPOT_MERGE_TWO_OBJECTS_OF_SAME_TYPE",
    MERGE_TWO_PRODUCTS_OF_THE_SAME_TYPE: "HUBSPOT_MERGE_TWO_PRODUCTS_OF_THE_SAME_TYPE",
    MERGE_TWO_QUOTES_OF_SAME_TYPE: "HUBSPOT_MERGE_TWO_QUOTES_OF_SAME_TYPE",
    MERGE_TWO_TICKETS_OF_SAME_TYPE: "HUBSPOT_MERGE_TWO_TICKETS_OF_SAME_TYPE",
    MODIFY_CALLING_EXTENSION_SETTINGS: "HUBSPOT_MODIFY_CALLING_EXTENSION_SETTINGS",
    PARTIALLY_UPDATE_CONTACT_USING_CONTACT_ID: "HUBSPOT_PARTIALLY_UPDATE_CONTACT_USING_CONTACT_ID",
    PARTIALLY_UPDATE_CRM_OBJECT_BY_ID: "HUBSPOT_PARTIALLY_UPDATE_CRM_OBJECT_BY_ID",
    PARTIALLY_UPDATE_DEAL_BY_ID: "HUBSPOT_PARTIALLY_UPDATE_DEAL_BY_ID",
    PARTIALLY_UPDATE_PIPELINE_BY_ID: "HUBSPOT_PARTIALLY_UPDATE_PIPELINE_BY_ID",
    PARTIALLY_UPDATE_PROPERTY_GROUP: "HUBSPOT_PARTIALLY_UPDATE_PROPERTY_GROUP",
    PARTIALLY_UPDATE_TICKET_BY_ID: "HUBSPOT_PARTIALLY_UPDATE_TICKET_BY_ID",
    PARTIAL_UPDATE_COMPANY_OBJECT_BY_ID: "HUBSPOT_PARTIAL_UPDATE_COMPANY_OBJECT_BY_ID",
    PARTIAL_UPDATE_QUOTE_BY_QUOTE_ID: "HUBSPOT_PARTIAL_UPDATE_QUOTE_BY_QUOTE_ID",
    PERMANENTLY_DELETE_COMPANY_FOR_GDPR_COMPLIANCE: "HUBSPOT_PERMANENTLY_DELETE_COMPANY_FOR_GDPR_COMPLIANCE",
    PERMANENTLY_DELETE_CONTACT_FOR_GDPR: "HUBSPOT_PERMANENTLY_DELETE_CONTACT_FOR_GDPR",
    PERMANENTLY_DELETE_CONTACT_FOR_GDPR_COMPLIANCE: "HUBSPOT_PERMANENTLY_DELETE_CONTACT_FOR_GDPR_COMPLIANCE",
    PERMANENTLY_DELETE_CONTACT_GDPR_COMPLIANCE: "HUBSPOT_PERMANENTLY_DELETE_CONTACT_GDPR_COMPLIANCE",
    PERMANENTLY_DELETE_CONTACT_VIA_GDPR: "HUBSPOT_PERMANENTLY_DELETE_CONTACT_VIA_GDPR",
    PERMANENTLY_DELETE_DEAL_FOR_GDPR_COMPLIANCE: "HUBSPOT_PERMANENTLY_DELETE_DEAL_FOR_GDPR_COMPLIANCE",
    PERMANENTLY_DELETE_GDPR_CONTACT_AND_CONTENT: "HUBSPOT_PERMANENTLY_DELETE_GDPR_CONTACT_AND_CONTENT",
    PERMANENTLY_DELETE_LINE_ITEMS_FOR_GDPR: "HUBSPOT_PERMANENTLY_DELETE_LINE_ITEMS_FOR_GDPR",
    PUBLISH_MARKETING_EMAIL: "HUBSPOT_PUBLISH_MARKETING_EMAIL",
    PURGE_SCHEMA_BY_OBJECT_TYPE: "HUBSPOT_PURGE_SCHEMA_BY_OBJECT_TYPE",
    READ: "HUBSPOT_READ",
    READ_ALL_PROPERTIES: "HUBSPOT_READ_ALL_PROPERTIES",
    READ_ALL_PROPERTIES_FOR_OBJECT_TYPE: "HUBSPOT_READ_ALL_PROPERTIES_FOR_OBJECT_TYPE",
    READ_ALL_PROPERTY_GROUPS: "HUBSPOT_READ_ALL_PROPERTY_GROUPS",
    READ_AN_OWNER_BY_GIVEN_ID_OR_USER_ID: "HUBSPOT_READ_AN_OWNER_BY_GIVEN_ID_OR_USER_ID",
    READ_APAGE_OF_OBJECTS_BY_TYPE: "HUBSPOT_READ_APAGE_OF_OBJECTS_BY_TYPE",
    READ_A_BATCH_OF_CAMPAIGNS: "HUBSPOT_READ_A_BATCH_OF_CAMPAIGNS",
    READ_A_BATCH_OF_CRM_OBJECT_PROPERTIES: "HUBSPOT_READ_A_BATCH_OF_CRM_OBJECT_PROPERTIES",
    READ_A_CAMPAIGN: "HUBSPOT_READ_A_CAMPAIGN",
    READ_A_CRM_PROPERTY_BY_NAME: "HUBSPOT_READ_A_CRM_PROPERTY_BY_NAME",
    READ_A_PAGE_OF_DEALS: "HUBSPOT_READ_A_PAGE_OF_DEALS",
    READ_A_PROPERTY: "HUBSPOT_READ_A_PROPERTY",
    READ_A_PROPERTY_GROUP: "HUBSPOT_READ_A_PROPERTY_GROUP",
    READ_BATCH_EMAILS: "HUBSPOT_READ_BATCH_EMAILS",
    READ_BATCH_FEEDBACK_SUBMISSIONS_BY_ID_OR_PROPERTY: "HUBSPOT_READ_BATCH_FEEDBACK_SUBMISSIONS_BY_ID_OR_PROPERTY",
    READ_BATCH_OF_CONTACTS_BY_ID_OR_PROPERTIES: "HUBSPOT_READ_BATCH_OF_CONTACTS_BY_ID_OR_PROPERTIES",
    READ_BATCH_OF_CRM_OBJECTS_BY_ID_OR_PROPERTY_VALUES: "HUBSPOT_READ_BATCH_OF_CRM_OBJECTS_BY_ID_OR_PROPERTY_VALUES",
    READ_BATCH_OF_LINE_ITEMS_BY_ID_OR_PROPERTY_VALUES: "HUBSPOT_READ_BATCH_OF_LINE_ITEMS_BY_ID_OR_PROPERTY_VALUES",
    READ_BATCH_OF_PRODUCTS_BY_ID_OR_PROPERTY: "HUBSPOT_READ_BATCH_OF_PRODUCTS_BY_ID_OR_PROPERTY",
    READ_BATCH_OF_QUOTES_BY_PROPERTY_VALUES: "HUBSPOT_READ_BATCH_OF_QUOTES_BY_PROPERTY_VALUES",
    READ_BATCH_OF_TICKETS_BY_ID_OR_UNIQUE_VALUES: "HUBSPOT_READ_BATCH_OF_TICKETS_BY_ID_OR_UNIQUE_VALUES",
    READ_BUDGET: "HUBSPOT_READ_BUDGET",
    READ_CRM_CONTACT_BY_ID: "HUBSPOT_READ_CRM_CONTACT_BY_ID",
    READ_CRM_OBJECT_BY_ID: "HUBSPOT_READ_CRM_OBJECT_BY_ID",
    READ_FEEDBACK_SUBMISSION_BY_ID: "HUBSPOT_READ_FEEDBACK_SUBMISSION_BY_ID",
    READ_PRODUCT_BY_ID: "HUBSPOT_READ_PRODUCT_BY_ID",
    READ_PROPERTY_GROUPS_FOR_OBJECT_TYPE: "HUBSPOT_READ_PROPERTY_GROUPS_FOR_OBJECT_TYPE",
    RECYCLE_CONTACT_BY_ID: "HUBSPOT_RECYCLE_CONTACT_BY_ID",
    REMOVE_ASSET_ASSOCIATION: "HUBSPOT_REMOVE_ASSET_ASSOCIATION",
    REMOVE_ASSOCIATION_FROM_SCHEMA: "HUBSPOT_REMOVE_ASSOCIATION_FROM_SCHEMA",
    REMOVE_DEAL_OBJECT_BY_ID: "HUBSPOT_REMOVE_DEAL_OBJECT_BY_ID",
    REMOVE_TOKEN_FROM_EVENT_TEMPLATE: "HUBSPOT_REMOVE_TOKEN_FROM_EVENT_TEMPLATE",
    RENDER_EVENT_DETAIL_TEMPLATE: "HUBSPOT_RENDER_EVENT_DETAIL_TEMPLATE",
    RENDER_EVENT_HEADER_OR_DETAIL_AS_HTML: "HUBSPOT_RENDER_EVENT_HEADER_OR_DETAIL_AS_HTML",
    REPLACE_ALL_PROPERTIES_OF_PIPELINE: "HUBSPOT_REPLACE_ALL_PROPERTIES_OF_PIPELINE",
    REPLACE_PIPELINE_STAGE_PROPERTIES: "HUBSPOT_REPLACE_PIPELINE_STAGE_PROPERTIES",
    RESET_DRAFT: "HUBSPOT_RESET_DRAFT",
    RESTORE_A_REVISION_OF_A_MARKETING_EMAIL: "HUBSPOT_RESTORE_A_REVISION_OF_A_MARKETING_EMAIL",
    RESTORE_EMAIL_REVISION: "HUBSPOT_RESTORE_EMAIL_REVISION",
    RETRIEVE_ALL_OBJECT_SCHEMAS: "HUBSPOT_RETRIEVE_ALL_OBJECT_SCHEMAS",
    RETRIEVE_ALL_PIPELINES: "HUBSPOT_RETRIEVE_ALL_PIPELINES",
    RETRIEVE_ALL_PIPELINES_FOR_SPECIFIED_OBJECT_TYPE: "HUBSPOT_RETRIEVE_ALL_PIPELINES_FOR_SPECIFIED_OBJECT_TYPE",
    RETRIEVE_BATCH_OF_DEALS_BY_IDS_OR_PROPERTY_VALUES: "HUBSPOT_RETRIEVE_BATCH_OF_DEALS_BY_IDS_OR_PROPERTY_VALUES",
    RETRIEVE_CALLING_SETTINGS_FOR_APP: "HUBSPOT_RETRIEVE_CALLING_SETTINGS_FOR_APP",
    RETRIEVE_COMPANY_OBJECT_BY_ID: "HUBSPOT_RETRIEVE_COMPANY_OBJECT_BY_ID",
    RETRIEVE_DEAL_BY_ID: "HUBSPOT_RETRIEVE_DEAL_BY_ID",
    RETRIEVE_EXISTING_OBJECT_SCHEMA: "HUBSPOT_RETRIEVE_EXISTING_OBJECT_SCHEMA",
    RETRIEVE_LINE_ITEMS_LIST: "HUBSPOT_RETRIEVE_LINE_ITEMS_LIST",
    RETRIEVE_LINE_ITEM_BY_ID: "HUBSPOT_RETRIEVE_LINE_ITEM_BY_ID",
    RETRIEVE_OWNERS: "HUBSPOT_RETRIEVE_OWNERS",
    RETRIEVE_OWNER_BY_ID_OR_USER_ID: "HUBSPOT_RETRIEVE_OWNER_BY_ID_OR_USER_ID",
    RETRIEVE_PAGE_OF_COMPANIES: "HUBSPOT_RETRIEVE_PAGE_OF_COMPANIES",
    RETRIEVE_PAGE_OF_CRM_OWNERS: "HUBSPOT_RETRIEVE_PAGE_OF_CRM_OWNERS",
    RETRIEVE_PIPELINE_STAGES: "HUBSPOT_RETRIEVE_PIPELINE_STAGES",
    RETRIEVE_PIPELINE_STAGE_BY_ID: "HUBSPOT_RETRIEVE_PIPELINE_STAGE_BY_ID",
    RETRIEVE_TICKET_BY_ID: "HUBSPOT_RETRIEVE_TICKET_BY_ID",
    RETRIEVE_TIMELINE_EVENT_BY_IDS: "HUBSPOT_RETRIEVE_TIMELINE_EVENT_BY_IDS",
    RETRIEVE_VIDEO_CONFERENCE_SETTINGS_BY_ID: "HUBSPOT_RETRIEVE_VIDEO_CONFERENCE_SETTINGS_BY_ID",
    RETURN_PIPELINE_BY_ID: "HUBSPOT_RETURN_PIPELINE_BY_ID",
    SEARCH_COMPANY_OBJECTS: "HUBSPOT_SEARCH_COMPANY_OBJECTS",
    SEARCH_CONTACTS_BY_CRITERIA: "HUBSPOT_SEARCH_CONTACTS_BY_CRITERIA",
    SEARCH_CRM_EMAIL_OBJECTS: "HUBSPOT_SEARCH_CRM_EMAIL_OBJECTS",
    SEARCH_CRM_OBJECTS_BY_CRITERIA: "HUBSPOT_SEARCH_CRM_OBJECTS_BY_CRITERIA",
    SEARCH_DEALS_BY_CRITERIA: "HUBSPOT_SEARCH_DEALS_BY_CRITERIA",
    SEARCH_FEEDBACK_SUBMISSIONS: "HUBSPOT_SEARCH_FEEDBACK_SUBMISSIONS",
    SEARCH_LINE_ITEMS_BY_CRITERIA: "HUBSPOT_SEARCH_LINE_ITEMS_BY_CRITERIA",
    SEARCH_PRODUCTS_BY_CRITERIA: "HUBSPOT_SEARCH_PRODUCTS_BY_CRITERIA",
    SEARCH_QUOTES_BY_CRITERIA: "HUBSPOT_SEARCH_QUOTES_BY_CRITERIA",
    SEARCH_TICKETS_BY_CRITERIA: "HUBSPOT_SEARCH_TICKETS_BY_CRITERIA",
    SECURE_PRODUCT_SEARCH_BY_CRITERIA: "HUBSPOT_SECURE_PRODUCT_SEARCH_BY_CRITERIA",
    SET_CALL_RECORDING_SETTINGS: "HUBSPOT_SET_CALL_RECORDING_SETTINGS",
    UPDATE: "HUBSPOT_UPDATE",
    UPDATE_A_BATCH_OF_CAMPAIGNS: "HUBSPOT_UPDATE_A_BATCH_OF_CAMPAIGNS",
    UPDATE_A_BATCH_OF_CONTACTS: "HUBSPOT_UPDATE_A_BATCH_OF_CONTACTS",
    UPDATE_A_BATCH_OF_EMAILS: "HUBSPOT_UPDATE_A_BATCH_OF_EMAILS",
    UPDATE_A_BATCH_OF_LINE_ITEMS: "HUBSPOT_UPDATE_A_BATCH_OF_LINE_ITEMS",
    UPDATE_A_MARKETING_EMAIL: "HUBSPOT_UPDATE_A_MARKETING_EMAIL",
    UPDATE_BATCH_FEEDBACK_SUBMISSIONS: "HUBSPOT_UPDATE_BATCH_FEEDBACK_SUBMISSIONS",
    UPDATE_BATCH_OF_COMPANIES: "HUBSPOT_UPDATE_BATCH_OF_COMPANIES",
    UPDATE_BATCH_OF_CRMOBJECTS_TICKETS: "HUBSPOT_UPDATE_BATCH_OF_CRMOBJECTS_TICKETS",
    UPDATE_BATCH_OF_DEALS: "HUBSPOT_UPDATE_BATCH_OF_DEALS",
    UPDATE_BATCH_OF_OBJECTS_BY_IDOR_PROPERTY_VALUES: "HUBSPOT_UPDATE_BATCH_OF_OBJECTS_BY_IDOR_PROPERTY_VALUES",
    UPDATE_BATCH_OF_PRODUCTS: "HUBSPOT_UPDATE_BATCH_OF_PRODUCTS",
    UPDATE_CALLING_APP_RECORDING_SETTINGS: "HUBSPOT_UPDATE_CALLING_APP_RECORDING_SETTINGS",
    UPDATE_CAMPAIGN: "HUBSPOT_UPDATE_CAMPAIGN",
    UPDATE_EXISTING_EVENT_TEMPLATE: "HUBSPOT_UPDATE_EXISTING_EVENT_TEMPLATE",
    UPDATE_EXISTING_OBJECT_SCHEMA: "HUBSPOT_UPDATE_EXISTING_OBJECT_SCHEMA",
    UPDATE_FEEDBACK_SUBMISSION_BY_ID: "HUBSPOT_UPDATE_FEEDBACK_SUBMISSION_BY_ID",
    UPDATE_LINE_ITEM_OBJECT_PARTIALLY: "HUBSPOT_UPDATE_LINE_ITEM_OBJECT_PARTIALLY",
    UPDATE_PIPELINE_STAGE_BY_IDS: "HUBSPOT_UPDATE_PIPELINE_STAGE_BY_IDS",
    UPDATE_PRODUCT: "HUBSPOT_UPDATE_PRODUCT",
    UPDATE_SPECIFIC_CRM_PROPERTY: "HUBSPOT_UPDATE_SPECIFIC_CRM_PROPERTY",
    UPDATE_TOKEN_ON_EVENT_TEMPLATE: "HUBSPOT_UPDATE_TOKEN_ON_EVENT_TEMPLATE",
    UPDATE_VIDEO_CONFERENCE_APP_SETTINGS: "HUBSPOT_UPDATE_VIDEO_CONFERENCE_APP_SETTINGS",
  },
  triggerTypes: {
    CONTACT_CREATED_TRIGGER: "HUBSPOT_CONTACT_CREATED_TRIGGER",
    DEAL_STAGE_UPDATED_TRIGGER: "HUBSPOT_DEAL_STAGE_UPDATED_TRIGGER",
  },
}
