// @composio/core/generated/intercom.ts
/**
 * Map of Composio's INTERCOM toolkit.
 */
export const INTERCOM = {
  slug: "intercom",
  tools: {
    ADD_SUBSCRIPTION_TO_A_CONTACT: "INTERCOM_ADD_SUBSCRIPTION_TO_A_CONTACT",
    ADD_TAG_TO_A_CONTACT: "INTERCOM_ADD_TAG_TO_A_CONTACT",
    ATTACH_A_CONTACT_TO_A_COMPANY: "INTERCOM_ATTACH_A_CONTACT_TO_A_COMPANY",
    CREATE_AN_ARTICLE: "INTERCOM_CREATE_AN_ARTICLE",
    CREATE_A_COLLECTION: "INTERCOM_CREATE_A_COLLECTION",
    CREATE_A_NOTE: "INTERCOM_CREATE_A_NOTE",
    CREATE_OR_UPDATE_A_COMPANY: "INTERCOM_CREATE_OR_UPDATE_A_COMPANY",
    DELETE_AN_ARTICLE: "INTERCOM_DELETE_AN_ARTICLE",
    DELETE_A_COLLECTION: "INTERCOM_DELETE_A_COLLECTION",
    DELETE_A_COMPANY: "INTERCOM_DELETE_A_COMPANY",
    DELETE_A_CONTACT: "INTERCOM_DELETE_A_CONTACT",
    DETACH_A_CONTACT_FROM_A_COMPANY: "INTERCOM_DETACH_A_CONTACT_FROM_A_COMPANY",
    GET_A_CONTACT: "INTERCOM_GET_A_CONTACT",
    IDENTIFY_AN_ADMIN: "INTERCOM_IDENTIFY_AN_ADMIN",
    LIST_ALL_ACTIVITY_LOGS: "INTERCOM_LIST_ALL_ACTIVITY_LOGS",
    LIST_ALL_ADMINS: "INTERCOM_LIST_ALL_ADMINS",
    LIST_ALL_ARTICLES: "INTERCOM_LIST_ALL_ARTICLES",
    LIST_ALL_COLLECTIONS: "INTERCOM_LIST_ALL_COLLECTIONS",
    LIST_ALL_COMPANIES: "INTERCOM_LIST_ALL_COMPANIES",
    LIST_ALL_HELP_CENTERS: "INTERCOM_LIST_ALL_HELP_CENTERS",
    LIST_ALL_NOTES: "INTERCOM_LIST_ALL_NOTES",
    LIST_ATTACHED_COMPANIES_FOR_CONTACT: "INTERCOM_LIST_ATTACHED_COMPANIES_FOR_CONTACT",
    LIST_ATTACHED_CONTACTS: "INTERCOM_LIST_ATTACHED_CONTACTS",
    LIST_ATTACHED_SEGMENTS_FOR_COMPANIES: "INTERCOM_LIST_ATTACHED_SEGMENTS_FOR_COMPANIES",
    LIST_ATTACHED_SEGMENTS_FOR_CONTACT: "INTERCOM_LIST_ATTACHED_SEGMENTS_FOR_CONTACT",
    LIST_SUBSCRIPTIONS_FOR_A_CONTACT: "INTERCOM_LIST_SUBSCRIPTIONS_FOR_A_CONTACT",
    LIST_TAGS_ATTACHED_TO_A_CONTACT: "INTERCOM_LIST_TAGS_ATTACHED_TO_A_CONTACT",
    MERGE_A_LEAD_AND_A_USER: "INTERCOM_MERGE_A_LEAD_AND_A_USER",
    REMOVE_SUBSCRIPTION_FROM_A_CONTACT: "INTERCOM_REMOVE_SUBSCRIPTION_FROM_A_CONTACT",
    REMOVE_TAG_FROM_A_CONTACT: "INTERCOM_REMOVE_TAG_FROM_A_CONTACT",
    RETRIEVE_AN_ADMIN: "INTERCOM_RETRIEVE_AN_ADMIN",
    RETRIEVE_AN_ARTICLE: "INTERCOM_RETRIEVE_AN_ARTICLE",
    RETRIEVE_A_COLLECTION: "INTERCOM_RETRIEVE_A_COLLECTION",
    RETRIEVE_A_COMPANY_BY_ID: "INTERCOM_RETRIEVE_A_COMPANY_BY_ID",
    RETRIEVE_A_HELP_CENTER: "INTERCOM_RETRIEVE_A_HELP_CENTER",
    RETRIEVE_COMPANIES: "INTERCOM_RETRIEVE_COMPANIES",
    SCROLL_OVER_ALL_COMPANIES: "INTERCOM_SCROLL_OVER_ALL_COMPANIES",
    SEARCH_FOR_ARTICLES: "INTERCOM_SEARCH_FOR_ARTICLES",
    SET_AN_ADMIN_TO_AWAY: "INTERCOM_SET_AN_ADMIN_TO_AWAY",
    UPDATE_AN_ARTICLE: "INTERCOM_UPDATE_AN_ARTICLE",
    UPDATE_A_COLLECTION: "INTERCOM_UPDATE_A_COLLECTION",
    UPDATE_A_COMPANY: "INTERCOM_UPDATE_A_COMPANY",
    UPDATE_A_CONTACT: "INTERCOM_UPDATE_A_CONTACT",
  },
  triggerTypes: {},
}
