// @composio/core/generated/jira.ts
/**
 * Map of Composio's JIRA toolkit.
 */
export const JIRA = {
  slug: "jira",
  tools: {
    ADD_ATTACHMENT_TO_ISSUE: "JIRA_ADD_ATTACHMENT_TO_ISSUE",
    ADD_COMMENT: "JIRA_ADD_COMMENT",
    ADD_WATCHER_TO_ISSUE: "JIRA_ADD_WATCHER_TO_ISSUE",
    ASSIGN_ISSUE: "JIRA_ASSIGN_ISSUE",
    BULK_CREATE_ISSUE: "JIRA_BULK_CREATE_ISSUE",
    CREATE_ISSUE: "JIRA_CREATE_ISSUE",
    CREATE_PROJECT: "JIRA_CREATE_PROJECT",
    CREATE_SPRINT: "JIRA_CREATE_SPRINT",
    CREATE_VERSION: "JIRA_CREATE_VERSION",
    DELETE_COMMENT: "JIRA_DELETE_COMMENT",
    DELETE_ISSUE: "JIRA_DELETE_ISSUE",
    DELETE_VERSION: "JIRA_DELETE_VERSION",
    DELETE_WORKLOG: "JIRA_DELETE_WORKLOG",
    EDIT_ISSUE: "JIRA_EDIT_ISSUE",
    FIND_USERS: "JIRA_FIND_USERS",
    GET_ALL_PROJECTS: "JIRA_GET_ALL_PROJECTS",
    GET_ALL_USERS: "JIRA_GET_ALL_USERS",
    GET_COMMENT: "JIRA_GET_COMMENT",
    GET_CURRENT_USER: "JIRA_GET_CURRENT_USER",
    GET_ISSUE: "JIRA_GET_ISSUE",
    GET_ISSUE_LINK_TYPES: "JIRA_GET_ISSUE_LINK_TYPES",
    GET_ISSUE_PROPERTY: "JIRA_GET_ISSUE_PROPERTY",
    GET_ISSUE_REMOTE_LINKS: "JIRA_GET_ISSUE_REMOTE_LINKS",
    GET_ISSUE_RESOLUTIONS: "JIRA_GET_ISSUE_RESOLUTIONS",
    GET_ISSUE_STATUSES: "JIRA_GET_ISSUE_STATUSES",
    GET_ISSUE_TYPES: "JIRA_GET_ISSUE_TYPES",
    GET_ISSUE_TYPE_SCHEME: "JIRA_GET_ISSUE_TYPE_SCHEME",
    GET_ISSUE_VOTES: "JIRA_GET_ISSUE_VOTES",
    GET_ISSUE_WATCHERS: "JIRA_GET_ISSUE_WATCHERS",
    GET_ISSUE_WORKLOGS: "JIRA_GET_ISSUE_WORKLOGS",
    GET_PROJECT_VERSIONS: "JIRA_GET_PROJECT_VERSIONS",
    GET_TRANSITIONS: "JIRA_GET_TRANSITIONS",
    GET_WORKLOG: "JIRA_GET_WORKLOG",
    LINK_ISSUES: "JIRA_LINK_ISSUES",
    LIST_BOARDS: "JIRA_LIST_BOARDS",
    LIST_ISSUE_COMMENTS: "JIRA_LIST_ISSUE_COMMENTS",
    LIST_SPRINTS: "JIRA_LIST_SPRINTS",
    MOVE_ISSUE_TO_SPRINT: "JIRA_MOVE_ISSUE_TO_SPRINT",
    NATURAL_LANGUAGE_TO_JQL: "JIRA_NATURAL_LANGUAGE_TO_JQL",
    REMOVE_WATCHER_FROM_ISSUE: "JIRA_REMOVE_WATCHER_FROM_ISSUE",
    SEARCH_FOR_ISSUES_USING_JQL_GET: "JIRA_SEARCH_FOR_ISSUES_USING_JQL_GET",
    SEARCH_FOR_ISSUES_USING_JQL_POST: "JIRA_SEARCH_FOR_ISSUES_USING_JQL_POST",
    SEARCH_ISSUES: "JIRA_SEARCH_ISSUES",
    SEND_NOTIFICATION_FOR_ISSUE: "JIRA_SEND_NOTIFICATION_FOR_ISSUE",
    TRANSITION_ISSUE: "JIRA_TRANSITION_ISSUE",
    UPDATE_COMMENT: "JIRA_UPDATE_COMMENT",
  },
  triggerTypes: {
    NEW_ISSUE_TRIGGER: "JIRA_NEW_ISSUE_TRIGGER",
    NEW_PROJECT_TRIGGER: "JIRA_NEW_PROJECT_TRIGGER",
    UPDATED_ISSUE_TRIGGER: "JIRA_UPDATED_ISSUE_TRIGGER",
  },
}
