// @composio/core/generated/kibana.ts
/**
 * Map of Composio's KIBANA toolkit.
 */
export const KIBANA = {
  slug: "kibana",
  tools: {
    DELETE_ACTION: "KIBANA_DELETE_ACTION",
    DELETE_ALERTING_RULES: "KIBANA_DELETE_ALERTING_RULES",
    DELETE_CONNECTORS: "KIBANA_DELETE_CONNECTORS",
    DELETE_FLEET_OUTPUT: "KIBANA_DELETE_FLEET_OUTPUT",
    DELETE_FLEET_PROXY: "KIBANA_DELETE_FLEET_PROXY",
    DELETE_LIST: "KIBANA_DELETE_LIST",
    DELETE_OSQUERY_SAVED_QUERIES: "KIBANA_DELETE_OSQUERY_SAVED_QUERIES",
    DELETE_SAVED_OBJECTS: "KIBANA_DELETE_SAVED_OBJECTS",
    FIND_ALERTS: "KIBANA_FIND_ALERTS",
    GET_ACTION_TYPES: "KIBANA_GET_ACTION_TYPES",
    GET_ALERTING_RULES: "KIBANA_GET_ALERTING_RULES",
    GET_ALERT_TYPES: "KIBANA_GET_ALERT_TYPES",
    GET_CASES: "KIBANA_GET_CASES",
    GET_CONNECTORS: "KIBANA_GET_CONNECTORS",
    GET_DATA_VIEWS: "KIBANA_GET_DATA_VIEWS",
    GET_DETECTION_ENGINE_RULES_FIND: "KIBANA_GET_DETECTION_ENGINE_RULES_FIND",
    GET_ENDPOINT_LIST_ITEMS: "KIBANA_GET_ENDPOINT_LIST_ITEMS",
    GET_ENTITY_STORE_ENGINES: "KIBANA_GET_ENTITY_STORE_ENGINES",
    GET_ENTITY_STORE_ENTITIES_LIST: "KIBANA_GET_ENTITY_STORE_ENTITIES_LIST",
    GET_ENTITY_STORE_STATUS: "KIBANA_GET_ENTITY_STORE_STATUS",
    GET_FLEET_AGENTS_AVAILABLE_VERSIONS: "KIBANA_GET_FLEET_AGENTS_AVAILABLE_VERSIONS",
    GET_FLEET_AGENTS_SETUP_STATUS: "KIBANA_GET_FLEET_AGENTS_SETUP_STATUS",
    GET_FLEET_AGENT_POLICIES: "KIBANA_GET_FLEET_AGENT_POLICIES",
    GET_FLEET_CHECK_PERMISSIONS: "KIBANA_GET_FLEET_CHECK_PERMISSIONS",
    GET_FLEET_DATA_STREAMS: "KIBANA_GET_FLEET_DATA_STREAMS",
    GET_FLEET_ENROLLMENT_API_KEY: "KIBANA_GET_FLEET_ENROLLMENT_API_KEY",
    GET_FLEET_ENROLLMENT_API_KEYS: "KIBANA_GET_FLEET_ENROLLMENT_API_KEYS",
    GET_FLEET_EPM_CATEGORIES: "KIBANA_GET_FLEET_EPM_CATEGORIES",
    GET_FLEET_EPM_DATA_STREAMS: "KIBANA_GET_FLEET_EPM_DATA_STREAMS",
    GET_FLEET_EPM_PACKAGES: "KIBANA_GET_FLEET_EPM_PACKAGES",
    GET_FLEET_EPM_PACKAGES_INSTALLED: "KIBANA_GET_FLEET_EPM_PACKAGES_INSTALLED",
    GET_FLEET_EPM_PACKAGES_LIMITED: "KIBANA_GET_FLEET_EPM_PACKAGES_LIMITED",
    GET_FLEET_EPM_PACKAGE_DETAILS: "KIBANA_GET_FLEET_EPM_PACKAGE_DETAILS",
    GET_FLEET_EPM_PACKAGE_FILE: "KIBANA_GET_FLEET_EPM_PACKAGE_FILE",
    GET_FLEET_EPM_PACKAGE_STATS: "KIBANA_GET_FLEET_EPM_PACKAGE_STATS",
    GET_FLEET_PACKAGE_POLICIES: "KIBANA_GET_FLEET_PACKAGE_POLICIES",
    GET_FLEET_SERVER_HOST: "KIBANA_GET_FLEET_SERVER_HOST",
    GET_FLEET_SERVER_HOSTS: "KIBANA_GET_FLEET_SERVER_HOSTS",
    GET_INDEX_MANAGEMENT_INDICES: "KIBANA_GET_INDEX_MANAGEMENT_INDICES",
    GET_METRICS: "KIBANA_GET_METRICS",
    GET_REPORTING_JOBS: "KIBANA_GET_REPORTING_JOBS",
    GET_SAVED_OBJECTS: "KIBANA_GET_SAVED_OBJECTS",
    GET_STATUS: "KIBANA_GET_STATUS",
    POST_ALERTING_RULES: "KIBANA_POST_ALERTING_RULES",
    POST_CASES: "KIBANA_POST_CASES",
    POST_CONNECTORS: "KIBANA_POST_CONNECTORS",
    POST_DASHBOARDS: "KIBANA_POST_DASHBOARDS",
    POST_DATA_VIEWS: "KIBANA_POST_DATA_VIEWS",
    POST_SAVED_OBJECTS: "KIBANA_POST_SAVED_OBJECTS",
  },
  triggerTypes: {},
}
