// @composio/core/generated/kit.ts
/**
 * Map of Composio's KIT toolkit.
 */
export const KIT = {
  slug: "kit",
  tools: {
    ADD_SUBSCRIBER_TO_FORM: "KIT_ADD_SUBSCRIBER_TO_FORM",
    CREATE_CUSTOM_FIELD: "KIT_CREATE_CUSTOM_FIELD",
    CREATE_TAG: "KIT_CREATE_TAG",
    CREATE_WEBHOOK: "KIT_CREATE_WEBHOOK",
    DELETE_BROADCAST: "KIT_DELETE_BROADCAST",
    DELETE_CUSTOM_FIELD: "KIT_DELETE_CUSTOM_FIELD",
    DELETE_SUBSCRIBER: "KIT_DELETE_SUBSCRIBER",
    DELETE_TAG: "KIT_DELETE_TAG",
    DELETE_WEBHOOK: "KIT_DELETE_WEBHOOK",
    GET_ACCOUNT: "KIT_GET_ACCOUNT",
    GET_ACCOUNT_COLORS: "KIT_GET_ACCOUNT_COLORS",
    GET_BROADCAST: "KIT_GET_BROADCAST",
    GET_BROADCAST_STATS: "KIT_GET_BROADCAST_STATS",
    GET_CREATOR_PROFILE: "KIT_GET_CREATOR_PROFILE",
    GET_EMAIL_STATS: "KIT_GET_EMAIL_STATS",
    LIST_BROADCASTS: "KIT_LIST_BROADCASTS",
    LIST_CUSTOM_FIELDS: "KIT_LIST_CUSTOM_FIELDS",
    LIST_FORMS: "KIT_LIST_FORMS",
    LIST_SEGMENTS: "KIT_LIST_SEGMENTS",
    LIST_SEQUENCES: "KIT_LIST_SEQUENCES",
    LIST_SUBSCRIBERS: "KIT_LIST_SUBSCRIBERS",
    LIST_SUBSCRIBERS_FOR_FORM: "KIT_LIST_SUBSCRIBERS_FOR_FORM",
    LIST_TAGS: "KIT_LIST_TAGS",
    LIST_TAG_SUBSCRIBERS: "KIT_LIST_TAG_SUBSCRIBERS",
    TAG_SUBSCRIBER: "KIT_TAG_SUBSCRIBER",
    TAG_SUBSCRIBER_BY_EMAIL: "KIT_TAG_SUBSCRIBER_BY_EMAIL",
    UPDATE_ACCOUNT_COLORS: "KIT_UPDATE_ACCOUNT_COLORS",
    UPDATE_CUSTOM_FIELD: "KIT_UPDATE_CUSTOM_FIELD",
    UPDATE_TAG: "KIT_UPDATE_TAG",
  },
  triggerTypes: {},
}
