// @composio/core/generated/klaviyo.ts
/**
 * Map of Composio's KLAVIYO toolkit.
 */
export const KLAVIYO = {
  slug: "klaviyo",
  tools: {
    ADD_PROFILE_TO_LIST: "KLAVIYO_ADD_PROFILE_TO_LIST",
    ASSIGN_CAMPAIGN_MESSAGE_TEMPLATE: "KLAVIYO_ASSIGN_CAMPAIGN_MESSAGE_TEMPLATE",
    BULK_CREATE_CLIENT_EVENTS: "KLAVIYO_BULK_CREATE_CLIENT_EVENTS",
    BULK_CREATE_EVENTS: "KLAVIYO_BULK_CREATE_EVENTS",
    CREATE_BACK_IN_STOCK_SUBSCRIPTION: "KLAVIYO_CREATE_BACK_IN_STOCK_SUBSCRIPTION",
    CREATE_CAMPAIGN: "KLAVIYO_CREATE_CAMPAIGN",
    CREATE_CAMPAIGN_CLONE: "KLAVIYO_CREATE_CAMPAIGN_CLONE",
    CREATE_CAMPAIGN_RECIPIENT_ESTIMATION_JOB: "KLAVIYO_CREATE_CAMPAIGN_RECIPIENT_ESTIMATION_JOB",
    CREATE_CAMPAIGN_SEND_JOB: "KLAVIYO_CREATE_CAMPAIGN_SEND_JOB",
    CREATE_CATALOG_CATEGORY: "KLAVIYO_CREATE_CATALOG_CATEGORY",
    CREATE_CATALOG_CATEGORY_RELATIONSHIPS_ITEMS: "KLAVIYO_CREATE_CATALOG_CATEGORY_RELATIONSHIPS_ITEMS",
    CREATE_CATALOG_ITEM: "KLAVIYO_CREATE_CATALOG_ITEM",
    CREATE_CATALOG_ITEM_RELATIONSHIPS_CATEGORIES: "KLAVIYO_CREATE_CATALOG_ITEM_RELATIONSHIPS_CATEGORIES",
    CREATE_CATALOG_VARIANT: "KLAVIYO_CREATE_CATALOG_VARIANT",
    CREATE_CLIENT_BACK_IN_STOCK_SUBSCRIPTION: "KLAVIYO_CREATE_CLIENT_BACK_IN_STOCK_SUBSCRIPTION",
    CREATE_CLIENT_EVENT: "KLAVIYO_CREATE_CLIENT_EVENT",
    CREATE_CLIENT_SUBSCRIPTION: "KLAVIYO_CREATE_CLIENT_SUBSCRIPTION",
    CREATE_COUPON: "KLAVIYO_CREATE_COUPON",
    CREATE_COUPON_CODE: "KLAVIYO_CREATE_COUPON_CODE",
    CREATE_EVENT: "KLAVIYO_CREATE_EVENT",
    CREATE_LIST: "KLAVIYO_CREATE_LIST",
    CREATE_OR_UPDATE_CLIENT_PROFILE: "KLAVIYO_CREATE_OR_UPDATE_CLIENT_PROFILE",
    CREATE_OR_UPDATE_CLIENT_PUSH_TOKEN: "KLAVIYO_CREATE_OR_UPDATE_CLIENT_PUSH_TOKEN",
    CREATE_OR_UPDATE_PROFILE: "KLAVIYO_CREATE_OR_UPDATE_PROFILE",
    CREATE_OR_UPDATE_PUSH_TOKEN: "KLAVIYO_CREATE_OR_UPDATE_PUSH_TOKEN",
    CREATE_PROFILE: "KLAVIYO_CREATE_PROFILE",
    CREATE_SEGMENT: "KLAVIYO_CREATE_SEGMENT",
    CREATE_TAG: "KLAVIYO_CREATE_TAG",
    CREATE_TAG_GROUP: "KLAVIYO_CREATE_TAG_GROUP",
    CREATE_TAG_RELATIONSHIPS_CAMPAIGNS: "KLAVIYO_CREATE_TAG_RELATIONSHIPS_CAMPAIGNS",
    CREATE_TAG_RELATIONSHIPS_FLOWS: "KLAVIYO_CREATE_TAG_RELATIONSHIPS_FLOWS",
    CREATE_TAG_RELATIONSHIPS_LISTS: "KLAVIYO_CREATE_TAG_RELATIONSHIPS_LISTS",
    CREATE_TAG_RELATIONSHIPS_SEGMENTS: "KLAVIYO_CREATE_TAG_RELATIONSHIPS_SEGMENTS",
    CREATE_TEMPLATE: "KLAVIYO_CREATE_TEMPLATE",
    CREATE_TEMPLATE_CLONE: "KLAVIYO_CREATE_TEMPLATE_CLONE",
    CREATE_TEMPLATE_RENDER: "KLAVIYO_CREATE_TEMPLATE_RENDER",
    CREATE_WEBHOOK: "KLAVIYO_CREATE_WEBHOOK",
    DELETE_CAMPAIGN: "KLAVIYO_DELETE_CAMPAIGN",
    DELETE_CATALOG_CATEGORY: "KLAVIYO_DELETE_CATALOG_CATEGORY",
    DELETE_CATALOG_CATEGORY_RELATIONSHIPS_ITEMS: "KLAVIYO_DELETE_CATALOG_CATEGORY_RELATIONSHIPS_ITEMS",
    DELETE_CATALOG_ITEM: "KLAVIYO_DELETE_CATALOG_ITEM",
    DELETE_CATALOG_ITEM_RELATIONSHIPS_CATEGORIES: "KLAVIYO_DELETE_CATALOG_ITEM_RELATIONSHIPS_CATEGORIES",
    DELETE_CATALOG_VARIANT: "KLAVIYO_DELETE_CATALOG_VARIANT",
    DELETE_COUPON: "KLAVIYO_DELETE_COUPON",
    DELETE_COUPON_CODE: "KLAVIYO_DELETE_COUPON_CODE",
    DELETE_FLOW: "KLAVIYO_DELETE_FLOW",
    DELETE_LIST: "KLAVIYO_DELETE_LIST",
    DELETE_SEGMENT: "KLAVIYO_DELETE_SEGMENT",
    DELETE_TAG: "KLAVIYO_DELETE_TAG",
    DELETE_TAG_GROUP: "KLAVIYO_DELETE_TAG_GROUP",
    DELETE_TAG_RELATIONSHIPS_CAMPAIGNS: "KLAVIYO_DELETE_TAG_RELATIONSHIPS_CAMPAIGNS",
    DELETE_TAG_RELATIONSHIPS_FLOWS: "KLAVIYO_DELETE_TAG_RELATIONSHIPS_FLOWS",
    DELETE_TAG_RELATIONSHIPS_LISTS: "KLAVIYO_DELETE_TAG_RELATIONSHIPS_LISTS",
    DELETE_TAG_RELATIONSHIPS_SEGMENTS: "KLAVIYO_DELETE_TAG_RELATIONSHIPS_SEGMENTS",
    DELETE_TEMPLATE: "KLAVIYO_DELETE_TEMPLATE",
    DELETE_WEBHOOK: "KLAVIYO_DELETE_WEBHOOK",
    GET_ACCOUNT: "KLAVIYO_GET_ACCOUNT",
    GET_ACCOUNTS: "KLAVIYO_GET_ACCOUNTS",
    GET_BULK_PROFILE_IMPORT_JOB: "KLAVIYO_GET_BULK_PROFILE_IMPORT_JOB",
    GET_BULK_PROFILE_IMPORT_JOBS: "KLAVIYO_GET_BULK_PROFILE_IMPORT_JOBS",
    GET_BULK_PROFILE_IMPORT_JOB_ERRORS: "KLAVIYO_GET_BULK_PROFILE_IMPORT_JOB_ERRORS",
    GET_BULK_PROFILE_IMPORT_JOB_LISTS: "KLAVIYO_GET_BULK_PROFILE_IMPORT_JOB_LISTS",
    GET_BULK_PROFILE_IMPORT_JOB_PROFILES: "KLAVIYO_GET_BULK_PROFILE_IMPORT_JOB_PROFILES",
    GET_BULK_PROFILE_IMPORT_JOB_RELATIONSHIPS_LISTS: "KLAVIYO_GET_BULK_PROFILE_IMPORT_JOB_RELATIONSHIPS_LISTS",
    GET_BULK_PROFILE_IMPORT_JOB_RELATIONSHIPS_PROFILES: "KLAVIYO_GET_BULK_PROFILE_IMPORT_JOB_RELATIONSHIPS_PROFILES",
    GET_CAMPAIGN: "KLAVIYO_GET_CAMPAIGN",
    GET_CAMPAIGNS: "KLAVIYO_GET_CAMPAIGNS",
    GET_CAMPAIGN_CAMPAIGN_MESSAGES: "KLAVIYO_GET_CAMPAIGN_CAMPAIGN_MESSAGES",
    GET_CAMPAIGN_MESSAGE: "KLAVIYO_GET_CAMPAIGN_MESSAGE",
    GET_CAMPAIGN_MESSAGE_CAMPAIGN: "KLAVIYO_GET_CAMPAIGN_MESSAGE_CAMPAIGN",
    GET_CAMPAIGN_MESSAGE_RELATIONSHIPS_CAMPAIGN: "KLAVIYO_GET_CAMPAIGN_MESSAGE_RELATIONSHIPS_CAMPAIGN",
    GET_CAMPAIGN_MESSAGE_RELATIONSHIPS_TEMPLATE: "KLAVIYO_GET_CAMPAIGN_MESSAGE_RELATIONSHIPS_TEMPLATE",
    GET_CAMPAIGN_MESSAGE_TEMPLATE: "KLAVIYO_GET_CAMPAIGN_MESSAGE_TEMPLATE",
    GET_CAMPAIGN_RECIPIENT_ESTIMATION: "KLAVIYO_GET_CAMPAIGN_RECIPIENT_ESTIMATION",
    GET_CAMPAIGN_RECIPIENT_ESTIMATION_JOB: "KLAVIYO_GET_CAMPAIGN_RECIPIENT_ESTIMATION_JOB",
    GET_CAMPAIGN_RELATIONSHIPS_CAMPAIGN_MESSAGES: "KLAVIYO_GET_CAMPAIGN_RELATIONSHIPS_CAMPAIGN_MESSAGES",
    GET_CAMPAIGN_RELATIONSHIPS_TAGS: "KLAVIYO_GET_CAMPAIGN_RELATIONSHIPS_TAGS",
    GET_CAMPAIGN_SEND_JOB: "KLAVIYO_GET_CAMPAIGN_SEND_JOB",
    GET_CAMPAIGN_TAGS: "KLAVIYO_GET_CAMPAIGN_TAGS",
    GET_CATALOG_CATEGORIES: "KLAVIYO_GET_CATALOG_CATEGORIES",
    GET_CATALOG_CATEGORY: "KLAVIYO_GET_CATALOG_CATEGORY",
    GET_CATALOG_CATEGORY_ITEMS: "KLAVIYO_GET_CATALOG_CATEGORY_ITEMS",
    GET_CATALOG_CATEGORY_RELATIONSHIPS_ITEMS: "KLAVIYO_GET_CATALOG_CATEGORY_RELATIONSHIPS_ITEMS",
    GET_CATALOG_ITEM: "KLAVIYO_GET_CATALOG_ITEM",
    GET_CATALOG_ITEMS: "KLAVIYO_GET_CATALOG_ITEMS",
    GET_CATALOG_ITEM_CATEGORIES: "KLAVIYO_GET_CATALOG_ITEM_CATEGORIES",
    GET_CATALOG_ITEM_RELATIONSHIPS_CATEGORIES: "KLAVIYO_GET_CATALOG_ITEM_RELATIONSHIPS_CATEGORIES",
    GET_CATALOG_ITEM_VARIANTS: "KLAVIYO_GET_CATALOG_ITEM_VARIANTS",
    GET_CATALOG_VARIANT: "KLAVIYO_GET_CATALOG_VARIANT",
    GET_CATALOG_VARIANTS: "KLAVIYO_GET_CATALOG_VARIANTS",
    GET_COUPON: "KLAVIYO_GET_COUPON",
    GET_COUPONS: "KLAVIYO_GET_COUPONS",
    GET_COUPON_CODE: "KLAVIYO_GET_COUPON_CODE",
    GET_COUPON_CODES: "KLAVIYO_GET_COUPON_CODES",
    GET_COUPON_CODES_FOR_COUPON: "KLAVIYO_GET_COUPON_CODES_FOR_COUPON",
    GET_COUPON_CODE_BULK_CREATE_JOB: "KLAVIYO_GET_COUPON_CODE_BULK_CREATE_JOB",
    GET_COUPON_CODE_BULK_CREATE_JOBS: "KLAVIYO_GET_COUPON_CODE_BULK_CREATE_JOBS",
    GET_COUPON_CODE_RELATIONSHIPS_COUPON: "KLAVIYO_GET_COUPON_CODE_RELATIONSHIPS_COUPON",
    GET_COUPON_FOR_COUPON_CODE: "KLAVIYO_GET_COUPON_FOR_COUPON_CODE",
    GET_COUPON_RELATIONSHIPS_COUPON_CODES: "KLAVIYO_GET_COUPON_RELATIONSHIPS_COUPON_CODES",
    GET_CREATE_CATEGORIES_JOB: "KLAVIYO_GET_CREATE_CATEGORIES_JOB",
    GET_CREATE_CATEGORIES_JOBS: "KLAVIYO_GET_CREATE_CATEGORIES_JOBS",
    GET_CREATE_ITEMS_JOB: "KLAVIYO_GET_CREATE_ITEMS_JOB",
    GET_CREATE_ITEMS_JOBS: "KLAVIYO_GET_CREATE_ITEMS_JOBS",
    GET_CREATE_VARIANTS_JOB: "KLAVIYO_GET_CREATE_VARIANTS_JOB",
    GET_CREATE_VARIANTS_JOBS: "KLAVIYO_GET_CREATE_VARIANTS_JOBS",
    GET_DELETE_CATEGORIES_JOB: "KLAVIYO_GET_DELETE_CATEGORIES_JOB",
    GET_DELETE_CATEGORIES_JOBS: "KLAVIYO_GET_DELETE_CATEGORIES_JOBS",
    GET_DELETE_ITEMS_JOB: "KLAVIYO_GET_DELETE_ITEMS_JOB",
    GET_DELETE_ITEMS_JOBS: "KLAVIYO_GET_DELETE_ITEMS_JOBS",
    GET_DELETE_VARIANTS_JOB: "KLAVIYO_GET_DELETE_VARIANTS_JOB",
    GET_DELETE_VARIANTS_JOBS: "KLAVIYO_GET_DELETE_VARIANTS_JOBS",
    GET_EVENT: "KLAVIYO_GET_EVENT",
    GET_EVENTS: "KLAVIYO_GET_EVENTS",
    GET_EVENT_METRIC: "KLAVIYO_GET_EVENT_METRIC",
    GET_EVENT_PROFILE: "KLAVIYO_GET_EVENT_PROFILE",
    GET_EVENT_RELATIONSHIPS_METRIC: "KLAVIYO_GET_EVENT_RELATIONSHIPS_METRIC",
    GET_EVENT_RELATIONSHIPS_PROFILE: "KLAVIYO_GET_EVENT_RELATIONSHIPS_PROFILE",
    GET_FLOW: "KLAVIYO_GET_FLOW",
    GET_FLOWS: "KLAVIYO_GET_FLOWS",
    GET_FLOW_ACTION: "KLAVIYO_GET_FLOW_ACTION",
    GET_FLOW_ACTION_FOR_MESSAGE: "KLAVIYO_GET_FLOW_ACTION_FOR_MESSAGE",
    GET_FLOW_ACTION_MESSAGES: "KLAVIYO_GET_FLOW_ACTION_MESSAGES",
    GET_FLOW_ACTION_RELATIONSHIPS_FLOW: "KLAVIYO_GET_FLOW_ACTION_RELATIONSHIPS_FLOW",
    GET_FLOW_ACTION_RELATIONSHIPS_MESSAGES: "KLAVIYO_GET_FLOW_ACTION_RELATIONSHIPS_MESSAGES",
    GET_FLOW_FLOW_ACTIONS: "KLAVIYO_GET_FLOW_FLOW_ACTIONS",
    GET_FLOW_FOR_FLOW_ACTION: "KLAVIYO_GET_FLOW_FOR_FLOW_ACTION",
    GET_FLOW_MESSAGE: "KLAVIYO_GET_FLOW_MESSAGE",
    GET_FLOW_MESSAGE_RELATIONSHIPS_ACTION: "KLAVIYO_GET_FLOW_MESSAGE_RELATIONSHIPS_ACTION",
    GET_FLOW_MESSAGE_RELATIONSHIPS_TEMPLATE: "KLAVIYO_GET_FLOW_MESSAGE_RELATIONSHIPS_TEMPLATE",
    GET_FLOW_MESSAGE_TEMPLATE: "KLAVIYO_GET_FLOW_MESSAGE_TEMPLATE",
    GET_FLOW_RELATIONSHIPS_FLOW_ACTIONS: "KLAVIYO_GET_FLOW_RELATIONSHIPS_FLOW_ACTIONS",
    GET_FLOW_RELATIONSHIPS_TAGS: "KLAVIYO_GET_FLOW_RELATIONSHIPS_TAGS",
    GET_FLOW_TAGS: "KLAVIYO_GET_FLOW_TAGS",
    GET_FORM: "KLAVIYO_GET_FORM",
    GET_FORMS: "KLAVIYO_GET_FORMS",
    GET_FORM_FOR_FORM_VERSION: "KLAVIYO_GET_FORM_FOR_FORM_VERSION",
    GET_FORM_ID_FOR_FORM_VERSION: "KLAVIYO_GET_FORM_ID_FOR_FORM_VERSION",
    GET_FORM_VERSION: "KLAVIYO_GET_FORM_VERSION",
    GET_IMAGE: "KLAVIYO_GET_IMAGE",
    GET_IMAGES: "KLAVIYO_GET_IMAGES",
    GET_LIST: "KLAVIYO_GET_LIST",
    GET_LISTS: "KLAVIYO_GET_LISTS",
    GET_LIST_PROFILES: "KLAVIYO_GET_LIST_PROFILES",
    GET_LIST_RELATIONSHIPS_PROFILES: "KLAVIYO_GET_LIST_RELATIONSHIPS_PROFILES",
    GET_LIST_RELATIONSHIPS_TAGS: "KLAVIYO_GET_LIST_RELATIONSHIPS_TAGS",
    GET_LIST_TAGS: "KLAVIYO_GET_LIST_TAGS",
    GET_METRIC: "KLAVIYO_GET_METRIC",
    GET_METRICS: "KLAVIYO_GET_METRICS",
    GET_PROFILE: "KLAVIYO_GET_PROFILE",
    GET_PROFILES: "KLAVIYO_GET_PROFILES",
    GET_PROFILE_LISTS: "KLAVIYO_GET_PROFILE_LISTS",
    GET_PROFILE_RELATIONSHIPS_LISTS: "KLAVIYO_GET_PROFILE_RELATIONSHIPS_LISTS",
    GET_PROFILE_RELATIONSHIPS_SEGMENTS: "KLAVIYO_GET_PROFILE_RELATIONSHIPS_SEGMENTS",
    GET_PROFILE_SEGMENTS: "KLAVIYO_GET_PROFILE_SEGMENTS",
    GET_SEGMENT: "KLAVIYO_GET_SEGMENT",
    GET_SEGMENTS: "KLAVIYO_GET_SEGMENTS",
    GET_SEGMENT_PROFILES: "KLAVIYO_GET_SEGMENT_PROFILES",
    GET_SEGMENT_RELATIONSHIPS_PROFILES: "KLAVIYO_GET_SEGMENT_RELATIONSHIPS_PROFILES",
    GET_SEGMENT_RELATIONSHIPS_TAGS: "KLAVIYO_GET_SEGMENT_RELATIONSHIPS_TAGS",
    GET_SEGMENT_TAGS: "KLAVIYO_GET_SEGMENT_TAGS",
    GET_TAG: "KLAVIYO_GET_TAG",
    GET_TAGS: "KLAVIYO_GET_TAGS",
    GET_TAG_GROUP: "KLAVIYO_GET_TAG_GROUP",
    GET_TAG_GROUPS: "KLAVIYO_GET_TAG_GROUPS",
    GET_TAG_GROUP_RELATIONSHIPS_TAGS: "KLAVIYO_GET_TAG_GROUP_RELATIONSHIPS_TAGS",
    GET_TAG_GROUP_TAGS: "KLAVIYO_GET_TAG_GROUP_TAGS",
    GET_TAG_RELATIONSHIPS_CAMPAIGNS: "KLAVIYO_GET_TAG_RELATIONSHIPS_CAMPAIGNS",
    GET_TAG_RELATIONSHIPS_FLOWS: "KLAVIYO_GET_TAG_RELATIONSHIPS_FLOWS",
    GET_TAG_RELATIONSHIPS_LISTS: "KLAVIYO_GET_TAG_RELATIONSHIPS_LISTS",
    GET_TAG_RELATIONSHIPS_SEGMENTS: "KLAVIYO_GET_TAG_RELATIONSHIPS_SEGMENTS",
    GET_TAG_RELATIONSHIPS_TAG_GROUP: "KLAVIYO_GET_TAG_RELATIONSHIPS_TAG_GROUP",
    GET_TAG_TAG_GROUP: "KLAVIYO_GET_TAG_TAG_GROUP",
    GET_TEMPLATE: "KLAVIYO_GET_TEMPLATE",
    GET_TEMPLATES: "KLAVIYO_GET_TEMPLATES",
    GET_UPDATE_CATEGORIES_JOB: "KLAVIYO_GET_UPDATE_CATEGORIES_JOB",
    GET_UPDATE_CATEGORIES_JOBS: "KLAVIYO_GET_UPDATE_CATEGORIES_JOBS",
    GET_UPDATE_ITEMS_JOB: "KLAVIYO_GET_UPDATE_ITEMS_JOB",
    GET_UPDATE_ITEMS_JOBS: "KLAVIYO_GET_UPDATE_ITEMS_JOBS",
    GET_UPDATE_VARIANTS_JOB: "KLAVIYO_GET_UPDATE_VARIANTS_JOB",
    GET_UPDATE_VARIANTS_JOBS: "KLAVIYO_GET_UPDATE_VARIANTS_JOBS",
    GET_VERSIONS_FOR_FORM: "KLAVIYO_GET_VERSIONS_FOR_FORM",
    GET_VERSION_IDS_FOR_FORM: "KLAVIYO_GET_VERSION_IDS_FOR_FORM",
    GET_WEBHOOK: "KLAVIYO_GET_WEBHOOK",
    GET_WEBHOOKS: "KLAVIYO_GET_WEBHOOKS",
    GET_WEBHOOK_TOPIC: "KLAVIYO_GET_WEBHOOK_TOPIC",
    GET_WEBHOOK_TOPICS: "KLAVIYO_GET_WEBHOOK_TOPICS",
    MERGE_PROFILES: "KLAVIYO_MERGE_PROFILES",
    QUERY_CAMPAIGN_VALUES: "KLAVIYO_QUERY_CAMPAIGN_VALUES",
    QUERY_FLOW_SERIES: "KLAVIYO_QUERY_FLOW_SERIES",
    QUERY_FLOW_VALUES: "KLAVIYO_QUERY_FLOW_VALUES",
    QUERY_METRIC_AGGREGATES: "KLAVIYO_QUERY_METRIC_AGGREGATES",
    REMOVE_PROFILE_FROM_LIST: "KLAVIYO_REMOVE_PROFILE_FROM_LIST",
    REQUEST_PROFILE_DELETION: "KLAVIYO_REQUEST_PROFILE_DELETION",
    SPAWN_BULK_PROFILE_IMPORT_JOB: "KLAVIYO_SPAWN_BULK_PROFILE_IMPORT_JOB",
    SPAWN_COUPON_CODE_BULK_CREATE_JOB: "KLAVIYO_SPAWN_COUPON_CODE_BULK_CREATE_JOB",
    SPAWN_CREATE_CATEGORIES_JOB: "KLAVIYO_SPAWN_CREATE_CATEGORIES_JOB",
    SPAWN_CREATE_ITEMS_JOB: "KLAVIYO_SPAWN_CREATE_ITEMS_JOB",
    SPAWN_CREATE_VARIANTS_JOB: "KLAVIYO_SPAWN_CREATE_VARIANTS_JOB",
    SPAWN_DELETE_CATEGORIES_JOB: "KLAVIYO_SPAWN_DELETE_CATEGORIES_JOB",
    SPAWN_DELETE_ITEMS_JOB: "KLAVIYO_SPAWN_DELETE_ITEMS_JOB",
    SPAWN_DELETE_VARIANTS_JOB: "KLAVIYO_SPAWN_DELETE_VARIANTS_JOB",
    SPAWN_UPDATE_CATEGORIES_JOB: "KLAVIYO_SPAWN_UPDATE_CATEGORIES_JOB",
    SPAWN_UPDATE_ITEMS_JOB: "KLAVIYO_SPAWN_UPDATE_ITEMS_JOB",
    SPAWN_UPDATE_VARIANTS_JOB: "KLAVIYO_SPAWN_UPDATE_VARIANTS_JOB",
    SUBSCRIBE_PROFILES: "KLAVIYO_SUBSCRIBE_PROFILES",
    SUPPRESS_PROFILES: "KLAVIYO_SUPPRESS_PROFILES",
    UNREGISTER_CLIENT_PUSH_TOKEN: "KLAVIYO_UNREGISTER_CLIENT_PUSH_TOKEN",
    UNSUBSCRIBE_PROFILES: "KLAVIYO_UNSUBSCRIBE_PROFILES",
    UNSUPPRESS_PROFILES: "KLAVIYO_UNSUPPRESS_PROFILES",
    UPDATE_CAMPAIGN: "KLAVIYO_UPDATE_CAMPAIGN",
    UPDATE_CAMPAIGN_MESSAGE: "KLAVIYO_UPDATE_CAMPAIGN_MESSAGE",
    UPDATE_CAMPAIGN_SEND_JOB: "KLAVIYO_UPDATE_CAMPAIGN_SEND_JOB",
    UPDATE_CATALOG_CATEGORY: "KLAVIYO_UPDATE_CATALOG_CATEGORY",
    UPDATE_CATALOG_CATEGORY_RELATIONSHIPS_ITEMS: "KLAVIYO_UPDATE_CATALOG_CATEGORY_RELATIONSHIPS_ITEMS",
    UPDATE_CATALOG_ITEM: "KLAVIYO_UPDATE_CATALOG_ITEM",
    UPDATE_CATALOG_ITEM_RELATIONSHIPS_CATEGORIES: "KLAVIYO_UPDATE_CATALOG_ITEM_RELATIONSHIPS_CATEGORIES",
    UPDATE_CATALOG_VARIANT: "KLAVIYO_UPDATE_CATALOG_VARIANT",
    UPDATE_COUPON: "KLAVIYO_UPDATE_COUPON",
    UPDATE_COUPON_CODE: "KLAVIYO_UPDATE_COUPON_CODE",
    UPDATE_FLOW_STATUS: "KLAVIYO_UPDATE_FLOW_STATUS",
    UPDATE_IMAGE: "KLAVIYO_UPDATE_IMAGE",
    UPDATE_LIST: "KLAVIYO_UPDATE_LIST",
    UPDATE_PROFILE: "KLAVIYO_UPDATE_PROFILE",
    UPDATE_SEGMENT: "KLAVIYO_UPDATE_SEGMENT",
    UPDATE_TAG: "KLAVIYO_UPDATE_TAG",
    UPDATE_TAG_GROUP: "KLAVIYO_UPDATE_TAG_GROUP",
    UPDATE_TEMPLATE: "KLAVIYO_UPDATE_TEMPLATE",
    UPDATE_WEBHOOK: "KLAVIYO_UPDATE_WEBHOOK",
    UPLOAD_IMAGE_FROM_FILE: "KLAVIYO_UPLOAD_IMAGE_FROM_FILE",
    UPLOAD_IMAGE_FROM_URL: "KLAVIYO_UPLOAD_IMAGE_FROM_URL",
  },
  triggerTypes: {},
}
