// @composio/core/generated/linear.ts
/**
 * Map of Composio's LINEAR toolkit.
 */
export const LINEAR = {
  slug: "linear",
  tools: {
    CREATE_LINEAR_ATTACHMENT: "LINEAR_CREATE_LINEAR_ATTACHMENT",
    CREATE_LINEAR_COMMENT: "LINEAR_CREATE_LINEAR_COMMENT",
    CREATE_LINEAR_ISSUE: "LINEAR_CREATE_LINEAR_ISSUE",
    CREATE_LINEAR_ISSUE_DETAILS: "LINEAR_CREATE_LINEAR_ISSUE_DETAILS",
    CREATE_LINEAR_LABEL: "LINEAR_CREATE_LINEAR_LABEL",
    DELETE_LINEAR_ISSUE: "LINEAR_DELETE_LINEAR_ISSUE",
    GET_ATTACHMENTS: "LINEAR_GET_ATTACHMENTS",
    GET_CYCLES_BY_TEAM_ID: "LINEAR_GET_CYCLES_BY_TEAM_ID",
    GET_LINEAR_ISSUE: "LINEAR_GET_LINEAR_ISSUE",
    LIST_LINEAR_CYCLES: "LINEAR_LIST_LINEAR_CYCLES",
    LIST_LINEAR_ISSUES: "LINEAR_LIST_LINEAR_ISSUES",
    LIST_LINEAR_LABELS: "LINEAR_LIST_LINEAR_LABELS",
    LIST_LINEAR_PROJECTS: "LINEAR_LIST_LINEAR_PROJECTS",
    LIST_LINEAR_STATES: "LINEAR_LIST_LINEAR_STATES",
    LIST_LINEAR_TEAMS: "LINEAR_LIST_LINEAR_TEAMS",
    REMOVE_ISSUE_LABEL: "LINEAR_REMOVE_ISSUE_LABEL",
    RUN_QUERY_OR_MUTATION: "LINEAR_RUN_QUERY_OR_MUTATION",
    UPDATE_ISSUE: "LINEAR_UPDATE_ISSUE",
  },
  triggerTypes: {
    COMMENT_EVENT_TRIGGER: "LINEAR_COMMENT_EVENT_TRIGGER",
    ISSUE_CREATED_TRIGGER: "LINEAR_ISSUE_CREATED_TRIGGER",
    ISSUE_UPDATED_TRIGGER: "LINEAR_ISSUE_UPDATED_TRIGGER",
  },
}
