// @composio/core/generated/listennotes.ts
/**
 * Map of Composio's LISTENNOTES toolkit.
 */
export const LISTENNOTES = {
  slug: "listennotes",
  tools: {
    BATCH_FETCH_BASIC_META_DATA_FOR_EPISODES: "LISTENNOTES_BATCH_FETCH_BASIC_META_DATA_FOR_EPISODES",
    BATCH_FETCH_BASIC_META_DATA_FOR_PODCASTS: "LISTENNOTES_BATCH_FETCH_BASIC_META_DATA_FOR_PODCASTS",
    FETCH_AUDIENCE_DEMOGRAPHICS_FOR_A_PODCAST: "LISTENNOTES_FETCH_AUDIENCE_DEMOGRAPHICS_FOR_A_PODCAST",
    FETCH_A_CURATED_LIST_OF_PODCASTS_BY_ID: "LISTENNOTES_FETCH_A_CURATED_LIST_OF_PODCASTS_BY_ID",
    FETCH_A_LIST_OF_BEST_PODCASTS_BY_GENRE: "LISTENNOTES_FETCH_A_LIST_OF_BEST_PODCASTS_BY_GENRE",
    FETCH_A_LIST_OF_PODCAST_GENRES: "LISTENNOTES_FETCH_A_LIST_OF_PODCAST_GENRES",
    FETCH_A_LIST_OF_YOUR_PLAYLISTS: "LISTENNOTES_FETCH_A_LIST_OF_YOUR_PLAYLISTS",
    FETCH_A_RANDOM_PODCAST_EPISODE: "LISTENNOTES_FETCH_A_RANDOM_PODCAST_EPISODE",
    FETCH_CURATED_LISTS_OF_PODCASTS: "LISTENNOTES_FETCH_CURATED_LISTS_OF_PODCASTS",
    FETCH_DETAILED_META_DATA_FOR_AN_EPISODE_BY_ID: "LISTENNOTES_FETCH_DETAILED_META_DATA_FOR_AN_EPISODE_BY_ID",
    FETCH_PLAYLIST_INFO: "LISTENNOTES_FETCH_PLAYLIST_INFO",
    FETCH_PODCASTS_BY_A_PUBLISHER_S_DOMAIN_NAME: "LISTENNOTES_FETCH_PODCASTS_BY_A_PUBLISHER_S_DOMAIN_NAME",
    FETCH_PODCAST_DETAILS_AND_EPISODES: "LISTENNOTES_FETCH_PODCAST_DETAILS_AND_EPISODES",
    FETCH_PODCAST_LANGUAGES: "LISTENNOTES_FETCH_PODCAST_LANGUAGES",
    FETCH_RECOMMENDATIONS_FOR_AN_EPISODE: "LISTENNOTES_FETCH_RECOMMENDATIONS_FOR_AN_EPISODE",
    FETCH_RECOMMENDATIONS_FOR_A_PODCAST: "LISTENNOTES_FETCH_RECOMMENDATIONS_FOR_A_PODCAST",
    FETCH_RELATED_SEARCH_TERMS: "LISTENNOTES_FETCH_RELATED_SEARCH_TERMS",
    FETCH_SUPPORTED_REGIONS: "LISTENNOTES_FETCH_SUPPORTED_REGIONS",
    FETCH_TRENDING_SEARCH_TERMS: "LISTENNOTES_FETCH_TRENDING_SEARCH_TERMS",
    FULL_TEXT_SEARCH: "LISTENNOTES_FULL_TEXT_SEARCH",
    REFRESH_RSS_FEED_OF_A_PODCAST: "LISTENNOTES_REFRESH_RSS_FEED_OF_A_PODCAST",
    REQUEST_TO_DELETE_A_PODCAST: "LISTENNOTES_REQUEST_TO_DELETE_A_PODCAST",
    SEARCH_EPISODE_TITLES: "LISTENNOTES_SEARCH_EPISODE_TITLES",
    SPELL_CHECK_ON_A_SEARCH_TERM: "LISTENNOTES_SPELL_CHECK_ON_A_SEARCH_TERM",
    SUBMIT_A_PODCAST_TO_LISTEN_NOTES_DATABASE: "LISTENNOTES_SUBMIT_A_PODCAST_TO_LISTEN_NOTES_DATABASE",
    TYPEAHEAD_SEARCH: "LISTENNOTES_TYPEAHEAD_SEARCH",
  },
  triggerTypes: {},
}
