// @composio/core/generated/mailchimp.ts
/**
 * Map of Composio's MAILCHIMP toolkit.
 */
export const MAILCHIMP = {
  slug: "mailchimp",
  tools: {
    ADD_AUTOMATION: "MAILCHIMP_ADD_AUTOMATION",
    ADD_BATCH_WEBHOOK: "MAILCHIMP_ADD_BATCH_WEBHOOK",
    ADD_CAMPAIGN: "MAILCHIMP_ADD_CAMPAIGN",
    ADD_CAMPAIGN_FEEDBACK: "MAILCHIMP_ADD_CAMPAIGN_FEEDBACK",
    ADD_CAMPAIGN_FOLDER: "MAILCHIMP_ADD_CAMPAIGN_FOLDER",
    ADD_CART: "MAILCHIMP_ADD_CART",
    ADD_CART_LINE_ITEM: "MAILCHIMP_ADD_CART_LINE_ITEM",
    ADD_CONNECTED_SITE: "MAILCHIMP_ADD_CONNECTED_SITE",
    ADD_CUSTOMER: "MAILCHIMP_ADD_CUSTOMER",
    ADD_DOMAIN_TO_ACCOUNT: "MAILCHIMP_ADD_DOMAIN_TO_ACCOUNT",
    ADD_EVENT: "MAILCHIMP_ADD_EVENT",
    ADD_EXPORT: "MAILCHIMP_ADD_EXPORT",
    ADD_FILE: "MAILCHIMP_ADD_FILE",
    ADD_FOLDER: "MAILCHIMP_ADD_FOLDER",
    ADD_INTEREST_CATEGORY: "MAILCHIMP_ADD_INTEREST_CATEGORY",
    ADD_INTEREST_IN_CATEGORY: "MAILCHIMP_ADD_INTEREST_IN_CATEGORY",
    ADD_LANDING_PAGE: "MAILCHIMP_ADD_LANDING_PAGE",
    ADD_LIST: "MAILCHIMP_ADD_LIST",
    ADD_MEMBER_NOTE: "MAILCHIMP_ADD_MEMBER_NOTE",
    ADD_MEMBER_TO_LIST: "MAILCHIMP_ADD_MEMBER_TO_LIST",
    ADD_MEMBER_TO_SEGMENT: "MAILCHIMP_ADD_MEMBER_TO_SEGMENT",
    ADD_MERGE_FIELD: "MAILCHIMP_ADD_MERGE_FIELD",
    ADD_ORDER: "MAILCHIMP_ADD_ORDER",
    ADD_ORDER_LINE_ITEM: "MAILCHIMP_ADD_ORDER_LINE_ITEM",
    ADD_OR_REMOVE_MEMBER_TAGS: "MAILCHIMP_ADD_OR_REMOVE_MEMBER_TAGS",
    ADD_OR_UPDATE_CUSTOMER: "MAILCHIMP_ADD_OR_UPDATE_CUSTOMER",
    ADD_OR_UPDATE_LIST_MEMBER: "MAILCHIMP_ADD_OR_UPDATE_LIST_MEMBER",
    ADD_OR_UPDATE_PRODUCT_VARIANT: "MAILCHIMP_ADD_OR_UPDATE_PRODUCT_VARIANT",
    ADD_PRODUCT: "MAILCHIMP_ADD_PRODUCT",
    ADD_PRODUCT_IMAGE: "MAILCHIMP_ADD_PRODUCT_IMAGE",
    ADD_PRODUCT_VARIANT: "MAILCHIMP_ADD_PRODUCT_VARIANT",
    ADD_PROMO_CODE: "MAILCHIMP_ADD_PROMO_CODE",
    ADD_PROMO_RULE: "MAILCHIMP_ADD_PROMO_RULE",
    ADD_SEGMENT: "MAILCHIMP_ADD_SEGMENT",
    ADD_STORE: "MAILCHIMP_ADD_STORE",
    ADD_SUBSCRIBER_TO_WORKFLOW_EMAIL: "MAILCHIMP_ADD_SUBSCRIBER_TO_WORKFLOW_EMAIL",
    ADD_TEMPLATE: "MAILCHIMP_ADD_TEMPLATE",
    ADD_TEMPLATE_FOLDER: "MAILCHIMP_ADD_TEMPLATE_FOLDER",
    ADD_WEBHOOK: "MAILCHIMP_ADD_WEBHOOK",
    ARCHIVE_AUTOMATION: "MAILCHIMP_ARCHIVE_AUTOMATION",
    ARCHIVE_LIST_MEMBER: "MAILCHIMP_ARCHIVE_LIST_MEMBER",
    BATCH_ADD_OR_REMOVE_MEMBERS: "MAILCHIMP_BATCH_ADD_OR_REMOVE_MEMBERS",
    BATCH_SUBSCRIBE_OR_UNSUBSCRIBE: "MAILCHIMP_BATCH_SUBSCRIBE_OR_UNSUBSCRIBE",
    CAMPAIGN_ABUSE_REPORT_DETAILS: "MAILCHIMP_CAMPAIGN_ABUSE_REPORT_DETAILS",
    CAMPAIGN_STATISTICS_FEEDBACK: "MAILCHIMP_CAMPAIGN_STATISTICS_FEEDBACK",
    CANCEL_CAMPAIGN: "MAILCHIMP_CANCEL_CAMPAIGN",
    CREATE_A_SURVEY_CAMPAIGN: "MAILCHIMP_CREATE_A_SURVEY_CAMPAIGN",
    CUSTOMER_JOURNEYS_API_TRIGGER_FOR_A_CONTACT: "MAILCHIMP_CUSTOMER_JOURNEYS_API_TRIGGER_FOR_A_CONTACT",
    CUSTOMIZE_SIGNUP_FORM: "MAILCHIMP_CUSTOMIZE_SIGNUP_FORM",
    DELETE_BATCH_REQUEST: "MAILCHIMP_DELETE_BATCH_REQUEST",
    DELETE_BATCH_WEBHOOK: "MAILCHIMP_DELETE_BATCH_WEBHOOK",
    DELETE_CAMPAIGN: "MAILCHIMP_DELETE_CAMPAIGN",
    DELETE_CAMPAIGN_FEEDBACK_MESSAGE: "MAILCHIMP_DELETE_CAMPAIGN_FEEDBACK_MESSAGE",
    DELETE_CAMPAIGN_FOLDER: "MAILCHIMP_DELETE_CAMPAIGN_FOLDER",
    DELETE_CART: "MAILCHIMP_DELETE_CART",
    DELETE_CART_LINE_ITEM: "MAILCHIMP_DELETE_CART_LINE_ITEM",
    DELETE_CONNECTED_SITE: "MAILCHIMP_DELETE_CONNECTED_SITE",
    DELETE_CUSTOMER: "MAILCHIMP_DELETE_CUSTOMER",
    DELETE_DOMAIN: "MAILCHIMP_DELETE_DOMAIN",
    DELETE_FILE: "MAILCHIMP_DELETE_FILE",
    DELETE_FOLDER: "MAILCHIMP_DELETE_FOLDER",
    DELETE_INTEREST_CATEGORY: "MAILCHIMP_DELETE_INTEREST_CATEGORY",
    DELETE_INTEREST_IN_CATEGORY: "MAILCHIMP_DELETE_INTEREST_IN_CATEGORY",
    DELETE_LANDING_PAGE: "MAILCHIMP_DELETE_LANDING_PAGE",
    DELETE_LIST: "MAILCHIMP_DELETE_LIST",
    DELETE_LIST_MEMBER: "MAILCHIMP_DELETE_LIST_MEMBER",
    DELETE_MERGE_FIELD: "MAILCHIMP_DELETE_MERGE_FIELD",
    DELETE_NOTE: "MAILCHIMP_DELETE_NOTE",
    DELETE_ORDER: "MAILCHIMP_DELETE_ORDER",
    DELETE_ORDER_LINE_ITEM: "MAILCHIMP_DELETE_ORDER_LINE_ITEM",
    DELETE_PRODUCT: "MAILCHIMP_DELETE_PRODUCT",
    DELETE_PRODUCT_IMAGE: "MAILCHIMP_DELETE_PRODUCT_IMAGE",
    DELETE_PRODUCT_VARIANT: "MAILCHIMP_DELETE_PRODUCT_VARIANT",
    DELETE_PROMO_CODE: "MAILCHIMP_DELETE_PROMO_CODE",
    DELETE_PROMO_RULE: "MAILCHIMP_DELETE_PROMO_RULE",
    DELETE_SEGMENT: "MAILCHIMP_DELETE_SEGMENT",
    DELETE_STORE: "MAILCHIMP_DELETE_STORE",
    DELETE_TEMPLATE: "MAILCHIMP_DELETE_TEMPLATE",
    DELETE_TEMPLATE_FOLDER: "MAILCHIMP_DELETE_TEMPLATE_FOLDER",
    DELETE_WEBHOOK: "MAILCHIMP_DELETE_WEBHOOK",
    DELETE_WORKFLOW_EMAIL: "MAILCHIMP_DELETE_WORKFLOW_EMAIL",
    GET_ABUSE_REPORT: "MAILCHIMP_GET_ABUSE_REPORT",
    GET_ACCOUNT_EXPORT_INFO: "MAILCHIMP_GET_ACCOUNT_EXPORT_INFO",
    GET_AUTHORIZED_APP_INFO: "MAILCHIMP_GET_AUTHORIZED_APP_INFO",
    GET_AUTOMATED_EMAIL_SUBSCRIBER: "MAILCHIMP_GET_AUTOMATED_EMAIL_SUBSCRIBER",
    GET_AUTOMATION_INFO: "MAILCHIMP_GET_AUTOMATION_INFO",
    GET_BATCH_OPERATION_STATUS: "MAILCHIMP_GET_BATCH_OPERATION_STATUS",
    GET_BATCH_WEBHOOK_INFO: "MAILCHIMP_GET_BATCH_WEBHOOK_INFO",
    GET_CAMPAIGN_CONTENT: "MAILCHIMP_GET_CAMPAIGN_CONTENT",
    GET_CAMPAIGN_FEEDBACK_MESSAGE: "MAILCHIMP_GET_CAMPAIGN_FEEDBACK_MESSAGE",
    GET_CAMPAIGN_FOLDER: "MAILCHIMP_GET_CAMPAIGN_FOLDER",
    GET_CAMPAIGN_INFO: "MAILCHIMP_GET_CAMPAIGN_INFO",
    GET_CAMPAIGN_LINK_DETAILS: "MAILCHIMP_GET_CAMPAIGN_LINK_DETAILS",
    GET_CAMPAIGN_RECIPIENT_INFO: "MAILCHIMP_GET_CAMPAIGN_RECIPIENT_INFO",
    GET_CAMPAIGN_REPORT: "MAILCHIMP_GET_CAMPAIGN_REPORT",
    GET_CAMPAIGN_SEND_CHECKLIST: "MAILCHIMP_GET_CAMPAIGN_SEND_CHECKLIST",
    GET_CART_INFO: "MAILCHIMP_GET_CART_INFO",
    GET_CART_LINE_ITEM: "MAILCHIMP_GET_CART_LINE_ITEM",
    GET_CLICKED_LINK_SUBSCRIBER: "MAILCHIMP_GET_CLICKED_LINK_SUBSCRIBER",
    GET_CONNECTED_SITE: "MAILCHIMP_GET_CONNECTED_SITE",
    GET_CONVERSATION: "MAILCHIMP_GET_CONVERSATION",
    GET_CUSTOMER_INFO: "MAILCHIMP_GET_CUSTOMER_INFO",
    GET_DOMAIN_INFO: "MAILCHIMP_GET_DOMAIN_INFO",
    GET_FACEBOOK_AD_INFO: "MAILCHIMP_GET_FACEBOOK_AD_INFO",
    GET_FACEBOOK_AD_REPORT: "MAILCHIMP_GET_FACEBOOK_AD_REPORT",
    GET_FILE: "MAILCHIMP_GET_FILE",
    GET_FOLDER: "MAILCHIMP_GET_FOLDER",
    GET_GROWTH_HISTORY_BY_MONTH: "MAILCHIMP_GET_GROWTH_HISTORY_BY_MONTH",
    GET_INFORMATION_ABOUT_ALL_SURVEYS_FOR_A_LIST: "MAILCHIMP_GET_INFORMATION_ABOUT_ALL_SURVEYS_FOR_A_LIST",
    GET_INTEREST_CATEGORY_INFO: "MAILCHIMP_GET_INTEREST_CATEGORY_INFO",
    GET_INTEREST_IN_CATEGORY: "MAILCHIMP_GET_INTEREST_IN_CATEGORY",
    GET_LANDING_PAGE_CONTENT: "MAILCHIMP_GET_LANDING_PAGE_CONTENT",
    GET_LANDING_PAGE_INFO: "MAILCHIMP_GET_LANDING_PAGE_INFO",
    GET_LANDING_PAGE_REPORT: "MAILCHIMP_GET_LANDING_PAGE_REPORT",
    GET_LATEST_CHIMP_CHATTER: "MAILCHIMP_GET_LATEST_CHIMP_CHATTER",
    GET_LISTS_INFO: "MAILCHIMP_GET_LISTS_INFO",
    GET_LIST_INFO: "MAILCHIMP_GET_LIST_INFO",
    GET_MEMBER_INFO: "MAILCHIMP_GET_MEMBER_INFO",
    GET_MEMBER_NOTE: "MAILCHIMP_GET_MEMBER_NOTE",
    GET_MERGE_FIELD: "MAILCHIMP_GET_MERGE_FIELD",
    GET_MESSAGE: "MAILCHIMP_GET_MESSAGE",
    GET_OPENED_CAMPAIGN_SUBSCRIBER: "MAILCHIMP_GET_OPENED_CAMPAIGN_SUBSCRIBER",
    GET_ORDER_INFO: "MAILCHIMP_GET_ORDER_INFO",
    GET_ORDER_LINE_ITEM: "MAILCHIMP_GET_ORDER_LINE_ITEM",
    GET_PRODUCT_IMAGE_INFO: "MAILCHIMP_GET_PRODUCT_IMAGE_INFO",
    GET_PRODUCT_INFO: "MAILCHIMP_GET_PRODUCT_INFO",
    GET_PRODUCT_VARIANT_INFO: "MAILCHIMP_GET_PRODUCT_VARIANT_INFO",
    GET_PROMO_CODE: "MAILCHIMP_GET_PROMO_CODE",
    GET_PROMO_RULE: "MAILCHIMP_GET_PROMO_RULE",
    GET_SEGMENT_INFO: "MAILCHIMP_GET_SEGMENT_INFO",
    GET_STORE_INFO: "MAILCHIMP_GET_STORE_INFO",
    GET_SUBSCRIBER_EMAIL_ACTIVITY: "MAILCHIMP_GET_SUBSCRIBER_EMAIL_ACTIVITY",
    GET_SUBSCRIBER_REMOVED_FROM_WORKFLOW: "MAILCHIMP_GET_SUBSCRIBER_REMOVED_FROM_WORKFLOW",
    GET_SURVEY: "MAILCHIMP_GET_SURVEY",
    GET_SURVEY_QUESTION_REPORT: "MAILCHIMP_GET_SURVEY_QUESTION_REPORT",
    GET_SURVEY_REPORT: "MAILCHIMP_GET_SURVEY_REPORT",
    GET_SURVEY_RESPONSE: "MAILCHIMP_GET_SURVEY_RESPONSE",
    GET_TEMPLATE_FOLDER: "MAILCHIMP_GET_TEMPLATE_FOLDER",
    GET_TEMPLATE_INFO: "MAILCHIMP_GET_TEMPLATE_INFO",
    GET_UNSUBSCRIBED_MEMBER: "MAILCHIMP_GET_UNSUBSCRIBED_MEMBER",
    GET_WEBHOOK_INFO: "MAILCHIMP_GET_WEBHOOK_INFO",
    GET_WORKFLOW_EMAIL_INFO: "MAILCHIMP_GET_WORKFLOW_EMAIL_INFO",
    LIST_ABUSE_REPORTS: "MAILCHIMP_LIST_ABUSE_REPORTS",
    LIST_ACCOUNT_EXPORTS: "MAILCHIMP_LIST_ACCOUNT_EXPORTS",
    LIST_ACCOUNT_ORDERS: "MAILCHIMP_LIST_ACCOUNT_ORDERS",
    LIST_ANSWERS_FOR_QUESTION: "MAILCHIMP_LIST_ANSWERS_FOR_QUESTION",
    LIST_API_ROOT_RESOURCES: "MAILCHIMP_LIST_API_ROOT_RESOURCES",
    LIST_AUTHORIZED_APPS: "MAILCHIMP_LIST_AUTHORIZED_APPS",
    LIST_AUTOMATED_EMAILS: "MAILCHIMP_LIST_AUTOMATED_EMAILS",
    LIST_AUTOMATED_EMAIL_SUBSCRIBERS: "MAILCHIMP_LIST_AUTOMATED_EMAIL_SUBSCRIBERS",
    LIST_AUTOMATIONS: "MAILCHIMP_LIST_AUTOMATIONS",
    LIST_BATCH_REQUESTS: "MAILCHIMP_LIST_BATCH_REQUESTS",
    LIST_BATCH_WEBHOOKS: "MAILCHIMP_LIST_BATCH_WEBHOOKS",
    LIST_CAMPAIGNS: "MAILCHIMP_LIST_CAMPAIGNS",
    LIST_CAMPAIGN_DETAILS: "MAILCHIMP_LIST_CAMPAIGN_DETAILS",
    LIST_CAMPAIGN_FEEDBACK: "MAILCHIMP_LIST_CAMPAIGN_FEEDBACK",
    LIST_CAMPAIGN_FOLDERS: "MAILCHIMP_LIST_CAMPAIGN_FOLDERS",
    LIST_CAMPAIGN_OPEN_DETAILS: "MAILCHIMP_LIST_CAMPAIGN_OPEN_DETAILS",
    LIST_CAMPAIGN_PRODUCT_ACTIVITY: "MAILCHIMP_LIST_CAMPAIGN_PRODUCT_ACTIVITY",
    LIST_CAMPAIGN_RECIPIENTS: "MAILCHIMP_LIST_CAMPAIGN_RECIPIENTS",
    LIST_CAMPAIGN_REPORTS: "MAILCHIMP_LIST_CAMPAIGN_REPORTS",
    LIST_CARTS: "MAILCHIMP_LIST_CARTS",
    LIST_CART_LINE_ITEMS: "MAILCHIMP_LIST_CART_LINE_ITEMS",
    LIST_CHILD_CAMPAIGN_REPORTS: "MAILCHIMP_LIST_CHILD_CAMPAIGN_REPORTS",
    LIST_CLICKED_LINK_SUBSCRIBERS: "MAILCHIMP_LIST_CLICKED_LINK_SUBSCRIBERS",
    LIST_CONNECTED_SITES: "MAILCHIMP_LIST_CONNECTED_SITES",
    LIST_CONVERSATIONS: "MAILCHIMP_LIST_CONVERSATIONS",
    LIST_CUSTOMERS: "MAILCHIMP_LIST_CUSTOMERS",
    LIST_DOMAIN_PERFORMANCE_STATS: "MAILCHIMP_LIST_DOMAIN_PERFORMANCE_STATS",
    LIST_EEP_URL_ACTIVITY: "MAILCHIMP_LIST_EEP_URL_ACTIVITY",
    LIST_EMAIL_ACTIVITY: "MAILCHIMP_LIST_EMAIL_ACTIVITY",
    LIST_FACEBOOK_ADS: "MAILCHIMP_LIST_FACEBOOK_ADS",
    LIST_FACEBOOK_ADS_REPORTS: "MAILCHIMP_LIST_FACEBOOK_ADS_REPORTS",
    LIST_FACEBOOK_ECOMMERCE_REPORT: "MAILCHIMP_LIST_FACEBOOK_ECOMMERCE_REPORT",
    LIST_FOLDERS: "MAILCHIMP_LIST_FOLDERS",
    LIST_GROWTH_HISTORY_DATA: "MAILCHIMP_LIST_GROWTH_HISTORY_DATA",
    LIST_INTERESTS_IN_CATEGORY: "MAILCHIMP_LIST_INTERESTS_IN_CATEGORY",
    LIST_INTEREST_CATEGORIES: "MAILCHIMP_LIST_INTEREST_CATEGORIES",
    LIST_LANDING_PAGES: "MAILCHIMP_LIST_LANDING_PAGES",
    LIST_LANDING_PAGES_REPORTS: "MAILCHIMP_LIST_LANDING_PAGES_REPORTS",
    LIST_LOCATIONS: "MAILCHIMP_LIST_LOCATIONS",
    LIST_MEMBERS_INFO: "MAILCHIMP_LIST_MEMBERS_INFO",
    LIST_MEMBERS_IN_SEGMENT: "MAILCHIMP_LIST_MEMBERS_IN_SEGMENT",
    LIST_MEMBER_EVENTS: "MAILCHIMP_LIST_MEMBER_EVENTS",
    LIST_MEMBER_GOAL_EVENTS: "MAILCHIMP_LIST_MEMBER_GOAL_EVENTS",
    LIST_MEMBER_TAGS: "MAILCHIMP_LIST_MEMBER_TAGS",
    LIST_MERGE_FIELDS: "MAILCHIMP_LIST_MERGE_FIELDS",
    LIST_MESSAGES: "MAILCHIMP_LIST_MESSAGES",
    LIST_ORDERS: "MAILCHIMP_LIST_ORDERS",
    LIST_ORDER_LINE_ITEMS: "MAILCHIMP_LIST_ORDER_LINE_ITEMS",
    LIST_PRODUCT: "MAILCHIMP_LIST_PRODUCT",
    LIST_PRODUCT_IMAGES: "MAILCHIMP_LIST_PRODUCT_IMAGES",
    LIST_PRODUCT_VARIANTS: "MAILCHIMP_LIST_PRODUCT_VARIANTS",
    LIST_PROMO_CODES: "MAILCHIMP_LIST_PROMO_CODES",
    LIST_PROMO_RULES: "MAILCHIMP_LIST_PROMO_RULES",
    LIST_RECENT_ACTIVITY: "MAILCHIMP_LIST_RECENT_ACTIVITY",
    LIST_RECENT_MEMBER_NOTES: "MAILCHIMP_LIST_RECENT_MEMBER_NOTES",
    LIST_SEGMENTS: "MAILCHIMP_LIST_SEGMENTS",
    LIST_SENDING_DOMAINS: "MAILCHIMP_LIST_SENDING_DOMAINS",
    LIST_SIGNUP_FORMS: "MAILCHIMP_LIST_SIGNUP_FORMS",
    LIST_STORED_FILES: "MAILCHIMP_LIST_STORED_FILES",
    LIST_STORES: "MAILCHIMP_LIST_STORES",
    LIST_SUBSCRIBERS_REMOVED_FROM_WORKFLOW: "MAILCHIMP_LIST_SUBSCRIBERS_REMOVED_FROM_WORKFLOW",
    LIST_SURVEY_QUESTION_REPORTS: "MAILCHIMP_LIST_SURVEY_QUESTION_REPORTS",
    LIST_SURVEY_REPORTS: "MAILCHIMP_LIST_SURVEY_REPORTS",
    LIST_SURVEY_RESPONSES: "MAILCHIMP_LIST_SURVEY_RESPONSES",
    LIST_TEMPLATES: "MAILCHIMP_LIST_TEMPLATES",
    LIST_TEMPLATE_FOLDERS: "MAILCHIMP_LIST_TEMPLATE_FOLDERS",
    LIST_TOP_EMAIL_CLIENTS: "MAILCHIMP_LIST_TOP_EMAIL_CLIENTS",
    LIST_TOP_OPEN_ACTIVITIES: "MAILCHIMP_LIST_TOP_OPEN_ACTIVITIES",
    LIST_UNSUBSCRIBED_MEMBERS: "MAILCHIMP_LIST_UNSUBSCRIBED_MEMBERS",
    LIST_WEBHOOKS: "MAILCHIMP_LIST_WEBHOOKS",
    PAUSE_AUTOMATED_EMAIL: "MAILCHIMP_PAUSE_AUTOMATED_EMAIL",
    PAUSE_AUTOMATION_EMAILS: "MAILCHIMP_PAUSE_AUTOMATION_EMAILS",
    PAUSE_RSS_CAMPAIGN: "MAILCHIMP_PAUSE_RSS_CAMPAIGN",
    PING: "MAILCHIMP_PING",
    PUBLISH_A_SURVEY: "MAILCHIMP_PUBLISH_A_SURVEY",
    PUBLISH_LANDING_PAGE: "MAILCHIMP_PUBLISH_LANDING_PAGE",
    REMOVE_LIST_MEMBER_FROM_SEGMENT: "MAILCHIMP_REMOVE_LIST_MEMBER_FROM_SEGMENT",
    REMOVE_SUBSCRIBER_FROM_WORKFLOW: "MAILCHIMP_REMOVE_SUBSCRIBER_FROM_WORKFLOW",
    REPLICATE_CAMPAIGN: "MAILCHIMP_REPLICATE_CAMPAIGN",
    RESEND_CAMPAIGN: "MAILCHIMP_RESEND_CAMPAIGN",
    RESUME_RSS_CAMPAIGN: "MAILCHIMP_RESUME_RSS_CAMPAIGN",
    RETRIEVE_CAMPAIGN_ABUSE_COMPLAINTS: "MAILCHIMP_RETRIEVE_CAMPAIGN_ABUSE_COMPLAINTS",
    RETRIEVE_FOLDER_CONTENTS: "MAILCHIMP_RETRIEVE_FOLDER_CONTENTS",
    SCHEDULE_CAMPAIGN: "MAILCHIMP_SCHEDULE_CAMPAIGN",
    SEARCH_CAMPAIGNS: "MAILCHIMP_SEARCH_CAMPAIGNS",
    SEARCH_FOR_TAGS_ON_A_LIST_BY_NAME: "MAILCHIMP_SEARCH_FOR_TAGS_ON_A_LIST_BY_NAME",
    SEARCH_MEMBERS: "MAILCHIMP_SEARCH_MEMBERS",
    SEND_CAMPAIGN: "MAILCHIMP_SEND_CAMPAIGN",
    SEND_TEST_EMAIL: "MAILCHIMP_SEND_TEST_EMAIL",
    SET_CAMPAIGN_CONTENT: "MAILCHIMP_SET_CAMPAIGN_CONTENT",
    START_AUTOMATED_EMAIL: "MAILCHIMP_START_AUTOMATED_EMAIL",
    START_AUTOMATION_EMAILS: "MAILCHIMP_START_AUTOMATION_EMAILS",
    START_BATCH_OPERATION: "MAILCHIMP_START_BATCH_OPERATION",
    UNPUBLISH_A_SURVEY: "MAILCHIMP_UNPUBLISH_A_SURVEY",
    UNPUBLISH_LANDING_PAGE: "MAILCHIMP_UNPUBLISH_LANDING_PAGE",
    UNSCHEDULE_CAMPAIGN: "MAILCHIMP_UNSCHEDULE_CAMPAIGN",
    UPDATE_BATCH_WEBHOOK: "MAILCHIMP_UPDATE_BATCH_WEBHOOK",
    UPDATE_CAMPAIGN_FEEDBACK_MESSAGE: "MAILCHIMP_UPDATE_CAMPAIGN_FEEDBACK_MESSAGE",
    UPDATE_CAMPAIGN_FOLDER: "MAILCHIMP_UPDATE_CAMPAIGN_FOLDER",
    UPDATE_CAMPAIGN_SETTINGS: "MAILCHIMP_UPDATE_CAMPAIGN_SETTINGS",
    UPDATE_CART: "MAILCHIMP_UPDATE_CART",
    UPDATE_CART_LINE_ITEM: "MAILCHIMP_UPDATE_CART_LINE_ITEM",
    UPDATE_CUSTOMER: "MAILCHIMP_UPDATE_CUSTOMER",
    UPDATE_FILE: "MAILCHIMP_UPDATE_FILE",
    UPDATE_FOLDER: "MAILCHIMP_UPDATE_FOLDER",
    UPDATE_INTEREST_CATEGORY: "MAILCHIMP_UPDATE_INTEREST_CATEGORY",
    UPDATE_INTEREST_IN_CATEGORY: "MAILCHIMP_UPDATE_INTEREST_IN_CATEGORY",
    UPDATE_LANDING_PAGE: "MAILCHIMP_UPDATE_LANDING_PAGE",
    UPDATE_LISTS: "MAILCHIMP_UPDATE_LISTS",
    UPDATE_LIST_MEMBER: "MAILCHIMP_UPDATE_LIST_MEMBER",
    UPDATE_MERGE_FIELD: "MAILCHIMP_UPDATE_MERGE_FIELD",
    UPDATE_NOTE: "MAILCHIMP_UPDATE_NOTE",
    UPDATE_ORDER: "MAILCHIMP_UPDATE_ORDER",
    UPDATE_ORDER_LINE_ITEM: "MAILCHIMP_UPDATE_ORDER_LINE_ITEM",
    UPDATE_PRODUCT: "MAILCHIMP_UPDATE_PRODUCT",
    UPDATE_PRODUCT_IMAGE: "MAILCHIMP_UPDATE_PRODUCT_IMAGE",
    UPDATE_PRODUCT_VARIANT: "MAILCHIMP_UPDATE_PRODUCT_VARIANT",
    UPDATE_PROMO_CODE: "MAILCHIMP_UPDATE_PROMO_CODE",
    UPDATE_PROMO_RULE: "MAILCHIMP_UPDATE_PROMO_RULE",
    UPDATE_STORE: "MAILCHIMP_UPDATE_STORE",
    UPDATE_TEMPLATE: "MAILCHIMP_UPDATE_TEMPLATE",
    UPDATE_TEMPLATE_FOLDER: "MAILCHIMP_UPDATE_TEMPLATE_FOLDER",
    UPDATE_WEBHOOK: "MAILCHIMP_UPDATE_WEBHOOK",
    UPDATE_WORKFLOW_EMAIL: "MAILCHIMP_UPDATE_WORKFLOW_EMAIL",
    VERIFY_CONNECTED_SITE_SCRIPT: "MAILCHIMP_VERIFY_CONNECTED_SITE_SCRIPT",
    VERIFY_DOMAIN: "MAILCHIMP_VERIFY_DOMAIN",
    VIEW_DEFAULT_CONTENT: "MAILCHIMP_VIEW_DEFAULT_CONTENT",
    VIEW_RECENT_ACTIVITY: "MAILCHIMP_VIEW_RECENT_ACTIVITY",
    VIEW_RECENT_ACTIVITY_50: "MAILCHIMP_VIEW_RECENT_ACTIVITY_50",
  },
  triggerTypes: {
    MAILCHIMP_CAMPAIGN_TRIGGER: "MAILCHIMP_MAILCHIMP_CAMPAIGN_TRIGGER",
    MAILCHIMP_PROFILE_UPDATE_TRIGGER: "MAILCHIMP_MAILCHIMP_PROFILE_UPDATE_TRIGGER",
    MAILCHIMP_SUBSCRIBE_TRIGGER: "MAILCHIMP_MAILCHIMP_SUBSCRIBE_TRIGGER",
    MAILCHIMP_UNSUBSCRIBE_TRIGGER: "MAILCHIMP_MAILCHIMP_UNSUBSCRIBE_TRIGGER",
  },
}
