// @composio/core/generated/mem0.ts
/**
 * Map of Composio's MEM0 toolkit.
 */
export const MEM0 = {
  slug: "mem0",
  tools: {
    ADD_MEMBER_TO_PROJECT: "MEM0_ADD_MEMBER_TO_PROJECT",
    ADD_NEW_MEMORY_RECORDS: "MEM0_ADD_NEW_MEMORY_RECORDS",
    ADD_ORGANIZATION_MEMBER: "MEM0_ADD_ORGANIZATION_MEMBER",
    CREATE_AN_EXPORT_JOB_WITH_SCHEMA: "MEM0_CREATE_AN_EXPORT_JOB_WITH_SCHEMA",
    CREATE_A_NEW_AGENT: "MEM0_CREATE_A_NEW_AGENT",
    CREATE_A_NEW_AGENT_RUN: "MEM0_CREATE_A_NEW_AGENT_RUN",
    CREATE_A_NEW_APPLICATION: "MEM0_CREATE_A_NEW_APPLICATION",
    CREATE_A_NEW_ORGANIZATION_ENTRY: "MEM0_CREATE_A_NEW_ORGANIZATION_ENTRY",
    CREATE_A_NEW_USER: "MEM0_CREATE_A_NEW_USER",
    CREATE_MEMORY_ENTRY: "MEM0_CREATE_MEMORY_ENTRY",
    CREATE_PROJECT: "MEM0_CREATE_PROJECT",
    DELETE_AN_ORGANIZATION: "MEM0_DELETE_AN_ORGANIZATION",
    DELETE_A_SPECIFIC_MEMORY_BY_ID: "MEM0_DELETE_A_SPECIFIC_MEMORY_BY_ID",
    DELETE_ENTITY_BY_TYPE_AND_ID: "MEM0_DELETE_ENTITY_BY_TYPE_AND_ID",
    DELETE_MEMORIES_ENDPOINT: "MEM0_DELETE_MEMORIES_ENDPOINT",
    DELETE_MEMORY_BATCH_WITH_UUIDS: "MEM0_DELETE_MEMORY_BATCH_WITH_UUIDS",
    DELETE_PROJECT: "MEM0_DELETE_PROJECT",
    DELETE_PROJECT_MEMBER: "MEM0_DELETE_PROJECT_MEMBER",
    EXPORT_DATA_BASED_ON_FILTERS: "MEM0_EXPORT_DATA_BASED_ON_FILTERS",
    FETCH_DETAILED_LIST_OF_ORGANIZATIONS: "MEM0_FETCH_DETAILED_LIST_OF_ORGANIZATIONS",
    FETCH_DETAILS_OF_A_SPECIFIC_ORGANIZATION: "MEM0_FETCH_DETAILS_OF_A_SPECIFIC_ORGANIZATION",
    FETCH_LIST_OF_ENTITY_FILTERS: "MEM0_FETCH_LIST_OF_ENTITY_FILTERS",
    FETCH_SPECIFIC_ENTITY_DETAILS_WITH_OPTIONAL_FILTERS: "MEM0_FETCH_SPECIFIC_ENTITY_DETAILS_WITH_OPTIONAL_FILTERS",
    GET_ORGANIZATION_MEMBERS: "MEM0_GET_ORGANIZATION_MEMBERS",
    GET_PROJECTS: "MEM0_GET_PROJECTS",
    GET_PROJECT_DETAILS: "MEM0_GET_PROJECT_DETAILS",
    GET_PROJECT_MEMBERS: "MEM0_GET_PROJECT_MEMBERS",
    GET_USER_MEMORY_STATS: "MEM0_GET_USER_MEMORY_STATS",
    LIST_ENTITIES_WITH_OPTIONAL_ORG_AND_PROJECT_FILTERS: "MEM0_LIST_ENTITIES_WITH_OPTIONAL_ORG_AND_PROJECT_FILTERS",
    PERFORM_SEMANTIC_SEARCH_ON_MEMORIES: "MEM0_PERFORM_SEMANTIC_SEARCH_ON_MEMORIES",
    REMOVE_A_MEMBER_FROM_THE_ORGANIZATION: "MEM0_REMOVE_A_MEMBER_FROM_THE_ORGANIZATION",
    RETRIEVE_ALL_EVENTS_FOR_THE_CURRENTLY_LOGGED_IN_USER: "MEM0_RETRIEVE_ALL_EVENTS_FOR_THE_CURRENTLY_LOGGED_IN_USER",
    RETRIEVE_ENTITY_SPECIFIC_MEMORIES: "MEM0_RETRIEVE_ENTITY_SPECIFIC_MEMORIES",
    RETRIEVE_LIST_OF_MEMORY_EVENTS: "MEM0_RETRIEVE_LIST_OF_MEMORY_EVENTS",
    RETRIEVE_MEMORY_BY_UNIQUE_IDENTIFIER: "MEM0_RETRIEVE_MEMORY_BY_UNIQUE_IDENTIFIER",
    RETRIEVE_MEMORY_HISTORY_BY_ID: "MEM0_RETRIEVE_MEMORY_HISTORY_BY_ID",
    RETRIEVE_MEMORY_LIST: "MEM0_RETRIEVE_MEMORY_LIST",
    SEARCH_MEMORIES_WITH_QUERY_FILTERS: "MEM0_SEARCH_MEMORIES_WITH_QUERY_FILTERS",
    UPDATE_MEMORY_BATCH_WITH_UUID: "MEM0_UPDATE_MEMORY_BATCH_WITH_UUID",
    UPDATE_MEMORY_DETAILS_BY_ID: "MEM0_UPDATE_MEMORY_DETAILS_BY_ID",
    UPDATE_ORGANIZATION_MEMBER_ROLE: "MEM0_UPDATE_ORGANIZATION_MEMBER_ROLE",
    UPDATE_PROJECT: "MEM0_UPDATE_PROJECT",
    UPDATE_PROJECT_MEMBER_ROLE: "MEM0_UPDATE_PROJECT_MEMBER_ROLE",
  },
  triggerTypes: {},
}
