// @composio/core/generated/mixpanel.ts
/**
 * Map of Composio's MIXPANEL toolkit.
 */
export const MIXPANEL = {
  slug: "mixpanel",
  tools: {
    AGGREGATED_EVENT_PROPERTY_VALUES: "MIXPANEL_AGGREGATED_EVENT_PROPERTY_VALUES",
    AGGREGATE_EVENTS: "MIXPANEL_AGGREGATE_EVENTS",
    AGGREGATE_EVENT_COUNTS: "MIXPANEL_AGGREGATE_EVENT_COUNTS",
    GET_ALL_PROJECTS: "MIXPANEL_GET_ALL_PROJECTS",
    JQL_QUERY: "MIXPANEL_JQL_QUERY",
    LIST_COHORTS: "MIXPANEL_LIST_COHORTS",
    LIST_FUNNELS: "MIXPANEL_LIST_FUNNELS",
    PROFILE_EVENT_ACTIVITY: "MIXPANEL_PROFILE_EVENT_ACTIVITY",
    QUERY_FREQUENCY_REPORT: "MIXPANEL_QUERY_FREQUENCY_REPORT",
    QUERY_FUNNEL: "MIXPANEL_QUERY_FUNNEL",
    QUERY_INSIGHT: "MIXPANEL_QUERY_INSIGHT",
    QUERY_NUMERIC_AVERAGE: "MIXPANEL_QUERY_NUMERIC_AVERAGE",
    QUERY_NUMERIC_SEGMENTATION: "MIXPANEL_QUERY_NUMERIC_SEGMENTATION",
    QUERY_NUMERIC_SUM: "MIXPANEL_QUERY_NUMERIC_SUM",
    QUERY_PROFILES: "MIXPANEL_QUERY_PROFILES",
    QUERY_RETENTION_REPORT: "MIXPANEL_QUERY_RETENTION_REPORT",
    QUERY_SEGMENTATION: "MIXPANEL_QUERY_SEGMENTATION",
    TODAYS_TOP_EVENTS: "MIXPANEL_TODAYS_TOP_EVENTS",
    TOP_EVENTS: "MIXPANEL_TOP_EVENTS",
    TOP_EVENT_PROPERTIES: "MIXPANEL_TOP_EVENT_PROPERTIES",
    TOP_EVENT_PROPERTY_VALUES: "MIXPANEL_TOP_EVENT_PROPERTY_VALUES",
  },
  triggerTypes: {},
}
