// @composio/core/generated/monday.ts
/**
 * Map of Composio's MONDAY toolkit.
 */
export const MONDAY = {
  slug: "monday",
  tools: {
    ADD_USERS_TO_BOARD: "MONDAY_ADD_USERS_TO_BOARD",
    ARCHIVE_BOARD: "MONDAY_ARCHIVE_BOARD",
    ARCHIVE_ITEM: "MONDAY_ARCHIVE_ITEM",
    CHANGE_SIMPLE_COLUMN_VALUE: "MONDAY_CHANGE_SIMPLE_COLUMN_VALUE",
    CREATE_BOARD: "MONDAY_CREATE_BOARD",
    CREATE_COLUMN: "MONDAY_CREATE_COLUMN",
    CREATE_GROUP: "MONDAY_CREATE_GROUP",
    CREATE_ITEM: "MONDAY_CREATE_ITEM",
    DELETE_COLUMN: "MONDAY_DELETE_COLUMN",
    DELETE_GROUP: "MONDAY_DELETE_GROUP",
    DELETE_ITEM: "MONDAY_DELETE_ITEM",
    DUPLICATE_ITEM: "MONDAY_DUPLICATE_ITEM",
    GET_GROUP_DETAILS: "MONDAY_GET_GROUP_DETAILS",
    LIST_BOARDS: "MONDAY_LIST_BOARDS",
    LIST_BOARD_ITEMS: "MONDAY_LIST_BOARD_ITEMS",
    LIST_COLUMNS: "MONDAY_LIST_COLUMNS",
    LIST_ITEMS: "MONDAY_LIST_ITEMS",
    LIST_USERS: "MONDAY_LIST_USERS",
    MOVE_ITEM_TO_BOARD: "MONDAY_MOVE_ITEM_TO_BOARD",
    MOVE_ITEM_TO_GROUP: "MONDAY_MOVE_ITEM_TO_GROUP",
    UPDATE_BOARD: "MONDAY_UPDATE_BOARD",
  },
  triggerTypes: {},
}
