// @composio/core/generated/neon.ts
/**
 * Map of Composio's NEON toolkit.
 */
export const NEON = {
  slug: "neon",
  tools: {
    ACCESS_PROJECT_DETAILS_BY_ID: "NEON_ACCESS_PROJECT_DETAILS_BY_ID",
    ADD_NEW_JWKS_TO_PROJECT_ENDPOINT: "NEON_ADD_NEW_JWKS_TO_PROJECT_ENDPOINT",
    ADD_PROJECT_EMAIL_PERMISSION: "NEON_ADD_PROJECT_EMAIL_PERMISSION",
    ADD_ROLE_TO_BRANCH: "NEON_ADD_ROLE_TO_BRANCH",
    CREATE_API_KEY_FOR_ORGANIZATION: "NEON_CREATE_API_KEY_FOR_ORGANIZATION",
    CREATE_BRANCH_DATABASE: "NEON_CREATE_BRANCH_DATABASE",
    CREATE_COMPUTE_ENDPOINT: "NEON_CREATE_COMPUTE_ENDPOINT",
    CREATE_NEW_API_KEY: "NEON_CREATE_NEW_API_KEY",
    CREATE_NEW_PROJECT_BRANCH: "NEON_CREATE_NEW_PROJECT_BRANCH",
    CREATE_PROJECT_WITH_QUOTA_AND_SETTINGS: "NEON_CREATE_PROJECT_WITH_QUOTA_AND_SETTINGS",
    CREATE_VPC_ENDPOINT_LABEL: "NEON_CREATE_VPC_ENDPOINT_LABEL",
    CREATE_VPC_ENDPOINT_WITH_LABEL: "NEON_CREATE_VPC_ENDPOINT_WITH_LABEL",
    DELETE_API_KEY_BY_ID: "NEON_DELETE_API_KEY_BY_ID",
    DELETE_DATABASE_FROM_BRANCH: "NEON_DELETE_DATABASE_FROM_BRANCH",
    DELETE_ORGANIZATION_API_KEY: "NEON_DELETE_ORGANIZATION_API_KEY",
    DELETE_ORGANIZATION_MEMBER: "NEON_DELETE_ORGANIZATION_MEMBER",
    DELETE_PROJECT_BRANCH_BY_ID: "NEON_DELETE_PROJECT_BRANCH_BY_ID",
    DELETE_PROJECT_BRANCH_ROLE: "NEON_DELETE_PROJECT_BRANCH_ROLE",
    DELETE_PROJECT_BY_ID: "NEON_DELETE_PROJECT_BY_ID",
    DELETE_PROJECT_ENDPOINT: "NEON_DELETE_PROJECT_ENDPOINT",
    DELETE_PROJECT_JWKS_BY_ID: "NEON_DELETE_PROJECT_JWKS_BY_ID",
    DELETE_PROJECT_PERMISSION: "NEON_DELETE_PROJECT_PERMISSION",
    DELETE_VPC_ENDPOINT_BY_IDS: "NEON_DELETE_VPC_ENDPOINT_BY_IDS",
    DELETE_VPC_ENDPOINT_BY_PROJECT_ID: "NEON_DELETE_VPC_ENDPOINT_BY_PROJECT_ID",
    FETCH_DATABASE_FOR_BRANCH: "NEON_FETCH_DATABASE_FOR_BRANCH",
    FETCH_ORGANIZATION_MEMBERS_BY_ID: "NEON_FETCH_ORGANIZATION_MEMBERS_BY_ID",
    FETCH_VPCENDPOINT_DETAILS_BY_ID: "NEON_FETCH_VPCENDPOINT_DETAILS_BY_ID",
    GET_BRANCHES_FOR_PROJECT: "NEON_GET_BRANCHES_FOR_PROJECT",
    GET_BRANCH_ROLES_FOR_PROJECT: "NEON_GET_BRANCH_ROLES_FOR_PROJECT",
    GET_CONSUMPTION_HISTORY_PROJECTS: "NEON_GET_CONSUMPTION_HISTORY_PROJECTS",
    GET_CURRENT_USER_INFORMATION: "NEON_GET_CURRENT_USER_INFORMATION",
    GET_ORGANIZATION_API_KEYS: "NEON_GET_ORGANIZATION_API_KEYS",
    GET_PROJECT_BRANCHES: "NEON_GET_PROJECT_BRANCHES",
    GET_PROJECT_BRANCH_ROLE: "NEON_GET_PROJECT_BRANCH_ROLE",
    GET_PROJECT_CONNECTION_URI: "NEON_GET_PROJECT_CONNECTION_URI",
    GET_PROJECT_ENDPOINT_INFORMATION: "NEON_GET_PROJECT_ENDPOINT_INFORMATION",
    GET_PROJECT_OPERATION_BY_ID: "NEON_GET_PROJECT_OPERATION_BY_ID",
    GET_SCHEMA_FOR_PROJECT_BRANCH: "NEON_GET_SCHEMA_FOR_PROJECT_BRANCH",
    GET_USER_ORGANIZATIONS: "NEON_GET_USER_ORGANIZATIONS",
    GET_VPC_REGION_ENDPOINTS: "NEON_GET_VPC_REGION_ENDPOINTS",
    LIST_API_KEYS: "NEON_LIST_API_KEYS",
    LIST_SHARED_PROJECTS: "NEON_LIST_SHARED_PROJECTS",
    MODIFY_BRANCH_DETAILS_IN_PROJECT: "NEON_MODIFY_BRANCH_DETAILS_IN_PROJECT",
    PATCH_BRANCH_DATABASE_INFORMATION: "NEON_PATCH_BRANCH_DATABASE_INFORMATION",
    RESET_ROLE_PASSWORD_FOR_BRANCH: "NEON_RESET_ROLE_PASSWORD_FOR_BRANCH",
    RESTART_PROJECT_ENDPOINT: "NEON_RESTART_PROJECT_ENDPOINT",
    RESTORE_PROJECT_BRANCH: "NEON_RESTORE_PROJECT_BRANCH",
    RETRIEVE_ACCOUNT_CONSUMPTION_HISTORY: "NEON_RETRIEVE_ACCOUNT_CONSUMPTION_HISTORY",
    RETRIEVE_ALL_REGIONS: "NEON_RETRIEVE_ALL_REGIONS",
    RETRIEVE_BRANCH_DATABASE_DETAILS: "NEON_RETRIEVE_BRANCH_DATABASE_DETAILS",
    RETRIEVE_BRANCH_ENDPOINTS: "NEON_RETRIEVE_BRANCH_ENDPOINTS",
    RETRIEVE_JWKS_FOR_PROJECT: "NEON_RETRIEVE_JWKS_FOR_PROJECT",
    RETRIEVE_ORGANIZATION_BY_ID: "NEON_RETRIEVE_ORGANIZATION_BY_ID",
    RETRIEVE_ORGANIZATION_INVITATIONS: "NEON_RETRIEVE_ORGANIZATION_INVITATIONS",
    RETRIEVE_ORGANIZATION_MEMBER_INFO: "NEON_RETRIEVE_ORGANIZATION_MEMBER_INFO",
    RETRIEVE_PROJECTS_LIST: "NEON_RETRIEVE_PROJECTS_LIST",
    RETRIEVE_PROJECT_ENDPOINT_DETAILS: "NEON_RETRIEVE_PROJECT_ENDPOINT_DETAILS",
    RETRIEVE_PROJECT_OPERATIONS: "NEON_RETRIEVE_PROJECT_OPERATIONS",
    RETRIEVE_PROJECT_PERMISSIONS: "NEON_RETRIEVE_PROJECT_PERMISSIONS",
    RETRIEVE_VPC_ENDPOINTS_FOR_PROJECT: "NEON_RETRIEVE_VPC_ENDPOINTS_FOR_PROJECT",
    REVEAL_ROLE_PASSWORD_IN_BRANCH: "NEON_REVEAL_ROLE_PASSWORD_IN_BRANCH",
    SEND_ORGANIZATION_INVITATIONS: "NEON_SEND_ORGANIZATION_INVITATIONS",
    SET_BRANCH_AS_DEFAULT: "NEON_SET_BRANCH_AS_DEFAULT",
    START_ENDPOINT_FOR_PROJECT: "NEON_START_ENDPOINT_FOR_PROJECT",
    SUSPEND_PROJECT_ENDPOINT_BY_ID: "NEON_SUSPEND_PROJECT_ENDPOINT_BY_ID",
    TRANSFER_USER_PROJECTS_TO_ORGANIZATION: "NEON_TRANSFER_USER_PROJECTS_TO_ORGANIZATION",
    UPDATE_ORGANIZATION_MEMBER_ROLE: "NEON_UPDATE_ORGANIZATION_MEMBER_ROLE",
    UPDATE_PROJECT_COMPUTE_ENDPOINT_SETTINGS: "NEON_UPDATE_PROJECT_COMPUTE_ENDPOINT_SETTINGS",
    UPDATE_PROJECT_SETTINGS_BY_ID: "NEON_UPDATE_PROJECT_SETTINGS_BY_ID",
  },
  triggerTypes: {},
}
