// @composio/core/generated/notion.ts
/**
 * Map of Composio's NOTION toolkit.
 */
export const NOTION = {
  slug: "notion",
  tools: {
    ADD_PAGE_CONTENT: "NOTION_ADD_PAGE_CONTENT",
    APPEND_BLOCK_CHILDREN: "NOTION_APPEND_BLOCK_CHILDREN",
    ARCHIVE_NOTION_PAGE: "NOTION_ARCHIVE_NOTION_PAGE",
    CREATE_COMMENT: "NOTION_CREATE_COMMENT",
    CREATE_DATABASE: "NOTION_CREATE_DATABASE",
    CREATE_NOTION_PAGE: "NOTION_CREATE_NOTION_PAGE",
    DELETE_BLOCK: "NOTION_DELETE_BLOCK",
    DUPLICATE_PAGE: "NOTION_DUPLICATE_PAGE",
    FETCH_COMMENTS: "NOTION_FETCH_COMMENTS",
    FETCH_DATA: "NOTION_FETCH_DATA",
    FETCH_DATABASE: "NOTION_FETCH_DATABASE",
    FETCH_NOTION_BLOCK: "NOTION_FETCH_NOTION_BLOCK",
    FETCH_NOTION_CHILD_BLOCK: "NOTION_FETCH_NOTION_CHILD_BLOCK",
    FETCH_ROW: "NOTION_FETCH_ROW",
    GET_ABOUT_ME: "NOTION_GET_ABOUT_ME",
    GET_ABOUT_USER: "NOTION_GET_ABOUT_USER",
    GET_PAGE_PROPERTY_ACTION: "NOTION_GET_PAGE_PROPERTY_ACTION",
    INSERT_ROW_DATABASE: "NOTION_INSERT_ROW_DATABASE",
    LIST_USERS: "NOTION_LIST_USERS",
    NOTION_UPDATE_BLOCK: "NOTION_NOTION_UPDATE_BLOCK",
    QUERY_DATABASE: "NOTION_QUERY_DATABASE",
    RETRIEVE_COMMENT: "NOTION_RETRIEVE_COMMENT",
    RETRIEVE_DATABASE_PROPERTY: "NOTION_RETRIEVE_DATABASE_PROPERTY",
    SEARCH_NOTION_PAGE: "NOTION_SEARCH_NOTION_PAGE",
    UPDATE_PAGE: "NOTION_UPDATE_PAGE",
    UPDATE_ROW_DATABASE: "NOTION_UPDATE_ROW_DATABASE",
    UPDATE_SCHEMA_DATABASE: "NOTION_UPDATE_SCHEMA_DATABASE",
  },
  triggerTypes: {
    PAGE_ADDED_TO_DATABASE: "NOTION_PAGE_ADDED_TO_DATABASE",
    PAGE_ADDED_TRIGGER: "NOTION_PAGE_ADDED_TRIGGER",
    PAGE_UPDATED_TRIGGER: "NOTION_PAGE_UPDATED_TRIGGER",
  },
}
