// @composio/core/generated/one_drive.ts
/**
 * Map of Composio's ONE_DRIVE toolkit.
 */
export const ONE_DRIVE = {
  slug: "one_drive",
  tools: {
    COPY_ITEM: "ONE_DRIVE_COPY_ITEM",
    CREATE_LINK: "ONE_DRIVE_CREATE_LINK",
    DELETE_ITEM: "ONE_DRIVE_DELETE_ITEM",
    DOWNLOAD_FILE: "ONE_DRIVE_DOWNLOAD_FILE",
    GET_DRIVE: "ONE_DRIVE_GET_DRIVE",
    GET_ITEM: "ONE_DRIVE_GET_ITEM",
    GET_ITEM_PERMISSIONS: "ONE_DRIVE_GET_ITEM_PERMISSIONS",
    GET_ITEM_THUMBNAILS: "ONE_DRIVE_GET_ITEM_THUMBNAILS",
    GET_ITEM_VERSIONS: "ONE_DRIVE_GET_ITEM_VERSIONS",
    GET_QUOTA: "ONE_DRIVE_GET_QUOTA",
    GET_RECENT_ITEMS: "ONE_DRIVE_GET_RECENT_ITEMS",
    GET_SHARED_ITEMS: "ONE_DRIVE_GET_SHARED_ITEMS",
    GET_SHAREPOINT_LIST_ITEMS: "ONE_DRIVE_GET_SHAREPOINT_LIST_ITEMS",
    GET_SITE_DETAILS: "ONE_DRIVE_GET_SITE_DETAILS",
    GET_SITE_PAGE_CONTENT: "ONE_DRIVE_GET_SITE_PAGE_CONTENT",
    INVITE_USER_TO_DRIVE_ITEM: "ONE_DRIVE_INVITE_USER_TO_DRIVE_ITEM",
    LIST_DRIVES: "ONE_DRIVE_LIST_DRIVES",
    LIST_DRIVE_ITEM_ACTIVITIES: "ONE_DRIVE_LIST_DRIVE_ITEM_ACTIVITIES",
    LIST_ROOT_DRIVE_CHANGES: "ONE_DRIVE_LIST_ROOT_DRIVE_CHANGES",
    LIST_SHAREPOINT_LIST_ITEMS_DELTA: "ONE_DRIVE_LIST_SHAREPOINT_LIST_ITEMS_DELTA",
    LIST_SITE_COLUMNS: "ONE_DRIVE_LIST_SITE_COLUMNS",
    LIST_SITE_DRIVE_ITEMS_DELTA: "ONE_DRIVE_LIST_SITE_DRIVE_ITEMS_DELTA",
    LIST_SITE_LISTS: "ONE_DRIVE_LIST_SITE_LISTS",
    LIST_SITE_SUBSITES: "ONE_DRIVE_LIST_SITE_SUBSITES",
    LIST_SUBSCRIPTIONS: "ONE_DRIVE_LIST_SUBSCRIPTIONS",
    MOVE_ITEM: "ONE_DRIVE_MOVE_ITEM",
    ONEDRIVE_CREATE_FOLDER: "ONE_DRIVE_ONEDRIVE_CREATE_FOLDER",
    ONEDRIVE_CREATE_TEXT_FILE: "ONE_DRIVE_ONEDRIVE_CREATE_TEXT_FILE",
    ONEDRIVE_FIND_FILE: "ONE_DRIVE_ONEDRIVE_FIND_FILE",
    ONEDRIVE_FIND_FOLDER: "ONE_DRIVE_ONEDRIVE_FIND_FOLDER",
    ONEDRIVE_LIST_ITEMS: "ONE_DRIVE_ONEDRIVE_LIST_ITEMS",
    ONEDRIVE_UPLOAD_FILE: "ONE_DRIVE_ONEDRIVE_UPLOAD_FILE",
    PREVIEW_DRIVE_ITEM: "ONE_DRIVE_PREVIEW_DRIVE_ITEM",
    SEARCH_ITEMS: "ONE_DRIVE_SEARCH_ITEMS",
    UPDATE_DRIVE_ITEM_METADATA: "ONE_DRIVE_UPDATE_DRIVE_ITEM_METADATA",
  },
  triggerTypes: {
    ONE_DRIVE_DELETED_TRIGGER: "ONE_DRIVE_ONE_DRIVE_DELETED_TRIGGER",
    ONE_DRIVE_FILE_CREATED_TRIGGER: "ONE_DRIVE_ONE_DRIVE_FILE_CREATED_TRIGGER",
    ONE_DRIVE_FOLDER_CREATED_TRIGGER: "ONE_DRIVE_ONE_DRIVE_FOLDER_CREATED_TRIGGER",
    ONE_DRIVE_UPDATED_TRIGGER: "ONE_DRIVE_ONE_DRIVE_UPDATED_TRIGGER",
  },
}
