// @composio/core/generated/open_sea.ts
/**
 * Map of Composio's OPEN_SEA toolkit.
 */
export const OPEN_SEA = {
  slug: "open_sea",
  tools: {
    BUILD_CRITERIA_OFFER: "OPEN_SEA_BUILD_CRITERIA_OFFER",
    CANCEL_ORDER: "OPEN_SEA_CANCEL_ORDER",
    CREATE_CRITERIA_OFFER: "OPEN_SEA_CREATE_CRITERIA_OFFER",
    CREATE_ITEM_OFFER: "OPEN_SEA_CREATE_ITEM_OFFER",
    CREATE_LISTING: "OPEN_SEA_CREATE_LISTING",
    FULFILL_LISTING: "OPEN_SEA_FULFILL_LISTING",
    FULFILL_OFFER: "OPEN_SEA_FULFILL_OFFER",
    GET_ACCOUNT: "OPEN_SEA_GET_ACCOUNT",
    GET_ALL_LISTINGS_BY_COLLECTION: "OPEN_SEA_GET_ALL_LISTINGS_BY_COLLECTION",
    GET_ALL_OFFERS_BY_COLLECTION: "OPEN_SEA_GET_ALL_OFFERS_BY_COLLECTION",
    GET_BEST_LISTINGS_BY_COLLECTION: "OPEN_SEA_GET_BEST_LISTINGS_BY_COLLECTION",
    GET_BEST_LISTING_BY_NFT: "OPEN_SEA_GET_BEST_LISTING_BY_NFT",
    GET_BEST_OFFER_BY_NFT: "OPEN_SEA_GET_BEST_OFFER_BY_NFT",
    GET_COLLECTION: "OPEN_SEA_GET_COLLECTION",
    GET_COLLECTIONS: "OPEN_SEA_GET_COLLECTIONS",
    GET_COLLECTION_STATS: "OPEN_SEA_GET_COLLECTION_STATS",
    GET_CONTRACT: "OPEN_SEA_GET_CONTRACT",
    GET_EVENTS: "OPEN_SEA_GET_EVENTS",
    GET_LISTINGS: "OPEN_SEA_GET_LISTINGS",
    GET_NFT: "OPEN_SEA_GET_NFT",
    GET_ORDER: "OPEN_SEA_GET_ORDER",
    GET_PAYMENT_TOKEN: "OPEN_SEA_GET_PAYMENT_TOKEN",
    GET_TRAITS: "OPEN_SEA_GET_TRAITS",
    REFRESH_NFT_METADATA: "OPEN_SEA_REFRESH_NFT_METADATA",
  },
  triggerTypes: {},
}
