// @composio/core/generated/outlook.ts
/**
 * Map of Composio's OUTLOOK toolkit.
 */
export const OUTLOOK = {
  slug: "outlook",
  tools: {
    DOWNLOAD_OUTLOOK_ATTACHMENT: "OUTLOOK_DOWNLOAD_OUTLOOK_ATTACHMENT",
    LIST_OUTLOOK_ATTACHMENTS: "OUTLOOK_LIST_OUTLOOK_ATTACHMENTS",
    OUTLOOK_CALENDAR_CREATE_EVENT: "OUTLOOK_OUTLOOK_CALENDAR_CREATE_EVENT",
    OUTLOOK_CREATE_CONTACT: "OUTLOOK_OUTLOOK_CREATE_CONTACT",
    OUTLOOK_CREATE_DRAFT: "OUTLOOK_OUTLOOK_CREATE_DRAFT",
    OUTLOOK_CREATE_DRAFT_REPLY: "OUTLOOK_OUTLOOK_CREATE_DRAFT_REPLY",
    OUTLOOK_DELETE_CONTACT: "OUTLOOK_OUTLOOK_DELETE_CONTACT",
    OUTLOOK_DELETE_EVENT: "OUTLOOK_OUTLOOK_DELETE_EVENT",
    OUTLOOK_GET_CONTACT: "OUTLOOK_OUTLOOK_GET_CONTACT",
    OUTLOOK_GET_EVENT: "OUTLOOK_OUTLOOK_GET_EVENT",
    OUTLOOK_GET_MESSAGE: "OUTLOOK_OUTLOOK_GET_MESSAGE",
    OUTLOOK_GET_PROFILE: "OUTLOOK_OUTLOOK_GET_PROFILE",
    OUTLOOK_GET_SCHEDULE: "OUTLOOK_OUTLOOK_GET_SCHEDULE",
    OUTLOOK_LIST_CONTACTS: "OUTLOOK_OUTLOOK_LIST_CONTACTS",
    OUTLOOK_LIST_EVENTS: "OUTLOOK_OUTLOOK_LIST_EVENTS",
    OUTLOOK_LIST_MESSAGES: "OUTLOOK_OUTLOOK_LIST_MESSAGES",
    OUTLOOK_REPLY_EMAIL: "OUTLOOK_OUTLOOK_REPLY_EMAIL",
    OUTLOOK_SEARCH_MESSAGES: "OUTLOOK_OUTLOOK_SEARCH_MESSAGES",
    OUTLOOK_SEND_EMAIL: "OUTLOOK_OUTLOOK_SEND_EMAIL",
    OUTLOOK_UPDATE_CALENDAR_EVENT: "OUTLOOK_OUTLOOK_UPDATE_CALENDAR_EVENT",
    OUTLOOK_UPDATE_CONTACT: "OUTLOOK_OUTLOOK_UPDATE_CONTACT",
    OUTLOOK_UPDATE_EMAIL: "OUTLOOK_OUTLOOK_UPDATE_EMAIL",
  },
  triggerTypes: {
    OUTLOOK_CONTACT_TRIGGER: "OUTLOOK_OUTLOOK_CONTACT_TRIGGER",
    OUTLOOK_EVENT_CHANGE_TRIGGER: "OUTLOOK_OUTLOOK_EVENT_CHANGE_TRIGGER",
    OUTLOOK_EVENT_TRIGGER: "OUTLOOK_OUTLOOK_EVENT_TRIGGER",
    OUTLOOK_MESSAGE_TRIGGER: "OUTLOOK_OUTLOOK_MESSAGE_TRIGGER",
    OUTLOOK_SENT_MESSAGE_TRIGGER: "OUTLOOK_OUTLOOK_SENT_MESSAGE_TRIGGER",
  },
}
