// @composio/core/generated/pagerduty.ts
/**
 * Map of Composio's PAGERDUTY toolkit.
 */
export const PAGERDUTY = {
  slug: "pagerduty",
  tools: {
    ADD_SERVICE_TO_INCIDENT_WORKFLOW_TRIGGER: "PAGERDUTY_ADD_SERVICE_TO_INCIDENT_WORKFLOW_TRIGGER",
    AGGREGATE_ESCALATION_POLICY_INCIDENT_METRICS: "PAGERDUTY_AGGREGATE_ESCALATION_POLICY_INCIDENT_METRICS",
    AGGREGATE_INCIDENT_METRICS_WITH_FILTERS: "PAGERDUTY_AGGREGATE_INCIDENT_METRICS_WITH_FILTERS",
    ASSOCIATE_SERVICE_DEPENDENCIES: "PAGERDUTY_ASSOCIATE_SERVICE_DEPENDENCIES",
    ASSOCIATE_TEAM_WITH_AUTOMATION_ACTION: "PAGERDUTY_ASSOCIATE_TEAM_WITH_AUTOMATION_ACTION",
    AUDIT_ESCALATION_POLICY_RECORDS: "PAGERDUTY_AUDIT_ESCALATION_POLICY_RECORDS",
    CONVERT_SERVICE_RULE_BY_ID: "PAGERDUTY_CONVERT_SERVICE_RULE_BY_ID",
    COUNT_PAUSED_INCIDENT_REPORTS: "PAGERDUTY_COUNT_PAUSED_INCIDENT_REPORTS",
    CREATE_AND_MANAGE_CUSTOM_INCIDENT_FIELDS: "PAGERDUTY_CREATE_AND_MANAGE_CUSTOM_INCIDENT_FIELDS",
    CREATE_AN_INCIDENT_TYPE: "PAGERDUTY_CREATE_AN_INCIDENT_TYPE",
    CREATE_AUTOMATION_RUNNER_ENDPOINT: "PAGERDUTY_CREATE_AUTOMATION_RUNNER_ENDPOINT",
    CREATE_A_CUSTOM_FIELD_FOR_AN_INCIDENT_TYPE: "PAGERDUTY_CREATE_A_CUSTOM_FIELD_FOR_AN_INCIDENT_TYPE",
    CREATE_A_FIELD_OPTION_FOR_A_CUSTOM_FIELD: "PAGERDUTY_CREATE_A_FIELD_OPTION_FOR_A_CUSTOM_FIELD",
    CREATE_A_RULESET: "PAGERDUTY_CREATE_A_RULESET",
    CREATE_A_TRIGGER: "PAGERDUTY_CREATE_A_TRIGGER",
    CREATE_BUSINESS_SERVICE: "PAGERDUTY_CREATE_BUSINESS_SERVICE",
    CREATE_ESCALATION_POLICY: "PAGERDUTY_CREATE_ESCALATION_POLICY",
    CREATE_EVENT_ORCHESTRATION: "PAGERDUTY_CREATE_EVENT_ORCHESTRATION",
    CREATE_EVENT_RULE_IN_RULESET: "PAGERDUTY_CREATE_EVENT_RULE_IN_RULESET",
    CREATE_EXTENSION_OBJECT: "PAGERDUTY_CREATE_EXTENSION_OBJECT",
    CREATE_HANDOFF_NOTIFICATION_RULE: "PAGERDUTY_CREATE_HANDOFF_NOTIFICATION_RULE",
    CREATE_INCIDENT_FIELD_OPTION: "PAGERDUTY_CREATE_INCIDENT_FIELD_OPTION",
    CREATE_INCIDENT_RECORD: "PAGERDUTY_CREATE_INCIDENT_RECORD",
    CREATE_INCIDENT_WORKFLOW: "PAGERDUTY_CREATE_INCIDENT_WORKFLOW",
    CREATE_INCIDENT_WORKFLOW_INSTANCE: "PAGERDUTY_CREATE_INCIDENT_WORKFLOW_INSTANCE",
    CREATE_INTEGRATION_FOR_ORCHESTRATION: "PAGERDUTY_CREATE_INTEGRATION_FOR_ORCHESTRATION",
    CREATE_INTEGRATION_FOR_SERVICE: "PAGERDUTY_CREATE_INTEGRATION_FOR_SERVICE",
    CREATE_MAINTENANCE_WINDOW_OBJECT: "PAGERDUTY_CREATE_MAINTENANCE_WINDOW_OBJECT",
    CREATE_NEW_SCHEDULE_LAYER: "PAGERDUTY_CREATE_NEW_SCHEDULE_LAYER",
    CREATE_NEW_SERVICE: "PAGERDUTY_CREATE_NEW_SERVICE",
    CREATE_NEW_TAG_IN_SYSTEM: "PAGERDUTY_CREATE_NEW_TAG_IN_SYSTEM",
    CREATE_NEW_TEAM_WITH_DETAILS: "PAGERDUTY_CREATE_NEW_TEAM_WITH_DETAILS",
    CREATE_NEW_WEBHOOK_SUBSCRIPTION: "PAGERDUTY_CREATE_NEW_WEBHOOK_SUBSCRIPTION",
    CREATE_OR_UPDATE_STATUS_PAGE_POST: "PAGERDUTY_CREATE_OR_UPDATE_STATUS_PAGE_POST",
    CREATE_OR_UPDATE_STATUS_PAGE_POSTMORTEM: "PAGERDUTY_CREATE_OR_UPDATE_STATUS_PAGE_POSTMORTEM",
    CREATE_RESPONSE_PLAY_FOR_INCIDENTS: "PAGERDUTY_CREATE_RESPONSE_PLAY_FOR_INCIDENTS",
    CREATE_SCHEDULE_OVERRIDES_CONFIGURATION: "PAGERDUTY_CREATE_SCHEDULE_OVERRIDES_CONFIGURATION",
    CREATE_SERVICE_EVENT_RULE: "PAGERDUTY_CREATE_SERVICE_EVENT_RULE",
    CREATE_STATUS_PAGE_SUBSCRIPTION: "PAGERDUTY_CREATE_STATUS_PAGE_SUBSCRIPTION",
    CREATE_STATUS_UPDATE_TEMPLATE: "PAGERDUTY_CREATE_STATUS_UPDATE_TEMPLATE",
    CREATE_USER_NOTIFICATION_RULE: "PAGERDUTY_CREATE_USER_NOTIFICATION_RULE",
    CREATE_USER_OBJECT: "PAGERDUTY_CREATE_USER_OBJECT",
    CREATE_USER_STATUS_UPDATE_NOTIFICATION_RULE: "PAGERDUTY_CREATE_USER_STATUS_UPDATE_NOTIFICATION_RULE",
    CREATE_WORKFLOW_INTEGRATION_CONNECTION: "PAGERDUTY_CREATE_WORKFLOW_INTEGRATION_CONNECTION",
    DELETE_ACCOUNT_SUBSCRIPTION_BY_ID: "PAGERDUTY_DELETE_ACCOUNT_SUBSCRIPTION_BY_ID",
    DELETE_ADDON_BY_ID: "PAGERDUTY_DELETE_ADDON_BY_ID",
    DELETE_ALERT_GROUPING_SETTING: "PAGERDUTY_DELETE_ALERT_GROUPING_SETTING",
    DELETE_ALL_OAUTH_DELEGATIONS: "PAGERDUTY_DELETE_ALL_OAUTH_DELEGATIONS",
    DELETE_AUTOMATION_ACTIONS_RUNNER_BY_ID: "PAGERDUTY_DELETE_AUTOMATION_ACTIONS_RUNNER_BY_ID",
    DELETE_AUTOMATION_ACTION_BY_ID: "PAGERDUTY_DELETE_AUTOMATION_ACTION_BY_ID",
    DELETE_AUTOMATION_ACTION_SERVICE_BY_ID: "PAGERDUTY_DELETE_AUTOMATION_ACTION_SERVICE_BY_ID",
    DELETE_A_CUSTOM_FIELD_FOR_AN_INCIDENT_TYPE: "PAGERDUTY_DELETE_A_CUSTOM_FIELD_FOR_AN_INCIDENT_TYPE",
    DELETE_A_FIELD_OPTION_FOR_A_CUSTOM_FIELD: "PAGERDUTY_DELETE_A_FIELD_OPTION_FOR_A_CUSTOM_FIELD",
    DELETE_BUSINESS_SERVICE_BY_ID: "PAGERDUTY_DELETE_BUSINESS_SERVICE_BY_ID",
    DELETE_BUSINESS_SERVICE_PRIORITY_THRESHOLDS: "PAGERDUTY_DELETE_BUSINESS_SERVICE_PRIORITY_THRESHOLDS",
    DELETE_CACHE_VARIABLE_FOR_EVENT: "PAGERDUTY_DELETE_CACHE_VARIABLE_FOR_EVENT",
    DELETE_CACHE_VARIABLE_FROM_SERVICE: "PAGERDUTY_DELETE_CACHE_VARIABLE_FROM_SERVICE",
    DELETE_CUSTOM_FIELD_BY_FIELD_ID: "PAGERDUTY_DELETE_CUSTOM_FIELD_BY_FIELD_ID",
    DELETE_ESCALATION_POLICY_BY_ID: "PAGERDUTY_DELETE_ESCALATION_POLICY_BY_ID",
    DELETE_EVENT_BY_ID: "PAGERDUTY_DELETE_EVENT_BY_ID",
    DELETE_EXTENSION_BY_ID: "PAGERDUTY_DELETE_EXTENSION_BY_ID",
    DELETE_FIELD_OPTION: "PAGERDUTY_DELETE_FIELD_OPTION",
    DELETE_INCIDENT_WORKFLOW_BY_ID: "PAGERDUTY_DELETE_INCIDENT_WORKFLOW_BY_ID",
    DELETE_INCIDENT_WORKFLOW_TRIGGER_BY_ID: "PAGERDUTY_DELETE_INCIDENT_WORKFLOW_TRIGGER_BY_ID",
    DELETE_INCIDENT_WORKFLOW_TRIGGER_SERVICE: "PAGERDUTY_DELETE_INCIDENT_WORKFLOW_TRIGGER_SERVICE",
    DELETE_INTEGRATION_FROM_EVENT_ORCHESTRATION: "PAGERDUTY_DELETE_INTEGRATION_FROM_EVENT_ORCHESTRATION",
    DELETE_MAINTENANCE_WINDOW_BY_ID: "PAGERDUTY_DELETE_MAINTENANCE_WINDOW_BY_ID",
    DELETE_ONCALL_HANDOFF_NOTIFICATION_RULE: "PAGERDUTY_DELETE_ONCALL_HANDOFF_NOTIFICATION_RULE",
    DELETE_POST_FROM_STATUS_PAGE: "PAGERDUTY_DELETE_POST_FROM_STATUS_PAGE",
    DELETE_POST_UPDATE_BY_ID: "PAGERDUTY_DELETE_POST_UPDATE_BY_ID",
    DELETE_RESPONSE_PLAY: "PAGERDUTY_DELETE_RESPONSE_PLAY",
    DELETE_RULESET_BY_ID: "PAGERDUTY_DELETE_RULESET_BY_ID",
    DELETE_RULE_FROM_RULESET_BY_ID: "PAGERDUTY_DELETE_RULE_FROM_RULESET_BY_ID",
    DELETE_RUNNER_TEAM_ASSOCIATION: "PAGERDUTY_DELETE_RUNNER_TEAM_ASSOCIATION",
    DELETE_SCHEDULE_BY_ID: "PAGERDUTY_DELETE_SCHEDULE_BY_ID",
    DELETE_SCHEDULE_OVERRIDE_BY_ID: "PAGERDUTY_DELETE_SCHEDULE_OVERRIDE_BY_ID",
    DELETE_SERVICE_BY_ID: "PAGERDUTY_DELETE_SERVICE_BY_ID",
    DELETE_SERVICE_RULE_BY_ID: "PAGERDUTY_DELETE_SERVICE_RULE_BY_ID",
    DELETE_STATUS_PAGE_POSTMORTEM: "PAGERDUTY_DELETE_STATUS_PAGE_POSTMORTEM",
    DELETE_STATUS_UPDATE_NOTIFICATION_RULE: "PAGERDUTY_DELETE_STATUS_UPDATE_NOTIFICATION_RULE",
    DELETE_SUBSCRIPTION_FROM_STATUS_PAGE: "PAGERDUTY_DELETE_SUBSCRIPTION_FROM_STATUS_PAGE",
    DELETE_TAG_BY_ID: "PAGERDUTY_DELETE_TAG_BY_ID",
    DELETE_TEAM_BY_ID: "PAGERDUTY_DELETE_TEAM_BY_ID",
    DELETE_TEAM_ESCALATION_POLICY: "PAGERDUTY_DELETE_TEAM_ESCALATION_POLICY",
    DELETE_TEAM_FROM_AUTOMATION_ACTION: "PAGERDUTY_DELETE_TEAM_FROM_AUTOMATION_ACTION",
    DELETE_TEMPLATE_BY_ID: "PAGERDUTY_DELETE_TEMPLATE_BY_ID",
    DELETE_USER_BY_ID: "PAGERDUTY_DELETE_USER_BY_ID",
    DELETE_USER_CONTACT_METHOD: "PAGERDUTY_DELETE_USER_CONTACT_METHOD",
    DELETE_USER_FROM_TEAM_BY_ID: "PAGERDUTY_DELETE_USER_FROM_TEAM_BY_ID",
    DELETE_USER_NOTIFICATION_RULE: "PAGERDUTY_DELETE_USER_NOTIFICATION_RULE",
    DELETE_USER_SESSION_BY_ID: "PAGERDUTY_DELETE_USER_SESSION_BY_ID",
    DELETE_USER_SESSION_BY_TYPE: "PAGERDUTY_DELETE_USER_SESSION_BY_TYPE",
    DELETE_WEBHOOK_SUBSCRIPTION_BY_ID: "PAGERDUTY_DELETE_WEBHOOK_SUBSCRIPTION_BY_ID",
    DELETE_WORKFLOW_INTEGRATION_CONNECTION: "PAGERDUTY_DELETE_WORKFLOW_INTEGRATION_CONNECTION",
    DISASSOCIATE_SERVICE_DEPENDENCIES: "PAGERDUTY_DISASSOCIATE_SERVICE_DEPENDENCIES",
    EDIT_WEBHOOK_SUBSCRIPTION_BY_ID: "PAGERDUTY_EDIT_WEBHOOK_SUBSCRIPTION_BY_ID",
    ENABLE_EXTENSION_BY_ID: "PAGERDUTY_ENABLE_EXTENSION_BY_ID",
    ENABLE_WEBHOOK_SUBSCRIPTION_BY_ID: "PAGERDUTY_ENABLE_WEBHOOK_SUBSCRIPTION_BY_ID",
    EXECUTE_RESPONSE_PLAY_BY_ID: "PAGERDUTY_EXECUTE_RESPONSE_PLAY_BY_ID",
    FETCH_CACHE_VARIABLE_FOR_EVENT_ORCHESTRATION: "PAGERDUTY_FETCH_CACHE_VARIABLE_FOR_EVENT_ORCHESTRATION",
    FETCH_CUSTOM_INCIDENT_FIELD_BY_ID: "PAGERDUTY_FETCH_CUSTOM_INCIDENT_FIELD_BY_ID",
    FETCH_ESCALATION_POLICES_LIST: "PAGERDUTY_FETCH_ESCALATION_POLICES_LIST",
    FETCH_EVENT_ORCHESTRATIONS: "PAGERDUTY_FETCH_EVENT_ORCHESTRATIONS",
    FETCH_INCIDENT_ANALYTICS_BY_ID: "PAGERDUTY_FETCH_INCIDENT_ANALYTICS_BY_ID",
    FETCH_INCIDENT_LIST: "PAGERDUTY_FETCH_INCIDENT_LIST",
    FETCH_OUTLIER_INCIDENT_BY_ID: "PAGERDUTY_FETCH_OUTLIER_INCIDENT_BY_ID",
    FETCH_POST_UPDATE_STATUS: "PAGERDUTY_FETCH_POST_UPDATE_STATUS",
    FETCH_PRIORITIES_LIST: "PAGERDUTY_FETCH_PRIORITIES_LIST",
    FETCH_RELATED_CHANGE_EVENTS_FOR_INCIDENT: "PAGERDUTY_FETCH_RELATED_CHANGE_EVENTS_FOR_INCIDENT",
    FETCH_RELATED_INCIDENTS_BY_ID: "PAGERDUTY_FETCH_RELATED_INCIDENTS_BY_ID",
    FETCH_RUNNER_TEAMS_INTEGRATION: "PAGERDUTY_FETCH_RUNNER_TEAMS_INTEGRATION",
    FETCH_STATUS_PAGES: "PAGERDUTY_FETCH_STATUS_PAGES",
    FETCH_USER_CONTACT_METHOD: "PAGERDUTY_FETCH_USER_CONTACT_METHOD",
    FETCH_VENDOR_LIST: "PAGERDUTY_FETCH_VENDOR_LIST",
    FILTER_AND_AGGREGATE_INCIDENT_METRICS: "PAGERDUTY_FILTER_AND_AGGREGATE_INCIDENT_METRICS",
    GET_ADDONS_LIST: "PAGERDUTY_GET_ADDONS_LIST",
    GET_AGGREGATED_PD_ADVANCE_USAGE_DATA: "PAGERDUTY_GET_AGGREGATED_PD_ADVANCE_USAGE_DATA",
    GET_ALERTS_BY_INCIDENT_ID: "PAGERDUTY_GET_ALERTS_BY_INCIDENT_ID",
    GET_ALERT_GROUPING_SETTINGS: "PAGERDUTY_GET_ALERT_GROUPING_SETTINGS",
    GET_AN_INCIDENT_TYPE: "PAGERDUTY_GET_AN_INCIDENT_TYPE",
    GET_AN_INCIDENT_TYPE_CUSTOM_FIELD: "PAGERDUTY_GET_AN_INCIDENT_TYPE_CUSTOM_FIELD",
    GET_AUDIT_RECORDS: "PAGERDUTY_GET_AUDIT_RECORDS",
    GET_AUTOMATION_ACTIONS_RUNNERS: "PAGERDUTY_GET_AUTOMATION_ACTIONS_RUNNERS",
    GET_AUTOMATION_ACTION_BY_ID: "PAGERDUTY_GET_AUTOMATION_ACTION_BY_ID",
    GET_AUTOMATION_ACTION_RUNNER_BY_ID: "PAGERDUTY_GET_AUTOMATION_ACTION_RUNNER_BY_ID",
    GET_AUTOMATION_ACTION_TEAM_BY_TEAM_ID: "PAGERDUTY_GET_AUTOMATION_ACTION_TEAM_BY_TEAM_ID",
    GET_A_FIELD_OPTION_ON_A_CUSTOM_FIELD: "PAGERDUTY_GET_A_FIELD_OPTION_ON_A_CUSTOM_FIELD",
    GET_A_TRIGGER: "PAGERDUTY_GET_A_TRIGGER",
    GET_BUSINESS_SERVICES_PRIORITY_THRESHOLDS: "PAGERDUTY_GET_BUSINESS_SERVICES_PRIORITY_THRESHOLDS",
    GET_BUSINESS_SERVICE_SUBSCRIBERS_BY_ID: "PAGERDUTY_GET_BUSINESS_SERVICE_SUBSCRIBERS_BY_ID",
    GET_CACHE_VARIABLES_FOR_SERVICE: "PAGERDUTY_GET_CACHE_VARIABLES_FOR_SERVICE",
    GET_CACHE_VARIABLE_BY_ID: "PAGERDUTY_GET_CACHE_VARIABLE_BY_ID",
    GET_ESCALATION_POLICY_BY_ID: "PAGERDUTY_GET_ESCALATION_POLICY_BY_ID",
    GET_EVENT_INTEGRATIONS_BY_ID: "PAGERDUTY_GET_EVENT_INTEGRATIONS_BY_ID",
    GET_EVENT_ORCHESTRATION_BY_ID: "PAGERDUTY_GET_EVENT_ORCHESTRATION_BY_ID",
    GET_EVENT_ORCHESTRATION_CACHE_VARIABLES: "PAGERDUTY_GET_EVENT_ORCHESTRATION_CACHE_VARIABLES",
    GET_EVENT_ORCHESTRATION_GLOBAL: "PAGERDUTY_GET_EVENT_ORCHESTRATION_GLOBAL",
    GET_EVENT_ORCHESTRATION_INTEGRATION: "PAGERDUTY_GET_EVENT_ORCHESTRATION_INTEGRATION",
    GET_EXTENSION_SCHEMA_BY_ID: "PAGERDUTY_GET_EXTENSION_SCHEMA_BY_ID",
    GET_IMPACT_BY_STATUS_PAGE_ID: "PAGERDUTY_GET_IMPACT_BY_STATUS_PAGE_ID",
    GET_INCIDENT_ALERT_DETAILS: "PAGERDUTY_GET_INCIDENT_ALERT_DETAILS",
    GET_INCIDENT_LOG_ENTRIES_BY_ID: "PAGERDUTY_GET_INCIDENT_LOG_ENTRIES_BY_ID",
    GET_INCIDENT_WORKFLOWS: "PAGERDUTY_GET_INCIDENT_WORKFLOWS",
    GET_INCIDENT_WORKFLOWS_ACTIONS: "PAGERDUTY_GET_INCIDENT_WORKFLOWS_ACTIONS",
    GET_INCIDENT_WORKFLOW_ACTION_BY_ID: "PAGERDUTY_GET_INCIDENT_WORKFLOW_ACTION_BY_ID",
    GET_LOG_ENTRIES: "PAGERDUTY_GET_LOG_ENTRIES",
    GET_MAINTENANCE_WINDOW_BY_ID: "PAGERDUTY_GET_MAINTENANCE_WINDOW_BY_ID",
    GET_OAUTH_DELEGATIONS_REVOCATION_REQUESTS_STATUS: "PAGERDUTY_GET_OAUTH_DELEGATIONS_REVOCATION_REQUESTS_STATUS",
    GET_PAUSED_INCIDENT_ALERTS: "PAGERDUTY_GET_PAUSED_INCIDENT_ALERTS",
    GET_POST_FROM_STATUS_PAGE_BY_ID: "PAGERDUTY_GET_POST_FROM_STATUS_PAGE_BY_ID",
    GET_RESPONSE_PLAYS: "PAGERDUTY_GET_RESPONSE_PLAYS",
    GET_RULE_FROM_RULESET_BY_ID: "PAGERDUTY_GET_RULE_FROM_RULESET_BY_ID",
    GET_SCHEDULES: "PAGERDUTY_GET_SCHEDULES",
    GET_SERVICE_IMPACTS_BY_URL_SLUG: "PAGERDUTY_GET_SERVICE_IMPACTS_BY_URL_SLUG",
    GET_SEVERITY_FOR_STATUS_PAGE: "PAGERDUTY_GET_SEVERITY_FOR_STATUS_PAGE",
    GET_SPECIFIC_POST_UPDATE_STATUS: "PAGERDUTY_GET_SPECIFIC_POST_UPDATE_STATUS",
    GET_STATUS_DASHBOARD_BY_ID: "PAGERDUTY_GET_STATUS_DASHBOARD_BY_ID",
    GET_STATUS_FOR_STATUS_PAGE_BY_ID: "PAGERDUTY_GET_STATUS_FOR_STATUS_PAGE_BY_ID",
    GET_STATUS_PAGE_IMPACTS_BY_ID: "PAGERDUTY_GET_STATUS_PAGE_IMPACTS_BY_ID",
    GET_STATUS_PAGE_SUBSCRIPTION: "PAGERDUTY_GET_STATUS_PAGE_SUBSCRIPTION",
    GET_TEAMS_ASSOCIATED_WITH_ACTION_ID: "PAGERDUTY_GET_TEAMS_ASSOCIATED_WITH_ACTION_ID",
    GET_TEAM_MEMBERS_BY_ID: "PAGERDUTY_GET_TEAM_MEMBERS_BY_ID",
    GET_TEAM_NOTIFICATION_SUBSCRIPTIONS: "PAGERDUTY_GET_TEAM_NOTIFICATION_SUBSCRIPTIONS",
    GET_TEMPLATE_BY_ID: "PAGERDUTY_GET_TEMPLATE_BY_ID",
    GET_THE_SERVICE_ORCHESTRATION_FOR_A_SERVICE: "PAGERDUTY_GET_THE_SERVICE_ORCHESTRATION_FOR_A_SERVICE",
    GET_USER_NOTIFICATION_SUBSCRIPTIONS: "PAGERDUTY_GET_USER_NOTIFICATION_SUBSCRIPTIONS",
    GET_USER_SESSIONS_BY_ID: "PAGERDUTY_GET_USER_SESSIONS_BY_ID",
    GET_USER_SESSION_BY_TYPE: "PAGERDUTY_GET_USER_SESSION_BY_TYPE",
    GET_USER_STATUS_UPDATE_NOTIFICATION_RULES: "PAGERDUTY_GET_USER_STATUS_UPDATE_NOTIFICATION_RULES",
    GET_WEBHOOK_SUBSCRIPTION_BY_ID: "PAGERDUTY_GET_WEBHOOK_SUBSCRIPTION_BY_ID",
    GET_WORKFLOW_INTEGRATION: "PAGERDUTY_GET_WORKFLOW_INTEGRATION",
    GET_WORKFLOW_INTEGRATION_CONNECTION: "PAGERDUTY_GET_WORKFLOW_INTEGRATION_CONNECTION",
    INSTALL_ADD_ON_ENDPOINT: "PAGERDUTY_INSTALL_ADD_ON_ENDPOINT",
    INVOKE_AUTOMATION_ACTION_BY_ID: "PAGERDUTY_INVOKE_AUTOMATION_ACTION_BY_ID",
    LIST_ALL_WORKFLOW_INTEGRATION_CONNECTIONS: "PAGERDUTY_LIST_ALL_WORKFLOW_INTEGRATION_CONNECTIONS",
    LIST_AUTOMATION_ACTION_DETAILS: "PAGERDUTY_LIST_AUTOMATION_ACTION_DETAILS",
    LIST_EXTENSION_SCHEMAS: "PAGERDUTY_LIST_EXTENSION_SCHEMAS",
    LIST_FIELD_OPTIONS_ON_A_CUSTOM_FIELD: "PAGERDUTY_LIST_FIELD_OPTIONS_ON_A_CUSTOM_FIELD",
    LIST_INCIDENT_STATUS_UPDATE_SUBSCRIBERS: "PAGERDUTY_LIST_INCIDENT_STATUS_UPDATE_SUBSCRIBERS",
    LIST_INCIDENT_TYPES: "PAGERDUTY_LIST_INCIDENT_TYPES",
    LIST_INCIDENT_TYPE_CUSTOM_FIELDS: "PAGERDUTY_LIST_INCIDENT_TYPE_CUSTOM_FIELDS",
    LIST_LICENSES: "PAGERDUTY_LIST_LICENSES",
    LIST_SUPPORTING_SERVICE_IMPACTS: "PAGERDUTY_LIST_SUPPORTING_SERVICE_IMPACTS",
    LIST_TEMPLATES: "PAGERDUTY_LIST_TEMPLATES",
    LIST_WORKFLOW_INTEGRATIONS: "PAGERDUTY_LIST_WORKFLOW_INTEGRATIONS",
    LIST_WORKFLOW_INTEGRATION_CONNECTIONS: "PAGERDUTY_LIST_WORKFLOW_INTEGRATION_CONNECTIONS",
    MANAGE_CACHE_VARIABLES_FOR_EVENT_SERVICE: "PAGERDUTY_MANAGE_CACHE_VARIABLES_FOR_EVENT_SERVICE",
    MERGE_SOURCE_INCIDENTS_INTO_TARGET_INCIDENT: "PAGERDUTY_MERGE_SOURCE_INCIDENTS_INTO_TARGET_INCIDENT",
    MIGRATE_INTEGRATION_BETWEEN_ORCHESTRATIONS: "PAGERDUTY_MIGRATE_INTEGRATION_BETWEEN_ORCHESTRATIONS",
    MODIFY_ENTITY_TAGS: "PAGERDUTY_MODIFY_ENTITY_TAGS",
    PING_WEBHOOK_SUBSCRIPTION: "PAGERDUTY_PING_WEBHOOK_SUBSCRIPTION",
    POST_ACCOUNT_SUBSCRIPTION_FOR_BUSINESS_SERVICE: "PAGERDUTY_POST_ACCOUNT_SUBSCRIPTION_FOR_BUSINESS_SERVICE",
    POST_ALERT_GROUPING_SETTINGS: "PAGERDUTY_POST_ALERT_GROUPING_SETTINGS",
    POST_ANALYTICS_METRICS_RESPONDER_FILTERS: "PAGERDUTY_POST_ANALYTICS_METRICS_RESPONDER_FILTERS",
    POST_ANALYTIC_METRICS_ON_ESCALATION_POLICIES: "PAGERDUTY_POST_ANALYTIC_METRICS_ON_ESCALATION_POLICIES",
    POST_CHANGE_EVENTS: "PAGERDUTY_POST_CHANGE_EVENTS",
    POST_EVENT_ORCHESTRATION_CACHE_VARIABLES: "PAGERDUTY_POST_EVENT_ORCHESTRATION_CACHE_VARIABLES",
    POST_INCIDENTS_ANALYTICS_WITH_FILTERS: "PAGERDUTY_POST_INCIDENTS_ANALYTICS_WITH_FILTERS",
    POST_INCIDENTS_METRICS_FILTERED_BY_SERVICE: "PAGERDUTY_POST_INCIDENTS_METRICS_FILTERED_BY_SERVICE",
    POST_INCIDENT_METRICS: "PAGERDUTY_POST_INCIDENT_METRICS",
    POST_INCIDENT_METRICS_WITH_FILTERS: "PAGERDUTY_POST_INCIDENT_METRICS_WITH_FILTERS",
    POST_INCIDENT_NOTE_USING_ID: "PAGERDUTY_POST_INCIDENT_NOTE_USING_ID",
    POST_INCIDENT_STATUS_UPDATE: "PAGERDUTY_POST_INCIDENT_STATUS_UPDATE",
    POST_RESPONDER_INCIDENTS_WITH_FILTERS: "PAGERDUTY_POST_RESPONDER_INCIDENTS_WITH_FILTERS",
    POST_SERVICE_AUTOMATION_ACTION: "PAGERDUTY_POST_SERVICE_AUTOMATION_ACTION",
    POST_TEAM_NOTIFICATION_SUBSCRIPTION: "PAGERDUTY_POST_TEAM_NOTIFICATION_SUBSCRIPTION",
    POST_TEAM_TO_RUNNER: "PAGERDUTY_POST_TEAM_TO_RUNNER",
    PREVIEW_SCHEDULE_OBJECT: "PAGERDUTY_PREVIEW_SCHEDULE_OBJECT",
    RENDER_TEMPLATE_FOR_INCIDENT: "PAGERDUTY_RENDER_TEMPLATE_FOR_INCIDENT",
    RETRIEVE_ABILITIES_LIST: "PAGERDUTY_RETRIEVE_ABILITIES_LIST",
    RETRIEVE_ABILITY_BY_ID: "PAGERDUTY_RETRIEVE_ABILITY_BY_ID",
    RETRIEVE_ACTION_SERVICES_BY_ID: "PAGERDUTY_RETRIEVE_ACTION_SERVICES_BY_ID",
    RETRIEVE_ACTIVE_SERVICE_EVENT_BY_ID: "PAGERDUTY_RETRIEVE_ACTIVE_SERVICE_EVENT_BY_ID",
    RETRIEVE_ADDON_BY_ID: "PAGERDUTY_RETRIEVE_ADDON_BY_ID",
    RETRIEVE_ALERT_GROUPING_SETTINGS_BY_ID: "PAGERDUTY_RETRIEVE_ALERT_GROUPING_SETTINGS_BY_ID",
    RETRIEVE_AUTOMATION_ACTION_INVOCATIONS: "PAGERDUTY_RETRIEVE_AUTOMATION_ACTION_INVOCATIONS",
    RETRIEVE_AUTOMATION_ACTION_INVOCATION_BY_ID: "PAGERDUTY_RETRIEVE_AUTOMATION_ACTION_INVOCATION_BY_ID",
    RETRIEVE_AUTOMATION_ACTION_RUNNER_TEAMS: "PAGERDUTY_RETRIEVE_AUTOMATION_ACTION_RUNNER_TEAMS",
    RETRIEVE_AUTOMATION_SERVICE_ACTION: "PAGERDUTY_RETRIEVE_AUTOMATION_SERVICE_ACTION",
    RETRIEVE_BUSINESS_SERVICES: "PAGERDUTY_RETRIEVE_BUSINESS_SERVICES",
    RETRIEVE_BUSINESS_SERVICE_BY_ID: "PAGERDUTY_RETRIEVE_BUSINESS_SERVICE_BY_ID",
    RETRIEVE_BUSINESS_SERVICE_DEPENDENCIES: "PAGERDUTY_RETRIEVE_BUSINESS_SERVICE_DEPENDENCIES",
    RETRIEVE_BUSINESS_SERVICE_IMPACTORS: "PAGERDUTY_RETRIEVE_BUSINESS_SERVICE_IMPACTORS",
    RETRIEVE_BUSINESS_SERVICE_IMPACTS: "PAGERDUTY_RETRIEVE_BUSINESS_SERVICE_IMPACTS",
    RETRIEVE_CHANGE_EVENT_BY_ID: "PAGERDUTY_RETRIEVE_CHANGE_EVENT_BY_ID",
    RETRIEVE_CHANGE_EVENT_INFORMATION: "PAGERDUTY_RETRIEVE_CHANGE_EVENT_INFORMATION",
    RETRIEVE_CURRENT_USER_INFO: "PAGERDUTY_RETRIEVE_CURRENT_USER_INFO",
    RETRIEVE_ENTITY_TAGS_BY_ID: "PAGERDUTY_RETRIEVE_ENTITY_TAGS_BY_ID",
    RETRIEVE_ENTITY_TAGS_BY_ID_AND_TYPE: "PAGERDUTY_RETRIEVE_ENTITY_TAGS_BY_ID_AND_TYPE",
    RETRIEVE_EVENT_ORCHESTRATION_ROUTER_BY_ID: "PAGERDUTY_RETRIEVE_EVENT_ORCHESTRATION_ROUTER_BY_ID",
    RETRIEVE_EXTENSION_BY_ID: "PAGERDUTY_RETRIEVE_EXTENSION_BY_ID",
    RETRIEVE_EXTENSION_LIST: "PAGERDUTY_RETRIEVE_EXTENSION_LIST",
    RETRIEVE_INCIDENT_BY_INCIDENT_ID: "PAGERDUTY_RETRIEVE_INCIDENT_BY_INCIDENT_ID",
    RETRIEVE_INCIDENT_CUSTOM_FIELDS: "PAGERDUTY_RETRIEVE_INCIDENT_CUSTOM_FIELDS",
    RETRIEVE_INCIDENT_CUSTOM_FIELD_VALUES: "PAGERDUTY_RETRIEVE_INCIDENT_CUSTOM_FIELD_VALUES",
    RETRIEVE_INCIDENT_FIELD_OPTIONS: "PAGERDUTY_RETRIEVE_INCIDENT_FIELD_OPTIONS",
    RETRIEVE_INCIDENT_WORKFLOW_BY_ID: "PAGERDUTY_RETRIEVE_INCIDENT_WORKFLOW_BY_ID",
    RETRIEVE_INCIDENT_WORKFLOW_TRIGGERS: "PAGERDUTY_RETRIEVE_INCIDENT_WORKFLOW_TRIGGERS",
    RETRIEVE_INTEGRATION_BY_SERVICE_ID: "PAGERDUTY_RETRIEVE_INTEGRATION_BY_SERVICE_ID",
    RETRIEVE_LICENSE_ALLOCATIONS: "PAGERDUTY_RETRIEVE_LICENSE_ALLOCATIONS",
    RETRIEVE_LIST_OF_SERVICES: "PAGERDUTY_RETRIEVE_LIST_OF_SERVICES",
    RETRIEVE_LOG_ENTRY_BY_ID: "PAGERDUTY_RETRIEVE_LOG_ENTRY_BY_ID",
    RETRIEVE_MAINTENANCE_WINDOWS: "PAGERDUTY_RETRIEVE_MAINTENANCE_WINDOWS",
    RETRIEVE_NOTES_FOR_INCIDENT: "PAGERDUTY_RETRIEVE_NOTES_FOR_INCIDENT",
    RETRIEVE_NOTIFICATIONS: "PAGERDUTY_RETRIEVE_NOTIFICATIONS",
    RETRIEVE_ONCALL_HANDOFF_NOTIFICATION_RULE: "PAGERDUTY_RETRIEVE_ONCALL_HANDOFF_NOTIFICATION_RULE",
    RETRIEVE_ONCALL_HANDOFF_NOTIFICATION_RULES: "PAGERDUTY_RETRIEVE_ONCALL_HANDOFF_NOTIFICATION_RULES",
    RETRIEVE_ONCALL_LIST: "PAGERDUTY_RETRIEVE_ONCALL_LIST",
    RETRIEVE_PAST_INCIDENTS: "PAGERDUTY_RETRIEVE_PAST_INCIDENTS",
    RETRIEVE_POSTMORTEM_BY_ID: "PAGERDUTY_RETRIEVE_POSTMORTEM_BY_ID",
    RETRIEVE_RESPONSE_PLAY_BY_ID: "PAGERDUTY_RETRIEVE_RESPONSE_PLAY_BY_ID",
    RETRIEVE_RULESET_BY_ID: "PAGERDUTY_RETRIEVE_RULESET_BY_ID",
    RETRIEVE_RULESET_LIST: "PAGERDUTY_RETRIEVE_RULESET_LIST",
    RETRIEVE_RULES_FOR_SERVICE_ID: "PAGERDUTY_RETRIEVE_RULES_FOR_SERVICE_ID",
    RETRIEVE_RULES_FROM_RULESET_BY_ID: "PAGERDUTY_RETRIEVE_RULES_FROM_RULESET_BY_ID",
    RETRIEVE_RULE_BY_SERVICE_ID: "PAGERDUTY_RETRIEVE_RULE_BY_SERVICE_ID",
    RETRIEVE_SCHEDULE_AUDIT_RECORDS_BY_ID: "PAGERDUTY_RETRIEVE_SCHEDULE_AUDIT_RECORDS_BY_ID",
    RETRIEVE_SCHEDULE_BY_ID: "PAGERDUTY_RETRIEVE_SCHEDULE_BY_ID",
    RETRIEVE_SCHEDULE_OVERRIDE_BY_ID: "PAGERDUTY_RETRIEVE_SCHEDULE_OVERRIDE_BY_ID",
    RETRIEVE_SERVICE_AUDIT_RECORDS_BY_ID: "PAGERDUTY_RETRIEVE_SERVICE_AUDIT_RECORDS_BY_ID",
    RETRIEVE_SERVICE_BY_ID: "PAGERDUTY_RETRIEVE_SERVICE_BY_ID",
    RETRIEVE_SERVICE_CHANGE_EVENTS_BY_ID: "PAGERDUTY_RETRIEVE_SERVICE_CHANGE_EVENTS_BY_ID",
    RETRIEVE_SERVICE_IMPACTS_FROM_STATUS_DASHBOARDS: "PAGERDUTY_RETRIEVE_SERVICE_IMPACTS_FROM_STATUS_DASHBOARDS",
    RETRIEVE_SERVICE_STATUS_BY_ID: "PAGERDUTY_RETRIEVE_SERVICE_STATUS_BY_ID",
    RETRIEVE_SERVICE_STATUS_PAGE: "PAGERDUTY_RETRIEVE_SERVICE_STATUS_PAGE",
    RETRIEVE_STANDARDS_LIST: "PAGERDUTY_RETRIEVE_STANDARDS_LIST",
    RETRIEVE_STANDARDS_SCORES_BY_RESOURCE_TYPE: "PAGERDUTY_RETRIEVE_STANDARDS_SCORES_BY_RESOURCE_TYPE",
    RETRIEVE_STANDARDS_SCORES_BY_RESOURCE_TYPE_AND_ID: "PAGERDUTY_RETRIEVE_STANDARDS_SCORES_BY_RESOURCE_TYPE_AND_ID",
    RETRIEVE_STATUS_DASHBOARDS_INFORMATION: "PAGERDUTY_RETRIEVE_STATUS_DASHBOARDS_INFORMATION",
    RETRIEVE_STATUS_DASHBOARD_BY_SLUG: "PAGERDUTY_RETRIEVE_STATUS_DASHBOARD_BY_SLUG",
    RETRIEVE_STATUS_PAGES_STATUS: "PAGERDUTY_RETRIEVE_STATUS_PAGES_STATUS",
    RETRIEVE_STATUS_PAGE_POST: "PAGERDUTY_RETRIEVE_STATUS_PAGE_POST",
    RETRIEVE_STATUS_PAGE_SEVERITIES_BY_ID: "PAGERDUTY_RETRIEVE_STATUS_PAGE_SEVERITIES_BY_ID",
    RETRIEVE_STATUS_PAGE_SUBSCRIPTION_BY_ID: "PAGERDUTY_RETRIEVE_STATUS_PAGE_SUBSCRIPTION_BY_ID",
    RETRIEVE_TAGS: "PAGERDUTY_RETRIEVE_TAGS",
    RETRIEVE_TAG_BY_ID: "PAGERDUTY_RETRIEVE_TAG_BY_ID",
    RETRIEVE_TEAM_AUDIT_RECORDS: "PAGERDUTY_RETRIEVE_TEAM_AUDIT_RECORDS",
    RETRIEVE_TEAM_DETAILS_BY_ID: "PAGERDUTY_RETRIEVE_TEAM_DETAILS_BY_ID",
    RETRIEVE_TEAM_LIST: "PAGERDUTY_RETRIEVE_TEAM_LIST",
    RETRIEVE_TECHNICAL_SERVICE_DETAILS: "PAGERDUTY_RETRIEVE_TECHNICAL_SERVICE_DETAILS",
    RETRIEVE_TEMPLATE_FIELDS: "PAGERDUTY_RETRIEVE_TEMPLATE_FIELDS",
    RETRIEVE_UNROUTED_EVENT_ORCHESTRATION_BY_ID: "PAGERDUTY_RETRIEVE_UNROUTED_EVENT_ORCHESTRATION_BY_ID",
    RETRIEVE_USERS_BY_SCHEDULE_ID: "PAGERDUTY_RETRIEVE_USERS_BY_SCHEDULE_ID",
    RETRIEVE_USERS_LIST: "PAGERDUTY_RETRIEVE_USERS_LIST",
    RETRIEVE_USER_AUDIT_RECORDS_BY_ID: "PAGERDUTY_RETRIEVE_USER_AUDIT_RECORDS_BY_ID",
    RETRIEVE_USER_BY_ID: "PAGERDUTY_RETRIEVE_USER_BY_ID",
    RETRIEVE_USER_CONTACT_METHODS_VIA_ID: "PAGERDUTY_RETRIEVE_USER_CONTACT_METHODS_VIA_ID",
    RETRIEVE_USER_LICENSE_INFORMATION: "PAGERDUTY_RETRIEVE_USER_LICENSE_INFORMATION",
    RETRIEVE_USER_NOTIFICATION_RULE: "PAGERDUTY_RETRIEVE_USER_NOTIFICATION_RULE",
    RETRIEVE_USER_NOTIFICATION_RULES: "PAGERDUTY_RETRIEVE_USER_NOTIFICATION_RULES",
    RETRIEVE_USER_STATUS_UPDATE_NOTIFICATION_RULE: "PAGERDUTY_RETRIEVE_USER_STATUS_UPDATE_NOTIFICATION_RULE",
    RETRIEVE_VENDOR_BY_ID: "PAGERDUTY_RETRIEVE_VENDOR_BY_ID",
    RETRIEVE_WEBHOOK_SUBSCRIPTIONS: "PAGERDUTY_RETRIEVE_WEBHOOK_SUBSCRIPTIONS",
    SEND_RESPONDER_REQUESTS_FOR_INCIDENTS: "PAGERDUTY_SEND_RESPONDER_REQUESTS_FOR_INCIDENTS",
    SET_BUSINESS_SERVICE_IMPACT_STATUS: "PAGERDUTY_SET_BUSINESS_SERVICE_IMPACT_STATUS",
    SET_GLOBAL_PRIORITY_THRESHOLD: "PAGERDUTY_SET_GLOBAL_PRIORITY_THRESHOLD",
    SNOOZE_INCIDENT_BY_DURATION: "PAGERDUTY_SNOOZE_INCIDENT_BY_DURATION",
    SUBSCRIBE_ENTITIES_TO_BUSINESS_SERVICES: "PAGERDUTY_SUBSCRIBE_ENTITIES_TO_BUSINESS_SERVICES",
    SUBSCRIBE_ENTITIES_TO_INCIDENT_STATUS_UPDATES: "PAGERDUTY_SUBSCRIBE_ENTITIES_TO_INCIDENT_STATUS_UPDATES",
    SUBSCRIBE_TO_USER_NOTIFICATIONS: "PAGERDUTY_SUBSCRIBE_TO_USER_NOTIFICATIONS",
    UNSUBSCRIBE_BUSINESS_SERVICE_ENTITY: "PAGERDUTY_UNSUBSCRIBE_BUSINESS_SERVICE_ENTITY",
    UNSUBSCRIBE_FROM_INCIDENT_STATUS_UPDATES: "PAGERDUTY_UNSUBSCRIBE_FROM_INCIDENT_STATUS_UPDATES",
    UNSUBSCRIBE_TEAM_NOTIFICATION_SUBSCRIPTIONS: "PAGERDUTY_UNSUBSCRIBE_TEAM_NOTIFICATION_SUBSCRIPTIONS",
    UNSUBSCRIBE_USER_NOTIFICATION_SUBSCRIPTIONS: "PAGERDUTY_UNSUBSCRIBE_USER_NOTIFICATION_SUBSCRIPTIONS",
    UPDATE_ADD_ON_BY_ID: "PAGERDUTY_UPDATE_ADD_ON_BY_ID",
    UPDATE_ALERT_GROUPING_SETTINGS_BY_ID: "PAGERDUTY_UPDATE_ALERT_GROUPING_SETTINGS_BY_ID",
    UPDATE_ALERT_IN_INCIDENT: "PAGERDUTY_UPDATE_ALERT_IN_INCIDENT",
    UPDATE_AN_INCIDENT_TYPE: "PAGERDUTY_UPDATE_AN_INCIDENT_TYPE",
    UPDATE_AUTOMATION_RUNNER_INFO: "PAGERDUTY_UPDATE_AUTOMATION_RUNNER_INFO",
    UPDATE_A_CUSTOM_FIELD_FOR_AN_INCIDENT_TYPE: "PAGERDUTY_UPDATE_A_CUSTOM_FIELD_FOR_AN_INCIDENT_TYPE",
    UPDATE_A_FIELD_OPTION_FOR_A_CUSTOM_FIELD: "PAGERDUTY_UPDATE_A_FIELD_OPTION_FOR_A_CUSTOM_FIELD",
    UPDATE_A_TRIGGER: "PAGERDUTY_UPDATE_A_TRIGGER",
    UPDATE_BUSINESS_SERVICE_BY_ID: "PAGERDUTY_UPDATE_BUSINESS_SERVICE_BY_ID",
    UPDATE_CHANGE_EVENT_BY_ID: "PAGERDUTY_UPDATE_CHANGE_EVENT_BY_ID",
    UPDATE_CUSTOM_FIELD_DISPLAY_NAME: "PAGERDUTY_UPDATE_CUSTOM_FIELD_DISPLAY_NAME",
    UPDATE_CUSTOM_FIELD_OPTION: "PAGERDUTY_UPDATE_CUSTOM_FIELD_OPTION",
    UPDATE_ESCALATION_POLICY_BY_ID: "PAGERDUTY_UPDATE_ESCALATION_POLICY_BY_ID",
    UPDATE_ESCALATION_POLICY_FOR_TEAM: "PAGERDUTY_UPDATE_ESCALATION_POLICY_FOR_TEAM",
    UPDATE_EVENT_ORCHESTRATION_BY_ID: "PAGERDUTY_UPDATE_EVENT_ORCHESTRATION_BY_ID",
    UPDATE_EVENT_ORCHESTRATION_CACHE_VARIABLE: "PAGERDUTY_UPDATE_EVENT_ORCHESTRATION_CACHE_VARIABLE",
    UPDATE_EVENT_RULE_BY_ID: "PAGERDUTY_UPDATE_EVENT_RULE_BY_ID",
    UPDATE_EVENT_RULE_FOR_SERVICE: "PAGERDUTY_UPDATE_EVENT_RULE_FOR_SERVICE",
    UPDATE_EXTENSION_BY_ID: "PAGERDUTY_UPDATE_EXTENSION_BY_ID",
    UPDATE_GLOBAL_ORCHESTRATION: "PAGERDUTY_UPDATE_GLOBAL_ORCHESTRATION",
    UPDATE_INCIDENT_ALERTS: "PAGERDUTY_UPDATE_INCIDENT_ALERTS",
    UPDATE_INCIDENT_BY_ID: "PAGERDUTY_UPDATE_INCIDENT_BY_ID",
    UPDATE_INCIDENT_CUSTOM_FIELD_VALUES: "PAGERDUTY_UPDATE_INCIDENT_CUSTOM_FIELD_VALUES",
    UPDATE_INCIDENT_DETAILS: "PAGERDUTY_UPDATE_INCIDENT_DETAILS",
    UPDATE_INCIDENT_WORKFLOW: "PAGERDUTY_UPDATE_INCIDENT_WORKFLOW",
    UPDATE_INTEGRATION_BY_ID_AND_INTEGRATION_ID: "PAGERDUTY_UPDATE_INTEGRATION_BY_ID_AND_INTEGRATION_ID",
    UPDATE_INTEGRATION_LABEL: "PAGERDUTY_UPDATE_INTEGRATION_LABEL",
    UPDATE_LOG_ENTRY_CHANNEL: "PAGERDUTY_UPDATE_LOG_ENTRY_CHANNEL",
    UPDATE_MAINTENANCE_WINDOW_BY_ID: "PAGERDUTY_UPDATE_MAINTENANCE_WINDOW_BY_ID",
    UPDATE_ONCALL_HANDOFF_NOTIFICATION_RULE: "PAGERDUTY_UPDATE_ONCALL_HANDOFF_NOTIFICATION_RULE",
    UPDATE_ORCHESTRATION_ROUTER_DETAILS: "PAGERDUTY_UPDATE_ORCHESTRATION_ROUTER_DETAILS",
    UPDATE_RESPONSE_PLAY_BY_ID: "PAGERDUTY_UPDATE_RESPONSE_PLAY_BY_ID",
    UPDATE_RULESET_BY_ID: "PAGERDUTY_UPDATE_RULESET_BY_ID",
    UPDATE_SCHEDULE_BY_ID: "PAGERDUTY_UPDATE_SCHEDULE_BY_ID",
    UPDATE_SERVICE_BY_ID: "PAGERDUTY_UPDATE_SERVICE_BY_ID",
    UPDATE_SERVICE_ORCHESTRATION_ACTIVE_STATUS: "PAGERDUTY_UPDATE_SERVICE_ORCHESTRATION_ACTIVE_STATUS",
    UPDATE_STANDARD_BY_ID: "PAGERDUTY_UPDATE_STANDARD_BY_ID",
    UPDATE_STATUS_PAGE_POST: "PAGERDUTY_UPDATE_STATUS_PAGE_POST",
    UPDATE_STATUS_PAGE_POST_INFO: "PAGERDUTY_UPDATE_STATUS_PAGE_POST_INFO",
    UPDATE_STATUS_PAGE_POST_POSTMORTEM: "PAGERDUTY_UPDATE_STATUS_PAGE_POST_POSTMORTEM",
    UPDATE_STATUS_PAGE_POST_RESOURCE: "PAGERDUTY_UPDATE_STATUS_PAGE_POST_RESOURCE",
    UPDATE_TEAM_BY_ID: "PAGERDUTY_UPDATE_TEAM_BY_ID",
    UPDATE_TEMPLATE_BY_ID: "PAGERDUTY_UPDATE_TEMPLATE_BY_ID",
    UPDATE_THE_SERVICE_ORCHESTRATION_FOR_A_SERVICE: "PAGERDUTY_UPDATE_THE_SERVICE_ORCHESTRATION_FOR_A_SERVICE",
    UPDATE_UNROUTED_ORCHESTRATION_RULES: "PAGERDUTY_UPDATE_UNROUTED_ORCHESTRATION_RULES",
    UPDATE_USER_INFORMATION: "PAGERDUTY_UPDATE_USER_INFORMATION",
    UPDATE_USER_NOTIFICATION_RULE_BY_ID: "PAGERDUTY_UPDATE_USER_NOTIFICATION_RULE_BY_ID",
    UPDATE_USER_ROLE_ON_TEAM: "PAGERDUTY_UPDATE_USER_ROLE_ON_TEAM",
    UPDATE_WORKFLOW_INTEGRATION_CONNECTION: "PAGERDUTY_UPDATE_WORKFLOW_INTEGRATION_CONNECTION",
  },
  triggerTypes: {},
}
