// @composio/core/generated/peopledatalabs.ts
/**
 * Map of Composio's PEOPLEDATALABS toolkit.
 */
export const PEOPLEDATALABS = {
  slug: "peopledatalabs",
  tools: {
    AUTOCOMPLETE_FIELD_SUGGESTIONS: "PEOPLEDATALABS_AUTOCOMPLETE_FIELD_SUGGESTIONS",
    CLEAN_COMPANY_DATA: "PEOPLEDATALABS_CLEAN_COMPANY_DATA",
    CLEAN_LOCATION_DATA: "PEOPLEDATALABS_CLEAN_LOCATION_DATA",
    CLEAN_SCHOOL_DATA: "PEOPLEDATALABS_CLEAN_SCHOOL_DATA",
    COMPANY_SEARCH_ELASTIC: "PEOPLEDATALABS_COMPANY_SEARCH_ELASTIC",
    ENRICH_COMPANY_DATA: "PEOPLEDATALABS_ENRICH_COMPANY_DATA",
    ENRICH_IP_DATA: "PEOPLEDATALABS_ENRICH_IP_DATA",
    ENRICH_JOB_TITLE_DATA: "PEOPLEDATALABS_ENRICH_JOB_TITLE_DATA",
    ENRICH_PERSON_DATA: "PEOPLEDATALABS_ENRICH_PERSON_DATA",
    ENRICH_SKILL_DATA: "PEOPLEDATALABS_ENRICH_SKILL_DATA",
    GET_COLUMN_DETAILS: "PEOPLEDATALABS_GET_COLUMN_DETAILS",
    GET_SCHEMA: "PEOPLEDATALABS_GET_SCHEMA",
    IDENTIFY_PERSON_DATA: "PEOPLEDATALABS_IDENTIFY_PERSON_DATA",
    NATURAL_LANGUAGE_QUERY_ACTION: "PEOPLEDATALABS_NATURAL_LANGUAGE_QUERY_ACTION",
    PEOPLE_SEARCH_ELASTIC: "PEOPLEDATALABS_PEOPLE_SEARCH_ELASTIC",
  },
  triggerTypes: {},
}
