// @composio/core/generated/pipedrive.ts
/**
 * Map of Composio's PIPEDRIVE toolkit.
 */
export const PIPEDRIVE = {
  slug: "pipedrive",
  tools: {
    ADD_AN_ACTIVITY: "PIPEDRIVE_ADD_AN_ACTIVITY",
    ADD_AN_INSTALLMENT_SUBSCRIPTION: "PIPEDRIVE_ADD_AN_INSTALLMENT_SUBSCRIPTION",
    ADD_AN_ORGANIZATION: "PIPEDRIVE_ADD_AN_ORGANIZATION",
    ADD_A_CALL_LOG: "PIPEDRIVE_ADD_A_CALL_LOG",
    ADD_A_CHANNEL: "PIPEDRIVE_ADD_A_CHANNEL",
    ADD_A_COMMENT_TO_A_NOTE: "PIPEDRIVE_ADD_A_COMMENT_TO_A_NOTE",
    ADD_A_DEAL: "PIPEDRIVE_ADD_A_DEAL",
    ADD_A_FOLLOWER_TO_AN_ORGANIZATION: "PIPEDRIVE_ADD_A_FOLLOWER_TO_AN_ORGANIZATION",
    ADD_A_FOLLOWER_TO_A_DEAL: "PIPEDRIVE_ADD_A_FOLLOWER_TO_A_DEAL",
    ADD_A_FOLLOWER_TO_A_PERSON: "PIPEDRIVE_ADD_A_FOLLOWER_TO_A_PERSON",
    ADD_A_FOLLOWER_TO_A_PRODUCT: "PIPEDRIVE_ADD_A_FOLLOWER_TO_A_PRODUCT",
    ADD_A_LEAD: "PIPEDRIVE_ADD_A_LEAD",
    ADD_A_LEAD_LABEL: "PIPEDRIVE_ADD_A_LEAD_LABEL",
    ADD_A_NEW_DEAL_FIELD: "PIPEDRIVE_ADD_A_NEW_DEAL_FIELD",
    ADD_A_NEW_FILTER: "PIPEDRIVE_ADD_A_NEW_FILTER",
    ADD_A_NEW_GOAL: "PIPEDRIVE_ADD_A_NEW_GOAL",
    ADD_A_NEW_ORGANIZATION_FIELD: "PIPEDRIVE_ADD_A_NEW_ORGANIZATION_FIELD",
    ADD_A_NEW_PERSON_FIELD: "PIPEDRIVE_ADD_A_NEW_PERSON_FIELD",
    ADD_A_NEW_PIPELINE: "PIPEDRIVE_ADD_A_NEW_PIPELINE",
    ADD_A_NEW_PRODUCT_FIELD: "PIPEDRIVE_ADD_A_NEW_PRODUCT_FIELD",
    ADD_A_NEW_STAGE: "PIPEDRIVE_ADD_A_NEW_STAGE",
    ADD_A_NEW_TEAM: "PIPEDRIVE_ADD_A_NEW_TEAM",
    ADD_A_NEW_USER: "PIPEDRIVE_ADD_A_NEW_USER",
    ADD_A_NOTE: "PIPEDRIVE_ADD_A_NOTE",
    ADD_A_PARTICIPANT_TO_A_DEAL: "PIPEDRIVE_ADD_A_PARTICIPANT_TO_A_DEAL",
    ADD_A_PERSON: "PIPEDRIVE_ADD_A_PERSON",
    ADD_A_PRODUCT: "PIPEDRIVE_ADD_A_PRODUCT",
    ADD_A_PRODUCT_TO_A_DEAL: "PIPEDRIVE_ADD_A_PRODUCT_TO_A_DEAL",
    ADD_A_PROJECT: "PIPEDRIVE_ADD_A_PROJECT",
    ADD_A_RECURRING_SUBSCRIPTION: "PIPEDRIVE_ADD_A_RECURRING_SUBSCRIPTION",
    ADD_A_ROLE: "PIPEDRIVE_ADD_A_ROLE",
    ADD_A_TASK: "PIPEDRIVE_ADD_A_TASK",
    ADD_FILE: "PIPEDRIVE_ADD_FILE",
    ADD_NEW_ACTIVITY_TYPE: "PIPEDRIVE_ADD_NEW_ACTIVITY_TYPE",
    ADD_OR_UPDATE_ROLE_SETTING: "PIPEDRIVE_ADD_OR_UPDATE_ROLE_SETTING",
    ADD_PERSON_PICTURE: "PIPEDRIVE_ADD_PERSON_PICTURE",
    ADD_ROLE_ASSIGNMENT: "PIPEDRIVE_ADD_ROLE_ASSIGNMENT",
    ADD_USERS_TO_A_TEAM: "PIPEDRIVE_ADD_USERS_TO_A_TEAM",
    ARCHIVE_A_PROJECT: "PIPEDRIVE_ARCHIVE_A_PROJECT",
    ATTACH_AN_AUDIO_FILE_TO_THE_CALL_LOG: "PIPEDRIVE_ATTACH_AN_AUDIO_FILE_TO_THE_CALL_LOG",
    CANCEL_A_RECURRING_SUBSCRIPTION: "PIPEDRIVE_CANCEL_A_RECURRING_SUBSCRIPTION",
    CREATE_AN_ORGANIZATION_RELATIONSHIP: "PIPEDRIVE_CREATE_AN_ORGANIZATION_RELATIONSHIP",
    CREATE_A_NEW_WEBHOOK: "PIPEDRIVE_CREATE_A_NEW_WEBHOOK",
    CREATE_A_REMOTE_FILE_AND_LINK_IT_TO_AN_ITEM: "PIPEDRIVE_CREATE_A_REMOTE_FILE_AND_LINK_IT_TO_AN_ITEM",
    DELETE_AN_ACTIVITY: "PIPEDRIVE_DELETE_AN_ACTIVITY",
    DELETE_AN_ACTIVITY_TYPE: "PIPEDRIVE_DELETE_AN_ACTIVITY_TYPE",
    DELETE_AN_ATTACHED_PRODUCT_FROM_A_DEAL: "PIPEDRIVE_DELETE_AN_ATTACHED_PRODUCT_FROM_A_DEAL",
    DELETE_AN_ORGANIZATION: "PIPEDRIVE_DELETE_AN_ORGANIZATION",
    DELETE_AN_ORGANIZATION_FIELD: "PIPEDRIVE_DELETE_AN_ORGANIZATION_FIELD",
    DELETE_AN_ORGANIZATION_RELATIONSHIP: "PIPEDRIVE_DELETE_AN_ORGANIZATION_RELATIONSHIP",
    DELETE_A_CALL_LOG: "PIPEDRIVE_DELETE_A_CALL_LOG",
    DELETE_A_CHANNEL: "PIPEDRIVE_DELETE_A_CHANNEL",
    DELETE_A_COMMENT_RELATED_TO_A_NOTE: "PIPEDRIVE_DELETE_A_COMMENT_RELATED_TO_A_NOTE",
    DELETE_A_CONVERSATION: "PIPEDRIVE_DELETE_A_CONVERSATION",
    DELETE_A_DEAL: "PIPEDRIVE_DELETE_A_DEAL",
    DELETE_A_DEAL_FIELD: "PIPEDRIVE_DELETE_A_DEAL_FIELD",
    DELETE_A_FILE: "PIPEDRIVE_DELETE_A_FILE",
    DELETE_A_FILTER: "PIPEDRIVE_DELETE_A_FILTER",
    DELETE_A_FOLLOWER_FROM_AN_ORGANIZATION: "PIPEDRIVE_DELETE_A_FOLLOWER_FROM_AN_ORGANIZATION",
    DELETE_A_FOLLOWER_FROM_A_DEAL: "PIPEDRIVE_DELETE_A_FOLLOWER_FROM_A_DEAL",
    DELETE_A_FOLLOWER_FROM_A_PERSON: "PIPEDRIVE_DELETE_A_FOLLOWER_FROM_A_PERSON",
    DELETE_A_FOLLOWER_FROM_A_PRODUCT: "PIPEDRIVE_DELETE_A_FOLLOWER_FROM_A_PRODUCT",
    DELETE_A_LEAD: "PIPEDRIVE_DELETE_A_LEAD",
    DELETE_A_LEAD_LABEL: "PIPEDRIVE_DELETE_A_LEAD_LABEL",
    DELETE_A_NOTE: "PIPEDRIVE_DELETE_A_NOTE",
    DELETE_A_PARTICIPANT_FROM_A_DEAL: "PIPEDRIVE_DELETE_A_PARTICIPANT_FROM_A_DEAL",
    DELETE_A_PERSON: "PIPEDRIVE_DELETE_A_PERSON",
    DELETE_A_PERSON_FIELD: "PIPEDRIVE_DELETE_A_PERSON_FIELD",
    DELETE_A_PIPELINE: "PIPEDRIVE_DELETE_A_PIPELINE",
    DELETE_A_PRODUCT: "PIPEDRIVE_DELETE_A_PRODUCT",
    DELETE_A_PRODUCT_FIELD: "PIPEDRIVE_DELETE_A_PRODUCT_FIELD",
    DELETE_A_PROJECT: "PIPEDRIVE_DELETE_A_PROJECT",
    DELETE_A_ROLE: "PIPEDRIVE_DELETE_A_ROLE",
    DELETE_A_ROLE_ASSIGNMENT: "PIPEDRIVE_DELETE_A_ROLE_ASSIGNMENT",
    DELETE_A_STAGE: "PIPEDRIVE_DELETE_A_STAGE",
    DELETE_A_SUBSCRIPTION: "PIPEDRIVE_DELETE_A_SUBSCRIPTION",
    DELETE_A_TASK: "PIPEDRIVE_DELETE_A_TASK",
    DELETE_EXISTING_GOAL: "PIPEDRIVE_DELETE_EXISTING_GOAL",
    DELETE_EXISTING_WEBHOOK: "PIPEDRIVE_DELETE_EXISTING_WEBHOOK",
    DELETE_MAIL_THREAD: "PIPEDRIVE_DELETE_MAIL_THREAD",
    DELETE_MULTIPLE_ACTIVITIES_IN_BULK: "PIPEDRIVE_DELETE_MULTIPLE_ACTIVITIES_IN_BULK",
    DELETE_MULTIPLE_ACTIVITY_TYPES_IN_BULK: "PIPEDRIVE_DELETE_MULTIPLE_ACTIVITY_TYPES_IN_BULK",
    DELETE_MULTIPLE_DEALS_IN_BULK: "PIPEDRIVE_DELETE_MULTIPLE_DEALS_IN_BULK",
    DELETE_MULTIPLE_DEAL_FIELDS_IN_BULK: "PIPEDRIVE_DELETE_MULTIPLE_DEAL_FIELDS_IN_BULK",
    DELETE_MULTIPLE_FILTERS_IN_BULK: "PIPEDRIVE_DELETE_MULTIPLE_FILTERS_IN_BULK",
    DELETE_MULTIPLE_ORGANIZATIONS_IN_BULK: "PIPEDRIVE_DELETE_MULTIPLE_ORGANIZATIONS_IN_BULK",
    DELETE_MULTIPLE_ORGANIZATION_FIELDS_IN_BULK: "PIPEDRIVE_DELETE_MULTIPLE_ORGANIZATION_FIELDS_IN_BULK",
    DELETE_MULTIPLE_PERSONS_IN_BULK: "PIPEDRIVE_DELETE_MULTIPLE_PERSONS_IN_BULK",
    DELETE_MULTIPLE_PERSON_FIELDS_IN_BULK: "PIPEDRIVE_DELETE_MULTIPLE_PERSON_FIELDS_IN_BULK",
    DELETE_MULTIPLE_PRODUCT_FIELDS_IN_BULK: "PIPEDRIVE_DELETE_MULTIPLE_PRODUCT_FIELDS_IN_BULK",
    DELETE_MULTIPLE_STAGES_IN_BULK: "PIPEDRIVE_DELETE_MULTIPLE_STAGES_IN_BULK",
    DELETE_PERSON_PICTURE: "PIPEDRIVE_DELETE_PERSON_PICTURE",
    DELETE_USERS_FROM_A_TEAM: "PIPEDRIVE_DELETE_USERS_FROM_A_TEAM",
    DOWNLOAD_ONE_FILE: "PIPEDRIVE_DOWNLOAD_ONE_FILE",
    DUPLICATE_DEAL: "PIPEDRIVE_DUPLICATE_DEAL",
    ENUMERATE_ACCESSIBLE_USERS_FOR_LEAD: "PIPEDRIVE_ENUMERATE_ACCESSIBLE_USERS_FOR_LEAD",
    FIND_GOALS: "PIPEDRIVE_FIND_GOALS",
    FIND_SUBSCRIPTION_BY_DEAL: "PIPEDRIVE_FIND_SUBSCRIPTION_BY_DEAL",
    FIND_USERS_BY_NAME: "PIPEDRIVE_FIND_USERS_BY_NAME",
    GET_ALL_ACTIVITIES_ASSIGNED_TO_A_PARTICULAR_USER: "PIPEDRIVE_GET_ALL_ACTIVITIES_ASSIGNED_TO_A_PARTICULAR_USER",
    GET_ALL_ACTIVITIES_BETA: "PIPEDRIVE_GET_ALL_ACTIVITIES_BETA",
    GET_ALL_ACTIVITY_FIELDS: "PIPEDRIVE_GET_ALL_ACTIVITY_FIELDS",
    GET_ALL_ACTIVITY_TYPES: "PIPEDRIVE_GET_ALL_ACTIVITY_TYPES",
    GET_ALL_ADD_ONS_FOR_A_SINGLE_COMPANY: "PIPEDRIVE_GET_ALL_ADD_ONS_FOR_A_SINGLE_COMPANY",
    GET_ALL_CALL_LOGS_ASSIGNED_TO_A_PARTICULAR_USER: "PIPEDRIVE_GET_ALL_CALL_LOGS_ASSIGNED_TO_A_PARTICULAR_USER",
    GET_ALL_COMMENTS_FOR_A_NOTE: "PIPEDRIVE_GET_ALL_COMMENTS_FOR_A_NOTE",
    GET_ALL_DEALS: "PIPEDRIVE_GET_ALL_DEALS",
    GET_ALL_DEALS_BETA: "PIPEDRIVE_GET_ALL_DEALS_BETA",
    GET_ALL_DEAL_FIELDS: "PIPEDRIVE_GET_ALL_DEAL_FIELDS",
    GET_ALL_FILES: "PIPEDRIVE_GET_ALL_FILES",
    GET_ALL_FILTERS: "PIPEDRIVE_GET_ALL_FILTERS",
    GET_ALL_FILTER_HELPERS: "PIPEDRIVE_GET_ALL_FILTER_HELPERS",
    GET_ALL_LEADS: "PIPEDRIVE_GET_ALL_LEADS",
    GET_ALL_LEAD_LABELS: "PIPEDRIVE_GET_ALL_LEAD_LABELS",
    GET_ALL_LEAD_SOURCES: "PIPEDRIVE_GET_ALL_LEAD_SOURCES",
    GET_ALL_MAIL_MESSAGES_OF_MAIL_THREAD: "PIPEDRIVE_GET_ALL_MAIL_MESSAGES_OF_MAIL_THREAD",
    GET_ALL_NOTES: "PIPEDRIVE_GET_ALL_NOTES",
    GET_ALL_NOTE_FIELDS: "PIPEDRIVE_GET_ALL_NOTE_FIELDS",
    GET_ALL_ORGANIZATIONS: "PIPEDRIVE_GET_ALL_ORGANIZATIONS",
    GET_ALL_ORGANIZATIONS_BETA: "PIPEDRIVE_GET_ALL_ORGANIZATIONS_BETA",
    GET_ALL_ORGANIZATION_FIELDS: "PIPEDRIVE_GET_ALL_ORGANIZATION_FIELDS",
    GET_ALL_PAYMENTS_OF_A_SUBSCRIPTION: "PIPEDRIVE_GET_ALL_PAYMENTS_OF_A_SUBSCRIPTION",
    GET_ALL_PERMISSION_SETS: "PIPEDRIVE_GET_ALL_PERMISSION_SETS",
    GET_ALL_PERSONS: "PIPEDRIVE_GET_ALL_PERSONS",
    GET_ALL_PERSONS_BETA: "PIPEDRIVE_GET_ALL_PERSONS_BETA",
    GET_ALL_PERSON_FIELDS: "PIPEDRIVE_GET_ALL_PERSON_FIELDS",
    GET_ALL_PIPELINES: "PIPEDRIVE_GET_ALL_PIPELINES",
    GET_ALL_PRODUCTS: "PIPEDRIVE_GET_ALL_PRODUCTS",
    GET_ALL_PRODUCT_FIELDS: "PIPEDRIVE_GET_ALL_PRODUCT_FIELDS",
    GET_ALL_PROJECTS: "PIPEDRIVE_GET_ALL_PROJECTS",
    GET_ALL_PROJECT_BOARDS: "PIPEDRIVE_GET_ALL_PROJECT_BOARDS",
    GET_ALL_PROJECT_TEMPLATES: "PIPEDRIVE_GET_ALL_PROJECT_TEMPLATES",
    GET_ALL_RELATIONSHIPS_FOR_ORGANIZATION: "PIPEDRIVE_GET_ALL_RELATIONSHIPS_FOR_ORGANIZATION",
    GET_ALL_ROLES: "PIPEDRIVE_GET_ALL_ROLES",
    GET_ALL_STAGES: "PIPEDRIVE_GET_ALL_STAGES",
    GET_ALL_SUPPORTED_CURRENCIES: "PIPEDRIVE_GET_ALL_SUPPORTED_CURRENCIES",
    GET_ALL_TASKS: "PIPEDRIVE_GET_ALL_TASKS",
    GET_ALL_TEAMS: "PIPEDRIVE_GET_ALL_TEAMS",
    GET_ALL_TEAMS_OF_A_USER: "PIPEDRIVE_GET_ALL_TEAMS_OF_A_USER",
    GET_ALL_USERS: "PIPEDRIVE_GET_ALL_USERS",
    GET_ALL_USERS_IN_A_TEAM: "PIPEDRIVE_GET_ALL_USERS_IN_A_TEAM",
    GET_ALL_USER_CONNECTIONS: "PIPEDRIVE_GET_ALL_USER_CONNECTIONS",
    GET_ALL_WEBHOOKS: "PIPEDRIVE_GET_ALL_WEBHOOKS",
    GET_A_SINGLE_TEAM: "PIPEDRIVE_GET_A_SINGLE_TEAM",
    GET_CURRENT_USER_DATA: "PIPEDRIVE_GET_CURRENT_USER_DATA",
    GET_DEALS_CONVERSION_RATES_IN_PIPELINE: "PIPEDRIVE_GET_DEALS_CONVERSION_RATES_IN_PIPELINE",
    GET_DEALS_IN_A_PIPELINE: "PIPEDRIVE_GET_DEALS_IN_A_PIPELINE",
    GET_DEALS_IN_A_STAGE: "PIPEDRIVE_GET_DEALS_IN_A_STAGE",
    GET_DEALS_MOVEMENTS_IN_PIPELINE: "PIPEDRIVE_GET_DEALS_MOVEMENTS_IN_PIPELINE",
    GET_DEALS_SUMMARY: "PIPEDRIVE_GET_DEALS_SUMMARY",
    GET_DEALS_TIMELINE: "PIPEDRIVE_GET_DEALS_TIMELINE",
    GET_DEALS_WHERE_A_PRODUCT_IS_ATTACHED_TO: "PIPEDRIVE_GET_DEALS_WHERE_A_PRODUCT_IS_ATTACHED_TO",
    GET_DETAILS_OF_AN_ACTIVITY: "PIPEDRIVE_GET_DETAILS_OF_AN_ACTIVITY",
    GET_DETAILS_OF_AN_ORGANIZATION: "PIPEDRIVE_GET_DETAILS_OF_AN_ORGANIZATION",
    GET_DETAILS_OF_A_BOARD: "PIPEDRIVE_GET_DETAILS_OF_A_BOARD",
    GET_DETAILS_OF_A_CALL_LOG: "PIPEDRIVE_GET_DETAILS_OF_A_CALL_LOG",
    GET_DETAILS_OF_A_DEAL: "PIPEDRIVE_GET_DETAILS_OF_A_DEAL",
    GET_DETAILS_OF_A_PERSON: "PIPEDRIVE_GET_DETAILS_OF_A_PERSON",
    GET_DETAILS_OF_A_PHASE: "PIPEDRIVE_GET_DETAILS_OF_A_PHASE",
    GET_DETAILS_OF_A_PROJECT: "PIPEDRIVE_GET_DETAILS_OF_A_PROJECT",
    GET_DETAILS_OF_A_SUBSCRIPTION: "PIPEDRIVE_GET_DETAILS_OF_A_SUBSCRIPTION",
    GET_DETAILS_OF_A_TASK: "PIPEDRIVE_GET_DETAILS_OF_A_TASK",
    GET_DETAILS_OF_A_TEMPLATE: "PIPEDRIVE_GET_DETAILS_OF_A_TEMPLATE",
    GET_MAIL_THREADS: "PIPEDRIVE_GET_MAIL_THREADS",
    GET_ONE_COMMENT: "PIPEDRIVE_GET_ONE_COMMENT",
    GET_ONE_DEAL_FIELD: "PIPEDRIVE_GET_ONE_DEAL_FIELD",
    GET_ONE_FILE: "PIPEDRIVE_GET_ONE_FILE",
    GET_ONE_FILTER: "PIPEDRIVE_GET_ONE_FILTER",
    GET_ONE_LEAD: "PIPEDRIVE_GET_ONE_LEAD",
    GET_ONE_MAIL_MESSAGE: "PIPEDRIVE_GET_ONE_MAIL_MESSAGE",
    GET_ONE_MAIL_THREAD: "PIPEDRIVE_GET_ONE_MAIL_THREAD",
    GET_ONE_NOTE: "PIPEDRIVE_GET_ONE_NOTE",
    GET_ONE_ORGANIZATION_FIELD: "PIPEDRIVE_GET_ONE_ORGANIZATION_FIELD",
    GET_ONE_ORGANIZATION_RELATIONSHIP: "PIPEDRIVE_GET_ONE_ORGANIZATION_RELATIONSHIP",
    GET_ONE_PERMISSION_SET: "PIPEDRIVE_GET_ONE_PERMISSION_SET",
    GET_ONE_PERSON_FIELD: "PIPEDRIVE_GET_ONE_PERSON_FIELD",
    GET_ONE_PIPELINE: "PIPEDRIVE_GET_ONE_PIPELINE",
    GET_ONE_PRODUCT: "PIPEDRIVE_GET_ONE_PRODUCT",
    GET_ONE_PRODUCT_FIELD: "PIPEDRIVE_GET_ONE_PRODUCT_FIELD",
    GET_ONE_ROLE: "PIPEDRIVE_GET_ONE_ROLE",
    GET_ONE_STAGE: "PIPEDRIVE_GET_ONE_STAGE",
    GET_ONE_USER: "PIPEDRIVE_GET_ONE_USER",
    GET_PROJECT_PHASES: "PIPEDRIVE_GET_PROJECT_PHASES",
    GET_RECENTS: "PIPEDRIVE_GET_RECENTS",
    GET_RESULT_OF_A_GOAL: "PIPEDRIVE_GET_RESULT_OF_A_GOAL",
    LINK_A_REMOTE_FILE_TO_AN_ITEM: "PIPEDRIVE_LINK_A_REMOTE_FILE_TO_AN_ITEM",
    LINK_USER_VIDEO_INTEGRATION: "PIPEDRIVE_LINK_USER_VIDEO_INTEGRATION",
    LIST_ACTIVITIES_ASSOCIATED_WITH_AN_ORGANIZATION: "PIPEDRIVE_LIST_ACTIVITIES_ASSOCIATED_WITH_AN_ORGANIZATION",
    LIST_ACTIVITIES_ASSOCIATED_WITH_A_DEAL: "PIPEDRIVE_LIST_ACTIVITIES_ASSOCIATED_WITH_A_DEAL",
    LIST_ACTIVITIES_ASSOCIATED_WITH_A_PERSON: "PIPEDRIVE_LIST_ACTIVITIES_ASSOCIATED_WITH_A_PERSON",
    LIST_ALL_PERSONS_ASSOCIATED_WITH_A_DEAL: "PIPEDRIVE_LIST_ALL_PERSONS_ASSOCIATED_WITH_A_DEAL",
    LIST_DEALS_ASSOCIATED_WITH_AN_ORGANIZATION: "PIPEDRIVE_LIST_DEALS_ASSOCIATED_WITH_AN_ORGANIZATION",
    LIST_DEALS_ASSOCIATED_WITH_A_PERSON: "PIPEDRIVE_LIST_DEALS_ASSOCIATED_WITH_A_PERSON",
    LIST_FILES_ATTACHED_TO_AN_ORGANIZATION: "PIPEDRIVE_LIST_FILES_ATTACHED_TO_AN_ORGANIZATION",
    LIST_FILES_ATTACHED_TO_A_DEAL: "PIPEDRIVE_LIST_FILES_ATTACHED_TO_A_DEAL",
    LIST_FILES_ATTACHED_TO_A_PERSON: "PIPEDRIVE_LIST_FILES_ATTACHED_TO_A_PERSON",
    LIST_FILES_ATTACHED_TO_A_PRODUCT: "PIPEDRIVE_LIST_FILES_ATTACHED_TO_A_PRODUCT",
    LIST_FOLLOWERS_OF_AN_ORGANIZATION: "PIPEDRIVE_LIST_FOLLOWERS_OF_AN_ORGANIZATION",
    LIST_FOLLOWERS_OF_A_DEAL: "PIPEDRIVE_LIST_FOLLOWERS_OF_A_DEAL",
    LIST_FOLLOWERS_OF_A_PERSON: "PIPEDRIVE_LIST_FOLLOWERS_OF_A_PERSON",
    LIST_FOLLOWERS_OF_A_PRODUCT: "PIPEDRIVE_LIST_FOLLOWERS_OF_A_PRODUCT",
    LIST_FOLLOWERS_OF_A_USER: "PIPEDRIVE_LIST_FOLLOWERS_OF_A_USER",
    LIST_MAIL_MESSAGES_ASSOCIATED_WITH_AN_ORGANIZATION: "PIPEDRIVE_LIST_MAIL_MESSAGES_ASSOCIATED_WITH_AN_ORGANIZATION",
    LIST_MAIL_MESSAGES_ASSOCIATED_WITH_A_DEAL: "PIPEDRIVE_LIST_MAIL_MESSAGES_ASSOCIATED_WITH_A_DEAL",
    LIST_MAIL_MESSAGES_ASSOCIATED_WITH_A_PERSON: "PIPEDRIVE_LIST_MAIL_MESSAGES_ASSOCIATED_WITH_A_PERSON",
    LIST_PARTICIPANTS_OF_A_DEAL: "PIPEDRIVE_LIST_PARTICIPANTS_OF_A_DEAL",
    LIST_PERMISSION_SET_ASSIGNMENTS: "PIPEDRIVE_LIST_PERMISSION_SET_ASSIGNMENTS",
    LIST_PERMITTED_USERS: "PIPEDRIVE_LIST_PERMITTED_USERS",
    LIST_PERSONS_OF_AN_ORGANIZATION: "PIPEDRIVE_LIST_PERSONS_OF_AN_ORGANIZATION",
    LIST_PERSON_ACCESS_USERS: "PIPEDRIVE_LIST_PERSON_ACCESS_USERS",
    LIST_PIPELINE_VISIBILITY_FOR_A_ROLE: "PIPEDRIVE_LIST_PIPELINE_VISIBILITY_FOR_A_ROLE",
    LIST_PRODUCTS_ASSOCIATED_WITH_A_PERSON: "PIPEDRIVE_LIST_PRODUCTS_ASSOCIATED_WITH_A_PERSON",
    LIST_PRODUCTS_ATTACHED_TO_A_DEAL: "PIPEDRIVE_LIST_PRODUCTS_ATTACHED_TO_A_DEAL",
    LIST_PRODUCT_PERMITTED_USERS: "PIPEDRIVE_LIST_PRODUCT_PERMITTED_USERS",
    LIST_ROLE_ASSIGNMENTS: "PIPEDRIVE_LIST_ROLE_ASSIGNMENTS",
    LIST_ROLE_SETTINGS: "PIPEDRIVE_LIST_ROLE_SETTINGS",
    LIST_SETTINGS_OF_AN_AUTHORIZED_USER: "PIPEDRIVE_LIST_SETTINGS_OF_AN_AUTHORIZED_USER",
    LIST_UPDATES_ABOUT_AN_ORGANIZATION: "PIPEDRIVE_LIST_UPDATES_ABOUT_AN_ORGANIZATION",
    LIST_UPDATES_ABOUT_A_DEAL: "PIPEDRIVE_LIST_UPDATES_ABOUT_A_DEAL",
    LIST_UPDATES_ABOUT_A_PERSON: "PIPEDRIVE_LIST_UPDATES_ABOUT_A_PERSON",
    LIST_UPDATES_ABOUT_PARTICIPANTS_OF_A_DEAL: "PIPEDRIVE_LIST_UPDATES_ABOUT_PARTICIPANTS_OF_A_DEAL",
    LIST_USER_PERMISSIONS: "PIPEDRIVE_LIST_USER_PERMISSIONS",
    LIST_USER_ROLE_ASSIGNMENTS: "PIPEDRIVE_LIST_USER_ROLE_ASSIGNMENTS",
    LIST_USER_ROLE_SETTINGS: "PIPEDRIVE_LIST_USER_ROLE_SETTINGS",
    MERGE_TWO_DEALS: "PIPEDRIVE_MERGE_TWO_DEALS",
    MERGE_TWO_ORGANIZATIONS: "PIPEDRIVE_MERGE_TWO_ORGANIZATIONS",
    MERGE_TWO_PERSONS: "PIPEDRIVE_MERGE_TWO_PERSONS",
    ORGANIZATION_ACCESSIBLE_USER_LIST: "PIPEDRIVE_ORGANIZATION_ACCESSIBLE_USER_LIST",
    PERFORM_A_SEARCH_FROM_MULTIPLE_ITEM_TYPES: "PIPEDRIVE_PERFORM_A_SEARCH_FROM_MULTIPLE_ITEM_TYPES",
    RECEIVES_AN_INCOMING_MESSAGE: "PIPEDRIVE_RECEIVES_AN_INCOMING_MESSAGE",
    REFRESHING_THE_TOKENS: "PIPEDRIVE_REFRESHING_THE_TOKENS",
    REQUESTING_AUTHORIZATION: "PIPEDRIVE_REQUESTING_AUTHORIZATION",
    RETURNS_PROJECT_ACTIVITIES: "PIPEDRIVE_RETURNS_PROJECT_ACTIVITIES",
    RETURNS_PROJECT_GROUPS: "PIPEDRIVE_RETURNS_PROJECT_GROUPS",
    RETURNS_PROJECT_PLAN: "PIPEDRIVE_RETURNS_PROJECT_PLAN",
    RETURNS_PROJECT_TASKS: "PIPEDRIVE_RETURNS_PROJECT_TASKS",
    SEARCH_DEALS: "PIPEDRIVE_SEARCH_DEALS",
    SEARCH_ITEM_BY_FIELD: "PIPEDRIVE_SEARCH_ITEM_BY_FIELD",
    SEARCH_LEADS: "PIPEDRIVE_SEARCH_LEADS",
    SEARCH_ORGANIZATIONS: "PIPEDRIVE_SEARCH_ORGANIZATIONS",
    SEARCH_PERSONS: "PIPEDRIVE_SEARCH_PERSONS",
    SEARCH_PRODUCTS: "PIPEDRIVE_SEARCH_PRODUCTS",
    UNLINK_USER_FROM_VIDEO_CALL_INTEGRATION: "PIPEDRIVE_UNLINK_USER_FROM_VIDEO_CALL_INTEGRATION",
    UPDATE_ACTIVITY_IN_PROJECT_PLAN: "PIPEDRIVE_UPDATE_ACTIVITY_IN_PROJECT_PLAN",
    UPDATE_AN_ACTIVITY: "PIPEDRIVE_UPDATE_AN_ACTIVITY",
    UPDATE_AN_ACTIVITY_TYPE: "PIPEDRIVE_UPDATE_AN_ACTIVITY_TYPE",
    UPDATE_AN_INSTALLMENT_SUBSCRIPTION: "PIPEDRIVE_UPDATE_AN_INSTALLMENT_SUBSCRIPTION",
    UPDATE_AN_ORGANIZATION: "PIPEDRIVE_UPDATE_AN_ORGANIZATION",
    UPDATE_AN_ORGANIZATION_FIELD: "PIPEDRIVE_UPDATE_AN_ORGANIZATION_FIELD",
    UPDATE_AN_ORGANIZATION_RELATIONSHIP: "PIPEDRIVE_UPDATE_AN_ORGANIZATION_RELATIONSHIP",
    UPDATE_A_COMMENT_RELATED_TO_A_NOTE: "PIPEDRIVE_UPDATE_A_COMMENT_RELATED_TO_A_NOTE",
    UPDATE_A_DEAL: "PIPEDRIVE_UPDATE_A_DEAL",
    UPDATE_A_DEAL_FIELD: "PIPEDRIVE_UPDATE_A_DEAL_FIELD",
    UPDATE_A_LEAD: "PIPEDRIVE_UPDATE_A_LEAD",
    UPDATE_A_LEAD_LABEL: "PIPEDRIVE_UPDATE_A_LEAD_LABEL",
    UPDATE_A_NOTE: "PIPEDRIVE_UPDATE_A_NOTE",
    UPDATE_A_PERSON: "PIPEDRIVE_UPDATE_A_PERSON",
    UPDATE_A_PERSON_FIELD: "PIPEDRIVE_UPDATE_A_PERSON_FIELD",
    UPDATE_A_PIPELINE: "PIPEDRIVE_UPDATE_A_PIPELINE",
    UPDATE_A_PRODUCT: "PIPEDRIVE_UPDATE_A_PRODUCT",
    UPDATE_A_PRODUCT_FIELD: "PIPEDRIVE_UPDATE_A_PRODUCT_FIELD",
    UPDATE_A_PROJECT: "PIPEDRIVE_UPDATE_A_PROJECT",
    UPDATE_A_RECURRING_SUBSCRIPTION: "PIPEDRIVE_UPDATE_A_RECURRING_SUBSCRIPTION",
    UPDATE_A_TASK: "PIPEDRIVE_UPDATE_A_TASK",
    UPDATE_A_TEAM: "PIPEDRIVE_UPDATE_A_TEAM",
    UPDATE_EXISTING_GOAL: "PIPEDRIVE_UPDATE_EXISTING_GOAL",
    UPDATE_FILE_DETAILS: "PIPEDRIVE_UPDATE_FILE_DETAILS",
    UPDATE_FILTER: "PIPEDRIVE_UPDATE_FILTER",
    UPDATE_MAIL_THREAD_DETAILS: "PIPEDRIVE_UPDATE_MAIL_THREAD_DETAILS",
    UPDATE_PIPELINE_VISIBILITY_FOR_A_ROLE: "PIPEDRIVE_UPDATE_PIPELINE_VISIBILITY_FOR_A_ROLE",
    UPDATE_ROLE_DETAILS: "PIPEDRIVE_UPDATE_ROLE_DETAILS",
    UPDATE_STAGE_DETAILS: "PIPEDRIVE_UPDATE_STAGE_DETAILS",
    UPDATE_TASK_IN_PROJECT_PLAN: "PIPEDRIVE_UPDATE_TASK_IN_PROJECT_PLAN",
    UPDATE_THE_PRODUCT_ATTACHED_TO_A_DEAL: "PIPEDRIVE_UPDATE_THE_PRODUCT_ATTACHED_TO_A_DEAL",
    UPDATE_USER_DETAILS: "PIPEDRIVE_UPDATE_USER_DETAILS",
  },
  triggerTypes: {
    PIPEDRIVE_NEW_DEAL_TRIGGER: "PIPEDRIVE_PIPEDRIVE_NEW_DEAL_TRIGGER",
    PIPEDRIVE_NEW_NOTE_TRIGGER: "PIPEDRIVE_PIPEDRIVE_NEW_NOTE_TRIGGER",
    PIPEDRIVE_NEW_ORGANIZATION_TRIGGER: "PIPEDRIVE_PIPEDRIVE_NEW_ORGANIZATION_TRIGGER",
  },
}
