// @composio/core/generated/posthog.ts
/**
 * Map of Composio's POSTHOG toolkit.
 */
export const POSTHOG = {
  slug: "posthog",
  tools: {
    ADD_DASHBOARD_COLLABORATORS_WITH_ACCESS_LEVEL: "POSTHOG_ADD_DASHBOARD_COLLABORATORS_WITH_ACCESS_LEVEL",
    ADD_MEMBER_TO_ORGANIZATION_ROLE: "POSTHOG_ADD_MEMBER_TO_ORGANIZATION_ROLE",
    ADD_PROJECT_SPECIFIC_MEMBER_ROLES: "POSTHOG_ADD_PROJECT_SPECIFIC_MEMBER_ROLES",
    ASSIGN_FEATURE_FLAG_ROLE_BASED_ACCESS: "POSTHOG_ASSIGN_FEATURE_FLAG_ROLE_BASED_ACCESS",
    BOOKMARK_PROJECT_ACTIVITY_NOTIFICATION: "POSTHOG_BOOKMARK_PROJECT_ACTIVITY_NOTIFICATION",
    BULK_CREATE_ORGANIZATION_INVITES: "POSTHOG_BULK_CREATE_ORGANIZATION_INVITES",
    CANCEL_INSIGHTS_FOR_A_PROJECT: "POSTHOG_CANCEL_INSIGHTS_FOR_A_PROJECT",
    CHECK_PIPELINE_DESTINATION_UPDATES: "POSTHOG_CHECK_PIPELINE_DESTINATION_UPDATES",
    CHECK_PIPELINE_IMPORT_APP_UPDATES: "POSTHOG_CHECK_PIPELINE_IMPORT_APP_UPDATES",
    CHECK_PIPELINE_PLUGIN_UPDATE_STATUS: "POSTHOG_CHECK_PIPELINE_PLUGIN_UPDATE_STATUS",
    CHECK_PROJECT_S_DEMO_DATA_GENERATION_STATUS: "POSTHOG_CHECK_PROJECT_S_DEMO_DATA_GENERATION_STATUS",
    COHORT_DUPLICATION_TO_STATIC: "POSTHOG_COHORT_DUPLICATION_TO_STATIC",
    CREATE_A_NOTEBOOK_IN_A_PROJECT: "POSTHOG_CREATE_A_NOTEBOOK_IN_A_PROJECT",
    CREATE_DASHBOARD_FROM_TEMPLATE_IN_PROJECT: "POSTHOG_CREATE_DASHBOARD_FROM_TEMPLATE_IN_PROJECT",
    CREATE_DOMAIN_WITH_SAML_FOR_ORGANIZATION: "POSTHOG_CREATE_DOMAIN_WITH_SAML_FOR_ORGANIZATION",
    CREATE_EXPERIMENT_EXPOSURE_COHORT: "POSTHOG_CREATE_EXPERIMENT_EXPOSURE_COHORT",
    CREATE_FEATURE_FLAGS_FOR_PROJECT: "POSTHOG_CREATE_FEATURE_FLAGS_FOR_PROJECT",
    CREATE_FEATURE_FLAG_DASHBOARD: "POSTHOG_CREATE_FEATURE_FLAG_DASHBOARD",
    CREATE_FRONTEND_APP_PIPELINE_CONFIG: "POSTHOG_CREATE_FRONTEND_APP_PIPELINE_CONFIG",
    CREATE_FUNNEL_ANALYSIS_FOR_PROJECT_PERSONS: "POSTHOG_CREATE_FUNNEL_ANALYSIS_FOR_PROJECT_PERSONS",
    CREATE_NEW_ORGANIZATION_WITH_CONFIGURABLE_ACCESS: "POSTHOG_CREATE_NEW_ORGANIZATION_WITH_CONFIGURABLE_ACCESS",
    CREATE_NEW_PROJECT_EXPERIMENT: "POSTHOG_CREATE_NEW_PROJECT_EXPERIMENT",
    CREATE_ORGANIZATION_INVITE_WITH_ROLE_AND_EMAIL: "POSTHOG_CREATE_ORGANIZATION_INVITE_WITH_ROLE_AND_EMAIL",
    CREATE_ORGANIZATION_ROLE_WITH_FEATURE_FLAG_LEVELS: "POSTHOG_CREATE_ORGANIZATION_ROLE_WITH_FEATURE_FLAG_LEVELS",
    CREATE_ORG_PROXY_RECORD_WITH_DOMAIN_AND_STATUS: "POSTHOG_CREATE_ORG_PROXY_RECORD_WITH_DOMAIN_AND_STATUS",
    CREATE_PERSON_FUNNEL_CORRELATION: "POSTHOG_CREATE_PERSON_FUNNEL_CORRELATION",
    CREATE_PERSON_PATH_WITH_FORMAT_OPTION: "POSTHOG_CREATE_PERSON_PATH_WITH_FORMAT_OPTION",
    CREATE_PIPELINE_DESTINATION_CONFIG: "POSTHOG_CREATE_PIPELINE_DESTINATION_CONFIG",
    CREATE_PIPELINE_DESTINATION_CONFIG_JOB: "POSTHOG_CREATE_PIPELINE_DESTINATION_CONFIG_JOB",
    CREATE_PIPELINE_IMPORT_APPS_CONFIG_JOB: "POSTHOG_CREATE_PIPELINE_IMPORT_APPS_CONFIG_JOB",
    CREATE_PLUGIN_JOB_CONFIGURATION: "POSTHOG_CREATE_PLUGIN_JOB_CONFIGURATION",
    CREATE_PROJECT_ACTION_WITH_FORMAT: "POSTHOG_CREATE_PROJECT_ACTION_WITH_FORMAT",
    CREATE_PROJECT_COHORTS: "POSTHOG_CREATE_PROJECT_COHORTS",
    CREATE_PROJECT_DASHBOARD: "POSTHOG_CREATE_PROJECT_DASHBOARD",
    CREATE_PROJECT_INSIGHTS_WITH_CUSTOMIZABLE_PARAMETERS: "POSTHOG_CREATE_PROJECT_INSIGHTS_WITH_CUSTOMIZABLE_PARAMETERS",
    CREATE_PROJECT_INSIGHTS_WITH_FORMAT_OPTION: "POSTHOG_CREATE_PROJECT_INSIGHTS_WITH_FORMAT_OPTION",
    CREATE_PROJECT_PIPELINE_APP_CONFIGS: "POSTHOG_CREATE_PROJECT_PIPELINE_APP_CONFIGS",
    CREATE_PROJECT_PIPELINE_FRONTEND_APP_JOB: "POSTHOG_CREATE_PROJECT_PIPELINE_FRONTEND_APP_JOB",
    CREATE_PROJECT_PIPELINE_TRANSFORMATION_CONFIG: "POSTHOG_CREATE_PROJECT_PIPELINE_TRANSFORMATION_CONFIG",
    CREATE_PROJECT_PLUGIN_CONFIGURATION: "POSTHOG_CREATE_PROJECT_PLUGIN_CONFIGURATION",
    CREATE_PROJECT_SUBSCRIPTION_WITH_DELIVERY_OPTIONS: "POSTHOG_CREATE_PROJECT_SUBSCRIPTION_WITH_DELIVERY_OPTIONS",
    CREATE_PROJECT_SURVEYS_WITH_QUESTIONS_AND_TARGETS: "POSTHOG_CREATE_PROJECT_SURVEYS_WITH_QUESTIONS_AND_TARGETS",
    CREATE_PROJECT_TIMING_INSIGHTS: "POSTHOG_CREATE_PROJECT_TIMING_INSIGHTS",
    CREATE_QUERY_IN_PROJECT_BY_ID: "POSTHOG_CREATE_QUERY_IN_PROJECT_BY_ID",
    CREATE_RETRIEVE_RETENTION_INSIGHTS_FOR_PROJECT: "POSTHOG_CREATE_RETRIEVE_RETENTION_INSIGHTS_FOR_PROJECT",
    CREATE_SESSION_RECORDING_IN_PLAYLIST: "POSTHOG_CREATE_SESSION_RECORDING_IN_PLAYLIST",
    CREATE_SESSION_RECORDING_PLAYLISTS: "POSTHOG_CREATE_SESSION_RECORDING_PLAYLISTS",
    CREATE_SESSION_RECORDING_SUMMARY: "POSTHOG_CREATE_SESSION_RECORDING_SUMMARY",
    CREATE_STATIC_COHORT_FOR_FEATURE_FLAG: "POSTHOG_CREATE_STATIC_COHORT_FOR_FEATURE_FLAG",
    CRUD_OPERATIONS_FOR_PROJECT_ANNOTATIONS: "POSTHOG_CRUD_OPERATIONS_FOR_PROJECT_ANNOTATIONS",
    DELETE_ORGANIZATION_BY_UUID: "POSTHOG_DELETE_ORGANIZATION_BY_UUID",
    DELETE_ORGANIZATION_PROJECT: "POSTHOG_DELETE_ORGANIZATION_PROJECT",
    DELETE_PERSON_AND_OPTIONALLY_ASSOCIATED_EVENTS: "POSTHOG_DELETE_PERSON_AND_OPTIONALLY_ASSOCIATED_EVENTS",
    DELETE_SESSION_RECORDING_BY_UUID: "POSTHOG_DELETE_SESSION_RECORDING_BY_UUID",
    DELETE_SURVEY_BY_ID: "POSTHOG_DELETE_SURVEY_BY_ID",
    ENABLE_PROJECT_EARLY_ACCESS_FEATURE: "POSTHOG_ENABLE_PROJECT_EARLY_ACCESS_FEATURE",
    FETCH_BATCH_EXPORT_DETAILS_VIA_UUID: "POSTHOG_FETCH_BATCH_EXPORT_DETAILS_VIA_UUID",
    FETCH_EXPORT_UNSUBSCRIBE_SETTINGS: "POSTHOG_FETCH_EXPORT_UNSUBSCRIBE_SETTINGS",
    FETCH_FRONTEND_APP_CONFIG_STATUS: "POSTHOG_FETCH_FRONTEND_APP_CONFIG_STATUS",
    FETCH_ORGANIZATION_DETAILS_BY_UUID: "POSTHOG_FETCH_ORGANIZATION_DETAILS_BY_UUID",
    FETCH_ORGANIZATION_PIPELINE_APP_REPOSITORIES: "POSTHOG_FETCH_ORGANIZATION_PIPELINE_APP_REPOSITORIES",
    FETCH_PIPELINE_DESTINATION_DETAILS: "POSTHOG_FETCH_PIPELINE_DESTINATION_DETAILS",
    FETCH_PIPELINE_TRANSFORMATION_DETAILS: "POSTHOG_FETCH_PIPELINE_TRANSFORMATION_DETAILS",
    FETCH_PROJECT_EXPERIMENT_SECONDARY_DATA: "POSTHOG_FETCH_PROJECT_EXPERIMENT_SECONDARY_DATA",
    FETCH_PROJECT_SURVEY_ACTIVITY_LOG: "POSTHOG_FETCH_PROJECT_SURVEY_ACTIVITY_LOG",
    FETCH_SESSION_RECORDING_PLAYLIST_DETAILS: "POSTHOG_FETCH_SESSION_RECORDING_PLAYLIST_DETAILS",
    FLAG_ANNOTATION_AS_DELETED: "POSTHOG_FLAG_ANNOTATION_AS_DELETED",
    GENERATE_PROJECT_FUNNEL_CORRELATION_INSIGHT: "POSTHOG_GENERATE_PROJECT_FUNNEL_CORRELATION_INSIGHT",
    INITIATE_DOMAIN_VERIFICATION_WITH_SAML_CONFIG: "POSTHOG_INITIATE_DOMAIN_VERIFICATION_WITH_SAML_CONFIG",
    INITIATE_PIPELINE_TRANSFORMATION_JOB: "POSTHOG_INITIATE_PIPELINE_TRANSFORMATION_JOB",
    LIST_AND_FILTER_PROJECT_EVENTS: "POSTHOG_LIST_AND_FILTER_PROJECT_EVENTS",
    LIST_AND_MANAGE_PROJECT_ANNOTATIONS: "POSTHOG_LIST_AND_MANAGE_PROJECT_ANNOTATIONS",
    LIST_AND_MANAGE_PROJECT_FEATURE_FLAGS: "POSTHOG_LIST_AND_MANAGE_PROJECT_FEATURE_FLAGS",
    LIST_BATCH_EXPORT_RUNS: "POSTHOG_LIST_BATCH_EXPORT_RUNS",
    LIST_DASHBOARD_SHARING_CONFIGS: "POSTHOG_LIST_DASHBOARD_SHARING_CONFIGS",
    LIST_FEATURE_FLAG_ROLE_ACCESS_DETAILS: "POSTHOG_LIST_FEATURE_FLAG_ROLE_ACCESS_DETAILS",
    LIST_NOTEBOOKS_WITH_FILTERS_AND_PAGINATION: "POSTHOG_LIST_NOTEBOOKS_WITH_FILTERS_AND_PAGINATION",
    LIST_OF_PROJECT_S_SESSION_RECORDING_PLAYLISTS: "POSTHOG_LIST_OF_PROJECT_S_SESSION_RECORDING_PLAYLISTS",
    LIST_ORGANIZATION_DOMAINS_WITH_PAGINATION: "POSTHOG_LIST_ORGANIZATION_DOMAINS_WITH_PAGINATION",
    LIST_ORGANIZATION_INVITES_WITH_PAGINATION: "POSTHOG_LIST_ORGANIZATION_INVITES_WITH_PAGINATION",
    LIST_ORGANIZATION_MEMBERS_WITH_PAGINATION: "POSTHOG_LIST_ORGANIZATION_MEMBERS_WITH_PAGINATION",
    LIST_ORGANIZATION_PLUGINS_WITH_PAGINATION: "POSTHOG_LIST_ORGANIZATION_PLUGINS_WITH_PAGINATION",
    LIST_ORGANIZATION_ROLES_WITH_PAGINATION: "POSTHOG_LIST_ORGANIZATION_ROLES_WITH_PAGINATION",
    LIST_ORGANIZATION_S_BATCH_EXPORTS_WITH_PAGINATION: "POSTHOG_LIST_ORGANIZATION_S_BATCH_EXPORTS_WITH_PAGINATION",
    LIST_ORGANIZATION_S_PAGINATED_PROXY_RECORDS: "POSTHOG_LIST_ORGANIZATION_S_PAGINATED_PROXY_RECORDS",
    LIST_ORGANIZATION_S_PIPELINE_DESTINATIONS: "POSTHOG_LIST_ORGANIZATION_S_PIPELINE_DESTINATIONS",
    LIST_ORGANIZATION_S_PIPELINE_FRONTEND_APPS: "POSTHOG_LIST_ORGANIZATION_S_PIPELINE_FRONTEND_APPS",
    LIST_ORGANIZATION_S_PIPELINE_IMPORT_APPS: "POSTHOG_LIST_ORGANIZATION_S_PIPELINE_IMPORT_APPS",
    LIST_ORGANIZATION_S_PIPELINE_TRANSFORMATIONS: "POSTHOG_LIST_ORGANIZATION_S_PIPELINE_TRANSFORMATIONS",
    LIST_OR_DELETE_PERSONS_WITH_OPTIONAL_FILTERS: "POSTHOG_LIST_OR_DELETE_PERSONS_WITH_OPTIONAL_FILTERS",
    LIST_PAGINATED_SURVEYS_FOR_A_PROJECT: "POSTHOG_LIST_PAGINATED_SURVEYS_FOR_A_PROJECT",
    LIST_PIPELINE_DESTINATION_CONFIGS_WITH_PAGINATION: "POSTHOG_LIST_PIPELINE_DESTINATION_CONFIGS_WITH_PAGINATION",
    LIST_PIPELINE_FRONTEND_APP_CONFIGS: "POSTHOG_LIST_PIPELINE_FRONTEND_APP_CONFIGS",
    LIST_PROJECTS_IN_ORGANIZATION_WITH_PAGINATION: "POSTHOG_LIST_PROJECTS_IN_ORGANIZATION_WITH_PAGINATION",
    LIST_PROJECT_ACTIONS_WITH_PAGINATION_AND_FILTERS: "POSTHOG_LIST_PROJECT_ACTIONS_WITH_PAGINATION_AND_FILTERS",
    LIST_PROJECT_BATCH_EXPORTS_WITH_PAGINATION: "POSTHOG_LIST_PROJECT_BATCH_EXPORTS_WITH_PAGINATION",
    LIST_PROJECT_CONFIGURATION_PLUGIN_LOGS: "POSTHOG_LIST_PROJECT_CONFIGURATION_PLUGIN_LOGS",
    LIST_PROJECT_DASHBOARDS_WITH_PAGINATION: "POSTHOG_LIST_PROJECT_DASHBOARDS_WITH_PAGINATION",
    LIST_PROJECT_DASHBOARD_COLLABORATORS: "POSTHOG_LIST_PROJECT_DASHBOARD_COLLABORATORS",
    LIST_PROJECT_DASHBOARD_TEMPLATES: "POSTHOG_LIST_PROJECT_DASHBOARD_TEMPLATES",
    LIST_PROJECT_EARLY_ACCESS_FEATURES: "POSTHOG_LIST_PROJECT_EARLY_ACCESS_FEATURES",
    LIST_PROJECT_EXPERIMENTS_WITH_PAGINATION: "POSTHOG_LIST_PROJECT_EXPERIMENTS_WITH_PAGINATION",
    LIST_PROJECT_EXPORT_ITEMS_WITH_PAGINATION: "POSTHOG_LIST_PROJECT_EXPORT_ITEMS_WITH_PAGINATION",
    LIST_PROJECT_GROUP_TYPES_BY_ID: "POSTHOG_LIST_PROJECT_GROUP_TYPES_BY_ID",
    LIST_PROJECT_MEMBERSHIP_DETAILS: "POSTHOG_LIST_PROJECT_MEMBERSHIP_DETAILS",
    LIST_PROJECT_PIPELINE_CONFIGURATIONS: "POSTHOG_LIST_PROJECT_PIPELINE_CONFIGURATIONS",
    LIST_PROJECT_PIPELINE_IMPORT_CONFIGS: "POSTHOG_LIST_PROJECT_PIPELINE_IMPORT_CONFIGS",
    LIST_PROJECT_PLUGIN_CONFIGURATIONS: "POSTHOG_LIST_PROJECT_PLUGIN_CONFIGURATIONS",
    LIST_PROJECT_SESSION_RECORDINGS: "POSTHOG_LIST_PROJECT_SESSION_RECORDINGS",
    LIST_PROJECT_SUBSCRIPTIONS_WITH_PAGINATION: "POSTHOG_LIST_PROJECT_SUBSCRIPTIONS_WITH_PAGINATION",
    LIST_ROLE_MEMBERSHIPS_PAGINATED: "POSTHOG_LIST_ROLE_MEMBERSHIPS_PAGINATED",
    LIST_SESSION_RECORDING_SHARING_CONFIGS: "POSTHOG_LIST_SESSION_RECORDING_SHARING_CONFIGS",
    LIST_SPECIFIC_TYPE_GROUPS_WITH_PAGINATION: "POSTHOG_LIST_SPECIFIC_TYPE_GROUPS_WITH_PAGINATION",
    LIST_UNUSED_ORGANIZATION_PIPELINE_TRANSFORMATIONS: "POSTHOG_LIST_UNUSED_ORGANIZATION_PIPELINE_TRANSFORMATIONS",
    LIST_USERS_WITH_STAFF_FILTER_AND_PAGINATION: "POSTHOG_LIST_USERS_WITH_STAFF_FILTER_AND_PAGINATION",
    MANAGE_FEATURE_FLAG_ENRICHMENT_FOR_USAGE_DASHBOARD: "POSTHOG_MANAGE_FEATURE_FLAG_ENRICHMENT_FOR_USAGE_DASHBOARD",
    MANAGE_PROJECT_FEATURE_FLAGS: "POSTHOG_MANAGE_PROJECT_FEATURE_FLAGS",
    MANAGE_PROJECT_FEATURE_FLAGS_FOR_LOCAL_EVALUATION: "POSTHOG_MANAGE_PROJECT_FEATURE_FLAGS_FOR_LOCAL_EVALUATION",
    MANAGE_USER_FEATURE_FLAGS: "POSTHOG_MANAGE_USER_FEATURE_FLAGS",
    MARK_DASHBOARD_TEMPLATE_AS_DELETED: "POSTHOG_MARK_DASHBOARD_TEMPLATE_AS_DELETED",
    MARK_INSIGHT_AS_VIEWED_WITH_CONFIGURABLE_FORMAT: "POSTHOG_MARK_INSIGHT_AS_VIEWED_WITH_CONFIGURABLE_FORMAT",
    MARK_PROJECT_ACTION_AS_DELETED: "POSTHOG_MARK_PROJECT_ACTION_AS_DELETED",
    MARK_SESSION_RECORDING_PLAYLIST_DELETED: "POSTHOG_MARK_SESSION_RECORDING_PLAYLIST_DELETED",
    MARK_SUBSCRIPTION_AS_DELETED_BY_ID: "POSTHOG_MARK_SUBSCRIPTION_AS_DELETED_BY_ID",
    MODIFY_ORG_INFO_VIA_UUID: "POSTHOG_MODIFY_ORG_INFO_VIA_UUID",
    MOVE_DASHBOARD_TILE: "POSTHOG_MOVE_DASHBOARD_TILE",
    PARTIALLY_UPDATE_PROJECT_PLUGIN_CONFIG: "POSTHOG_PARTIALLY_UPDATE_PROJECT_PLUGIN_CONFIG",
    PARTIALLY_UPDATE_PROJECT_PROPERTY_DEFINITION: "POSTHOG_PARTIALLY_UPDATE_PROJECT_PROPERTY_DEFINITION",
    PARTIAL_EXPERIMENT_UPDATE_IN_PROJECT: "POSTHOG_PARTIAL_EXPERIMENT_UPDATE_IN_PROJECT",
    PARTIAL_PIPELINE_FRONTEND_APP_CONFIG_UPDATE: "POSTHOG_PARTIAL_PIPELINE_FRONTEND_APP_CONFIG_UPDATE",
    PARTIAL_UPDATE_OF_A_PROJECT_INSIGHT: "POSTHOG_PARTIAL_UPDATE_OF_A_PROJECT_INSIGHT",
    PARTIAL_UPDATE_OF_A_PROJECT_S_DASHBOARD: "POSTHOG_PARTIAL_UPDATE_OF_A_PROJECT_S_DASHBOARD",
    PARTIAL_UPDATE_OF_A_PROJECT_S_FEATURE_FLAG: "POSTHOG_PARTIAL_UPDATE_OF_A_PROJECT_S_FEATURE_FLAG",
    PARTIAL_UPDATE_OF_PROJECT_NOTEBOOK: "POSTHOG_PARTIAL_UPDATE_OF_PROJECT_NOTEBOOK",
    PARTIAL_UPDATE_PROJECT_ANNOTATION: "POSTHOG_PARTIAL_UPDATE_PROJECT_ANNOTATION",
    PERSIST_SESSION_RECORDING_DATA_BY_UUID: "POSTHOG_PERSIST_SESSION_RECORDING_DATA_BY_UUID",
    PERSON_PARTIAL_UPDATE_VIA_PROJECT_ID: "POSTHOG_PERSON_PARTIAL_UPDATE_VIA_PROJECT_ID",
    PERSON_PROPERTY_DELETION_ENDPOINT: "POSTHOG_PERSON_PROPERTY_DELETION_ENDPOINT",
    POST_PROJECT_TRENDS_WITH_FILTERS: "POSTHOG_POST_PROJECT_TRENDS_WITH_FILTERS",
    PREVENT_HARD_DELETE_NOTEBOOK_BY_FLAGGING: "POSTHOG_PREVENT_HARD_DELETE_NOTEBOOK_BY_FLAGGING",
    PROJECT_FUNNEL_ADVANCED_ANALYTICS: "POSTHOG_PROJECT_FUNNEL_ADVANCED_ANALYTICS",
    REARRANGE_FRONTEND_APP_PIPELINE_CONFIGS: "POSTHOG_REARRANGE_FRONTEND_APP_PIPELINE_CONFIGS",
    REARRANGE_PIPELINE_TRANSFORMATION_CONFIGS: "POSTHOG_REARRANGE_PIPELINE_TRANSFORMATION_CONFIGS",
    REARRANGE_PROJECT_IMPORT_APP_CONFIGURATIONS: "POSTHOG_REARRANGE_PROJECT_IMPORT_APP_CONFIGURATIONS",
    REARRANGE_PROJECT_PIPELINE_DESTINATION_CONFIGS: "POSTHOG_REARRANGE_PROJECT_PIPELINE_DESTINATION_CONFIGS",
    RECONFIGURE_PLUGIN_ORDER_AND_STATUS_IN_PROJECT: "POSTHOG_RECONFIGURE_PLUGIN_ORDER_AND_STATUS_IN_PROJECT",
    REMOVE_DOMAIN_BY_UUID_FROM_ORGANIZATION: "POSTHOG_REMOVE_DOMAIN_BY_UUID_FROM_ORGANIZATION",
    REMOVE_EARLY_ACCESS_FEATURE_FROM_PROJECT: "POSTHOG_REMOVE_EARLY_ACCESS_FEATURE_FROM_PROJECT",
    REMOVE_EXPLICIT_PROJECT_MEMBER_BY_UUID: "POSTHOG_REMOVE_EXPLICIT_PROJECT_MEMBER_BY_UUID",
    REMOVE_FEATURE_FLAG_ROLE_ACCESS_BY_ID: "POSTHOG_REMOVE_FEATURE_FLAG_ROLE_ACCESS_BY_ID",
    REMOVE_MEMBER_FROM_ORGANIZATION_BY_UUID: "POSTHOG_REMOVE_MEMBER_FROM_ORGANIZATION_BY_UUID",
    REMOVE_ORGANIZATION_BATCH_EXPORT_BY_UUID: "POSTHOG_REMOVE_ORGANIZATION_BATCH_EXPORT_BY_UUID",
    REMOVE_ORGANIZATION_PIPELINE_DESTINATION: "POSTHOG_REMOVE_ORGANIZATION_PIPELINE_DESTINATION",
    REMOVE_ORGANIZATION_PIPELINE_TRANSFORMATION: "POSTHOG_REMOVE_ORGANIZATION_PIPELINE_TRANSFORMATION",
    REMOVE_ORGANIZATION_PLUGIN_BY_ID: "POSTHOG_REMOVE_ORGANIZATION_PLUGIN_BY_ID",
    REMOVE_ORGANIZATION_PROXY_RECORD: "POSTHOG_REMOVE_ORGANIZATION_PROXY_RECORD",
    REMOVE_ORGANIZATION_ROLE_BY_ID: "POSTHOG_REMOVE_ORGANIZATION_ROLE_BY_ID",
    REMOVE_ORG_PIPELINE_IMPORT_APP_BY_ID: "POSTHOG_REMOVE_ORG_PIPELINE_IMPORT_APP_BY_ID",
    REMOVE_PIPELINE_DESTINATION_CONFIG: "POSTHOG_REMOVE_PIPELINE_DESTINATION_CONFIG",
    REMOVE_PIPELINE_FRONTEND_APP_BY_ID: "POSTHOG_REMOVE_PIPELINE_FRONTEND_APP_BY_ID",
    REMOVE_PIPELINE_FRONTEND_APP_CONFIG: "POSTHOG_REMOVE_PIPELINE_FRONTEND_APP_CONFIG",
    REMOVE_PIPELINE_IMPORT_APP_CONFIG: "POSTHOG_REMOVE_PIPELINE_IMPORT_APP_CONFIG",
    REMOVE_PIPELINE_TRANSFORMATION_CONFIG_BY_ID: "POSTHOG_REMOVE_PIPELINE_TRANSFORMATION_CONFIG_BY_ID",
    REMOVE_PROJECT_DASHBOARD_COLLABORATOR: "POSTHOG_REMOVE_PROJECT_DASHBOARD_COLLABORATOR",
    REMOVE_PROJECT_EVENT_DEFINITION: "POSTHOG_REMOVE_PROJECT_EVENT_DEFINITION",
    REMOVE_PROJECT_EXPERIMENT_BY_ID: "POSTHOG_REMOVE_PROJECT_EXPERIMENT_BY_ID",
    REMOVE_PROJECT_PLUGIN_CONFIGURATION: "POSTHOG_REMOVE_PROJECT_PLUGIN_CONFIGURATION",
    REMOVE_PROPERTY_DEFINITION_BY_UUID: "POSTHOG_REMOVE_PROPERTY_DEFINITION_BY_UUID",
    REMOVE_ROLE_MEMBERSHIP_BY_UUID: "POSTHOG_REMOVE_ROLE_MEMBERSHIP_BY_UUID",
    REMOVE_SESSION_RECORDING_FROM_PLAYLIST: "POSTHOG_REMOVE_SESSION_RECORDING_FROM_PLAYLIST",
    REMOVE_SPECIFIC_PROJECT_BATCH_EXPORT: "POSTHOG_REMOVE_SPECIFIC_PROJECT_BATCH_EXPORT",
    REMOVE_SPECIFIC_PROJECT_QUERY: "POSTHOG_REMOVE_SPECIFIC_PROJECT_QUERY",
    RETRIEVE_AND_DELETE_PROJECT_PERSON_TRENDS: "POSTHOG_RETRIEVE_AND_DELETE_PROJECT_PERSON_TRENDS",
    RETRIEVE_APP_METRICS_BY_IDS: "POSTHOG_RETRIEVE_APP_METRICS_BY_IDS",
    RETRIEVE_APP_METRICS_ERROR_DETAILS: "POSTHOG_RETRIEVE_APP_METRICS_ERROR_DETAILS",
    RETRIEVE_ASYNC_QUERY_STATUS_BY_ID: "POSTHOG_RETRIEVE_ASYNC_QUERY_STATUS_BY_ID",
    RETRIEVE_BATCH_EXPORT_DETAILS: "POSTHOG_RETRIEVE_BATCH_EXPORT_DETAILS",
    RETRIEVE_BATCH_EXPORT_LOGS_BY_UUID: "POSTHOG_RETRIEVE_BATCH_EXPORT_LOGS_BY_UUID",
    RETRIEVE_BATCH_EXPORT_RUN_LOGS: "POSTHOG_RETRIEVE_BATCH_EXPORT_RUN_LOGS",
    RETRIEVE_COHORT_ACTIVITY_DETAILS: "POSTHOG_RETRIEVE_COHORT_ACTIVITY_DETAILS",
    RETRIEVE_COHORT_PERSON_DETAILS: "POSTHOG_RETRIEVE_COHORT_PERSON_DETAILS",
    RETRIEVE_DASHBOARD_TEMPLATE_DETAILS_BY_UUID: "POSTHOG_RETRIEVE_DASHBOARD_TEMPLATE_DETAILS_BY_UUID",
    RETRIEVE_DASHBOARD_TEMPLATE_SCHEMA: "POSTHOG_RETRIEVE_DASHBOARD_TEMPLATE_SCHEMA",
    RETRIEVE_EVENT_DEFINITIONS_BY_PROJECT_ID: "POSTHOG_RETRIEVE_EVENT_DEFINITIONS_BY_PROJECT_ID",
    RETRIEVE_EVENT_DEFINITION_BY_UUID: "POSTHOG_RETRIEVE_EVENT_DEFINITION_BY_UUID",
    RETRIEVE_EXPERIMENTS_REQUIRING_FLAG_IMPLEMENTATION: "POSTHOG_RETRIEVE_EXPERIMENTS_REQUIRING_FLAG_IMPLEMENTATION",
    RETRIEVE_EXPERIMENT_DETAILS_BY_IDS: "POSTHOG_RETRIEVE_EXPERIMENT_DETAILS_BY_IDS",
    RETRIEVE_EXPORT_UNSUBSCRIBE_CONFIGS: "POSTHOG_RETRIEVE_EXPORT_UNSUBSCRIBE_CONFIGS",
    RETRIEVE_FEATURE_FLAGS_ACTIVITY_LOG: "POSTHOG_RETRIEVE_FEATURE_FLAGS_ACTIVITY_LOG",
    RETRIEVE_FEATURE_FLAGS_EVALUATION_REASONS: "POSTHOG_RETRIEVE_FEATURE_FLAGS_EVALUATION_REASONS",
    RETRIEVE_FEATURE_FLAG_ACTIVITY_DETAILS: "POSTHOG_RETRIEVE_FEATURE_FLAG_ACTIVITY_DETAILS",
    RETRIEVE_FEATURE_FLAG_DETAILS: "POSTHOG_RETRIEVE_FEATURE_FLAG_DETAILS",
    RETRIEVE_FEATURE_FLAG_ROLE_ACCESS_DETAILS: "POSTHOG_RETRIEVE_FEATURE_FLAG_ROLE_ACCESS_DETAILS",
    RETRIEVE_FILTERED_PROJECT_PROPERTY_DEFINITIONS: "POSTHOG_RETRIEVE_FILTERED_PROJECT_PROPERTY_DEFINITIONS",
    RETRIEVE_FUNNEL_INSIGHTS_IN_PROJECT: "POSTHOG_RETRIEVE_FUNNEL_INSIGHTS_IN_PROJECT",
    RETRIEVE_GROUP_PROPERTY_DEFINITIONS_BY_PROJECT_ID: "POSTHOG_RETRIEVE_GROUP_PROPERTY_DEFINITIONS_BY_PROJECT_ID",
    RETRIEVE_GROUP_PROPERTY_VALUES_BY_INDEX_AND_KEY: "POSTHOG_RETRIEVE_GROUP_PROPERTY_VALUES_BY_INDEX_AND_KEY",
    RETRIEVE_HEDGEHOG_CONFIGURATION_BY_UUID: "POSTHOG_RETRIEVE_HEDGEHOG_CONFIGURATION_BY_UUID",
    RETRIEVE_HISTORICAL_APP_METRICS_EXPORT: "POSTHOG_RETRIEVE_HISTORICAL_APP_METRICS_EXPORT",
    RETRIEVE_IMPORTANT_PROJECT_ACTIVITY_LOG: "POSTHOG_RETRIEVE_IMPORTANT_PROJECT_ACTIVITY_LOG",
    RETRIEVE_INSIGHT_SHARING_CONFIGURATIONS: "POSTHOG_RETRIEVE_INSIGHT_SHARING_CONFIGURATIONS",
    RETRIEVE_LAST_VIEWED_INSIGHTS_FOR_USER: "POSTHOG_RETRIEVE_LAST_VIEWED_INSIGHTS_FOR_USER",
    RETRIEVE_MATCHING_EVENTS_FOR_SESSION_RECORDING: "POSTHOG_RETRIEVE_MATCHING_EVENTS_FOR_SESSION_RECORDING",
    RETRIEVE_NOTEBOOK_ACTIVITY_FOR_PROJECT: "POSTHOG_RETRIEVE_NOTEBOOK_ACTIVITY_FOR_PROJECT",
    RETRIEVE_NOTEBOOK_ACTIVITY_LOG: "POSTHOG_RETRIEVE_NOTEBOOK_ACTIVITY_LOG",
    RETRIEVE_NOTEBOOK_DETAILS: "POSTHOG_RETRIEVE_NOTEBOOK_DETAILS",
    RETRIEVE_ORGANIZATION_PIPELINE_FRONTEND_APP_DETAILS: "POSTHOG_RETRIEVE_ORGANIZATION_PIPELINE_FRONTEND_APP_DETAILS",
    RETRIEVE_ORGANIZATION_PIPELINE_UNSUBSCRIBE_CONFIGS: "POSTHOG_RETRIEVE_ORGANIZATION_PIPELINE_UNSUBSCRIBE_CONFIGS",
    RETRIEVE_ORGANIZATION_PLUGIN_ACTIVITY: "POSTHOG_RETRIEVE_ORGANIZATION_PLUGIN_ACTIVITY",
    RETRIEVE_ORGANIZATION_PLUGIN_DETAILS: "POSTHOG_RETRIEVE_ORGANIZATION_PLUGIN_DETAILS",
    RETRIEVE_ORGANIZATION_PLUGIN_REPOSITORY_DETAILS: "POSTHOG_RETRIEVE_ORGANIZATION_PLUGIN_REPOSITORY_DETAILS",
    RETRIEVE_ORGANIZATION_PLUGIN_UNSUBSCRIBE_CONFIGS: "POSTHOG_RETRIEVE_ORGANIZATION_PLUGIN_UNSUBSCRIBE_CONFIGS",
    RETRIEVE_ORGANIZATION_PROJECT_DETAILS: "POSTHOG_RETRIEVE_ORGANIZATION_PROJECT_DETAILS",
    RETRIEVE_ORGANIZATION_ROLE_DETAILS: "POSTHOG_RETRIEVE_ORGANIZATION_ROLE_DETAILS",
    RETRIEVE_ORGANIZATION_S_UNUSED_PIPELINE_APPS: "POSTHOG_RETRIEVE_ORGANIZATION_S_UNUSED_PIPELINE_APPS",
    RETRIEVE_ORG_PIPELINE_REPO_DETAILS: "POSTHOG_RETRIEVE_ORG_PIPELINE_REPO_DETAILS",
    RETRIEVE_ORG_PIPELINE_TRANSFORMATION_ACTIVITY: "POSTHOG_RETRIEVE_ORG_PIPELINE_TRANSFORMATION_ACTIVITY",
    RETRIEVE_OR_DELETE_PERSON_RETENTION_DATA: "POSTHOG_RETRIEVE_OR_DELETE_PERSON_RETENTION_DATA",
    RETRIEVE_OR_DELETE_PROJECT_PERSONS_FUNNEL_DATA: "POSTHOG_RETRIEVE_OR_DELETE_PROJECT_PERSONS_FUNNEL_DATA",
    RETRIEVE_PAGINATED_ORGANIZATION_LIST: "POSTHOG_RETRIEVE_PAGINATED_ORGANIZATION_LIST",
    RETRIEVE_PERSONS_FUNNEL_CORRELATION_DATA: "POSTHOG_RETRIEVE_PERSONS_FUNNEL_CORRELATION_DATA",
    RETRIEVE_PERSON_DETAILS_BY_ID: "POSTHOG_RETRIEVE_PERSON_DETAILS_BY_ID",
    RETRIEVE_PERSON_DETAILS_BY_PROJECT: "POSTHOG_RETRIEVE_PERSON_DETAILS_BY_PROJECT",
    RETRIEVE_PERSON_LIFECYCLE_DETAILS: "POSTHOG_RETRIEVE_PERSON_LIFECYCLE_DETAILS",
    RETRIEVE_PERSON_PROPERTIES_TIMELINE: "POSTHOG_RETRIEVE_PERSON_PROPERTIES_TIMELINE",
    RETRIEVE_PERSON_S_ACTIVITY_AND_DELETE_OPTIONS: "POSTHOG_RETRIEVE_PERSON_S_ACTIVITY_AND_DELETE_OPTIONS",
    RETRIEVE_PERSON_VALUES_BY_PROJECT: "POSTHOG_RETRIEVE_PERSON_VALUES_BY_PROJECT",
    RETRIEVE_PIPELINE_DESTINATION_ACTIVITY: "POSTHOG_RETRIEVE_PIPELINE_DESTINATION_ACTIVITY",
    RETRIEVE_PIPELINE_DESTINATION_CONFIG_DETAILS: "POSTHOG_RETRIEVE_PIPELINE_DESTINATION_CONFIG_DETAILS",
    RETRIEVE_PIPELINE_DESTINATION_DETAILS: "POSTHOG_RETRIEVE_PIPELINE_DESTINATION_DETAILS",
    RETRIEVE_PIPELINE_DESTINATION_FRONTEND_CONFIG: "POSTHOG_RETRIEVE_PIPELINE_DESTINATION_FRONTEND_CONFIG",
    RETRIEVE_PIPELINE_DESTINATION_PLUGIN_DETAILS: "POSTHOG_RETRIEVE_PIPELINE_DESTINATION_PLUGIN_DETAILS",
    RETRIEVE_PIPELINE_FRONTEND_APP_ACTIVITY: "POSTHOG_RETRIEVE_PIPELINE_FRONTEND_APP_ACTIVITY",
    RETRIEVE_PIPELINE_FRONTEND_APP_DETAILS: "POSTHOG_RETRIEVE_PIPELINE_FRONTEND_APP_DETAILS",
    RETRIEVE_PIPELINE_FRONTEND_APP_SOURCE_DETAILS: "POSTHOG_RETRIEVE_PIPELINE_FRONTEND_APP_SOURCE_DETAILS",
    RETRIEVE_PIPELINE_FRONTEND_PLUGIN_CONFIG: "POSTHOG_RETRIEVE_PIPELINE_FRONTEND_PLUGIN_CONFIG",
    RETRIEVE_PIPELINE_IMPORT_APPS_ACTIVITY: "POSTHOG_RETRIEVE_PIPELINE_IMPORT_APPS_ACTIVITY",
    RETRIEVE_PIPELINE_IMPORT_APP_DETAILS: "POSTHOG_RETRIEVE_PIPELINE_IMPORT_APP_DETAILS",
    RETRIEVE_PIPELINE_IMPORT_APP_SOURCE_DETAILS: "POSTHOG_RETRIEVE_PIPELINE_IMPORT_APP_SOURCE_DETAILS",
    RETRIEVE_PIPELINE_PLUGIN_DETAILS_BY_ID: "POSTHOG_RETRIEVE_PIPELINE_PLUGIN_DETAILS_BY_ID",
    RETRIEVE_PIPELINE_PLUGIN_SOURCE: "POSTHOG_RETRIEVE_PIPELINE_PLUGIN_SOURCE",
    RETRIEVE_PIPELINE_TRANSFORMATION_ACTIVITY: "POSTHOG_RETRIEVE_PIPELINE_TRANSFORMATION_ACTIVITY",
    RETRIEVE_PIPELINE_TRANSFORMATION_CONFIG: "POSTHOG_RETRIEVE_PIPELINE_TRANSFORMATION_CONFIG",
    RETRIEVE_PIPELINE_TRANSFORMATION_FRONTEND_CONFIG: "POSTHOG_RETRIEVE_PIPELINE_TRANSFORMATION_FRONTEND_CONFIG",
    RETRIEVE_PLUGIN_CONFIG_ACTIVITY: "POSTHOG_RETRIEVE_PLUGIN_CONFIG_ACTIVITY",
    RETRIEVE_PLUGIN_CONFIG_ACTIVITY_DETAILS: "POSTHOG_RETRIEVE_PLUGIN_CONFIG_ACTIVITY_DETAILS",
    RETRIEVE_PLUGIN_CONFIG_DETAILS: "POSTHOG_RETRIEVE_PLUGIN_CONFIG_DETAILS",
    RETRIEVE_PLUGIN_CONFIG_FRONTEND: "POSTHOG_RETRIEVE_PLUGIN_CONFIG_FRONTEND",
    RETRIEVE_PLUGIN_FRONTEND_CONFIG: "POSTHOG_RETRIEVE_PLUGIN_FRONTEND_CONFIG",
    RETRIEVE_PLUGIN_SOURCE_DETAILS: "POSTHOG_RETRIEVE_PLUGIN_SOURCE_DETAILS",
    RETRIEVE_PLUGIN_UPDATE_INFO: "POSTHOG_RETRIEVE_PLUGIN_UPDATE_INFO",
    RETRIEVE_PLUGIN_UPDATE_STATUS: "POSTHOG_RETRIEVE_PLUGIN_UPDATE_STATUS",
    RETRIEVE_PROJECT_ACTION_DETAILS_BY_ID: "POSTHOG_RETRIEVE_PROJECT_ACTION_DETAILS_BY_ID",
    RETRIEVE_PROJECT_ACTIVITY_DETAILS: "POSTHOG_RETRIEVE_PROJECT_ACTIVITY_DETAILS",
    RETRIEVE_PROJECT_ACTIVITY_INSIGHTS: "POSTHOG_RETRIEVE_PROJECT_ACTIVITY_INSIGHTS",
    RETRIEVE_PROJECT_ACTIVITY_LOG_WITH_PAGINATION: "POSTHOG_RETRIEVE_PROJECT_ACTIVITY_LOG_WITH_PAGINATION",
    RETRIEVE_PROJECT_ANNOTATION_DETAILS: "POSTHOG_RETRIEVE_PROJECT_ANNOTATION_DETAILS",
    RETRIEVE_PROJECT_COHORTS_ACTIVITY_LOG: "POSTHOG_RETRIEVE_PROJECT_COHORTS_ACTIVITY_LOG",
    RETRIEVE_PROJECT_COHORTS_WITH_PAGINATION: "POSTHOG_RETRIEVE_PROJECT_COHORTS_WITH_PAGINATION",
    RETRIEVE_PROJECT_COHORT_DETAILS: "POSTHOG_RETRIEVE_PROJECT_COHORT_DETAILS",
    RETRIEVE_PROJECT_DRAFT_SQL_QUERY: "POSTHOG_RETRIEVE_PROJECT_DRAFT_SQL_QUERY",
    RETRIEVE_PROJECT_EARLY_ACCESS_FEATURE_DETAILS: "POSTHOG_RETRIEVE_PROJECT_EARLY_ACCESS_FEATURE_DETAILS",
    RETRIEVE_PROJECT_EVENT_DETAILS_BY_ID: "POSTHOG_RETRIEVE_PROJECT_EVENT_DETAILS_BY_ID",
    RETRIEVE_PROJECT_EVENT_VALUES: "POSTHOG_RETRIEVE_PROJECT_EVENT_VALUES",
    RETRIEVE_PROJECT_EXPERIMENT_RESULTS: "POSTHOG_RETRIEVE_PROJECT_EXPERIMENT_RESULTS",
    RETRIEVE_PROJECT_EXPORT_DETAILS: "POSTHOG_RETRIEVE_PROJECT_EXPORT_DETAILS",
    RETRIEVE_PROJECT_FUNNEL_CORRELATION_INSIGHTS: "POSTHOG_RETRIEVE_PROJECT_FUNNEL_CORRELATION_INSIGHTS",
    RETRIEVE_PROJECT_GROUP_DETAILS_BY_KEY_AND_TYPE: "POSTHOG_RETRIEVE_PROJECT_GROUP_DETAILS_BY_KEY_AND_TYPE",
    RETRIEVE_PROJECT_INSIGHTS_DETAILS: "POSTHOG_RETRIEVE_PROJECT_INSIGHTS_DETAILS",
    RETRIEVE_PROJECT_INSIGHTS_WITH_PAGINATION: "POSTHOG_RETRIEVE_PROJECT_INSIGHTS_WITH_PAGINATION",
    RETRIEVE_PROJECT_INSIGHT_ACTIVITY_DATA: "POSTHOG_RETRIEVE_PROJECT_INSIGHT_ACTIVITY_DATA",
    RETRIEVE_PROJECT_INSIGHT_DETAILS: "POSTHOG_RETRIEVE_PROJECT_INSIGHT_DETAILS",
    RETRIEVE_PROJECT_MEMBER_DETAILS_BY_UUID: "POSTHOG_RETRIEVE_PROJECT_MEMBER_DETAILS_BY_UUID",
    RETRIEVE_PROJECT_PERSON_ACTIVITY: "POSTHOG_RETRIEVE_PROJECT_PERSON_ACTIVITY",
    RETRIEVE_PROJECT_PERSON_COHORTS: "POSTHOG_RETRIEVE_PROJECT_PERSON_COHORTS",
    RETRIEVE_PROJECT_PERSON_STICKINESS: "POSTHOG_RETRIEVE_PROJECT_PERSON_STICKINESS",
    RETRIEVE_PROJECT_PIPELINE_PLUGIN_CONFIG: "POSTHOG_RETRIEVE_PROJECT_PIPELINE_PLUGIN_CONFIG",
    RETRIEVE_PROJECT_SESSION_VALUES: "POSTHOG_RETRIEVE_PROJECT_SESSION_VALUES",
    RETRIEVE_PROJECT_SUBSCRIPTION_DETAILS: "POSTHOG_RETRIEVE_PROJECT_SUBSCRIPTION_DETAILS",
    RETRIEVE_PROJECT_TREND_INSIGHTS: "POSTHOG_RETRIEVE_PROJECT_TREND_INSIGHTS",
    RETRIEVE_PROPERTIES_AND_EVENTS_ASSOCIATION: "POSTHOG_RETRIEVE_PROPERTIES_AND_EVENTS_ASSOCIATION",
    RETRIEVE_PROPERTY_DEFINITION_DETAILS: "POSTHOG_RETRIEVE_PROPERTY_DEFINITION_DETAILS",
    RETRIEVE_PROXY_RECORD_DETAILS: "POSTHOG_RETRIEVE_PROXY_RECORD_DETAILS",
    RETRIEVE_RELATED_PROJECT_GROUPS: "POSTHOG_RETRIEVE_RELATED_PROJECT_GROUPS",
    RETRIEVE_RETENTION_INSIGHTS: "POSTHOG_RETRIEVE_RETENTION_INSIGHTS",
    RETRIEVE_SESSION_PROPERTY_DEFINITIONS: "POSTHOG_RETRIEVE_SESSION_PROPERTY_DEFINITIONS",
    RETRIEVE_SESSION_RECORDING_DETAILS: "POSTHOG_RETRIEVE_SESSION_RECORDING_DETAILS",
    RETRIEVE_SESSION_RECORDING_ERROR_CLUSTERS: "POSTHOG_RETRIEVE_SESSION_RECORDING_ERROR_CLUSTERS",
    RETRIEVE_SESSION_RECORDING_PLAYLIST_DETAILS: "POSTHOG_RETRIEVE_SESSION_RECORDING_PLAYLIST_DETAILS",
    RETRIEVE_SESSION_RECORDING_PROPERTIES: "POSTHOG_RETRIEVE_SESSION_RECORDING_PROPERTIES",
    RETRIEVE_SESSION_RECORDING_SNAPSHOTS: "POSTHOG_RETRIEVE_SESSION_RECORDING_SNAPSHOTS",
    RETRIEVE_SIMILAR_SESSION_RECORDINGS: "POSTHOG_RETRIEVE_SIMILAR_SESSION_RECORDINGS",
    RETRIEVE_SPECIFIC_BATCH_EXPORT_DETAILS: "POSTHOG_RETRIEVE_SPECIFIC_BATCH_EXPORT_DETAILS",
    RETRIEVE_SPECIFIC_BATCH_EXPORT_RUN_DETAILS: "POSTHOG_RETRIEVE_SPECIFIC_BATCH_EXPORT_RUN_DETAILS",
    RETRIEVE_SPECIFIC_DOMAIN_DETAILS: "POSTHOG_RETRIEVE_SPECIFIC_DOMAIN_DETAILS",
    RETRIEVE_SPECIFIC_EXPORTED_ASSET_CONTENT: "POSTHOG_RETRIEVE_SPECIFIC_EXPORTED_ASSET_CONTENT",
    RETRIEVE_SPECIFIC_HISTORICAL_PLUGIN_EXPORT: "POSTHOG_RETRIEVE_SPECIFIC_HISTORICAL_PLUGIN_EXPORT",
    RETRIEVE_SPECIFIC_PIPELINE_FRONTEND_APP_CONFIG: "POSTHOG_RETRIEVE_SPECIFIC_PIPELINE_FRONTEND_APP_CONFIG",
    RETRIEVE_SPECIFIC_PROJECT_DASHBOARD_DETAILS: "POSTHOG_RETRIEVE_SPECIFIC_PROJECT_DASHBOARD_DETAILS",
    RETRIEVE_SURVEY_ACTIVITY_LOGS: "POSTHOG_RETRIEVE_SURVEY_ACTIVITY_LOGS",
    RETRIEVE_SURVEY_DETAILS_WITH_UUID_AND_PROJECT_ID: "POSTHOG_RETRIEVE_SURVEY_DETAILS_WITH_UUID_AND_PROJECT_ID",
    RETRIEVE_SURVEY_RESPONSES_COUNT_BY_PROJECT_ID: "POSTHOG_RETRIEVE_SURVEY_RESPONSES_COUNT_BY_PROJECT_ID",
    RETRIEVE_UNSUBSCRIBED_PIPELINE_CONFIGS: "POSTHOG_RETRIEVE_UNSUBSCRIBED_PIPELINE_CONFIGS",
    RETRIEVE_UNUSED_ORGANIZATION_PIPELINE_FRONTEND_APPS: "POSTHOG_RETRIEVE_UNUSED_ORGANIZATION_PIPELINE_FRONTEND_APPS",
    RETRIEVE_UNUSED_ORGANIZATION_PLUGINS: "POSTHOG_RETRIEVE_UNUSED_ORGANIZATION_PLUGINS",
    RETRIEVE_UNUSED_PIPELINE_DESTINATIONS: "POSTHOG_RETRIEVE_UNUSED_PIPELINE_DESTINATIONS",
    RETRIEVE_USER_DETAILS_FOR_INITIATING_2_FA_SETUP: "POSTHOG_RETRIEVE_USER_DETAILS_FOR_INITIATING_2_FA_SETUP",
    RETRIEVE_USER_PROFILE_AND_TEAM_DETAILS: "POSTHOG_RETRIEVE_USER_PROFILE_AND_TEAM_DETAILS",
    REVOKE_ORGANIZATION_INVITE: "POSTHOG_REVOKE_ORGANIZATION_INVITE",
    SET_INSIGHT_DELETION_FLAG: "POSTHOG_SET_INSIGHT_DELETION_FLAG",
    SOFT_DELETE_A_SPECIFIC_COHORT_BY_ID: "POSTHOG_SOFT_DELETE_A_SPECIFIC_COHORT_BY_ID",
    SOFT_DELETE_FEATURE_FLAG_BY_SETTING_STATUS: "POSTHOG_SOFT_DELETE_FEATURE_FLAG_BY_SETTING_STATUS",
    SOFT_DELETE_PROJECT_DASHBOARD: "POSTHOG_SOFT_DELETE_PROJECT_DASHBOARD",
    SPLIT_PERSON_ENTITIES_IN_PROJECT: "POSTHOG_SPLIT_PERSON_ENTITIES_IN_PROJECT",
    TRIGGER_PROJECT_EXPORT_WITH_FORMAT_OPTION: "POSTHOG_TRIGGER_PROJECT_EXPORT_WITH_FORMAT_OPTION",
    UPDATE_ANNOTATION_DETAILS: "POSTHOG_UPDATE_ANNOTATION_DETAILS",
    UPDATE_EARLY_ACCESS_PROJECT_FEATURE: "POSTHOG_UPDATE_EARLY_ACCESS_PROJECT_FEATURE",
    UPDATE_EVENT_DEFINITION_BY_UUID: "POSTHOG_UPDATE_EVENT_DEFINITION_BY_UUID",
    UPDATE_EXPERIMENT_DETAILS: "POSTHOG_UPDATE_EXPERIMENT_DETAILS",
    UPDATE_FEATURE_FLAG_CONFIGURATION: "POSTHOG_UPDATE_FEATURE_FLAG_CONFIGURATION",
    UPDATE_ORGANIZATION_DETAILS_BY_UUID: "POSTHOG_UPDATE_ORGANIZATION_DETAILS_BY_UUID",
    UPDATE_ORGANIZATION_DOMAIN_INFO_AND_SAML_CONFIG: "POSTHOG_UPDATE_ORGANIZATION_DOMAIN_INFO_AND_SAML_CONFIG",
    UPDATE_ORGANIZATION_DOMAIN_PROPERTIES: "POSTHOG_UPDATE_ORGANIZATION_DOMAIN_PROPERTIES",
    UPDATE_ORGANIZATION_MEMBER_BY_UUIDS: "POSTHOG_UPDATE_ORGANIZATION_MEMBER_BY_UUIDS",
    UPDATE_ORGANIZATION_MEMBER_DETAILS: "POSTHOG_UPDATE_ORGANIZATION_MEMBER_DETAILS",
    UPDATE_ORGANIZATION_S_PROXY_RECORD_DOMAIN: "POSTHOG_UPDATE_ORGANIZATION_S_PROXY_RECORD_DOMAIN",
    UPDATE_PARTIAL_PROJECT_COHORT_DETAILS: "POSTHOG_UPDATE_PARTIAL_PROJECT_COHORT_DETAILS",
    UPDATE_PARTIAL_ROLE_DETAILS_IN_ORGANIZATION: "POSTHOG_UPDATE_PARTIAL_ROLE_DETAILS_IN_ORGANIZATION",
    UPDATE_PARTIAL_SUBSCRIPTION_DETAILS: "POSTHOG_UPDATE_PARTIAL_SUBSCRIPTION_DETAILS",
    UPDATE_PARTIAL_SURVEY_DETAILS: "POSTHOG_UPDATE_PARTIAL_SURVEY_DETAILS",
    UPDATE_PERSON_PROPERTIES: "POSTHOG_UPDATE_PERSON_PROPERTIES",
    UPDATE_PERSON_PROPERTY_ENDPOINT: "POSTHOG_UPDATE_PERSON_PROPERTY_ENDPOINT",
    UPDATE_PIPELINE_DESTINATION_CONFIG: "POSTHOG_UPDATE_PIPELINE_DESTINATION_CONFIG",
    UPDATE_PIPELINE_FRONTEND_APP_CONFIG: "POSTHOG_UPDATE_PIPELINE_FRONTEND_APP_CONFIG",
    UPDATE_PIPELINE_PLUGIN_CONFIG: "POSTHOG_UPDATE_PIPELINE_PLUGIN_CONFIG",
    UPDATE_PIPELINE_TRANSFORMATION_CONFIG: "POSTHOG_UPDATE_PIPELINE_TRANSFORMATION_CONFIG",
    UPDATE_PLUGIN_CONFIG_FOR_PROJECT: "POSTHOG_UPDATE_PLUGIN_CONFIG_FOR_PROJECT",
    UPDATE_PROJECT_ACTION_BY_ID: "POSTHOG_UPDATE_PROJECT_ACTION_BY_ID",
    UPDATE_PROJECT_ACTION_WITH_PARTIAL_DATA: "POSTHOG_UPDATE_PROJECT_ACTION_WITH_PARTIAL_DATA",
    UPDATE_PROJECT_COHORT_DETAILS: "POSTHOG_UPDATE_PROJECT_COHORT_DETAILS",
    UPDATE_PROJECT_DASHBOARD_DETAILS: "POSTHOG_UPDATE_PROJECT_DASHBOARD_DETAILS",
    UPDATE_PROJECT_EARLY_ACCESS_FEATURE: "POSTHOG_UPDATE_PROJECT_EARLY_ACCESS_FEATURE",
    UPDATE_PROJECT_EVENT_DEFINITION: "POSTHOG_UPDATE_PROJECT_EVENT_DEFINITION",
    UPDATE_PROJECT_GROUP_TYPE_METADATA: "POSTHOG_UPDATE_PROJECT_GROUP_TYPE_METADATA",
    UPDATE_PROJECT_INSIGHT_DETAILS: "POSTHOG_UPDATE_PROJECT_INSIGHT_DETAILS",
    UPDATE_PROJECT_MEMBER_ROLE_BY_UUID: "POSTHOG_UPDATE_PROJECT_MEMBER_ROLE_BY_UUID",
    UPDATE_PROJECT_MEMBER_S_ROLE_BY_UUID: "POSTHOG_UPDATE_PROJECT_MEMBER_S_ROLE_BY_UUID",
    UPDATE_PROJECT_NOTEBOOK_DETAILS: "POSTHOG_UPDATE_PROJECT_NOTEBOOK_DETAILS",
    UPDATE_PROJECT_PIPELINE_DESTINATION_CONFIG: "POSTHOG_UPDATE_PROJECT_PIPELINE_DESTINATION_CONFIG",
    UPDATE_PROJECT_PIPELINE_TRANSFORMATION_CONFIG: "POSTHOG_UPDATE_PROJECT_PIPELINE_TRANSFORMATION_CONFIG",
    UPDATE_PROJECT_PLUGIN_CONFIGURATION: "POSTHOG_UPDATE_PROJECT_PLUGIN_CONFIGURATION",
    UPDATE_PROJECT_PROPERTY_DEFINITION: "POSTHOG_UPDATE_PROJECT_PROPERTY_DEFINITION",
    UPDATE_PROJECT_SESSION_RECORDING_PLAYLIST: "POSTHOG_UPDATE_PROJECT_SESSION_RECORDING_PLAYLIST",
    UPDATE_PROJECT_SUBSCRIPTION_DETAILS: "POSTHOG_UPDATE_PROJECT_SUBSCRIPTION_DETAILS",
    UPDATE_PROJECT_SURVEY_DETAILS: "POSTHOG_UPDATE_PROJECT_SURVEY_DETAILS",
    UPDATE_ROLE_DETAILS_IN_ORGANIZATION: "POSTHOG_UPDATE_ROLE_DETAILS_IN_ORGANIZATION",
    UPDATE_SESSION_RECORDING_PLAYLIST_PARTIALLY: "POSTHOG_UPDATE_SESSION_RECORDING_PLAYLIST_PARTIALLY",
    UPDATE_SPECIFIC_PROXY_RECORD_FIELDS: "POSTHOG_UPDATE_SPECIFIC_PROXY_RECORD_FIELDS",
  },
  triggerTypes: {},
}
