// @composio/core/generated/rocketlane.ts
/**
 * Map of Composio's ROCKETLANE toolkit.
 */
export const ROCKETLANE = {
  slug: "rocketlane",
  tools: {
    ARCHIVE_PROJECT_BY_ID: "ROCKETLANE_ARCHIVE_PROJECT_BY_ID",
    CREATE_COMPANY: "ROCKETLANE_CREATE_COMPANY",
    CREATE_TASK: "ROCKETLANE_CREATE_TASK",
    CREATE_TIME_ENTRY: "ROCKETLANE_CREATE_TIME_ENTRY",
    DELETE_PROJECT: "ROCKETLANE_DELETE_PROJECT",
    DELETE_TASK_BY_ID: "ROCKETLANE_DELETE_TASK_BY_ID",
    DELETE_TIME_ENTRY_BY_ID: "ROCKETLANE_DELETE_TIME_ENTRY_BY_ID",
    GET_ALL_FIELDS: "ROCKETLANE_GET_ALL_FIELDS",
    GET_COMPANY: "ROCKETLANE_GET_COMPANY",
    GET_FIELD_BY_ID: "ROCKETLANE_GET_FIELD_BY_ID",
    GET_PROJECT_BY_ID: "ROCKETLANE_GET_PROJECT_BY_ID",
    GET_TASK_BY_ID: "ROCKETLANE_GET_TASK_BY_ID",
    GET_TEMPLATE_BY_ID: "ROCKETLANE_GET_TEMPLATE_BY_ID",
    GET_TIME_ENTRIES: "ROCKETLANE_GET_TIME_ENTRIES",
    GET_TIME_ENTRY_BY_ID: "ROCKETLANE_GET_TIME_ENTRY_BY_ID",
    GET_USER_BY_ID: "ROCKETLANE_GET_USER_BY_ID",
    LIST_COMPANIES: "ROCKETLANE_LIST_COMPANIES",
    LIST_COMPANY_FIELDS: "ROCKETLANE_LIST_COMPANY_FIELDS",
    LIST_COMPANY_NOTE_FIELDS: "ROCKETLANE_LIST_COMPANY_NOTE_FIELDS",
    LIST_CURRENCIES: "ROCKETLANE_LIST_CURRENCIES",
    LIST_CUSTOMER_USERS: "ROCKETLANE_LIST_CUSTOMER_USERS",
    LIST_PROJECTS: "ROCKETLANE_LIST_PROJECTS",
    LIST_PROJECT_FIELDS: "ROCKETLANE_LIST_PROJECT_FIELDS",
    LIST_PROJECT_PHASES: "ROCKETLANE_LIST_PROJECT_PHASES",
    LIST_TASK_FIELDS: "ROCKETLANE_LIST_TASK_FIELDS",
    LIST_TEMPLATES: "ROCKETLANE_LIST_TEMPLATES",
    LIST_USERS: "ROCKETLANE_LIST_USERS",
    LIST_VENDOR_USERS: "ROCKETLANE_LIST_VENDOR_USERS",
    RETRIEVE_SUBSCRIPTION_DETAILS: "ROCKETLANE_RETRIEVE_SUBSCRIPTION_DETAILS",
    SEARCH_USER_BY_EMAIL: "ROCKETLANE_SEARCH_USER_BY_EMAIL",
    UPDATE_COMPANY: "ROCKETLANE_UPDATE_COMPANY",
    UPDATE_PROJECT_BY_ID: "ROCKETLANE_UPDATE_PROJECT_BY_ID",
    UPDATE_TIME_ENTRY_BY_ID: "ROCKETLANE_UPDATE_TIME_ENTRY_BY_ID",
  },
  triggerTypes: {},
}
