// @composio/core/generated/salesforce.ts
/**
 * Map of Composio's SALESFORCE toolkit.
 */
export const SALESFORCE = {
  slug: "salesforce",
  tools: {
    ACCOUNT_CREATION_WITH_CONTENT_TYPE_OPTION: "SALESFORCE_ACCOUNT_CREATION_WITH_CONTENT_TYPE_OPTION",
    CREATE_CAMPAIGN_RECORD_VIA_POST: "SALESFORCE_CREATE_CAMPAIGN_RECORD_VIA_POST",
    CREATE_LEAD_WITH_SPECIFIED_CONTENT_TYPE: "SALESFORCE_CREATE_LEAD_WITH_SPECIFIED_CONTENT_TYPE",
    CREATE_NEW_CONTACT_WITH_JSON_HEADER: "SALESFORCE_CREATE_NEW_CONTACT_WITH_JSON_HEADER",
    CREATE_NOTE_RECORD_WITH_CONTENT_TYPE_HEADER: "SALESFORCE_CREATE_NOTE_RECORD_WITH_CONTENT_TYPE_HEADER",
    CREATE_OPPORTUNITY_RECORD: "SALESFORCE_CREATE_OPPORTUNITY_RECORD",
    DELETE_A_LEAD_OBJECT_BY_ITS_ID: "SALESFORCE_DELETE_A_LEAD_OBJECT_BY_ITS_ID",
    EXECUTE_SOQL_QUERY: "SALESFORCE_EXECUTE_SOQL_QUERY",
    FETCH_ACCOUNT_BY_ID_WITH_QUERY: "SALESFORCE_FETCH_ACCOUNT_BY_ID_WITH_QUERY",
    FETCH_MODIFIED_OR_UNMODIFIED_SOBJECTS: "SALESFORCE_FETCH_MODIFIED_OR_UNMODIFIED_SOBJECTS",
    QUERY_CONTACTS_BY_NAME: "SALESFORCE_QUERY_CONTACTS_BY_NAME",
    QUERY_REPORT: "SALESFORCE_QUERY_REPORT",
    REMOVE_ACCOUNT_BY_UNIQUE_IDENTIFIER: "SALESFORCE_REMOVE_ACCOUNT_BY_UNIQUE_IDENTIFIER",
    REMOVE_A_SPECIFIC_CONTACT_BY_ID: "SALESFORCE_REMOVE_A_SPECIFIC_CONTACT_BY_ID",
    REMOVE_CAMPAIGN_OBJECT_BY_ID: "SALESFORCE_REMOVE_CAMPAIGN_OBJECT_BY_ID",
    REMOVE_NOTE_OBJECT_BY_ID: "SALESFORCE_REMOVE_NOTE_OBJECT_BY_ID",
    REMOVE_OPPORTUNITY_BY_ID: "SALESFORCE_REMOVE_OPPORTUNITY_BY_ID",
    RETRIEVE_ACCOUNT_DATA_AND_ERROR_RESPONSES: "SALESFORCE_RETRIEVE_ACCOUNT_DATA_AND_ERROR_RESPONSES",
    RETRIEVE_CAMPAIGN_DATA_WITH_ERROR_HANDLING: "SALESFORCE_RETRIEVE_CAMPAIGN_DATA_WITH_ERROR_HANDLING",
    RETRIEVE_CONTACT_INFO_WITH_STANDARD_RESPONSES: "SALESFORCE_RETRIEVE_CONTACT_INFO_WITH_STANDARD_RESPONSES",
    RETRIEVE_LEAD_BY_ID: "SALESFORCE_RETRIEVE_LEAD_BY_ID",
    RETRIEVE_LEAD_DATA_WITH_VARIOUS_RESPONSES: "SALESFORCE_RETRIEVE_LEAD_DATA_WITH_VARIOUS_RESPONSES",
    RETRIEVE_NOTE_OBJECT_INFORMATION: "SALESFORCE_RETRIEVE_NOTE_OBJECT_INFORMATION",
    RETRIEVE_NOTE_WITH_CONDITIONS: "SALESFORCE_RETRIEVE_NOTE_WITH_CONDITIONS",
    RETRIEVE_OPPORTUNITIES_DATA: "SALESFORCE_RETRIEVE_OPPORTUNITIES_DATA",
    RETRIEVE_OPPORTUNITY_BY_ID_WITH_OPTIONAL_FIELDS: "SALESFORCE_RETRIEVE_OPPORTUNITY_BY_ID_WITH_OPTIONAL_FIELDS",
    RETRIEVE_SPECIFIC_CAMPAIGN_OBJECT_DETAILS: "SALESFORCE_RETRIEVE_SPECIFIC_CAMPAIGN_OBJECT_DETAILS",
    RETRIEVE_SPECIFIC_CONTACT_BY_ID: "SALESFORCE_RETRIEVE_SPECIFIC_CONTACT_BY_ID",
    UPDATE_ACCOUNT_OBJECT_BY_ID: "SALESFORCE_UPDATE_ACCOUNT_OBJECT_BY_ID",
    UPDATE_CAMPAIGN_BY_ID_WITH_JSON: "SALESFORCE_UPDATE_CAMPAIGN_BY_ID_WITH_JSON",
    UPDATE_CONTACT_BY_ID: "SALESFORCE_UPDATE_CONTACT_BY_ID",
    UPDATE_LEAD_BY_ID_WITH_JSON_PAYLOAD: "SALESFORCE_UPDATE_LEAD_BY_ID_WITH_JSON_PAYLOAD",
    UPDATE_OPPORTUNITY_BY_ID: "SALESFORCE_UPDATE_OPPORTUNITY_BY_ID",
    UPDATE_SPECIFIC_NOTE_BY_ID: "SALESFORCE_UPDATE_SPECIFIC_NOTE_BY_ID",
  },
  triggerTypes: {
    NEW_CONTACT_TRIGGER: "SALESFORCE_NEW_CONTACT_TRIGGER",
    NEW_LEAD_TRIGGER: "SALESFORCE_NEW_LEAD_TRIGGER",
  },
}
