// @composio/core/generated/scrape_do.ts
/**
 * Map of Composio's SCRAPE_DO toolkit.
 */
export const SCRAPE_DO = {
  slug: "scrape_do",
  tools: {
    GET_ACCOUNT_INFO: "SCRAPE_DO_GET_ACCOUNT_INFO",
    GET_RENDER_PAGE: "SCRAPE_DO_GET_RENDER_PAGE",
    SCRAPE_DO_GET_PAGE: "SCRAPE_DO_SCRAPE_DO_GET_PAGE",
    SCRAPE_DO_PROXY_MODE: "SCRAPE_DO_SCRAPE_DO_PROXY_MODE",
    SCRAPE_DO_SET_COOKIES: "SCRAPE_DO_SCRAPE_DO_SET_COOKIES",
    SCRAPE_DO_SET_SUPER_MODE: "SCRAPE_DO_SCRAPE_DO_SET_SUPER_MODE",
    SET_BLOCK_URLS: "SCRAPE_DO_SET_BLOCK_URLS",
    SET_CUSTOM_HEADERS: "SCRAPE_DO_SET_CUSTOM_HEADERS",
    SET_CUSTOM_WAIT: "SCRAPE_DO_SET_CUSTOM_WAIT",
    SET_DEVICE_TYPE: "SCRAPE_DO_SET_DEVICE_TYPE",
    SET_DISABLE_REDIRECTION: "SCRAPE_DO_SET_DISABLE_REDIRECTION",
    SET_PURE_COOKIES: "SCRAPE_DO_SET_PURE_COOKIES",
    SET_REGIONAL_GEO_CODE: "SCRAPE_DO_SET_REGIONAL_GEO_CODE",
    SET_RETRY_TIMEOUT: "SCRAPE_DO_SET_RETRY_TIMEOUT",
    SET_SCREENSHOT: "SCRAPE_DO_SET_SCREENSHOT",
    SET_SESSION_ID: "SCRAPE_DO_SET_SESSION_ID",
    SET_WAIT_FOR_SELECTOR: "SCRAPE_DO_SET_WAIT_FOR_SELECTOR",
    SET_WAIT_UNTIL: "SCRAPE_DO_SET_WAIT_UNTIL",
    SHOW_WEBSOCKET_REQUESTS: "SCRAPE_DO_SHOW_WEBSOCKET_REQUESTS",
  },
  triggerTypes: {},
}
