// @composio/core/generated/semanticscholar.ts
/**
 * Map of Composio's SEMANTICSCHOLAR toolkit.
 */
export const SEMANTICSCHOLAR = {
  slug: "semanticscholar",
  tools: {
    DETAILS_ABOUT_AN_AUTHOR: "SEMANTICSCHOLAR_DETAILS_ABOUT_AN_AUTHOR",
    DETAILS_ABOUT_AN_AUTHOR_S_PAPERS: "SEMANTICSCHOLAR_DETAILS_ABOUT_AN_AUTHOR_S_PAPERS",
    DETAILS_ABOUT_A_PAPER: "SEMANTICSCHOLAR_DETAILS_ABOUT_A_PAPER",
    DETAILS_ABOUT_A_PAPER_S_AUTHORS: "SEMANTICSCHOLAR_DETAILS_ABOUT_A_PAPER_S_AUTHORS",
    DETAILS_ABOUT_A_PAPER_S_CITATIONS: "SEMANTICSCHOLAR_DETAILS_ABOUT_A_PAPER_S_CITATIONS",
    DETAILS_ABOUT_A_PAPER_S_REFERENCES: "SEMANTICSCHOLAR_DETAILS_ABOUT_A_PAPER_S_REFERENCES",
    GET_DETAILS_FOR_MULTIPLE_AUTHORS_AT_ONCE: "SEMANTICSCHOLAR_GET_DETAILS_FOR_MULTIPLE_AUTHORS_AT_ONCE",
    GET_DETAILS_FOR_MULTIPLE_PAPERS_AT_ONCE: "SEMANTICSCHOLAR_GET_DETAILS_FOR_MULTIPLE_PAPERS_AT_ONCE",
    PAPER_BULK_SEARCH: "SEMANTICSCHOLAR_PAPER_BULK_SEARCH",
    PAPER_RELEVANCE_SEARCH: "SEMANTICSCHOLAR_PAPER_RELEVANCE_SEARCH",
    PAPER_TITLE_SEARCH: "SEMANTICSCHOLAR_PAPER_TITLE_SEARCH",
    SEARCH_FOR_AUTHORS_BY_NAME: "SEMANTICSCHOLAR_SEARCH_FOR_AUTHORS_BY_NAME",
    SUGGEST_PAPER_QUERY_COMPLETIONS: "SEMANTICSCHOLAR_SUGGEST_PAPER_QUERY_COMPLETIONS",
    TEXT_SNIPPET_SEARCH: "SEMANTICSCHOLAR_TEXT_SNIPPET_SEARCH",
  },
  triggerTypes: {},
}
