// @composio/core/generated/semrush.ts
/**
 * Map of Composio's SEMRUSH toolkit.
 */
export const SEMRUSH = {
  slug: "semrush",
  tools: {
    ADS_COPIES: "SEMRUSH_ADS_COPIES",
    ANCHORS: "SEMRUSH_ANCHORS",
    AUTHORITY_SCORE_PROFILE: "SEMRUSH_AUTHORITY_SCORE_PROFILE",
    BACKLINKS: "SEMRUSH_BACKLINKS",
    BACKLINKS_OVERVIEW: "SEMRUSH_BACKLINKS_OVERVIEW",
    BATCH_COMPARISON: "SEMRUSH_BATCH_COMPARISON",
    BATCH_KEYWORD_OVERVIEW: "SEMRUSH_BATCH_KEYWORD_OVERVIEW",
    BROAD_MATCH_KEYWORD: "SEMRUSH_BROAD_MATCH_KEYWORD",
    CATEGORIES: "SEMRUSH_CATEGORIES",
    CATEGORIES_PROFILE: "SEMRUSH_CATEGORIES_PROFILE",
    COMPETITORS: "SEMRUSH_COMPETITORS",
    COMPETITORS_IN_ORGANIC_SEARCH: "SEMRUSH_COMPETITORS_IN_ORGANIC_SEARCH",
    COMPETITORS_IN_PAID_SEARCH: "SEMRUSH_COMPETITORS_IN_PAID_SEARCH",
    DOMAIN_AD_HISTORY: "SEMRUSH_DOMAIN_AD_HISTORY",
    DOMAIN_ORGANIC_PAGES: "SEMRUSH_DOMAIN_ORGANIC_PAGES",
    DOMAIN_ORGANIC_SEARCH_KEYWORDS: "SEMRUSH_DOMAIN_ORGANIC_SEARCH_KEYWORDS",
    DOMAIN_ORGANIC_SUBDOMAINS: "SEMRUSH_DOMAIN_ORGANIC_SUBDOMAINS",
    DOMAIN_PAID_SEARCH_KEYWORDS: "SEMRUSH_DOMAIN_PAID_SEARCH_KEYWORDS",
    DOMAIN_PLA_SEARCH_KEYWORDS: "SEMRUSH_DOMAIN_PLA_SEARCH_KEYWORDS",
    DOMAIN_VS_DOMAIN: "SEMRUSH_DOMAIN_VS_DOMAIN",
    HISTORICAL_DATA: "SEMRUSH_HISTORICAL_DATA",
    INDEXED_PAGES: "SEMRUSH_INDEXED_PAGES",
    KEYWORDS_ADS_HISTORY: "SEMRUSH_KEYWORDS_ADS_HISTORY",
    KEYWORD_DIFFICULTY: "SEMRUSH_KEYWORD_DIFFICULTY",
    KEYWORD_OVERVIEW_ALL_DATABASES: "SEMRUSH_KEYWORD_OVERVIEW_ALL_DATABASES",
    KEYWORD_OVERVIEW_ONE_DATABASE: "SEMRUSH_KEYWORD_OVERVIEW_ONE_DATABASE",
    ORGANIC_RESULTS: "SEMRUSH_ORGANIC_RESULTS",
    PAID_RESULTS: "SEMRUSH_PAID_RESULTS",
    PHRASE_QUESTIONS: "SEMRUSH_PHRASE_QUESTIONS",
    PLA_COMPETITORS: "SEMRUSH_PLA_COMPETITORS",
    PLA_COPIES: "SEMRUSH_PLA_COPIES",
    REFERRING_DOMAINS: "SEMRUSH_REFERRING_DOMAINS",
    REFERRING_DOMAINS_BY_COUNTRY: "SEMRUSH_REFERRING_DOMAINS_BY_COUNTRY",
    REFERRING_I_PS: "SEMRUSH_REFERRING_I_PS",
    RELATED_KEYWORDS: "SEMRUSH_RELATED_KEYWORDS",
    TLD_DISTRIBUTION: "SEMRUSH_TLD_DISTRIBUTION",
  },
  triggerTypes: {},
}
