// @composio/core/generated/sendgrid.ts
/**
 * Map of Composio's SENDGRID toolkit.
 */
export const SENDGRID = {
  slug: "sendgrid",
  tools: {
    ACTIVATE_TEMPLATE_VERSION: "SENDGRID_ACTIVATE_TEMPLATE_VERSION",
    ADD_AN_IP_ADDRESS_TO_A_POOL: "SENDGRID_ADD_AN_IP_ADDRESS_TO_A_POOL",
    ADD_AN_IP_TO_AN_AUTHENTICATED_DOMAIN: "SENDGRID_ADD_AN_IP_TO_AN_AUTHENTICATED_DOMAIN",
    ADD_A_BATCH_OF_IPS_TO_AN_IP_POOL: "SENDGRID_ADD_A_BATCH_OF_IPS_TO_AN_IP_POOL",
    ADD_A_SINGLE_RECIPIENT_TO_A_LIST: "SENDGRID_ADD_A_SINGLE_RECIPIENT_TO_A_LIST",
    ADD_A_TWILIO_SEND_GRID_IP_ADDRESS: "SENDGRID_ADD_A_TWILIO_SEND_GRID_IP_ADDRESS",
    ADD_IPS: "SENDGRID_ADD_IPS",
    ADD_ONE_OR_MORE_IPS_TO_THE_ALLOW_LIST: "SENDGRID_ADD_ONE_OR_MORE_IPS_TO_THE_ALLOW_LIST",
    ADD_OR_UPDATE_A_CONTACT: "SENDGRID_ADD_OR_UPDATE_A_CONTACT",
    ADD_SUPPRESSIONS_TO_A_SUPPRESSION_GROUP: "SENDGRID_ADD_SUPPRESSIONS_TO_A_SUPPRESSION_GROUP",
    ADD_TO_GLOBAL_SUPPRESSIONS_GROUP: "SENDGRID_ADD_TO_GLOBAL_SUPPRESSIONS_GROUP",
    APPROVE_ACCESS_REQUEST: "SENDGRID_APPROVE_ACCESS_REQUEST",
    ASSIGN_A_BATCH_OF_SUBUSERS_TO_AN_IP: "SENDGRID_ASSIGN_A_BATCH_OF_SUBUSERS_TO_AN_IP",
    ASSOCIATE_AN_AUTHENTICATED_DOMAIN_WITH_A_GIVEN_USER: "SENDGRID_ASSOCIATE_AN_AUTHENTICATED_DOMAIN_WITH_A_GIVEN_USER",
    ASSOCIATE_A_BRANDED_LINK_WITH_A_SUBUSER: "SENDGRID_ASSOCIATE_A_BRANDED_LINK_WITH_A_SUBUSER",
    AUTHENTICATE_AN_ACCOUNT_WITH_SINGLE_SIGN_ON: "SENDGRID_AUTHENTICATE_AN_ACCOUNT_WITH_SINGLE_SIGN_ON",
    AUTHENTICATE_A_DOMAIN: "SENDGRID_AUTHENTICATE_A_DOMAIN",
    BIND_AUTHENTICATED_DOMAINS_TO_USER: "SENDGRID_BIND_AUTHENTICATED_DOMAINS_TO_USER",
    BULK_DELETE_SINGLE_SENDS: "SENDGRID_BULK_DELETE_SINGLE_SENDS",
    CANCEL_OR_PAUSE_A_SCHEDULED_SEND: "SENDGRID_CANCEL_OR_PAUSE_A_SCHEDULED_SEND",
    COMPLETED_STEPS: "SENDGRID_COMPLETED_STEPS",
    CREATE_AN_ACCOUNT: "SENDGRID_CREATE_AN_ACCOUNT",
    CREATE_AN_IP_POOL: "SENDGRID_CREATE_AN_IP_POOL",
    CREATE_AN_IP_POOL_WITH_A_NAME_AND_IP_ASSIGNMENTS: "SENDGRID_CREATE_AN_IP_POOL_WITH_A_NAME_AND_IP_ASSIGNMENTS",
    CREATE_AN_SSO_CERTIFICATE: "SENDGRID_CREATE_AN_SSO_CERTIFICATE",
    CREATE_AN_SSO_INTEGRATION: "SENDGRID_CREATE_AN_SSO_INTEGRATION",
    CREATE_AN_SSO_TEAMMATE: "SENDGRID_CREATE_AN_SSO_TEAMMATE",
    CREATE_API_KEYS: "SENDGRID_CREATE_API_KEYS",
    CREATE_A_BATCH_ID: "SENDGRID_CREATE_A_BATCH_ID",
    CREATE_A_BRANDED_LINK: "SENDGRID_CREATE_A_BRANDED_LINK",
    CREATE_A_CAMPAIGN: "SENDGRID_CREATE_A_CAMPAIGN",
    CREATE_A_CUSTOM_FIELD: "SENDGRID_CREATE_A_CUSTOM_FIELD",
    CREATE_A_LIST: "SENDGRID_CREATE_A_LIST",
    CREATE_A_NEW_ALERT: "SENDGRID_CREATE_A_NEW_ALERT",
    CREATE_A_NEW_EVENT_WEBHOOK: "SENDGRID_CREATE_A_NEW_EVENT_WEBHOOK",
    CREATE_A_NEW_SUPPRESSION_GROUP: "SENDGRID_CREATE_A_NEW_SUPPRESSION_GROUP",
    CREATE_A_NEW_TRANSACTIONAL_TEMPLATE_VERSION: "SENDGRID_CREATE_A_NEW_TRANSACTIONAL_TEMPLATE_VERSION",
    CREATE_A_PARSE_SETTING: "SENDGRID_CREATE_A_PARSE_SETTING",
    CREATE_A_SEGMENT: "SENDGRID_CREATE_A_SEGMENT",
    CREATE_A_SENDER: "SENDGRID_CREATE_A_SENDER",
    CREATE_A_SENDER_IDENTITY: "SENDGRID_CREATE_A_SENDER_IDENTITY",
    CREATE_A_TRANSACTIONAL_TEMPLATE: "SENDGRID_CREATE_A_TRANSACTIONAL_TEMPLATE",
    CREATE_CUSTOM_FIELD_DEFINITION: "SENDGRID_CREATE_CUSTOM_FIELD_DEFINITION",
    CREATE_DESIGN: "SENDGRID_CREATE_DESIGN",
    CREATE_INTEGRATION: "SENDGRID_CREATE_INTEGRATION",
    CREATE_LIST: "SENDGRID_CREATE_LIST",
    CREATE_SEGMENT: "SENDGRID_CREATE_SEGMENT",
    CREATE_SINGLE_SEND: "SENDGRID_CREATE_SINGLE_SEND",
    CREATE_SUBUSER: "SENDGRID_CREATE_SUBUSER",
    CREATE_VERIFIED_SENDER_REQUEST: "SENDGRID_CREATE_VERIFIED_SENDER_REQUEST",
    DELETE_AN_ACCOUNT: "SENDGRID_DELETE_AN_ACCOUNT",
    DELETE_AN_ALERT: "SENDGRID_DELETE_AN_ALERT",
    DELETE_AN_AUTHENTICATED_DOMAIN: "SENDGRID_DELETE_AN_AUTHENTICATED_DOMAIN",
    DELETE_AN_IP_POOL: "SENDGRID_DELETE_AN_IP_POOL",
    DELETE_AN_SSO_CERTIFICATE: "SENDGRID_DELETE_AN_SSO_CERTIFICATE",
    DELETE_AN_SSO_INTEGRATION: "SENDGRID_DELETE_AN_SSO_INTEGRATION",
    DELETE_API_KEYS: "SENDGRID_DELETE_API_KEYS",
    DELETE_A_BATCH_OF_IPS_FROM_AN_IP_POOL: "SENDGRID_DELETE_A_BATCH_OF_IPS_FROM_AN_IP_POOL",
    DELETE_A_BATCH_OF_SUBUSERS_FROM_AN_IP: "SENDGRID_DELETE_A_BATCH_OF_SUBUSERS_FROM_AN_IP",
    DELETE_A_BOUNCE: "SENDGRID_DELETE_A_BOUNCE",
    DELETE_A_BRANDED_LINK: "SENDGRID_DELETE_A_BRANDED_LINK",
    DELETE_A_CAMPAIGN: "SENDGRID_DELETE_A_CAMPAIGN",
    DELETE_A_CONTACT_IDENTIFIER: "SENDGRID_DELETE_A_CONTACT_IDENTIFIER",
    DELETE_A_CUSTOM_FIELD: "SENDGRID_DELETE_A_CUSTOM_FIELD",
    DELETE_A_GLOBAL_SUPPRESSION: "SENDGRID_DELETE_A_GLOBAL_SUPPRESSION",
    DELETE_A_LIST: "SENDGRID_DELETE_A_LIST",
    DELETE_A_PARSE_SETTING: "SENDGRID_DELETE_A_PARSE_SETTING",
    DELETE_A_RECIPIENT: "SENDGRID_DELETE_A_RECIPIENT",
    DELETE_A_REVERSE_DNS_RECORD: "SENDGRID_DELETE_A_REVERSE_DNS_RECORD",
    DELETE_A_SEGMENT: "SENDGRID_DELETE_A_SEGMENT",
    DELETE_A_SENDER: "SENDGRID_DELETE_A_SENDER",
    DELETE_A_SENDER_IDENTITY: "SENDGRID_DELETE_A_SENDER_IDENTITY",
    DELETE_A_SINGLE_EVENT_WEBHOOK_BY_ID: "SENDGRID_DELETE_A_SINGLE_EVENT_WEBHOOK_BY_ID",
    DELETE_A_SINGLE_RECIPIENT_FROM_A_SINGLE_LIST: "SENDGRID_DELETE_A_SINGLE_RECIPIENT_FROM_A_SINGLE_LIST",
    DELETE_A_SPECIFIC_BLOCK: "SENDGRID_DELETE_A_SPECIFIC_BLOCK",
    DELETE_A_SPECIFIC_INVALID_EMAIL: "SENDGRID_DELETE_A_SPECIFIC_INVALID_EMAIL",
    DELETE_A_SPECIFIC_SPAM_REPORT: "SENDGRID_DELETE_A_SPECIFIC_SPAM_REPORT",
    DELETE_A_SUBUSER: "SENDGRID_DELETE_A_SUBUSER",
    DELETE_A_SUPPRESSION_FROM_A_SUPPRESSION_GROUP: "SENDGRID_DELETE_A_SUPPRESSION_FROM_A_SUPPRESSION_GROUP",
    DELETE_A_SUPPRESSION_GROUP: "SENDGRID_DELETE_A_SUPPRESSION_GROUP",
    DELETE_A_TEMPLATE: "SENDGRID_DELETE_A_TEMPLATE",
    DELETE_A_TRANSACTIONAL_TEMPLATE_VERSION: "SENDGRID_DELETE_A_TRANSACTIONAL_TEMPLATE_VERSION",
    DELETE_BLOCKS: "SENDGRID_DELETE_BLOCKS",
    DELETE_BOUNCES: "SENDGRID_DELETE_BOUNCES",
    DELETE_BULK_INTEGRATION: "SENDGRID_DELETE_BULK_INTEGRATION",
    DELETE_CONTACTS: "SENDGRID_DELETE_CONTACTS",
    DELETE_CUSTOM_FIELD_DEFINITION: "SENDGRID_DELETE_CUSTOM_FIELD_DEFINITION",
    DELETE_DESIGN: "SENDGRID_DELETE_DESIGN",
    DELETE_INVALID_EMAILS: "SENDGRID_DELETE_INVALID_EMAILS",
    DELETE_IP_POOL: "SENDGRID_DELETE_IP_POOL",
    DELETE_PENDING_TEAMMATE: "SENDGRID_DELETE_PENDING_TEAMMATE",
    DELETE_SEGMENT: "SENDGRID_DELETE_SEGMENT",
    DELETE_SINGLE_SEND_BY_ID: "SENDGRID_DELETE_SINGLE_SEND_BY_ID",
    DELETE_SINGLE_SEND_SCHEDULE: "SENDGRID_DELETE_SINGLE_SEND_SCHEDULE",
    DELETE_SPAM_REPORTS: "SENDGRID_DELETE_SPAM_REPORTS",
    DELETE_TEAMMATE: "SENDGRID_DELETE_TEAMMATE",
    DELETE_VERIFIED_SENDER: "SENDGRID_DELETE_VERIFIED_SENDER",
    DENY_ACCESS_REQUEST: "SENDGRID_DENY_ACCESS_REQUEST",
    DISASSOCIATE_A_BRANDED_LINK_FROM_A_SUBUSER: "SENDGRID_DISASSOCIATE_A_BRANDED_LINK_FROM_A_SUBUSER",
    DOMAIN_WARN_LIST: "SENDGRID_DOMAIN_WARN_LIST",
    DOWNLOAD_CSV: "SENDGRID_DOWNLOAD_CSV",
    DUPLICATE_A_TRANSACTIONAL_TEMPLATE: "SENDGRID_DUPLICATE_A_TRANSACTIONAL_TEMPLATE",
    DUPLICATE_DESIGN: "SENDGRID_DUPLICATE_DESIGN",
    DUPLICATE_SEND_GRID_PRE_BUILT_DESIGN: "SENDGRID_DUPLICATE_SEND_GRID_PRE_BUILT_DESIGN",
    DUPLICATE_SINGLE_SEND: "SENDGRID_DUPLICATE_SINGLE_SEND",
    EDIT_AN_SSO_TEAMMATE: "SENDGRID_EDIT_AN_SSO_TEAMMATE",
    EDIT_A_TRANSACTIONAL_TEMPLATE: "SENDGRID_EDIT_A_TRANSACTIONAL_TEMPLATE",
    EDIT_A_TRANSACTIONAL_TEMPLATE_VERSION: "SENDGRID_EDIT_A_TRANSACTIONAL_TEMPLATE_VERSION",
    EDIT_VERIFIED_SENDER: "SENDGRID_EDIT_VERIFIED_SENDER",
    EMAIL_DNS_RECORDS_TO_A_CO_WORKER: "SENDGRID_EMAIL_DNS_RECORDS_TO_A_CO_WORKER",
    ENABLE_DISABLE_A_SUBUSER: "SENDGRID_ENABLE_DISABLE_A_SUBUSER",
    ENABLE_DISABLE_WEBSITE_ACCESS_FOR_A_SUBUSER: "SENDGRID_ENABLE_DISABLE_WEBSITE_ACCESS_FOR_A_SUBUSER",
    ERASE_RECIPIENTS_EMAIL_DATA: "SENDGRID_ERASE_RECIPIENTS_EMAIL_DATA",
    EXPORT_CONTACTS: "SENDGRID_EXPORT_CONTACTS",
    EXPORT_CONTACTS_STATUS: "SENDGRID_EXPORT_CONTACTS_STATUS",
    EXPORT_SINGLE_SEND_STATS: "SENDGRID_EXPORT_SINGLE_SEND_STATS",
    FILTER_ALL_MESSAGES: "SENDGRID_FILTER_ALL_MESSAGES",
    FILTER_MESSAGES_BY_MESSAGE_ID: "SENDGRID_FILTER_MESSAGES_BY_MESSAGE_ID",
    FILTER_SEGMENTS_BY_PARENT_IDS: "SENDGRID_FILTER_SEGMENTS_BY_PARENT_IDS",
    GET_ACCOUNT_OFFERINGS: "SENDGRID_GET_ACCOUNT_OFFERINGS",
    GET_ALL_ACCOUNTS: "SENDGRID_GET_ALL_ACCOUNTS",
    GET_ALL_AUTOMATION_STATS: "SENDGRID_GET_ALL_AUTOMATION_STATS",
    GET_ALL_AVAILABLE_OFFERINGS: "SENDGRID_GET_ALL_AVAILABLE_OFFERINGS",
    GET_ALL_CATEGORIES: "SENDGRID_GET_ALL_CATEGORIES",
    GET_ALL_EXISTING_EXPORTS: "SENDGRID_GET_ALL_EXISTING_EXPORTS",
    GET_ALL_FIELD_DEFINITIONS: "SENDGRID_GET_ALL_FIELD_DEFINITIONS",
    GET_ALL_IP_POOLS_THAT_HAVE_ASSOCIATED_IPS: "SENDGRID_GET_ALL_IP_POOLS_THAT_HAVE_ASSOCIATED_IPS",
    GET_ALL_LISTS: "SENDGRID_GET_ALL_LISTS",
    GET_ALL_SENDER_IDENTITIES: "SENDGRID_GET_ALL_SENDER_IDENTITIES",
    GET_ALL_SINGLE_SENDS: "SENDGRID_GET_ALL_SINGLE_SENDS",
    GET_ALL_SINGLE_SENDS_STATS: "SENDGRID_GET_ALL_SINGLE_SENDS_STATS",
    GET_ALL_SSO_CERTIFICATES_BY_INTEGRATION: "SENDGRID_GET_ALL_SSO_CERTIFICATES_BY_INTEGRATION",
    GET_ALL_SSO_INTEGRATIONS: "SENDGRID_GET_ALL_SSO_INTEGRATIONS",
    GET_ALL_VERIFIED_SENDERS: "SENDGRID_GET_ALL_VERIFIED_SENDERS",
    GET_AN_ACCOUNT_S_STATE: "SENDGRID_GET_AN_ACCOUNT_S_STATE",
    GET_AN_SSO_CERTIFICATE: "SENDGRID_GET_AN_SSO_CERTIFICATE",
    GET_AN_SSO_INTEGRATION: "SENDGRID_GET_AN_SSO_INTEGRATION",
    GET_AUTOMATION_CLICK_TRACKING_STATS_BY_ID: "SENDGRID_GET_AUTOMATION_CLICK_TRACKING_STATS_BY_ID",
    GET_AUTOMATION_STATS_BY_ID: "SENDGRID_GET_AUTOMATION_STATS_BY_ID",
    GET_A_CONTACT_BY_ID: "SENDGRID_GET_A_CONTACT_BY_ID",
    GET_A_LIST_BY_ID: "SENDGRID_GET_A_LIST_BY_ID",
    GET_A_LIST_OF_ALL_IP_ADDRESSES_ON_YOUR_ACCOUNT: "SENDGRID_GET_A_LIST_OF_ALL_IP_ADDRESSES_ON_YOUR_ACCOUNT",
    GET_A_LIST_OF_ALL_SENDERS: "SENDGRID_GET_A_LIST_OF_ALL_SENDERS",
    GET_A_LIST_OF_SUBUSERS_ASSIGNED_TO_AN_IP: "SENDGRID_GET_A_LIST_OF_SUBUSERS_ASSIGNED_TO_AN_IP",
    GET_A_SPECIFIC_SENDER: "SENDGRID_GET_A_SPECIFIC_SENDER",
    GET_A_USER_S_ACCOUNT_INFORMATION: "SENDGRID_GET_A_USER_S_ACCOUNT_INFORMATION",
    GET_A_USER_S_PROFILE: "SENDGRID_GET_A_USER_S_PROFILE",
    GET_BATCHED_CONTACTS_BY_IDS: "SENDGRID_GET_BATCHED_CONTACTS_BY_IDS",
    GET_BOUNCE_CLASS_STATS: "SENDGRID_GET_BOUNCE_CLASS_STATS",
    GET_BULK_VALIDATION_JOB: "SENDGRID_GET_BULK_VALIDATION_JOB",
    GET_CONTACTS_BY_EMAILS: "SENDGRID_GET_CONTACTS_BY_EMAILS",
    GET_CONTACTS_BY_IDENTIFIERS: "SENDGRID_GET_CONTACTS_BY_IDENTIFIERS",
    GET_DESIGN: "SENDGRID_GET_DESIGN",
    GET_DETAILS_FOR_AN_IP_ADDRESS: "SENDGRID_GET_DETAILS_FOR_AN_IP_ADDRESS",
    GET_DETAILS_FOR_AN_IP_POOL: "SENDGRID_GET_DETAILS_FOR_AN_IP_POOL",
    GET_EMAIL_SUPPRESSION_GROUPS: "SENDGRID_GET_EMAIL_SUPPRESSION_GROUPS",
    GET_ENGAGEMENT_QUALITY_SCORES: "SENDGRID_GET_ENGAGEMENT_QUALITY_SCORES",
    GET_GEO_STATS: "SENDGRID_GET_GEO_STATS",
    GET_INFORMATION_ON_A_SINGLE_SUPPRESSION_GROUP: "SENDGRID_GET_INFORMATION_ON_A_SINGLE_SUPPRESSION_GROUP",
    GET_INTEGRATION: "SENDGRID_GET_INTEGRATION",
    GET_IPS_ASSIGNED_TO_AN_IP_POOL: "SENDGRID_GET_IPS_ASSIGNED_TO_AN_IP_POOL",
    GET_IP_WARMUP_STATUS: "SENDGRID_GET_IP_WARMUP_STATUS",
    GET_LIST_CONTACT_COUNT: "SENDGRID_GET_LIST_CONTACT_COUNT",
    GET_LIST_OF_SEGMENTS: "SENDGRID_GET_LIST_OF_SEGMENTS",
    GET_OPEN_TRACKING_SETTINGS: "SENDGRID_GET_OPEN_TRACKING_SETTINGS",
    GET_RECIPIENT_UPLOAD_STATUS: "SENDGRID_GET_RECIPIENT_UPLOAD_STATUS",
    GET_REMAINING_IPS_COUNT: "SENDGRID_GET_REMAINING_IPS_COUNT",
    GET_SAMPLE_CONTACTS: "SENDGRID_GET_SAMPLE_CONTACTS",
    GET_SEGMENT_BY_ID: "SENDGRID_GET_SEGMENT_BY_ID",
    GET_SEND_GRID_PRE_BUILT_DESIGN: "SENDGRID_GET_SEND_GRID_PRE_BUILT_DESIGN",
    GET_SIGNED_EVENT_WEBHOOK_S_PUBLIC_KEY: "SENDGRID_GET_SIGNED_EVENT_WEBHOOK_S_PUBLIC_KEY",
    GET_SINGLE_SENDS_SEARCH: "SENDGRID_GET_SINGLE_SENDS_SEARCH",
    GET_SINGLE_SEND_BY_ID: "SENDGRID_GET_SINGLE_SEND_BY_ID",
    GET_SINGLE_SEND_CLICK_TRACKING_STATS_BY_ID: "SENDGRID_GET_SINGLE_SEND_CLICK_TRACKING_STATS_BY_ID",
    GET_SINGLE_SEND_STATS_BY_ID: "SENDGRID_GET_SINGLE_SEND_STATS_BY_ID",
    GET_SUBUSERS_ENGAGEMENT_QUALITY_SCORES: "SENDGRID_GET_SUBUSERS_ENGAGEMENT_QUALITY_SCORES",
    GET_SUBUSER_MONTHLY_STATS: "SENDGRID_GET_SUBUSER_MONTHLY_STATS",
    GET_SUPPRESSION_GROUPS: "SENDGRID_GET_SUPPRESSION_GROUPS",
    GET_TEAMMATE_SUBUSER_ACCESS: "SENDGRID_GET_TEAMMATE_SUBUSER_ACCESS",
    GET_THE_CREDITS_FOR_A_SUBUSER: "SENDGRID_GET_THE_CREDITS_FOR_A_SUBUSER",
    GET_THE_DEFAULT_AUTHENTICATION: "SENDGRID_GET_THE_DEFAULT_AUTHENTICATION",
    GET_THE_SETTINGS_FOR_A_SINGLE_EVENT_WEBHOOK: "SENDGRID_GET_THE_SETTINGS_FOR_A_SINGLE_EVENT_WEBHOOK",
    GET_TOTAL_CONTACT_COUNT: "SENDGRID_GET_TOTAL_CONTACT_COUNT",
    GET_USER_SCOPES: "SENDGRID_GET_USER_SCOPES",
    IMPORT_CONTACTS: "SENDGRID_IMPORT_CONTACTS",
    IMPORT_CONTACTS_STATUS: "SENDGRID_IMPORT_CONTACTS_STATUS",
    INVITE_TEAMMATE: "SENDGRID_INVITE_TEAMMATE",
    LIST_ALL_AUTHENTICATED_DOMAINS: "SENDGRID_LIST_ALL_AUTHENTICATED_DOMAINS",
    LIST_ALL_SUBUSERS: "SENDGRID_LIST_ALL_SUBUSERS",
    LIST_API_KEYS: "SENDGRID_LIST_API_KEYS",
    LIST_BULK_EMAIL_VALIDATION_JOBS: "SENDGRID_LIST_BULK_EMAIL_VALIDATION_JOBS",
    LIST_DESIGNS: "SENDGRID_LIST_DESIGNS",
    LIST_INTEGRATION: "SENDGRID_LIST_INTEGRATION",
    LIST_SEND_GRID_PRE_BUILT_DESIGNS: "SENDGRID_LIST_SEND_GRID_PRE_BUILT_DESIGNS",
    LIST_USER_AUTH_DOMAINS: "SENDGRID_LIST_USER_AUTH_DOMAINS",
    MANUALLY_REFRESH_A_SEGMENT: "SENDGRID_MANUALLY_REFRESH_A_SEGMENT",
    PRESIGNED_URL_HEADERS_FOR_BULK_EMAIL_VALIDATION: "SENDGRID_PRESIGNED_URL_HEADERS_FOR_BULK_EMAIL_VALIDATION",
    REMOVE_AN_IP_ADDRESS_FROM_A_POOL: "SENDGRID_REMOVE_AN_IP_ADDRESS_FROM_A_POOL",
    REMOVE_AN_IP_FROM_AN_AUTHENTICATED_DOMAIN: "SENDGRID_REMOVE_AN_IP_FROM_AN_AUTHENTICATED_DOMAIN",
    REMOVE_A_SPECIFIC_IP_FROM_THE_ALLOWED_LIST: "SENDGRID_REMOVE_A_SPECIFIC_IP_FROM_THE_ALLOWED_LIST",
    REMOVE_CONTACTS_FROM_A_LIST: "SENDGRID_REMOVE_CONTACTS_FROM_A_LIST",
    REMOVE_LIST_AND_OPTIONAL_CONTACTS: "SENDGRID_REMOVE_LIST_AND_OPTIONAL_CONTACTS",
    REMOVE_ONE_OR_MORE_IPS_FROM_THE_ALLOW_LIST: "SENDGRID_REMOVE_ONE_OR_MORE_IPS_FROM_THE_ALLOW_LIST",
    REMOVE_SEGMENT_WITHOUT_AFFECTING_CONTACTS: "SENDGRID_REMOVE_SEGMENT_WITHOUT_AFFECTING_CONTACTS",
    REMOVE_USER_DOMAIN: "SENDGRID_REMOVE_USER_DOMAIN",
    RENAME_AN_IP_POOL: "SENDGRID_RENAME_AN_IP_POOL",
    REQUEST_CSV: "SENDGRID_REQUEST_CSV",
    RESEND_A_SENDER_VERIFICATION: "SENDGRID_RESEND_A_SENDER_VERIFICATION",
    RESEND_SENDER_IDENTITY_VERIFICATION: "SENDGRID_RESEND_SENDER_IDENTITY_VERIFICATION",
    RESEND_TEAMMATE_INVITE: "SENDGRID_RESEND_TEAMMATE_INVITE",
    RESEND_VERIFIED_SENDER_REQUEST: "SENDGRID_RESEND_VERIFIED_SENDER_REQUEST",
    RESUME_SCHEDULED_SEND: "SENDGRID_RESUME_SCHEDULED_SEND",
    RETRIEVES_INBOUND_PARSE_WEBHOOK_STATISTICS: "SENDGRID_RETRIEVES_INBOUND_PARSE_WEBHOOK_STATISTICS",
    RETRIEVE_ACCESS_REQUESTS: "SENDGRID_RETRIEVE_ACCESS_REQUESTS",
    RETRIEVE_ADDRESS_WHITELIST_MAIL_SETTINGS: "SENDGRID_RETRIEVE_ADDRESS_WHITELIST_MAIL_SETTINGS",
    RETRIEVE_ALL_ALERTS: "SENDGRID_RETRIEVE_ALL_ALERTS",
    RETRIEVE_ALL_ASSIGNED_IPS: "SENDGRID_RETRIEVE_ALL_ASSIGNED_IPS",
    RETRIEVE_ALL_BLOCKS: "SENDGRID_RETRIEVE_ALL_BLOCKS",
    RETRIEVE_ALL_BOUNCES: "SENDGRID_RETRIEVE_ALL_BOUNCES",
    RETRIEVE_ALL_BRANDED_LINKS: "SENDGRID_RETRIEVE_ALL_BRANDED_LINKS",
    RETRIEVE_ALL_CAMPAIGNS: "SENDGRID_RETRIEVE_ALL_CAMPAIGNS",
    RETRIEVE_ALL_CATEGORIES: "SENDGRID_RETRIEVE_ALL_CATEGORIES",
    RETRIEVE_ALL_CUSTOM_FIELDS: "SENDGRID_RETRIEVE_ALL_CUSTOM_FIELDS",
    RETRIEVE_ALL_GLOBAL_SUPPRESSIONS: "SENDGRID_RETRIEVE_ALL_GLOBAL_SUPPRESSIONS",
    RETRIEVE_ALL_INVALID_EMAILS: "SENDGRID_RETRIEVE_ALL_INVALID_EMAILS",
    RETRIEVE_ALL_IPS_CURRENTLY_IN_WARMUP: "SENDGRID_RETRIEVE_ALL_IPS_CURRENTLY_IN_WARMUP",
    RETRIEVE_ALL_IP_ADDRESSES: "SENDGRID_RETRIEVE_ALL_IP_ADDRESSES",
    RETRIEVE_ALL_IP_POOLS: "SENDGRID_RETRIEVE_ALL_IP_POOLS",
    RETRIEVE_ALL_IP_POOLS_AN_IP_ADDRESS_BELONGS_TO: "SENDGRID_RETRIEVE_ALL_IP_POOLS_AN_IP_ADDRESS_BELONGS_TO",
    RETRIEVE_ALL_LISTS: "SENDGRID_RETRIEVE_ALL_LISTS",
    RETRIEVE_ALL_MAIL_SETTINGS: "SENDGRID_RETRIEVE_ALL_MAIL_SETTINGS",
    RETRIEVE_ALL_OF_YOUR_EVENT_WEBHOOKS: "SENDGRID_RETRIEVE_ALL_OF_YOUR_EVENT_WEBHOOKS",
    RETRIEVE_ALL_PARSE_SETTINGS: "SENDGRID_RETRIEVE_ALL_PARSE_SETTINGS",
    RETRIEVE_ALL_PENDING_TEAMMATES: "SENDGRID_RETRIEVE_ALL_PENDING_TEAMMATES",
    RETRIEVE_ALL_RECENT_ACCESS_ATTEMPTS: "SENDGRID_RETRIEVE_ALL_RECENT_ACCESS_ATTEMPTS",
    RETRIEVE_ALL_RECIPIENTS_ON_A_LIST: "SENDGRID_RETRIEVE_ALL_RECIPIENTS_ON_A_LIST",
    RETRIEVE_ALL_REVERSE_DNS_RECORDS: "SENDGRID_RETRIEVE_ALL_REVERSE_DNS_RECORDS",
    RETRIEVE_ALL_SCHEDULED_SENDS: "SENDGRID_RETRIEVE_ALL_SCHEDULED_SENDS",
    RETRIEVE_ALL_SEGMENTS: "SENDGRID_RETRIEVE_ALL_SEGMENTS",
    RETRIEVE_ALL_SPAM_REPORTS: "SENDGRID_RETRIEVE_ALL_SPAM_REPORTS",
    RETRIEVE_ALL_SUPPRESSIONS: "SENDGRID_RETRIEVE_ALL_SUPPRESSIONS",
    RETRIEVE_ALL_SUPPRESSIONS_FOR_A_SUPPRESSION_GROUP: "SENDGRID_RETRIEVE_ALL_SUPPRESSIONS_FOR_A_SUPPRESSION_GROUP",
    RETRIEVE_ALL_TEAMMATES: "SENDGRID_RETRIEVE_ALL_TEAMMATES",
    RETRIEVE_ALL_THE_IPS_IN_A_SPECIFIED_POOL: "SENDGRID_RETRIEVE_ALL_THE_IPS_IN_A_SPECIFIED_POOL",
    RETRIEVE_AN_AUTHENTICATED_DOMAIN: "SENDGRID_RETRIEVE_AN_AUTHENTICATED_DOMAIN",
    RETRIEVE_AN_EXISTING_API_KEY: "SENDGRID_RETRIEVE_AN_EXISTING_API_KEY",
    RETRIEVE_A_BOUNCE: "SENDGRID_RETRIEVE_A_BOUNCE",
    RETRIEVE_A_BRANDED_LINK: "SENDGRID_RETRIEVE_A_BRANDED_LINK",
    RETRIEVE_A_COUNT_OF_RECIPIENTS: "SENDGRID_RETRIEVE_A_COUNT_OF_RECIPIENTS",
    RETRIEVE_A_CUSTOM_FIELD: "SENDGRID_RETRIEVE_A_CUSTOM_FIELD",
    RETRIEVE_A_GLOBAL_SUPPRESSION: "SENDGRID_RETRIEVE_A_GLOBAL_SUPPRESSION",
    RETRIEVE_A_LIST_OF_CURRENTLY_ALLOWED_IPS: "SENDGRID_RETRIEVE_A_LIST_OF_CURRENTLY_ALLOWED_IPS",
    RETRIEVE_A_REVERSE_DNS_RECORD: "SENDGRID_RETRIEVE_A_REVERSE_DNS_RECORD",
    RETRIEVE_A_SEGMENT: "SENDGRID_RETRIEVE_A_SEGMENT",
    RETRIEVE_A_SINGLE_CAMPAIGN: "SENDGRID_RETRIEVE_A_SINGLE_CAMPAIGN",
    RETRIEVE_A_SINGLE_LIST: "SENDGRID_RETRIEVE_A_SINGLE_LIST",
    RETRIEVE_A_SINGLE_RECIPIENT: "SENDGRID_RETRIEVE_A_SINGLE_RECIPIENT",
    RETRIEVE_A_SINGLE_TRANSACTIONAL_TEMPLATE: "SENDGRID_RETRIEVE_A_SINGLE_TRANSACTIONAL_TEMPLATE",
    RETRIEVE_A_SPECIFIC_ALERT: "SENDGRID_RETRIEVE_A_SPECIFIC_ALERT",
    RETRIEVE_A_SPECIFIC_ALLOWED_IP: "SENDGRID_RETRIEVE_A_SPECIFIC_ALLOWED_IP",
    RETRIEVE_A_SPECIFIC_BLOCK: "SENDGRID_RETRIEVE_A_SPECIFIC_BLOCK",
    RETRIEVE_A_SPECIFIC_INVALID_EMAIL: "SENDGRID_RETRIEVE_A_SPECIFIC_INVALID_EMAIL",
    RETRIEVE_A_SPECIFIC_PARSE_SETTING: "SENDGRID_RETRIEVE_A_SPECIFIC_PARSE_SETTING",
    RETRIEVE_A_SPECIFIC_SPAM_REPORT: "SENDGRID_RETRIEVE_A_SPECIFIC_SPAM_REPORT",
    RETRIEVE_A_SPECIFIC_TRANSACTIONAL_TEMPLATE_VERSION: "SENDGRID_RETRIEVE_A_SPECIFIC_TRANSACTIONAL_TEMPLATE_VERSION",
    RETRIEVE_A_SUBUSER_S_BRANDED_LINK: "SENDGRID_RETRIEVE_A_SUBUSER_S_BRANDED_LINK",
    RETRIEVE_BOUNCE_CLASSIFICATION_TOTALS: "SENDGRID_RETRIEVE_BOUNCE_CLASSIFICATION_TOTALS",
    RETRIEVE_BOUNCE_PURGE_MAIL_SETTINGS: "SENDGRID_RETRIEVE_BOUNCE_PURGE_MAIL_SETTINGS",
    RETRIEVE_CLICK_TRACK_SETTINGS: "SENDGRID_RETRIEVE_CLICK_TRACK_SETTINGS",
    RETRIEVE_CURRENT_ENFORCED_TLS_SETTINGS: "SENDGRID_RETRIEVE_CURRENT_ENFORCED_TLS_SETTINGS",
    RETRIEVE_EMAIL_STATISTICS_BY_BROWSER: "SENDGRID_RETRIEVE_EMAIL_STATISTICS_BY_BROWSER",
    RETRIEVE_EMAIL_STATISTICS_BY_CLIENT_TYPE: "SENDGRID_RETRIEVE_EMAIL_STATISTICS_BY_CLIENT_TYPE",
    RETRIEVE_EMAIL_STATISTICS_BY_DEVICE_TYPE: "SENDGRID_RETRIEVE_EMAIL_STATISTICS_BY_DEVICE_TYPE",
    RETRIEVE_EMAIL_STATISTICS_BY_MAILBOX_PROVIDER: "SENDGRID_RETRIEVE_EMAIL_STATISTICS_BY_MAILBOX_PROVIDER",
    RETRIEVE_EMAIL_STATISTICS_FOR_CATEGORIES: "SENDGRID_RETRIEVE_EMAIL_STATISTICS_FOR_CATEGORIES",
    RETRIEVE_EMAIL_STATISTICS_FOR_YOUR_SUBUSERS: "SENDGRID_RETRIEVE_EMAIL_STATISTICS_FOR_YOUR_SUBUSERS",
    RETRIEVE_FOOTER_MAIL_SETTINGS: "SENDGRID_RETRIEVE_FOOTER_MAIL_SETTINGS",
    RETRIEVE_FORWARD_BOUNCE_MAIL_SETTINGS: "SENDGRID_RETRIEVE_FORWARD_BOUNCE_MAIL_SETTINGS",
    RETRIEVE_FORWARD_SPAM_MAIL_SETTINGS: "SENDGRID_RETRIEVE_FORWARD_SPAM_MAIL_SETTINGS",
    RETRIEVE_GLOBAL_EMAIL_STATISTICS: "SENDGRID_RETRIEVE_GLOBAL_EMAIL_STATISTICS",
    RETRIEVE_GOOGLE_ANALYTICS_SETTINGS: "SENDGRID_RETRIEVE_GOOGLE_ANALYTICS_SETTINGS",
    RETRIEVE_LEGACY_TEMPLATE_MAIL_SETTINGS: "SENDGRID_RETRIEVE_LEGACY_TEMPLATE_MAIL_SETTINGS",
    RETRIEVE_MONTHLY_STATS_FOR_ALL_SUBUSERS: "SENDGRID_RETRIEVE_MONTHLY_STATS_FOR_ALL_SUBUSERS",
    RETRIEVE_PAGED_TRANSACTIONAL_TEMPLATES: "SENDGRID_RETRIEVE_PAGED_TRANSACTIONAL_TEMPLATES",
    RETRIEVE_RECIPIENTS: "SENDGRID_RETRIEVE_RECIPIENTS",
    RETRIEVE_RECIPIENTS_ON_A_SEGMENT: "SENDGRID_RETRIEVE_RECIPIENTS_ON_A_SEGMENT",
    RETRIEVE_RESERVED_FIELDS: "SENDGRID_RETRIEVE_RESERVED_FIELDS",
    RETRIEVE_SCHEDULED_SEND: "SENDGRID_RETRIEVE_SCHEDULED_SEND",
    RETRIEVE_SINGLE_SEGMENT_ENDPOINT: "SENDGRID_RETRIEVE_SINGLE_SEGMENT_ENDPOINT",
    RETRIEVE_SPECIFIC_TEAMMATE: "SENDGRID_RETRIEVE_SPECIFIC_TEAMMATE",
    RETRIEVE_STATS_BY_A_SPECIFIC_CLIENT_TYPE: "SENDGRID_RETRIEVE_STATS_BY_A_SPECIFIC_CLIENT_TYPE",
    RETRIEVE_SUBSCRIPTION_TRACKING_SETTINGS: "SENDGRID_RETRIEVE_SUBSCRIPTION_TRACKING_SETTINGS",
    RETRIEVE_SUBUSER_REPUTATIONS: "SENDGRID_RETRIEVE_SUBUSER_REPUTATIONS",
    RETRIEVE_SUMS_OF_EMAIL_STATS_FOR_EACH_CATEGORY: "SENDGRID_RETRIEVE_SUMS_OF_EMAIL_STATS_FOR_EACH_CATEGORY",
    RETRIEVE_THE_COUNT_OF_BILLABLE_RECIPIENTS: "SENDGRID_RETRIEVE_THE_COUNT_OF_BILLABLE_RECIPIENTS",
    RETRIEVE_THE_DEFAULT_BRANDED_LINK: "SENDGRID_RETRIEVE_THE_DEFAULT_BRANDED_LINK",
    RETRIEVE_THE_LISTS_THAT_A_RECIPIENT_IS_ON: "SENDGRID_RETRIEVE_THE_LISTS_THAT_A_RECIPIENT_IS_ON",
    RETRIEVE_TRACKING_SETTINGS: "SENDGRID_RETRIEVE_TRACKING_SETTINGS",
    RETRIEVE_USER_S_AUTHENTICATED_DOMAINS: "SENDGRID_RETRIEVE_USER_S_AUTHENTICATED_DOMAINS",
    RETRIEVE_YOUR_ACCOUNT_EMAIL_ADDRESS: "SENDGRID_RETRIEVE_YOUR_ACCOUNT_EMAIL_ADDRESS",
    RETRIEVE_YOUR_CREDIT_BALANCE: "SENDGRID_RETRIEVE_YOUR_CREDIT_BALANCE",
    RETRIEVE_YOUR_USERNAME: "SENDGRID_RETRIEVE_YOUR_USERNAME",
    RETURNS_A_LIST_OF_ALL_PARTNER_SETTINGS: "SENDGRID_RETURNS_A_LIST_OF_ALL_PARTNER_SETTINGS",
    SCHEDULE_A_CAMPAIGN: "SENDGRID_SCHEDULE_A_CAMPAIGN",
    SCHEDULE_SINGLE_SEND: "SENDGRID_SCHEDULE_SINGLE_SEND",
    SEARCH_CONTACTS: "SENDGRID_SEARCH_CONTACTS",
    SEARCH_FOR_SUPPRESSIONS_WITHIN_A_GROUP: "SENDGRID_SEARCH_FOR_SUPPRESSIONS_WITHIN_A_GROUP",
    SEARCH_RECIPIENTS: "SENDGRID_SEARCH_RECIPIENTS",
    SEGMENT_JSON_DATA_BY_SPECIFIC_CRITERIA: "SENDGRID_SEGMENT_JSON_DATA_BY_SPECIFIC_CRITERIA",
    SEND_A_CAMPAIGN: "SENDGRID_SEND_A_CAMPAIGN",
    SEND_A_TEST_CAMPAIGN: "SENDGRID_SEND_A_TEST_CAMPAIGN",
    SEND_A_TEST_MARKETING_EMAIL: "SENDGRID_SEND_A_TEST_MARKETING_EMAIL",
    SEND_EMAIL_WITH_TWILIO_SEND_GRID: "SENDGRID_SEND_EMAIL_WITH_TWILIO_SEND_GRID",
    SET_UP_REVERSE_DNS: "SENDGRID_SET_UP_REVERSE_DNS",
    SINGLE_SEND_CSV_STATS_RETRIEVAL: "SENDGRID_SINGLE_SEND_CSV_STATS_RETRIEVAL",
    START_WARMING_UP_AN_IP_ADDRESS: "SENDGRID_START_WARMING_UP_AN_IP_ADDRESS",
    STOP_WARMING_UP_AN_IP_ADDRESS: "SENDGRID_STOP_WARMING_UP_AN_IP_ADDRESS",
    SUM_EMAIL_STATS_FOR_SUBUSERS: "SENDGRID_SUM_EMAIL_STATS_FOR_SUBUSERS",
    TEST_AN_EVENT_WEBHOOK_S_SETTINGS: "SENDGRID_TEST_AN_EVENT_WEBHOOK_S_SETTINGS",
    TOGGLE_WEBHOOK_SIGNATURE: "SENDGRID_TOGGLE_WEBHOOK_SIGNATURE",
    UNLINK_SUBUSER_DOMAIN: "SENDGRID_UNLINK_SUBUSER_DOMAIN",
    UNSCHEDULE_A_SCHEDULED_CAMPAIGN: "SENDGRID_UNSCHEDULE_A_SCHEDULED_CAMPAIGN",
    UPDATE_ACCOUNT_OFFERINGS: "SENDGRID_UPDATE_ACCOUNT_OFFERINGS",
    UPDATE_ADDRESS_WHITELIST_MAIL_SETTINGS: "SENDGRID_UPDATE_ADDRESS_WHITELIST_MAIL_SETTINGS",
    UPDATE_AN_ACCOUNT_S_STATE: "SENDGRID_UPDATE_AN_ACCOUNT_S_STATE",
    UPDATE_AN_ALERT: "SENDGRID_UPDATE_AN_ALERT",
    UPDATE_AN_AUTHENTICATED_DOMAIN: "SENDGRID_UPDATE_AN_AUTHENTICATED_DOMAIN",
    UPDATE_AN_IP_POOL_NAME: "SENDGRID_UPDATE_AN_IP_POOL_NAME",
    UPDATE_AN_SSO_INTEGRATION: "SENDGRID_UPDATE_AN_SSO_INTEGRATION",
    UPDATE_API_KEY_NAME: "SENDGRID_UPDATE_API_KEY_NAME",
    UPDATE_API_KEY_NAME_AND_SCOPES: "SENDGRID_UPDATE_API_KEY_NAME_AND_SCOPES",
    UPDATE_A_BRANDED_LINK: "SENDGRID_UPDATE_A_BRANDED_LINK",
    UPDATE_A_CAMPAIGN: "SENDGRID_UPDATE_A_CAMPAIGN",
    UPDATE_A_LIST: "SENDGRID_UPDATE_A_LIST",
    UPDATE_A_PARSE_SETTING: "SENDGRID_UPDATE_A_PARSE_SETTING",
    UPDATE_A_SCHEDULED_CAMPAIGN: "SENDGRID_UPDATE_A_SCHEDULED_CAMPAIGN",
    UPDATE_A_SCHEDULED_SEND: "SENDGRID_UPDATE_A_SCHEDULED_SEND",
    UPDATE_A_SEGMENT: "SENDGRID_UPDATE_A_SEGMENT",
    UPDATE_A_SENDER: "SENDGRID_UPDATE_A_SENDER",
    UPDATE_A_SENDER_IDENTITY: "SENDGRID_UPDATE_A_SENDER_IDENTITY",
    UPDATE_A_SINGLE_EVENT_WEBHOOK_BY_ID: "SENDGRID_UPDATE_A_SINGLE_EVENT_WEBHOOK_BY_ID",
    UPDATE_A_SUPPRESSION_GROUP: "SENDGRID_UPDATE_A_SUPPRESSION_GROUP",
    UPDATE_A_USER_S_PROFILE: "SENDGRID_UPDATE_A_USER_S_PROFILE",
    UPDATE_BOUNCE_PURGE_MAIL_SETTINGS: "SENDGRID_UPDATE_BOUNCE_PURGE_MAIL_SETTINGS",
    UPDATE_CLICK_TRACKING_SETTINGS: "SENDGRID_UPDATE_CLICK_TRACKING_SETTINGS",
    UPDATE_CUSTOM_FIELD_DEFINITION: "SENDGRID_UPDATE_CUSTOM_FIELD_DEFINITION",
    UPDATE_DESIGN: "SENDGRID_UPDATE_DESIGN",
    UPDATE_DETAILS_FOR_AN_IP_ADDRESS: "SENDGRID_UPDATE_DETAILS_FOR_AN_IP_ADDRESS",
    UPDATE_ENFORCED_TLS_SETTINGS: "SENDGRID_UPDATE_ENFORCED_TLS_SETTINGS",
    UPDATE_FOOTER_MAIL_SETTINGS: "SENDGRID_UPDATE_FOOTER_MAIL_SETTINGS",
    UPDATE_FORWARD_BOUNCE_MAIL_SETTINGS: "SENDGRID_UPDATE_FORWARD_BOUNCE_MAIL_SETTINGS",
    UPDATE_FORWARD_SPAM_MAIL_SETTINGS: "SENDGRID_UPDATE_FORWARD_SPAM_MAIL_SETTINGS",
    UPDATE_GOOGLE_ANALYTICS_SETTINGS: "SENDGRID_UPDATE_GOOGLE_ANALYTICS_SETTINGS",
    UPDATE_INTEGRATION: "SENDGRID_UPDATE_INTEGRATION",
    UPDATE_LIST: "SENDGRID_UPDATE_LIST",
    UPDATE_OPEN_TRACKING_SETTINGS: "SENDGRID_UPDATE_OPEN_TRACKING_SETTINGS",
    UPDATE_SEGMENT: "SENDGRID_UPDATE_SEGMENT",
    UPDATE_SINGLE_SEND: "SENDGRID_UPDATE_SINGLE_SEND",
    UPDATE_SSO_CERTIFICATE: "SENDGRID_UPDATE_SSO_CERTIFICATE",
    UPDATE_SUBSCRIPTION_TRACKING_SETTINGS: "SENDGRID_UPDATE_SUBSCRIPTION_TRACKING_SETTINGS",
    UPDATE_TEAMMATE_S_PERMISSIONS: "SENDGRID_UPDATE_TEAMMATE_S_PERMISSIONS",
    UPDATE_TEMPLATE_MAIL_SETTINGS: "SENDGRID_UPDATE_TEMPLATE_MAIL_SETTINGS",
    UPDATE_THE_CREDITS_FOR_A_SUBUSER: "SENDGRID_UPDATE_THE_CREDITS_FOR_A_SUBUSER",
    UPDATE_THE_REMAINING_CREDITS_FOR_A_SUBUSER: "SENDGRID_UPDATE_THE_REMAINING_CREDITS_FOR_A_SUBUSER",
    UPDATE_YOUR_ACCOUNT_EMAIL_ADDRESS: "SENDGRID_UPDATE_YOUR_ACCOUNT_EMAIL_ADDRESS",
    UPDATE_YOUR_PASSWORD: "SENDGRID_UPDATE_YOUR_PASSWORD",
    UPDATE_YOUR_USERNAME: "SENDGRID_UPDATE_YOUR_USERNAME",
    VALIDATE_AN_EMAIL: "SENDGRID_VALIDATE_AN_EMAIL",
    VALIDATE_A_BATCH_ID: "SENDGRID_VALIDATE_A_BATCH_ID",
    VALIDATE_A_BRANDED_LINK: "SENDGRID_VALIDATE_A_BRANDED_LINK",
    VALIDATE_A_DOMAIN_AUTHENTICATION: "SENDGRID_VALIDATE_A_DOMAIN_AUTHENTICATION",
    VALIDATE_A_REVERSE_DNS_RECORD: "SENDGRID_VALIDATE_A_REVERSE_DNS_RECORD",
    VERIFY_SENDER_REQUEST: "SENDGRID_VERIFY_SENDER_REQUEST",
    VIEW_A_SENDER_IDENTITY: "SENDGRID_VIEW_A_SENDER_IDENTITY",
    VIEW_SCHEDULED_TIME_OF_A_CAMPAIGN: "SENDGRID_VIEW_SCHEDULED_TIME_OF_A_CAMPAIGN",
  },
  triggerTypes: {},
}
