// @composio/core/generated/sentry.ts
/**
 * Map of Composio's SENTRY toolkit.
 */
export const SENTRY = {
  slug: "sentry",
  tools: {
    ACCESS_PROJECT_INFORMATION: "SENTRY_ACCESS_PROJECT_INFORMATION",
    ADD_ORGANIZATION_MEMBER_VIA_EMAIL: "SENTRY_ADD_ORGANIZATION_MEMBER_VIA_EMAIL",
    ADD_REMOVE_USER_EMAIL_BY_ID: "SENTRY_ADD_REMOVE_USER_EMAIL_BY_ID",
    ADD_TEAM_MEMBER_IN_ORGANIZATION: "SENTRY_ADD_TEAM_MEMBER_IN_ORGANIZATION",
    ADD_TEAM_TO_PROJECT: "SENTRY_ADD_TEAM_TO_PROJECT",
    CREATE_DASHBOARD_WITH_WIDGETS: "SENTRY_CREATE_DASHBOARD_WITH_WIDGETS",
    CREATE_EXTERNAL_USER_FOR_ORGANIZATION: "SENTRY_CREATE_EXTERNAL_USER_FOR_ORGANIZATION",
    CREATE_ORGANIZATION_ALERT_RULE: "SENTRY_CREATE_ORGANIZATION_ALERT_RULE",
    CREATE_ORGANIZATION_MONITOR: "SENTRY_CREATE_ORGANIZATION_MONITOR",
    CREATE_ORGANIZATION_TEAM: "SENTRY_CREATE_ORGANIZATION_TEAM",
    CREATE_PROJECT_KEY_WITH_OPTIONAL_RATE_LIMITING: "SENTRY_CREATE_PROJECT_KEY_WITH_OPTIONAL_RATE_LIMITING",
    CREATE_PROJECT_RULE_FOR_ALERTS: "SENTRY_CREATE_PROJECT_RULE_FOR_ALERTS",
    CREATE_PROJECT_WEBHOOK_SUBSCRIPTION: "SENTRY_CREATE_PROJECT_WEBHOOK_SUBSCRIPTION",
    CREATE_RELEASE_DEPLOY_FOR_ORG: "SENTRY_CREATE_RELEASE_DEPLOY_FOR_ORG",
    CREATE_RELEASE_FOR_ORGANIZATION: "SENTRY_CREATE_RELEASE_FOR_ORGANIZATION",
    CREATE_SCIM_GROUP_FOR_ORGANIZATION: "SENTRY_CREATE_SCIM_GROUP_FOR_ORGANIZATION",
    CREATE_SENTRY_EXTERNAL_ISSUE_LINK: "SENTRY_CREATE_SENTRY_EXTERNAL_ISSUE_LINK",
    CREATE_TEAM_PROJECT_FOR_ORGANIZATION: "SENTRY_CREATE_TEAM_PROJECT_FOR_ORGANIZATION",
    CREATE_USER_FOR_SAML_INTEGRATION: "SENTRY_CREATE_USER_FOR_SAML_INTEGRATION",
    DELETE_DSYMS_FOR_PROJECT: "SENTRY_DELETE_DSYMS_FOR_PROJECT",
    DELETE_EXTERNAL_ISSUE_BY_UUID: "SENTRY_DELETE_EXTERNAL_ISSUE_BY_UUID",
    DELETE_EXTERNAL_TEAM_BY_ID: "SENTRY_DELETE_EXTERNAL_TEAM_BY_ID",
    DELETE_EXTERNAL_USER_FROM_ORGANIZATION: "SENTRY_DELETE_EXTERNAL_USER_FROM_ORGANIZATION",
    DELETE_MEMBER_FROM_TEAM: "SENTRY_DELETE_MEMBER_FROM_TEAM",
    DELETE_ORGANIZATION_ALERT_RULE: "SENTRY_DELETE_ORGANIZATION_ALERT_RULE",
    DELETE_ORGANIZATION_DASHBOARD: "SENTRY_DELETE_ORGANIZATION_DASHBOARD",
    DELETE_ORGANIZATION_DISCOVER_QUERY: "SENTRY_DELETE_ORGANIZATION_DISCOVER_QUERY",
    DELETE_ORGANIZATION_INTEGRATION: "SENTRY_DELETE_ORGANIZATION_INTEGRATION",
    DELETE_ORGANIZATION_ISSUE: "SENTRY_DELETE_ORGANIZATION_ISSUE",
    DELETE_ORGANIZATION_MEMBER: "SENTRY_DELETE_ORGANIZATION_MEMBER",
    DELETE_ORGANIZATION_MONITOR: "SENTRY_DELETE_ORGANIZATION_MONITOR",
    DELETE_ORGANIZATION_RELEASE: "SENTRY_DELETE_ORGANIZATION_RELEASE",
    DELETE_ORG_NOTIFICATION_ACTION: "SENTRY_DELETE_ORG_NOTIFICATION_ACTION",
    DELETE_PROJECT_BY_ID: "SENTRY_DELETE_PROJECT_BY_ID",
    DELETE_PROJECT_HOOK: "SENTRY_DELETE_PROJECT_HOOK",
    DELETE_PROJECT_ISSUES: "SENTRY_DELETE_PROJECT_ISSUES",
    DELETE_PROJECT_KEY: "SENTRY_DELETE_PROJECT_KEY",
    DELETE_PROJECT_MONITOR: "SENTRY_DELETE_PROJECT_MONITOR",
    DELETE_PROJECT_REPLAY: "SENTRY_DELETE_PROJECT_REPLAY",
    DELETE_PROJECT_RULE: "SENTRY_DELETE_PROJECT_RULE",
    DELETE_PROJECT_SYMBOL_SOURCES: "SENTRY_DELETE_PROJECT_SYMBOL_SOURCES",
    DELETE_PROJECT_TEAM_ASSOCIATION: "SENTRY_DELETE_PROJECT_TEAM_ASSOCIATION",
    DELETE_RELEASE_FILE: "SENTRY_DELETE_RELEASE_FILE",
    DELETE_RELEASE_FILE_BY_ID: "SENTRY_DELETE_RELEASE_FILE_BY_ID",
    DELETE_TEAM_BY_ORGANIZATION_OR_TEAM_SLUG: "SENTRY_DELETE_TEAM_BY_ORGANIZATION_OR_TEAM_SLUG",
    DELETE_TEAM_FROM_ORG_SCIM_V2: "SENTRY_DELETE_TEAM_FROM_ORG_SCIM_V2",
    DELETE_USER_EMAILS_BY_ID: "SENTRY_DELETE_USER_EMAILS_BY_ID",
    DELETE_USER_FROM_ORG: "SENTRY_DELETE_USER_FROM_ORG",
    FETCH_ISSUE_EVENT_BY_ID: "SENTRY_FETCH_ISSUE_EVENT_BY_ID",
    FETCH_ORGANIZATION_ALERT_RULES: "SENTRY_FETCH_ORGANIZATION_ALERT_RULES",
    FETCH_ORGANIZATION_RELEASE_THRESHOLD_STATUSES: "SENTRY_FETCH_ORGANIZATION_RELEASE_THRESHOLD_STATUSES",
    FETCH_ORGANIZATION_REPLAY_COUNT: "SENTRY_FETCH_ORGANIZATION_REPLAY_COUNT",
    FETCH_ORGANIZATION_REPLAY_DETAILS: "SENTRY_FETCH_ORGANIZATION_REPLAY_DETAILS",
    FETCH_ORGANIZATION_REPLAY_SELECTORS: "SENTRY_FETCH_ORGANIZATION_REPLAY_SELECTORS",
    FETCH_PROJECT_ENVIRONMENT_DETAILS: "SENTRY_FETCH_PROJECT_ENVIRONMENT_DETAILS",
    FETCH_PROJECT_OWNERSHIP_DETAILS: "SENTRY_FETCH_PROJECT_OWNERSHIP_DETAILS",
    FETCH_PROJECT_RELEASE_FILES: "SENTRY_FETCH_PROJECT_RELEASE_FILES",
    FETCH_PROJECT_REPLAY_CLICKS: "SENTRY_FETCH_PROJECT_REPLAY_CLICKS",
    FETCH_REPLAY_RECORDING_SEGMENT: "SENTRY_FETCH_REPLAY_RECORDING_SEGMENT",
    FETCH_TAG_VALUES_FOR_ISSUE: "SENTRY_FETCH_TAG_VALUES_FOR_ISSUE",
    GET_ACTIVATION_OF_ALERT_RULE_FOR_ORGANIZATION: "SENTRY_GET_ACTIVATION_OF_ALERT_RULE_FOR_ORGANIZATION",
    GET_INTEGRATION_DETAILS_BY_ORG: "SENTRY_GET_INTEGRATION_DETAILS_BY_ORG",
    GET_ORGANIZATION_BY_ID_OR_SLUG: "SENTRY_GET_ORGANIZATION_BY_ID_OR_SLUG",
    GET_ORGANIZATION_DETAILS: "SENTRY_GET_ORGANIZATION_DETAILS",
    GET_ORGANIZATION_ENVIRONMENTS: "SENTRY_GET_ORGANIZATION_ENVIRONMENTS",
    GET_ORGANIZATION_ISSUE_DETAILS: "SENTRY_GET_ORGANIZATION_ISSUE_DETAILS",
    GET_ORGANIZATION_MONITOR_BY_ID_OR_SLUG: "SENTRY_GET_ORGANIZATION_MONITOR_BY_ID_OR_SLUG",
    GET_ORGANIZATION_RELEASE_VERSION: "SENTRY_GET_ORGANIZATION_RELEASE_VERSION",
    GET_ORGANIZATION_SESSIONS: "SENTRY_GET_ORGANIZATION_SESSIONS",
    GET_ORGANIZATION_STATS_SUMMARY: "SENTRY_GET_ORGANIZATION_STATS_SUMMARY",
    GET_PROJECT_EVENTS: "SENTRY_GET_PROJECT_EVENTS",
    GET_PROJECT_EVENT_STATS: "SENTRY_GET_PROJECT_EVENT_STATS",
    GET_PROJECT_HOOK: "SENTRY_GET_PROJECT_HOOK",
    GET_PROJECT_LIST: "SENTRY_GET_PROJECT_LIST",
    GET_PROJECT_MONITOR_BY_ID: "SENTRY_GET_PROJECT_MONITOR_BY_ID",
    GET_PROJECT_RULE_DETAILS: "SENTRY_GET_PROJECT_RULE_DETAILS",
    GET_SCIM_GROUP_BY_TEAM_ID: "SENTRY_GET_SCIM_GROUP_BY_TEAM_ID",
    GET_SENTRY_APP_INSTALLATIONS_FOR_ORGANIZATION: "SENTRY_GET_SENTRY_APP_INSTALLATIONS_FOR_ORGANIZATION",
    GET_TEAM_MEMBERS_BY_ID_OR_SLUG: "SENTRY_GET_TEAM_MEMBERS_BY_ID_OR_SLUG",
    GET_WHO_VIEWED_REPLAY_BY_PROJECT: "SENTRY_GET_WHO_VIEWED_REPLAY_BY_PROJECT",
    LIST_ORGANIZATION_DASHBOARDS: "SENTRY_LIST_ORGANIZATION_DASHBOARDS",
    LIST_ORGANIZATION_MEMBERS: "SENTRY_LIST_ORGANIZATION_MEMBERS",
    LIST_ORGANIZATION_RELEASES: "SENTRY_LIST_ORGANIZATION_RELEASES",
    LIST_ORGANIZATION_REPOSITORIES: "SENTRY_LIST_ORGANIZATION_REPOSITORIES",
    LIST_ORGANIZATION_USER_TEAMS: "SENTRY_LIST_ORGANIZATION_USER_TEAMS",
    LIST_PROJECT_USERS: "SENTRY_LIST_PROJECT_USERS",
    LIST_SCIM_V2_ORGANIZATION_USERS: "SENTRY_LIST_SCIM_V2_ORGANIZATION_USERS",
    LIST_TEAMS_IN_ORGANIZATION: "SENTRY_LIST_TEAMS_IN_ORGANIZATION",
    MANAGE_TEAM_EXTERNAL_INTEGRATIONS: "SENTRY_MANAGE_TEAM_EXTERNAL_INTEGRATIONS",
    MODIFY_ORGANIZATION_MONITOR_DATA: "SENTRY_MODIFY_ORGANIZATION_MONITOR_DATA",
    MODIFY_ORGANIZATION_NOTIFICATION_ACTION: "SENTRY_MODIFY_ORGANIZATION_NOTIFICATION_ACTION",
    MODIFY_ORGANIZATION_SETTINGS: "SENTRY_MODIFY_ORGANIZATION_SETTINGS",
    MODIFY_RELEASE_FILE_ATTRIBUTES: "SENTRY_MODIFY_RELEASE_FILE_ATTRIBUTES",
    PATCH_SCIM_GROUP_OPERATIONS: "SENTRY_PATCH_SCIM_GROUP_OPERATIONS",
    PATCH_USER_ACTIVE_STATUS_IN_ORGANIZATION: "SENTRY_PATCH_USER_ACTIVE_STATUS_IN_ORGANIZATION",
    POST_PROJECT_SYMBOL_SOURCES: "SENTRY_POST_PROJECT_SYMBOL_SOURCES",
    POST_SPIKE_PROTECTION_FOR_ORGANIZATION: "SENTRY_POST_SPIKE_PROTECTION_FOR_ORGANIZATION",
    RETRIEVE_ALERT_RULE_DETAILS: "SENTRY_RETRIEVE_ALERT_RULE_DETAILS",
    RETRIEVE_COMMITS_FOR_ORGANIZATION_REPO: "SENTRY_RETRIEVE_COMMITS_FOR_ORGANIZATION_REPO",
    RETRIEVE_COMMIT_FILES_FOR_RELEASE: "SENTRY_RETRIEVE_COMMIT_FILES_FOR_RELEASE",
    RETRIEVE_DSYM_FILES_FOR_PROJECT: "SENTRY_RETRIEVE_DSYM_FILES_FOR_PROJECT",
    RETRIEVE_EVENT_ID_FOR_ORGANIZATION: "SENTRY_RETRIEVE_EVENT_ID_FOR_ORGANIZATION",
    RETRIEVE_FILES_FOR_RELEASE: "SENTRY_RETRIEVE_FILES_FOR_RELEASE",
    RETRIEVE_ISSUE_EVENTS_BY_ID: "SENTRY_RETRIEVE_ISSUE_EVENTS_BY_ID",
    RETRIEVE_ISSUE_HASHES_FOR_ORGANIZATION: "SENTRY_RETRIEVE_ISSUE_HASHES_FOR_ORGANIZATION",
    RETRIEVE_ISSUE_TAG_DETAILS: "SENTRY_RETRIEVE_ISSUE_TAG_DETAILS",
    RETRIEVE_MONITOR_CHECKINS: "SENTRY_RETRIEVE_MONITOR_CHECKINS",
    RETRIEVE_MONITOR_CHECKINS_BY_ORG: "SENTRY_RETRIEVE_MONITOR_CHECKINS_BY_ORG",
    RETRIEVE_NOTIFICATION_ACTION_BY_ORG_ID: "SENTRY_RETRIEVE_NOTIFICATION_ACTION_BY_ORG_ID",
    RETRIEVE_ORGANIZATION_DASHBOARD: "SENTRY_RETRIEVE_ORGANIZATION_DASHBOARD",
    RETRIEVE_ORGANIZATION_EVENTS: "SENTRY_RETRIEVE_ORGANIZATION_EVENTS",
    RETRIEVE_ORGANIZATION_INTEGRATIONS_LIST: "SENTRY_RETRIEVE_ORGANIZATION_INTEGRATIONS_LIST",
    RETRIEVE_ORGANIZATION_MEMBER: "SENTRY_RETRIEVE_ORGANIZATION_MEMBER",
    RETRIEVE_ORGANIZATION_MONITORS: "SENTRY_RETRIEVE_ORGANIZATION_MONITORS",
    RETRIEVE_ORGANIZATION_PROJECTS: "SENTRY_RETRIEVE_ORGANIZATION_PROJECTS",
    RETRIEVE_ORGANIZATION_RELAY_USAGE: "SENTRY_RETRIEVE_ORGANIZATION_RELAY_USAGE",
    RETRIEVE_ORGANIZATION_RELEASE_COMMITS: "SENTRY_RETRIEVE_ORGANIZATION_RELEASE_COMMITS",
    RETRIEVE_ORGANIZATION_REPLAYS: "SENTRY_RETRIEVE_ORGANIZATION_REPLAYS",
    RETRIEVE_ORGANIZATION_SCIM_GROUPS: "SENTRY_RETRIEVE_ORGANIZATION_SCIM_GROUPS",
    RETRIEVE_ORGANIZATION_STATS_V2: "SENTRY_RETRIEVE_ORGANIZATION_STATS_V2",
    RETRIEVE_ORG_INTEGRATION_CONFIG: "SENTRY_RETRIEVE_ORG_INTEGRATION_CONFIG",
    RETRIEVE_PROJECT_ENVIRONMENTS: "SENTRY_RETRIEVE_PROJECT_ENVIRONMENTS",
    RETRIEVE_PROJECT_EVENT_BY_ID: "SENTRY_RETRIEVE_PROJECT_EVENT_BY_ID",
    RETRIEVE_PROJECT_FILTER_DATA: "SENTRY_RETRIEVE_PROJECT_FILTER_DATA",
    RETRIEVE_PROJECT_HOOKS: "SENTRY_RETRIEVE_PROJECT_HOOKS",
    RETRIEVE_PROJECT_ISSUES_LIST: "SENTRY_RETRIEVE_PROJECT_ISSUES_LIST",
    RETRIEVE_PROJECT_KEYS_BY_ORG_AND_PROJECT: "SENTRY_RETRIEVE_PROJECT_KEYS_BY_ORG_AND_PROJECT",
    RETRIEVE_PROJECT_KEY_DETAILS: "SENTRY_RETRIEVE_PROJECT_KEY_DETAILS",
    RETRIEVE_PROJECT_MEMBERS_LIST: "SENTRY_RETRIEVE_PROJECT_MEMBERS_LIST",
    RETRIEVE_PROJECT_RELEASE_FILE_DETAILS: "SENTRY_RETRIEVE_PROJECT_RELEASE_FILE_DETAILS",
    RETRIEVE_PROJECT_RULES_BY_ORG_AND_PROJECT_ID: "SENTRY_RETRIEVE_PROJECT_RULES_BY_ORG_AND_PROJECT_ID",
    RETRIEVE_PROJECT_SYMBOL_SOURCES: "SENTRY_RETRIEVE_PROJECT_SYMBOL_SOURCES",
    RETRIEVE_PROJECT_TAG_VALUES: "SENTRY_RETRIEVE_PROJECT_TAG_VALUES",
    RETRIEVE_PROJECT_TEAMS: "SENTRY_RETRIEVE_PROJECT_TEAMS",
    RETRIEVE_RELEASE_COMMITS: "SENTRY_RETRIEVE_RELEASE_COMMITS",
    RETRIEVE_RELEASE_DEPLOYMENTS: "SENTRY_RETRIEVE_RELEASE_DEPLOYMENTS",
    RETRIEVE_RELEASE_FILE_BY_ID: "SENTRY_RETRIEVE_RELEASE_FILE_BY_ID",
    RETRIEVE_REPLAY_RECORDING_SEGMENTS: "SENTRY_RETRIEVE_REPLAY_RECORDING_SEGMENTS",
    RETRIEVE_SAVED_DISCOVER_QUERIES: "SENTRY_RETRIEVE_SAVED_DISCOVER_QUERIES",
    RETRIEVE_SAVED_DISCOVER_QUERY_FOR_ORGANIZATION: "SENTRY_RETRIEVE_SAVED_DISCOVER_QUERY_FOR_ORGANIZATION",
    RETRIEVE_SHORT_ID_FOR_ORGANIZATION: "SENTRY_RETRIEVE_SHORT_ID_FOR_ORGANIZATION",
    RETRIEVE_SOURCE_MAP_DEBUG_EVENT: "SENTRY_RETRIEVE_SOURCE_MAP_DEBUG_EVENT",
    RETRIEVE_TEAM_INFO_VIA_ORGANIZATION_ID_OR_SLUG: "SENTRY_RETRIEVE_TEAM_INFO_VIA_ORGANIZATION_ID_OR_SLUG",
    RETRIEVE_TEAM_PROJECTS: "SENTRY_RETRIEVE_TEAM_PROJECTS",
    RETRIEVE_USER_EMAIL_INFORMATION: "SENTRY_RETRIEVE_USER_EMAIL_INFORMATION",
    RETRIEVE_USER_FEEDBACK_FOR_PROJECT: "SENTRY_RETRIEVE_USER_FEEDBACK_FOR_PROJECT",
    RETRIEVE_USER_VIA_SCIM_API: "SENTRY_RETRIEVE_USER_VIA_SCIM_API",
    SAVE_ORGANIZATION_DISCOVER_QUERY: "SENTRY_SAVE_ORGANIZATION_DISCOVER_QUERY",
    SUBMIT_NOTIFICATION_ACTION_API_DATA: "SENTRY_SUBMIT_NOTIFICATION_ACTION_API_DATA",
    SUBMIT_PROJECT_USER_FEEDBACK: "SENTRY_SUBMIT_PROJECT_USER_FEEDBACK",
    SWITCH_TEAM_ROLE_FOR_MEMBER: "SENTRY_SWITCH_TEAM_ROLE_FOR_MEMBER",
    TOGGLE_PROJECT_FILTER_STATUS: "SENTRY_TOGGLE_PROJECT_FILTER_STATUS",
    UPDATE_ENVIRONMENT_VISIBILITY: "SENTRY_UPDATE_ENVIRONMENT_VISIBILITY",
    UPDATE_EXTERNAL_TEAM_INTEGRATION: "SENTRY_UPDATE_EXTERNAL_TEAM_INTEGRATION",
    UPDATE_EXTERNAL_USER_FOR_ORGANIZATION: "SENTRY_UPDATE_EXTERNAL_USER_FOR_ORGANIZATION",
    UPDATE_ISSUE_ATTRIBUTES_IN_ORGANIZATION: "SENTRY_UPDATE_ISSUE_ATTRIBUTES_IN_ORGANIZATION",
    UPDATE_ORGANIZATION_ALERT_RULES: "SENTRY_UPDATE_ORGANIZATION_ALERT_RULES",
    UPDATE_ORGANIZATION_DASHBOARD: "SENTRY_UPDATE_ORGANIZATION_DASHBOARD",
    UPDATE_ORGANIZATION_MEMBER_ROLE: "SENTRY_UPDATE_ORGANIZATION_MEMBER_ROLE",
    UPDATE_PROJECT_DETAILS: "SENTRY_UPDATE_PROJECT_DETAILS",
    UPDATE_PROJECT_ISSUE_STATUS_AND_DETAILS: "SENTRY_UPDATE_PROJECT_ISSUE_STATUS_AND_DETAILS",
    UPDATE_PROJECT_KEY_CONFIGURATION: "SENTRY_UPDATE_PROJECT_KEY_CONFIGURATION",
    UPDATE_PROJECT_MONITOR: "SENTRY_UPDATE_PROJECT_MONITOR",
    UPDATE_PROJECT_OWNERSHIP_SETTINGS: "SENTRY_UPDATE_PROJECT_OWNERSHIP_SETTINGS",
    UPDATE_PROJECT_RELEASE_FILE_DETAILS: "SENTRY_UPDATE_PROJECT_RELEASE_FILE_DETAILS",
    UPDATE_PROJECT_RULE_BY_ID: "SENTRY_UPDATE_PROJECT_RULE_BY_ID",
    UPDATE_RELEASE_DETAILS_FOR_ORGANIZATION: "SENTRY_UPDATE_RELEASE_DETAILS_FOR_ORGANIZATION",
    UPDATE_SAVED_QUERY_FOR_ORGANIZATION: "SENTRY_UPDATE_SAVED_QUERY_FOR_ORGANIZATION",
    UPDATE_SYMBOL_SOURCE_SETTINGS: "SENTRY_UPDATE_SYMBOL_SOURCE_SETTINGS",
    UPDATE_TEAM_INFORMATION_BY_ORGANIZATION_ID: "SENTRY_UPDATE_TEAM_INFORMATION_BY_ORGANIZATION_ID",
    UPDATE_USER_EMAIL: "SENTRY_UPDATE_USER_EMAIL",
    UPDATE_WEBHOOK_CONFIGURATION_XP: "SENTRY_UPDATE_WEBHOOK_CONFIGURATION_XP",
    UPLOAD_DSYMS_FILE_TO_PROJECT: "SENTRY_UPLOAD_DSYMS_FILE_TO_PROJECT",
    UPLOAD_FILE_TO_PROJECT_RELEASE: "SENTRY_UPLOAD_FILE_TO_PROJECT_RELEASE",
    UPLOAD_RELEASE_FILE_TO_ORGANIZATION: "SENTRY_UPLOAD_RELEASE_FILE_TO_ORGANIZATION",
    VIEW_ORGANIZATION_NOTIFICATION_ACTIONS: "SENTRY_VIEW_ORGANIZATION_NOTIFICATION_ACTIONS",
  },
  triggerTypes: {},
}
