// @composio/core/generated/serpapi.ts
/**
 * Map of Composio's SERPAPI toolkit.
 */
export const SERPAPI = {
  slug: "serpapi",
  tools: {
    BAIDU_SEARCH: "SERPAPI_BAIDU_SEARCH",
    BING_SEARCH: "SERPAPI_BING_SEARCH",
    DUCK_DUCK_GO_SEARCH: "SERPAPI_DUCK_DUCK_GO_SEARCH",
    EBAY_SEARCH: "SERPAPI_EBAY_SEARCH",
    EVENT_SEARCH: "SERPAPI_EVENT_SEARCH",
    FINANCE_SEARCH: "SERPAPI_FINANCE_SEARCH",
    GOOGLE_DOMAINS_LIST: "SERPAPI_GOOGLE_DOMAINS_LIST",
    GOOGLE_JOBS_SEARCH: "SERPAPI_GOOGLE_JOBS_SEARCH",
    GOOGLE_LIGHT_SEARCH: "SERPAPI_GOOGLE_LIGHT_SEARCH",
    GOOGLE_MAPS_SEARCH: "SERPAPI_GOOGLE_MAPS_SEARCH",
    HOTEL_SEARCH: "SERPAPI_HOTEL_SEARCH",
    IMAGE_SEARCH: "SERPAPI_IMAGE_SEARCH",
    LOCATIONS_LIST: "SERPAPI_LOCATIONS_LIST",
    NEWS_SEARCH: "SERPAPI_NEWS_SEARCH",
    PLAY_SEARCH: "SERPAPI_PLAY_SEARCH",
    SCHOLAR_SEARCH: "SERPAPI_SCHOLAR_SEARCH",
    SEARCH: "SERPAPI_SEARCH",
    SHOPPING_SEARCH: "SERPAPI_SHOPPING_SEARCH",
    TRENDS_SEARCH: "SERPAPI_TRENDS_SEARCH",
    WALMART_SEARCH: "SERPAPI_WALMART_SEARCH",
    YAHOO_SEARCH: "SERPAPI_YAHOO_SEARCH",
    YANDEX_SEARCH: "SERPAPI_YANDEX_SEARCH",
    YOU_TUBE_SEARCH: "SERPAPI_YOU_TUBE_SEARCH",
  },
  triggerTypes: {},
}
