// @composio/core/generated/servicem8.ts
/**
 * Map of Composio's SERVICEM8 toolkit.
 */
export const SERVICEM8 = {
  slug: "servicem8",
  tools: {
    CREATE_JOBS: "SERVICEM8_CREATE_JOBS",
    CREATE_JOB_NOTE: "SERVICEM8_CREATE_JOB_NOTE",
    CREATE_JOB_PAYMENT: "SERVICEM8_CREATE_JOB_PAYMENT",
    DELETE_JOB_NOTE: "SERVICEM8_DELETE_JOB_NOTE",
    DELETE_JOB_PAYMENT: "SERVICEM8_DELETE_JOB_PAYMENT",
    LIST_ALL_ASSETS: "SERVICEM8_LIST_ALL_ASSETS",
    LIST_ALL_CLIENTS: "SERVICEM8_LIST_ALL_CLIENTS",
    LIST_ALL_DOCUMENT_TEMPLATES: "SERVICEM8_LIST_ALL_DOCUMENT_TEMPLATES",
    LIST_ALL_FORMS: "SERVICEM8_LIST_ALL_FORMS",
    LIST_ALL_JOBS: "SERVICEM8_LIST_ALL_JOBS",
    LIST_ALL_JOB_NOTES: "SERVICEM8_LIST_ALL_JOB_NOTES",
    LIST_ALL_JOB_QUEUES: "SERVICEM8_LIST_ALL_JOB_QUEUES",
    LIST_ALL_LOCATIONS: "SERVICEM8_LIST_ALL_LOCATIONS",
    LIST_ALL_MATERIALS: "SERVICEM8_LIST_ALL_MATERIALS",
    LIST_ALL_TASKS: "SERVICEM8_LIST_ALL_TASKS",
    RETRIEVE_CLIENT: "SERVICEM8_RETRIEVE_CLIENT",
    RETRIEVE_FORM: "SERVICEM8_RETRIEVE_FORM",
    RETRIEVE_JOB: "SERVICEM8_RETRIEVE_JOB",
    RETRIEVE_JOB_ACTIVITY: "SERVICEM8_RETRIEVE_JOB_ACTIVITY",
    RETRIEVE_JOB_NOTE: "SERVICEM8_RETRIEVE_JOB_NOTE",
    RETRIEVE_JOB_PAYMENT: "SERVICEM8_RETRIEVE_JOB_PAYMENT",
    RETRIEVE_JOB_QUEUE: "SERVICEM8_RETRIEVE_JOB_QUEUE",
    RETRIEVE_LOCATION: "SERVICEM8_RETRIEVE_LOCATION",
    RETRIEVE_MATERIAL: "SERVICEM8_RETRIEVE_MATERIAL",
    RETRIEVE_STAFF_MEMBER: "SERVICEM8_RETRIEVE_STAFF_MEMBER",
    SERVICE_M8_CREATE_JOB: "SERVICEM8_SERVICE_M8_CREATE_JOB",
    UPDATE_JOB_NOTE: "SERVICEM8_UPDATE_JOB_NOTE",
    UPDATE_JOB_PAYMENT: "SERVICEM8_UPDATE_JOB_PAYMENT",
  },
  triggerTypes: {},
}
