// @composio/core/generated/shopify.ts
/**
 * Map of Composio's SHOPIFY toolkit.
 */
export const SHOPIFY = {
  slug: "shopify",
  tools: {
    ADD_PRODUCT_TO_COLLECTION: "SHOPIFY_ADD_PRODUCT_TO_COLLECTION",
    COUNT_PRODUCT_IMAGES: "SHOPIFY_COUNT_PRODUCT_IMAGES",
    CREATE_CUSTOMER: "SHOPIFY_CREATE_CUSTOMER",
    CREATE_CUSTOM_COLLECTION: "SHOPIFY_CREATE_CUSTOM_COLLECTION",
    CREATE_ORDER: "SHOPIFY_CREATE_ORDER",
    CREATE_PRODUCT: "SHOPIFY_CREATE_PRODUCT",
    CREATE_PRODUCT_IMAGE: "SHOPIFY_CREATE_PRODUCT_IMAGE",
    DELETE_CUSTOM_COLLECTION: "SHOPIFY_DELETE_CUSTOM_COLLECTION",
    DELETE_PRODUCT: "SHOPIFY_DELETE_PRODUCT",
    DELETE_PRODUCT_IMAGE: "SHOPIFY_DELETE_PRODUCT_IMAGE",
    GET_ALL_CUSTOMERS: "SHOPIFY_GET_ALL_CUSTOMERS",
    GET_COLLECTION_BY_ID: "SHOPIFY_GET_COLLECTION_BY_ID",
    GET_COLLECTS: "SHOPIFY_GET_COLLECTS",
    GET_COLLECTS_COUNT: "SHOPIFY_GET_COLLECTS_COUNT",
    GET_CUSTOMER: "SHOPIFY_GET_CUSTOMER",
    GET_CUSTOMER_ORDERS: "SHOPIFY_GET_CUSTOMER_ORDERS",
    GET_CUSTOM_COLLECTIONS: "SHOPIFY_GET_CUSTOM_COLLECTIONS",
    GET_CUSTOM_COLLECTIONS_COUNT: "SHOPIFY_GET_CUSTOM_COLLECTIONS_COUNT",
    GET_ORDERSBY_ID: "SHOPIFY_GET_ORDERSBY_ID",
    GET_ORDER_LIST: "SHOPIFY_GET_ORDER_LIST",
    GET_PRODUCT: "SHOPIFY_GET_PRODUCT",
    GET_PRODUCTS: "SHOPIFY_GET_PRODUCTS",
    GET_PRODUCTS_COUNT: "SHOPIFY_GET_PRODUCTS_COUNT",
    GET_PRODUCTS_IN_COLLECTION: "SHOPIFY_GET_PRODUCTS_IN_COLLECTION",
    GET_PRODUCT_IMAGE: "SHOPIFY_GET_PRODUCT_IMAGE",
    GET_PRODUCT_IMAGES: "SHOPIFY_GET_PRODUCT_IMAGES",
    GET_SHOP_DETAILS: "SHOPIFY_GET_SHOP_DETAILS",
    UPDATE_ORDER: "SHOPIFY_UPDATE_ORDER",
  },
  triggerTypes: {},
}
