// @composio/core/generated/shortcut.ts
/**
 * Map of Composio's SHORTCUT toolkit.
 */
export const SHORTCUT = {
  slug: "shortcut",
  tools: {
    CREATE_CATEGORY: "SHORTCUT_CREATE_CATEGORY",
    CREATE_ENTITY_TEMPLATE: "SHORTCUT_CREATE_ENTITY_TEMPLATE",
    CREATE_EPIC: "SHORTCUT_CREATE_EPIC",
    CREATE_EPIC_COMMENT: "SHORTCUT_CREATE_EPIC_COMMENT",
    CREATE_EPIC_COMMENT_COMMENT: "SHORTCUT_CREATE_EPIC_COMMENT_COMMENT",
    CREATE_GROUP: "SHORTCUT_CREATE_GROUP",
    CREATE_ITERATION: "SHORTCUT_CREATE_ITERATION",
    CREATE_LABEL: "SHORTCUT_CREATE_LABEL",
    CREATE_LINKED_FILE: "SHORTCUT_CREATE_LINKED_FILE",
    CREATE_MILESTONE: "SHORTCUT_CREATE_MILESTONE",
    CREATE_MULTIPLE_STORIES: "SHORTCUT_CREATE_MULTIPLE_STORIES",
    CREATE_OBJECTIVE: "SHORTCUT_CREATE_OBJECTIVE",
    CREATE_PROJECT: "SHORTCUT_CREATE_PROJECT",
    CREATE_STORY: "SHORTCUT_CREATE_STORY",
    CREATE_STORY_COMMENT: "SHORTCUT_CREATE_STORY_COMMENT",
    CREATE_STORY_FROM_TEMPLATE: "SHORTCUT_CREATE_STORY_FROM_TEMPLATE",
    CREATE_STORY_LINK: "SHORTCUT_CREATE_STORY_LINK",
    CREATE_STORY_REACTION: "SHORTCUT_CREATE_STORY_REACTION",
    CREATE_TASK: "SHORTCUT_CREATE_TASK",
    DELETE_CATEGORY: "SHORTCUT_DELETE_CATEGORY",
    DELETE_CUSTOM_FIELD: "SHORTCUT_DELETE_CUSTOM_FIELD",
    DELETE_ENTITY_TEMPLATE: "SHORTCUT_DELETE_ENTITY_TEMPLATE",
    DELETE_EPIC: "SHORTCUT_DELETE_EPIC",
    DELETE_EPIC_COMMENT: "SHORTCUT_DELETE_EPIC_COMMENT",
    DELETE_FILE: "SHORTCUT_DELETE_FILE",
    DELETE_ITERATION: "SHORTCUT_DELETE_ITERATION",
    DELETE_LABEL: "SHORTCUT_DELETE_LABEL",
    DELETE_LINKED_FILE: "SHORTCUT_DELETE_LINKED_FILE",
    DELETE_MILESTONE: "SHORTCUT_DELETE_MILESTONE",
    DELETE_MULTIPLE_STORIES: "SHORTCUT_DELETE_MULTIPLE_STORIES",
    DELETE_OBJECTIVE: "SHORTCUT_DELETE_OBJECTIVE",
    DELETE_PROJECT: "SHORTCUT_DELETE_PROJECT",
    DELETE_STORY: "SHORTCUT_DELETE_STORY",
    DELETE_STORY_COMMENT: "SHORTCUT_DELETE_STORY_COMMENT",
    DELETE_STORY_LINK: "SHORTCUT_DELETE_STORY_LINK",
    DELETE_STORY_REACTION: "SHORTCUT_DELETE_STORY_REACTION",
    DELETE_TASK: "SHORTCUT_DELETE_TASK",
    DISABLE_ITERATIONS: "SHORTCUT_DISABLE_ITERATIONS",
    DISABLE_STORY_TEMPLATES: "SHORTCUT_DISABLE_STORY_TEMPLATES",
    ENABLE_ITERATIONS: "SHORTCUT_ENABLE_ITERATIONS",
    ENABLE_STORY_TEMPLATES: "SHORTCUT_ENABLE_STORY_TEMPLATES",
    GET_CATEGORY: "SHORTCUT_GET_CATEGORY",
    GET_CURRENT_MEMBER_INFO: "SHORTCUT_GET_CURRENT_MEMBER_INFO",
    GET_CUSTOM_FIELD: "SHORTCUT_GET_CUSTOM_FIELD",
    GET_ENTITY_TEMPLATE: "SHORTCUT_GET_ENTITY_TEMPLATE",
    GET_EPIC: "SHORTCUT_GET_EPIC",
    GET_EPIC_COMMENT: "SHORTCUT_GET_EPIC_COMMENT",
    GET_EPIC_WORKFLOW: "SHORTCUT_GET_EPIC_WORKFLOW",
    GET_EXTERNAL_LINK_STORIES: "SHORTCUT_GET_EXTERNAL_LINK_STORIES",
    GET_FILE: "SHORTCUT_GET_FILE",
    GET_GROUP: "SHORTCUT_GET_GROUP",
    GET_ITERATION: "SHORTCUT_GET_ITERATION",
    GET_KEY_RESULT: "SHORTCUT_GET_KEY_RESULT",
    GET_LABEL: "SHORTCUT_GET_LABEL",
    GET_LINKED_FILE: "SHORTCUT_GET_LINKED_FILE",
    GET_MEMBER: "SHORTCUT_GET_MEMBER",
    GET_MILESTONE: "SHORTCUT_GET_MILESTONE",
    GET_OBJECTIVE: "SHORTCUT_GET_OBJECTIVE",
    GET_PROJECT: "SHORTCUT_GET_PROJECT",
    GET_REPOSITORY: "SHORTCUT_GET_REPOSITORY",
    GET_STORY: "SHORTCUT_GET_STORY",
    GET_STORY_COMMENT: "SHORTCUT_GET_STORY_COMMENT",
    GET_STORY_LINK: "SHORTCUT_GET_STORY_LINK",
    GET_TASK: "SHORTCUT_GET_TASK",
    GET_WORKFLOW: "SHORTCUT_GET_WORKFLOW",
    LIST_CATEGORIES: "SHORTCUT_LIST_CATEGORIES",
    LIST_CATEGORY_MILESTONES: "SHORTCUT_LIST_CATEGORY_MILESTONES",
    LIST_CATEGORY_OBJECTIVES: "SHORTCUT_LIST_CATEGORY_OBJECTIVES",
    LIST_CUSTOM_FIELDS: "SHORTCUT_LIST_CUSTOM_FIELDS",
    LIST_ENTITY_TEMPLATES: "SHORTCUT_LIST_ENTITY_TEMPLATES",
    LIST_EPICS: "SHORTCUT_LIST_EPICS",
    LIST_EPIC_COMMENTS: "SHORTCUT_LIST_EPIC_COMMENTS",
    LIST_EPIC_STORIES: "SHORTCUT_LIST_EPIC_STORIES",
    LIST_FILES: "SHORTCUT_LIST_FILES",
    LIST_GROUPS: "SHORTCUT_LIST_GROUPS",
    LIST_GROUP_STORIES: "SHORTCUT_LIST_GROUP_STORIES",
    LIST_ITERATIONS: "SHORTCUT_LIST_ITERATIONS",
    LIST_ITERATION_STORIES: "SHORTCUT_LIST_ITERATION_STORIES",
    LIST_LABELS: "SHORTCUT_LIST_LABELS",
    LIST_LABEL_EPICS: "SHORTCUT_LIST_LABEL_EPICS",
    LIST_LABEL_STORIES: "SHORTCUT_LIST_LABEL_STORIES",
    LIST_LINKED_FILES: "SHORTCUT_LIST_LINKED_FILES",
    LIST_MEMBERS: "SHORTCUT_LIST_MEMBERS",
    LIST_MILESTONES: "SHORTCUT_LIST_MILESTONES",
    LIST_MILESTONE_EPICS: "SHORTCUT_LIST_MILESTONE_EPICS",
    LIST_OBJECTIVES: "SHORTCUT_LIST_OBJECTIVES",
    LIST_OBJECTIVE_EPICS: "SHORTCUT_LIST_OBJECTIVE_EPICS",
    LIST_PROJECTS: "SHORTCUT_LIST_PROJECTS",
    LIST_REPOSITORIES: "SHORTCUT_LIST_REPOSITORIES",
    LIST_STORIES: "SHORTCUT_LIST_STORIES",
    LIST_STORY_COMMENT: "SHORTCUT_LIST_STORY_COMMENT",
    LIST_WORKFLOWS: "SHORTCUT_LIST_WORKFLOWS",
    SEARCH: "SHORTCUT_SEARCH",
    SEARCH_EPICS: "SHORTCUT_SEARCH_EPICS",
    SEARCH_ITERATIONS: "SHORTCUT_SEARCH_ITERATIONS",
    SEARCH_MILESTONES: "SHORTCUT_SEARCH_MILESTONES",
    SEARCH_OBJECTIVES: "SHORTCUT_SEARCH_OBJECTIVES",
    SEARCH_STORIES: "SHORTCUT_SEARCH_STORIES",
    SEARCH_STORIES_OLD: "SHORTCUT_SEARCH_STORIES_OLD",
    STORY_HISTORY: "SHORTCUT_STORY_HISTORY",
    UNLINK_COMMENT_THREAD_FROM_SLACK: "SHORTCUT_UNLINK_COMMENT_THREAD_FROM_SLACK",
    UNLINK_PRODUCTBOARD_FROM_EPIC: "SHORTCUT_UNLINK_PRODUCTBOARD_FROM_EPIC",
    UPDATE_CATEGORY: "SHORTCUT_UPDATE_CATEGORY",
    UPDATE_CUSTOM_FIELD: "SHORTCUT_UPDATE_CUSTOM_FIELD",
    UPDATE_ENTITY_TEMPLATE: "SHORTCUT_UPDATE_ENTITY_TEMPLATE",
    UPDATE_EPIC: "SHORTCUT_UPDATE_EPIC",
    UPDATE_EPIC_COMMENT: "SHORTCUT_UPDATE_EPIC_COMMENT",
    UPDATE_FILE: "SHORTCUT_UPDATE_FILE",
    UPDATE_GROUP: "SHORTCUT_UPDATE_GROUP",
    UPDATE_ITERATION: "SHORTCUT_UPDATE_ITERATION",
    UPDATE_KEY_RESULT: "SHORTCUT_UPDATE_KEY_RESULT",
    UPDATE_LABEL: "SHORTCUT_UPDATE_LABEL",
    UPDATE_LINKED_FILE: "SHORTCUT_UPDATE_LINKED_FILE",
    UPDATE_MILESTONE: "SHORTCUT_UPDATE_MILESTONE",
    UPDATE_MULTIPLE_STORIES: "SHORTCUT_UPDATE_MULTIPLE_STORIES",
    UPDATE_OBJECTIVE: "SHORTCUT_UPDATE_OBJECTIVE",
    UPDATE_PROJECT: "SHORTCUT_UPDATE_PROJECT",
    UPDATE_STORY: "SHORTCUT_UPDATE_STORY",
    UPDATE_STORY_COMMENT: "SHORTCUT_UPDATE_STORY_COMMENT",
    UPDATE_STORY_LINK: "SHORTCUT_UPDATE_STORY_LINK",
    UPDATE_TASK: "SHORTCUT_UPDATE_TASK",
    UPLOAD_FILES: "SHORTCUT_UPLOAD_FILES",
  },
  triggerTypes: {},
}
