// @composio/core/generated/slackbot.ts
/**
 * Map of Composio's SLACKBOT toolkit.
 */
export const SLACKBOT = {
  slug: "slackbot",
  tools: {
    ACTIVATE_OR_MODIFY_DO_NOT_DISTURB_DURATION: "SLACKBOT_ACTIVATE_OR_MODIFY_DO_NOT_DISTURB_DURATION",
    ADD_AN_EMOJI_ALIAS_IN_SLACK: "SLACKBOT_ADD_AN_EMOJI_ALIAS_IN_SLACK",
    ADD_A_CUSTOM_EMOJI_TO_A_SLACK_TEAM: "SLACKBOT_ADD_A_CUSTOM_EMOJI_TO_A_SLACK_TEAM",
    ADD_A_REMOTE_FILE_FROM_A_SERVICE: "SLACKBOT_ADD_A_REMOTE_FILE_FROM_A_SERVICE",
    ADD_A_STAR_TO_AN_ITEM: "SLACKBOT_ADD_A_STAR_TO_AN_ITEM",
    ADD_DEFAULT_CHANNELS_TO_IDP_GROUP: "SLACKBOT_ADD_DEFAULT_CHANNELS_TO_IDP_GROUP",
    ADD_ENTERPRISE_USER_TO_WORKSPACE: "SLACKBOT_ADD_ENTERPRISE_USER_TO_WORKSPACE",
    ADD_IDP_GROUP_ALLOWLIST_TO_CHANNEL: "SLACKBOT_ADD_IDP_GROUP_ALLOWLIST_TO_CHANNEL",
    ADD_REACTION_TO_AN_ITEM: "SLACKBOT_ADD_REACTION_TO_AN_ITEM",
    APPROVE_AN_APP_INSTALLATION_IN_A_WORKSPACE: "SLACKBOT_APPROVE_AN_APP_INSTALLATION_IN_A_WORKSPACE",
    APPROVE_WORKSPACE_INVITE_REQUEST: "SLACKBOT_APPROVE_WORKSPACE_INVITE_REQUEST",
    APPS_PERMISSIONS_LIST_PERMISSIONS: "SLACKBOT_APPS_PERMISSIONS_LIST_PERMISSIONS",
    APPS_PERMISSIONS_SCOPES_GET_LIST: "SLACKBOT_APPS_PERMISSIONS_SCOPES_GET_LIST",
    ARCHIVE_A_PUBLIC_OR_PRIVATE_CHANNEL: "SLACKBOT_ARCHIVE_A_PUBLIC_OR_PRIVATE_CHANNEL",
    ARCHIVE_A_SLACK_CONVERSATION: "SLACKBOT_ARCHIVE_A_SLACK_CONVERSATION",
    ASSOCIATE_DEFAULT_WORKSPACES_WITH_IDP_GROUP: "SLACKBOT_ASSOCIATE_DEFAULT_WORKSPACES_WITH_IDP_GROUP",
    CHAT_POST_MESSAGE: "SLACKBOT_CHAT_POST_MESSAGE",
    CHECKS_API_CALL_AND_RETURNS_SUCCESS_OR_ERROR: "SLACKBOT_CHECKS_API_CALL_AND_RETURNS_SUCCESS_OR_ERROR",
    CHECKS_AUTHENTICATION_AND_IDENTITY: "SLACKBOT_CHECKS_AUTHENTICATION_AND_IDENTITY",
    CLOSE_DM_OR_MULTI_PERSON_DM: "SLACKBOT_CLOSE_DM_OR_MULTI_PERSON_DM",
    CONVERSATIONS_HISTORY: "SLACKBOT_CONVERSATIONS_HISTORY",
    CONVERSATIONS_INFO: "SLACKBOT_CONVERSATIONS_INFO",
    CONVERSATIONS_LIST: "SLACKBOT_CONVERSATIONS_LIST",
    CONVERT_PUBLIC_CHANNEL_TO_PRIVATE: "SLACKBOT_CONVERT_PUBLIC_CHANNEL_TO_PRIVATE",
    CREATE_AN_ENTERPRISE_TEAM: "SLACKBOT_CREATE_AN_ENTERPRISE_TEAM",
    CREATE_A_REMINDER: "SLACKBOT_CREATE_A_REMINDER",
    CREATE_A_SLACK_USER_GROUP: "SLACKBOT_CREATE_A_SLACK_USER_GROUP",
    CREATE_CHANNEL_BASED_CONVERSATION: "SLACKBOT_CREATE_CHANNEL_BASED_CONVERSATION",
    CUSTOMIZE_URL_UNFURLING_IN_MESSAGES: "SLACKBOT_CUSTOMIZE_URL_UNFURLING_IN_MESSAGES",
    DELETES_A_MESSAGE_FROM_A_CHAT: "SLACKBOT_DELETES_A_MESSAGE_FROM_A_CHAT",
    DELETE_A_COMMENT_ON_A_FILE: "SLACKBOT_DELETE_A_COMMENT_ON_A_FILE",
    DELETE_A_FILE_BY_ID: "SLACKBOT_DELETE_A_FILE_BY_ID",
    DELETE_A_PUBLIC_OR_PRIVATE_CHANNEL: "SLACKBOT_DELETE_A_PUBLIC_OR_PRIVATE_CHANNEL",
    DELETE_A_SCHEDULED_MESSAGE_IN_A_CHAT: "SLACKBOT_DELETE_A_SCHEDULED_MESSAGE_IN_A_CHAT",
    DELETE_A_SLACK_REMINDER: "SLACKBOT_DELETE_A_SLACK_REMINDER",
    DELETE_USER_PROFILE_PHOTO: "SLACKBOT_DELETE_USER_PROFILE_PHOTO",
    DENY_WORKSPACE_INVITE_REQUEST: "SLACKBOT_DENY_WORKSPACE_INVITE_REQUEST",
    DISABLE_AN_EXISTING_SLACK_USER_GROUP: "SLACKBOT_DISABLE_AN_EXISTING_SLACK_USER_GROUP",
    DISCONNECT_CHANNEL_FROM_WORKSPACES: "SLACKBOT_DISCONNECT_CHANNEL_FROM_WORKSPACES",
    ENABLE_A_SPECIFIED_USER_GROUP: "SLACKBOT_ENABLE_A_SPECIFIED_USER_GROUP",
    ENABLE_PUBLIC_SHARING_OF_A_FILE: "SLACKBOT_ENABLE_PUBLIC_SHARING_OF_A_FILE",
    END_A_CALL_WITH_DURATION_AND_ID: "SLACKBOT_END_A_CALL_WITH_DURATION_AND_ID",
    END_USER_DO_NOT_DISTURB_SESSION: "SLACKBOT_END_USER_DO_NOT_DISTURB_SESSION",
    END_USER_SNOOZE_MODE_IMMEDIATELY: "SLACKBOT_END_USER_SNOOZE_MODE_IMMEDIATELY",
    EXCHANGE_OAUTH_CODE_FOR_ACCESS_TOKEN: "SLACKBOT_EXCHANGE_OAUTH_CODE_FOR_ACCESS_TOKEN",
    EXCHANGE_OAUTH_CODE_FOR_WORKSPACE_TOKEN: "SLACKBOT_EXCHANGE_OAUTH_CODE_FOR_WORKSPACE_TOKEN",
    EXCHANGE_OAUTH_VERIFIER_FOR_ACCESS_TOKEN: "SLACKBOT_EXCHANGE_OAUTH_VERIFIER_FOR_ACCESS_TOKEN",
    FETCH_BOT_USER_INFORMATION: "SLACKBOT_FETCH_BOT_USER_INFORMATION",
    FETCH_CONVERSATION_HISTORY: "SLACKBOT_FETCH_CONVERSATION_HISTORY",
    FETCH_CURRENT_TEAM_INFO_WITH_OPTIONAL_TEAM_SCOPE: "SLACKBOT_FETCH_CURRENT_TEAM_INFO_WITH_OPTIONAL_TEAM_SCOPE",
    FETCH_DND_STATUS_FOR_MULTIPLE_TEAM_MEMBERS: "SLACKBOT_FETCH_DND_STATUS_FOR_MULTIPLE_TEAM_MEMBERS",
    FETCH_ITEM_REACTIONS: "SLACKBOT_FETCH_ITEM_REACTIONS",
    FETCH_MESSAGE_THREAD_FROM_A_CONVERSATION: "SLACKBOT_FETCH_MESSAGE_THREAD_FROM_A_CONVERSATION",
    FETCH_WORKSPACE_SETTINGS_INFORMATION: "SLACKBOT_FETCH_WORKSPACE_SETTINGS_INFORMATION",
    FIND_USER_BY_EMAIL_ADDRESS: "SLACKBOT_FIND_USER_BY_EMAIL_ADDRESS",
    GET_BILLABLE_TEAM_USER_INFO: "SLACKBOT_GET_BILLABLE_TEAM_USER_INFO",
    GET_CHANNEL_CONVERSATION_PREFERENCES: "SLACKBOT_GET_CHANNEL_CONVERSATION_PREFERENCES",
    GET_CHANNEL_S_CONNECTED_WORKSPACES_IN_ENTERPRISE: "SLACKBOT_GET_CHANNEL_S_CONNECTED_WORKSPACES_IN_ENTERPRISE",
    GET_CURRENT_TEAM_INTEGRATION_LOGS: "SLACKBOT_GET_CURRENT_TEAM_INTEGRATION_LOGS",
    GET_CURRENT_TEAM_S_ACCESS_LOGS: "SLACKBOT_GET_CURRENT_TEAM_S_ACCESS_LOGS",
    GET_REMINDER_INFORMATION: "SLACKBOT_GET_REMINDER_INFORMATION",
    GET_USER_PRESENCE_INFO: "SLACKBOT_GET_USER_PRESENCE_INFO",
    INDICATE_WORKFLOW_STEP_FAILURE: "SLACKBOT_INDICATE_WORKFLOW_STEP_FAILURE",
    INITIATES_CHANNEL_BASED_CONVERSATIONS: "SLACKBOT_INITIATES_CHANNEL_BASED_CONVERSATIONS",
    INVALIDATE_A_USER_SESSION_BY_ID: "SLACKBOT_INVALIDATE_A_USER_SESSION_BY_ID",
    INVITE_USERS_TO_A_SLACK_CHANNEL: "SLACKBOT_INVITE_USERS_TO_A_SLACK_CHANNEL",
    INVITE_USER_TO_CHANNEL: "SLACKBOT_INVITE_USER_TO_CHANNEL",
    INVITE_USER_TO_WORKSPACE_WITH_OPTIONAL_CHANNEL_INVITES: "SLACKBOT_INVITE_USER_TO_WORKSPACE_WITH_OPTIONAL_CHANNEL_INVITES",
    JOIN_AN_EXISTING_CONVERSATION: "SLACKBOT_JOIN_AN_EXISTING_CONVERSATION",
    LEAVE_A_CONVERSATION: "SLACKBOT_LEAVE_A_CONVERSATION",
    LISTS_PINNED_ITEMS_IN_A_CHANNEL: "SLACKBOT_LISTS_PINNED_ITEMS_IN_A_CHANNEL",
    LISTS_USER_S_STARRED_ITEMS_WITH_PAGINATION: "SLACKBOT_LISTS_USER_S_STARRED_ITEMS_WITH_PAGINATION",
    LIST_ACCESSIBLE_CONVERSATIONS_FOR_A_USER: "SLACKBOT_LIST_ACCESSIBLE_CONVERSATIONS_FOR_A_USER",
    LIST_ALL_SLACK_TEAM_CHANNELS_WITH_VARIOUS_FILTERS: "SLACKBOT_LIST_ALL_SLACK_TEAM_CHANNELS_WITH_VARIOUS_FILTERS",
    LIST_ALL_SLACK_TEAM_USERS_WITH_PAGINATION: "SLACKBOT_LIST_ALL_SLACK_TEAM_USERS_WITH_PAGINATION",
    LIST_ALL_TEAMS_IN_AN_ENTERPRISE_ORGANIZATION: "SLACKBOT_LIST_ALL_TEAMS_IN_AN_ENTERPRISE_ORGANIZATION",
    LIST_ALL_USERS_IN_A_USER_GROUP: "SLACKBOT_LIST_ALL_USERS_IN_A_USER_GROUP",
    LIST_APPROVED_APPS_FOR_ORG_OR_WORKSPACE: "SLACKBOT_LIST_APPROVED_APPS_FOR_ORG_OR_WORKSPACE",
    LIST_APPROVED_WORKSPACE_INVITE_REQUESTS: "SLACKBOT_LIST_APPROVED_WORKSPACE_INVITE_REQUESTS",
    LIST_APP_PERMISSIONS_ON_A_TEAM: "SLACKBOT_LIST_APP_PERMISSIONS_ON_A_TEAM",
    LIST_APP_PERMISSIONS_SCOPES_ON_A_TEAM: "SLACKBOT_LIST_APP_PERMISSIONS_SCOPES_ON_A_TEAM",
    LIST_APP_RESOURCE_GRANTS_ON_A_TEAM: "SLACKBOT_LIST_APP_RESOURCE_GRANTS_ON_A_TEAM",
    LIST_APP_USER_GRANTS_AND_SCOPES_ON_TEAM: "SLACKBOT_LIST_APP_USER_GRANTS_AND_SCOPES_ON_TEAM",
    LIST_DENIED_WORKSPACE_INVITE_REQUESTS: "SLACKBOT_LIST_DENIED_WORKSPACE_INVITE_REQUESTS",
    LIST_DISCONNECTED_CHANNELS_WITH_ORIGINAL_IDS_FOR_EKM: "SLACKBOT_LIST_DISCONNECTED_CHANNELS_WITH_ORIGINAL_IDS_FOR_EKM",
    LIST_ENTERPRISE_GRID_ORGANIZATION_EMOJIS: "SLACKBOT_LIST_ENTERPRISE_GRID_ORGANIZATION_EMOJIS",
    LIST_EVENT_AUTHORIZATIONS_FOR_APPS: "SLACKBOT_LIST_EVENT_AUTHORIZATIONS_FOR_APPS",
    LIST_FILES_WITH_FILTERS_IN_SLACK: "SLACKBOT_LIST_FILES_WITH_FILTERS_IN_SLACK",
    LIST_IDP_GROUPS_LINKED_TO_A_CHANNEL: "SLACKBOT_LIST_IDP_GROUPS_LINKED_TO_A_CHANNEL",
    LIST_ORG_LEVEL_IDP_GROUP_CHANNELS: "SLACKBOT_LIST_ORG_LEVEL_IDP_GROUP_CHANNELS",
    LIST_PENDING_WORKSPACE_INVITE_REQUESTS: "SLACKBOT_LIST_PENDING_WORKSPACE_INVITE_REQUESTS",
    LIST_RESTRICTED_APPS_FOR_AN_ORG_OR_WORKSPACE: "SLACKBOT_LIST_RESTRICTED_APPS_FOR_AN_ORG_OR_WORKSPACE",
    LIST_SCHEDULED_MESSAGES_IN_A_CHANNEL: "SLACKBOT_LIST_SCHEDULED_MESSAGES_IN_A_CHANNEL",
    LIST_SLACK_S_REMOTE_FILES_WITH_FILTERS: "SLACKBOT_LIST_SLACK_S_REMOTE_FILES_WITH_FILTERS",
    LIST_TEAM_CUSTOM_EMOJIS: "SLACKBOT_LIST_TEAM_CUSTOM_EMOJIS",
    LIST_TEAM_WORKSPACE_APP_REQUESTS: "SLACKBOT_LIST_TEAM_WORKSPACE_APP_REQUESTS",
    LIST_USER_GROUPS_FOR_TEAM_WITH_OPTIONS: "SLACKBOT_LIST_USER_GROUPS_FOR_TEAM_WITH_OPTIONS",
    LIST_USER_REACTIONS: "SLACKBOT_LIST_USER_REACTIONS",
    LIST_USER_REMINDERS_WITH_DETAILS: "SLACKBOT_LIST_USER_REMINDERS_WITH_DETAILS",
    LIST_WORKSPACE_ADMINS: "SLACKBOT_LIST_WORKSPACE_ADMINS",
    LIST_WORKSPACE_OWNERS: "SLACKBOT_LIST_WORKSPACE_OWNERS",
    LIST_WORKSPACE_USERS: "SLACKBOT_LIST_WORKSPACE_USERS",
    MANUALLY_SET_USER_PRESENCE: "SLACKBOT_MANUALLY_SET_USER_PRESENCE",
    MAP_USER_IDS_FOR_ENTERPRISE_GRID_WORKSPACES: "SLACKBOT_MAP_USER_IDS_FOR_ENTERPRISE_GRID_WORKSPACES",
    MARK_A_USER_AS_ACTIVE_DEPRECATED: "SLACKBOT_MARK_A_USER_AS_ACTIVE_DEPRECATED",
    MARK_REMINDER_AS_COMPLETE: "SLACKBOT_MARK_REMINDER_AS_COMPLETE",
    MARK_WORKFLOW_STEP_AS_COMPLETED: "SLACKBOT_MARK_WORKFLOW_STEP_AS_COMPLETED",
    OPEN_A_VIEW_FOR_A_SLACK_USER: "SLACKBOT_OPEN_A_VIEW_FOR_A_SLACK_USER",
    OPEN_OR_RESUME_DIRECT_OR_MULTI_PERSON_MESSAGES: "SLACKBOT_OPEN_OR_RESUME_DIRECT_OR_MULTI_PERSON_MESSAGES",
    OPEN_USER_DIALOG_WITH_JSON_DEFINED_UI: "SLACKBOT_OPEN_USER_DIALOG_WITH_JSON_DEFINED_UI",
    PINS_AN_ITEM_TO_A_CHANNEL: "SLACKBOT_PINS_AN_ITEM_TO_A_CHANNEL",
    PROMOTE_USER_TO_ADMIN_STATUS: "SLACKBOT_PROMOTE_USER_TO_ADMIN_STATUS",
    PROMOTE_USER_TO_WORKSPACE_OWNER: "SLACKBOT_PROMOTE_USER_TO_WORKSPACE_OWNER",
    PUBLISH_STATIC_VIEW_FOR_USER: "SLACKBOT_PUBLISH_STATIC_VIEW_FOR_USER",
    PUSH_VIEW_TO_ROOT_VIEW_STACK: "SLACKBOT_PUSH_VIEW_TO_ROOT_VIEW_STACK",
    REGISTERS_A_NEW_CALL_WITH_PARTICIPANTS: "SLACKBOT_REGISTERS_A_NEW_CALL_WITH_PARTICIPANTS",
    REGISTERS_NEW_CALL_PARTICIPANTS: "SLACKBOT_REGISTERS_NEW_CALL_PARTICIPANTS",
    REGISTER_CALL_PARTICIPANTS_REMOVAL: "SLACKBOT_REGISTER_CALL_PARTICIPANTS_REMOVAL",
    REMINDERS_ADD: "SLACKBOT_REMINDERS_ADD",
    REMOVE_A_REMOTE_FILE: "SLACKBOT_REMOVE_A_REMOTE_FILE",
    REMOVE_A_STAR_FROM_AN_ITEM: "SLACKBOT_REMOVE_A_STAR_FROM_AN_ITEM",
    REMOVE_A_USER_FROM_A_CONVERSATION: "SLACKBOT_REMOVE_A_USER_FROM_A_CONVERSATION",
    REMOVE_CHANNELS_FROM_IDP_GROUP: "SLACKBOT_REMOVE_CHANNELS_FROM_IDP_GROUP",
    REMOVE_ENTERPRISE_GRID_ORG_EMOJI: "SLACKBOT_REMOVE_ENTERPRISE_GRID_ORG_EMOJI",
    REMOVE_IDP_GROUP_FROM_PRIVATE_CHANNEL: "SLACKBOT_REMOVE_IDP_GROUP_FROM_PRIVATE_CHANNEL",
    REMOVE_REACTION_FROM_ITEM: "SLACKBOT_REMOVE_REACTION_FROM_ITEM",
    REMOVE_USER_FROM_WORKSPACE: "SLACKBOT_REMOVE_USER_FROM_WORKSPACE",
    RENAME_AN_EMOJI: "SLACKBOT_RENAME_AN_EMOJI",
    RENAME_A_CONVERSATION: "SLACKBOT_RENAME_A_CONVERSATION",
    RENAME_A_SLACK_CHANNEL: "SLACKBOT_RENAME_A_SLACK_CHANNEL",
    REQUEST_ADDITIONAL_APP_PERMISSIONS: "SLACKBOT_REQUEST_ADDITIONAL_APP_PERMISSIONS",
    RESTRICT_APP_INSTALLATION_ON_WORKSPACE: "SLACKBOT_RESTRICT_APP_INSTALLATION_ON_WORKSPACE",
    RETRIEVE_A_USER_S_IDENTITY_DETAILS: "SLACKBOT_RETRIEVE_A_USER_S_IDENTITY_DETAILS",
    RETRIEVE_CALL_INFORMATION: "SLACKBOT_RETRIEVE_CALL_INFORMATION",
    RETRIEVE_CONVERSATION_INFORMATION: "SLACKBOT_RETRIEVE_CONVERSATION_INFORMATION",
    RETRIEVE_CONVERSATION_MEMBERS_LIST: "SLACKBOT_RETRIEVE_CONVERSATION_MEMBERS_LIST",
    RETRIEVE_CURRENT_USER_DND_STATUS: "SLACKBOT_RETRIEVE_CURRENT_USER_DND_STATUS",
    RETRIEVE_DETAILED_INFORMATION_ABOUT_A_FILE: "SLACKBOT_RETRIEVE_DETAILED_INFORMATION_ABOUT_A_FILE",
    RETRIEVE_DETAILED_USER_INFORMATION: "SLACKBOT_RETRIEVE_DETAILED_USER_INFORMATION",
    RETRIEVE_MESSAGE_PERMALINK_URL: "SLACKBOT_RETRIEVE_MESSAGE_PERMALINK_URL",
    RETRIEVE_REMOTE_FILE_INFO_IN_SLACK: "SLACKBOT_RETRIEVE_REMOTE_FILE_INFO_IN_SLACK",
    RETRIEVE_TEAM_PROFILE_DETAILS: "SLACKBOT_RETRIEVE_TEAM_PROFILE_DETAILS",
    RETRIEVE_USER_PROFILE_INFORMATION: "SLACKBOT_RETRIEVE_USER_PROFILE_INFORMATION",
    REVERSE_A_CONVERSATION_S_ARCHIVAL_STATUS: "SLACKBOT_REVERSE_A_CONVERSATION_S_ARCHIVAL_STATUS",
    REVOKES_A_TOKEN_WITH_OPTIONAL_TEST_MODE: "SLACKBOT_REVOKES_A_TOKEN_WITH_OPTIONAL_TEST_MODE",
    REVOKE_PUBLIC_SHARING_ACCESS_FOR_A_FILE: "SLACKBOT_REVOKE_PUBLIC_SHARING_ACCESS_FOR_A_FILE",
    SCHEDULES_A_MESSAGE_TO_A_CHANNEL_AT_A_SPECIFIED_TIME: "SLACKBOT_SCHEDULES_A_MESSAGE_TO_A_CHANNEL_AT_A_SPECIFIED_TIME",
    SEARCH_CHANNELS_IN_AN_ENTERPRISE_ORGANIZATION: "SLACKBOT_SEARCH_CHANNELS_IN_AN_ENTERPRISE_ORGANIZATION",
    SEARCH_FOR_MESSAGES_WITH_QUERY: "SLACKBOT_SEARCH_FOR_MESSAGES_WITH_QUERY",
    SEARCH_MESSAGES: "SLACKBOT_SEARCH_MESSAGES",
    SENDS_A_MESSAGE_TO_A_SLACK_CHANNEL: "SLACKBOT_SENDS_A_MESSAGE_TO_A_SLACK_CHANNEL",
    SENDS_EPHEMERAL_MESSAGES_TO_CHANNEL_USERS: "SLACKBOT_SENDS_EPHEMERAL_MESSAGES_TO_CHANNEL_USERS",
    SET_A_CONVERSATION_S_PURPOSE: "SLACKBOT_SET_A_CONVERSATION_S_PURPOSE",
    SET_CHANNEL_POSTING_PERMISSIONS: "SLACKBOT_SET_CHANNEL_POSTING_PERMISSIONS",
    SET_EXPIRATION_FOR_A_GUEST_USER: "SLACKBOT_SET_EXPIRATION_FOR_A_GUEST_USER",
    SET_READ_CURSOR_IN_A_CONVERSATION: "SLACKBOT_SET_READ_CURSOR_IN_A_CONVERSATION",
    SET_SLACK_USER_PROFILE_INFORMATION: "SLACKBOT_SET_SLACK_USER_PROFILE_INFORMATION",
    SET_THE_TOPIC_OF_A_CONVERSATION: "SLACKBOT_SET_THE_TOPIC_OF_A_CONVERSATION",
    SET_USER_PROFILE_PHOTO_WITH_CROPPING_OPTIONS: "SLACKBOT_SET_USER_PROFILE_PHOTO_WITH_CROPPING_OPTIONS",
    SET_USER_TO_REGULAR_STATUS: "SLACKBOT_SET_USER_TO_REGULAR_STATUS",
    SET_WORKSPACE_CONNECTIONS_FOR_A_CHANNEL: "SLACKBOT_SET_WORKSPACE_CONNECTIONS_FOR_A_CHANNEL",
    SET_WORKSPACE_DEFAULT_CHANNELS: "SLACKBOT_SET_WORKSPACE_DEFAULT_CHANNELS",
    SET_WORKSPACE_DESCRIPTION: "SLACKBOT_SET_WORKSPACE_DESCRIPTION",
    SET_WORKSPACE_DISCOVERABILITY_SETTINGS: "SLACKBOT_SET_WORKSPACE_DISCOVERABILITY_SETTINGS",
    SET_WORKSPACE_ICON: "SLACKBOT_SET_WORKSPACE_ICON",
    SET_WORKSPACE_NAME: "SLACKBOT_SET_WORKSPACE_NAME",
    SHARE_A_ME_MESSAGE_IN_A_CHANNEL: "SLACKBOT_SHARE_A_ME_MESSAGE_IN_A_CHANNEL",
    SHARE_REMOTE_FILE_IN_CHANNELS: "SLACKBOT_SHARE_REMOTE_FILE_IN_CHANNELS",
    START_REAL_TIME_MESSAGING_SESSION: "SLACKBOT_START_REAL_TIME_MESSAGING_SESSION",
    TRIGGER_USER_PERMISSIONS_MODAL: "SLACKBOT_TRIGGER_USER_PERMISSIONS_MODAL",
    UNARCHIVE_A_PUBLIC_OR_PRIVATE_CHANNEL: "SLACKBOT_UNARCHIVE_A_PUBLIC_OR_PRIVATE_CHANNEL",
    UNINSTALL_APP_FROM_WORKSPACE: "SLACKBOT_UNINSTALL_APP_FROM_WORKSPACE",
    UNPIN_ITEM_FROM_CHANNEL: "SLACKBOT_UNPIN_ITEM_FROM_CHANNEL",
    UPDATES_AN_EXISTING_REMOTE_FILE: "SLACKBOT_UPDATES_AN_EXISTING_REMOTE_FILE",
    UPDATES_A_SLACK_MESSAGE: "SLACKBOT_UPDATES_A_SLACK_MESSAGE",
    UPDATE_AN_EXISTING_SLACK_USER_GROUP: "SLACKBOT_UPDATE_AN_EXISTING_SLACK_USER_GROUP",
    UPDATE_AN_EXISTING_SLACK_VIEW: "SLACKBOT_UPDATE_AN_EXISTING_SLACK_VIEW",
    UPDATE_SLACK_CALL_INFORMATION: "SLACKBOT_UPDATE_SLACK_CALL_INFORMATION",
    UPDATE_USER_GROUP_MEMBERS: "SLACKBOT_UPDATE_USER_GROUP_MEMBERS",
    UPDATE_WORKFLOW_EXTENSION_STEP_CONFIGURATION: "SLACKBOT_UPDATE_WORKFLOW_EXTENSION_STEP_CONFIGURATION",
    UPLOAD_OR_CREATE_A_FILE_IN_SLACK: "SLACKBOT_UPLOAD_OR_CREATE_A_FILE_IN_SLACK",
    USERS_INFO: "SLACKBOT_USERS_INFO",
    USERS_LIST: "SLACKBOT_USERS_LIST",
    USERS_LOOKUP_BY_EMAIL: "SLACKBOT_USERS_LOOKUP_BY_EMAIL",
    USERS_PROFILE_GET_PROFILE_INFO: "SLACKBOT_USERS_PROFILE_GET_PROFILE_INFO",
    WIPE_USER_SESSIONS_ACROSS_DEVICES: "SLACKBOT_WIPE_USER_SESSIONS_ACROSS_DEVICES",
  },
  triggerTypes: {
    CHANNEL_CREATED: "SLACKBOT_CHANNEL_CREATED",
    REACTION_ADDED: "SLACKBOT_REACTION_ADDED",
    REACTION_REMOVED: "SLACKBOT_REACTION_REMOVED",
    RECEIVE_BOT_MESSAGE: "SLACKBOT_RECEIVE_BOT_MESSAGE",
    RECEIVE_DIRECT_MESSAGE: "SLACKBOT_RECEIVE_DIRECT_MESSAGE",
    RECEIVE_GROUP_MESSAGE: "SLACKBOT_RECEIVE_GROUP_MESSAGE",
    RECEIVE_MESSAGE: "SLACKBOT_RECEIVE_MESSAGE",
    RECEIVE_MPIM_MESSAGE: "SLACKBOT_RECEIVE_MPIM_MESSAGE",
    RECEIVE_THREAD_REPLY: "SLACKBOT_RECEIVE_THREAD_REPLY",
  },
}
