// @composio/core/generated/stripe.ts
/**
 * Map of Composio's STRIPE toolkit.
 */
export const STRIPE = {
  slug: "stripe",
  tools: {
    CANCEL_SUBSCRIPTION: "STRIPE_CANCEL_SUBSCRIPTION",
    CONFIRM_PAYMENT_INTENT: "STRIPE_CONFIRM_PAYMENT_INTENT",
    CREATE_CUSTOMER: "STRIPE_CREATE_CUSTOMER",
    CREATE_INVOICE: "STRIPE_CREATE_INVOICE",
    CREATE_PAYMENT_INTENT: "STRIPE_CREATE_PAYMENT_INTENT",
    CREATE_PRICE: "STRIPE_CREATE_PRICE",
    CREATE_PRODUCT: "STRIPE_CREATE_PRODUCT",
    CREATE_REFUND: "STRIPE_CREATE_REFUND",
    CREATE_SUBSCRIPTION: "STRIPE_CREATE_SUBSCRIPTION",
    DELETE_CUSTOMER: "STRIPE_DELETE_CUSTOMER",
    LIST_CHARGES: "STRIPE_LIST_CHARGES",
    LIST_COUPONS: "STRIPE_LIST_COUPONS",
    LIST_CUSTOMERS: "STRIPE_LIST_CUSTOMERS",
    LIST_CUSTOMER_PAYMENT_METHODS: "STRIPE_LIST_CUSTOMER_PAYMENT_METHODS",
    LIST_INVOICES: "STRIPE_LIST_INVOICES",
    LIST_PAYMENT_INTENTS: "STRIPE_LIST_PAYMENT_INTENTS",
    LIST_PAYMENT_LINKS: "STRIPE_LIST_PAYMENT_LINKS",
    LIST_PRODUCTS: "STRIPE_LIST_PRODUCTS",
    LIST_REFUNDS: "STRIPE_LIST_REFUNDS",
    LIST_SHIPPING_RATES: "STRIPE_LIST_SHIPPING_RATES",
    LIST_SUBSCRIPTIONS: "STRIPE_LIST_SUBSCRIPTIONS",
    LIST_TAX_CODES: "STRIPE_LIST_TAX_CODES",
    LIST_TAX_RATES: "STRIPE_LIST_TAX_RATES",
    RETRIEVE_BALANCE: "STRIPE_RETRIEVE_BALANCE",
    RETRIEVE_CHARGE: "STRIPE_RETRIEVE_CHARGE",
    RETRIEVE_CUSTOMER: "STRIPE_RETRIEVE_CUSTOMER",
    RETRIEVE_PAYMENT_INTENT: "STRIPE_RETRIEVE_PAYMENT_INTENT",
    RETRIEVE_REFUND: "STRIPE_RETRIEVE_REFUND",
    RETRIEVE_SUBSCRIPTION: "STRIPE_RETRIEVE_SUBSCRIPTION",
    SEARCH_CUSTOMERS: "STRIPE_SEARCH_CUSTOMERS",
    UPDATE_CUSTOMER: "STRIPE_UPDATE_CUSTOMER",
    UPDATE_PAYMENT_INTENT: "STRIPE_UPDATE_PAYMENT_INTENT",
    UPDATE_SUBSCRIPTION: "STRIPE_UPDATE_SUBSCRIPTION",
  },
  triggerTypes: {
    STRIPE_CHARGE_FAILED_TRIGGER: "STRIPE_STRIPE_CHARGE_FAILED_TRIGGER",
    STRIPE_CHECKOUT_SESSION_COMPLETED_TRIGGER: "STRIPE_STRIPE_CHECKOUT_SESSION_COMPLETED_TRIGGER",
    STRIPE_INVOICE_PAYMENT_SUCCEEDED_TRIGGER: "STRIPE_STRIPE_INVOICE_PAYMENT_SUCCEEDED_TRIGGER",
    STRIPE_PAYMENT_FAILED_TRIGGER: "STRIPE_STRIPE_PAYMENT_FAILED_TRIGGER",
    STRIPE_PRODUCT_CREATED_TRIGGER: "STRIPE_STRIPE_PRODUCT_CREATED_TRIGGER",
    STRIPE_SUBSCRIPTION_ADDED_TRIGGER: "STRIPE_STRIPE_SUBSCRIPTION_ADDED_TRIGGER",
    STRIPE_SUBSCRIPTION_DELETED_TRIGGER: "STRIPE_STRIPE_SUBSCRIPTION_DELETED_TRIGGER",
  },
}
