// @composio/core/generated/supabase.ts
/**
 * Map of Composio's SUPABASE toolkit.
 */
export const SUPABASE = {
  slug: "supabase",
  tools: {
    ALPHA_CREATES_A_NEW_API_KEY_FOR_THE_PROJECT: "SUPABASE_ALPHA_CREATES_A_NEW_API_KEY_FOR_THE_PROJECT",
    ALPHA_DELETES_AN_API_KEY_FOR_THE_PROJECT: "SUPABASE_ALPHA_DELETES_AN_API_KEY_FOR_THE_PROJECT",
    ALPHA_GET_A_THIRD_PARTY_INTEGRATION: "SUPABASE_ALPHA_GET_A_THIRD_PARTY_INTEGRATION",
    ALPHA_LISTS_ALL_THIRD_PARTY_AUTH_INTEGRATIONS: "SUPABASE_ALPHA_LISTS_ALL_THIRD_PARTY_AUTH_INTEGRATIONS",
    ALPHA_REMOVES_A_THIRD_PARTY_AUTH_INTEGRATION: "SUPABASE_ALPHA_REMOVES_A_THIRD_PARTY_AUTH_INTEGRATION",
    ALPHA_UPDATES_AN_API_KEY_FOR_THE_PROJECT: "SUPABASE_ALPHA_UPDATES_AN_API_KEY_FOR_THE_PROJECT",
    BETA_ACTIVATES_A_CUSTOM_HOSTNAME_FOR_A_PROJECT: "SUPABASE_BETA_ACTIVATES_A_CUSTOM_HOSTNAME_FOR_A_PROJECT",
    BETA_ACTIVATES_A_VANITY_SUBDOMAIN_FOR_A_PROJECT: "SUPABASE_BETA_ACTIVATES_A_VANITY_SUBDOMAIN_FOR_A_PROJECT",
    BETA_AUTHORIZE_USER_THROUGH_OAUTH: "SUPABASE_BETA_AUTHORIZE_USER_THROUGH_OAUTH",
    BETA_CHECKS_VANITY_SUBDOMAIN_AVAILABILITY: "SUPABASE_BETA_CHECKS_VANITY_SUBDOMAIN_AVAILABILITY",
    BETA_ENABLES_DATABASE_WEBHOOKS_ON_THE_PROJECT: "SUPABASE_BETA_ENABLES_DATABASE_WEBHOOKS_ON_THE_PROJECT",
    BETA_GETS_CURRENT_VANITY_SUBDOMAIN_CONFIG: "SUPABASE_BETA_GETS_CURRENT_VANITY_SUBDOMAIN_CONFIG",
    BETA_GETS_PROJECT_S_CUSTOM_HOSTNAME_CONFIG: "SUPABASE_BETA_GETS_PROJECT_S_CUSTOM_HOSTNAME_CONFIG",
    BETA_GETS_PROJECT_S_NETWORK_BANS: "SUPABASE_BETA_GETS_PROJECT_S_NETWORK_BANS",
    BETA_GETS_PROJECT_S_NETWORK_RESTRICTIONS: "SUPABASE_BETA_GETS_PROJECT_S_NETWORK_RESTRICTIONS",
    BETA_GETS_PROJECT_S_PGSODIUM_CONFIG: "SUPABASE_BETA_GETS_PROJECT_S_PGSODIUM_CONFIG",
    BETA_GET_PROJECT_S_SSL_ENFORCEMENT_CONFIGURATION: "SUPABASE_BETA_GET_PROJECT_S_SSL_ENFORCEMENT_CONFIGURATION",
    BETA_REMOVE_A_READ_REPLICA: "SUPABASE_BETA_REMOVE_A_READ_REPLICA",
    BETA_REMOVE_NETWORK_BANS: "SUPABASE_BETA_REMOVE_NETWORK_BANS",
    BETA_RUN_SQL_QUERY: "SUPABASE_BETA_RUN_SQL_QUERY",
    BETA_SET_UP_A_READ_REPLICA: "SUPABASE_BETA_SET_UP_A_READ_REPLICA",
    BETA_UPDATES_PROJECT_S_NETWORK_RESTRICTIONS: "SUPABASE_BETA_UPDATES_PROJECT_S_NETWORK_RESTRICTIONS",
    BETA_UPGRADES_THE_PROJECT_S_POSTGRES_VERSION: "SUPABASE_BETA_UPGRADES_THE_PROJECT_S_POSTGRES_VERSION",
    CONFIG_PGSODIUM_UPDATE_WITH_ROOT_KEY_WARNING: "SUPABASE_CONFIG_PGSODIUM_UPDATE_WITH_ROOT_KEY_WARNING",
    CREATES_A_NEW_SSO_PROVIDER: "SUPABASE_CREATES_A_NEW_SSO_PROVIDER",
    CREATES_A_NEW_THIRD_PARTY_AUTH_INTEGRATION: "SUPABASE_CREATES_A_NEW_THIRD_PARTY_AUTH_INTEGRATION",
    CREATE_AN_ORGANIZATION: "SUPABASE_CREATE_AN_ORGANIZATION",
    CREATE_A_DATABASE_BRANCH: "SUPABASE_CREATE_A_DATABASE_BRANCH",
    CREATE_A_FUNCTION: "SUPABASE_CREATE_A_FUNCTION",
    CREATE_A_PROJECT: "SUPABASE_CREATE_A_PROJECT",
    CUSTOM_HOSTNAME_DNS_VERIFICATION: "SUPABASE_CUSTOM_HOSTNAME_DNS_VERIFICATION",
    DELETES_THE_GIVEN_PROJECT: "SUPABASE_DELETES_THE_GIVEN_PROJECT",
    DELETE_A_DATABASE_BRANCH: "SUPABASE_DELETE_A_DATABASE_BRANCH",
    DELETE_A_FUNCTION: "SUPABASE_DELETE_A_FUNCTION",
    DELETE_CUSTOM_HOSTNAME_CONFIG: "SUPABASE_DELETE_CUSTOM_HOSTNAME_CONFIG",
    DELETE_PROJECTS_VANITY_SUBDOMAIN: "SUPABASE_DELETE_PROJECTS_VANITY_SUBDOMAIN",
    DISABLES_PREVIEW_BRANCHING: "SUPABASE_DISABLES_PREVIEW_BRANCHING",
    DISABLE_PROJECT_READONLY_MODE: "SUPABASE_DISABLE_PROJECT_READONLY_MODE",
    EXCHANGE_AUTH_CODE: "SUPABASE_EXCHANGE_AUTH_CODE",
    GENERATE_TYPE_SCRIPT_TYPES: "SUPABASE_GENERATE_TYPE_SCRIPT_TYPES",
    GETS_A_SPECIFIC_SQL_SNIPPET: "SUPABASE_GETS_A_SPECIFIC_SQL_SNIPPET",
    GETS_A_SSO_PROVIDER_BY_ITS_UUID: "SUPABASE_GETS_A_SSO_PROVIDER_BY_ITS_UUID",
    GETS_INFORMATION_ABOUT_THE_ORGANIZATION: "SUPABASE_GETS_INFORMATION_ABOUT_THE_ORGANIZATION",
    GETS_PROJECT_S_AUTH_CONFIG: "SUPABASE_GETS_PROJECT_S_AUTH_CONFIG",
    GETS_PROJECT_S_POSTGREST_CONFIG: "SUPABASE_GETS_PROJECT_S_POSTGREST_CONFIG",
    GETS_PROJECT_S_POSTGRES_CONFIG: "SUPABASE_GETS_PROJECT_S_POSTGRES_CONFIG",
    GETS_PROJECT_S_SERVICE_HEALTH_STATUS: "SUPABASE_GETS_PROJECT_S_SERVICE_HEALTH_STATUS",
    GETS_PROJECT_S_SUPAVISOR_CONFIG: "SUPABASE_GETS_PROJECT_S_SUPAVISOR_CONFIG",
    GET_DATABASE_BRANCH_CONFIG: "SUPABASE_GET_DATABASE_BRANCH_CONFIG",
    GET_PROJECT_API_KEYS: "SUPABASE_GET_PROJECT_API_KEYS",
    GET_PROJECT_S_PGBOUNCER_CONFIG: "SUPABASE_GET_PROJECT_S_PGBOUNCER_CONFIG",
    GET_PROJECT_UPGRADE_ELIGIBILITY: "SUPABASE_GET_PROJECT_UPGRADE_ELIGIBILITY",
    GET_PROJECT_UPGRADE_STATUS: "SUPABASE_GET_PROJECT_UPGRADE_STATUS",
    LISTS_ALL_BACKUPS: "SUPABASE_LISTS_ALL_BACKUPS",
    LISTS_ALL_BUCKETS: "SUPABASE_LISTS_ALL_BUCKETS",
    LISTS_ALL_SSO_PROVIDERS: "SUPABASE_LISTS_ALL_SSO_PROVIDERS",
    LISTS_SQL_SNIPPETS_FOR_THE_LOGGED_IN_USER: "SUPABASE_LISTS_SQL_SNIPPETS_FOR_THE_LOGGED_IN_USER",
    LIST_ALL_DATABASE_BRANCHES: "SUPABASE_LIST_ALL_DATABASE_BRANCHES",
    LIST_ALL_FUNCTIONS: "SUPABASE_LIST_ALL_FUNCTIONS",
    LIST_ALL_ORGANIZATIONS: "SUPABASE_LIST_ALL_ORGANIZATIONS",
    LIST_ALL_PROJECTS: "SUPABASE_LIST_ALL_PROJECTS",
    LIST_ALL_SECRETS: "SUPABASE_LIST_ALL_SECRETS",
    LIST_MEMBERS_OF_AN_ORGANIZATION: "SUPABASE_LIST_MEMBERS_OF_AN_ORGANIZATION",
    REMOVES_A_SSO_PROVIDER_BY_ITS_UUID: "SUPABASE_REMOVES_A_SSO_PROVIDER_BY_ITS_UUID",
    RESETS_A_DATABASE_BRANCH: "SUPABASE_RESETS_A_DATABASE_BRANCH",
    RESTORES_A_PITR_BACKUP_FOR_A_DATABASE: "SUPABASE_RESTORES_A_PITR_BACKUP_FOR_A_DATABASE",
    RETRIEVE_A_FUNCTION: "SUPABASE_RETRIEVE_A_FUNCTION",
    RETRIEVE_A_FUNCTION_BODY: "SUPABASE_RETRIEVE_A_FUNCTION_BODY",
    RETURNS_PROJECT_S_READONLY_MODE_STATUS: "SUPABASE_RETURNS_PROJECT_S_READONLY_MODE_STATUS",
    UPDATES_A_SSO_PROVIDER_BY_ITS_UUID: "SUPABASE_UPDATES_A_SSO_PROVIDER_BY_ITS_UUID",
    UPDATES_PROJECT_S_POSTGREST_CONFIG: "SUPABASE_UPDATES_PROJECT_S_POSTGREST_CONFIG",
    UPDATES_PROJECT_S_POSTGRES_CONFIG: "SUPABASE_UPDATES_PROJECT_S_POSTGRES_CONFIG",
    UPDATES_PROJECT_S_SUPAVISOR_CONFIG: "SUPABASE_UPDATES_PROJECT_S_SUPAVISOR_CONFIG",
    UPDATE_A_FUNCTION: "SUPABASE_UPDATE_A_FUNCTION",
    UPDATE_CUSTOM_HOSTNAME: "SUPABASE_UPDATE_CUSTOM_HOSTNAME",
    UPDATE_DATABASE_BRANCH_CONFIG: "SUPABASE_UPDATE_DATABASE_BRANCH_CONFIG",
    UPDATE_SSL_ENFORCEMENT_CONFIG: "SUPABASE_UPDATE_SSL_ENFORCEMENT_CONFIG",
  },
  triggerTypes: {},
}
