// @composio/core/generated/trello.ts
/**
 * Map of Composio's TRELLO toolkit.
 */
export const TRELLO = {
  slug: "trello",
  tools: {
    ACTION_GET_BOARD_BY_ID_ACTION: "TRELLO_ACTION_GET_BOARD_BY_ID_ACTION",
    ACTION_GET_BY_ID: "TRELLO_ACTION_GET_BY_ID",
    ACTION_GET_LIST_BY_ID_ACTION: "TRELLO_ACTION_GET_LIST_BY_ID_ACTION",
    ADD_BOARDS: "TRELLO_ADD_BOARDS",
    ADD_BOARDS_CALENDAR_KEY_GENERATE_BY_ID_BOARD: "TRELLO_ADD_BOARDS_CALENDAR_KEY_GENERATE_BY_ID_BOARD",
    ADD_BOARDS_CHECKLISTS_BY_ID_BOARD: "TRELLO_ADD_BOARDS_CHECKLISTS_BY_ID_BOARD",
    ADD_BOARDS_EMAIL_KEY_GENERATE_BY_ID_BOARD: "TRELLO_ADD_BOARDS_EMAIL_KEY_GENERATE_BY_ID_BOARD",
    ADD_BOARDS_LABELS_BY_ID_BOARD: "TRELLO_ADD_BOARDS_LABELS_BY_ID_BOARD",
    ADD_BOARDS_LISTS_BY_ID_BOARD: "TRELLO_ADD_BOARDS_LISTS_BY_ID_BOARD",
    ADD_BOARDS_MARK_AS_VIEWED_BY_ID_BOARD: "TRELLO_ADD_BOARDS_MARK_AS_VIEWED_BY_ID_BOARD",
    ADD_BOARDS_POWER_UPS_BY_ID_BOARD: "TRELLO_ADD_BOARDS_POWER_UPS_BY_ID_BOARD",
    ADD_CARDS: "TRELLO_ADD_CARDS",
    ADD_CARDS_ACTIONS_COMMENTS_BY_ID_CARD: "TRELLO_ADD_CARDS_ACTIONS_COMMENTS_BY_ID_CARD",
    ADD_CARDS_ATTACHMENTS_BY_ID_CARD: "TRELLO_ADD_CARDS_ATTACHMENTS_BY_ID_CARD",
    ADD_CARDS_CHECKLISTS_BY_ID_CARD: "TRELLO_ADD_CARDS_CHECKLISTS_BY_ID_CARD",
    ADD_CARDS_ID_LABELS_BY_ID_CARD: "TRELLO_ADD_CARDS_ID_LABELS_BY_ID_CARD",
    ADD_CARDS_ID_MEMBERS_BY_ID_CARD: "TRELLO_ADD_CARDS_ID_MEMBERS_BY_ID_CARD",
    ADD_CARDS_LABELS_BY_ID_CARD: "TRELLO_ADD_CARDS_LABELS_BY_ID_CARD",
    ADD_CARDS_MEMBERS_VOTED_BY_ID_CARD: "TRELLO_ADD_CARDS_MEMBERS_VOTED_BY_ID_CARD",
    ADD_CARDS_STICKERS_BY_ID_CARD: "TRELLO_ADD_CARDS_STICKERS_BY_ID_CARD",
    ADD_CHECKLISTS: "TRELLO_ADD_CHECKLISTS",
    ADD_CHECKLISTS_CHECK_ITEMS_BY_ID_CHECKLIST: "TRELLO_ADD_CHECKLISTS_CHECK_ITEMS_BY_ID_CHECKLIST",
    ADD_CHECKLIST_ITEM: "TRELLO_ADD_CHECKLIST_ITEM",
    ADD_LABELS: "TRELLO_ADD_LABELS",
    ADD_LISTS: "TRELLO_ADD_LISTS",
    ADD_LISTS_ARCHIVE_ALL_CARDS_BY_ID_LIST: "TRELLO_ADD_LISTS_ARCHIVE_ALL_CARDS_BY_ID_LIST",
    ADD_LISTS_CARDS_BY_ID_LIST: "TRELLO_ADD_LISTS_CARDS_BY_ID_LIST",
    ADD_LISTS_MOVE_ALL_CARDS_BY_ID_LIST: "TRELLO_ADD_LISTS_MOVE_ALL_CARDS_BY_ID_LIST",
    ADD_MEMBERS_AVATAR_BY_ID_MEMBER: "TRELLO_ADD_MEMBERS_AVATAR_BY_ID_MEMBER",
    ADD_MEMBERS_BOARD_BACKGROUNDS_BY_ID_MEMBER: "TRELLO_ADD_MEMBERS_BOARD_BACKGROUNDS_BY_ID_MEMBER",
    ADD_MEMBERS_BOARD_STARS_BY_ID_MEMBER: "TRELLO_ADD_MEMBERS_BOARD_STARS_BY_ID_MEMBER",
    ADD_MEMBERS_CUSTOM_BOARD_BACKGROUNDS_BY_ID_MEMBER: "TRELLO_ADD_MEMBERS_CUSTOM_BOARD_BACKGROUNDS_BY_ID_MEMBER",
    ADD_MEMBERS_CUSTOM_EMOJI_BY_ID_MEMBER: "TRELLO_ADD_MEMBERS_CUSTOM_EMOJI_BY_ID_MEMBER",
    ADD_MEMBERS_CUSTOM_STICKERS_BY_ID_MEMBER: "TRELLO_ADD_MEMBERS_CUSTOM_STICKERS_BY_ID_MEMBER",
    ADD_MEMBERS_SAVED_SEARCHES_BY_ID_MEMBER: "TRELLO_ADD_MEMBERS_SAVED_SEARCHES_BY_ID_MEMBER",
    ADD_NOTIFICATIONS_ALL_READ: "TRELLO_ADD_NOTIFICATIONS_ALL_READ",
    ADD_ORGANIZATIONS: "TRELLO_ADD_ORGANIZATIONS",
    ADD_ORGANIZATIONS_LOGO_BY_ID_ORG: "TRELLO_ADD_ORGANIZATIONS_LOGO_BY_ID_ORG",
    ADD_SESSIONS: "TRELLO_ADD_SESSIONS",
    ADD_TOKENS_WEBHOOKS_BY_TOKEN: "TRELLO_ADD_TOKENS_WEBHOOKS_BY_TOKEN",
    BOARD_CREATE_BOARD: "TRELLO_BOARD_CREATE_BOARD",
    BOARD_FILTER_CARDS_BY_ID_BOARD: "TRELLO_BOARD_FILTER_CARDS_BY_ID_BOARD",
    BOARD_GET_BY_ID: "TRELLO_BOARD_GET_BY_ID",
    BOARD_GET_CARDS_BY_ID_BOARD: "TRELLO_BOARD_GET_CARDS_BY_ID_BOARD",
    BOARD_GET_LISTS_BY_ID_BOARD: "TRELLO_BOARD_GET_LISTS_BY_ID_BOARD",
    CARD_CREATE_AND_UPDATE: "TRELLO_CARD_CREATE_AND_UPDATE",
    CARD_GET_BY_ID: "TRELLO_CARD_GET_BY_ID",
    CARD_GET_BY_ID_FIELD: "TRELLO_CARD_GET_BY_ID_FIELD",
    CARD_UPDATE_BY_ID_CARD: "TRELLO_CARD_UPDATE_BY_ID_CARD",
    CARD_UPDATE_ID_LIST_BY_ID_CARD: "TRELLO_CARD_UPDATE_ID_LIST_BY_ID_CARD",
    CARD_UPDATE_POS_BY_ID_CARD: "TRELLO_CARD_UPDATE_POS_BY_ID_CARD",
    CONVERT_CHECKLIST_ITEM_TO_CARD: "TRELLO_CONVERT_CHECKLIST_ITEM_TO_CARD",
    DELETE_ACTIONS_BY_ID_ACTION: "TRELLO_DELETE_ACTIONS_BY_ID_ACTION",
    DELETE_BOARDS_MEMBERS_BY_ID_BOARD_BY_ID_MEMBER: "TRELLO_DELETE_BOARDS_MEMBERS_BY_ID_BOARD_BY_ID_MEMBER",
    DELETE_BOARDS_POWER_UPS_BY_ID_BOARD_BY_POWER_UP: "TRELLO_DELETE_BOARDS_POWER_UPS_BY_ID_BOARD_BY_POWER_UP",
    DELETE_CARDS_ACTIONS_COMMENTS_BY_ID_CARD_BY_ID_ACTION: "TRELLO_DELETE_CARDS_ACTIONS_COMMENTS_BY_ID_CARD_BY_ID_ACTION",
    DELETE_CARDS_ATTACHMENTS_BY_ID_CARD_BY_ID_ATTACHMENT: "TRELLO_DELETE_CARDS_ATTACHMENTS_BY_ID_CARD_BY_ID_ATTACHMENT",
    DELETE_CARDS_BY_ID_CARD: "TRELLO_DELETE_CARDS_BY_ID_CARD",
    DELETE_CARDS_CHECKLISTS_BY_ID_CARD_BY_ID_CHECKLIST: "TRELLO_DELETE_CARDS_CHECKLISTS_BY_ID_CARD_BY_ID_CHECKLIST",
    DELETE_CARDS_ID_LABELS_BY_ID_CARD_BY_ID_LABEL: "TRELLO_DELETE_CARDS_ID_LABELS_BY_ID_CARD_BY_ID_LABEL",
    DELETE_CARDS_ID_MEMBERS_BY_ID_CARD_BY_ID_MEMBER: "TRELLO_DELETE_CARDS_ID_MEMBERS_BY_ID_CARD_BY_ID_MEMBER",
    DELETE_CARDS_LABELS_BY_ID_CARD_BY_COLOR: "TRELLO_DELETE_CARDS_LABELS_BY_ID_CARD_BY_COLOR",
    DELETE_CARDS_MEMBERS_VOTED_BY_ID_CARD_BY_ID_MEMBER: "TRELLO_DELETE_CARDS_MEMBERS_VOTED_BY_ID_CARD_BY_ID_MEMBER",
    DELETE_CARDS_STICKERS_BY_ID_CARD_BY_ID_STICKER: "TRELLO_DELETE_CARDS_STICKERS_BY_ID_CARD_BY_ID_STICKER",
    DELETE_CHECKLISTS_BY_ID_CHECKLIST: "TRELLO_DELETE_CHECKLISTS_BY_ID_CHECKLIST",
    DELETE_CHECKLIST_ITEM: "TRELLO_DELETE_CHECKLIST_ITEM",
    DELETE_LABELS_BY_ID_LABEL: "TRELLO_DELETE_LABELS_BY_ID_LABEL",
    DELETE_MEMBER_BOARD_BACKGROUND: "TRELLO_DELETE_MEMBER_BOARD_BACKGROUND",
    DELETE_MEMBER_CUSTOM_STICKER: "TRELLO_DELETE_MEMBER_CUSTOM_STICKER",
    DELETE_MEMBER_SAVED_SEARCH: "TRELLO_DELETE_MEMBER_SAVED_SEARCH",
    DELETE_ORGANIZATIONS_BY_ID_ORG: "TRELLO_DELETE_ORGANIZATIONS_BY_ID_ORG",
    DELETE_ORGANIZATIONS_LOGO_BY_ID_ORG: "TRELLO_DELETE_ORGANIZATIONS_LOGO_BY_ID_ORG",
    DELETE_ORGANIZATIONS_MEMBERS_BY_ID_ORG_BY_ID_MEMBER: "TRELLO_DELETE_ORGANIZATIONS_MEMBERS_BY_ID_ORG_BY_ID_MEMBER",
    DELETE_ORG_ASSOCIATED_DOMAIN: "TRELLO_DELETE_ORG_ASSOCIATED_DOMAIN",
    DELETE_ORG_INVITE_RESTRICT: "TRELLO_DELETE_ORG_INVITE_RESTRICT",
    DELETE_TOKENS_BY_TOKEN: "TRELLO_DELETE_TOKENS_BY_TOKEN",
    DELETE_TOKENS_WEBHOOKS_BY_TOKEN_BY_ID_WEBHOOK: "TRELLO_DELETE_TOKENS_WEBHOOKS_BY_TOKEN_BY_ID_WEBHOOK",
    DELETE_WEBHOOKS_BY_ID_WEBHOOK: "TRELLO_DELETE_WEBHOOKS_BY_ID_WEBHOOK",
    DISMISS_MEMBER_MESSAGE: "TRELLO_DISMISS_MEMBER_MESSAGE",
    GET_ACTIONS_BOARD_BY_ID_ACTION: "TRELLO_GET_ACTIONS_BOARD_BY_ID_ACTION",
    GET_ACTIONS_BOARD_BY_ID_ACTION_BY_FIELD: "TRELLO_GET_ACTIONS_BOARD_BY_ID_ACTION_BY_FIELD",
    GET_ACTIONS_BY_ID_ACTION: "TRELLO_GET_ACTIONS_BY_ID_ACTION",
    GET_ACTIONS_BY_ID_ACTION_BY_FIELD: "TRELLO_GET_ACTIONS_BY_ID_ACTION_BY_FIELD",
    GET_ACTIONS_CARD_BY_ID_ACTION: "TRELLO_GET_ACTIONS_CARD_BY_ID_ACTION",
    GET_ACTIONS_CARD_BY_ID_ACTION_BY_FIELD: "TRELLO_GET_ACTIONS_CARD_BY_ID_ACTION_BY_FIELD",
    GET_ACTIONS_DISPLAY_BY_ID_ACTION: "TRELLO_GET_ACTIONS_DISPLAY_BY_ID_ACTION",
    GET_ACTIONS_ENTITIES_BY_ID_ACTION: "TRELLO_GET_ACTIONS_ENTITIES_BY_ID_ACTION",
    GET_ACTIONS_LIST_BY_ID_ACTION: "TRELLO_GET_ACTIONS_LIST_BY_ID_ACTION",
    GET_ACTIONS_LIST_BY_ID_ACTION_BY_FIELD: "TRELLO_GET_ACTIONS_LIST_BY_ID_ACTION_BY_FIELD",
    GET_ACTIONS_MEMBER_BY_ID_ACTION: "TRELLO_GET_ACTIONS_MEMBER_BY_ID_ACTION",
    GET_ACTIONS_MEMBER_BY_ID_ACTION_BY_FIELD: "TRELLO_GET_ACTIONS_MEMBER_BY_ID_ACTION_BY_FIELD",
    GET_ACTIONS_MEMBER_CREATOR_BY_ID_ACTION: "TRELLO_GET_ACTIONS_MEMBER_CREATOR_BY_ID_ACTION",
    GET_ACTIONS_MEMBER_CREATOR_BY_ID_ACTION_BY_FIELD: "TRELLO_GET_ACTIONS_MEMBER_CREATOR_BY_ID_ACTION_BY_FIELD",
    GET_ACTIONS_ORGANIZATION_BY_ID_ACTION: "TRELLO_GET_ACTIONS_ORGANIZATION_BY_ID_ACTION",
    GET_ACTIONS_ORGANIZATION_BY_ID_ACTION_BY_FIELD: "TRELLO_GET_ACTIONS_ORGANIZATION_BY_ID_ACTION_BY_FIELD",
    GET_BATCH: "TRELLO_GET_BATCH",
    GET_BOARDS_ACTIONS_BY_ID_BOARD: "TRELLO_GET_BOARDS_ACTIONS_BY_ID_BOARD",
    GET_BOARDS_BOARD_STARS_BY_ID_BOARD: "TRELLO_GET_BOARDS_BOARD_STARS_BY_ID_BOARD",
    GET_BOARDS_BY_ID_BOARD: "TRELLO_GET_BOARDS_BY_ID_BOARD",
    GET_BOARDS_BY_ID_BOARD_BY_FIELD: "TRELLO_GET_BOARDS_BY_ID_BOARD_BY_FIELD",
    GET_BOARDS_CARDS_BY_ID_BOARD: "TRELLO_GET_BOARDS_CARDS_BY_ID_BOARD",
    GET_BOARDS_CARDS_BY_ID_BOARD_BY_FILTER: "TRELLO_GET_BOARDS_CARDS_BY_ID_BOARD_BY_FILTER",
    GET_BOARDS_CARDS_BY_ID_BOARD_BY_ID_CARD: "TRELLO_GET_BOARDS_CARDS_BY_ID_BOARD_BY_ID_CARD",
    GET_BOARDS_CHECKLISTS_BY_ID_BOARD: "TRELLO_GET_BOARDS_CHECKLISTS_BY_ID_BOARD",
    GET_BOARDS_DELTAS_BY_ID_BOARD: "TRELLO_GET_BOARDS_DELTAS_BY_ID_BOARD",
    GET_BOARDS_LABELS_BY_ID_BOARD: "TRELLO_GET_BOARDS_LABELS_BY_ID_BOARD",
    GET_BOARDS_LABELS_BY_ID_BOARD_BY_ID_LABEL: "TRELLO_GET_BOARDS_LABELS_BY_ID_BOARD_BY_ID_LABEL",
    GET_BOARDS_LISTS_BY_ID_BOARD: "TRELLO_GET_BOARDS_LISTS_BY_ID_BOARD",
    GET_BOARDS_LISTS_BY_ID_BOARD_BY_FILTER: "TRELLO_GET_BOARDS_LISTS_BY_ID_BOARD_BY_FILTER",
    GET_BOARDS_MEMBERSHIPS_BY_ID_BOARD: "TRELLO_GET_BOARDS_MEMBERSHIPS_BY_ID_BOARD",
    GET_BOARDS_MEMBERSHIPS_BY_ID_BOARD_BY_ID_MEMBERSHIP: "TRELLO_GET_BOARDS_MEMBERSHIPS_BY_ID_BOARD_BY_ID_MEMBERSHIP",
    GET_BOARDS_MEMBERS_BY_ID_BOARD: "TRELLO_GET_BOARDS_MEMBERS_BY_ID_BOARD",
    GET_BOARDS_MEMBERS_BY_ID_BOARD_BY_FILTER: "TRELLO_GET_BOARDS_MEMBERS_BY_ID_BOARD_BY_FILTER",
    GET_BOARDS_MEMBERS_CARDS_BY_ID_BOARD_BY_ID_MEMBER: "TRELLO_GET_BOARDS_MEMBERS_CARDS_BY_ID_BOARD_BY_ID_MEMBER",
    GET_BOARDS_MEMBERS_INVITED_BY_ID_BOARD: "TRELLO_GET_BOARDS_MEMBERS_INVITED_BY_ID_BOARD",
    GET_BOARDS_MEMBERS_INVITED_BY_ID_BOARD_BY_FIELD: "TRELLO_GET_BOARDS_MEMBERS_INVITED_BY_ID_BOARD_BY_FIELD",
    GET_BOARDS_MY_PREFS_BY_ID_BOARD: "TRELLO_GET_BOARDS_MY_PREFS_BY_ID_BOARD",
    GET_BOARDS_ORGANIZATION_BY_ID_BOARD: "TRELLO_GET_BOARDS_ORGANIZATION_BY_ID_BOARD",
    GET_BOARDS_ORGANIZATION_BY_ID_BOARD_BY_FIELD: "TRELLO_GET_BOARDS_ORGANIZATION_BY_ID_BOARD_BY_FIELD",
    GET_CARDS_ACTIONS_BY_ID_CARD: "TRELLO_GET_CARDS_ACTIONS_BY_ID_CARD",
    GET_CARDS_ATTACHMENTS_BY_ID_CARD: "TRELLO_GET_CARDS_ATTACHMENTS_BY_ID_CARD",
    GET_CARDS_ATTACHMENTS_BY_ID_CARD_BY_ID_ATTACHMENT: "TRELLO_GET_CARDS_ATTACHMENTS_BY_ID_CARD_BY_ID_ATTACHMENT",
    GET_CARDS_BOARD_BY_ID_CARD: "TRELLO_GET_CARDS_BOARD_BY_ID_CARD",
    GET_CARDS_BOARD_BY_ID_CARD_BY_FIELD: "TRELLO_GET_CARDS_BOARD_BY_ID_CARD_BY_FIELD",
    GET_CARDS_BY_ID_CARD: "TRELLO_GET_CARDS_BY_ID_CARD",
    GET_CARDS_BY_ID_CARD_BY_FIELD: "TRELLO_GET_CARDS_BY_ID_CARD_BY_FIELD",
    GET_CARDS_CHECKLISTS_BY_ID_CARD: "TRELLO_GET_CARDS_CHECKLISTS_BY_ID_CARD",
    GET_CARDS_CHECK_ITEM_STATES_BY_ID_CARD: "TRELLO_GET_CARDS_CHECK_ITEM_STATES_BY_ID_CARD",
    GET_CARDS_LIST_BY_ID_CARD: "TRELLO_GET_CARDS_LIST_BY_ID_CARD",
    GET_CARDS_LIST_BY_ID_CARD_BY_FIELD: "TRELLO_GET_CARDS_LIST_BY_ID_CARD_BY_FIELD",
    GET_CARDS_MEMBERS_BY_ID_CARD: "TRELLO_GET_CARDS_MEMBERS_BY_ID_CARD",
    GET_CARDS_MEMBERS_VOTED_BY_ID_CARD: "TRELLO_GET_CARDS_MEMBERS_VOTED_BY_ID_CARD",
    GET_CARDS_STICKERS_BY_ID_CARD: "TRELLO_GET_CARDS_STICKERS_BY_ID_CARD",
    GET_CARDS_STICKERS_BY_ID_CARD_BY_ID_STICKER: "TRELLO_GET_CARDS_STICKERS_BY_ID_CARD_BY_ID_STICKER",
    GET_CHECKLISTS_BOARD_BY_ID_CHECKLIST: "TRELLO_GET_CHECKLISTS_BOARD_BY_ID_CHECKLIST",
    GET_CHECKLISTS_BOARD_BY_ID_CHECKLIST_BY_FIELD: "TRELLO_GET_CHECKLISTS_BOARD_BY_ID_CHECKLIST_BY_FIELD",
    GET_CHECKLISTS_BY_ID_CHECKLIST: "TRELLO_GET_CHECKLISTS_BY_ID_CHECKLIST",
    GET_CHECKLISTS_BY_ID_CHECKLIST_BY_FIELD: "TRELLO_GET_CHECKLISTS_BY_ID_CHECKLIST_BY_FIELD",
    GET_CHECKLISTS_CARDS_BY_ID_CHECKLIST: "TRELLO_GET_CHECKLISTS_CARDS_BY_ID_CHECKLIST",
    GET_CHECKLISTS_CARDS_BY_ID_CHECKLIST_BY_FILTER: "TRELLO_GET_CHECKLISTS_CARDS_BY_ID_CHECKLIST_BY_FILTER",
    GET_CHECKLISTS_CHECK_ITEMS_BY_ID_CHECKLIST: "TRELLO_GET_CHECKLISTS_CHECK_ITEMS_BY_ID_CHECKLIST",
    GET_CHECK_ITEM_BY_ID: "TRELLO_GET_CHECK_ITEM_BY_ID",
    GET_LABELS_BOARD_BY_ID_LABEL: "TRELLO_GET_LABELS_BOARD_BY_ID_LABEL",
    GET_LABELS_BOARD_BY_ID_LABEL_BY_FIELD: "TRELLO_GET_LABELS_BOARD_BY_ID_LABEL_BY_FIELD",
    GET_LABELS_BY_ID_LABEL: "TRELLO_GET_LABELS_BY_ID_LABEL",
    GET_LISTS_ACTIONS_BY_ID_LIST: "TRELLO_GET_LISTS_ACTIONS_BY_ID_LIST",
    GET_LISTS_BOARD_BY_ID_LIST: "TRELLO_GET_LISTS_BOARD_BY_ID_LIST",
    GET_LISTS_BOARD_BY_ID_LIST_BY_FIELD: "TRELLO_GET_LISTS_BOARD_BY_ID_LIST_BY_FIELD",
    GET_LISTS_BY_ID_LIST: "TRELLO_GET_LISTS_BY_ID_LIST",
    GET_LISTS_BY_ID_LIST_BY_FIELD: "TRELLO_GET_LISTS_BY_ID_LIST_BY_FIELD",
    GET_LISTS_CARDS_BY_ID_LIST: "TRELLO_GET_LISTS_CARDS_BY_ID_LIST",
    GET_LISTS_CARDS_BY_ID_LIST_BY_FILTER: "TRELLO_GET_LISTS_CARDS_BY_ID_LIST_BY_FILTER",
    GET_MEMBERS_ACTIONS_BY_ID_MEMBER: "TRELLO_GET_MEMBERS_ACTIONS_BY_ID_MEMBER",
    GET_MEMBERS_BOARDS_BY_ID_MEMBER: "TRELLO_GET_MEMBERS_BOARDS_BY_ID_MEMBER",
    GET_MEMBERS_BOARDS_BY_ID_MEMBER_BY_FILTER: "TRELLO_GET_MEMBERS_BOARDS_BY_ID_MEMBER_BY_FILTER",
    GET_MEMBERS_BOARDS_INVITED_BY_ID_MEMBER: "TRELLO_GET_MEMBERS_BOARDS_INVITED_BY_ID_MEMBER",
    GET_MEMBERS_BOARDS_INVITED_BY_ID_MEMBER_BY_FIELD: "TRELLO_GET_MEMBERS_BOARDS_INVITED_BY_ID_MEMBER_BY_FIELD",
    GET_MEMBERS_BOARD_BACKGROUNDS_BY_ID_MEMBER: "TRELLO_GET_MEMBERS_BOARD_BACKGROUNDS_BY_ID_MEMBER",
    GET_MEMBERS_BOARD_STARS_BY_ID_MEMBER: "TRELLO_GET_MEMBERS_BOARD_STARS_BY_ID_MEMBER",
    GET_MEMBERS_BOARD_STARS_BY_ID_MEMBER_BY_ID_BOARD_STAR: "TRELLO_GET_MEMBERS_BOARD_STARS_BY_ID_MEMBER_BY_ID_BOARD_STAR",
    GET_MEMBERS_BY_ID_MEMBER: "TRELLO_GET_MEMBERS_BY_ID_MEMBER",
    GET_MEMBERS_BY_ID_MEMBER_BY_FIELD: "TRELLO_GET_MEMBERS_BY_ID_MEMBER_BY_FIELD",
    GET_MEMBERS_CARDS_BY_ID_MEMBER: "TRELLO_GET_MEMBERS_CARDS_BY_ID_MEMBER",
    GET_MEMBERS_CARDS_BY_ID_MEMBER_BY_FILTER: "TRELLO_GET_MEMBERS_CARDS_BY_ID_MEMBER_BY_FILTER",
    GET_MEMBERS_CUSTOM_BOARD_BACKGROUNDS_BY_ID_MEMBER: "TRELLO_GET_MEMBERS_CUSTOM_BOARD_BACKGROUNDS_BY_ID_MEMBER",
    GET_MEMBERS_CUSTOM_EMOJI_BY_ID_MEMBER: "TRELLO_GET_MEMBERS_CUSTOM_EMOJI_BY_ID_MEMBER",
    GET_MEMBERS_CUSTOM_STICKERS_BY_ID_MEMBER: "TRELLO_GET_MEMBERS_CUSTOM_STICKERS_BY_ID_MEMBER",
    GET_MEMBERS_DELTAS_BY_ID_MEMBER: "TRELLO_GET_MEMBERS_DELTAS_BY_ID_MEMBER",
    GET_MEMBERS_NOTIFICATIONS_BY_ID_MEMBER: "TRELLO_GET_MEMBERS_NOTIFICATIONS_BY_ID_MEMBER",
    GET_MEMBERS_NOTIFICATIONS_BY_ID_MEMBER_BY_FILTER: "TRELLO_GET_MEMBERS_NOTIFICATIONS_BY_ID_MEMBER_BY_FILTER",
    GET_MEMBERS_ORGANIZATIONS_BY_ID_MEMBER: "TRELLO_GET_MEMBERS_ORGANIZATIONS_BY_ID_MEMBER",
    GET_MEMBERS_ORGANIZATIONS_BY_ID_MEMBER_BY_FILTER: "TRELLO_GET_MEMBERS_ORGANIZATIONS_BY_ID_MEMBER_BY_FILTER",
    GET_MEMBERS_ORGANIZATIONS_INVITED_BY_ID_MEMBER: "TRELLO_GET_MEMBERS_ORGANIZATIONS_INVITED_BY_ID_MEMBER",
    GET_MEMBERS_SAVED_SEARCHES_BY_ID_MEMBER: "TRELLO_GET_MEMBERS_SAVED_SEARCHES_BY_ID_MEMBER",
    GET_MEMBERS_TOKENS_BY_ID_MEMBER: "TRELLO_GET_MEMBERS_TOKENS_BY_ID_MEMBER",
    GET_MEMBER_BOARD_BACKGROUND: "TRELLO_GET_MEMBER_BOARD_BACKGROUND",
    GET_MEMBER_CUSTOM_BOARD_BACKGROUND: "TRELLO_GET_MEMBER_CUSTOM_BOARD_BACKGROUND",
    GET_MEMBER_CUSTOM_EMOJI: "TRELLO_GET_MEMBER_CUSTOM_EMOJI",
    GET_MEMBER_CUSTOM_STICKER: "TRELLO_GET_MEMBER_CUSTOM_STICKER",
    GET_MEMBER_INVITED_ORG: "TRELLO_GET_MEMBER_INVITED_ORG",
    GET_MEMBER_ORG_CARDS: "TRELLO_GET_MEMBER_ORG_CARDS",
    GET_MEMBER_SAVED_SEARCH: "TRELLO_GET_MEMBER_SAVED_SEARCH",
    GET_NOTIFICATIONS_BOARD_BY_ID_NOTIFICATION: "TRELLO_GET_NOTIFICATIONS_BOARD_BY_ID_NOTIFICATION",
    GET_NOTIFICATIONS_BOARD_BY_ID_NOTIFICATION_BY_FIELD: "TRELLO_GET_NOTIFICATIONS_BOARD_BY_ID_NOTIFICATION_BY_FIELD",
    GET_NOTIFICATIONS_BY_ID_NOTIFICATION: "TRELLO_GET_NOTIFICATIONS_BY_ID_NOTIFICATION",
    GET_NOTIFICATIONS_BY_ID_NOTIFICATION_BY_FIELD: "TRELLO_GET_NOTIFICATIONS_BY_ID_NOTIFICATION_BY_FIELD",
    GET_NOTIFICATIONS_CARD_BY_ID_NOTIFICATION: "TRELLO_GET_NOTIFICATIONS_CARD_BY_ID_NOTIFICATION",
    GET_NOTIFICATIONS_CARD_BY_ID_NOTIFICATION_BY_FIELD: "TRELLO_GET_NOTIFICATIONS_CARD_BY_ID_NOTIFICATION_BY_FIELD",
    GET_NOTIFICATIONS_DISPLAY_BY_ID_NOTIFICATION: "TRELLO_GET_NOTIFICATIONS_DISPLAY_BY_ID_NOTIFICATION",
    GET_NOTIFICATIONS_ENTITIES_BY_ID_NOTIFICATION: "TRELLO_GET_NOTIFICATIONS_ENTITIES_BY_ID_NOTIFICATION",
    GET_NOTIFICATIONS_LIST_BY_ID_NOTIFICATION: "TRELLO_GET_NOTIFICATIONS_LIST_BY_ID_NOTIFICATION",
    GET_NOTIFICATIONS_LIST_BY_ID_NOTIFICATION_BY_FIELD: "TRELLO_GET_NOTIFICATIONS_LIST_BY_ID_NOTIFICATION_BY_FIELD",
    GET_NOTIFICATIONS_MEMBER_BY_ID_NOTIFICATION: "TRELLO_GET_NOTIFICATIONS_MEMBER_BY_ID_NOTIFICATION",
    GET_NOTIFICATIONS_MEMBER_BY_ID_NOTIFICATION_BY_FIELD: "TRELLO_GET_NOTIFICATIONS_MEMBER_BY_ID_NOTIFICATION_BY_FIELD",
    GET_NOTIFICATIONS_MEMBER_CREATOR_BY_ID_NOTIFICATION: "TRELLO_GET_NOTIFICATIONS_MEMBER_CREATOR_BY_ID_NOTIFICATION",
    GET_NOTIFICATIONS_ORGANIZATION_BY_ID_NOTIFICATION: "TRELLO_GET_NOTIFICATIONS_ORGANIZATION_BY_ID_NOTIFICATION",
    GET_NOTIFICATION_ORG_FIELD: "TRELLO_GET_NOTIFICATION_ORG_FIELD",
    GET_NOTIF_CREATOR_FIELD: "TRELLO_GET_NOTIF_CREATOR_FIELD",
    GET_ORGANIZATIONS_ACTIONS_BY_ID_ORG: "TRELLO_GET_ORGANIZATIONS_ACTIONS_BY_ID_ORG",
    GET_ORGANIZATIONS_BOARDS_BY_ID_ORG: "TRELLO_GET_ORGANIZATIONS_BOARDS_BY_ID_ORG",
    GET_ORGANIZATIONS_BOARDS_BY_ID_ORG_BY_FILTER: "TRELLO_GET_ORGANIZATIONS_BOARDS_BY_ID_ORG_BY_FILTER",
    GET_ORGANIZATIONS_BY_ID_ORG: "TRELLO_GET_ORGANIZATIONS_BY_ID_ORG",
    GET_ORGANIZATIONS_BY_ID_ORG_BY_FIELD: "TRELLO_GET_ORGANIZATIONS_BY_ID_ORG_BY_FIELD",
    GET_ORGANIZATIONS_DELTAS_BY_ID_ORG: "TRELLO_GET_ORGANIZATIONS_DELTAS_BY_ID_ORG",
    GET_ORGANIZATIONS_MEMBERSHIPS_BY_ID_ORG: "TRELLO_GET_ORGANIZATIONS_MEMBERSHIPS_BY_ID_ORG",
    GET_ORGANIZATIONS_MEMBERS_BY_ID_ORG: "TRELLO_GET_ORGANIZATIONS_MEMBERS_BY_ID_ORG",
    GET_ORGANIZATIONS_MEMBERS_BY_ID_ORG_BY_FILTER: "TRELLO_GET_ORGANIZATIONS_MEMBERS_BY_ID_ORG_BY_FILTER",
    GET_ORGANIZATIONS_MEMBERS_INVITED_BY_ID_ORG: "TRELLO_GET_ORGANIZATIONS_MEMBERS_INVITED_BY_ID_ORG",
    GET_ORGANIZATIONS_MEMBERS_INVITED_BY_ID_ORG_BY_FIELD: "TRELLO_GET_ORGANIZATIONS_MEMBERS_INVITED_BY_ID_ORG_BY_FIELD",
    GET_ORG_MEMBERSHIP: "TRELLO_GET_ORG_MEMBERSHIP",
    GET_SEARCH: "TRELLO_GET_SEARCH",
    GET_SEARCH_MEMBERS: "TRELLO_GET_SEARCH_MEMBERS",
    GET_SESSIONS_SOCKET: "TRELLO_GET_SESSIONS_SOCKET",
    GET_TOKENS_BY_TOKEN: "TRELLO_GET_TOKENS_BY_TOKEN",
    GET_TOKENS_BY_TOKEN_BY_FIELD: "TRELLO_GET_TOKENS_BY_TOKEN_BY_FIELD",
    GET_TOKENS_MEMBER_BY_TOKEN: "TRELLO_GET_TOKENS_MEMBER_BY_TOKEN",
    GET_TOKENS_MEMBER_BY_TOKEN_BY_FIELD: "TRELLO_GET_TOKENS_MEMBER_BY_TOKEN_BY_FIELD",
    GET_TOKENS_WEBHOOKS_BY_TOKEN: "TRELLO_GET_TOKENS_WEBHOOKS_BY_TOKEN",
    GET_TOKENS_WEBHOOKS_BY_TOKEN_BY_ID_WEBHOOK: "TRELLO_GET_TOKENS_WEBHOOKS_BY_TOKEN_BY_ID_WEBHOOK",
    GET_TYPES_BY_ID: "TRELLO_GET_TYPES_BY_ID",
    GET_WEBHOOKS_BY_ID_WEBHOOK: "TRELLO_GET_WEBHOOKS_BY_ID_WEBHOOK",
    GET_WEBHOOKS_BY_ID_WEBHOOK_BY_FIELD: "TRELLO_GET_WEBHOOKS_BY_ID_WEBHOOK_BY_FIELD",
    LIST_CREATE_LIST: "TRELLO_LIST_CREATE_LIST",
    LIST_GET_BY_ID_LIST: "TRELLO_LIST_GET_BY_ID_LIST",
    LIST_GET_CARDS_BY_ID_LIST: "TRELLO_LIST_GET_CARDS_BY_ID_LIST",
    LIST_ID_BOARD_GET: "TRELLO_LIST_ID_BOARD_GET",
    MARK_CARD_NOTIFICATIONS_READ: "TRELLO_MARK_CARD_NOTIFICATIONS_READ",
    MEMBER_GET_BOARDS: "TRELLO_MEMBER_GET_BOARDS",
    MEMBER_GET_BOARDS_BY_ID_MEMBER: "TRELLO_MEMBER_GET_BOARDS_BY_ID_MEMBER",
    MEMBER_GET_BY_ID: "TRELLO_MEMBER_GET_BY_ID",
    REMOVE_BOARD_STAR: "TRELLO_REMOVE_BOARD_STAR",
    REMOVE_CHECKLIST_ITEM_FROM_CARD_BY_IDS: "TRELLO_REMOVE_CHECKLIST_ITEM_FROM_CARD_BY_IDS",
    REMOVE_MEMBER_FROM_ORG: "TRELLO_REMOVE_MEMBER_FROM_ORG",
    REMOVE_SPECIFIC_MEMBER_S_BOARD_BACKGROUNDS: "TRELLO_REMOVE_SPECIFIC_MEMBER_S_BOARD_BACKGROUNDS",
    SEARCH_GET_RESULTS: "TRELLO_SEARCH_GET_RESULTS",
    TOKEN_GET_MEMBER_BY_TOKEN: "TRELLO_TOKEN_GET_MEMBER_BY_TOKEN",
    UPDATE_ACTIONS_BY_ID_ACTION: "TRELLO_UPDATE_ACTIONS_BY_ID_ACTION",
    UPDATE_ACTIONS_TEXT_BY_ID_ACTION: "TRELLO_UPDATE_ACTIONS_TEXT_BY_ID_ACTION",
    UPDATE_BOARDS_BY_ID_BOARD: "TRELLO_UPDATE_BOARDS_BY_ID_BOARD",
    UPDATE_BOARDS_CLOSED_BY_ID_BOARD: "TRELLO_UPDATE_BOARDS_CLOSED_BY_ID_BOARD",
    UPDATE_BOARDS_DESC_BY_ID_BOARD: "TRELLO_UPDATE_BOARDS_DESC_BY_ID_BOARD",
    UPDATE_BOARDS_ID_ORGANIZATION_BY_ID_BOARD: "TRELLO_UPDATE_BOARDS_ID_ORGANIZATION_BY_ID_BOARD",
    UPDATE_BOARDS_LABEL_NAMES_BLUE_BY_ID_BOARD: "TRELLO_UPDATE_BOARDS_LABEL_NAMES_BLUE_BY_ID_BOARD",
    UPDATE_BOARDS_LABEL_NAMES_GREEN_BY_ID_BOARD: "TRELLO_UPDATE_BOARDS_LABEL_NAMES_GREEN_BY_ID_BOARD",
    UPDATE_BOARDS_LABEL_NAMES_ORANGE_BY_ID_BOARD: "TRELLO_UPDATE_BOARDS_LABEL_NAMES_ORANGE_BY_ID_BOARD",
    UPDATE_BOARDS_LABEL_NAMES_PURPLE_BY_ID_BOARD: "TRELLO_UPDATE_BOARDS_LABEL_NAMES_PURPLE_BY_ID_BOARD",
    UPDATE_BOARDS_LABEL_NAMES_RED_BY_ID_BOARD: "TRELLO_UPDATE_BOARDS_LABEL_NAMES_RED_BY_ID_BOARD",
    UPDATE_BOARDS_LABEL_NAMES_YELLOW_BY_ID_BOARD: "TRELLO_UPDATE_BOARDS_LABEL_NAMES_YELLOW_BY_ID_BOARD",
    UPDATE_BOARDS_MEMBERS_BY_ID_BOARD: "TRELLO_UPDATE_BOARDS_MEMBERS_BY_ID_BOARD",
    UPDATE_BOARDS_MEMBERS_BY_ID_BOARD_BY_ID_MEMBER: "TRELLO_UPDATE_BOARDS_MEMBERS_BY_ID_BOARD_BY_ID_MEMBER",
    UPDATE_BOARDS_MY_PREFS_EMAIL_POSITION_BY_ID_BOARD: "TRELLO_UPDATE_BOARDS_MY_PREFS_EMAIL_POSITION_BY_ID_BOARD",
    UPDATE_BOARDS_MY_PREFS_ID_EMAIL_LIST_BY_ID_BOARD: "TRELLO_UPDATE_BOARDS_MY_PREFS_ID_EMAIL_LIST_BY_ID_BOARD",
    UPDATE_BOARDS_MY_PREFS_SHOW_LIST_GUIDE_BY_ID_BOARD: "TRELLO_UPDATE_BOARDS_MY_PREFS_SHOW_LIST_GUIDE_BY_ID_BOARD",
    UPDATE_BOARDS_MY_PREFS_SHOW_SIDEBAR_BY_ID_BOARD: "TRELLO_UPDATE_BOARDS_MY_PREFS_SHOW_SIDEBAR_BY_ID_BOARD",
    UPDATE_BOARDS_NAME_BY_ID_BOARD: "TRELLO_UPDATE_BOARDS_NAME_BY_ID_BOARD",
    UPDATE_BOARDS_PREFS_BACKGROUND_BY_ID_BOARD: "TRELLO_UPDATE_BOARDS_PREFS_BACKGROUND_BY_ID_BOARD",
    UPDATE_BOARDS_PREFS_CALENDAR_FEED_ENABLED_BY_ID_BOARD: "TRELLO_UPDATE_BOARDS_PREFS_CALENDAR_FEED_ENABLED_BY_ID_BOARD",
    UPDATE_BOARDS_PREFS_CARD_AGING_BY_ID_BOARD: "TRELLO_UPDATE_BOARDS_PREFS_CARD_AGING_BY_ID_BOARD",
    UPDATE_BOARDS_PREFS_CARD_COVERS_BY_ID_BOARD: "TRELLO_UPDATE_BOARDS_PREFS_CARD_COVERS_BY_ID_BOARD",
    UPDATE_BOARDS_PREFS_COMMENTS_BY_ID_BOARD: "TRELLO_UPDATE_BOARDS_PREFS_COMMENTS_BY_ID_BOARD",
    UPDATE_BOARDS_PREFS_INVITATIONS_BY_ID_BOARD: "TRELLO_UPDATE_BOARDS_PREFS_INVITATIONS_BY_ID_BOARD",
    UPDATE_BOARDS_PREFS_PERMISSION_LEVEL_BY_ID_BOARD: "TRELLO_UPDATE_BOARDS_PREFS_PERMISSION_LEVEL_BY_ID_BOARD",
    UPDATE_BOARDS_PREFS_SELF_JOIN_BY_ID_BOARD: "TRELLO_UPDATE_BOARDS_PREFS_SELF_JOIN_BY_ID_BOARD",
    UPDATE_BOARDS_PREFS_VOTING_BY_ID_BOARD: "TRELLO_UPDATE_BOARDS_PREFS_VOTING_BY_ID_BOARD",
    UPDATE_BOARDS_SUBSCRIBED_BY_ID_BOARD: "TRELLO_UPDATE_BOARDS_SUBSCRIBED_BY_ID_BOARD",
    UPDATE_BOARD_MEMBERSHIP: "TRELLO_UPDATE_BOARD_MEMBERSHIP",
    UPDATE_BOARD_SIDEBAR_ACTIONS_PREFS: "TRELLO_UPDATE_BOARD_SIDEBAR_ACTIONS_PREFS",
    UPDATE_BOARD_SIDEBAR_ACTIVITY_PREFS: "TRELLO_UPDATE_BOARD_SIDEBAR_ACTIVITY_PREFS",
    UPDATE_BOARD_SIDEBAR_MEMBERS_PREF: "TRELLO_UPDATE_BOARD_SIDEBAR_MEMBERS_PREF",
    UPDATE_BOARD_STAR_POSITION: "TRELLO_UPDATE_BOARD_STAR_POSITION",
    UPDATE_CARDS_BY_ID_CARD: "TRELLO_UPDATE_CARDS_BY_ID_CARD",
    UPDATE_CARDS_CLOSED_BY_ID_CARD: "TRELLO_UPDATE_CARDS_CLOSED_BY_ID_CARD",
    UPDATE_CARDS_DESC_BY_ID_CARD: "TRELLO_UPDATE_CARDS_DESC_BY_ID_CARD",
    UPDATE_CARDS_DUE_BY_ID_CARD: "TRELLO_UPDATE_CARDS_DUE_BY_ID_CARD",
    UPDATE_CARDS_ID_ATTACHMENT_COVER_BY_ID_CARD: "TRELLO_UPDATE_CARDS_ID_ATTACHMENT_COVER_BY_ID_CARD",
    UPDATE_CARDS_ID_BOARD_BY_ID_CARD: "TRELLO_UPDATE_CARDS_ID_BOARD_BY_ID_CARD",
    UPDATE_CARDS_ID_LIST_BY_ID_CARD: "TRELLO_UPDATE_CARDS_ID_LIST_BY_ID_CARD",
    UPDATE_CARDS_ID_MEMBERS_BY_ID_CARD: "TRELLO_UPDATE_CARDS_ID_MEMBERS_BY_ID_CARD",
    UPDATE_CARDS_LABELS_BY_ID_CARD: "TRELLO_UPDATE_CARDS_LABELS_BY_ID_CARD",
    UPDATE_CARDS_NAME_BY_ID_CARD: "TRELLO_UPDATE_CARDS_NAME_BY_ID_CARD",
    UPDATE_CARDS_POS_BY_ID_CARD: "TRELLO_UPDATE_CARDS_POS_BY_ID_CARD",
    UPDATE_CARDS_STICKERS_BY_ID_CARD_BY_ID_STICKER: "TRELLO_UPDATE_CARDS_STICKERS_BY_ID_CARD_BY_ID_STICKER",
    UPDATE_CARDS_SUBSCRIBED_BY_ID_CARD: "TRELLO_UPDATE_CARDS_SUBSCRIBED_BY_ID_CARD",
    UPDATE_CARD_CHECKLIST_ITEM_POSITION: "TRELLO_UPDATE_CARD_CHECKLIST_ITEM_POSITION",
    UPDATE_CARD_CHECKLIST_ITEM_STATE_BY_IDS: "TRELLO_UPDATE_CARD_CHECKLIST_ITEM_STATE_BY_IDS",
    UPDATE_CARD_COMMENT: "TRELLO_UPDATE_CARD_COMMENT",
    UPDATE_CHECKLISTS_BY_ID_CHECKLIST: "TRELLO_UPDATE_CHECKLISTS_BY_ID_CHECKLIST",
    UPDATE_CHECKLISTS_ID_CARD_BY_ID_CHECKLIST: "TRELLO_UPDATE_CHECKLISTS_ID_CARD_BY_ID_CHECKLIST",
    UPDATE_CHECKLISTS_NAME_BY_ID_CHECKLIST: "TRELLO_UPDATE_CHECKLISTS_NAME_BY_ID_CHECKLIST",
    UPDATE_CHECKLISTS_POS_BY_ID_CHECKLIST: "TRELLO_UPDATE_CHECKLISTS_POS_BY_ID_CHECKLIST",
    UPDATE_CHECKLIST_ITEM_BY_IDS: "TRELLO_UPDATE_CHECKLIST_ITEM_BY_IDS",
    UPDATE_CHECKLIST_ITEM_NAME_IN_CARD: "TRELLO_UPDATE_CHECKLIST_ITEM_NAME_IN_CARD",
    UPDATE_LABELS_BY_ID_LABEL: "TRELLO_UPDATE_LABELS_BY_ID_LABEL",
    UPDATE_LABELS_COLOR_BY_ID_LABEL: "TRELLO_UPDATE_LABELS_COLOR_BY_ID_LABEL",
    UPDATE_LABELS_NAME_BY_ID_LABEL: "TRELLO_UPDATE_LABELS_NAME_BY_ID_LABEL",
    UPDATE_LISTS_BY_ID_LIST: "TRELLO_UPDATE_LISTS_BY_ID_LIST",
    UPDATE_LISTS_CLOSED_BY_ID_LIST: "TRELLO_UPDATE_LISTS_CLOSED_BY_ID_LIST",
    UPDATE_LISTS_ID_BOARD_BY_ID_LIST: "TRELLO_UPDATE_LISTS_ID_BOARD_BY_ID_LIST",
    UPDATE_LISTS_NAME_BY_ID_LIST: "TRELLO_UPDATE_LISTS_NAME_BY_ID_LIST",
    UPDATE_LISTS_POS_BY_ID_LIST: "TRELLO_UPDATE_LISTS_POS_BY_ID_LIST",
    UPDATE_LISTS_SUBSCRIBED_BY_ID_LIST: "TRELLO_UPDATE_LISTS_SUBSCRIBED_BY_ID_LIST",
    UPDATE_MEMBERS_AVATAR_SOURCE_BY_ID_MEMBER: "TRELLO_UPDATE_MEMBERS_AVATAR_SOURCE_BY_ID_MEMBER",
    UPDATE_MEMBERS_BIO_BY_ID_MEMBER: "TRELLO_UPDATE_MEMBERS_BIO_BY_ID_MEMBER",
    UPDATE_MEMBERS_BY_ID_MEMBER: "TRELLO_UPDATE_MEMBERS_BY_ID_MEMBER",
    UPDATE_MEMBERS_FULL_NAME_BY_ID_MEMBER: "TRELLO_UPDATE_MEMBERS_FULL_NAME_BY_ID_MEMBER",
    UPDATE_MEMBERS_INITIALS_BY_ID_MEMBER: "TRELLO_UPDATE_MEMBERS_INITIALS_BY_ID_MEMBER",
    UPDATE_MEMBERS_PREFS_COLOR_BLIND_BY_ID_MEMBER: "TRELLO_UPDATE_MEMBERS_PREFS_COLOR_BLIND_BY_ID_MEMBER",
    UPDATE_MEMBERS_PREFS_LOCALE_BY_ID_MEMBER: "TRELLO_UPDATE_MEMBERS_PREFS_LOCALE_BY_ID_MEMBER",
    UPDATE_MEMBERS_USERNAME_BY_ID_MEMBER: "TRELLO_UPDATE_MEMBERS_USERNAME_BY_ID_MEMBER",
    UPDATE_MEMBER_BOARD_BACKGROUND: "TRELLO_UPDATE_MEMBER_BOARD_BACKGROUND",
    UPDATE_MEMBER_BOARD_STAR: "TRELLO_UPDATE_MEMBER_BOARD_STAR",
    UPDATE_MEMBER_SAVED_SEARCH: "TRELLO_UPDATE_MEMBER_SAVED_SEARCH",
    UPDATE_MEMBER_SAVED_SEARCH_NAME: "TRELLO_UPDATE_MEMBER_SAVED_SEARCH_NAME",
    UPDATE_MEMBER_SAVED_SEARCH_POS: "TRELLO_UPDATE_MEMBER_SAVED_SEARCH_POS",
    UPDATE_MEMBER_SAVED_SEARCH_QUERY: "TRELLO_UPDATE_MEMBER_SAVED_SEARCH_QUERY",
    UPDATE_MEMBER_SUMMARY_INTERVAL: "TRELLO_UPDATE_MEMBER_SUMMARY_INTERVAL",
    UPDATE_MEMBER_S_CUSTOM_BOARD_BACKGROUNDS: "TRELLO_UPDATE_MEMBER_S_CUSTOM_BOARD_BACKGROUNDS",
    UPDATE_NOTIFICATIONS_BY_ID_NOTIFICATION: "TRELLO_UPDATE_NOTIFICATIONS_BY_ID_NOTIFICATION",
    UPDATE_NOTIFICATIONS_UNREAD_BY_ID_NOTIFICATION: "TRELLO_UPDATE_NOTIFICATIONS_UNREAD_BY_ID_NOTIFICATION",
    UPDATE_ORGANIZATIONS_BY_ID_ORG: "TRELLO_UPDATE_ORGANIZATIONS_BY_ID_ORG",
    UPDATE_ORGANIZATIONS_DESC_BY_ID_ORG: "TRELLO_UPDATE_ORGANIZATIONS_DESC_BY_ID_ORG",
    UPDATE_ORGANIZATIONS_DISPLAY_NAME_BY_ID_ORG: "TRELLO_UPDATE_ORGANIZATIONS_DISPLAY_NAME_BY_ID_ORG",
    UPDATE_ORGANIZATIONS_MEMBERS_BY_ID_ORG: "TRELLO_UPDATE_ORGANIZATIONS_MEMBERS_BY_ID_ORG",
    UPDATE_ORGANIZATIONS_MEMBERS_BY_ID_ORG_BY_ID_MEMBER: "TRELLO_UPDATE_ORGANIZATIONS_MEMBERS_BY_ID_ORG_BY_ID_MEMBER",
    UPDATE_ORGANIZATIONS_NAME_BY_ID_ORG: "TRELLO_UPDATE_ORGANIZATIONS_NAME_BY_ID_ORG",
    UPDATE_ORGANIZATIONS_PREFS_PERMISSION_LEVEL_BY_ID_ORG: "TRELLO_UPDATE_ORGANIZATIONS_PREFS_PERMISSION_LEVEL_BY_ID_ORG",
    UPDATE_ORGANIZATIONS_WEBSITE_BY_ID_ORG: "TRELLO_UPDATE_ORGANIZATIONS_WEBSITE_BY_ID_ORG",
    UPDATE_ORG_ASSOCIATED_DOMAIN_PREFS: "TRELLO_UPDATE_ORG_ASSOCIATED_DOMAIN_PREFS",
    UPDATE_ORG_BOARD_VISIBILITY_ORG_PREF: "TRELLO_UPDATE_ORG_BOARD_VISIBILITY_ORG_PREF",
    UPDATE_ORG_EXTERNAL_MEMBERS_ACCESS: "TRELLO_UPDATE_ORG_EXTERNAL_MEMBERS_ACCESS",
    UPDATE_ORG_GOOGLE_APPS_VERSION: "TRELLO_UPDATE_ORG_GOOGLE_APPS_VERSION",
    UPDATE_ORG_INVITE_RESTRICTIONS: "TRELLO_UPDATE_ORG_INVITE_RESTRICTIONS",
    UPDATE_ORG_MEMBERSHIP: "TRELLO_UPDATE_ORG_MEMBERSHIP",
    UPDATE_ORG_MEMBER_DEACTIVATION_STATUS: "TRELLO_UPDATE_ORG_MEMBER_DEACTIVATION_STATUS",
    UPDATE_ORG_PRIVATE_BOARD_VISIBILITY: "TRELLO_UPDATE_ORG_PRIVATE_BOARD_VISIBILITY",
    UPDATE_ORG_PUBLIC_BOARD_VISIBILITY: "TRELLO_UPDATE_ORG_PUBLIC_BOARD_VISIBILITY",
    UPDATE_SESSIONS_BY_ID_SESSION: "TRELLO_UPDATE_SESSIONS_BY_ID_SESSION",
    UPDATE_SESSIONS_STATUS_BY_ID_SESSION: "TRELLO_UPDATE_SESSIONS_STATUS_BY_ID_SESSION",
    UPDATE_TOKENS_WEBHOOKS_BY_TOKEN: "TRELLO_UPDATE_TOKENS_WEBHOOKS_BY_TOKEN",
    UPDATE_WEBHOOKS: "TRELLO_UPDATE_WEBHOOKS",
    UPDATE_WEBHOOKS_ACTIVE_BY_ID_WEBHOOK: "TRELLO_UPDATE_WEBHOOKS_ACTIVE_BY_ID_WEBHOOK",
    UPDATE_WEBHOOKS_BY_ID_WEBHOOK: "TRELLO_UPDATE_WEBHOOKS_BY_ID_WEBHOOK",
    UPDATE_WEBHOOKS_CALLBACK_URLBY_ID_WEBHOOK: "TRELLO_UPDATE_WEBHOOKS_CALLBACK_URLBY_ID_WEBHOOK",
    UPDATE_WEBHOOKS_DESCRIPTION_BY_ID_WEBHOOK: "TRELLO_UPDATE_WEBHOOKS_DESCRIPTION_BY_ID_WEBHOOK",
    UPDATE_WEBHOOKS_ID_MODEL_BY_ID_WEBHOOK: "TRELLO_UPDATE_WEBHOOKS_ID_MODEL_BY_ID_WEBHOOK",
  },
  triggerTypes: {
    TRELLO_ARCHIVED_CARD_TRIGGER: "TRELLO_TRELLO_ARCHIVED_CARD_TRIGGER",
    TRELLO_NEW_ACTIVITY_TRIGGER: "TRELLO_TRELLO_NEW_ACTIVITY_TRIGGER",
    TRELLO_NEW_BOARD_TRIGGER: "TRELLO_TRELLO_NEW_BOARD_TRIGGER",
    TRELLO_NEW_CARD_TRIGGER: "TRELLO_TRELLO_NEW_CARD_TRIGGER",
    TRELLO_UPDATED_CARD_TRIGGER: "TRELLO_TRELLO_UPDATED_CARD_TRIGGER",
  },
}
