// @composio/core/generated/twitter.ts
/**
 * Map of Composio's TWITTER toolkit.
 */
export const TWITTER = {
  slug: "twitter",
  tools: {
    ADD_A_LIST_MEMBER: "TWITTER_ADD_A_LIST_MEMBER",
    ADD_POST_TO_BOOKMARKS: "TWITTER_ADD_POST_TO_BOOKMARKS",
    BOOKMARKS_BY_USER: "TWITTER_BOOKMARKS_BY_USER",
    CREATE_A_NEW_DM_CONVERSATION: "TWITTER_CREATE_A_NEW_DM_CONVERSATION",
    CREATE_COMPLIANCE_JOB_REQUEST: "TWITTER_CREATE_COMPLIANCE_JOB_REQUEST",
    CREATE_LIST: "TWITTER_CREATE_LIST",
    CREATION_OF_A_POST: "TWITTER_CREATION_OF_A_POST",
    DELETE_DM: "TWITTER_DELETE_DM",
    DELETE_LIST: "TWITTER_DELETE_LIST",
    FETCH_LIST_MEMBERS_BY_ID: "TWITTER_FETCH_LIST_MEMBERS_BY_ID",
    FETCH_SPACE_TICKET_BUYERS_LIST: "TWITTER_FETCH_SPACE_TICKET_BUYERS_LIST",
    FOLLOWERS_BY_USER_ID: "TWITTER_FOLLOWERS_BY_USER_ID",
    FOLLOWING_BY_USER_ID: "TWITTER_FOLLOWING_BY_USER_ID",
    FOLLOW_A_LIST: "TWITTER_FOLLOW_A_LIST",
    FOLLOW_USER: "TWITTER_FOLLOW_USER",
    FULL_ARCHIVE_SEARCH: "TWITTER_FULL_ARCHIVE_SEARCH",
    FULL_ARCHIVE_SEARCH_COUNTS: "TWITTER_FULL_ARCHIVE_SEARCH_COUNTS",
    GET_A_USER_S_LIST_MEMBERSHIPS: "TWITTER_GET_A_USER_S_LIST_MEMBERSHIPS",
    GET_A_USER_S_OWNED_LISTS: "TWITTER_GET_A_USER_S_OWNED_LISTS",
    GET_A_USER_S_PINNED_LISTS: "TWITTER_GET_A_USER_S_PINNED_LISTS",
    GET_BLOCKED_USERS: "TWITTER_GET_BLOCKED_USERS",
    GET_DM_EVENTS_BY_ID: "TWITTER_GET_DM_EVENTS_BY_ID",
    GET_DM_EVENTS_FOR_A_DM_CONVERSATION: "TWITTER_GET_DM_EVENTS_FOR_A_DM_CONVERSATION",
    GET_LIST_FOLLOWERS: "TWITTER_GET_LIST_FOLLOWERS",
    GET_MUTED_USERS: "TWITTER_GET_MUTED_USERS",
    GET_POST_RETWEETERS_ACTION: "TWITTER_GET_POST_RETWEETERS_ACTION",
    GET_RECENT_DM_EVENTS: "TWITTER_GET_RECENT_DM_EVENTS",
    GET_USER_S_FOLLOWED_LISTS: "TWITTER_GET_USER_S_FOLLOWED_LISTS",
    HIDE_REPLIES: "TWITTER_HIDE_REPLIES",
    LIST_LOOKUP_BY_LIST_ID: "TWITTER_LIST_LOOKUP_BY_LIST_ID",
    LIST_POSTS_TIMELINE_BY_LIST_ID: "TWITTER_LIST_POSTS_TIMELINE_BY_LIST_ID",
    LIST_POST_LIKERS: "TWITTER_LIST_POST_LIKERS",
    MEDIA_UPLOAD_MEDIA: "TWITTER_MEDIA_UPLOAD_MEDIA",
    MUTE_USER_BY_USER_ID: "TWITTER_MUTE_USER_BY_USER_ID",
    PIN_A_LIST: "TWITTER_PIN_A_LIST",
    POSTS_LABEL_STREAM: "TWITTER_POSTS_LABEL_STREAM",
    POST_DELETE_BY_POST_ID: "TWITTER_POST_DELETE_BY_POST_ID",
    POST_LOOKUP_BY_POST_ID: "TWITTER_POST_LOOKUP_BY_POST_ID",
    POST_LOOKUP_BY_POST_IDS: "TWITTER_POST_LOOKUP_BY_POST_IDS",
    POST_USAGE: "TWITTER_POST_USAGE",
    RECENT_SEARCH: "TWITTER_RECENT_SEARCH",
    RECENT_SEARCH_COUNTS: "TWITTER_RECENT_SEARCH_COUNTS",
    REMOVE_A_BOOKMARKED_POST: "TWITTER_REMOVE_A_BOOKMARKED_POST",
    REMOVE_A_LIST_MEMBER: "TWITTER_REMOVE_A_LIST_MEMBER",
    RETRIEVE_COMPLIANCE_JOBS: "TWITTER_RETRIEVE_COMPLIANCE_JOBS",
    RETRIEVE_COMPLIANCE_JOB_BY_ID: "TWITTER_RETRIEVE_COMPLIANCE_JOB_BY_ID",
    RETRIEVE_DM_CONVERSATION_EVENTS: "TWITTER_RETRIEVE_DM_CONVERSATION_EVENTS",
    RETRIEVE_POSTS_FROM_A_SPACE: "TWITTER_RETRIEVE_POSTS_FROM_A_SPACE",
    RETRIEVE_POSTS_THAT_QUOTE_A_POST: "TWITTER_RETRIEVE_POSTS_THAT_QUOTE_A_POST",
    RETRIEVE_POSTS_THAT_REPOST_A_POST: "TWITTER_RETRIEVE_POSTS_THAT_REPOST_A_POST",
    RETURNS_POST_OBJECTS_LIKED_BY_THE_PROVIDED_USER_ID: "TWITTER_RETURNS_POST_OBJECTS_LIKED_BY_THE_PROVIDED_USER_ID",
    RETURNS_THE_OPEN_API_SPECIFICATION_DOCUMENT: "TWITTER_RETURNS_THE_OPEN_API_SPECIFICATION_DOCUMENT",
    RETWEET_POST: "TWITTER_RETWEET_POST",
    SEARCH_FOR_SPACES: "TWITTER_SEARCH_FOR_SPACES",
    SEND_A_NEW_MESSAGE_TO_A_DM_CONVERSATION: "TWITTER_SEND_A_NEW_MESSAGE_TO_A_DM_CONVERSATION",
    SEND_A_NEW_MESSAGE_TO_A_USER: "TWITTER_SEND_A_NEW_MESSAGE_TO_A_USER",
    SPACE_LOOKUP_BY_SPACE_ID: "TWITTER_SPACE_LOOKUP_BY_SPACE_ID",
    SPACE_LOOKUP_BY_THEIR_CREATORS: "TWITTER_SPACE_LOOKUP_BY_THEIR_CREATORS",
    SPACE_LOOKUP_UP_SPACE_IDS: "TWITTER_SPACE_LOOKUP_UP_SPACE_IDS",
    UNFOLLOW_A_LIST: "TWITTER_UNFOLLOW_A_LIST",
    UNFOLLOW_USER: "TWITTER_UNFOLLOW_USER",
    UNLIKE_POST: "TWITTER_UNLIKE_POST",
    UNMUTE_USER_BY_USER_ID: "TWITTER_UNMUTE_USER_BY_USER_ID",
    UNPIN_A_LIST: "TWITTER_UNPIN_A_LIST",
    UNRETWEET_POST: "TWITTER_UNRETWEET_POST",
    UPDATE_LIST: "TWITTER_UPDATE_LIST",
    USER_HOME_TIMELINE_BY_USER_ID: "TWITTER_USER_HOME_TIMELINE_BY_USER_ID",
    USER_LIKE_POST: "TWITTER_USER_LIKE_POST",
    USER_LOOKUP_BY_ID: "TWITTER_USER_LOOKUP_BY_ID",
    USER_LOOKUP_BY_IDS: "TWITTER_USER_LOOKUP_BY_IDS",
    USER_LOOKUP_BY_USERNAME: "TWITTER_USER_LOOKUP_BY_USERNAME",
    USER_LOOKUP_BY_USERNAMES: "TWITTER_USER_LOOKUP_BY_USERNAMES",
    USER_LOOKUP_ME: "TWITTER_USER_LOOKUP_ME",
  },
  triggerTypes: {},
}
