// @composio/core/generated/webex.ts
/**
 * Map of Composio's WEBEX toolkit.
 */
export const WEBEX = {
  slug: "webex",
  tools: {
    CREATE_ROOM: "WEBEX_CREATE_ROOM",
    CREATE_TEAM: "WEBEX_CREATE_TEAM",
    GET_TEAM_DETAILS: "WEBEX_GET_TEAM_DETAILS",
    LIST_TEAMS: "WEBEX_LIST_TEAMS",
    LIST_WEBHOOKS: "WEBEX_LIST_WEBHOOKS",
    MESSAGING_CREATE_MESSAGE: "WEBEX_MESSAGING_CREATE_MESSAGE",
    MESSAGING_CREATE_ROOM: "WEBEX_MESSAGING_CREATE_ROOM",
    MESSAGING_CREATE_TEAM_MEMBERSHIP: "WEBEX_MESSAGING_CREATE_TEAM_MEMBERSHIP",
    MESSAGING_DELETE_MEMBERSHIP: "WEBEX_MESSAGING_DELETE_MEMBERSHIP",
    MESSAGING_DELETE_MESSAGE: "WEBEX_MESSAGING_DELETE_MESSAGE",
    MESSAGING_DELETE_ROOM: "WEBEX_MESSAGING_DELETE_ROOM",
    MESSAGING_GET_MEMBERSHIP_DETAILS: "WEBEX_MESSAGING_GET_MEMBERSHIP_DETAILS",
    MESSAGING_GET_MESSAGE_DETAILS: "WEBEX_MESSAGING_GET_MESSAGE_DETAILS",
    MESSAGING_GET_ROOM_DETAILS: "WEBEX_MESSAGING_GET_ROOM_DETAILS",
    MESSAGING_GET_TEAM_MEMBERSHIP_DETAILS: "WEBEX_MESSAGING_GET_TEAM_MEMBERSHIP_DETAILS",
    MESSAGING_LIST_MEMBERSHIPS: "WEBEX_MESSAGING_LIST_MEMBERSHIPS",
    MESSAGING_LIST_MESSAGES: "WEBEX_MESSAGING_LIST_MESSAGES",
    MESSAGING_LIST_ROOMS: "WEBEX_MESSAGING_LIST_ROOMS",
    MESSAGING_LIST_TEAM_MEMBERSHIPS: "WEBEX_MESSAGING_LIST_TEAM_MEMBERSHIPS",
    PEOPLE_GET_PERSON: "WEBEX_PEOPLE_GET_PERSON",
    PEOPLE_LIST_PEOPLE: "WEBEX_PEOPLE_LIST_PEOPLE",
    ROOMS_GET_ROOM_DETAILS: "WEBEX_ROOMS_GET_ROOM_DETAILS",
    UPDATE_MEMBERSHIP: "WEBEX_UPDATE_MEMBERSHIP",
    UPDATE_ROOM: "WEBEX_UPDATE_ROOM",
    UPDATE_TEAM: "WEBEX_UPDATE_TEAM",
    WEBHOOKS_CREATE_WEBHOOK: "WEBEX_WEBHOOKS_CREATE_WEBHOOK",
    WEBHOOKS_DELETE_WEBHOOK: "WEBEX_WEBHOOKS_DELETE_WEBHOOK",
    WEBHOOKS_GET_WEBHOOK: "WEBEX_WEBHOOKS_GET_WEBHOOK",
  },
  triggerTypes: {},
}
