// @composio/core/generated/webflow.ts
/**
 * Map of Composio's WEBFLOW toolkit.
 */
export const WEBFLOW = {
  slug: "webflow",
  tools: {
    CREATE_COLLECTION_ITEM: "WEBFLOW_CREATE_COLLECTION_ITEM",
    DELETE_COLLECTION_ITEM: "WEBFLOW_DELETE_COLLECTION_ITEM",
    FULFILL_ORDER: "WEBFLOW_FULFILL_ORDER",
    GET_COLLECTION: "WEBFLOW_GET_COLLECTION",
    GET_COLLECTION_ITEM: "WEBFLOW_GET_COLLECTION_ITEM",
    GET_ITEM_INVENTORY: "WEBFLOW_GET_ITEM_INVENTORY",
    GET_ORDER: "WEBFLOW_GET_ORDER",
    GET_SITE_INFO: "WEBFLOW_GET_SITE_INFO",
    LIST_COLLECTIONS: "WEBFLOW_LIST_COLLECTIONS",
    LIST_COLLECTION_ITEMS: "WEBFLOW_LIST_COLLECTION_ITEMS",
    LIST_FORM_SUBMISSIONS: "WEBFLOW_LIST_FORM_SUBMISSIONS",
    LIST_ORDERS: "WEBFLOW_LIST_ORDERS",
    LIST_PAGES: "WEBFLOW_LIST_PAGES",
    LIST_WEBFLOW_SITES: "WEBFLOW_LIST_WEBFLOW_SITES",
    PUBLISH_SITE: "WEBFLOW_PUBLISH_SITE",
    REFUND_ORDER: "WEBFLOW_REFUND_ORDER",
    UNFULFILL_ORDER: "WEBFLOW_UNFULFILL_ORDER",
    UPDATE_COLLECTION_ITEM: "WEBFLOW_UPDATE_COLLECTION_ITEM",
    UPDATE_ITEM_INVENTORY: "WEBFLOW_UPDATE_ITEM_INVENTORY",
    UPDATE_ORDER: "WEBFLOW_UPDATE_ORDER",
    UPLOAD_ASSET: "WEBFLOW_UPLOAD_ASSET",
  },
  triggerTypes: {},
}
