// @composio/core/generated/wrike.ts
/**
 * Map of Composio's WRIKE toolkit.
 */
export const WRIKE = {
  slug: "wrike",
  tools: {
    BULK_MODIFY_GROUPS: "WRIKE_BULK_MODIFY_GROUPS",
    CREATE_FOLDER: "WRIKE_CREATE_FOLDER",
    CREATE_GROUP: "WRIKE_CREATE_GROUP",
    CREATE_INVITATION: "WRIKE_CREATE_INVITATION",
    CREATE_TASK: "WRIKE_CREATE_TASK",
    DELETE_CUSTOM_FIELD: "WRIKE_DELETE_CUSTOM_FIELD",
    DELETE_FOLDER: "WRIKE_DELETE_FOLDER",
    DELETE_GROUP: "WRIKE_DELETE_GROUP",
    DELETE_INIVTATION: "WRIKE_DELETE_INIVTATION",
    DELETE_TASK: "WRIKE_DELETE_TASK",
    FETCH_ALL_TASKS: "WRIKE_FETCH_ALL_TASKS",
    GET_ACCOUNT_INFORMATION: "WRIKE_GET_ACCOUNT_INFORMATION",
    GET_ALL_CUSTOM_FIELDS: "WRIKE_GET_ALL_CUSTOM_FIELDS",
    GET_CONTACTS: "WRIKE_GET_CONTACTS",
    GET_FOLDERS: "WRIKE_GET_FOLDERS",
    GET_INFORMATION_ABOUT_SPECIFIC_CONTACT: "WRIKE_GET_INFORMATION_ABOUT_SPECIFIC_CONTACT",
    GET_SPECIFIC_USER: "WRIKE_GET_SPECIFIC_USER",
    GET_TASK_BY_ID: "WRIKE_GET_TASK_BY_ID",
    LAUNCH_FOLDER_BLUEPRINT_ASYNC: "WRIKE_LAUNCH_FOLDER_BLUEPRINT_ASYNC",
    LAUNCH_TASK_BLUEPRINT_ASYNC: "WRIKE_LAUNCH_TASK_BLUEPRINT_ASYNC",
    LIST_FOLDER_BLUEPRINTS: "WRIKE_LIST_FOLDER_BLUEPRINTS",
    LIST_SPACE_FOLDER_BLUEPRINTS: "WRIKE_LIST_SPACE_FOLDER_BLUEPRINTS",
    LIST_SPACE_TASK_BLUEPRINTS: "WRIKE_LIST_SPACE_TASK_BLUEPRINTS",
    LIST_SUBFOLDERS_BY_FOLDER_ID: "WRIKE_LIST_SUBFOLDERS_BY_FOLDER_ID",
    LIST_TASK_BLUEPRINTS: "WRIKE_LIST_TASK_BLUEPRINTS",
    MODIFY_ACCOUNT: "WRIKE_MODIFY_ACCOUNT",
    MODIFY_FOLDER: "WRIKE_MODIFY_FOLDER",
    MODIFY_GROUP: "WRIKE_MODIFY_GROUP",
    MODIFY_TASK: "WRIKE_MODIFY_TASK",
    QUERY_CUSTOMFIELDS: "WRIKE_QUERY_CUSTOMFIELDS",
    QUERY_INVITATIONS: "WRIKE_QUERY_INVITATIONS",
    QUERY_SPECIFIC_GROUP: "WRIKE_QUERY_SPECIFIC_GROUP",
    QUERY_WORKFLOWS: "WRIKE_QUERY_WORKFLOWS",
    RETRIEVE_LIST_OF_GROUPS: "WRIKE_RETRIEVE_LIST_OF_GROUPS",
    UPDATE_CUSTOM_FIELD_BY_ID: "WRIKE_UPDATE_CUSTOM_FIELD_BY_ID",
    UPDATE_INVITATION: "WRIKE_UPDATE_INVITATION",
    UPDATE_METADATA_ON_SPECIFIC_CONTACT: "WRIKE_UPDATE_METADATA_ON_SPECIFIC_CONTACT",
    UPDATE_SPECIFIC_USER: "WRIKE_UPDATE_SPECIFIC_USER",
  },
  triggerTypes: {},
}
