// @composio/core/generated/zenrows.ts
/**
 * Map of Composio's ZENROWS toolkit.
 */
export const ZENROWS = {
  slug: "zenrows",
  tools: {
    GET_API_USAGE: "ZENROWS_GET_API_USAGE",
    GET_CONCURRENCY_STATUS: "ZENROWS_GET_CONCURRENCY_STATUS",
    GET_CONCURRENCY_STATUS_DETAILED: "ZENROWS_GET_CONCURRENCY_STATUS_DETAILED",
    GET_ORIGINAL_STATUS: "ZENROWS_GET_ORIGINAL_STATUS",
    GET_PDF_FROM_URL: "ZENROWS_GET_PDF_FROM_URL",
    GET_PLAINTEXT: "ZENROWS_GET_PLAINTEXT",
    GET_REAL_ESTATE_DATA: "ZENROWS_GET_REAL_ESTATE_DATA",
    GET_RESPONSE_COMPRESSION: "ZENROWS_GET_RESPONSE_COMPRESSION",
    GET_RESPONSE_HEADERS: "ZENROWS_GET_RESPONSE_HEADERS",
    GET_SESSION_ID: "ZENROWS_GET_SESSION_ID",
    GET_WALMART_PRODUCT: "ZENROWS_GET_WALMART_PRODUCT",
    SCRAPE_URL: "ZENROWS_SCRAPE_URL",
    SCRAPE_URL_AUTOPARSE: "ZENROWS_SCRAPE_URL_AUTOPARSE",
    SCRAPE_URL_HTML: "ZENROWS_SCRAPE_URL_HTML",
    SCRAPE_WITH_CSS_SELECTORS: "ZENROWS_SCRAPE_WITH_CSS_SELECTORS",
    SCREENSHOT_URL: "ZENROWS_SCREENSHOT_URL",
  },
  triggerTypes: {},
}
