// @composio/core/generated/zoho.ts
/**
 * Map of Composio's ZOHO toolkit.
 */
export const ZOHO = {
  slug: "zoho",
  tools: {
    BIGIN_ADD_RECORDS: "ZOHO_BIGIN_ADD_RECORDS",
    BIGIN_ADD_TAGS_TO_RECORDS: "ZOHO_BIGIN_ADD_TAGS_TO_RECORDS",
    BIGIN_CREATE_TAGS: "ZOHO_BIGIN_CREATE_TAGS",
    BIGIN_DELETE_ATTACHMENT: "ZOHO_BIGIN_DELETE_ATTACHMENT",
    BIGIN_DELETE_RECORDS: "ZOHO_BIGIN_DELETE_RECORDS",
    BIGIN_DOWNLOAD_ATTACHMENT: "ZOHO_BIGIN_DOWNLOAD_ATTACHMENT",
    BIGIN_GET_ATTACHMENTS: "ZOHO_BIGIN_GET_ATTACHMENTS",
    BIGIN_GET_DELETED_RECORDS: "ZOHO_BIGIN_GET_DELETED_RECORDS",
    BIGIN_GET_MODULES: "ZOHO_BIGIN_GET_MODULES",
    BIGIN_GET_RECORDS: "ZOHO_BIGIN_GET_RECORDS",
    BIGIN_UPDATE_RECORDS: "ZOHO_BIGIN_UPDATE_RECORDS",
    BIGIN_UPLOAD_ATTACHMENT: "ZOHO_BIGIN_UPLOAD_ATTACHMENT",
    CONVERT_ZOHO_LEAD: "ZOHO_CONVERT_ZOHO_LEAD",
    CREATE_ZOHO_RECORD: "ZOHO_CREATE_ZOHO_RECORD",
    CREATE_ZOHO_TAG: "ZOHO_CREATE_ZOHO_TAG",
    GET_ZOHO_RECORDS: "ZOHO_GET_ZOHO_RECORDS",
    INVOICE_GET_ITEM: "ZOHO_INVOICE_GET_ITEM",
    INVOICE_LIST_CONTACTS: "ZOHO_INVOICE_LIST_CONTACTS",
    INVOICE_LIST_EXPENSES: "ZOHO_INVOICE_LIST_EXPENSES",
    INVOICE_LIST_INVOICES: "ZOHO_INVOICE_LIST_INVOICES",
    INVOICE_LIST_ITEMS: "ZOHO_INVOICE_LIST_ITEMS",
    INVOICE_LIST_PAYMENTS: "ZOHO_INVOICE_LIST_PAYMENTS",
    UPDATE_RELATED_RECORDS: "ZOHO_UPDATE_RELATED_RECORDS",
    UPDATE_ZOHO_RECORD: "ZOHO_UPDATE_ZOHO_RECORD",
  },
  triggerTypes: {},
}
