// @composio/core/generated/zoom.ts
/**
 * Map of Composio's ZOOM toolkit.
 */
export const ZOOM = {
  slug: "zoom",
  tools: {
    ADD_A_MEETING_REGISTRANT: "ZOOM_ADD_A_MEETING_REGISTRANT",
    ADD_A_NEW_DEVICE: "ZOOM_ADD_A_NEW_DEVICE",
    ADD_A_USER_S_TSP_ACCOUNT: "ZOOM_ADD_A_USER_S_TSP_ACCOUNT",
    ADD_A_WEBINAR_REGISTRANT: "ZOOM_ADD_A_WEBINAR_REGISTRANT",
    ADD_PANELISTS: "ZOOM_ADD_PANELISTS",
    ASSIGN_A_DEVICE_TO_A_USER_OR_COMMONAREA: "ZOOM_ASSIGN_A_DEVICE_TO_A_USER_OR_COMMONAREA",
    CHANGE_DEVICE: "ZOOM_CHANGE_DEVICE",
    CHANGE_DEVICE_ASSOCIATION: "ZOOM_CHANGE_DEVICE_ASSOCIATION",
    CREATE_A_H_323_SIP_DEVICE: "ZOOM_CREATE_A_H_323_SIP_DEVICE",
    CREATE_A_MEETING: "ZOOM_CREATE_A_MEETING",
    CREATE_A_MEETING_POLL: "ZOOM_CREATE_A_MEETING_POLL",
    CREATE_A_MEETING_TEMPLATE_FROM_AN_EXISTING_MEETING: "ZOOM_CREATE_A_MEETING_TEMPLATE_FROM_AN_EXISTING_MEETING",
    CREATE_A_RECORDING_REGISTRANT: "ZOOM_CREATE_A_RECORDING_REGISTRANT",
    CREATE_A_TRACKING_FIELD: "ZOOM_CREATE_A_TRACKING_FIELD",
    CREATE_A_WEBINAR: "ZOOM_CREATE_A_WEBINAR",
    CREATE_A_WEBINAR_S_BRANDING_NAME_TAG: "ZOOM_CREATE_A_WEBINAR_S_BRANDING_NAME_TAG",
    CREATE_A_WEBINAR_S_POLL: "ZOOM_CREATE_A_WEBINAR_S_POLL",
    CREATE_A_WEBINAR_TEMPLATE: "ZOOM_CREATE_A_WEBINAR_TEMPLATE",
    CREATE_MEETING_S_INVITE_LINKS: "ZOOM_CREATE_MEETING_S_INVITE_LINKS",
    CREATE_WEBINAR_S_INVITE_LINKS: "ZOOM_CREATE_WEBINAR_S_INVITE_LINKS",
    DELETE_A_H_323_SIP_DEVICE: "ZOOM_DELETE_A_H_323_SIP_DEVICE",
    DELETE_A_LIVE_MEETING_MESSAGE: "ZOOM_DELETE_A_LIVE_MEETING_MESSAGE",
    DELETE_A_LIVE_WEBINAR_MESSAGE: "ZOOM_DELETE_A_LIVE_WEBINAR_MESSAGE",
    DELETE_A_MEETING: "ZOOM_DELETE_A_MEETING",
    DELETE_A_MEETING_POLL: "ZOOM_DELETE_A_MEETING_POLL",
    DELETE_A_MEETING_RECORDING_FILE: "ZOOM_DELETE_A_MEETING_RECORDING_FILE",
    DELETE_A_MEETING_REGISTRANT: "ZOOM_DELETE_A_MEETING_REGISTRANT",
    DELETE_A_MEETING_SURVEY: "ZOOM_DELETE_A_MEETING_SURVEY",
    DELETE_A_MEETING_S_ARCHIVED_FILES: "ZOOM_DELETE_A_MEETING_S_ARCHIVED_FILES",
    DELETE_A_TRACKING_FIELD: "ZOOM_DELETE_A_TRACKING_FIELD",
    DELETE_A_USER_S_TSP_ACCOUNT: "ZOOM_DELETE_A_USER_S_TSP_ACCOUNT",
    DELETE_A_WEBINAR: "ZOOM_DELETE_A_WEBINAR",
    DELETE_A_WEBINAR_POLL: "ZOOM_DELETE_A_WEBINAR_POLL",
    DELETE_A_WEBINAR_REGISTRANT: "ZOOM_DELETE_A_WEBINAR_REGISTRANT",
    DELETE_A_WEBINAR_SURVEY: "ZOOM_DELETE_A_WEBINAR_SURVEY",
    DELETE_A_WEBINAR_S_BRANDING_NAME_TAG: "ZOOM_DELETE_A_WEBINAR_S_BRANDING_NAME_TAG",
    DELETE_A_WEBINAR_S_BRANDING_VIRTUAL_BACKGROUNDS: "ZOOM_DELETE_A_WEBINAR_S_BRANDING_VIRTUAL_BACKGROUNDS",
    DELETE_A_WEBINAR_S_BRANDING_WALLPAPER: "ZOOM_DELETE_A_WEBINAR_S_BRANDING_WALLPAPER",
    DELETE_DEVICE: "ZOOM_DELETE_DEVICE",
    DELETE_MEETING_RECORDINGS: "ZOOM_DELETE_MEETING_RECORDINGS",
    DELETE_SIP_PHONE: "ZOOM_DELETE_SIP_PHONE",
    DELETE_ZPA_DEVICE_BY_VENDOR_AND_MAC_ADDRESS: "ZOOM_DELETE_ZPA_DEVICE_BY_VENDOR_AND_MAC_ADDRESS",
    ENABLE_SIP_PHONE: "ZOOM_ENABLE_SIP_PHONE",
    GET_ACCOUNT_S_TSP_INFORMATION: "ZOOM_GET_ACCOUNT_S_TSP_INFORMATION",
    GET_ACTIVE_INACTIVE_HOST_REPORTS: "ZOOM_GET_ACTIVE_INACTIVE_HOST_REPORTS",
    GET_ARCHIVED_FILE_STATISTICS: "ZOOM_GET_ARCHIVED_FILE_STATISTICS",
    GET_A_MEETING: "ZOOM_GET_A_MEETING",
    GET_A_MEETING_POLL: "ZOOM_GET_A_MEETING_POLL",
    GET_A_MEETING_REGISTRANT: "ZOOM_GET_A_MEETING_REGISTRANT",
    GET_A_MEETING_SIP_URI_WITH_PASSCODE: "ZOOM_GET_A_MEETING_SIP_URI_WITH_PASSCODE",
    GET_A_MEETING_SUMMARY: "ZOOM_GET_A_MEETING_SUMMARY",
    GET_A_MEETING_SURVEY: "ZOOM_GET_A_MEETING_SURVEY",
    GET_A_MEETING_S_ARCHIVED_FILES: "ZOOM_GET_A_MEETING_S_ARCHIVED_FILES",
    GET_A_MEETING_S_ARCHIVE_TOKEN_FOR_LOCAL_ARCHIVING: "ZOOM_GET_A_MEETING_S_ARCHIVE_TOKEN_FOR_LOCAL_ARCHIVING",
    GET_A_MEETING_S_JOIN_TOKEN_FOR_LIVE_STREAMING: "ZOOM_GET_A_MEETING_S_JOIN_TOKEN_FOR_LIVE_STREAMING",
    GET_A_MEETING_S_JOIN_TOKEN_FOR_LOCAL_RECORDING: "ZOOM_GET_A_MEETING_S_JOIN_TOKEN_FOR_LOCAL_RECORDING",
    GET_A_TRACKING_FIELD: "ZOOM_GET_A_TRACKING_FIELD",
    GET_A_USER_S_TSP_ACCOUNT: "ZOOM_GET_A_USER_S_TSP_ACCOUNT",
    GET_A_WEBINAR: "ZOOM_GET_A_WEBINAR",
    GET_A_WEBINAR_POLL: "ZOOM_GET_A_WEBINAR_POLL",
    GET_A_WEBINAR_REGISTRANT: "ZOOM_GET_A_WEBINAR_REGISTRANT",
    GET_A_WEBINAR_SIP_URI_WITH_PASSCODE: "ZOOM_GET_A_WEBINAR_SIP_URI_WITH_PASSCODE",
    GET_A_WEBINAR_SURVEY: "ZOOM_GET_A_WEBINAR_SURVEY",
    GET_A_WEBINAR_S_ARCHIVE_TOKEN_FOR_LOCAL_ARCHIVING: "ZOOM_GET_A_WEBINAR_S_ARCHIVE_TOKEN_FOR_LOCAL_ARCHIVING",
    GET_A_WEBINAR_S_JOIN_TOKEN_FOR_LIVE_STREAMING: "ZOOM_GET_A_WEBINAR_S_JOIN_TOKEN_FOR_LIVE_STREAMING",
    GET_A_WEBINAR_S_JOIN_TOKEN_FOR_LOCAL_RECORDING: "ZOOM_GET_A_WEBINAR_S_JOIN_TOKEN_FOR_LOCAL_RECORDING",
    GET_BILLING_INVOICE_REPORTS: "ZOOM_GET_BILLING_INVOICE_REPORTS",
    GET_BILLING_REPORTS: "ZOOM_GET_BILLING_REPORTS",
    GET_CLOUD_RECORDING_USAGE_REPORT: "ZOOM_GET_CLOUD_RECORDING_USAGE_REPORT",
    GET_DAILY_USAGE_REPORT: "ZOOM_GET_DAILY_USAGE_REPORT",
    GET_DEVICE_DETAIL: "ZOOM_GET_DEVICE_DETAIL",
    GET_LIVESTREAM_DETAILS: "ZOOM_GET_LIVESTREAM_DETAILS",
    GET_LIVE_STREAM_DETAILS: "ZOOM_GET_LIVE_STREAM_DETAILS",
    GET_MEETING_DETAIL_REPORTS: "ZOOM_GET_MEETING_DETAIL_REPORTS",
    GET_MEETING_INVITATION: "ZOOM_GET_MEETING_INVITATION",
    GET_MEETING_PARTICIPANT_REPORTS: "ZOOM_GET_MEETING_PARTICIPANT_REPORTS",
    GET_MEETING_POLL_REPORTS: "ZOOM_GET_MEETING_POLL_REPORTS",
    GET_MEETING_Q_A_REPORT: "ZOOM_GET_MEETING_Q_A_REPORT",
    GET_MEETING_RECORDINGS: "ZOOM_GET_MEETING_RECORDINGS",
    GET_MEETING_RECORDING_SETTINGS: "ZOOM_GET_MEETING_RECORDING_SETTINGS",
    GET_MEETING_RECORDING_S_ANALYTICS_DETAILS: "ZOOM_GET_MEETING_RECORDING_S_ANALYTICS_DETAILS",
    GET_MEETING_RECORDING_S_ANALYTICS_SUMMARY: "ZOOM_GET_MEETING_RECORDING_S_ANALYTICS_SUMMARY",
    GET_MEETING_REPORTS: "ZOOM_GET_MEETING_REPORTS",
    GET_MEETING_SURVEY_REPORT: "ZOOM_GET_MEETING_SURVEY_REPORT",
    GET_MEETING_S_TOKEN: "ZOOM_GET_MEETING_S_TOKEN",
    GET_OPERATION_LOGS_REPORT: "ZOOM_GET_OPERATION_LOGS_REPORT",
    GET_PAST_MEETING_DETAILS: "ZOOM_GET_PAST_MEETING_DETAILS",
    GET_PAST_MEETING_PARTICIPANTS: "ZOOM_GET_PAST_MEETING_PARTICIPANTS",
    GET_REGISTRATION_QUESTIONS: "ZOOM_GET_REGISTRATION_QUESTIONS",
    GET_SIGN_IN_SIGN_OUT_ACTIVITY_REPORT: "ZOOM_GET_SIGN_IN_SIGN_OUT_ACTIVITY_REPORT",
    GET_TELEPHONE_REPORTS: "ZOOM_GET_TELEPHONE_REPORTS",
    GET_UPCOMING_EVENTS_REPORT: "ZOOM_GET_UPCOMING_EVENTS_REPORT",
    GET_WEBINAR_ABSENTEES: "ZOOM_GET_WEBINAR_ABSENTEES",
    GET_WEBINAR_DETAIL_REPORTS: "ZOOM_GET_WEBINAR_DETAIL_REPORTS",
    GET_WEBINAR_PARTICIPANT_REPORTS: "ZOOM_GET_WEBINAR_PARTICIPANT_REPORTS",
    GET_WEBINAR_POLL_REPORTS: "ZOOM_GET_WEBINAR_POLL_REPORTS",
    GET_WEBINAR_Q_A_REPORT: "ZOOM_GET_WEBINAR_Q_A_REPORT",
    GET_WEBINAR_SURVEY_REPORT: "ZOOM_GET_WEBINAR_SURVEY_REPORT",
    GET_WEBINAR_S_SESSION_BRANDING: "ZOOM_GET_WEBINAR_S_SESSION_BRANDING",
    GET_WEBINAR_S_TOKEN: "ZOOM_GET_WEBINAR_S_TOKEN",
    GET_WEBINAR_TRACKING_SOURCES: "ZOOM_GET_WEBINAR_TRACKING_SOURCES",
    GET_ZDM_GROUP_INFO: "ZOOM_GET_ZDM_GROUP_INFO",
    GET_ZPA_VERSION_INFO: "ZOOM_GET_ZPA_VERSION_INFO",
    LIST_ALL_RECORDINGS: "ZOOM_LIST_ALL_RECORDINGS",
    LIST_ARCHIVED_FILES: "ZOOM_LIST_ARCHIVED_FILES",
    LIST_A_USER_S_PAC_ACCOUNTS: "ZOOM_LIST_A_USER_S_PAC_ACCOUNTS",
    LIST_A_WEBINAR_S_POLLS: "ZOOM_LIST_A_WEBINAR_S_POLLS",
    LIST_DEVICES: "ZOOM_LIST_DEVICES",
    LIST_H_323_SIP_DEVICES: "ZOOM_LIST_H_323_SIP_DEVICES",
    LIST_MEETINGS: "ZOOM_LIST_MEETINGS",
    LIST_MEETING_POLLS: "ZOOM_LIST_MEETING_POLLS",
    LIST_MEETING_REGISTRANTS: "ZOOM_LIST_MEETING_REGISTRANTS",
    LIST_MEETING_SUMMARIES_OF_AN_ACCOUNT: "ZOOM_LIST_MEETING_SUMMARIES_OF_AN_ACCOUNT",
    LIST_MEETING_TEMPLATES: "ZOOM_LIST_MEETING_TEMPLATES",
    LIST_PANELISTS: "ZOOM_LIST_PANELISTS",
    LIST_PAST_MEETINGS_Q_A: "ZOOM_LIST_PAST_MEETINGS_Q_A",
    LIST_PAST_MEETING_INSTANCES: "ZOOM_LIST_PAST_MEETING_INSTANCES",
    LIST_PAST_MEETING_S_POLL_RESULTS: "ZOOM_LIST_PAST_MEETING_S_POLL_RESULTS",
    LIST_PAST_WEBINAR_INSTANCES: "ZOOM_LIST_PAST_WEBINAR_INSTANCES",
    LIST_PAST_WEBINAR_POLL_RESULTS: "ZOOM_LIST_PAST_WEBINAR_POLL_RESULTS",
    LIST_Q_AS_OF_A_PAST_WEBINAR: "ZOOM_LIST_Q_AS_OF_A_PAST_WEBINAR",
    LIST_RECORDING_REGISTRANTS: "ZOOM_LIST_RECORDING_REGISTRANTS",
    LIST_REGISTRATION_QUESTIONS_FOR_MEETING: "ZOOM_LIST_REGISTRATION_QUESTIONS_FOR_MEETING",
    LIST_REGISTRATION_QUESTIONS_FOR_WEBINAR: "ZOOM_LIST_REGISTRATION_QUESTIONS_FOR_WEBINAR",
    LIST_SIP_PHONES: "ZOOM_LIST_SIP_PHONES",
    LIST_TRACKING_FIELDS: "ZOOM_LIST_TRACKING_FIELDS",
    LIST_UPCOMING_MEETINGS: "ZOOM_LIST_UPCOMING_MEETINGS",
    LIST_USER_S_TSP_ACCOUNTS: "ZOOM_LIST_USER_S_TSP_ACCOUNTS",
    LIST_WEBINARS: "ZOOM_LIST_WEBINARS",
    LIST_WEBINAR_PARTICIPANTS: "ZOOM_LIST_WEBINAR_PARTICIPANTS",
    LIST_WEBINAR_REGISTRANTS: "ZOOM_LIST_WEBINAR_REGISTRANTS",
    LIST_WEBINAR_TEMPLATES: "ZOOM_LIST_WEBINAR_TEMPLATES",
    PERFORM_BATCH_POLL_CREATION: "ZOOM_PERFORM_BATCH_POLL_CREATION",
    PERFORM_BATCH_REGISTRATION: "ZOOM_PERFORM_BATCH_REGISTRATION",
    RECOVER_A_SINGLE_RECORDING: "ZOOM_RECOVER_A_SINGLE_RECORDING",
    RECOVER_MEETING_RECORDINGS: "ZOOM_RECOVER_MEETING_RECORDINGS",
    REMOVE_A_PANELIST: "ZOOM_REMOVE_A_PANELIST",
    REMOVE_WEBINAR_PANELISTS: "ZOOM_REMOVE_WEBINAR_PANELISTS",
    SCHEDULE_USER_WEBINAR_WITH_REGISTRATION: "ZOOM_SCHEDULE_USER_WEBINAR_WITH_REGISTRATION",
    SET_GLOBAL_DIAL_IN_URL_FOR_A_TSP_USER: "ZOOM_SET_GLOBAL_DIAL_IN_URL_FOR_A_TSP_USER",
    SET_WEBINAR_S_DEFAULT_BRANDING_VIRTUAL_BACKGROUND: "ZOOM_SET_WEBINAR_S_DEFAULT_BRANDING_VIRTUAL_BACKGROUND",
    UPDATE_ACCOUNT_S_TSP_INFORMATION: "ZOOM_UPDATE_ACCOUNT_S_TSP_INFORMATION",
    UPDATE_AN_ARCHIVED_FILE_S_AUTO_DELETE_STATUS: "ZOOM_UPDATE_AN_ARCHIVED_FILE_S_AUTO_DELETE_STATUS",
    UPDATE_A_H_323_SIP_DEVICE: "ZOOM_UPDATE_A_H_323_SIP_DEVICE",
    UPDATE_A_LIVESTREAM: "ZOOM_UPDATE_A_LIVESTREAM",
    UPDATE_A_LIVE_MEETING_MESSAGE: "ZOOM_UPDATE_A_LIVE_MEETING_MESSAGE",
    UPDATE_A_LIVE_STREAM: "ZOOM_UPDATE_A_LIVE_STREAM",
    UPDATE_A_MEETING: "ZOOM_UPDATE_A_MEETING",
    UPDATE_A_MEETING_POLL: "ZOOM_UPDATE_A_MEETING_POLL",
    UPDATE_A_MEETING_SURVEY: "ZOOM_UPDATE_A_MEETING_SURVEY",
    UPDATE_A_TRACKING_FIELD: "ZOOM_UPDATE_A_TRACKING_FIELD",
    UPDATE_A_TSP_ACCOUNT: "ZOOM_UPDATE_A_TSP_ACCOUNT",
    UPDATE_A_WEBINAR: "ZOOM_UPDATE_A_WEBINAR",
    UPDATE_A_WEBINAR_POLL: "ZOOM_UPDATE_A_WEBINAR_POLL",
    UPDATE_A_WEBINAR_SURVEY: "ZOOM_UPDATE_A_WEBINAR_SURVEY",
    UPDATE_A_WEBINAR_S_BRANDING_NAME_TAG: "ZOOM_UPDATE_A_WEBINAR_S_BRANDING_NAME_TAG",
    UPDATE_LIVESTREAM_STATUS: "ZOOM_UPDATE_LIVESTREAM_STATUS",
    UPDATE_LIVE_STREAM_STATUS: "ZOOM_UPDATE_LIVE_STREAM_STATUS",
    UPDATE_MEETING_RECORDING_SETTINGS: "ZOOM_UPDATE_MEETING_RECORDING_SETTINGS",
    UPDATE_MEETING_REGISTRANT_STATUS: "ZOOM_UPDATE_MEETING_REGISTRANT_STATUS",
    UPDATE_MEETING_STATUS: "ZOOM_UPDATE_MEETING_STATUS",
    UPDATE_REGISTRANT_S_STATUS: "ZOOM_UPDATE_REGISTRANT_S_STATUS",
    UPDATE_REGISTRATION_QUESTIONS: "ZOOM_UPDATE_REGISTRATION_QUESTIONS",
    UPDATE_SIP_PHONE: "ZOOM_UPDATE_SIP_PHONE",
    UPDATE_WEBINAR_REGISTRANT_STATUS: "ZOOM_UPDATE_WEBINAR_REGISTRANT_STATUS",
    UPDATE_WEBINAR_REGISTRATION_FIELDS: "ZOOM_UPDATE_WEBINAR_REGISTRATION_FIELDS",
    UPDATE_WEBINAR_STATUS: "ZOOM_UPDATE_WEBINAR_STATUS",
    UPDATE_ZOOM_MEETING_REGISTRATION_QUESTIONS: "ZOOM_UPDATE_ZOOM_MEETING_REGISTRATION_QUESTIONS",
    UPGRADE_ZPA_OS_APP: "ZOOM_UPGRADE_ZPA_OS_APP",
    UPLOAD_A_WEBINAR_S_BRANDING_VIRTUAL_BACKGROUND: "ZOOM_UPLOAD_A_WEBINAR_S_BRANDING_VIRTUAL_BACKGROUND",
    UPLOAD_A_WEBINAR_S_BRANDING_WALLPAPER: "ZOOM_UPLOAD_A_WEBINAR_S_BRANDING_WALLPAPER",
    USE_IN_MEETING_CONTROLS: "ZOOM_USE_IN_MEETING_CONTROLS",
  },
  triggerTypes: {},
}
